<!--
  ~ The Exomiser - A tool to annotate and prioritize genomic variants
  ~
  ~ Copyright (c) 2016-2019 Queen Mary University of London.
  ~ Copyright (c) 2012-2016 Charité Universitätsmedizin Berlin and Genome Research Ltd.
  ~
  ~ This program is free software: you can redistribute it and/or modify
  ~ it under the terms of the GNU Affero General Public License as
  ~ published by the Free Software Foundation, either version 3 of the
  ~ License, or (at your option) any later version.
  ~
  ~ This program is distributed in the hope that it will be useful,
  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  ~ GNU Affero General Public License for more details.
  ~
  ~ You should have received a copy of the GNU Affero General Public License
  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  -->

<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>
    <title>The Exomiser - A Tool to Annotate and Prioritise Whole-Exome Sequencing Data</title>
    <meta charset="UTF-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <style>
        /*!
 * Bootstrap v3.3.7 (http://getbootstrap.com)
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 *//*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */html{font-family:sans-serif;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block;vertical-align:baseline}audio:not([controls]){display:none;height:0}[hidden],template{display:none}a{background-color:transparent}a:active,a:hover{outline:0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:700}dfn{font-style:italic}h1{margin:.67em 0;font-size:2em}mark{color:#000;background:#ff0}small{font-size:80%}sub,sup{position:relative;font-size:75%;line-height:0;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}img{border:0}svg:not(:root){overflow:hidden}figure{margin:1em 40px}hr{height:0;-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}pre{overflow:auto}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}button,input,optgroup,select,textarea{margin:0;font:inherit;color:inherit}button{overflow:visible}button,select{text-transform:none}button,html input[type=button],input[type=reset],input[type=submit]{-webkit-appearance:button;cursor:pointer}button[disabled],html input[disabled]{cursor:default}button::-moz-focus-inner,input::-moz-focus-inner{padding:0;border:0}input{line-height:normal}input[type=checkbox],input[type=radio]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;padding:0}input[type=number]::-webkit-inner-spin-button,input[type=number]::-webkit-outer-spin-button{height:auto}input[type=search]{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;-webkit-appearance:textfield}input[type=search]::-webkit-search-cancel-button,input[type=search]::-webkit-search-decoration{-webkit-appearance:none}fieldset{padding:.35em .625em .75em;margin:0 2px;border:1px solid silver}legend{padding:0;border:0}textarea{overflow:auto}optgroup{font-weight:700}table{border-spacing:0;border-collapse:collapse}td,th{padding:0}/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */@media print{*,:after,:before{color:#000!important;text-shadow:none!important;background:0 0!important;-webkit-box-shadow:none!important;box-shadow:none!important}a,a:visited{text-decoration:underline}a[href]:after{content:" (" attr(href) ")"}abbr[title]:after{content:" (" attr(title) ")"}a[href^="javascript:"]:after,a[href^="#"]:after{content:""}blockquote,pre{border:1px solid #999;page-break-inside:avoid}thead{display:table-header-group}img,tr{page-break-inside:avoid}img{max-width:100%!important}h2,h3,p{orphans:3;widows:3}h2,h3{page-break-after:avoid}.navbar{display:none}.btn>.caret,.dropup>.btn>.caret{border-top-color:#000!important}.label{border:1px solid #000}.table{border-collapse:collapse!important}.table td,.table th{background-color:#fff!important}.table-bordered td,.table-bordered th{border:1px solid #ddd!important}}@font-face{font-family:'Glyphicons Halflings';src:url(../fonts/glyphicons-halflings-regular.eot);src:url(../fonts/glyphicons-halflings-regular.eot?#iefix) format('embedded-opentype'),url(../fonts/glyphicons-halflings-regular.woff2) format('woff2'),url(../fonts/glyphicons-halflings-regular.woff) format('woff'),url(../fonts/glyphicons-halflings-regular.ttf) format('truetype'),url(../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular) format('svg')}.glyphicon{position:relative;top:1px;display:inline-block;font-family:'Glyphicons Halflings';font-style:normal;font-weight:400;line-height:1;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.glyphicon-asterisk:before{content:"\002a"}.glyphicon-plus:before{content:"\002b"}.glyphicon-eur:before,.glyphicon-euro:before{content:"\20ac"}.glyphicon-minus:before{content:"\2212"}.glyphicon-cloud:before{content:"\2601"}.glyphicon-envelope:before{content:"\2709"}.glyphicon-pencil:before{content:"\270f"}.glyphicon-glass:before{content:"\e001"}.glyphicon-music:before{content:"\e002"}.glyphicon-search:before{content:"\e003"}.glyphicon-heart:before{content:"\e005"}.glyphicon-star:before{content:"\e006"}.glyphicon-star-empty:before{content:"\e007"}.glyphicon-user:before{content:"\e008"}.glyphicon-film:before{content:"\e009"}.glyphicon-th-large:before{content:"\e010"}.glyphicon-th:before{content:"\e011"}.glyphicon-th-list:before{content:"\e012"}.glyphicon-ok:before{content:"\e013"}.glyphicon-remove:before{content:"\e014"}.glyphicon-zoom-in:before{content:"\e015"}.glyphicon-zoom-out:before{content:"\e016"}.glyphicon-off:before{content:"\e017"}.glyphicon-signal:before{content:"\e018"}.glyphicon-cog:before{content:"\e019"}.glyphicon-trash:before{content:"\e020"}.glyphicon-home:before{content:"\e021"}.glyphicon-file:before{content:"\e022"}.glyphicon-time:before{content:"\e023"}.glyphicon-road:before{content:"\e024"}.glyphicon-download-alt:before{content:"\e025"}.glyphicon-download:before{content:"\e026"}.glyphicon-upload:before{content:"\e027"}.glyphicon-inbox:before{content:"\e028"}.glyphicon-play-circle:before{content:"\e029"}.glyphicon-repeat:before{content:"\e030"}.glyphicon-refresh:before{content:"\e031"}.glyphicon-list-alt:before{content:"\e032"}.glyphicon-lock:before{content:"\e033"}.glyphicon-flag:before{content:"\e034"}.glyphicon-headphones:before{content:"\e035"}.glyphicon-volume-off:before{content:"\e036"}.glyphicon-volume-down:before{content:"\e037"}.glyphicon-volume-up:before{content:"\e038"}.glyphicon-qrcode:before{content:"\e039"}.glyphicon-barcode:before{content:"\e040"}.glyphicon-tag:before{content:"\e041"}.glyphicon-tags:before{content:"\e042"}.glyphicon-book:before{content:"\e043"}.glyphicon-bookmark:before{content:"\e044"}.glyphicon-print:before{content:"\e045"}.glyphicon-camera:before{content:"\e046"}.glyphicon-font:before{content:"\e047"}.glyphicon-bold:before{content:"\e048"}.glyphicon-italic:before{content:"\e049"}.glyphicon-text-height:before{content:"\e050"}.glyphicon-text-width:before{content:"\e051"}.glyphicon-align-left:before{content:"\e052"}.glyphicon-align-center:before{content:"\e053"}.glyphicon-align-right:before{content:"\e054"}.glyphicon-align-justify:before{content:"\e055"}.glyphicon-list:before{content:"\e056"}.glyphicon-indent-left:before{content:"\e057"}.glyphicon-indent-right:before{content:"\e058"}.glyphicon-facetime-video:before{content:"\e059"}.glyphicon-picture:before{content:"\e060"}.glyphicon-map-marker:before{content:"\e062"}.glyphicon-adjust:before{content:"\e063"}.glyphicon-tint:before{content:"\e064"}.glyphicon-edit:before{content:"\e065"}.glyphicon-share:before{content:"\e066"}.glyphicon-check:before{content:"\e067"}.glyphicon-move:before{content:"\e068"}.glyphicon-step-backward:before{content:"\e069"}.glyphicon-fast-backward:before{content:"\e070"}.glyphicon-backward:before{content:"\e071"}.glyphicon-play:before{content:"\e072"}.glyphicon-pause:before{content:"\e073"}.glyphicon-stop:before{content:"\e074"}.glyphicon-forward:before{content:"\e075"}.glyphicon-fast-forward:before{content:"\e076"}.glyphicon-step-forward:before{content:"\e077"}.glyphicon-eject:before{content:"\e078"}.glyphicon-chevron-left:before{content:"\e079"}.glyphicon-chevron-right:before{content:"\e080"}.glyphicon-plus-sign:before{content:"\e081"}.glyphicon-minus-sign:before{content:"\e082"}.glyphicon-remove-sign:before{content:"\e083"}.glyphicon-ok-sign:before{content:"\e084"}.glyphicon-question-sign:before{content:"\e085"}.glyphicon-info-sign:before{content:"\e086"}.glyphicon-screenshot:before{content:"\e087"}.glyphicon-remove-circle:before{content:"\e088"}.glyphicon-ok-circle:before{content:"\e089"}.glyphicon-ban-circle:before{content:"\e090"}.glyphicon-arrow-left:before{content:"\e091"}.glyphicon-arrow-right:before{content:"\e092"}.glyphicon-arrow-up:before{content:"\e093"}.glyphicon-arrow-down:before{content:"\e094"}.glyphicon-share-alt:before{content:"\e095"}.glyphicon-resize-full:before{content:"\e096"}.glyphicon-resize-small:before{content:"\e097"}.glyphicon-exclamation-sign:before{content:"\e101"}.glyphicon-gift:before{content:"\e102"}.glyphicon-leaf:before{content:"\e103"}.glyphicon-fire:before{content:"\e104"}.glyphicon-eye-open:before{content:"\e105"}.glyphicon-eye-close:before{content:"\e106"}.glyphicon-warning-sign:before{content:"\e107"}.glyphicon-plane:before{content:"\e108"}.glyphicon-calendar:before{content:"\e109"}.glyphicon-random:before{content:"\e110"}.glyphicon-comment:before{content:"\e111"}.glyphicon-magnet:before{content:"\e112"}.glyphicon-chevron-up:before{content:"\e113"}.glyphicon-chevron-down:before{content:"\e114"}.glyphicon-retweet:before{content:"\e115"}.glyphicon-shopping-cart:before{content:"\e116"}.glyphicon-folder-close:before{content:"\e117"}.glyphicon-folder-open:before{content:"\e118"}.glyphicon-resize-vertical:before{content:"\e119"}.glyphicon-resize-horizontal:before{content:"\e120"}.glyphicon-hdd:before{content:"\e121"}.glyphicon-bullhorn:before{content:"\e122"}.glyphicon-bell:before{content:"\e123"}.glyphicon-certificate:before{content:"\e124"}.glyphicon-thumbs-up:before{content:"\e125"}.glyphicon-thumbs-down:before{content:"\e126"}.glyphicon-hand-right:before{content:"\e127"}.glyphicon-hand-left:before{content:"\e128"}.glyphicon-hand-up:before{content:"\e129"}.glyphicon-hand-down:before{content:"\e130"}.glyphicon-circle-arrow-right:before{content:"\e131"}.glyphicon-circle-arrow-left:before{content:"\e132"}.glyphicon-circle-arrow-up:before{content:"\e133"}.glyphicon-circle-arrow-down:before{content:"\e134"}.glyphicon-globe:before{content:"\e135"}.glyphicon-wrench:before{content:"\e136"}.glyphicon-tasks:before{content:"\e137"}.glyphicon-filter:before{content:"\e138"}.glyphicon-briefcase:before{content:"\e139"}.glyphicon-fullscreen:before{content:"\e140"}.glyphicon-dashboard:before{content:"\e141"}.glyphicon-paperclip:before{content:"\e142"}.glyphicon-heart-empty:before{content:"\e143"}.glyphicon-link:before{content:"\e144"}.glyphicon-phone:before{content:"\e145"}.glyphicon-pushpin:before{content:"\e146"}.glyphicon-usd:before{content:"\e148"}.glyphicon-gbp:before{content:"\e149"}.glyphicon-sort:before{content:"\e150"}.glyphicon-sort-by-alphabet:before{content:"\e151"}.glyphicon-sort-by-alphabet-alt:before{content:"\e152"}.glyphicon-sort-by-order:before{content:"\e153"}.glyphicon-sort-by-order-alt:before{content:"\e154"}.glyphicon-sort-by-attributes:before{content:"\e155"}.glyphicon-sort-by-attributes-alt:before{content:"\e156"}.glyphicon-unchecked:before{content:"\e157"}.glyphicon-expand:before{content:"\e158"}.glyphicon-collapse-down:before{content:"\e159"}.glyphicon-collapse-up:before{content:"\e160"}.glyphicon-log-in:before{content:"\e161"}.glyphicon-flash:before{content:"\e162"}.glyphicon-log-out:before{content:"\e163"}.glyphicon-new-window:before{content:"\e164"}.glyphicon-record:before{content:"\e165"}.glyphicon-save:before{content:"\e166"}.glyphicon-open:before{content:"\e167"}.glyphicon-saved:before{content:"\e168"}.glyphicon-import:before{content:"\e169"}.glyphicon-export:before{content:"\e170"}.glyphicon-send:before{content:"\e171"}.glyphicon-floppy-disk:before{content:"\e172"}.glyphicon-floppy-saved:before{content:"\e173"}.glyphicon-floppy-remove:before{content:"\e174"}.glyphicon-floppy-save:before{content:"\e175"}.glyphicon-floppy-open:before{content:"\e176"}.glyphicon-credit-card:before{content:"\e177"}.glyphicon-transfer:before{content:"\e178"}.glyphicon-cutlery:before{content:"\e179"}.glyphicon-header:before{content:"\e180"}.glyphicon-compressed:before{content:"\e181"}.glyphicon-earphone:before{content:"\e182"}.glyphicon-phone-alt:before{content:"\e183"}.glyphicon-tower:before{content:"\e184"}.glyphicon-stats:before{content:"\e185"}.glyphicon-sd-video:before{content:"\e186"}.glyphicon-hd-video:before{content:"\e187"}.glyphicon-subtitles:before{content:"\e188"}.glyphicon-sound-stereo:before{content:"\e189"}.glyphicon-sound-dolby:before{content:"\e190"}.glyphicon-sound-5-1:before{content:"\e191"}.glyphicon-sound-6-1:before{content:"\e192"}.glyphicon-sound-7-1:before{content:"\e193"}.glyphicon-copyright-mark:before{content:"\e194"}.glyphicon-registration-mark:before{content:"\e195"}.glyphicon-cloud-download:before{content:"\e197"}.glyphicon-cloud-upload:before{content:"\e198"}.glyphicon-tree-conifer:before{content:"\e199"}.glyphicon-tree-deciduous:before{content:"\e200"}.glyphicon-cd:before{content:"\e201"}.glyphicon-save-file:before{content:"\e202"}.glyphicon-open-file:before{content:"\e203"}.glyphicon-level-up:before{content:"\e204"}.glyphicon-copy:before{content:"\e205"}.glyphicon-paste:before{content:"\e206"}.glyphicon-alert:before{content:"\e209"}.glyphicon-equalizer:before{content:"\e210"}.glyphicon-king:before{content:"\e211"}.glyphicon-queen:before{content:"\e212"}.glyphicon-pawn:before{content:"\e213"}.glyphicon-bishop:before{content:"\e214"}.glyphicon-knight:before{content:"\e215"}.glyphicon-baby-formula:before{content:"\e216"}.glyphicon-tent:before{content:"\26fa"}.glyphicon-blackboard:before{content:"\e218"}.glyphicon-bed:before{content:"\e219"}.glyphicon-apple:before{content:"\f8ff"}.glyphicon-erase:before{content:"\e221"}.glyphicon-hourglass:before{content:"\231b"}.glyphicon-lamp:before{content:"\e223"}.glyphicon-duplicate:before{content:"\e224"}.glyphicon-piggy-bank:before{content:"\e225"}.glyphicon-scissors:before{content:"\e226"}.glyphicon-bitcoin:before{content:"\e227"}.glyphicon-btc:before{content:"\e227"}.glyphicon-xbt:before{content:"\e227"}.glyphicon-yen:before{content:"\00a5"}.glyphicon-jpy:before{content:"\00a5"}.glyphicon-ruble:before{content:"\20bd"}.glyphicon-rub:before{content:"\20bd"}.glyphicon-scale:before{content:"\e230"}.glyphicon-ice-lolly:before{content:"\e231"}.glyphicon-ice-lolly-tasted:before{content:"\e232"}.glyphicon-education:before{content:"\e233"}.glyphicon-option-horizontal:before{content:"\e234"}.glyphicon-option-vertical:before{content:"\e235"}.glyphicon-menu-hamburger:before{content:"\e236"}.glyphicon-modal-window:before{content:"\e237"}.glyphicon-oil:before{content:"\e238"}.glyphicon-grain:before{content:"\e239"}.glyphicon-sunglasses:before{content:"\e240"}.glyphicon-text-size:before{content:"\e241"}.glyphicon-text-color:before{content:"\e242"}.glyphicon-text-background:before{content:"\e243"}.glyphicon-object-align-top:before{content:"\e244"}.glyphicon-object-align-bottom:before{content:"\e245"}.glyphicon-object-align-horizontal:before{content:"\e246"}.glyphicon-object-align-left:before{content:"\e247"}.glyphicon-object-align-vertical:before{content:"\e248"}.glyphicon-object-align-right:before{content:"\e249"}.glyphicon-triangle-right:before{content:"\e250"}.glyphicon-triangle-left:before{content:"\e251"}.glyphicon-triangle-bottom:before{content:"\e252"}.glyphicon-triangle-top:before{content:"\e253"}.glyphicon-console:before{content:"\e254"}.glyphicon-superscript:before{content:"\e255"}.glyphicon-subscript:before{content:"\e256"}.glyphicon-menu-left:before{content:"\e257"}.glyphicon-menu-right:before{content:"\e258"}.glyphicon-menu-down:before{content:"\e259"}.glyphicon-menu-up:before{content:"\e260"}*{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}:after,:before{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}html{font-size:10px;-webkit-tap-highlight-color:rgba(0,0,0,0)}body{font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;line-height:1.42857143;color:#333;background-color:#fff}button,input,select,textarea{font-family:inherit;font-size:inherit;line-height:inherit}a{color:#337ab7;text-decoration:none}a:focus,a:hover{color:#23527c;text-decoration:underline}a:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}figure{margin:0}img{vertical-align:middle}.carousel-inner>.item>a>img,.carousel-inner>.item>img,.img-responsive,.thumbnail a>img,.thumbnail>img{display:block;max-width:100%;height:auto}.img-rounded{border-radius:6px}.img-thumbnail{display:inline-block;max-width:100%;height:auto;padding:4px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:all .2s ease-in-out;-o-transition:all .2s ease-in-out;transition:all .2s ease-in-out}.img-circle{border-radius:50%}hr{margin-top:20px;margin-bottom:20px;border:0;border-top:1px solid #eee}.sr-only{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0,0,0,0);border:0}.sr-only-focusable:active,.sr-only-focusable:focus{position:static;width:auto;height:auto;margin:0;overflow:visible;clip:auto}[role=button]{cursor:pointer}.h1,.h2,.h3,.h4,.h5,.h6,h1,h2,h3,h4,h5,h6{font-family:inherit;font-weight:500;line-height:1.1;color:inherit}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-weight:400;line-height:1;color:#777}.h1,.h2,.h3,h1,h2,h3{margin-top:20px;margin-bottom:10px}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small{font-size:65%}.h4,.h5,.h6,h4,h5,h6{margin-top:10px;margin-bottom:10px}.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-size:75%}.h1,h1{font-size:36px}.h2,h2{font-size:30px}.h3,h3{font-size:24px}.h4,h4{font-size:18px}.h5,h5{font-size:14px}.h6,h6{font-size:12px}p{margin:0 0 10px}.lead{margin-bottom:20px;font-size:16px;font-weight:300;line-height:1.4}@media (min-width:768px){.lead{font-size:21px}}.small,small{font-size:85%}.mark,mark{padding:.2em;background-color:#fcf8e3}.text-left{text-align:left}.text-right{text-align:right}.text-center{text-align:center}.text-justify{text-align:justify}.text-nowrap{white-space:nowrap}.text-lowercase{text-transform:lowercase}.text-uppercase{text-transform:uppercase}.text-capitalize{text-transform:capitalize}.text-muted{color:#777}.text-primary{color:#337ab7}a.text-primary:focus,a.text-primary:hover{color:#286090}.text-success{color:#3c763d}a.text-success:focus,a.text-success:hover{color:#2b542c}.text-info{color:#31708f}a.text-info:focus,a.text-info:hover{color:#245269}.text-warning{color:#8a6d3b}a.text-warning:focus,a.text-warning:hover{color:#66512c}.text-danger{color:#a94442}a.text-danger:focus,a.text-danger:hover{color:#843534}.bg-primary{color:#fff;background-color:#337ab7}a.bg-primary:focus,a.bg-primary:hover{background-color:#286090}.bg-success{background-color:#dff0d8}a.bg-success:focus,a.bg-success:hover{background-color:#c1e2b3}.bg-info{background-color:#d9edf7}a.bg-info:focus,a.bg-info:hover{background-color:#afd9ee}.bg-warning{background-color:#fcf8e3}a.bg-warning:focus,a.bg-warning:hover{background-color:#f7ecb5}.bg-danger{background-color:#f2dede}a.bg-danger:focus,a.bg-danger:hover{background-color:#e4b9b9}.page-header{padding-bottom:9px;margin:40px 0 20px;border-bottom:1px solid #eee}ol,ul{margin-top:0;margin-bottom:10px}ol ol,ol ul,ul ol,ul ul{margin-bottom:0}.list-unstyled{padding-left:0;list-style:none}.list-inline{padding-left:0;margin-left:-5px;list-style:none}.list-inline>li{display:inline-block;padding-right:5px;padding-left:5px}dl{margin-top:0;margin-bottom:20px}dd,dt{line-height:1.42857143}dt{font-weight:700}dd{margin-left:0}@media (min-width:768px){.dl-horizontal dt{float:left;width:160px;overflow:hidden;clear:left;text-align:right;text-overflow:ellipsis;white-space:nowrap}.dl-horizontal dd{margin-left:180px}}abbr[data-original-title],abbr[title]{cursor:help;border-bottom:1px dotted #777}.initialism{font-size:90%;text-transform:uppercase}blockquote{padding:10px 20px;margin:0 0 20px;font-size:17.5px;border-left:5px solid #eee}blockquote ol:last-child,blockquote p:last-child,blockquote ul:last-child{margin-bottom:0}blockquote .small,blockquote footer,blockquote small{display:block;font-size:80%;line-height:1.42857143;color:#777}blockquote .small:before,blockquote footer:before,blockquote small:before{content:'\2014 \00A0'}.blockquote-reverse,blockquote.pull-right{padding-right:15px;padding-left:0;text-align:right;border-right:5px solid #eee;border-left:0}.blockquote-reverse .small:before,.blockquote-reverse footer:before,.blockquote-reverse small:before,blockquote.pull-right .small:before,blockquote.pull-right footer:before,blockquote.pull-right small:before{content:''}.blockquote-reverse .small:after,.blockquote-reverse footer:after,.blockquote-reverse small:after,blockquote.pull-right .small:after,blockquote.pull-right footer:after,blockquote.pull-right small:after{content:'\00A0 \2014'}address{margin-bottom:20px;font-style:normal;line-height:1.42857143}code,kbd,pre,samp{font-family:Menlo,Monaco,Consolas,"Courier New",monospace}code{padding:2px 4px;font-size:90%;color:#c7254e;background-color:#f9f2f4;border-radius:4px}kbd{padding:2px 4px;font-size:90%;color:#fff;background-color:#333;border-radius:3px;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.25);box-shadow:inset 0 -1px 0 rgba(0,0,0,.25)}kbd kbd{padding:0;font-size:100%;font-weight:700;-webkit-box-shadow:none;box-shadow:none}pre{display:block;padding:9.5px;margin:0 0 10px;font-size:13px;line-height:1.42857143;color:#333;word-break:break-all;word-wrap:break-word;background-color:#f5f5f5;border:1px solid #ccc;border-radius:4px}pre code{padding:0;font-size:inherit;color:inherit;white-space:pre-wrap;background-color:transparent;border-radius:0}.pre-scrollable{max-height:340px;overflow-y:scroll}.container{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}@media (min-width:768px){.container{width:750px}}@media (min-width:992px){.container{width:970px}}@media (min-width:1200px){.container{width:1170px}}.container-fluid{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}.row{margin-right:-15px;margin-left:-15px}.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9,.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9,.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9,.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{position:relative;min-height:1px;padding-right:15px;padding-left:15px}.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{float:left}.col-xs-12{width:100%}.col-xs-11{width:91.66666667%}.col-xs-10{width:83.33333333%}.col-xs-9{width:75%}.col-xs-8{width:66.66666667%}.col-xs-7{width:58.33333333%}.col-xs-6{width:50%}.col-xs-5{width:41.66666667%}.col-xs-4{width:33.33333333%}.col-xs-3{width:25%}.col-xs-2{width:16.66666667%}.col-xs-1{width:8.33333333%}.col-xs-pull-12{right:100%}.col-xs-pull-11{right:91.66666667%}.col-xs-pull-10{right:83.33333333%}.col-xs-pull-9{right:75%}.col-xs-pull-8{right:66.66666667%}.col-xs-pull-7{right:58.33333333%}.col-xs-pull-6{right:50%}.col-xs-pull-5{right:41.66666667%}.col-xs-pull-4{right:33.33333333%}.col-xs-pull-3{right:25%}.col-xs-pull-2{right:16.66666667%}.col-xs-pull-1{right:8.33333333%}.col-xs-pull-0{right:auto}.col-xs-push-12{left:100%}.col-xs-push-11{left:91.66666667%}.col-xs-push-10{left:83.33333333%}.col-xs-push-9{left:75%}.col-xs-push-8{left:66.66666667%}.col-xs-push-7{left:58.33333333%}.col-xs-push-6{left:50%}.col-xs-push-5{left:41.66666667%}.col-xs-push-4{left:33.33333333%}.col-xs-push-3{left:25%}.col-xs-push-2{left:16.66666667%}.col-xs-push-1{left:8.33333333%}.col-xs-push-0{left:auto}.col-xs-offset-12{margin-left:100%}.col-xs-offset-11{margin-left:91.66666667%}.col-xs-offset-10{margin-left:83.33333333%}.col-xs-offset-9{margin-left:75%}.col-xs-offset-8{margin-left:66.66666667%}.col-xs-offset-7{margin-left:58.33333333%}.col-xs-offset-6{margin-left:50%}.col-xs-offset-5{margin-left:41.66666667%}.col-xs-offset-4{margin-left:33.33333333%}.col-xs-offset-3{margin-left:25%}.col-xs-offset-2{margin-left:16.66666667%}.col-xs-offset-1{margin-left:8.33333333%}.col-xs-offset-0{margin-left:0}@media (min-width:768px){.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9{float:left}.col-sm-12{width:100%}.col-sm-11{width:91.66666667%}.col-sm-10{width:83.33333333%}.col-sm-9{width:75%}.col-sm-8{width:66.66666667%}.col-sm-7{width:58.33333333%}.col-sm-6{width:50%}.col-sm-5{width:41.66666667%}.col-sm-4{width:33.33333333%}.col-sm-3{width:25%}.col-sm-2{width:16.66666667%}.col-sm-1{width:8.33333333%}.col-sm-pull-12{right:100%}.col-sm-pull-11{right:91.66666667%}.col-sm-pull-10{right:83.33333333%}.col-sm-pull-9{right:75%}.col-sm-pull-8{right:66.66666667%}.col-sm-pull-7{right:58.33333333%}.col-sm-pull-6{right:50%}.col-sm-pull-5{right:41.66666667%}.col-sm-pull-4{right:33.33333333%}.col-sm-pull-3{right:25%}.col-sm-pull-2{right:16.66666667%}.col-sm-pull-1{right:8.33333333%}.col-sm-pull-0{right:auto}.col-sm-push-12{left:100%}.col-sm-push-11{left:91.66666667%}.col-sm-push-10{left:83.33333333%}.col-sm-push-9{left:75%}.col-sm-push-8{left:66.66666667%}.col-sm-push-7{left:58.33333333%}.col-sm-push-6{left:50%}.col-sm-push-5{left:41.66666667%}.col-sm-push-4{left:33.33333333%}.col-sm-push-3{left:25%}.col-sm-push-2{left:16.66666667%}.col-sm-push-1{left:8.33333333%}.col-sm-push-0{left:auto}.col-sm-offset-12{margin-left:100%}.col-sm-offset-11{margin-left:91.66666667%}.col-sm-offset-10{margin-left:83.33333333%}.col-sm-offset-9{margin-left:75%}.col-sm-offset-8{margin-left:66.66666667%}.col-sm-offset-7{margin-left:58.33333333%}.col-sm-offset-6{margin-left:50%}.col-sm-offset-5{margin-left:41.66666667%}.col-sm-offset-4{margin-left:33.33333333%}.col-sm-offset-3{margin-left:25%}.col-sm-offset-2{margin-left:16.66666667%}.col-sm-offset-1{margin-left:8.33333333%}.col-sm-offset-0{margin-left:0}}@media (min-width:992px){.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9{float:left}.col-md-12{width:100%}.col-md-11{width:91.66666667%}.col-md-10{width:83.33333333%}.col-md-9{width:75%}.col-md-8{width:66.66666667%}.col-md-7{width:58.33333333%}.col-md-6{width:50%}.col-md-5{width:41.66666667%}.col-md-4{width:33.33333333%}.col-md-3{width:25%}.col-md-2{width:16.66666667%}.col-md-1{width:8.33333333%}.col-md-pull-12{right:100%}.col-md-pull-11{right:91.66666667%}.col-md-pull-10{right:83.33333333%}.col-md-pull-9{right:75%}.col-md-pull-8{right:66.66666667%}.col-md-pull-7{right:58.33333333%}.col-md-pull-6{right:50%}.col-md-pull-5{right:41.66666667%}.col-md-pull-4{right:33.33333333%}.col-md-pull-3{right:25%}.col-md-pull-2{right:16.66666667%}.col-md-pull-1{right:8.33333333%}.col-md-pull-0{right:auto}.col-md-push-12{left:100%}.col-md-push-11{left:91.66666667%}.col-md-push-10{left:83.33333333%}.col-md-push-9{left:75%}.col-md-push-8{left:66.66666667%}.col-md-push-7{left:58.33333333%}.col-md-push-6{left:50%}.col-md-push-5{left:41.66666667%}.col-md-push-4{left:33.33333333%}.col-md-push-3{left:25%}.col-md-push-2{left:16.66666667%}.col-md-push-1{left:8.33333333%}.col-md-push-0{left:auto}.col-md-offset-12{margin-left:100%}.col-md-offset-11{margin-left:91.66666667%}.col-md-offset-10{margin-left:83.33333333%}.col-md-offset-9{margin-left:75%}.col-md-offset-8{margin-left:66.66666667%}.col-md-offset-7{margin-left:58.33333333%}.col-md-offset-6{margin-left:50%}.col-md-offset-5{margin-left:41.66666667%}.col-md-offset-4{margin-left:33.33333333%}.col-md-offset-3{margin-left:25%}.col-md-offset-2{margin-left:16.66666667%}.col-md-offset-1{margin-left:8.33333333%}.col-md-offset-0{margin-left:0}}@media (min-width:1200px){.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9{float:left}.col-lg-12{width:100%}.col-lg-11{width:91.66666667%}.col-lg-10{width:83.33333333%}.col-lg-9{width:75%}.col-lg-8{width:66.66666667%}.col-lg-7{width:58.33333333%}.col-lg-6{width:50%}.col-lg-5{width:41.66666667%}.col-lg-4{width:33.33333333%}.col-lg-3{width:25%}.col-lg-2{width:16.66666667%}.col-lg-1{width:8.33333333%}.col-lg-pull-12{right:100%}.col-lg-pull-11{right:91.66666667%}.col-lg-pull-10{right:83.33333333%}.col-lg-pull-9{right:75%}.col-lg-pull-8{right:66.66666667%}.col-lg-pull-7{right:58.33333333%}.col-lg-pull-6{right:50%}.col-lg-pull-5{right:41.66666667%}.col-lg-pull-4{right:33.33333333%}.col-lg-pull-3{right:25%}.col-lg-pull-2{right:16.66666667%}.col-lg-pull-1{right:8.33333333%}.col-lg-pull-0{right:auto}.col-lg-push-12{left:100%}.col-lg-push-11{left:91.66666667%}.col-lg-push-10{left:83.33333333%}.col-lg-push-9{left:75%}.col-lg-push-8{left:66.66666667%}.col-lg-push-7{left:58.33333333%}.col-lg-push-6{left:50%}.col-lg-push-5{left:41.66666667%}.col-lg-push-4{left:33.33333333%}.col-lg-push-3{left:25%}.col-lg-push-2{left:16.66666667%}.col-lg-push-1{left:8.33333333%}.col-lg-push-0{left:auto}.col-lg-offset-12{margin-left:100%}.col-lg-offset-11{margin-left:91.66666667%}.col-lg-offset-10{margin-left:83.33333333%}.col-lg-offset-9{margin-left:75%}.col-lg-offset-8{margin-left:66.66666667%}.col-lg-offset-7{margin-left:58.33333333%}.col-lg-offset-6{margin-left:50%}.col-lg-offset-5{margin-left:41.66666667%}.col-lg-offset-4{margin-left:33.33333333%}.col-lg-offset-3{margin-left:25%}.col-lg-offset-2{margin-left:16.66666667%}.col-lg-offset-1{margin-left:8.33333333%}.col-lg-offset-0{margin-left:0}}table{background-color:transparent}caption{padding-top:8px;padding-bottom:8px;color:#777;text-align:left}th{text-align:left}.table{width:100%;max-width:100%;margin-bottom:20px}.table>tbody>tr>td,.table>tbody>tr>th,.table>tfoot>tr>td,.table>tfoot>tr>th,.table>thead>tr>td,.table>thead>tr>th{padding:8px;line-height:1.42857143;vertical-align:top;border-top:1px solid #ddd}.table>thead>tr>th{vertical-align:bottom;border-bottom:2px solid #ddd}.table>caption+thead>tr:first-child>td,.table>caption+thead>tr:first-child>th,.table>colgroup+thead>tr:first-child>td,.table>colgroup+thead>tr:first-child>th,.table>thead:first-child>tr:first-child>td,.table>thead:first-child>tr:first-child>th{border-top:0}.table>tbody+tbody{border-top:2px solid #ddd}.table .table{background-color:#fff}.table-condensed>tbody>tr>td,.table-condensed>tbody>tr>th,.table-condensed>tfoot>tr>td,.table-condensed>tfoot>tr>th,.table-condensed>thead>tr>td,.table-condensed>thead>tr>th{padding:5px}.table-bordered{border:1px solid #ddd}.table-bordered>tbody>tr>td,.table-bordered>tbody>tr>th,.table-bordered>tfoot>tr>td,.table-bordered>tfoot>tr>th,.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border:1px solid #ddd}.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border-bottom-width:2px}.table-striped>tbody>tr:nth-of-type(odd){background-color:#f9f9f9}.table-hover>tbody>tr:hover{background-color:#f5f5f5}table col[class*=col-]{position:static;display:table-column;float:none}table td[class*=col-],table th[class*=col-]{position:static;display:table-cell;float:none}.table>tbody>tr.active>td,.table>tbody>tr.active>th,.table>tbody>tr>td.active,.table>tbody>tr>th.active,.table>tfoot>tr.active>td,.table>tfoot>tr.active>th,.table>tfoot>tr>td.active,.table>tfoot>tr>th.active,.table>thead>tr.active>td,.table>thead>tr.active>th,.table>thead>tr>td.active,.table>thead>tr>th.active{background-color:#f5f5f5}.table-hover>tbody>tr.active:hover>td,.table-hover>tbody>tr.active:hover>th,.table-hover>tbody>tr:hover>.active,.table-hover>tbody>tr>td.active:hover,.table-hover>tbody>tr>th.active:hover{background-color:#e8e8e8}.table>tbody>tr.success>td,.table>tbody>tr.success>th,.table>tbody>tr>td.success,.table>tbody>tr>th.success,.table>tfoot>tr.success>td,.table>tfoot>tr.success>th,.table>tfoot>tr>td.success,.table>tfoot>tr>th.success,.table>thead>tr.success>td,.table>thead>tr.success>th,.table>thead>tr>td.success,.table>thead>tr>th.success{background-color:#dff0d8}.table-hover>tbody>tr.success:hover>td,.table-hover>tbody>tr.success:hover>th,.table-hover>tbody>tr:hover>.success,.table-hover>tbody>tr>td.success:hover,.table-hover>tbody>tr>th.success:hover{background-color:#d0e9c6}.table>tbody>tr.info>td,.table>tbody>tr.info>th,.table>tbody>tr>td.info,.table>tbody>tr>th.info,.table>tfoot>tr.info>td,.table>tfoot>tr.info>th,.table>tfoot>tr>td.info,.table>tfoot>tr>th.info,.table>thead>tr.info>td,.table>thead>tr.info>th,.table>thead>tr>td.info,.table>thead>tr>th.info{background-color:#d9edf7}.table-hover>tbody>tr.info:hover>td,.table-hover>tbody>tr.info:hover>th,.table-hover>tbody>tr:hover>.info,.table-hover>tbody>tr>td.info:hover,.table-hover>tbody>tr>th.info:hover{background-color:#c4e3f3}.table>tbody>tr.warning>td,.table>tbody>tr.warning>th,.table>tbody>tr>td.warning,.table>tbody>tr>th.warning,.table>tfoot>tr.warning>td,.table>tfoot>tr.warning>th,.table>tfoot>tr>td.warning,.table>tfoot>tr>th.warning,.table>thead>tr.warning>td,.table>thead>tr.warning>th,.table>thead>tr>td.warning,.table>thead>tr>th.warning{background-color:#fcf8e3}.table-hover>tbody>tr.warning:hover>td,.table-hover>tbody>tr.warning:hover>th,.table-hover>tbody>tr:hover>.warning,.table-hover>tbody>tr>td.warning:hover,.table-hover>tbody>tr>th.warning:hover{background-color:#faf2cc}.table>tbody>tr.danger>td,.table>tbody>tr.danger>th,.table>tbody>tr>td.danger,.table>tbody>tr>th.danger,.table>tfoot>tr.danger>td,.table>tfoot>tr.danger>th,.table>tfoot>tr>td.danger,.table>tfoot>tr>th.danger,.table>thead>tr.danger>td,.table>thead>tr.danger>th,.table>thead>tr>td.danger,.table>thead>tr>th.danger{background-color:#f2dede}.table-hover>tbody>tr.danger:hover>td,.table-hover>tbody>tr.danger:hover>th,.table-hover>tbody>tr:hover>.danger,.table-hover>tbody>tr>td.danger:hover,.table-hover>tbody>tr>th.danger:hover{background-color:#ebcccc}.table-responsive{min-height:.01%;overflow-x:auto}@media screen and (max-width:767px){.table-responsive{width:100%;margin-bottom:15px;overflow-y:hidden;-ms-overflow-style:-ms-autohiding-scrollbar;border:1px solid #ddd}.table-responsive>.table{margin-bottom:0}.table-responsive>.table>tbody>tr>td,.table-responsive>.table>tbody>tr>th,.table-responsive>.table>tfoot>tr>td,.table-responsive>.table>tfoot>tr>th,.table-responsive>.table>thead>tr>td,.table-responsive>.table>thead>tr>th{white-space:nowrap}.table-responsive>.table-bordered{border:0}.table-responsive>.table-bordered>tbody>tr>td:first-child,.table-responsive>.table-bordered>tbody>tr>th:first-child,.table-responsive>.table-bordered>tfoot>tr>td:first-child,.table-responsive>.table-bordered>tfoot>tr>th:first-child,.table-responsive>.table-bordered>thead>tr>td:first-child,.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.table-responsive>.table-bordered>tbody>tr>td:last-child,.table-responsive>.table-bordered>tbody>tr>th:last-child,.table-responsive>.table-bordered>tfoot>tr>td:last-child,.table-responsive>.table-bordered>tfoot>tr>th:last-child,.table-responsive>.table-bordered>thead>tr>td:last-child,.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.table-responsive>.table-bordered>tbody>tr:last-child>td,.table-responsive>.table-bordered>tbody>tr:last-child>th,.table-responsive>.table-bordered>tfoot>tr:last-child>td,.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}}fieldset{min-width:0;padding:0;margin:0;border:0}legend{display:block;width:100%;padding:0;margin-bottom:20px;font-size:21px;line-height:inherit;color:#333;border:0;border-bottom:1px solid #e5e5e5}label{display:inline-block;max-width:100%;margin-bottom:5px;font-weight:700}input[type=search]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}input[type=checkbox],input[type=radio]{margin:4px 0 0;margin-top:1px\9;line-height:normal}input[type=file]{display:block}input[type=range]{display:block;width:100%}select[multiple],select[size]{height:auto}input[type=file]:focus,input[type=checkbox]:focus,input[type=radio]:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}output{display:block;padding-top:7px;font-size:14px;line-height:1.42857143;color:#555}.form-control{display:block;width:100%;height:34px;padding:6px 12px;font-size:14px;line-height:1.42857143;color:#555;background-color:#fff;background-image:none;border:1px solid #ccc;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075);-webkit-transition:border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;-o-transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s;transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s}.form-control:focus{border-color:#66afe9;outline:0;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6);box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6)}.form-control::-moz-placeholder{color:#999;opacity:1}.form-control:-ms-input-placeholder{color:#999}.form-control::-webkit-input-placeholder{color:#999}.form-control::-ms-expand{background-color:transparent;border:0}.form-control[disabled],.form-control[readonly],fieldset[disabled] .form-control{background-color:#eee;opacity:1}.form-control[disabled],fieldset[disabled] .form-control{cursor:not-allowed}textarea.form-control{height:auto}input[type=search]{-webkit-appearance:none}@media screen and (-webkit-min-device-pixel-ratio:0){input[type=date].form-control,input[type=time].form-control,input[type=datetime-local].form-control,input[type=month].form-control{line-height:34px}.input-group-sm input[type=date],.input-group-sm input[type=time],.input-group-sm input[type=datetime-local],.input-group-sm input[type=month],input[type=date].input-sm,input[type=time].input-sm,input[type=datetime-local].input-sm,input[type=month].input-sm{line-height:30px}.input-group-lg input[type=date],.input-group-lg input[type=time],.input-group-lg input[type=datetime-local],.input-group-lg input[type=month],input[type=date].input-lg,input[type=time].input-lg,input[type=datetime-local].input-lg,input[type=month].input-lg{line-height:46px}}.form-group{margin-bottom:15px}.checkbox,.radio{position:relative;display:block;margin-top:10px;margin-bottom:10px}.checkbox label,.radio label{min-height:20px;padding-left:20px;margin-bottom:0;font-weight:400;cursor:pointer}.checkbox input[type=checkbox],.checkbox-inline input[type=checkbox],.radio input[type=radio],.radio-inline input[type=radio]{position:absolute;margin-top:4px\9;margin-left:-20px}.checkbox+.checkbox,.radio+.radio{margin-top:-5px}.checkbox-inline,.radio-inline{position:relative;display:inline-block;padding-left:20px;margin-bottom:0;font-weight:400;vertical-align:middle;cursor:pointer}.checkbox-inline+.checkbox-inline,.radio-inline+.radio-inline{margin-top:0;margin-left:10px}fieldset[disabled] input[type=checkbox],fieldset[disabled] input[type=radio],input[type=checkbox].disabled,input[type=checkbox][disabled],input[type=radio].disabled,input[type=radio][disabled]{cursor:not-allowed}.checkbox-inline.disabled,.radio-inline.disabled,fieldset[disabled] .checkbox-inline,fieldset[disabled] .radio-inline{cursor:not-allowed}.checkbox.disabled label,.radio.disabled label,fieldset[disabled] .checkbox label,fieldset[disabled] .radio label{cursor:not-allowed}.form-control-static{min-height:34px;padding-top:7px;padding-bottom:7px;margin-bottom:0}.form-control-static.input-lg,.form-control-static.input-sm{padding-right:0;padding-left:0}.input-sm{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-sm{height:30px;line-height:30px}select[multiple].input-sm,textarea.input-sm{height:auto}.form-group-sm .form-control{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.form-group-sm select.form-control{height:30px;line-height:30px}.form-group-sm select[multiple].form-control,.form-group-sm textarea.form-control{height:auto}.form-group-sm .form-control-static{height:30px;min-height:32px;padding:6px 10px;font-size:12px;line-height:1.5}.input-lg{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-lg{height:46px;line-height:46px}select[multiple].input-lg,textarea.input-lg{height:auto}.form-group-lg .form-control{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.form-group-lg select.form-control{height:46px;line-height:46px}.form-group-lg select[multiple].form-control,.form-group-lg textarea.form-control{height:auto}.form-group-lg .form-control-static{height:46px;min-height:38px;padding:11px 16px;font-size:18px;line-height:1.3333333}.has-feedback{position:relative}.has-feedback .form-control{padding-right:42.5px}.form-control-feedback{position:absolute;top:0;right:0;z-index:2;display:block;width:34px;height:34px;line-height:34px;text-align:center;pointer-events:none}.form-group-lg .form-control+.form-control-feedback,.input-group-lg+.form-control-feedback,.input-lg+.form-control-feedback{width:46px;height:46px;line-height:46px}.form-group-sm .form-control+.form-control-feedback,.input-group-sm+.form-control-feedback,.input-sm+.form-control-feedback{width:30px;height:30px;line-height:30px}.has-success .checkbox,.has-success .checkbox-inline,.has-success .control-label,.has-success .help-block,.has-success .radio,.has-success .radio-inline,.has-success.checkbox label,.has-success.checkbox-inline label,.has-success.radio label,.has-success.radio-inline label{color:#3c763d}.has-success .form-control{border-color:#3c763d;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-success .form-control:focus{border-color:#2b542c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168}.has-success .input-group-addon{color:#3c763d;background-color:#dff0d8;border-color:#3c763d}.has-success .form-control-feedback{color:#3c763d}.has-warning .checkbox,.has-warning .checkbox-inline,.has-warning .control-label,.has-warning .help-block,.has-warning .radio,.has-warning .radio-inline,.has-warning.checkbox label,.has-warning.checkbox-inline label,.has-warning.radio label,.has-warning.radio-inline label{color:#8a6d3b}.has-warning .form-control{border-color:#8a6d3b;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-warning .form-control:focus{border-color:#66512c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b}.has-warning .input-group-addon{color:#8a6d3b;background-color:#fcf8e3;border-color:#8a6d3b}.has-warning .form-control-feedback{color:#8a6d3b}.has-error .checkbox,.has-error .checkbox-inline,.has-error .control-label,.has-error .help-block,.has-error .radio,.has-error .radio-inline,.has-error.checkbox label,.has-error.checkbox-inline label,.has-error.radio label,.has-error.radio-inline label{color:#a94442}.has-error .form-control{border-color:#a94442;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-error .form-control:focus{border-color:#843534;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483}.has-error .input-group-addon{color:#a94442;background-color:#f2dede;border-color:#a94442}.has-error .form-control-feedback{color:#a94442}.has-feedback label~.form-control-feedback{top:25px}.has-feedback label.sr-only~.form-control-feedback{top:0}.help-block{display:block;margin-top:5px;margin-bottom:10px;color:#737373}@media (min-width:768px){.form-inline .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.form-inline .form-control{display:inline-block;width:auto;vertical-align:middle}.form-inline .form-control-static{display:inline-block}.form-inline .input-group{display:inline-table;vertical-align:middle}.form-inline .input-group .form-control,.form-inline .input-group .input-group-addon,.form-inline .input-group .input-group-btn{width:auto}.form-inline .input-group>.form-control{width:100%}.form-inline .control-label{margin-bottom:0;vertical-align:middle}.form-inline .checkbox,.form-inline .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.form-inline .checkbox label,.form-inline .radio label{padding-left:0}.form-inline .checkbox input[type=checkbox],.form-inline .radio input[type=radio]{position:relative;margin-left:0}.form-inline .has-feedback .form-control-feedback{top:0}}.form-horizontal .checkbox,.form-horizontal .checkbox-inline,.form-horizontal .radio,.form-horizontal .radio-inline{padding-top:7px;margin-top:0;margin-bottom:0}.form-horizontal .checkbox,.form-horizontal .radio{min-height:27px}.form-horizontal .form-group{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.form-horizontal .control-label{padding-top:7px;margin-bottom:0;text-align:right}}.form-horizontal .has-feedback .form-control-feedback{right:15px}@media (min-width:768px){.form-horizontal .form-group-lg .control-label{padding-top:11px;font-size:18px}}@media (min-width:768px){.form-horizontal .form-group-sm .control-label{padding-top:6px;font-size:12px}}.btn{display:inline-block;padding:6px 12px;margin-bottom:0;font-size:14px;font-weight:400;line-height:1.42857143;text-align:center;white-space:nowrap;vertical-align:middle;-ms-touch-action:manipulation;touch-action:manipulation;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;background-image:none;border:1px solid transparent;border-radius:4px}.btn.active.focus,.btn.active:focus,.btn.focus,.btn:active.focus,.btn:active:focus,.btn:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}.btn.focus,.btn:focus,.btn:hover{color:#333;text-decoration:none}.btn.active,.btn:active{background-image:none;outline:0;-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn.disabled,.btn[disabled],fieldset[disabled] .btn{cursor:not-allowed;filter:alpha(opacity=65);-webkit-box-shadow:none;box-shadow:none;opacity:.65}a.btn.disabled,fieldset[disabled] a.btn{pointer-events:none}.btn-default{color:#333;background-color:#fff;border-color:#ccc}.btn-default.focus,.btn-default:focus{color:#333;background-color:#e6e6e6;border-color:#8c8c8c}.btn-default:hover{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active.focus,.btn-default.active:focus,.btn-default.active:hover,.btn-default:active.focus,.btn-default:active:focus,.btn-default:active:hover,.open>.dropdown-toggle.btn-default.focus,.open>.dropdown-toggle.btn-default:focus,.open>.dropdown-toggle.btn-default:hover{color:#333;background-color:#d4d4d4;border-color:#8c8c8c}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{background-image:none}.btn-default.disabled.focus,.btn-default.disabled:focus,.btn-default.disabled:hover,.btn-default[disabled].focus,.btn-default[disabled]:focus,.btn-default[disabled]:hover,fieldset[disabled] .btn-default.focus,fieldset[disabled] .btn-default:focus,fieldset[disabled] .btn-default:hover{background-color:#fff;border-color:#ccc}.btn-default .badge{color:#fff;background-color:#333}.btn-primary{color:#fff;background-color:#337ab7;border-color:#2e6da4}.btn-primary.focus,.btn-primary:focus{color:#fff;background-color:#286090;border-color:#122b40}.btn-primary:hover{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active.focus,.btn-primary.active:focus,.btn-primary.active:hover,.btn-primary:active.focus,.btn-primary:active:focus,.btn-primary:active:hover,.open>.dropdown-toggle.btn-primary.focus,.open>.dropdown-toggle.btn-primary:focus,.open>.dropdown-toggle.btn-primary:hover{color:#fff;background-color:#204d74;border-color:#122b40}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{background-image:none}.btn-primary.disabled.focus,.btn-primary.disabled:focus,.btn-primary.disabled:hover,.btn-primary[disabled].focus,.btn-primary[disabled]:focus,.btn-primary[disabled]:hover,fieldset[disabled] .btn-primary.focus,fieldset[disabled] .btn-primary:focus,fieldset[disabled] .btn-primary:hover{background-color:#337ab7;border-color:#2e6da4}.btn-primary .badge{color:#337ab7;background-color:#fff}.btn-success{color:#fff;background-color:#5cb85c;border-color:#4cae4c}.btn-success.focus,.btn-success:focus{color:#fff;background-color:#449d44;border-color:#255625}.btn-success:hover{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active.focus,.btn-success.active:focus,.btn-success.active:hover,.btn-success:active.focus,.btn-success:active:focus,.btn-success:active:hover,.open>.dropdown-toggle.btn-success.focus,.open>.dropdown-toggle.btn-success:focus,.open>.dropdown-toggle.btn-success:hover{color:#fff;background-color:#398439;border-color:#255625}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{background-image:none}.btn-success.disabled.focus,.btn-success.disabled:focus,.btn-success.disabled:hover,.btn-success[disabled].focus,.btn-success[disabled]:focus,.btn-success[disabled]:hover,fieldset[disabled] .btn-success.focus,fieldset[disabled] .btn-success:focus,fieldset[disabled] .btn-success:hover{background-color:#5cb85c;border-color:#4cae4c}.btn-success .badge{color:#5cb85c;background-color:#fff}.btn-info{color:#fff;background-color:#5bc0de;border-color:#46b8da}.btn-info.focus,.btn-info:focus{color:#fff;background-color:#31b0d5;border-color:#1b6d85}.btn-info:hover{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active.focus,.btn-info.active:focus,.btn-info.active:hover,.btn-info:active.focus,.btn-info:active:focus,.btn-info:active:hover,.open>.dropdown-toggle.btn-info.focus,.open>.dropdown-toggle.btn-info:focus,.open>.dropdown-toggle.btn-info:hover{color:#fff;background-color:#269abc;border-color:#1b6d85}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{background-image:none}.btn-info.disabled.focus,.btn-info.disabled:focus,.btn-info.disabled:hover,.btn-info[disabled].focus,.btn-info[disabled]:focus,.btn-info[disabled]:hover,fieldset[disabled] .btn-info.focus,fieldset[disabled] .btn-info:focus,fieldset[disabled] .btn-info:hover{background-color:#5bc0de;border-color:#46b8da}.btn-info .badge{color:#5bc0de;background-color:#fff}.btn-warning{color:#fff;background-color:#f0ad4e;border-color:#eea236}.btn-warning.focus,.btn-warning:focus{color:#fff;background-color:#ec971f;border-color:#985f0d}.btn-warning:hover{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active.focus,.btn-warning.active:focus,.btn-warning.active:hover,.btn-warning:active.focus,.btn-warning:active:focus,.btn-warning:active:hover,.open>.dropdown-toggle.btn-warning.focus,.open>.dropdown-toggle.btn-warning:focus,.open>.dropdown-toggle.btn-warning:hover{color:#fff;background-color:#d58512;border-color:#985f0d}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{background-image:none}.btn-warning.disabled.focus,.btn-warning.disabled:focus,.btn-warning.disabled:hover,.btn-warning[disabled].focus,.btn-warning[disabled]:focus,.btn-warning[disabled]:hover,fieldset[disabled] .btn-warning.focus,fieldset[disabled] .btn-warning:focus,fieldset[disabled] .btn-warning:hover{background-color:#f0ad4e;border-color:#eea236}.btn-warning .badge{color:#f0ad4e;background-color:#fff}.btn-danger{color:#fff;background-color:#d9534f;border-color:#d43f3a}.btn-danger.focus,.btn-danger:focus{color:#fff;background-color:#c9302c;border-color:#761c19}.btn-danger:hover{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active.focus,.btn-danger.active:focus,.btn-danger.active:hover,.btn-danger:active.focus,.btn-danger:active:focus,.btn-danger:active:hover,.open>.dropdown-toggle.btn-danger.focus,.open>.dropdown-toggle.btn-danger:focus,.open>.dropdown-toggle.btn-danger:hover{color:#fff;background-color:#ac2925;border-color:#761c19}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{background-image:none}.btn-danger.disabled.focus,.btn-danger.disabled:focus,.btn-danger.disabled:hover,.btn-danger[disabled].focus,.btn-danger[disabled]:focus,.btn-danger[disabled]:hover,fieldset[disabled] .btn-danger.focus,fieldset[disabled] .btn-danger:focus,fieldset[disabled] .btn-danger:hover{background-color:#d9534f;border-color:#d43f3a}.btn-danger .badge{color:#d9534f;background-color:#fff}.btn-link{font-weight:400;color:#337ab7;border-radius:0}.btn-link,.btn-link.active,.btn-link:active,.btn-link[disabled],fieldset[disabled] .btn-link{background-color:transparent;-webkit-box-shadow:none;box-shadow:none}.btn-link,.btn-link:active,.btn-link:focus,.btn-link:hover{border-color:transparent}.btn-link:focus,.btn-link:hover{color:#23527c;text-decoration:underline;background-color:transparent}.btn-link[disabled]:focus,.btn-link[disabled]:hover,fieldset[disabled] .btn-link:focus,fieldset[disabled] .btn-link:hover{color:#777;text-decoration:none}.btn-group-lg>.btn,.btn-lg{padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.btn-group-sm>.btn,.btn-sm{padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.btn-group-xs>.btn,.btn-xs{padding:1px 5px;font-size:12px;line-height:1.5;border-radius:3px}.btn-block{display:block;width:100%}.btn-block+.btn-block{margin-top:5px}input[type=button].btn-block,input[type=reset].btn-block,input[type=submit].btn-block{width:100%}.fade{opacity:0;-webkit-transition:opacity .15s linear;-o-transition:opacity .15s linear;transition:opacity .15s linear}.fade.in{opacity:1}.collapse{display:none}.collapse.in{display:block}tr.collapse.in{display:table-row}tbody.collapse.in{display:table-row-group}.collapsing{position:relative;height:0;overflow:hidden;-webkit-transition-timing-function:ease;-o-transition-timing-function:ease;transition-timing-function:ease;-webkit-transition-duration:.35s;-o-transition-duration:.35s;transition-duration:.35s;-webkit-transition-property:height,visibility;-o-transition-property:height,visibility;transition-property:height,visibility}.caret{display:inline-block;width:0;height:0;margin-left:2px;vertical-align:middle;border-top:4px dashed;border-top:4px solid\9;border-right:4px solid transparent;border-left:4px solid transparent}.dropdown,.dropup{position:relative}.dropdown-toggle:focus{outline:0}.dropdown-menu{position:absolute;top:100%;left:0;z-index:1000;display:none;float:left;min-width:160px;padding:5px 0;margin:2px 0 0;font-size:14px;text-align:left;list-style:none;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.15);border-radius:4px;-webkit-box-shadow:0 6px 12px rgba(0,0,0,.175);box-shadow:0 6px 12px rgba(0,0,0,.175)}.dropdown-menu.pull-right{right:0;left:auto}.dropdown-menu .divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.dropdown-menu>li>a{display:block;padding:3px 20px;clear:both;font-weight:400;line-height:1.42857143;color:#333;white-space:nowrap}.dropdown-menu>li>a:focus,.dropdown-menu>li>a:hover{color:#262626;text-decoration:none;background-color:#f5f5f5}.dropdown-menu>.active>a,.dropdown-menu>.active>a:focus,.dropdown-menu>.active>a:hover{color:#fff;text-decoration:none;background-color:#337ab7;outline:0}.dropdown-menu>.disabled>a,.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{color:#777}.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{text-decoration:none;cursor:not-allowed;background-color:transparent;background-image:none;filter:progid:DXImageTransform.Microsoft.gradient(enabled=false)}.open>.dropdown-menu{display:block}.open>a{outline:0}.dropdown-menu-right{right:0;left:auto}.dropdown-menu-left{right:auto;left:0}.dropdown-header{display:block;padding:3px 20px;font-size:12px;line-height:1.42857143;color:#777;white-space:nowrap}.dropdown-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:990}.pull-right>.dropdown-menu{right:0;left:auto}.dropup .caret,.navbar-fixed-bottom .dropdown .caret{content:"";border-top:0;border-bottom:4px dashed;border-bottom:4px solid\9}.dropup .dropdown-menu,.navbar-fixed-bottom .dropdown .dropdown-menu{top:auto;bottom:100%;margin-bottom:2px}@media (min-width:768px){.navbar-right .dropdown-menu{right:0;left:auto}.navbar-right .dropdown-menu-left{right:auto;left:0}}.btn-group,.btn-group-vertical{position:relative;display:inline-block;vertical-align:middle}.btn-group-vertical>.btn,.btn-group>.btn{position:relative;float:left}.btn-group-vertical>.btn.active,.btn-group-vertical>.btn:active,.btn-group-vertical>.btn:focus,.btn-group-vertical>.btn:hover,.btn-group>.btn.active,.btn-group>.btn:active,.btn-group>.btn:focus,.btn-group>.btn:hover{z-index:2}.btn-group .btn+.btn,.btn-group .btn+.btn-group,.btn-group .btn-group+.btn,.btn-group .btn-group+.btn-group{margin-left:-1px}.btn-toolbar{margin-left:-5px}.btn-toolbar .btn,.btn-toolbar .btn-group,.btn-toolbar .input-group{float:left}.btn-toolbar>.btn,.btn-toolbar>.btn-group,.btn-toolbar>.input-group{margin-left:5px}.btn-group>.btn:not(:first-child):not(:last-child):not(.dropdown-toggle){border-radius:0}.btn-group>.btn:first-child{margin-left:0}.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn:last-child:not(:first-child),.btn-group>.dropdown-toggle:not(:first-child){border-top-left-radius:0;border-bottom-left-radius:0}.btn-group>.btn-group{float:left}.btn-group>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-bottom-left-radius:0}.btn-group .dropdown-toggle:active,.btn-group.open .dropdown-toggle{outline:0}.btn-group>.btn+.dropdown-toggle{padding-right:8px;padding-left:8px}.btn-group>.btn-lg+.dropdown-toggle{padding-right:12px;padding-left:12px}.btn-group.open .dropdown-toggle{-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn-group.open .dropdown-toggle.btn-link{-webkit-box-shadow:none;box-shadow:none}.btn .caret{margin-left:0}.btn-lg .caret{border-width:5px 5px 0;border-bottom-width:0}.dropup .btn-lg .caret{border-width:0 5px 5px}.btn-group-vertical>.btn,.btn-group-vertical>.btn-group,.btn-group-vertical>.btn-group>.btn{display:block;float:none;width:100%;max-width:100%}.btn-group-vertical>.btn-group>.btn{float:none}.btn-group-vertical>.btn+.btn,.btn-group-vertical>.btn+.btn-group,.btn-group-vertical>.btn-group+.btn,.btn-group-vertical>.btn-group+.btn-group{margin-top:-1px;margin-left:0}.btn-group-vertical>.btn:not(:first-child):not(:last-child){border-radius:0}.btn-group-vertical>.btn:first-child:not(:last-child){border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn:last-child:not(:first-child){border-top-left-radius:0;border-top-right-radius:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}.btn-group-vertical>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group-vertical>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group-vertical>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-top-right-radius:0}.btn-group-justified{display:table;width:100%;table-layout:fixed;border-collapse:separate}.btn-group-justified>.btn,.btn-group-justified>.btn-group{display:table-cell;float:none;width:1%}.btn-group-justified>.btn-group .btn{width:100%}.btn-group-justified>.btn-group .dropdown-menu{left:auto}[data-toggle=buttons]>.btn input[type=checkbox],[data-toggle=buttons]>.btn input[type=radio],[data-toggle=buttons]>.btn-group>.btn input[type=checkbox],[data-toggle=buttons]>.btn-group>.btn input[type=radio]{position:absolute;clip:rect(0,0,0,0);pointer-events:none}.input-group{position:relative;display:table;border-collapse:separate}.input-group[class*=col-]{float:none;padding-right:0;padding-left:0}.input-group .form-control{position:relative;z-index:2;float:left;width:100%;margin-bottom:0}.input-group .form-control:focus{z-index:3}.input-group-lg>.form-control,.input-group-lg>.input-group-addon,.input-group-lg>.input-group-btn>.btn{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-group-lg>.form-control,select.input-group-lg>.input-group-addon,select.input-group-lg>.input-group-btn>.btn{height:46px;line-height:46px}select[multiple].input-group-lg>.form-control,select[multiple].input-group-lg>.input-group-addon,select[multiple].input-group-lg>.input-group-btn>.btn,textarea.input-group-lg>.form-control,textarea.input-group-lg>.input-group-addon,textarea.input-group-lg>.input-group-btn>.btn{height:auto}.input-group-sm>.form-control,.input-group-sm>.input-group-addon,.input-group-sm>.input-group-btn>.btn{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-group-sm>.form-control,select.input-group-sm>.input-group-addon,select.input-group-sm>.input-group-btn>.btn{height:30px;line-height:30px}select[multiple].input-group-sm>.form-control,select[multiple].input-group-sm>.input-group-addon,select[multiple].input-group-sm>.input-group-btn>.btn,textarea.input-group-sm>.form-control,textarea.input-group-sm>.input-group-addon,textarea.input-group-sm>.input-group-btn>.btn{height:auto}.input-group .form-control,.input-group-addon,.input-group-btn{display:table-cell}.input-group .form-control:not(:first-child):not(:last-child),.input-group-addon:not(:first-child):not(:last-child),.input-group-btn:not(:first-child):not(:last-child){border-radius:0}.input-group-addon,.input-group-btn{width:1%;white-space:nowrap;vertical-align:middle}.input-group-addon{padding:6px 12px;font-size:14px;font-weight:400;line-height:1;color:#555;text-align:center;background-color:#eee;border:1px solid #ccc;border-radius:4px}.input-group-addon.input-sm{padding:5px 10px;font-size:12px;border-radius:3px}.input-group-addon.input-lg{padding:10px 16px;font-size:18px;border-radius:6px}.input-group-addon input[type=checkbox],.input-group-addon input[type=radio]{margin-top:0}.input-group .form-control:first-child,.input-group-addon:first-child,.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group>.btn,.input-group-btn:first-child>.dropdown-toggle,.input-group-btn:last-child>.btn-group:not(:last-child)>.btn,.input-group-btn:last-child>.btn:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.input-group-addon:first-child{border-right:0}.input-group .form-control:last-child,.input-group-addon:last-child,.input-group-btn:first-child>.btn-group:not(:first-child)>.btn,.input-group-btn:first-child>.btn:not(:first-child),.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group>.btn,.input-group-btn:last-child>.dropdown-toggle{border-top-left-radius:0;border-bottom-left-radius:0}.input-group-addon:last-child{border-left:0}.input-group-btn{position:relative;font-size:0;white-space:nowrap}.input-group-btn>.btn{position:relative}.input-group-btn>.btn+.btn{margin-left:-1px}.input-group-btn>.btn:active,.input-group-btn>.btn:focus,.input-group-btn>.btn:hover{z-index:2}.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group{margin-right:-1px}.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group{z-index:2;margin-left:-1px}.nav{padding-left:0;margin-bottom:0;list-style:none}.nav>li{position:relative;display:block}.nav>li>a{position:relative;display:block;padding:10px 15px}.nav>li>a:focus,.nav>li>a:hover{text-decoration:none;background-color:#eee}.nav>li.disabled>a{color:#777}.nav>li.disabled>a:focus,.nav>li.disabled>a:hover{color:#777;text-decoration:none;cursor:not-allowed;background-color:transparent}.nav .open>a,.nav .open>a:focus,.nav .open>a:hover{background-color:#eee;border-color:#337ab7}.nav .nav-divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.nav>li>a>img{max-width:none}.nav-tabs{border-bottom:1px solid #ddd}.nav-tabs>li{float:left;margin-bottom:-1px}.nav-tabs>li>a{margin-right:2px;line-height:1.42857143;border:1px solid transparent;border-radius:4px 4px 0 0}.nav-tabs>li>a:hover{border-color:#eee #eee #ddd}.nav-tabs>li.active>a,.nav-tabs>li.active>a:focus,.nav-tabs>li.active>a:hover{color:#555;cursor:default;background-color:#fff;border:1px solid #ddd;border-bottom-color:transparent}.nav-tabs.nav-justified{width:100%;border-bottom:0}.nav-tabs.nav-justified>li{float:none}.nav-tabs.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-tabs.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-tabs.nav-justified>li{display:table-cell;width:1%}.nav-tabs.nav-justified>li>a{margin-bottom:0}}.nav-tabs.nav-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs.nav-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border-bottom-color:#fff}}.nav-pills>li{float:left}.nav-pills>li>a{border-radius:4px}.nav-pills>li+li{margin-left:2px}.nav-pills>li.active>a,.nav-pills>li.active>a:focus,.nav-pills>li.active>a:hover{color:#fff;background-color:#337ab7}.nav-stacked>li{float:none}.nav-stacked>li+li{margin-top:2px;margin-left:0}.nav-justified{width:100%}.nav-justified>li{float:none}.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-justified>li{display:table-cell;width:1%}.nav-justified>li>a{margin-bottom:0}}.nav-tabs-justified{border-bottom:0}.nav-tabs-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border-bottom-color:#fff}}.tab-content>.tab-pane{display:none}.tab-content>.active{display:block}.nav-tabs .dropdown-menu{margin-top:-1px;border-top-left-radius:0;border-top-right-radius:0}.navbar{position:relative;min-height:50px;margin-bottom:20px;border:1px solid transparent}@media (min-width:768px){.navbar{border-radius:4px}}@media (min-width:768px){.navbar-header{float:left}}.navbar-collapse{padding-right:15px;padding-left:15px;overflow-x:visible;-webkit-overflow-scrolling:touch;border-top:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1)}.navbar-collapse.in{overflow-y:auto}@media (min-width:768px){.navbar-collapse{width:auto;border-top:0;-webkit-box-shadow:none;box-shadow:none}.navbar-collapse.collapse{display:block!important;height:auto!important;padding-bottom:0;overflow:visible!important}.navbar-collapse.in{overflow-y:visible}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse,.navbar-static-top .navbar-collapse{padding-right:0;padding-left:0}}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:340px}@media (max-device-width:480px) and (orientation:landscape){.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:200px}}.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:0;margin-left:0}}.navbar-static-top{z-index:1000;border-width:0 0 1px}@media (min-width:768px){.navbar-static-top{border-radius:0}}.navbar-fixed-bottom,.navbar-fixed-top{position:fixed;right:0;left:0;z-index:1030}@media (min-width:768px){.navbar-fixed-bottom,.navbar-fixed-top{border-radius:0}}.navbar-fixed-top{top:0;border-width:0 0 1px}.navbar-fixed-bottom{bottom:0;margin-bottom:0;border-width:1px 0 0}.navbar-brand{float:left;height:50px;padding:15px 15px;font-size:18px;line-height:20px}.navbar-brand:focus,.navbar-brand:hover{text-decoration:none}.navbar-brand>img{display:block}@media (min-width:768px){.navbar>.container .navbar-brand,.navbar>.container-fluid .navbar-brand{margin-left:-15px}}.navbar-toggle{position:relative;float:right;padding:9px 10px;margin-top:8px;margin-right:15px;margin-bottom:8px;background-color:transparent;background-image:none;border:1px solid transparent;border-radius:4px}.navbar-toggle:focus{outline:0}.navbar-toggle .icon-bar{display:block;width:22px;height:2px;border-radius:1px}.navbar-toggle .icon-bar+.icon-bar{margin-top:4px}@media (min-width:768px){.navbar-toggle{display:none}}.navbar-nav{margin:7.5px -15px}.navbar-nav>li>a{padding-top:10px;padding-bottom:10px;line-height:20px}@media (max-width:767px){.navbar-nav .open .dropdown-menu{position:static;float:none;width:auto;margin-top:0;background-color:transparent;border:0;-webkit-box-shadow:none;box-shadow:none}.navbar-nav .open .dropdown-menu .dropdown-header,.navbar-nav .open .dropdown-menu>li>a{padding:5px 15px 5px 25px}.navbar-nav .open .dropdown-menu>li>a{line-height:20px}.navbar-nav .open .dropdown-menu>li>a:focus,.navbar-nav .open .dropdown-menu>li>a:hover{background-image:none}}@media (min-width:768px){.navbar-nav{float:left;margin:0}.navbar-nav>li{float:left}.navbar-nav>li>a{padding-top:15px;padding-bottom:15px}}.navbar-form{padding:10px 15px;margin-top:8px;margin-right:-15px;margin-bottom:8px;margin-left:-15px;border-top:1px solid transparent;border-bottom:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1)}@media (min-width:768px){.navbar-form .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.navbar-form .form-control{display:inline-block;width:auto;vertical-align:middle}.navbar-form .form-control-static{display:inline-block}.navbar-form .input-group{display:inline-table;vertical-align:middle}.navbar-form .input-group .form-control,.navbar-form .input-group .input-group-addon,.navbar-form .input-group .input-group-btn{width:auto}.navbar-form .input-group>.form-control{width:100%}.navbar-form .control-label{margin-bottom:0;vertical-align:middle}.navbar-form .checkbox,.navbar-form .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.navbar-form .checkbox label,.navbar-form .radio label{padding-left:0}.navbar-form .checkbox input[type=checkbox],.navbar-form .radio input[type=radio]{position:relative;margin-left:0}.navbar-form .has-feedback .form-control-feedback{top:0}}@media (max-width:767px){.navbar-form .form-group{margin-bottom:5px}.navbar-form .form-group:last-child{margin-bottom:0}}@media (min-width:768px){.navbar-form{width:auto;padding-top:0;padding-bottom:0;margin-right:0;margin-left:0;border:0;-webkit-box-shadow:none;box-shadow:none}}.navbar-nav>li>.dropdown-menu{margin-top:0;border-top-left-radius:0;border-top-right-radius:0}.navbar-fixed-bottom .navbar-nav>li>.dropdown-menu{margin-bottom:0;border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.navbar-btn{margin-top:8px;margin-bottom:8px}.navbar-btn.btn-sm{margin-top:10px;margin-bottom:10px}.navbar-btn.btn-xs{margin-top:14px;margin-bottom:14px}.navbar-text{margin-top:15px;margin-bottom:15px}@media (min-width:768px){.navbar-text{float:left;margin-right:15px;margin-left:15px}}@media (min-width:768px){.navbar-left{float:left!important}.navbar-right{float:right!important;margin-right:-15px}.navbar-right~.navbar-right{margin-right:0}}.navbar-default{background-color:#f8f8f8;border-color:#e7e7e7}.navbar-default .navbar-brand{color:#777}.navbar-default .navbar-brand:focus,.navbar-default .navbar-brand:hover{color:#5e5e5e;background-color:transparent}.navbar-default .navbar-text{color:#777}.navbar-default .navbar-nav>li>a{color:#777}.navbar-default .navbar-nav>li>a:focus,.navbar-default .navbar-nav>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav>.active>a,.navbar-default .navbar-nav>.active>a:focus,.navbar-default .navbar-nav>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav>.disabled>a,.navbar-default .navbar-nav>.disabled>a:focus,.navbar-default .navbar-nav>.disabled>a:hover{color:#ccc;background-color:transparent}.navbar-default .navbar-toggle{border-color:#ddd}.navbar-default .navbar-toggle:focus,.navbar-default .navbar-toggle:hover{background-color:#ddd}.navbar-default .navbar-toggle .icon-bar{background-color:#888}.navbar-default .navbar-collapse,.navbar-default .navbar-form{border-color:#e7e7e7}.navbar-default .navbar-nav>.open>a,.navbar-default .navbar-nav>.open>a:focus,.navbar-default .navbar-nav>.open>a:hover{color:#555;background-color:#e7e7e7}@media (max-width:767px){.navbar-default .navbar-nav .open .dropdown-menu>li>a{color:#777}.navbar-default .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav .open .dropdown-menu>.active>a,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#ccc;background-color:transparent}}.navbar-default .navbar-link{color:#777}.navbar-default .navbar-link:hover{color:#333}.navbar-default .btn-link{color:#777}.navbar-default .btn-link:focus,.navbar-default .btn-link:hover{color:#333}.navbar-default .btn-link[disabled]:focus,.navbar-default .btn-link[disabled]:hover,fieldset[disabled] .navbar-default .btn-link:focus,fieldset[disabled] .navbar-default .btn-link:hover{color:#ccc}.navbar-inverse{background-color:#222;border-color:#080808}.navbar-inverse .navbar-brand{color:#9d9d9d}.navbar-inverse .navbar-brand:focus,.navbar-inverse .navbar-brand:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-text{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a:focus,.navbar-inverse .navbar-nav>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav>.active>a,.navbar-inverse .navbar-nav>.active>a:focus,.navbar-inverse .navbar-nav>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav>.disabled>a,.navbar-inverse .navbar-nav>.disabled>a:focus,.navbar-inverse .navbar-nav>.disabled>a:hover{color:#444;background-color:transparent}.navbar-inverse .navbar-toggle{border-color:#333}.navbar-inverse .navbar-toggle:focus,.navbar-inverse .navbar-toggle:hover{background-color:#333}.navbar-inverse .navbar-toggle .icon-bar{background-color:#fff}.navbar-inverse .navbar-collapse,.navbar-inverse .navbar-form{border-color:#101010}.navbar-inverse .navbar-nav>.open>a,.navbar-inverse .navbar-nav>.open>a:focus,.navbar-inverse .navbar-nav>.open>a:hover{color:#fff;background-color:#080808}@media (max-width:767px){.navbar-inverse .navbar-nav .open .dropdown-menu>.dropdown-header{border-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu .divider{background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#444;background-color:transparent}}.navbar-inverse .navbar-link{color:#9d9d9d}.navbar-inverse .navbar-link:hover{color:#fff}.navbar-inverse .btn-link{color:#9d9d9d}.navbar-inverse .btn-link:focus,.navbar-inverse .btn-link:hover{color:#fff}.navbar-inverse .btn-link[disabled]:focus,.navbar-inverse .btn-link[disabled]:hover,fieldset[disabled] .navbar-inverse .btn-link:focus,fieldset[disabled] .navbar-inverse .btn-link:hover{color:#444}.breadcrumb{padding:8px 15px;margin-bottom:20px;list-style:none;background-color:#f5f5f5;border-radius:4px}.breadcrumb>li{display:inline-block}.breadcrumb>li+li:before{padding:0 5px;color:#ccc;content:"/\00a0"}.breadcrumb>.active{color:#777}.pagination{display:inline-block;padding-left:0;margin:20px 0;border-radius:4px}.pagination>li{display:inline}.pagination>li>a,.pagination>li>span{position:relative;float:left;padding:6px 12px;margin-left:-1px;line-height:1.42857143;color:#337ab7;text-decoration:none;background-color:#fff;border:1px solid #ddd}.pagination>li:first-child>a,.pagination>li:first-child>span{margin-left:0;border-top-left-radius:4px;border-bottom-left-radius:4px}.pagination>li:last-child>a,.pagination>li:last-child>span{border-top-right-radius:4px;border-bottom-right-radius:4px}.pagination>li>a:focus,.pagination>li>a:hover,.pagination>li>span:focus,.pagination>li>span:hover{z-index:2;color:#23527c;background-color:#eee;border-color:#ddd}.pagination>.active>a,.pagination>.active>a:focus,.pagination>.active>a:hover,.pagination>.active>span,.pagination>.active>span:focus,.pagination>.active>span:hover{z-index:3;color:#fff;cursor:default;background-color:#337ab7;border-color:#337ab7}.pagination>.disabled>a,.pagination>.disabled>a:focus,.pagination>.disabled>a:hover,.pagination>.disabled>span,.pagination>.disabled>span:focus,.pagination>.disabled>span:hover{color:#777;cursor:not-allowed;background-color:#fff;border-color:#ddd}.pagination-lg>li>a,.pagination-lg>li>span{padding:10px 16px;font-size:18px;line-height:1.3333333}.pagination-lg>li:first-child>a,.pagination-lg>li:first-child>span{border-top-left-radius:6px;border-bottom-left-radius:6px}.pagination-lg>li:last-child>a,.pagination-lg>li:last-child>span{border-top-right-radius:6px;border-bottom-right-radius:6px}.pagination-sm>li>a,.pagination-sm>li>span{padding:5px 10px;font-size:12px;line-height:1.5}.pagination-sm>li:first-child>a,.pagination-sm>li:first-child>span{border-top-left-radius:3px;border-bottom-left-radius:3px}.pagination-sm>li:last-child>a,.pagination-sm>li:last-child>span{border-top-right-radius:3px;border-bottom-right-radius:3px}.pager{padding-left:0;margin:20px 0;text-align:center;list-style:none}.pager li{display:inline}.pager li>a,.pager li>span{display:inline-block;padding:5px 14px;background-color:#fff;border:1px solid #ddd;border-radius:15px}.pager li>a:focus,.pager li>a:hover{text-decoration:none;background-color:#eee}.pager .next>a,.pager .next>span{float:right}.pager .previous>a,.pager .previous>span{float:left}.pager .disabled>a,.pager .disabled>a:focus,.pager .disabled>a:hover,.pager .disabled>span{color:#777;cursor:not-allowed;background-color:#fff}.label{display:inline;padding:.2em .6em .3em;font-size:75%;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:baseline;border-radius:.25em}a.label:focus,a.label:hover{color:#fff;text-decoration:none;cursor:pointer}.label:empty{display:none}.btn .label{position:relative;top:-1px}.label-default{background-color:#777}.label-default[href]:focus,.label-default[href]:hover{background-color:#5e5e5e}.label-primary{background-color:#337ab7}.label-primary[href]:focus,.label-primary[href]:hover{background-color:#286090}.label-success{background-color:#5cb85c}.label-success[href]:focus,.label-success[href]:hover{background-color:#449d44}.label-info{background-color:#5bc0de}.label-info[href]:focus,.label-info[href]:hover{background-color:#31b0d5}.label-warning{background-color:#f0ad4e}.label-warning[href]:focus,.label-warning[href]:hover{background-color:#ec971f}.label-danger{background-color:#d9534f}.label-danger[href]:focus,.label-danger[href]:hover{background-color:#c9302c}.badge{display:inline-block;min-width:10px;padding:3px 7px;font-size:12px;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:middle;background-color:#777;border-radius:10px}.badge:empty{display:none}.btn .badge{position:relative;top:-1px}.btn-group-xs>.btn .badge,.btn-xs .badge{top:0;padding:1px 5px}a.badge:focus,a.badge:hover{color:#fff;text-decoration:none;cursor:pointer}.list-group-item.active>.badge,.nav-pills>.active>a>.badge{color:#337ab7;background-color:#fff}.list-group-item>.badge{float:right}.list-group-item>.badge+.badge{margin-right:5px}.nav-pills>li>a>.badge{margin-left:3px}.jumbotron{padding-top:30px;padding-bottom:30px;margin-bottom:30px;color:inherit;background-color:#eee}.jumbotron .h1,.jumbotron h1{color:inherit}.jumbotron p{margin-bottom:15px;font-size:21px;font-weight:200}.jumbotron>hr{border-top-color:#d5d5d5}.container .jumbotron,.container-fluid .jumbotron{padding-right:15px;padding-left:15px;border-radius:6px}.jumbotron .container{max-width:100%}@media screen and (min-width:768px){.jumbotron{padding-top:48px;padding-bottom:48px}.container .jumbotron,.container-fluid .jumbotron{padding-right:60px;padding-left:60px}.jumbotron .h1,.jumbotron h1{font-size:63px}}.thumbnail{display:block;padding:4px;margin-bottom:20px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:border .2s ease-in-out;-o-transition:border .2s ease-in-out;transition:border .2s ease-in-out}.thumbnail a>img,.thumbnail>img{margin-right:auto;margin-left:auto}a.thumbnail.active,a.thumbnail:focus,a.thumbnail:hover{border-color:#337ab7}.thumbnail .caption{padding:9px;color:#333}.alert{padding:15px;margin-bottom:20px;border:1px solid transparent;border-radius:4px}.alert h4{margin-top:0;color:inherit}.alert .alert-link{font-weight:700}.alert>p,.alert>ul{margin-bottom:0}.alert>p+p{margin-top:5px}.alert-dismissable,.alert-dismissible{padding-right:35px}.alert-dismissable .close,.alert-dismissible .close{position:relative;top:-2px;right:-21px;color:inherit}.alert-success{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.alert-success hr{border-top-color:#c9e2b3}.alert-success .alert-link{color:#2b542c}.alert-info{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.alert-info hr{border-top-color:#a6e1ec}.alert-info .alert-link{color:#245269}.alert-warning{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.alert-warning hr{border-top-color:#f7e1b5}.alert-warning .alert-link{color:#66512c}.alert-danger{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.alert-danger hr{border-top-color:#e4b9c0}.alert-danger .alert-link{color:#843534}@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@-o-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}.progress{height:20px;margin-bottom:20px;overflow:hidden;background-color:#f5f5f5;border-radius:4px;-webkit-box-shadow:inset 0 1px 2px rgba(0,0,0,.1);box-shadow:inset 0 1px 2px rgba(0,0,0,.1)}.progress-bar{float:left;width:0;height:100%;font-size:12px;line-height:20px;color:#fff;text-align:center;background-color:#337ab7;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);-webkit-transition:width .6s ease;-o-transition:width .6s ease;transition:width .6s ease}.progress-bar-striped,.progress-striped .progress-bar{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);-webkit-background-size:40px 40px;background-size:40px 40px}.progress-bar.active,.progress.active .progress-bar{-webkit-animation:progress-bar-stripes 2s linear infinite;-o-animation:progress-bar-stripes 2s linear infinite;animation:progress-bar-stripes 2s linear infinite}.progress-bar-success{background-color:#5cb85c}.progress-striped .progress-bar-success{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-info{background-color:#5bc0de}.progress-striped .progress-bar-info{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-warning{background-color:#f0ad4e}.progress-striped .progress-bar-warning{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-danger{background-color:#d9534f}.progress-striped .progress-bar-danger{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.media{margin-top:15px}.media:first-child{margin-top:0}.media,.media-body{overflow:hidden;zoom:1}.media-body{width:10000px}.media-object{display:block}.media-object.img-thumbnail{max-width:none}.media-right,.media>.pull-right{padding-left:10px}.media-left,.media>.pull-left{padding-right:10px}.media-body,.media-left,.media-right{display:table-cell;vertical-align:top}.media-middle{vertical-align:middle}.media-bottom{vertical-align:bottom}.media-heading{margin-top:0;margin-bottom:5px}.media-list{padding-left:0;list-style:none}.list-group{padding-left:0;margin-bottom:20px}.list-group-item{position:relative;display:block;padding:10px 15px;margin-bottom:-1px;background-color:#fff;border:1px solid #ddd}.list-group-item:first-child{border-top-left-radius:4px;border-top-right-radius:4px}.list-group-item:last-child{margin-bottom:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}a.list-group-item,button.list-group-item{color:#555}a.list-group-item .list-group-item-heading,button.list-group-item .list-group-item-heading{color:#333}a.list-group-item:focus,a.list-group-item:hover,button.list-group-item:focus,button.list-group-item:hover{color:#555;text-decoration:none;background-color:#f5f5f5}button.list-group-item{width:100%;text-align:left}.list-group-item.disabled,.list-group-item.disabled:focus,.list-group-item.disabled:hover{color:#777;cursor:not-allowed;background-color:#eee}.list-group-item.disabled .list-group-item-heading,.list-group-item.disabled:focus .list-group-item-heading,.list-group-item.disabled:hover .list-group-item-heading{color:inherit}.list-group-item.disabled .list-group-item-text,.list-group-item.disabled:focus .list-group-item-text,.list-group-item.disabled:hover .list-group-item-text{color:#777}.list-group-item.active,.list-group-item.active:focus,.list-group-item.active:hover{z-index:2;color:#fff;background-color:#337ab7;border-color:#337ab7}.list-group-item.active .list-group-item-heading,.list-group-item.active .list-group-item-heading>.small,.list-group-item.active .list-group-item-heading>small,.list-group-item.active:focus .list-group-item-heading,.list-group-item.active:focus .list-group-item-heading>.small,.list-group-item.active:focus .list-group-item-heading>small,.list-group-item.active:hover .list-group-item-heading,.list-group-item.active:hover .list-group-item-heading>.small,.list-group-item.active:hover .list-group-item-heading>small{color:inherit}.list-group-item.active .list-group-item-text,.list-group-item.active:focus .list-group-item-text,.list-group-item.active:hover .list-group-item-text{color:#c7ddef}.list-group-item-success{color:#3c763d;background-color:#dff0d8}a.list-group-item-success,button.list-group-item-success{color:#3c763d}a.list-group-item-success .list-group-item-heading,button.list-group-item-success .list-group-item-heading{color:inherit}a.list-group-item-success:focus,a.list-group-item-success:hover,button.list-group-item-success:focus,button.list-group-item-success:hover{color:#3c763d;background-color:#d0e9c6}a.list-group-item-success.active,a.list-group-item-success.active:focus,a.list-group-item-success.active:hover,button.list-group-item-success.active,button.list-group-item-success.active:focus,button.list-group-item-success.active:hover{color:#fff;background-color:#3c763d;border-color:#3c763d}.list-group-item-info{color:#31708f;background-color:#d9edf7}a.list-group-item-info,button.list-group-item-info{color:#31708f}a.list-group-item-info .list-group-item-heading,button.list-group-item-info .list-group-item-heading{color:inherit}a.list-group-item-info:focus,a.list-group-item-info:hover,button.list-group-item-info:focus,button.list-group-item-info:hover{color:#31708f;background-color:#c4e3f3}a.list-group-item-info.active,a.list-group-item-info.active:focus,a.list-group-item-info.active:hover,button.list-group-item-info.active,button.list-group-item-info.active:focus,button.list-group-item-info.active:hover{color:#fff;background-color:#31708f;border-color:#31708f}.list-group-item-warning{color:#8a6d3b;background-color:#fcf8e3}a.list-group-item-warning,button.list-group-item-warning{color:#8a6d3b}a.list-group-item-warning .list-group-item-heading,button.list-group-item-warning .list-group-item-heading{color:inherit}a.list-group-item-warning:focus,a.list-group-item-warning:hover,button.list-group-item-warning:focus,button.list-group-item-warning:hover{color:#8a6d3b;background-color:#faf2cc}a.list-group-item-warning.active,a.list-group-item-warning.active:focus,a.list-group-item-warning.active:hover,button.list-group-item-warning.active,button.list-group-item-warning.active:focus,button.list-group-item-warning.active:hover{color:#fff;background-color:#8a6d3b;border-color:#8a6d3b}.list-group-item-danger{color:#a94442;background-color:#f2dede}a.list-group-item-danger,button.list-group-item-danger{color:#a94442}a.list-group-item-danger .list-group-item-heading,button.list-group-item-danger .list-group-item-heading{color:inherit}a.list-group-item-danger:focus,a.list-group-item-danger:hover,button.list-group-item-danger:focus,button.list-group-item-danger:hover{color:#a94442;background-color:#ebcccc}a.list-group-item-danger.active,a.list-group-item-danger.active:focus,a.list-group-item-danger.active:hover,button.list-group-item-danger.active,button.list-group-item-danger.active:focus,button.list-group-item-danger.active:hover{color:#fff;background-color:#a94442;border-color:#a94442}.list-group-item-heading{margin-top:0;margin-bottom:5px}.list-group-item-text{margin-bottom:0;line-height:1.3}.panel{margin-bottom:20px;background-color:#fff;border:1px solid transparent;border-radius:4px;-webkit-box-shadow:0 1px 1px rgba(0,0,0,.05);box-shadow:0 1px 1px rgba(0,0,0,.05)}.panel-body{padding:15px}.panel-heading{padding:10px 15px;border-bottom:1px solid transparent;border-top-left-radius:3px;border-top-right-radius:3px}.panel-heading>.dropdown .dropdown-toggle{color:inherit}.panel-title{margin-top:0;margin-bottom:0;font-size:16px;color:inherit}.panel-title>.small,.panel-title>.small>a,.panel-title>a,.panel-title>small,.panel-title>small>a{color:inherit}.panel-footer{padding:10px 15px;background-color:#f5f5f5;border-top:1px solid #ddd;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.list-group,.panel>.panel-collapse>.list-group{margin-bottom:0}.panel>.list-group .list-group-item,.panel>.panel-collapse>.list-group .list-group-item{border-width:1px 0;border-radius:0}.panel>.list-group:first-child .list-group-item:first-child,.panel>.panel-collapse>.list-group:first-child .list-group-item:first-child{border-top:0;border-top-left-radius:3px;border-top-right-radius:3px}.panel>.list-group:last-child .list-group-item:last-child,.panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{border-bottom:0;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.panel-heading+.panel-collapse>.list-group .list-group-item:first-child{border-top-left-radius:0;border-top-right-radius:0}.panel-heading+.list-group .list-group-item:first-child{border-top-width:0}.list-group+.panel-footer{border-top-width:0}.panel>.panel-collapse>.table,.panel>.table,.panel>.table-responsive>.table{margin-bottom:0}.panel>.panel-collapse>.table caption,.panel>.table caption,.panel>.table-responsive>.table caption{padding-right:15px;padding-left:15px}.panel>.table-responsive:first-child>.table:first-child,.panel>.table:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child,.panel>.table:first-child>thead:first-child>tr:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table:first-child>thead:first-child>tr:first-child th:first-child{border-top-left-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table:first-child>thead:first-child>tr:first-child th:last-child{border-top-right-radius:3px}.panel>.table-responsive:last-child>.table:last-child,.panel>.table:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:first-child{border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:last-child{border-bottom-right-radius:3px}.panel>.panel-body+.table,.panel>.panel-body+.table-responsive,.panel>.table+.panel-body,.panel>.table-responsive+.panel-body{border-top:1px solid #ddd}.panel>.table>tbody:first-child>tr:first-child td,.panel>.table>tbody:first-child>tr:first-child th{border-top:0}.panel>.table-bordered,.panel>.table-responsive>.table-bordered{border:0}.panel>.table-bordered>tbody>tr>td:first-child,.panel>.table-bordered>tbody>tr>th:first-child,.panel>.table-bordered>tfoot>tr>td:first-child,.panel>.table-bordered>tfoot>tr>th:first-child,.panel>.table-bordered>thead>tr>td:first-child,.panel>.table-bordered>thead>tr>th:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:first-child,.panel>.table-responsive>.table-bordered>thead>tr>td:first-child,.panel>.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.panel>.table-bordered>tbody>tr>td:last-child,.panel>.table-bordered>tbody>tr>th:last-child,.panel>.table-bordered>tfoot>tr>td:last-child,.panel>.table-bordered>tfoot>tr>th:last-child,.panel>.table-bordered>thead>tr>td:last-child,.panel>.table-bordered>thead>tr>th:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:last-child,.panel>.table-responsive>.table-bordered>thead>tr>td:last-child,.panel>.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.panel>.table-bordered>tbody>tr:first-child>td,.panel>.table-bordered>tbody>tr:first-child>th,.panel>.table-bordered>thead>tr:first-child>td,.panel>.table-bordered>thead>tr:first-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>th,.panel>.table-responsive>.table-bordered>thead>tr:first-child>td,.panel>.table-responsive>.table-bordered>thead>tr:first-child>th{border-bottom:0}.panel>.table-bordered>tbody>tr:last-child>td,.panel>.table-bordered>tbody>tr:last-child>th,.panel>.table-bordered>tfoot>tr:last-child>td,.panel>.table-bordered>tfoot>tr:last-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>th,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>td,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}.panel>.table-responsive{margin-bottom:0;border:0}.panel-group{margin-bottom:20px}.panel-group .panel{margin-bottom:0;border-radius:4px}.panel-group .panel+.panel{margin-top:5px}.panel-group .panel-heading{border-bottom:0}.panel-group .panel-heading+.panel-collapse>.list-group,.panel-group .panel-heading+.panel-collapse>.panel-body{border-top:1px solid #ddd}.panel-group .panel-footer{border-top:0}.panel-group .panel-footer+.panel-collapse .panel-body{border-bottom:1px solid #ddd}.panel-default{border-color:#ddd}.panel-default>.panel-heading{color:#333;background-color:#f5f5f5;border-color:#ddd}.panel-default>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ddd}.panel-default>.panel-heading .badge{color:#f5f5f5;background-color:#333}.panel-default>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ddd}.panel-primary{border-color:#337ab7}.panel-primary>.panel-heading{color:#fff;background-color:#337ab7;border-color:#337ab7}.panel-primary>.panel-heading+.panel-collapse>.panel-body{border-top-color:#337ab7}.panel-primary>.panel-heading .badge{color:#337ab7;background-color:#fff}.panel-primary>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#337ab7}.panel-success{border-color:#d6e9c6}.panel-success>.panel-heading{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.panel-success>.panel-heading+.panel-collapse>.panel-body{border-top-color:#d6e9c6}.panel-success>.panel-heading .badge{color:#dff0d8;background-color:#3c763d}.panel-success>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#d6e9c6}.panel-info{border-color:#bce8f1}.panel-info>.panel-heading{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.panel-info>.panel-heading+.panel-collapse>.panel-body{border-top-color:#bce8f1}.panel-info>.panel-heading .badge{color:#d9edf7;background-color:#31708f}.panel-info>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#bce8f1}.panel-warning{border-color:#faebcc}.panel-warning>.panel-heading{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.panel-warning>.panel-heading+.panel-collapse>.panel-body{border-top-color:#faebcc}.panel-warning>.panel-heading .badge{color:#fcf8e3;background-color:#8a6d3b}.panel-warning>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#faebcc}.panel-danger{border-color:#ebccd1}.panel-danger>.panel-heading{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.panel-danger>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ebccd1}.panel-danger>.panel-heading .badge{color:#f2dede;background-color:#a94442}.panel-danger>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ebccd1}.embed-responsive{position:relative;display:block;height:0;padding:0;overflow:hidden}.embed-responsive .embed-responsive-item,.embed-responsive embed,.embed-responsive iframe,.embed-responsive object,.embed-responsive video{position:absolute;top:0;bottom:0;left:0;width:100%;height:100%;border:0}.embed-responsive-16by9{padding-bottom:56.25%}.embed-responsive-4by3{padding-bottom:75%}.well{min-height:20px;padding:19px;margin-bottom:20px;background-color:#f5f5f5;border:1px solid #e3e3e3;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.05);box-shadow:inset 0 1px 1px rgba(0,0,0,.05)}.well blockquote{border-color:#ddd;border-color:rgba(0,0,0,.15)}.well-lg{padding:24px;border-radius:6px}.well-sm{padding:9px;border-radius:3px}.close{float:right;font-size:21px;font-weight:700;line-height:1;color:#000;text-shadow:0 1px 0 #fff;filter:alpha(opacity=20);opacity:.2}.close:focus,.close:hover{color:#000;text-decoration:none;cursor:pointer;filter:alpha(opacity=50);opacity:.5}button.close{-webkit-appearance:none;padding:0;cursor:pointer;background:0 0;border:0}.modal-open{overflow:hidden}.modal{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1050;display:none;overflow:hidden;-webkit-overflow-scrolling:touch;outline:0}.modal.fade .modal-dialog{-webkit-transition:-webkit-transform .3s ease-out;-o-transition:-o-transform .3s ease-out;transition:transform .3s ease-out;-webkit-transform:translate(0,-25%);-ms-transform:translate(0,-25%);-o-transform:translate(0,-25%);transform:translate(0,-25%)}.modal.in .modal-dialog{-webkit-transform:translate(0,0);-ms-transform:translate(0,0);-o-transform:translate(0,0);transform:translate(0,0)}.modal-open .modal{overflow-x:hidden;overflow-y:auto}.modal-dialog{position:relative;width:auto;margin:10px}.modal-content{position:relative;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #999;border:1px solid rgba(0,0,0,.2);border-radius:6px;outline:0;-webkit-box-shadow:0 3px 9px rgba(0,0,0,.5);box-shadow:0 3px 9px rgba(0,0,0,.5)}.modal-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1040;background-color:#000}.modal-backdrop.fade{filter:alpha(opacity=0);opacity:0}.modal-backdrop.in{filter:alpha(opacity=50);opacity:.5}.modal-header{padding:15px;border-bottom:1px solid #e5e5e5}.modal-header .close{margin-top:-2px}.modal-title{margin:0;line-height:1.42857143}.modal-body{position:relative;padding:15px}.modal-footer{padding:15px;text-align:right;border-top:1px solid #e5e5e5}.modal-footer .btn+.btn{margin-bottom:0;margin-left:5px}.modal-footer .btn-group .btn+.btn{margin-left:-1px}.modal-footer .btn-block+.btn-block{margin-left:0}.modal-scrollbar-measure{position:absolute;top:-9999px;width:50px;height:50px;overflow:scroll}@media (min-width:768px){.modal-dialog{width:600px;margin:30px auto}.modal-content{-webkit-box-shadow:0 5px 15px rgba(0,0,0,.5);box-shadow:0 5px 15px rgba(0,0,0,.5)}.modal-sm{width:300px}}@media (min-width:992px){.modal-lg{width:900px}}.tooltip{position:absolute;z-index:1070;display:block;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:12px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;filter:alpha(opacity=0);opacity:0;line-break:auto}.tooltip.in{filter:alpha(opacity=90);opacity:.9}.tooltip.top{padding:5px 0;margin-top:-3px}.tooltip.right{padding:0 5px;margin-left:3px}.tooltip.bottom{padding:5px 0;margin-top:3px}.tooltip.left{padding:0 5px;margin-left:-3px}.tooltip-inner{max-width:200px;padding:3px 8px;color:#fff;text-align:center;background-color:#000;border-radius:4px}.tooltip-arrow{position:absolute;width:0;height:0;border-color:transparent;border-style:solid}.tooltip.top .tooltip-arrow{bottom:0;left:50%;margin-left:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-left .tooltip-arrow{right:5px;bottom:0;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-right .tooltip-arrow{bottom:0;left:5px;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.right .tooltip-arrow{top:50%;left:0;margin-top:-5px;border-width:5px 5px 5px 0;border-right-color:#000}.tooltip.left .tooltip-arrow{top:50%;right:0;margin-top:-5px;border-width:5px 0 5px 5px;border-left-color:#000}.tooltip.bottom .tooltip-arrow{top:0;left:50%;margin-left:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-left .tooltip-arrow{top:0;right:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-right .tooltip-arrow{top:0;left:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.popover{position:absolute;top:0;left:0;z-index:1060;display:none;max-width:276px;padding:1px;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.2);border-radius:6px;-webkit-box-shadow:0 5px 10px rgba(0,0,0,.2);box-shadow:0 5px 10px rgba(0,0,0,.2);line-break:auto}.popover.top{margin-top:-10px}.popover.right{margin-left:10px}.popover.bottom{margin-top:10px}.popover.left{margin-left:-10px}.popover-title{padding:8px 14px;margin:0;font-size:14px;background-color:#f7f7f7;border-bottom:1px solid #ebebeb;border-radius:5px 5px 0 0}.popover-content{padding:9px 14px}.popover>.arrow,.popover>.arrow:after{position:absolute;display:block;width:0;height:0;border-color:transparent;border-style:solid}.popover>.arrow{border-width:11px}.popover>.arrow:after{content:"";border-width:10px}.popover.top>.arrow{bottom:-11px;left:50%;margin-left:-11px;border-top-color:#999;border-top-color:rgba(0,0,0,.25);border-bottom-width:0}.popover.top>.arrow:after{bottom:1px;margin-left:-10px;content:" ";border-top-color:#fff;border-bottom-width:0}.popover.right>.arrow{top:50%;left:-11px;margin-top:-11px;border-right-color:#999;border-right-color:rgba(0,0,0,.25);border-left-width:0}.popover.right>.arrow:after{bottom:-10px;left:1px;content:" ";border-right-color:#fff;border-left-width:0}.popover.bottom>.arrow{top:-11px;left:50%;margin-left:-11px;border-top-width:0;border-bottom-color:#999;border-bottom-color:rgba(0,0,0,.25)}.popover.bottom>.arrow:after{top:1px;margin-left:-10px;content:" ";border-top-width:0;border-bottom-color:#fff}.popover.left>.arrow{top:50%;right:-11px;margin-top:-11px;border-right-width:0;border-left-color:#999;border-left-color:rgba(0,0,0,.25)}.popover.left>.arrow:after{right:1px;bottom:-10px;content:" ";border-right-width:0;border-left-color:#fff}.carousel{position:relative}.carousel-inner{position:relative;width:100%;overflow:hidden}.carousel-inner>.item{position:relative;display:none;-webkit-transition:.6s ease-in-out left;-o-transition:.6s ease-in-out left;transition:.6s ease-in-out left}.carousel-inner>.item>a>img,.carousel-inner>.item>img{line-height:1}@media all and (transform-3d),(-webkit-transform-3d){.carousel-inner>.item{-webkit-transition:-webkit-transform .6s ease-in-out;-o-transition:-o-transform .6s ease-in-out;transition:transform .6s ease-in-out;-webkit-backface-visibility:hidden;backface-visibility:hidden;-webkit-perspective:1000px;perspective:1000px}.carousel-inner>.item.active.right,.carousel-inner>.item.next{left:0;-webkit-transform:translate3d(100%,0,0);transform:translate3d(100%,0,0)}.carousel-inner>.item.active.left,.carousel-inner>.item.prev{left:0;-webkit-transform:translate3d(-100%,0,0);transform:translate3d(-100%,0,0)}.carousel-inner>.item.active,.carousel-inner>.item.next.left,.carousel-inner>.item.prev.right{left:0;-webkit-transform:translate3d(0,0,0);transform:translate3d(0,0,0)}}.carousel-inner>.active,.carousel-inner>.next,.carousel-inner>.prev{display:block}.carousel-inner>.active{left:0}.carousel-inner>.next,.carousel-inner>.prev{position:absolute;top:0;width:100%}.carousel-inner>.next{left:100%}.carousel-inner>.prev{left:-100%}.carousel-inner>.next.left,.carousel-inner>.prev.right{left:0}.carousel-inner>.active.left{left:-100%}.carousel-inner>.active.right{left:100%}.carousel-control{position:absolute;top:0;bottom:0;left:0;width:15%;font-size:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6);background-color:rgba(0,0,0,0);filter:alpha(opacity=50);opacity:.5}.carousel-control.left{background-image:-webkit-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.5)),to(rgba(0,0,0,.0001)));background-image:linear-gradient(to right,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#80000000', endColorstr='#00000000', GradientType=1);background-repeat:repeat-x}.carousel-control.right{right:0;left:auto;background-image:-webkit-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.0001)),to(rgba(0,0,0,.5)));background-image:linear-gradient(to right,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#00000000', endColorstr='#80000000', GradientType=1);background-repeat:repeat-x}.carousel-control:focus,.carousel-control:hover{color:#fff;text-decoration:none;filter:alpha(opacity=90);outline:0;opacity:.9}.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{position:absolute;top:50%;z-index:5;display:inline-block;margin-top:-10px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{left:50%;margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{right:50%;margin-right:-10px}.carousel-control .icon-next,.carousel-control .icon-prev{width:20px;height:20px;font-family:serif;line-height:1}.carousel-control .icon-prev:before{content:'\2039'}.carousel-control .icon-next:before{content:'\203a'}.carousel-indicators{position:absolute;bottom:10px;left:50%;z-index:15;width:60%;padding-left:0;margin-left:-30%;text-align:center;list-style:none}.carousel-indicators li{display:inline-block;width:10px;height:10px;margin:1px;text-indent:-999px;cursor:pointer;background-color:#000\9;background-color:rgba(0,0,0,0);border:1px solid #fff;border-radius:10px}.carousel-indicators .active{width:12px;height:12px;margin:0;background-color:#fff}.carousel-caption{position:absolute;right:15%;bottom:20px;left:15%;z-index:10;padding-top:20px;padding-bottom:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6)}.carousel-caption .btn{text-shadow:none}@media screen and (min-width:768px){.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{width:30px;height:30px;margin-top:-10px;font-size:30px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{margin-right:-10px}.carousel-caption{right:20%;left:20%;padding-bottom:30px}.carousel-indicators{bottom:20px}}.btn-group-vertical>.btn-group:after,.btn-group-vertical>.btn-group:before,.btn-toolbar:after,.btn-toolbar:before,.clearfix:after,.clearfix:before,.container-fluid:after,.container-fluid:before,.container:after,.container:before,.dl-horizontal dd:after,.dl-horizontal dd:before,.form-horizontal .form-group:after,.form-horizontal .form-group:before,.modal-footer:after,.modal-footer:before,.modal-header:after,.modal-header:before,.nav:after,.nav:before,.navbar-collapse:after,.navbar-collapse:before,.navbar-header:after,.navbar-header:before,.navbar:after,.navbar:before,.pager:after,.pager:before,.panel-body:after,.panel-body:before,.row:after,.row:before{display:table;content:" "}.btn-group-vertical>.btn-group:after,.btn-toolbar:after,.clearfix:after,.container-fluid:after,.container:after,.dl-horizontal dd:after,.form-horizontal .form-group:after,.modal-footer:after,.modal-header:after,.nav:after,.navbar-collapse:after,.navbar-header:after,.navbar:after,.pager:after,.panel-body:after,.row:after{clear:both}.center-block{display:block;margin-right:auto;margin-left:auto}.pull-right{float:right!important}.pull-left{float:left!important}.hide{display:none!important}.show{display:block!important}.invisible{visibility:hidden}.text-hide{font:0/0 a;color:transparent;text-shadow:none;background-color:transparent;border:0}.hidden{display:none!important}.affix{position:fixed}@-ms-viewport{width:device-width}.visible-lg,.visible-md,.visible-sm,.visible-xs{display:none!important}.visible-lg-block,.visible-lg-inline,.visible-lg-inline-block,.visible-md-block,.visible-md-inline,.visible-md-inline-block,.visible-sm-block,.visible-sm-inline,.visible-sm-inline-block,.visible-xs-block,.visible-xs-inline,.visible-xs-inline-block{display:none!important}@media (max-width:767px){.visible-xs{display:block!important}table.visible-xs{display:table!important}tr.visible-xs{display:table-row!important}td.visible-xs,th.visible-xs{display:table-cell!important}}@media (max-width:767px){.visible-xs-block{display:block!important}}@media (max-width:767px){.visible-xs-inline{display:inline!important}}@media (max-width:767px){.visible-xs-inline-block{display:inline-block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm{display:block!important}table.visible-sm{display:table!important}tr.visible-sm{display:table-row!important}td.visible-sm,th.visible-sm{display:table-cell!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-block{display:block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline{display:inline!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline-block{display:inline-block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md{display:block!important}table.visible-md{display:table!important}tr.visible-md{display:table-row!important}td.visible-md,th.visible-md{display:table-cell!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-block{display:block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline{display:inline!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline-block{display:inline-block!important}}@media (min-width:1200px){.visible-lg{display:block!important}table.visible-lg{display:table!important}tr.visible-lg{display:table-row!important}td.visible-lg,th.visible-lg{display:table-cell!important}}@media (min-width:1200px){.visible-lg-block{display:block!important}}@media (min-width:1200px){.visible-lg-inline{display:inline!important}}@media (min-width:1200px){.visible-lg-inline-block{display:inline-block!important}}@media (max-width:767px){.hidden-xs{display:none!important}}@media (min-width:768px) and (max-width:991px){.hidden-sm{display:none!important}}@media (min-width:992px) and (max-width:1199px){.hidden-md{display:none!important}}@media (min-width:1200px){.hidden-lg{display:none!important}}.visible-print{display:none!important}@media print{.visible-print{display:block!important}table.visible-print{display:table!important}tr.visible-print{display:table-row!important}td.visible-print,th.visible-print{display:table-cell!important}}.visible-print-block{display:none!important}@media print{.visible-print-block{display:block!important}}.visible-print-inline{display:none!important}@media print{.visible-print-inline{display:inline!important}}.visible-print-inline-block{display:none!important}@media print{.visible-print-inline-block{display:inline-block!important}}@media print{.hidden-print{display:none!important}}
        /*# sourceMappingURL=bootstrap.min.css.map */
    </style>
    <!--<link rel="stylesheet" type="text/css" media="all" href="../../resources/bootstrap-3.2.0/css/bootstrap.min.css" th:href="@{resources/bootstrap-3.2.0/css/bootstrap.min.css}"/>-->
</head>
<body>
<div id="content" class="container">
    <h2>
        Exomiser Analysis Results for
        proband
    </h2>
    <ul class="nav nav-pills" id="tabs">
        <li><a href="#settings" role="tab" data-toggle="tab"><b>Analysis Settings</b></a></li>
        <li><a href="#filtering" role="tab" data-toggle="tab"><b>Filtering Summary</b></a></li>
        <li><a href="#distribution" role="tab" data-toggle="tab"><b>Variant Type Distribution</b></a></li>
        <li><a href="#prioritisation" role="tab" data-toggle="tab"><b>Prioritised Genes</b></a></li>
        
        <li><a href="#about" role="tab" data-toggle="tab"><b>About</b></a></li>
    </ul>
    <div class="panel panel-default" id="settings">
        <div class="panel-heading">
            <h3>Analysis Settings</h3>
        </div>
        <div class="panel-body"><p>Settings used for this analysis:</p>
            <pre>---
vcf: &quot;/mnt/scratch/Public/TRAINING/GenomeAnalysisModule/CaseInformation/CaseFiles/Case2/Case2_Merged.vcf.gz&quot;
genomeAssembly: &quot;HG38&quot;
pedigree:
  individuals:
  - familyId: &quot;FAM&quot;
    id: &quot;proband&quot;
    motherId: &quot;mother&quot;
    fatherId: &quot;father&quot;
    sex: &quot;MALE&quot;
    status: &quot;AFFECTED&quot;
  - familyId: &quot;FAM&quot;
    id: &quot;mother&quot;
    motherId: &quot;&quot;
    fatherId: &quot;&quot;
    sex: &quot;FEMALE&quot;
    status: &quot;UNAFFECTED&quot;
  - familyId: &quot;FAM&quot;
    id: &quot;father&quot;
    motherId: &quot;&quot;
    fatherId: &quot;&quot;
    sex: &quot;MALE&quot;
    status: &quot;UNAFFECTED&quot;
proband: &quot;proband&quot;
hpoIds:
- &quot;HP:0040275&quot;
- &quot;HP:0012378&quot;
- &quot;HP:0002027&quot;
inheritanceModes:
  maxFreq: 2.0
  empty: false
  definedSubModes:
  - &quot;AUTOSOMAL_DOMINANT&quot;
  - &quot;AUTOSOMAL_RECESSIVE_COMP_HET&quot;
  - &quot;AUTOSOMAL_RECESSIVE_HOM_ALT&quot;
  - &quot;X_RECESSIVE_COMP_HET&quot;
  - &quot;X_RECESSIVE_HOM_ALT&quot;
  - &quot;X_DOMINANT&quot;
  - &quot;MITOCHONDRIAL&quot;
  definedModes:
  - &quot;AUTOSOMAL_DOMINANT&quot;
  - &quot;AUTOSOMAL_RECESSIVE&quot;
  - &quot;X_RECESSIVE&quot;
  - &quot;X_DOMINANT&quot;
  - &quot;MITOCHONDRIAL&quot;
analysisMode: &quot;PASS_ONLY&quot;
frequencySources:
- &quot;THOUSAND_GENOMES&quot;
- &quot;TOPMED&quot;
- &quot;UK10K&quot;
- &quot;ESP_AFRICAN_AMERICAN&quot;
- &quot;ESP_EUROPEAN_AMERICAN&quot;
- &quot;ESP_ALL&quot;
- &quot;EXAC_AFRICAN_INC_AFRICAN_AMERICAN&quot;
- &quot;EXAC_AMERICAN&quot;
- &quot;EXAC_EAST_ASIAN&quot;
- &quot;EXAC_FINNISH&quot;
- &quot;EXAC_NON_FINNISH_EUROPEAN&quot;
- &quot;EXAC_OTHER&quot;
- &quot;EXAC_SOUTH_ASIAN&quot;
- &quot;GNOMAD_E_AFR&quot;
- &quot;GNOMAD_E_AMR&quot;
- &quot;GNOMAD_E_ASJ&quot;
- &quot;GNOMAD_E_EAS&quot;
- &quot;GNOMAD_E_FIN&quot;
- &quot;GNOMAD_E_NFE&quot;
- &quot;GNOMAD_E_OTH&quot;
- &quot;GNOMAD_E_SAS&quot;
- &quot;GNOMAD_G_AFR&quot;
- &quot;GNOMAD_G_AMR&quot;
- &quot;GNOMAD_G_ASJ&quot;
- &quot;GNOMAD_G_EAS&quot;
- &quot;GNOMAD_G_FIN&quot;
- &quot;GNOMAD_G_NFE&quot;
- &quot;GNOMAD_G_OTH&quot;
- &quot;GNOMAD_G_SAS&quot;
pathogenicitySources:
- &quot;POLYPHEN&quot;
- &quot;MUTATION_TASTER&quot;
- &quot;SIFT&quot;
- &quot;CADD&quot;
analysisSteps:
- offTargetVariantTypes:
  - &quot;CODING_TRANSCRIPT_INTRON_VARIANT&quot;
  - &quot;FIVE_PRIME_UTR_EXON_VARIANT&quot;
  - &quot;THREE_PRIME_UTR_EXON_VARIANT&quot;
  - &quot;FIVE_PRIME_UTR_INTRON_VARIANT&quot;
  - &quot;THREE_PRIME_UTR_INTRON_VARIANT&quot;
  - &quot;NON_CODING_TRANSCRIPT_EXON_VARIANT&quot;
  - &quot;NON_CODING_TRANSCRIPT_INTRON_VARIANT&quot;
  - &quot;UPSTREAM_GENE_VARIANT&quot;
  - &quot;DOWNSTREAM_GENE_VARIANT&quot;
  - &quot;INTERGENIC_VARIANT&quot;
  - &quot;REGULATORY_REGION_VARIANT&quot;
- decoratedFilter:
    maxFreq: 2.0
- decoratedFilter:
    keepNonPathogenic: true
- compatibleModes:
  - &quot;AUTOSOMAL_DOMINANT&quot;
  - &quot;AUTOSOMAL_RECESSIVE&quot;
  - &quot;X_RECESSIVE&quot;
  - &quot;X_DOMINANT&quot;
  - &quot;MITOCHONDRIAL&quot;
- priorityType: &quot;OMIM_PRIORITY&quot;
- priorityType: &quot;HIPHIVE_PRIORITY&quot;
---
outputContributingVariantsOnly: false
outputPrefix: &quot;Case2_exomiser&quot;
outputFormats:
- &quot;HTML&quot;
- &quot;VCF&quot;
- &quot;TSV_GENE&quot;
- &quot;TSV_VARIANT&quot;
- &quot;JSON&quot;
numGenes: 0
</pre>
        </div>
    </div>

    <div class="panel panel-default" id="filtering">
        <div class="panel-heading">
            <h3>Filtering Summary</h3>
        </div>
        <div class="panel-body">
            <table class="table">
                <thead>
                <tr>
                    <th>Filter</th>
                    <th>Report</th>
                    <th>Passed filter</th>
                    <th>Failed filter</th>
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td><b>Variant effect</b></td>
                    <td>
                        <ul>Removed variants with effects of type: [CODING_TRANSCRIPT_INTRON_VARIANT, FIVE_PRIME_UTR_EXON_VARIANT, THREE_PRIME_UTR_EXON_VARIANT, FIVE_PRIME_UTR_INTRON_VARIANT, THREE_PRIME_UTR_INTRON_VARIANT, NON_CODING_TRANSCRIPT_EXON_VARIANT, NON_CODING_TRANSCRIPT_INTRON_VARIANT, UPSTREAM_GENE_VARIANT, DOWNSTREAM_GENE_VARIANT, INTERGENIC_VARIANT, REGULATORY_REGION_VARIANT]</ul>
                    </td>
                    <td>250</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Frequency</b></td>
                    <td>
                        <ul>Variants filtered for maximum allele frequency of 2.00%</ul>
                    </td>
                    <td>250</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Pathogenicity</b></td>
                    <td>
                        <ul>Retained all non-pathogenic variants of all types. Scoring was applied, but the filter passed all variants.</ul>
                    </td>
                    <td>250</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Inheritance</b></td>
                    <td>
                        <ul>Genes filtered for compatibility with AUTOSOMAL_DOMINANT, AUTOSOMAL_RECESSIVE, X_RECESSIVE, X_DOMINANT, MITOCHONDRIAL inheritance.</ul>
                    </td>
                    <td>126</td>
                    <td>0</td>
                </tr>
                </tbody>
            </table>
        </div>
    </div>
    <div class="panel panel-default" id="distribution">
        <div class="panel-heading">
            <h3>Variant Type Distribution</h3>
        </div>
        <div class="panel-body">
            <table class="table-striped table-condensed">
                <thead>
                <tr>
                    <th>Variant Type</th>
                    
                        <th>proband</th>
                    
                        <th>mother</th>
                    
                        <th>father</th>
                    
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td>FRAMESHIFT_ELONGATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_TRUNCATION</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_VARIANT</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INTERNAL_FEATURE_ELONGATION</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FEATURE_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>MNV</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>STOP_GAINED</td>
                    
                        <td>4</td>
                    
                        <td>3</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>STOP_LOST</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>START_LOST</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_ACCEPTOR_VARIANT</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_DONOR_VARIANT</td>
                    
                        <td>3</td>
                    
                        <td>0</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>MISSENSE_VARIANT</td>
                    
                        <td>150</td>
                    
                        <td>53</td>
                    
                        <td>61</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_INSERTION</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_INSERTION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_DELETION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_DELETION</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_REGION_VARIANT</td>
                    
                        <td>14</td>
                    
                        <td>6</td>
                    
                        <td>3</td>
                    
                </tr>
                <tr>
                    <td>STOP_RETAINED_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INITIATOR_CODON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SYNONYMOUS_VARIANT</td>
                    
                        <td>72</td>
                    
                        <td>32</td>
                    
                        <td>30</td>
                    
                </tr>
                <tr>
                    <td>CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>UPSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DOWNSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INTERGENIC_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>REGULATORY_REGION_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                </tbody>
            </table>
        </div>
    </div>

    

    <div class="panel panel-default" id="prioritisation">
        <div class="panel-heading">
            <h3>Prioritised Genes</h3>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000095002">MSH2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.965</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.715</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.617 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=144">Lynch syndrome</a> associated with MSH2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0003003, Colon cancer</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl><dt>Phenotypic similarity 0.715 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MSH2">MSH2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0002957, increased intestinal adenocarcinoma incidence</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0002957, increased intestinal adenocarcinoma incidence</dd></dl><dl><dt>Proximity score 0.515 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MSH2%0DMUTYH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MUTYH</a> and phenotypic similarity 0.807 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=247798">MUTYH-related attenuated familial adenomatous polyposis</a> associated with MUTYH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0040276, Adenocarcinoma of the colon</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/120435">OMIM:120435</a> Colorectal cancer, hereditary nonpolyposis, type 1 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/158320">OMIM:158320</a> Muir-Torre syndrome - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/276300">OMIM:276300</a> Mismatch repair cancer syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=144">ORPHA:144</a> Lynch syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=587">ORPHA:587</a> Muir-Torre syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.965</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.715</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr2:g.47403219C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=63751099">rs63751099</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs63751099&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-danger">PATHOGENIC</span>
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=96530%5Balleleid%5D">(reviewed_by_expert_panel)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                <span class="label label-danger">WHITELIST VARIANT</span>
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000233146.7">MSH2:ENST00000233146.7:c.28C&gt;T:p.(Gln10*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000406134.5">MSH2:ENST00000406134.5:c.28C&gt;T:p.(Gln10*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000645506.1">MSH2:ENST00000645506.1:c.28C&gt;T:p.(Gln10*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543555.6">MSH2:ENST00000543555.6:c.-31+44C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 1.000 (42.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000078328">RBFOX1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.759</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.998</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RBFOX1%0DATXN2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ATXN2</a> and phenotypic similarity 0.605 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=803">Amyotrophic lateral sclerosis</a> associated with ATXN2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002017, Nausea and vomiting</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RBFOX1%0DATXN2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ATXN2</a> and phenotypic similarity 0.254 to mouse mutant of ATXN2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0005415, intrahepatic cholestasis</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0005415, intrahepatic cholestasis</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.759</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.998</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.7333041G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=778728291">rs778728291</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs778728291&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.998</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311745.9">RBFOX1:ENST00000311745.9:c.40G&gt;A:p.(Val14Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355637.8">RBFOX1:ENST00000355637.8:c.40G&gt;A:p.(Val14Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000436368.6">RBFOX1:ENST00000436368.6:c.40G&gt;A:p.(Val14Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422070.8">RBFOX1:ENST00000422070.8:c.157-185106G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000535565.6">RBFOX1:ENST00000535565.6:c.136-185106G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000547338.5">RBFOX1:ENST00000547338.5:c.28-185106G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000547372.5">RBFOX1:ENST00000547372.5:c.157-185106G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000550418.6">RBFOX1:ENST00000550418.6:c.28-185106G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000552089.5">RBFOX1:ENST00000552089.5:c.136-185106G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000553186.5">RBFOX1:ENST00000553186.5:c.28-185106G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000641259.1">RBFOX1:ENST00000641259.1:c.505-185106G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340209.8">RBFOX1:ENST00000340209.8:c.-6G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620507.4">RBFOX1:ENST00000620507.4:c.-6G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.024 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.092 (T)</dd>
                                    <dd>CADD: 0.990 (20.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0063%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000274391">TPTE</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.756</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.508</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.508 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TPTE%0DSPRTN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SPRTN</a> and phenotypic similarity 0.345 to <a href="http://www.omim.org/616200">Ruijs-Aalfs syndrome</a> associated with SPRTN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0001402, Hepatocellular carcinoma</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TPTE%0DSPRTN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SPRTN</a> and phenotypic similarity 0.642 to mouse mutant of SPRTN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - MP:0012106, impaired exercise endurance</dd><dd>HP:0002027, Abdominal pain - MP:0012106, impaired exercise endurance</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.756</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr21:g.10577479A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=74553878">rs74553878</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs74553878&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427445.6">TPTE:ENST00000427445.6:c.701A&gt;G:p.(Asp234Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612746.1">TPTE:ENST00000612746.1:c.401A&gt;G:p.(Asp134Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618007.4">TPTE:ENST00000618007.4:c.815A&gt;G:p.(Asp272Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622113.4">TPTE:ENST00000622113.4:c.761A&gt;G:p.(Asp254Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99668866</dd>
                                    
                                    <dd>CADD: 0.997 (24.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr21:g.10569454G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1810856">rs1810856</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1810856&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427445.6">TPTE:ENST00000427445.6:c.470G&gt;A:p.(Arg157Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612746.1">TPTE:ENST00000612746.1:c.170G&gt;A:p.(Arg57Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618007.4">TPTE:ENST00000618007.4:c.584G&gt;A:p.(Arg195Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622113.4">TPTE:ENST00000622113.4:c.530G&gt;A:p.(Arg177Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9922375</dd>
                                    
                                    <dd>CADD: 0.992 (21.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                    <b>chr21:g.10569526TAAG&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=113444703">rs113444703</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs113444703&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.992</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427445.6">TPTE:ENST00000427445.6:c.546_548del:p.(Arg184del)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612746.1">TPTE:ENST00000612746.1:c.246_248del:p.(Arg84del)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618007.4">TPTE:ENST00000618007.4:c.660_662del:p.(Arg222del)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622113.4">TPTE:ENST00000622113.4:c.606_608del:p.(Arg204del)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9922375</dd>
                        
                        <dd>CADD: 0.992 (21.100)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104093">DMXL2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.749</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.222 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DMXL2">DMXL2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0014018, embryo tumor</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DMXL2%0DSLC18A3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC18A3</a> and phenotypic similarity 0.628 to <a href="http://www.omim.org/617239">Myasthenic syndrome, congenital, 21, presynaptic</a> associated with SLC18A3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0011968, Feeding difficulties</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DMXL2%0DSLC18A3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC18A3</a> and phenotypic similarity 0.642 to mouse mutant of SLC18A3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - MP:0012106, impaired exercise endurance</dd><dd>HP:0002027, Abdominal pain - MP:0012106, impaired exercise endurance</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/616113">OMIM:616113</a> ?Polyendocrine-polyneuropathy syndrome (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/617605">OMIM:617605</a> ?Deafness, autosomal dominant 71 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=453533">ORPHA:453533</a> Polyendocrine-polyneuropathy syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.749</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_ACCEPTOR_VARIANT</span>
                                <b>chr15:g.51576183T&gt;TAAA [1/1:0/1:-/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3078066">rs3078066</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs3078066&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=779794%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251076.9">DMXL2:ENST00000251076.9:c.88-3_88-2insTTT:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449909.7">DMXL2:ENST00000449909.7:c.88-3_88-2insTTT:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543779.6">DMXL2:ENST00000543779.6:c.88-3_88-2insTTT:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.63144696</dd>
                                    
                                    <dd>CADD: 0.631 (4.335)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163389">POGLUT1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.744</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=POGLUT1%0DVGLL2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VGLL2</a> and phenotypic similarity 0.642 to mouse mutant of VGLL2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - MP:0012106, impaired exercise endurance</dd><dd>HP:0002027, Abdominal pain - MP:0012106, impaired exercise endurance</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/615696">OMIM:615696</a> Dowling-Degos disease 4 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/617232">OMIM:617232</a> ?Muscular dystrophy, limb-girdle, autosomal recessive 21 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.744</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.119477440T&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=528041638">rs528041638</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs528041638&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295588.9">POGLUT1:ENST00000295588.9:c.448T&gt;A:p.(Phe150Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999999</dd>
                                    
                                    <dd>Polyphen2: 0.926 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.005 (D)</dd>
                                    <dd>CADD: 0.998 (27.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182240">BACE2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.744</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=BACE2%0DATP2A2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ATP2A2</a> and phenotypic similarity 0.264 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=218">Darier disease</a> associated with ATP2A2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0005212, Anal mucosal leukoplakia</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=BACE2%0DATP2A2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ATP2A2</a> and phenotypic similarity 0.746 to mouse mutant of ATP2A2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - MP:0002899, fatigue</dd><dd>HP:0002027, Abdominal pain - MP:0002899, fatigue</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.744</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr21:g.41243404A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=530813869">rs530813869</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs530813869&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000328735.10">BACE2:ENST00000328735.10:c.776A&gt;G:p.(Tyr259Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000330333.11">BACE2:ENST00000330333.11:c.776A&gt;G:p.(Tyr259Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000347667.5">BACE2:ENST00000347667.5:c.776A&gt;G:p.(Tyr259Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.994 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.999 (32.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185420">SMYD3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.739</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SMYD3%0DFLT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FLT1</a> and phenotypic similarity 0.631 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=275555">Preeclampsia</a> associated with FLT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0005202, Helicobacter pylori infection</dd><dd>HP:0012378, Fatigue - HP:0002027, Abdominal pain</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.739</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.246507191G&gt;A [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000403792.7">SMYD3:ENST00000403792.7:c.27C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000490107.6">SMYD3:ENST00000490107.6:c.27C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99323916</dd>
                                    
                                    <dd>CADD: 0.993 (21.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000114251">WNT5A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.734</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.985</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.208 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3107">Autosomal dominant Robinow syndrome</a> associated with WNT5A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0005306, Capillary hemangioma</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl><dt>Phenotypic similarity 0.413 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=WNT5A">WNT5A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0000495, abnormal colon morphology</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0000495, abnormal colon morphology</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=WNT5A%0DVGLL2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VGLL2</a> and phenotypic similarity 0.642 to mouse mutant of VGLL2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - MP:0012106, impaired exercise endurance</dd><dd>HP:0002027, Abdominal pain - MP:0012106, impaired exercise endurance</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/180700">OMIM:180700</a> Robinow syndrome, autosomal dominant 1 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3107">ORPHA:3107</a> Autosomal dominant Robinow syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.734</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.985</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr3:g.55474496C&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=546487800">rs546487800</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs546487800&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.985</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264634.8">WNT5A:ENST00000264634.8:c.525G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000474267.5">WNT5A:ENST00000474267.5:c.525G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000497027.5">WNT5A:ENST00000497027.5:c.480G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9897198</dd>
                                    
                                    <dd>CADD: 0.990 (19.880)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC AFR: 0.0292%</dd>
                                    <dd>ExAC AMR: 0.0229%</dd>
                                    <dd>gnomAD_E_AFR: 0.0285%</dd>
                                    <dd>gnomAD_E_AMR: 0.0034%</dd>
                                    <dd>gnomAD_E_OTH: 0.0218%</dd>
                                    <dd>gnomAD_G_AFR: 0.0347%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000173559">NABP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.731</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.668</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.668 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=520">Acute promyelocytic leukemia</a> associated with NABP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NABP1%0DSLC25A4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC25A4</a> and phenotypic similarity 0.606 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=254892">Autosomal dominant progressive external ophthalmoplegia</a> associated with SLC25A4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002020, Gastroesophageal reflux</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0003326, Myalgia</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=520">ORPHA:520</a> Acute promyelocytic leukemia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.731</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.668</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr2:g.191678403CT&gt;C [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=781513296">rs781513296</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs781513296&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409510.5">NABP1:ENST00000409510.5:c.-150+8del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000425611.7">NABP1:ENST00000425611.7:c.-188del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000410026.6">NABP1:ENST00000410026.6:c.-150+163del:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7192628</dd>
                                    
                                    <dd>CADD: 0.719 (5.517)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000131373">HACL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.729</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=HACL1%0DRNF168&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RNF168</a> and phenotypic similarity 0.621 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=420741">RIDDLE syndrome</a> associated with RNF168.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0002027, Abdominal pain</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.729</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.15596397A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=555682297">rs555682297</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs555682297&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000321169.10">HACL1:ENST00000321169.10:c.214T&gt;C:p.(Tyr72His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000451445.6">HACL1:ENST00000451445.6:c.214T&gt;C:p.(Tyr72His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456194.6">HACL1:ENST00000456194.6:c.214T&gt;C:p.(Tyr72His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457447.6">HACL1:ENST00000457447.6:c.214T&gt;C:p.(Tyr72His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000628377.2">HACL1:ENST00000628377.2:c.214T&gt;C:p.(Tyr72His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000435217.6">HACL1:ENST00000435217.6:c.186+4693T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 1.000 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.997 (25.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ExAC AFR: 0.0291%</dd>
                                    <dd>ExAC AMR: 0.0347%</dd>
                                    <dd>gnomAD_E_AFR: 0.0327%</dd>
                                    <dd>gnomAD_E_AMR: 0.0238%</dd>
                                    <dd>gnomAD_G_AFR: 0.0801%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136488">CSH1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.724</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.984</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSH1%0DMPL&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MPL</a> and phenotypic similarity 0.631 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=729">Polycythemia vera</a> associated with MPL.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002863, Myelodysplasia</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.724</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.984</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr17:g.63895426G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2320124">rs2320124</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2320124&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329882.8">CSH1:ENST00000329882.8:c.503C&gt;A:p.(Ser168*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000316193.13">CSH1:ENST00000316193.13:c.456+47C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453363.7">CSH1:ENST00000453363.7:c.172-207C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000610991.1">CSH1:ENST00000610991.1:c.189+47C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.003 (0.011)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0799%</dd>
                                    <dd>ESP AA: 0.0228%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AFR: 0.0291%</dd>
                                    <dd>ExAC EAS: 0.0578%</dd>
                                    <dd>ExAC NFE: 0.0330%</dd>
                                    <dd>ExAC SAS: 0.0485%</dd>
                                    <dd>gnomAD_E_AFR: 0.0459%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0754%</dd>
                                    <dd>gnomAD_E_NFE: 0.0313%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0357%</dd>
                                    <dd>gnomAD_G_AFR: 0.0117%</dd>
                                    <dd>gnomAD_G_EAS: 0.0618%</dd>
                                    <dd>gnomAD_G_NFE: 0.0402%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_LOST</span>
                                <b>chr17:g.63895023T&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=141565530">rs141565530</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs141565530&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.980</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000316193.13">CSH1:ENST00000316193.13:c.653A&gt;C:p.(*218Serext*4)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453363.7">CSH1:ENST00000453363.7:c.368A&gt;C:p.(*123Serext*4)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000610991.1">CSH1:ENST00000610991.1:c.386A&gt;C:p.(*129Serext*4)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329882.8">CSH1:ENST00000329882.8:c.*135A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.933 (11.740)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1398%</dd>
                                    <dd>TOPMed: 0.0191%</dd>
                                    <dd>ESP AA: 0.0912%</dd>
                                    <dd>ESP All: 0.0308%</dd>
                                    <dd>ExAC AFR: 0.1352%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.1049%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_G_AFR: 0.0581%</dd>
                                    <dd>gnomAD_G_AMR: 0.1196%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187627">RGPD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.721</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.984</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.428 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RGPD1">RGPD1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0003331, increased hepatocellular carcinoma incidence</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RGPD1%0DFOXP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FOXP1</a> and phenotypic similarity 0.672 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=52417">MALT lymphoma</a> associated with FOXP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0012191, B-cell lymphoma</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.721</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.984</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.86986821G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1437744213">rs1437744213</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1437744213&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.984</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398193.8">RGPD1:ENST00000398193.8:c.3922G&gt;A:p.(Val1308Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409776.6">RGPD1:ENST00000409776.6:c.3898G&gt;A:p.(Val1300Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000559485.2">RGPD1:ENST00000559485.2:c.3898G&gt;A:p.(Val1300Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000641458.1">RGPD1:ENST00000641458.1:c.3922G&gt;A:p.(Val1308Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9958313</dd>
                                    
                                    <dd>Polyphen2: 0.949 (P)</dd>
                                    <dd>SIFT: 0.226 (T)</dd>
                                    <dd>CADD: 0.996 (23.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_G_AFR: 0.0829%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130538">OR11H1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.711</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.979</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR11H1%0DPALLD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PALLD</a> and phenotypic similarity 0.627 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1333">Familial pancreatic carcinoma</a> associated with PALLD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0003003, Colon cancer</dd><dd>HP:0012378, Fatigue - HP:0012432, Chronic fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.711</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.979</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.15528537T&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1341580021">rs1341580021</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1341580021&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.979</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000252835.5">OR11H1:ENST00000252835.5:c.379T&gt;A:p.(Cys127Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000643195.1">OR11H1:ENST00000643195.1:c.346T&gt;A:p.(Cys116Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.979</dd>
                                    
                                    <dd>SIFT: 0.021 (D)</dd>
                                    <dd>CADD: 0.974 (15.890)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000070371">CLTCL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.711</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.975</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CLTCL1%0DDNAJC6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DNAJC6</a> and phenotypic similarity 0.619 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=391411">Atypical juvenile parkinsonism</a> associated with DNAJC6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0012378, Fatigue</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.711</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.975</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr22:g.19233557A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=371115799">rs371115799</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs371115799&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.975</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427926.6">CLTCL1:ENST00000427926.6:c.1233T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621271.4">CLTCL1:ENST00000621271.4:c.1233T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98574394</dd>
                                    
                                    <dd>CADD: 0.986 (18.460)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0143%</dd>
                                    <dd>ESP AA: 0.0264%</dd>
                                    <dd>ESP All: 0.0083%</dd>
                                    <dd>ExAC AFR: 0.0613%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0654%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_AFR: 0.0458%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000172199">OR8U1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.694</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.970</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR8U1%0DPALLD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PALLD</a> and phenotypic similarity 0.627 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1333">Familial pancreatic carcinoma</a> associated with PALLD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0003003, Colon cancer</dd><dd>HP:0012378, Fatigue - HP:0012432, Chronic fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.694</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.970</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.56375784G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=75727721">rs75727721</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs75727721&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.970</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302270.1">OR8U1:ENST00000302270.1:c.161G&gt;A:p.(Ser54Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.971</dd>
                                    
                                    <dd>Polyphen2: 0.010 (B)</dd>
                                    <dd>SIFT: 0.029 (D)</dd>
                                    <dd>CADD: 0.931 (11.620)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_FIN: 0.0058%</dd>
                                    <dd>gnomAD_G_NFE: 0.0070%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000077097">TOP2B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.665</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.953</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TOP2B%0DMSH6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MSH6</a> and phenotypic similarity 0.648 to <a href="http://www.omim.org/614350">Colorectal cancer, hereditary nonpolyposis, type 5</a> associated with MSH6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0006716, Hereditary nonpolyposis colorectal carcinoma</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TOP2B%0DMSH6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MSH6</a> and phenotypic similarity 0.511 to mouse mutant of MSH6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0002404, increased intestinal adenoma incidence</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0002404, increased intestinal adenoma incidence</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.665</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.953</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr3:g.25638314T&gt;TAAA [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=56986587">rs56986587</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs56986587&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.953</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264331.8">TOP2B:ENST00000264331.8:c.396-5_396-4insTTT:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000435706.6">TOP2B:ENST00000435706.6:c.381-5_381-4insTTT:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9533341</dd>
                                    
                                    <dd>CADD: 0.953 (13.310)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000144749">LRIG1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.633</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.939</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRIG1%0DERBB4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ERBB4</a> and phenotypic similarity 0.605 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=803">Amyotrophic lateral sclerosis</a> associated with ERBB4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002017, Nausea and vomiting</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.633</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.939</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.66380679C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=529567262">rs529567262</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs529567262&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.939</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000273261.7">LRIG1:ENST00000273261.7:c.2953G&gt;A:p.(Ala985Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000383703.3">LRIG1:ENST00000383703.3:c.2884G&gt;A:p.(Ala962Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9416555</dd>
                                    
                                    <dd>Polyphen2: 0.124 (B)</dd>
                                    <dd>Mutation Taster: 0.645</dd>
                                    <dd>SIFT: 0.432 (T)</dd>
                                    <dd>CADD: 0.942 (12.340)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000110799">VWF</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.603</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.922</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.280 to <a href="http://www.omim.org/193400">von Willebrand disease, type 1</a> associated with VWF.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0000471, Gastrointestinal angiodysplasia</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - HP:0002239, Gastrointestinal hemorrhage</dd></dl><dl><dt>Phenotypic similarity 0.362 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=VWF">VWF</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0003292, melena</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0003292, melena</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=VWF%0DSERPING1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SERPING1</a> and phenotypic similarity 0.635 to <a href="http://www.omim.org/106100">Angioedema, hereditary, types I and II</a> associated with SERPING1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0005225, Intestinal edema</dd><dd>HP:0012378, Fatigue - HP:0002027, Abdominal pain</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/193400">OMIM:193400</a> von Willebrand disease, type 1 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/277480">OMIM:277480</a> von Willibrand disease, type 3 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/613554">OMIM:613554</a> von Willebrand disease, types 2A, 2B, 2M, and 2N - autosomal dominant/recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.603</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.922</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.5969369G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=374690023">rs374690023</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs374690023&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.987</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261405.9">VWF:ENST00000261405.9:c.7571C&gt;T:p.(Pro2524Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99339306</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.123 (T)</dd>
                                    <dd>CADD: 0.993 (21.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0120%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0491%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC FIN: 0.0152%</dd>
                                    <dd>gnomAD_E_AFR: 0.0393%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_FIN: 0.0135%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.5985084G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=184921605">rs184921605</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs184921605&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.858</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261405.9">VWF:ENST00000261405.9:c.6937C&gt;T:p.(Arg2313Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99920565</dd>
                                    
                                    <dd>Polyphen2: 0.828 (P)</dd>
                                    <dd>Mutation Taster: 0.785</dd>
                                    <dd>SIFT: 0.024 (D)</dd>
                                    <dd>CADD: 0.999 (31.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0589%</dd>
                                    <dd>ExAC AFR: 0.0481%</dd>
                                    <dd>ExAC AMR: 0.6824%</dd>
                                    <dd>ExAC EAS: 0.0231%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 0.0392%</dd>
                                    <dd>gnomAD_E_AMR: 0.6283%</dd>
                                    <dd>gnomAD_E_EAS: 0.0232%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_OTH: 0.1094%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.7160%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000129465">RIPK3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.594</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.917</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RIPK3%0DTICAM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TICAM1</a> and phenotypic similarity 0.629 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1930">Herpes simplex virus encephalitis</a> associated with TICAM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002017, Nausea and vomiting</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RIPK3%0DTICAM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TICAM1</a> and phenotypic similarity 0.327 to mouse mutant of TICAM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0008882, abnormal enterocyte physiology</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0008882, abnormal enterocyte physiology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.594</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.917</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.24337227C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=143073502">rs143073502</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs143073502&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.943</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000216274.10">RIPK3:ENST00000216274.10:c.1134G&gt;C:p.(Gln378His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.975</dd>
                                    
                                    <dd>Polyphen2: 0.034 (B)</dd>
                                    <dd>SIFT: 0.025 (D)</dd>
                                    <dd>CADD: 0.919 (10.920)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0462%</dd>
                                    <dd>ESP AA: 0.1362%</dd>
                                    <dd>ESP All: 0.0461%</dd>
                                    <dd>ExAC AFR: 0.2021%</dd>
                                    <dd>gnomAD_E_AFR: 0.1895%</dd>
                                    <dd>gnomAD_G_AFR: 0.2179%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.24337926T&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=7153640">rs7153640</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs7153640&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.890</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000216274.10">RIPK3:ENST00000216274.10:c.779A&gt;T:p.(Glu260Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9945046</dd>
                                    
                                    <dd>Polyphen2: 0.543 (P)</dd>
                                    <dd>SIFT: 0.008 (D)</dd>
                                    <dd>CADD: 0.995 (22.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2396%</dd>
                                    <dd>TOPMed: 0.2087%</dd>
                                    <dd>ESP AA: 0.5674%</dd>
                                    <dd>ESP All: 0.1922%</dd>
                                    <dd>ExAC AFR: 0.5306%</dd>
                                    <dd>gnomAD_E_AFR: 0.5359%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.5730%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000278845">MRPL45</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.576</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.914</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MRPL45%0DRAD54B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAD54B</a> and phenotypic similarity 0.640 to <a href="http://www.omim.org/114500">Colon cancer, somatic</a> associated with RAD54B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0006716, Hereditary nonpolyposis colorectal carcinoma</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.576</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.914</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.38318702G&gt;A [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=59939216">rs59939216</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs59939216&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.914</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613675.5">MRPL45:ENST00000613675.5:c.477G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000619548.1">MRPL45:ENST00000619548.1:c.477G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9144933</dd>
                                    
                                    <dd>CADD: 0.914 (10.680)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169432">SCN9A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.570</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.422</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.422 to <a href="http://www.omim.org/133020">Small fiber neuropathy</a> associated with SCN9A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0032147, Erythromelalgia</dd><dd>HP:0002027, Abdominal pain - HP:0002014, Diarrhea</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/133020">OMIM:133020</a> Small fiber neuropathy - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/167400">OMIM:167400</a> Paroxysmal extreme pain disorder - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/243000">OMIM:243000</a> HSAN2D, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/607208">OMIM:607208</a> Dravet syndrome, modifier of (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/613863">OMIM:613863</a> Epilepsy, generalized, with febrile seizures plus, type 7 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=33069">ORPHA:33069</a> Dravet syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=36387">ORPHA:36387</a> Generalized epilepsy with febrile seizures-plus</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=46348">ORPHA:46348</a> Paroxysmal extreme pain disorder</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=90026">ORPHA:90026</a> Primary erythromelalgia</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=970">ORPHA:970</a> Hereditary sensory and autonomic neuropathy type 2</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.570</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.422</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.166199148C&gt;T [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000303354.11">SCN9A:ENST00000303354.11:c.5491G&gt;A:p.(Val1831Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409435.6">SCN9A:ENST00000409435.6:c.5491G&gt;A:p.(Val1831Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409672.5">SCN9A:ENST00000409672.5:c.5458G&gt;A:p.(Val1820Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000642356.1">SCN9A:ENST00000642356.1:c.5491G&gt;A:p.(Val1831Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000644316.1">SCN9A:ENST00000644316.1:c.5335G&gt;A:p.(Val1779Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000645907.1">SCN9A:ENST00000645907.1:c.5458G&gt;A:p.(Val1820Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.998 (26.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000125498">KIR2DL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.538</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.897</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIR2DL1%0DKLRC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KLRC4</a> and phenotypic similarity 0.658 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=117">Behçet disease</a> associated with KLRC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002024, Malabsorption</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.538</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.897</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54778656A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=75232650">rs75232650</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs75232650&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.930</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000291633.7">KIR2DL1:ENST00000291633.7:c.709A&gt;G:p.(Lys237Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336077.11">KIR2DL1:ENST00000336077.11:c.709A&gt;G:p.(Lys237Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.95773315</dd>
                                    
                                    <dd>Polyphen2: 0.010 (B)</dd>
                                    <dd>SIFT: 0.135 (T)</dd>
                                    <dd>CADD: 0.958 (13.740)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>gnomAD_E_AMR: 0.0064%</dd>
                                    <dd>gnomAD_E_EAS: 0.0710%</dd>
                                    <dd>gnomAD_E_FIN: 0.0048%</dd>
                                    <dd>gnomAD_E_NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_SAS: 0.0120%</dd>
                                    <dd>gnomAD_G_EAS: 0.1928%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54775345A&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=151330111">rs151330111</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs151330111&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.864</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000291633.7">KIR2DL1:ENST00000291633.7:c.551A&gt;T:p.(Asp184Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336077.11">KIR2DL1:ENST00000336077.11:c.551A&gt;T:p.(Asp184Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.625 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.992 (20.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3994%</dd>
                                    <dd>TOPMed: 0.0996%</dd>
                                    <dd>ESP AA: 0.6164%</dd>
                                    <dd>ESP All: 0.2067%</dd>
                                    <dd>ExAC AFR: 0.6973%</dd>
                                    <dd>ExAC AMR: 0.0735%</dd>
                                    <dd>gnomAD_E_AFR: 0.6116%</dd>
                                    <dd>gnomAD_E_AMR: 0.1023%</dd>
                                    <dd>gnomAD_E_NFE: 0.0010%</dd>
                                    <dd>gnomAD_E_OTH: 0.0409%</dd>
                                    <dd>gnomAD_G_AFR: 0.4658%</dd>
                                    <dd>gnomAD_G_AMR: 0.1295%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr19:g.54782930C&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=149300913">rs149300913</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs149300913&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.616</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000291633.7">KIR2DL1:ENST00000291633.7:c.802C&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336077.11">KIR2DL1:ENST00000336077.11:c.724C&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.7515723</dd>
                        
                        <dd>CADD: 0.752 (6.048)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.3395%</dd>
                        <dd>ESP AA: 0.4852%</dd>
                        <dd>ESP All: 0.1664%</dd>
                        <dd>gnomAD_E_AFR: 0.6658%</dd>
                        <dd>gnomAD_E_AMR: 0.0486%</dd>
                        <dd>gnomAD_E_NFE: 0.0031%</dd>
                        <dd>gnomAD_E_OTH: 0.0345%</dd>
                        <dd>gnomAD_E_SAS: 0.0544%</dd>
                        <dd>gnomAD_G_AFR: 0.8488%</dd>
                        <dd>gnomAD_G_NFE: 0.0170%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000006607">FARP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.535</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.896</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FARP2%0DSEMA3D&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SEMA3D</a> and phenotypic similarity 0.632 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=388">Hirschsprung disease</a> associated with SEMA3D.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0200008, Intestinal polyposis</dd><dd>HP:0012378, Fatigue - HP:0002027, Abdominal pain</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.535</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.896</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.241456914A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=548966753">rs548966753</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs548966753&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264042.8">FARP2:ENST00000264042.8:c.1579A&gt;G:p.(Arg527Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373287.8">FARP2:ENST00000373287.8:c.1579A&gt;G:p.(Arg527Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000627550.2">FARP2:ENST00000627550.2:c.1579A&gt;G:p.(Arg527Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.997</dd>
                                    
                                    <dd>Polyphen2: 0.079 (B)</dd>
                                    <dd>SIFT: 0.003 (D)</dd>
                                    <dd>CADD: 0.991 (20.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0110%</dd>
                                    <dd>gnomAD_E_AFR: 0.0068%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr2:g.241463894C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537031260">rs537031260</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537031260&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.797</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264042.8">FARP2:ENST00000264042.8:c.1812-5C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373287.8">FARP2:ENST00000373287.8:c.1812-5C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000627550.2">FARP2:ENST00000627550.2:c.1812-5C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.20329052</dd>
                                    
                                    <dd>CADD: 0.203 (0.987)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ExAC AFR: 0.0206%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_AFR: 0.0262%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr2:g.241468274G&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=147634480">rs147634480</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs147634480&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.021</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264042.8">FARP2:ENST00000264042.8:c.2028G&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.024561286</dd>
                        
                        <dd>CADD: 0.025 (0.108)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.1597%</dd>
                        <dd>TOPMed: 0.2182%</dd>
                        <dd>ESP AA: 0.4993%</dd>
                        <dd>ESP All: 0.1692%</dd>
                        <dd>ExAC AFR: 0.6742%</dd>
                        <dd>ExAC AMR: 0.0173%</dd>
                        <dd>gnomAD_E_AFR: 0.7842%</dd>
                        <dd>gnomAD_E_AMR: 0.0328%</dd>
                        <dd>gnomAD_E_NFE: 0.0009%</dd>
                        <dd>gnomAD_G_AFR: 0.5955%</dd>
                        <dd>gnomAD_G_OTH: 0.1018%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162881">OXER1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.518</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.887</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OXER1%0DAPLNR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to APLNR</a> and phenotypic similarity 0.642 to mouse mutant of APLNR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - MP:0012106, impaired exercise endurance</dd><dd>HP:0002027, Abdominal pain - MP:0012106, impaired exercise endurance</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.518</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.887</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.42763588G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=541716654">rs541716654</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs541716654&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378661.3">OXER1:ENST00000378661.3:c.592C&gt;T:p.(His198Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.993</dd>
                                    
                                    <dd>Polyphen2: 0.989 (D)</dd>
                                    <dd>SIFT: 0.007 (D)</dd>
                                    <dd>CADD: 0.993 (21.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0225%</dd>
                                    <dd>gnomAD_E_AFR: 0.0077%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.42763097G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=144634528">rs144634528</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs144634528&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.785</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378661.3">OXER1:ENST00000378661.3:c.1083C&gt;G:p.(Asn361Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96301717</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.250 (T)</dd>
                                    <dd>CADD: 0.963 (14.320)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3794%</dd>
                                    <dd>TOPMed: 0.3018%</dd>
                                    <dd>ESP AA: 0.7263%</dd>
                                    <dd>ESP All: 0.2460%</dd>
                                    <dd>ExAC AFR: 0.8614%</dd>
                                    <dd>ExAC AMR: 0.0347%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.8576%</dd>
                                    <dd>gnomAD_E_AMR: 0.0983%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_AFR: 0.6989%</dd>
                                    <dd>gnomAD_G_AMR: 0.3580%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr2:g.42763379G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=139916967">rs139916967</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs139916967&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.683</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378661.3">OXER1:ENST00000378661.3:c.801C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.94363624</dd>
                        
                        <dd>CADD: 0.944 (12.490)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.3794%</dd>
                        <dd>TOPMed: 0.2939%</dd>
                        <dd>ESP AA: 0.6585%</dd>
                        <dd>ESP All: 0.2230%</dd>
                        <dd>ExAC AFR: 1.1111%</dd>
                        <dd>ExAC AMR: 0.0515%</dd>
                        <dd>ExAC NFE: 0.0022%</dd>
                        <dd>gnomAD_E_AFR: 0.8724%</dd>
                        <dd>gnomAD_E_AMR: 0.1202%</dd>
                        <dd>gnomAD_E_NFE: 0.0038%</dd>
                        <dd>gnomAD_E_OTH: 0.0192%</dd>
                        <dd>gnomAD_G_AFR: 0.6874%</dd>
                        <dd>gnomAD_G_AMR: 0.3580%</dd>
                        <dd>gnomAD_G_OTH: 0.1018%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170322">NFRKB</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.411</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.842</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NFRKB%0DFOXP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FOXP1</a> and phenotypic similarity 0.672 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=52417">MALT lymphoma</a> associated with FOXP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0012191, B-cell lymphoma</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.411</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.842</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.129882499G&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=116056970">rs116056970</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs116056970&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.913</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000446488.7">NFRKB:ENST00000446488.7:c.1034C&gt;A:p.(Pro345His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524746.5">NFRKB:ENST00000524746.5:c.1034C&gt;A:p.(Pro345His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524794.5">NFRKB:ENST00000524794.5:c.1109C&gt;A:p.(Pro370His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999</dd>
                                    
                                    <dd>Polyphen2: 0.223 (B)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.994 (22.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1398%</dd>
                                    <dd>TOPMed: 0.1410%</dd>
                                    <dd>ESP AA: 0.4089%</dd>
                                    <dd>ESP All: 0.1385%</dd>
                                    <dd>ExAC AFR: 0.4132%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>gnomAD_E_AFR: 0.4901%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_G_AFR: 0.4581%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr11:g.129874866A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=143517925">rs143517925</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs143517925&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.770</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000446488.7">NFRKB:ENST00000446488.7:c.1905T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524746.5">NFRKB:ENST00000524746.5:c.1905T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524794.5">NFRKB:ENST00000524794.5:c.1980T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9060277</dd>
                                    
                                    <dd>CADD: 0.906 (10.270)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2196%</dd>
                                    <dd>TOPMed: 0.1752%</dd>
                                    <dd>ESP AA: 0.3862%</dd>
                                    <dd>ESP All: 0.1308%</dd>
                                    <dd>ExAC AFR: 0.6919%</dd>
                                    <dd>ExAC AMR: 0.0432%</dd>
                                    <dd>gnomAD_E_AFR: 0.6926%</dd>
                                    <dd>gnomAD_E_AMR: 0.0655%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_G_AFR: 0.7447%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.129874229T&gt;C [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=150132863">rs150132863</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs150132863&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.698</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000446488.7">NFRKB:ENST00000446488.7:c.2163A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524746.5">NFRKB:ENST00000524746.5:c.2163A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524794.5">NFRKB:ENST00000524794.5:c.2238A&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.818323</dd>
                        
                        <dd>CADD: 0.818 (7.407)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.2196%</dd>
                        <dd>TOPMed: 0.1744%</dd>
                        <dd>ESP AA: 0.3862%</dd>
                        <dd>ESP All: 0.1308%</dd>
                        <dd>ExAC AFR: 0.6950%</dd>
                        <dd>ExAC AMR: 0.0435%</dd>
                        <dd>gnomAD_E_AFR: 0.7003%</dd>
                        <dd>gnomAD_E_AMR: 0.0875%</dd>
                        <dd>gnomAD_E_OTH: 0.0575%</dd>
                        <dd>gnomAD_G_AFR: 0.7368%</dd>
                        <dd>gnomAD_G_AMR: 0.1193%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000053747">LAMA3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.399</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.834</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.382 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79404">Junctional epidermolysis bullosa, generalized severe</a> associated with LAMA3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002043, Esophageal stricture</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - HP:0011968, Feeding difficulties</dd></dl><dl><dt>Phenotypic similarity 0.300 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LAMA3">LAMA3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0002691, small stomach</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0002691, small stomach</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LAMA3%0DDMD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DMD</a> and phenotypic similarity 0.626 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98895">Becker muscular dystrophy</a> associated with DMD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0003326, Myalgia</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LAMA3%0DDMD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DMD</a> and phenotypic similarity 0.746 to mouse mutant of DMD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - MP:0002899, fatigue</dd><dd>HP:0002027, Abdominal pain - MP:0002899, fatigue</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/226650">OMIM:226650</a> Epidermolysis bullosa, generalized atrophic benign - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/226700">OMIM:226700</a> Epidermolysis bullosa, junctional, Herlitz type - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/245660">OMIM:245660</a> Laryngoonychocutaneous syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79402">ORPHA:79402</a> Junctional epidermolysis bullosa, generalized intermediate</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79404">ORPHA:79404</a> Junctional epidermolysis bullosa, generalized severe</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.399</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.834</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr18:g.23921477A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=147208526">rs147208526</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs147208526&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=756409%5Balleleid%5D">(criteria_provided,_conflicting_interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.912</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000269217.10">LAMA3:ENST00000269217.10:c.3242A&gt;G:p.(Gln1081Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313654.13">LAMA3:ENST00000313654.13:c.8069A&gt;G:p.(Gln2690Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399516.7">LAMA3:ENST00000399516.7:c.7901A&gt;G:p.(Gln2634Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000587184.5">LAMA3:ENST00000587184.5:c.3074A&gt;G:p.(Gln1025Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96837723</dd>
                                    
                                    <dd>SIFT: 0.206 (T)</dd>
                                    <dd>CADD: 0.968 (15.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0398%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0349%</dd>
                                    <dd>ESP All: 0.0308%</dd>
                                    <dd>ExAC AFR: 0.0387%</dd>
                                    <dd>ExAC AMR: 0.0607%</dd>
                                    <dd>ExAC NFE: 0.0361%</dd>
                                    <dd>gnomAD_E_AFR: 0.0458%</dd>
                                    <dd>gnomAD_E_AMR: 0.0746%</dd>
                                    <dd>gnomAD_E_ASJ: 0.3560%</dd>
                                    <dd>gnomAD_E_NFE: 0.0180%</dd>
                                    <dd>gnomAD_E_OTH: 0.1461%</dd>
                                    <dd>gnomAD_G_AFR: 0.0458%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr18:g.23932212C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=35153125">rs35153125</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs35153125&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=346738%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.756</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000269217.10">LAMA3:ENST00000269217.10:c.3802C&gt;T:p.(His1268Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313654.13">LAMA3:ENST00000313654.13:c.8629C&gt;T:p.(His2877Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399516.7">LAMA3:ENST00000399516.7:c.8461C&gt;T:p.(His2821Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000587184.5">LAMA3:ENST00000587184.5:c.3634C&gt;T:p.(His1212Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.959</dd>
                                    
                                    <dd>SIFT: 0.041 (D)</dd>
                                    <dd>CADD: 0.958 (13.750)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2995%</dd>
                                    <dd>TOPMed: 0.2413%</dd>
                                    <dd>ESP AA: 0.7717%</dd>
                                    <dd>ESP All: 0.2614%</dd>
                                    <dd>ExAC AFR: 0.9418%</dd>
                                    <dd>ExAC AMR: 0.0605%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 0.8756%</dd>
                                    <dd>gnomAD_E_AMR: 0.0625%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_G_AFR: 0.8135%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167210">LOXHD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.390</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.832</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LOXHD1%0DCDH23&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDH23</a> and phenotypic similarity 0.691 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=96253">Cushing disease</a> associated with CDH23.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002893, Pituitary adenoma</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LOXHD1%0DCDH23&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDH23</a> and phenotypic similarity 0.401 to mouse mutant of CDH23.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0003866, abnormal defecation</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0003866, abnormal defecation</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/613079">OMIM:613079</a> Deafness, autosomal recessive 77 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.390</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.832</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr18:g.46639657C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=551430371">rs551430371</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs551430371&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441551.6">LOXHD1:ENST00000441551.6:c.470G&gt;A:p.(Arg157His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536736.5">LOXHD1:ENST00000536736.5:c.470G&gt;A:p.(Arg157His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000642948.1">LOXHD1:ENST00000642948.1:c.470G&gt;A:p.(Arg157His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999755</dd>
                                    
                                    <dd>Polyphen2: 0.998 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.003 (D)</dd>
                                    <dd>CADD: 0.999 (29.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC SAS: 0.0254%</dd>
                                    <dd>gnomAD_E_SAS: 0.0044%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr18:g.46579731C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=140437150">rs140437150</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs140437150&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=57083%5Balleleid%5D">(criteria_provided,_conflicting_interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.667</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441551.6">LOXHD1:ENST00000441551.6:c.1708G&gt;A:p.(Asp570Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536736.5">LOXHD1:ENST00000536736.5:c.1708G&gt;A:p.(Asp570Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000642948.1">LOXHD1:ENST00000642948.1:c.1708G&gt;A:p.(Asp570Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.997665</dd>
                                    
                                    <dd>Polyphen2: 0.066 (B)</dd>
                                    <dd>Mutation Taster: 0.998 (P)</dd>
                                    <dd>SIFT: 0.137 (T)</dd>
                                    <dd>CADD: 0.992 (20.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3594%</dd>
                                    <dd>TOPMed: 0.3568%</dd>
                                    <dd>ESP AA: 0.8671%</dd>
                                    <dd>ESP All: 0.2628%</dd>
                                    <dd>ExAC AFR: 0.9380%</dd>
                                    <dd>ExAC NFE: 0.0114%</dd>
                                    <dd>gnomAD_E_AFR: 1.1019%</dd>
                                    <dd>gnomAD_E_AMR: 0.0543%</dd>
                                    <dd>gnomAD_E_NFE: 0.0034%</dd>
                                    <dd>gnomAD_E_OTH: 0.0770%</dd>
                                    <dd>gnomAD_G_AFR: 1.2385%</dd>
                                    <dd>gnomAD_G_AMR: 0.1196%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr18:g.46577732C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=141932807">rs141932807</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs141932807&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                <div class="col-sm-12">
                    <b>ClinVar: </b>
                    
                        
                        
                        
                        
                        
                        
                        <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                        <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=176572%5Balleleid%5D">(criteria_provided,_conflicting_interpretations)</a>
                    
                </div>
                <div class="col-sm-6">
                    <b>Variant score: 0.595</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441551.6">LOXHD1:ENST00000441551.6:c.1945G&gt;A:p.(Asp649Asn)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536736.5">LOXHD1:ENST00000536736.5:c.1945G&gt;A:p.(Asp649Asn)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000642948.1">LOXHD1:ENST00000642948.1:c.1945G&gt;A:p.(Asp649Asn)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.99323916</dd>
                        
                        <dd>Polyphen2: 0.247 (B)</dd>
                        <dd>SIFT: 0.161 (T)</dd>
                        <dd>CADD: 0.993 (21.700)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.4393%</dd>
                        <dd>TOPMed: 0.3337%</dd>
                        <dd>ESP AA: 1.3728%</dd>
                        <dd>ESP All: 0.4161%</dd>
                        <dd>ExAC AFR: 1.3091%</dd>
                        <dd>gnomAD_E_AFR: 1.3769%</dd>
                        <dd>gnomAD_E_AMR: 0.0752%</dd>
                        <dd>gnomAD_E_EAS: 0.0098%</dd>
                        <dd>gnomAD_E_NFE: 0.0035%</dd>
                        <dd>gnomAD_E_OTH: 0.0518%</dd>
                        <dd>gnomAD_G_AFR: 1.1126%</dd>
                        <dd>gnomAD_G_NFE: 0.0133%</dd>
                        <dd>gnomAD_G_OTH: 0.1018%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100228">RAB36</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.378</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.827</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RAB36%0DDCTN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DCTN1</a> and phenotypic similarity 0.605 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=803">Amyotrophic lateral sclerosis</a> associated with DCTN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002017, Nausea and vomiting</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.378</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.827</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.23145402G&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=551752308">rs551752308</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs551752308&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.827</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263116.6">RAB36:ENST00000263116.6:c.37G&gt;T:p.(Ala13Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341989.8">RAB36:ENST00000341989.8:c.37G&gt;T:p.(Ala13Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.829</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.171 (T)</dd>
                                    <dd>CADD: 0.826 (7.586)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0104%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000075914">EXOSC7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.346</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.513</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.797</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.513 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=EXOSC7%0DDIS3L2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DIS3L2</a> and phenotypic similarity 0.630 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=654">Nephroblastoma</a> associated with DIS3L2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002896, Neoplasm of the liver</dd><dd>HP:0012378, Fatigue - HP:0002027, Abdominal pain</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=EXOSC7%0DDIS3L2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DIS3L2</a> and phenotypic similarity 0.220 to fish mutant of DIS3L2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - ZP:0001436, abnormal(ly) disrupted cloaca development</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.346</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.513</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.797</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr3:g.44976339C&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=374024896">rs374024896</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs374024896&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.797</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265564.8">EXOSC7:ENST00000265564.8:c.57+5C&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7837281</dd>
                                    
                                    <dd>CADD: 0.784 (6.650)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0278%</dd>
                                    <dd>gnomAD_E_AFR: 0.0187%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167447">SMG8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.320</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.798</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SMG8%0DRPL27&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RPL27</a> and phenotypic similarity 0.623 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=124">Blackfan-Diamond anemia</a> associated with RPL27.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002488, Acute leukemia</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0012378, Fatigue</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.320</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.798</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.59210405A&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=533126243">rs533126243</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs533126243&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.971</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000300917.9">SMG8:ENST00000300917.9:c.354A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543872.6">SMG8:ENST00000543872.6:c.354A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000578922.1">SMG8:ENST00000578922.1:c.354A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000577660.1">SMG8:ENST00000577660.1:c.136-4400A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97326994</dd>
                                    
                                    <dd>CADD: 0.973 (15.730)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0100%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.59213141C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=146340466">rs146340466</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs146340466&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=715600%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.625</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000300917.9">SMG8:ENST00000300917.9:c.2318C&gt;T:p.(Pro773Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543872.6">SMG8:ENST00000543872.6:c.2318C&gt;T:p.(Pro773Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000577660.1">SMG8:ENST00000577660.1:c.136-1664C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.999 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.002 (D)</dd>
                                    <dd>CADD: 0.998 (27.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4792%</dd>
                                    <dd>TOPMed: 0.4792%</dd>
                                    <dd>ESP AA: 1.2937%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.4459%</dd>
                                    <dd>ExAC AFR: 1.3168%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 1.3266%</dd>
                                    <dd>gnomAD_E_AMR: 0.0268%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0097%</dd>
                                    <dd>gnomAD_G_AFR: 0.9851%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.2037%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100345">MYH9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.317</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.796</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MYH9%0DACTG2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTG2</a> and phenotypic similarity 0.610 to <a href="http://www.omim.org/155310">Visceral myopathy</a> associated with ACTG2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0004388, Microcolon</dd><dd>HP:0012378, Fatigue - HP:0002027, Abdominal pain</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/155100">OMIM:155100</a> Macrothrombocytopenia and granulocyte inclusions with or without nephritis or sensorineural hearing loss - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/603622">OMIM:603622</a> Deafness, autosomal dominant 17 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=182050">ORPHA:182050</a> MYH9-related disease</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.317</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.796</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr22:g.36288420G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=559921359">rs559921359</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs559921359&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.796</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000216181.10">MYH9:ENST00000216181.10:c.4771-7C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.601526</dd>
                                    
                                    <dd>CADD: 0.602 (3.996)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0399%</dd>
                                    <dd>ExAC AFR: 0.0193%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000142207">URB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.316</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.797</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=URB1%0DNUMA1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NUMA1</a> and phenotypic similarity 0.668 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=520">Acute promyelocytic leukemia</a> associated with NUMA1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.316</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.797</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr21:g.32341485A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=566281589">rs566281589</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs566281589&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382751.4">URB1:ENST00000382751.4:c.4297T&gt;C:p.(Phe1433Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9948714</dd>
                                    
                                    <dd>Polyphen2: 0.108 (B)</dd>
                                    <dd>Mutation Taster: 0.942 (P)</dd>
                                    <dd>SIFT: 0.271 (T)</dd>
                                    <dd>CADD: 0.995 (22.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_E_AFR: 0.0394%</dd>
                                    <dd>gnomAD_G_AFR: 0.0458%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr21:g.32355478T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=138144587">rs138144587</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs138144587&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.605</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382751.4">URB1:ENST00000382751.4:c.2077A&gt;G:p.(Lys693Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9885449</dd>
                                    
                                    <dd>Polyphen2: 0.067 (B)</dd>
                                    <dd>SIFT: 0.029 (D)</dd>
                                    <dd>CADD: 0.989 (19.410)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3594%</dd>
                                    <dd>TOPMed: 0.4547%</dd>
                                    <dd>ESP AA: 0.6503%</dd>
                                    <dd>ESP All: 0.1971%</dd>
                                    <dd>ExAC AFR: 0.9132%</dd>
                                    <dd>ExAC AMR: 0.2439%</dd>
                                    <dd>ExAC SAS: 0.0126%</dd>
                                    <dd>gnomAD_E_AFR: 1.3524%</dd>
                                    <dd>gnomAD_E_AMR: 0.0876%</dd>
                                    <dd>gnomAD_E_NFE: 0.0051%</dd>
                                    <dd>gnomAD_E_OTH: 0.0256%</dd>
                                    <dd>gnomAD_E_SAS: 0.0044%</dd>
                                    <dd>gnomAD_G_AFR: 1.2251%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000111254">AKAP3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.301</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.789</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=AKAP3%0DLRRC8A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRRC8A</a> and phenotypic similarity 0.630 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=33110">Autosomal agammaglobulinemia</a> associated with LRRC8A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002024, Malabsorption</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002014, Diarrhea</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.301</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.789</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.4627830C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=137999689">rs137999689</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs137999689&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.911</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000228850.6">AKAP3:ENST00000228850.6:c.1072G&gt;A:p.(Val358Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545990.6">AKAP3:ENST00000545990.6:c.1072G&gt;A:p.(Val358Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536588.1">AKAP3:ENST00000536588.1:c.144-3492C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.472 (P)</dd>
                                    <dd>Mutation Taster: 0.859</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.996 (23.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1997%</dd>
                                    <dd>TOPMed: 0.1027%</dd>
                                    <dd>ESP AA: 0.4993%</dd>
                                    <dd>ESP All: 0.1692%</dd>
                                    <dd>ExAC AFR: 0.3945%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 0.3725%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AFR: 0.5039%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.4615833G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=34903224">rs34903224</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs34903224&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.667</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000228850.6">AKAP3:ENST00000228850.6:c.2468C&gt;T:p.(Ser823Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545990.6">AKAP3:ENST00000545990.6:c.2468C&gt;T:p.(Ser823Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536588.1">AKAP3:ENST00000536588.1:c.143+10747G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9930817</dd>
                                    
                                    <dd>Polyphen2: 0.016 (B)</dd>
                                    <dd>SIFT: 0.103 (T)</dd>
                                    <dd>CADD: 0.993 (21.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4393%</dd>
                                    <dd>TOPMed: 0.3504%</dd>
                                    <dd>ESP AA: 0.7263%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.2537%</dd>
                                    <dd>ExAC AFR: 1.1953%</dd>
                                    <dd>ExAC AMR: 0.0866%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>ExAC OTH: 0.1109%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 1.2305%</dd>
                                    <dd>gnomAD_E_AMR: 0.0924%</dd>
                                    <dd>gnomAD_E_NFE: 0.0099%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 1.0880%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000124232">RBPJL</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.289</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.780</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RBPJL%0DTBX19&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TBX19</a> and phenotypic similarity 0.619 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=199296">Congenital isolated ACTH deficiency</a> associated with TBX19.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0012378, Fatigue</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.289</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.780</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr20:g.45313523G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=141335503">rs141335503</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs141335503&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.831</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343694.8">RBPJL:ENST00000343694.8:c.675G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372741.7">RBPJL:ENST00000372741.7:c.675G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372743.5">RBPJL:ENST00000372743.5:c.675G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9112844</dd>
                                    
                                    <dd>CADD: 0.911 (10.520)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2196%</dd>
                                    <dd>TOPMed: 0.1426%</dd>
                                    <dd>ESP AA: 0.3858%</dd>
                                    <dd>ESP All: 0.1307%</dd>
                                    <dd>ExAC AFR: 0.4667%</dd>
                                    <dd>ExAC AMR: 0.0530%</dd>
                                    <dd>gnomAD_E_AFR: 0.5037%</dd>
                                    <dd>gnomAD_E_AMR: 0.0357%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_G_AFR: 0.3779%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr20:g.45311884G&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=35472429">rs35472429</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs35472429&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.729</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343694.8">RBPJL:ENST00000343694.8:c.374G&gt;T:p.(Gly125Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372741.7">RBPJL:ENST00000372741.7:c.374G&gt;T:p.(Gly125Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372743.5">RBPJL:ENST00000372743.5:c.374G&gt;T:p.(Gly125Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.997 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.999 (29.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3994%</dd>
                                    <dd>TOPMed: 0.2732%</dd>
                                    <dd>ESP AA: 0.4132%</dd>
                                    <dd>ESP All: 0.1400%</dd>
                                    <dd>ExAC AFR: 0.7114%</dd>
                                    <dd>ExAC OTH: 1.0989%</dd>
                                    <dd>gnomAD_E_AFR: 0.8375%</dd>
                                    <dd>gnomAD_E_AMR: 0.0584%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_OTH: 0.0526%</dd>
                                    <dd>gnomAD_G_AFR: 0.6870%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000228198">OR2M3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.253</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.763</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR2M3%0DPALLD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PALLD</a> and phenotypic similarity 0.627 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1333">Familial pancreatic carcinoma</a> associated with PALLD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0003003, Colon cancer</dd><dd>HP:0012378, Fatigue - HP:0012432, Chronic fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.253</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.763</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.248203431C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=143836422">rs143836422</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs143836422&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.838</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456743.3">OR2M3:ENST00000456743.3:c.364C&gt;T:p.(Arg122Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000641626.1">OR2M3:ENST00000641626.1:c.364C&gt;T:p.(Arg122Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9908799</dd>
                                    
                                    <dd>Polyphen2: 0.020 (B)</dd>
                                    <dd>SIFT: 0.068 (T)</dd>
                                    <dd>CADD: 0.991 (20.400)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3195%</dd>
                                    <dd>TOPMed: 0.1879%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 0.7263%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.2537%</dd>
                                    <dd>ExAC AFR: 0.7592%</dd>
                                    <dd>ExAC AMR: 0.0346%</dd>
                                    <dd>ExAC EAS: 0.0231%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.6993%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_EAS: 0.0174%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_E_SAS: 0.0097%</dd>
                                    <dd>gnomAD_G_AFR: 0.6889%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.248203944C&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=116642815">rs116642815</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs116642815&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.688</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456743.3">OR2M3:ENST00000456743.3:c.877C&gt;A:p.(Arg293Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000641626.1">OR2M3:ENST00000641626.1:c.877C&gt;A:p.(Arg293Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.614 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.989 (19.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2596%</dd>
                                    <dd>TOPMed: 0.3544%</dd>
                                    <dd>ESP AA: 0.7717%</dd>
                                    <dd>ESP All: 0.2614%</dd>
                                    <dd>ExAC AFR: 1.1244%</dd>
                                    <dd>ExAC AMR: 0.0435%</dd>
                                    <dd>gnomAD_E_AFR: 1.1962%</dd>
                                    <dd>gnomAD_E_AMR: 0.0269%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_AFR: 1.1804%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000099834">CDHR5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.220</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.743</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CDHR5%0DPTEN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTEN</a> and phenotypic similarity 0.616 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79076">Juvenile polyposis of infancy</a> associated with PTEN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0005227, Adenomatous colonic polyposis</dd><dd>HP:0012378, Fatigue - HP:0002027, Abdominal pain</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CDHR5%0DPTEN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTEN</a> and phenotypic similarity 0.539 to mouse mutant of PTEN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0010279, increased gastrointestinal tumor incidence</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0010279, increased gastrointestinal tumor incidence</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.220</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.743</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.618713C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=139730150">rs139730150</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs139730150&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.743</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358353.8">CDHR5:ENST00000358353.8:c.1828G&gt;A:p.(Gly610Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397542.7">CDHR5:ENST00000397542.7:c.1846G&gt;A:p.(Gly616Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000674088.1">CDHR5:ENST00000674088.1:c.1846G&gt;A:p.(Gly616Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349570.11">CDHR5:ENST00000349570.11:c.1378+593G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.747</dd>
                                    
                                    <dd>Polyphen2: 0.180 (B)</dd>
                                    <dd>SIFT: 0.253 (T)</dd>
                                    <dd>CADD: 0.253 (1.265)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0349%</dd>
                                    <dd>gnomAD_E_NFE: 0.0063%</dd>
                                    <dd>gnomAD_E_SAS: 0.0163%</dd>
                                    <dd>gnomAD_G_AFR: 0.0249%</dd>
                                    <dd>gnomAD_G_NFE: 0.0072%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000085982">USP40</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.206</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.734</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=USP40%0DPOLG2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLG2</a> and phenotypic similarity 0.606 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=254892">Autosomal dominant progressive external ophthalmoplegia</a> associated with POLG2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002020, Gastroesophageal reflux</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0003326, Myalgia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.206</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.734</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.233496774C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200152165">rs200152165</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200152165&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.969</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251722.10">USP40:ENST00000251722.10:c.2771G&gt;A:p.(Gly924Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409945.1">USP40:ENST00000409945.1:c.299G&gt;A:p.(Gly100Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427112.6">USP40:ENST00000427112.6:c.2771G&gt;A:p.(Gly924Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000450966.5">USP40:ENST00000450966.5:c.2807G&gt;A:p.(Gly936Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 1.000 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.997 (25.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0151%</dd>
                                    <dd>ESP AA: 0.0545%</dd>
                                    <dd>ESP All: 0.0169%</dd>
                                    <dd>ExAC AFR: 0.1023%</dd>
                                    <dd>ExAC AMR: 0.0088%</dd>
                                    <dd>gnomAD_E_AFR: 0.0655%</dd>
                                    <dd>gnomAD_E_AMR: 0.0090%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_FIN: 0.0287%</dd>
                                    <dd>gnomAD_G_OTH: 0.2045%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.233481276C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=145560089">rs145560089</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs145560089&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.498</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251722.10">USP40:ENST00000251722.10:c.3523G&gt;A:p.(Asp1175Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427112.6">USP40:ENST00000427112.6:c.3523G&gt;A:p.(Asp1175Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000450966.5">USP40:ENST00000450966.5:c.3559G&gt;A:p.(Asp1187Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9821351</dd>
                                    
                                    <dd>Polyphen2: 0.051 (B)</dd>
                                    <dd>SIFT: 0.063 (T)</dd>
                                    <dd>CADD: 0.982 (17.480)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.5791%</dd>
                                    <dd>TOPMed: 0.3209%</dd>
                                    <dd>ESP AA: 1.0139%</dd>
                                    <dd>ESP All: 0.3183%</dd>
                                    <dd>ExAC AFR: 1.5349%</dd>
                                    <dd>ExAC AMR: 0.0899%</dd>
                                    <dd>ExAC NFE: 0.0095%</dd>
                                    <dd>ExAC OTH: 0.2008%</dd>
                                    <dd>gnomAD_E_AFR: 1.0955%</dd>
                                    <dd>gnomAD_E_AMR: 0.0782%</dd>
                                    <dd>gnomAD_E_NFE: 0.0029%</dd>
                                    <dd>gnomAD_E_OTH: 0.0192%</dd>
                                    <dd>gnomAD_G_AFR: 1.1921%</dd>
                                    <dd>gnomAD_G_AMR: 0.1196%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182923">CEP63</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.174</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CEP63%0DAURKA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AURKA</a> and phenotypic similarity 0.640 to <a href="http://www.omim.org/114500">Colon cancer, susceptibility to</a> associated with AURKA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0006716, Hereditary nonpolyposis colorectal carcinoma</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CEP63%0DAURKA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AURKA</a> and phenotypic similarity 0.413 to mouse mutant of AURKA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0008714, increased lung carcinoma incidence</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614728">OMIM:614728</a> ?Seckel syndrome 6 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2512">ORPHA:2512</a> Autosomal recessive primary microcephaly</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.174</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.134537198T&gt;C [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=542172267">rs542172267</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs542172267&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000332047.9">CEP63:ENST00000332047.9:c.485T&gt;C:p.(Ile162Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000337090.7">CEP63:ENST00000337090.7:c.485T&gt;C:p.(Ile162Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354446.7">CEP63:ENST00000354446.7:c.485T&gt;C:p.(Ile162Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000383229.7">CEP63:ENST00000383229.7:c.485T&gt;C:p.(Ile162Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513612.6">CEP63:ENST00000513612.6:c.485T&gt;C:p.(Ile162Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000606977.5">CEP63:ENST00000606977.5:c.485T&gt;C:p.(Ile162Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620544.4">CEP63:ENST00000620544.4:c.485T&gt;C:p.(Ile162Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99636924</dd>
                                    
                                    <dd>Polyphen2: 0.795 (P)</dd>
                                    <dd>SIFT: 0.004 (D)</dd>
                                    <dd>CADD: 0.996 (24.400)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000066279">ASPM</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.145</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.688</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ASPM%0DKIF23&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KIF23</a> and phenotypic similarity 0.622 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98870">Congenital dyserythropoietic anemia type III</a> associated with KIF23.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002249, Melena</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0012378, Fatigue</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/608716">OMIM:608716</a> Microcephaly 5, primary, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2512">ORPHA:2512</a> Autosomal recessive primary microcephaly</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.145</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.688</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr1:g.197122305T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=149303254">rs149303254</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs149303254&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=167652%5Balleleid%5D">(criteria_provided,_conflicting_interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.703</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000294732.11">ASPM:ENST00000294732.11:c.3599-4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367408.5">ASPM:ENST00000367408.5:c.1349-4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367409.9">ASPM:ENST00000367409.9:c.3599-4A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9190904</dd>
                                    
                                    <dd>CADD: 0.919 (10.920)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3195%</dd>
                                    <dd>TOPMed: 0.3034%</dd>
                                    <dd>ESP AA: 0.9083%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.3154%</dd>
                                    <dd>ExAC AFR: 0.8726%</dd>
                                    <dd>ExAC NFE: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.8882%</dd>
                                    <dd>gnomAD_E_AMR: 0.0239%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_AFR: 1.0082%</dd>
                                    <dd>gnomAD_G_OTH: 0.1025%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.197121994C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=150125249">rs150125249</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs150125249&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=278319%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.673</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000294732.11">ASPM:ENST00000294732.11:c.3791G&gt;A:p.(Arg1264His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367408.5">ASPM:ENST00000367408.5:c.1541G&gt;A:p.(Arg514His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367409.9">ASPM:ENST00000367409.9:c.3791G&gt;A:p.(Arg1264His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.998645</dd>
                                    
                                    <dd>Polyphen2: 0.994 (D)</dd>
                                    <dd>Mutation Taster: 0.999 (P)</dd>
                                    <dd>SIFT: 0.003 (D)</dd>
                                    <dd>CADD: 0.998 (26.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4792%</dd>
                                    <dd>TOPMed: 0.3711%</dd>
                                    <dd>ESP AA: 1.2256%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.4306%</dd>
                                    <dd>ExAC AFR: 1.2159%</dd>
                                    <dd>ExAC AMR: 0.0709%</dd>
                                    <dd>ExAC NFE: 0.0107%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 1.1215%</dd>
                                    <dd>gnomAD_E_AMR: 0.0898%</dd>
                                    <dd>gnomAD_E_NFE: 0.0099%</dd>
                                    <dd>gnomAD_E_OTH: 0.0367%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.9871%</dd>
                                    <dd>gnomAD_G_AMR: 0.1202%</dd>
                                    <dd>gnomAD_G_NFE: 0.0201%</dd>
                                    <dd>gnomAD_G_OTH: 0.1025%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000108469">RECQL5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.130</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.673</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RECQL5%0DRAD54B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAD54B</a> and phenotypic similarity 0.640 to <a href="http://www.omim.org/114500">Colon cancer, somatic</a> associated with RAD54B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0006716, Hereditary nonpolyposis colorectal carcinoma</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.130</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.673</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.75631581G&gt;A [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201940860">rs201940860</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201940860&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.673</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317905.10">RECQL5:ENST00000317905.10:c.1317C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423245.6">RECQL5:ENST00000423245.6:c.1236C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.71568483</dd>
                                    
                                    <dd>CADD: 0.716 (5.462)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0876%</dd>
                                    <dd>ESP AA: 0.3634%</dd>
                                    <dd>ESP All: 0.1198%</dd>
                                    <dd>ExAC AFR: 0.2861%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC NFE: 0.0062%</dd>
                                    <dd>gnomAD_E_AFR: 0.3042%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.3555%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163395">IGFN1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.129</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.673</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=IGFN1%0DACTG2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTG2</a> and phenotypic similarity 0.610 to <a href="http://www.omim.org/155310">Visceral myopathy</a> associated with ACTG2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0004388, Microcolon</dd><dd>HP:0012378, Fatigue - HP:0002027, Abdominal pain</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.129</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.673</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.201210955G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=542948447">rs542948447</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs542948447&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.970</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335211.9">IGFN1:ENST00000335211.9:c.6062G&gt;A:p.(Gly2021Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295591.12">IGFN1:ENST00000295591.12:c.1242-2551G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.987265</dd>
                                    
                                    <dd>SIFT: 0.197 (T)</dd>
                                    <dd>CADD: 0.987 (18.950)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0998%</dd>
                                    <dd>TOPMed: 0.0998%</dd>
                                    <dd>gnomAD_E_AFR: 0.0581%</dd>
                                    <dd>gnomAD_E_AMR: 0.0050%</dd>
                                    <dd>gnomAD_E_OTH: 0.0671%</dd>
                                    <dd>gnomAD_G_AFR: 0.0947%</dd>
                                    <dd>gnomAD_G_AMR: 0.1220%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.201210874A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=188248150">rs188248150</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs188248150&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.376</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335211.9">IGFN1:ENST00000335211.9:c.5981A&gt;G:p.(Asp1994Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295591.12">IGFN1:ENST00000295591.12:c.1242-2632A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.52313006</dd>
                                    
                                    <dd>SIFT: 0.761 (T)</dd>
                                    <dd>CADD: 0.523 (3.216)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4393%</dd>
                                    <dd>TOPMed: 0.4393%</dd>
                                    <dd>gnomAD_E_AFR: 1.1221%</dd>
                                    <dd>gnomAD_E_AMR: 0.0252%</dd>
                                    <dd>gnomAD_E_NFE: 0.0022%</dd>
                                    <dd>gnomAD_E_OTH: 0.0673%</dd>
                                    <dd>gnomAD_E_SAS: 0.0057%</dd>
                                    <dd>gnomAD_G_AFR: 0.9157%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.201221669G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=141949142">rs141949142</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs141949142&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.966</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295591.12">IGFN1:ENST00000295591.12:c.2753G&gt;A:p.(Arg918Gln)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335211.9">IGFN1:ENST00000335211.9:c.10124G&gt;A:p.(Arg3375Gln)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9958313</dd>
                        
                        <dd>Mutation Taster: 0.854</dd>
                        <dd>SIFT: 0.005 (D)</dd>
                        <dd>CADD: 0.996 (23.800)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.1198%</dd>
                        <dd>TOPMed: 0.0565%</dd>
                        <dd>ESP AA: 0.1589%</dd>
                        <dd>ESP All: 0.0538%</dd>
                        <dd>ExAC AFR: 0.1736%</dd>
                        <dd>ExAC AMR: 0.0087%</dd>
                        <dd>ExAC EAS: 0.0116%</dd>
                        <dd>ExAC NFE: 0.0196%</dd>
                        <dd>gnomAD_E_AFR: 0.2027%</dd>
                        <dd>gnomAD_E_AMR: 0.0089%</dd>
                        <dd>gnomAD_E_EAS: 0.0116%</dd>
                        <dd>gnomAD_E_NFE: 0.0135%</dd>
                        <dd>gnomAD_E_OTH: 0.0548%</dd>
                        <dd>gnomAD_G_AFR: 0.1720%</dd>
                        <dd>gnomAD_G_AMR: 0.1193%</dd>
                        <dd>gnomAD_G_EAS: 0.0617%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr1:g.201225837C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=545610095">rs545610095</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs545610095&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.005</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295591.12">IGFN1:ENST00000295591.12:c.3129C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335211.9">IGFN1:ENST00000335211.9:c.10500C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.0045945644</dd>
                        
                        <dd>CADD: 0.005 (0.020)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0200%</dd>
                        <dd>TOPMed: 0.0056%</dd>
                        <dd>ExAC AFR: 0.0209%</dd>
                        <dd>gnomAD_E_AFR: 0.0198%</dd>
                        <dd>gnomAD_G_AFR: 0.0115%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188162">OTOG</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.120</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.664</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OTOG%0DCDH23&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDH23</a> and phenotypic similarity 0.691 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=96253">Cushing disease</a> associated with CDH23.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002893, Pituitary adenoma</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OTOG%0DCDH23&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDH23</a> and phenotypic similarity 0.401 to mouse mutant of CDH23.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0003866, abnormal defecation</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0003866, abnormal defecation</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/614945">OMIM:614945</a> Deafness, autosomal recessive 18B - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.120</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.664</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.17555877G&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=61736015">rs61736015</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs61736015&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=230050%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.724</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399391.7">OTOG:ENST00000399391.7:c.675G&gt;T:p.(Glu225Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399397.5">OTOG:ENST00000399397.5:c.639G&gt;T:p.(Glu213Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9939744</dd>
                                    
                                    <dd>SIFT: 0.027 (D)</dd>
                                    <dd>CADD: 0.994 (22.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3794%</dd>
                                    <dd>TOPMed: 0.3735%</dd>
                                    <dd>ExAC AFR: 0.7273%</dd>
                                    <dd>gnomAD_E_AFR: 1.0619%</dd>
                                    <dd>gnomAD_E_AMR: 0.0293%</dd>
                                    <dd>gnomAD_E_NFE: 0.0019%</dd>
                                    <dd>gnomAD_E_OTH: 0.0259%</dd>
                                    <dd>gnomAD_G_AFR: 1.1009%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.17640936C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=567966154">rs567966154</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs567966154&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=230169%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.603</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399391.7">OTOG:ENST00000399391.7:c.8071C&gt;T:p.(Arg2691Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399397.5">OTOG:ENST00000399397.5:c.8035C&gt;T:p.(Arg2679Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99765575</dd>
                                    
                                    <dd>Mutation Taster: 0.846</dd>
                                    <dd>SIFT: 0.004 (D)</dd>
                                    <dd>CADD: 0.998 (26.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4992%</dd>
                                    <dd>TOPMed: 0.4237%</dd>
                                    <dd>UK10K: 0.0397%</dd>
                                    <dd>ExAC AFR: 1.3667%</dd>
                                    <dd>ExAC SAS: 0.0132%</dd>
                                    <dd>gnomAD_E_AFR: 1.2793%</dd>
                                    <dd>gnomAD_E_AMR: 0.0923%</dd>
                                    <dd>gnomAD_E_EAS: 0.0099%</dd>
                                    <dd>gnomAD_E_FIN: 0.0073%</dd>
                                    <dd>gnomAD_E_NFE: 0.0255%</dd>
                                    <dd>gnomAD_E_OTH: 0.1076%</dd>
                                    <dd>gnomAD_E_SAS: 0.0089%</dd>
                                    <dd>gnomAD_G_AFR: 1.2618%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.17638480C&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=61995750">rs61995750</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs61995750&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                <div class="col-sm-12">
                    <b>ClinVar: </b>
                    
                        
                        <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                        
                        
                        
                        
                        
                        <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=230161%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                    
                </div>
                <div class="col-sm-6">
                    <b>Variant score: 0.422</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399391.7">OTOG:ENST00000399391.7:c.7861C&gt;A:p.(Gln2621Lys)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399397.5">OTOG:ENST00000399397.5:c.7825C&gt;A:p.(Gln2609Lys)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.98827803</dd>
                        
                        <dd>SIFT: 0.313 (T)</dd>
                        <dd>CADD: 0.988 (19.310)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.5990%</dd>
                        <dd>TOPMed: 0.4850%</dd>
                        <dd>ExAC AFR: 1.6556%</dd>
                        <dd>gnomAD_E_AFR: 1.5754%</dd>
                        <dd>gnomAD_E_AMR: 0.0965%</dd>
                        <dd>gnomAD_E_FIN: 0.0062%</dd>
                        <dd>gnomAD_E_NFE: 0.0213%</dd>
                        <dd>gnomAD_E_OTH: 0.1064%</dd>
                        <dd>gnomAD_G_AFR: 1.4780%</dd>
                        <dd>gnomAD_G_AMR: 0.2387%</dd>
                        <dd>gnomAD_G_NFE: 0.0133%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000241973">PI4KA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.115</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.253</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.938</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PI4KA%0DVAPB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VAPB</a> and phenotypic similarity 0.605 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=803">Amyotrophic lateral sclerosis</a> associated with VAPB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002017, Nausea and vomiting</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/616531">OMIM:616531</a> Polymicrogyria, perisylvian, with cerebellar hypoplasia and arthrogryposis - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.115</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.253</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.938</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr22:g.20824380G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=547780642">rs547780642</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs547780642&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.938</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000255882.11">PI4KA:ENST00000255882.11:c.402C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9409799</dd>
                                    
                                    <dd>CADD: 0.941 (12.290)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0098%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                    <b>chr22:g.20734391G&gt;A [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=543444936">rs543444936</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs543444936&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.798</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000255882.11">PI4KA:ENST00000255882.11:c.3900+4C&gt;T:p.?</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.024561286</dd>
                        
                        <dd>CADD: 0.025 (0.108)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0200%</dd>
                        <dd>TOPMed: 0.0200%</dd>
                        <dd>gnomAD_E_AFR: 0.0075%</dd>
                        <dd>gnomAD_E_SAS: 0.0103%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197748">CFAP43</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.112</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.200</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.399 to <a href="http://www.omim.org/236690">Hydrocephalus, normal pressure, 1</a> associated with CFAP43.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002607, Bowel incontinence</dd><dd>HP:0012378, Fatigue - HP:0002607, Bowel incontinence</dd><dd>HP:0002027, Abdominal pain - HP:0002607, Bowel incontinence</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/236690">OMIM:236690</a> Hydrocephalus, normal pressure, 1 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/617592">OMIM:617592</a> Spermatogenic failure 19 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.112</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.200</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.104207797G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=777065695">rs777065695</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs777065695&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000278064.7">CFAP43:ENST00000278064.7:c.763C&gt;T:p.(His255Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357060.8">CFAP43:ENST00000357060.8:c.763C&gt;T:p.(His255Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369719.2">CFAP43:ENST00000369719.2:c.763C&gt;T:p.(His255Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369720.6">CFAP43:ENST00000369720.6:c.763C&gt;T:p.(His255Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99645185</dd>
                                    
                                    <dd>Polyphen2: 0.917 (P)</dd>
                                    <dd>SIFT: 0.247 (T)</dd>
                                    <dd>CADD: 0.996 (24.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000150990">DHX37</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.110</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.653</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DHX37%0DTSR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TSR2</a> and phenotypic similarity 0.623 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=124">Blackfan-Diamond anemia</a> associated with TSR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002488, Acute leukemia</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0012378, Fatigue</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/273250">OMIM:273250</a> 46, XY sex reversal 11 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/618731">OMIM:618731</a> Neurodevelopmental disorder with brain anomalies and with or without vertebral or cardiac anomalies - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.110</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.653</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.124968548C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=143588731">rs143588731</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs143588731&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.983</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308736.7">DHX37:ENST00000308736.7:c.1394G&gt;A:p.(Arg465Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999999</dd>
                                    
                                    <dd>Polyphen2: 0.156 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.101 (T)</dd>
                                    <dd>CADD: 0.996 (23.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0111%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0391%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC SAS: 0.0304%</dd>
                                    <dd>gnomAD_E_AFR: 0.0524%</dd>
                                    <dd>gnomAD_E_AMR: 0.0238%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_SAS: 0.0195%</dd>
                                    <dd>gnomAD_G_AFR: 0.0458%</dd>
                                    <dd>gnomAD_G_AMR: 0.1196%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.124986230C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=78936225">rs78936225</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs78936225&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.324</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308736.7">DHX37:ENST00000308736.7:c.142G&gt;A:p.(Ala48Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.993661</dd>
                                    
                                    <dd>Polyphen2: 0.082 (B)</dd>
                                    <dd>Mutation Taster: 0.994 (P)</dd>
                                    <dd>SIFT: 0.051 (D)</dd>
                                    <dd>CADD: 0.962 (14.230)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.6789%</dd>
                                    <dd>TOPMed: 0.4675%</dd>
                                    <dd>ESP AA: 1.2256%</dd>
                                    <dd>ESP All: 0.4152%</dd>
                                    <dd>ExAC AFR: 1.5376%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 1.6009%</dd>
                                    <dd>gnomAD_E_AMR: 0.0417%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0729%</dd>
                                    <dd>gnomAD_E_SAS: 0.0097%</dd>
                                    <dd>gnomAD_G_AFR: 1.7890%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr12:g.124950515G&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=144251258">rs144251258</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs144251258&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                <div class="col-sm-12">
                    <b>ClinVar: </b>
                    
                        
                        
                        <span class="label label-info">LIKELY_BENIGN</span>
                        
                        
                        
                        
                        <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=738537%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                    
                </div>
                <div class="col-sm-6">
                    <b>Variant score: 0.953</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308736.7">DHX37:ENST00000308736.7:c.3019C&gt;G:p.(Leu1007Val)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.999999</dd>
                        
                        <dd>Polyphen2: 0.255 (B)</dd>
                        <dd>Mutation Taster: 1.000 (P)</dd>
                        <dd>SIFT: 0.012 (D)</dd>
                        <dd>CADD: 0.997 (25.000)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.1597%</dd>
                        <dd>TOPMed: 0.0900%</dd>
                        <dd>ESP AA: 0.2953%</dd>
                        <dd>ESP All: 0.1000%</dd>
                        <dd>ExAC AFR: 0.2973%</dd>
                        <dd>ExAC AMR: 0.0530%</dd>
                        <dd>gnomAD_E_AFR: 0.2994%</dd>
                        <dd>gnomAD_E_AMR: 0.0708%</dd>
                        <dd>gnomAD_E_OTH: 0.0227%</dd>
                        <dd>gnomAD_G_AFR: 0.2638%</dd>
                        <dd>gnomAD_G_AMR: 0.2387%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000133422">MORC2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.067</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.600</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.481</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.600 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=466768">Autosomal dominant Charcot-Marie-Tooth disease type 2Z</a> associated with MORC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0012378, Fatigue</dd></dl><dl><dt>Phenotypic similarity 0.322 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MORC2">MORC2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0014023, abnormal intestine placement</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0014023, abnormal intestine placement</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MORC2%0DSLC26A4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC26A4</a> and phenotypic similarity 0.612 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=95713">Athyreosis</a> associated with SLC26A4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0011968, Feeding difficulties</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MORC2%0DSLC26A4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC26A4</a> and phenotypic similarity 0.312 to mouse mutant of SLC26A4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0008537, increased susceptibility to induced colitis</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0008537, increased susceptibility to induced colitis</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/616688">OMIM:616688</a> Charcot-Marie-Tooth disease, axonal, type 2Z - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=466768">ORPHA:466768</a> Autosomal dominant Charcot-Marie-Tooth disease type 2Z</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.067</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.600</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.481</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr22:g.30932625G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=565441094">rs565441094</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs565441094&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    <span class="label label-info">LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=534716%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.481</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000215862.8">MORC2:ENST00000215862.8:c.2481C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397641.8">MORC2:ENST00000397641.8:c.2667C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.48251235</dd>
                                    
                                    <dd>CADD: 0.483 (2.861)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0056%</dd>
                                    <dd>ExAC AFR: 0.0193%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.0261%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204120">GIGYF2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.059</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.862</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.309 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=411602">Hereditary late-onset Parkinson disease</a> associated with GIGYF2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002015, Dysphagia</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - HP:0012450, Chronic constipation</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GIGYF2%0DPON2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PON2</a> and phenotypic similarity 0.605 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=803">Amyotrophic lateral sclerosis</a> associated with PON2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002017, Nausea and vomiting</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/607688">OMIM:607688</a> Parkinson disease 11 (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=411602">ORPHA:411602</a> Hereditary late-onset Parkinson disease</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.059</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.862</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.232815714C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=552038398">rs552038398</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs552038398&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373563.8">GIGYF2:ENST00000373563.8:c.2185C&gt;G:p.(Leu729Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409196.7">GIGYF2:ENST00000409196.7:c.2167C&gt;G:p.(Leu723Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409451.7">GIGYF2:ENST00000409451.7:c.2248C&gt;G:p.(Leu750Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409480.5">GIGYF2:ENST00000409480.5:c.2251C&gt;G:p.(Leu751Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409547.5">GIGYF2:ENST00000409547.5:c.2185C&gt;G:p.(Leu729Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000629305.2">GIGYF2:ENST00000629305.2:c.2251C&gt;G:p.(Leu751Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99369043</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>Mutation Taster: 0.993 (P)</dd>
                                    <dd>SIFT: 0.057 (D)</dd>
                                    <dd>CADD: 0.994 (22.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC AFR: 0.0295%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.232811255C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=74560358">rs74560358</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs74560358&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.735</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373563.8">GIGYF2:ENST00000373563.8:c.1910C&gt;T:p.(Ala637Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409196.7">GIGYF2:ENST00000409196.7:c.1892C&gt;T:p.(Ala631Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409451.7">GIGYF2:ENST00000409451.7:c.1973C&gt;T:p.(Ala658Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409480.5">GIGYF2:ENST00000409480.5:c.1976C&gt;T:p.(Ala659Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409547.5">GIGYF2:ENST00000409547.5:c.1910C&gt;T:p.(Ala637Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000629305.2">GIGYF2:ENST00000629305.2:c.1976C&gt;T:p.(Ala659Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999984</dd>
                                    
                                    <dd>Polyphen2: 0.008 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.039 (D)</dd>
                                    <dd>CADD: 0.995 (23.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4593%</dd>
                                    <dd>TOPMed: 0.2684%</dd>
                                    <dd>ESP AA: 0.9759%</dd>
                                    <dd>ESP All: 0.3306%</dd>
                                    <dd>ExAC AFR: 1.0499%</dd>
                                    <dd>ExAC AMR: 0.0779%</dd>
                                    <dd>gnomAD_E_AFR: 1.0851%</dd>
                                    <dd>gnomAD_E_AMR: 0.0447%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.7675%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_OTH: 0.3055%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr2:g.232794920A&gt;G [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=547560862">rs547560862</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs547560862&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.946</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373563.8">GIGYF2:ENST00000373563.8:c.1455A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409196.7">GIGYF2:ENST00000409196.7:c.1437A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409451.7">GIGYF2:ENST00000409451.7:c.1518A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409480.5">GIGYF2:ENST00000409480.5:c.1521A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409547.5">GIGYF2:ENST00000409547.5:c.1455A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000629305.2">GIGYF2:ENST00000629305.2:c.1521A&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9489495</dd>
                        
                        <dd>CADD: 0.949 (12.920)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0200%</dd>
                        <dd>TOPMed: 0.0200%</dd>
                        <dd>ExAC AFR: 0.0096%</dd>
                        <dd>gnomAD_E_AFR: 0.0065%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197321">SVIL</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.058</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.578</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SVIL%0DVAPB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VAPB</a> and phenotypic similarity 0.605 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=803">Amyotrophic lateral sclerosis</a> associated with VAPB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002017, Nausea and vomiting</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.058</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.578</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr10:g.29686658G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=538397571">rs538397571</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs538397571&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.798</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375400.7">SVIL:ENST00000375400.7:c.-399-7C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.037944794</dd>
                                    
                                    <dd>CADD: 0.038 (0.168)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.29512751C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=61737022">rs61737022</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs61737022&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.359</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355867.8">SVIL:ENST00000355867.8:c.3500G&gt;A:p.(Arg1167Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375398.6">SVIL:ENST00000375398.6:c.3404G&gt;A:p.(Arg1135Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375400.7">SVIL:ENST00000375400.7:c.2222G&gt;A:p.(Arg741Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98584205</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>CADD: 0.986 (18.490)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.6190%</dd>
                                    <dd>TOPMed: 0.5256%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 1.6568%</dd>
                                    <dd>ESP EA: 0.0349%</dd>
                                    <dd>ESP All: 0.5843%</dd>
                                    <dd>ExAC AFR: 1.6532%</dd>
                                    <dd>ExAC AMR: 0.1641%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>ExAC OTH: 0.1101%</dd>
                                    <dd>gnomAD_E_AFR: 1.6728%</dd>
                                    <dd>gnomAD_E_AMR: 0.1548%</dd>
                                    <dd>gnomAD_E_NFE: 0.0090%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 1.7407%</dd>
                                    <dd>gnomAD_G_AMR: 0.5967%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204624">DISP3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.041</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.537</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DISP3%0DCCNF&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCNF</a> and phenotypic similarity 0.605 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=803">Amyotrophic lateral sclerosis</a> associated with CCNF.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002017, Nausea and vomiting</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.041</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.537</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr1:g.11525172G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=191852506">rs191852506</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs191852506&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=776980%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.564</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000294484.7">DISP3:ENST00000294484.7:c.2477-4G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.18304151</dd>
                                    
                                    <dd>CADD: 0.183 (0.878)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3395%</dd>
                                    <dd>TOPMed: 0.3815%</dd>
                                    <dd>ESP AA: 1.0249%</dd>
                                    <dd>ESP EA: 0.0120%</dd>
                                    <dd>ESP All: 0.3463%</dd>
                                    <dd>ExAC AFR: 1.1218%</dd>
                                    <dd>ExAC AMR: 0.0259%</dd>
                                    <dd>ExAC EAS: 0.0696%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 1.1576%</dd>
                                    <dd>gnomAD_E_AMR: 0.0984%</dd>
                                    <dd>gnomAD_E_EAS: 0.0580%</dd>
                                    <dd>gnomAD_E_NFE: 0.0189%</dd>
                                    <dd>gnomAD_E_OTH: 0.0366%</dd>
                                    <dd>gnomAD_E_SAS: 0.0098%</dd>
                                    <dd>gnomAD_G_AFR: 1.1577%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.11531617G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=141767050">rs141767050</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs141767050&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.509</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000294484.7">DISP3:ENST00000294484.7:c.3282G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.88709843</dd>
                                    
                                    <dd>CADD: 0.887 (9.473)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3594%</dd>
                                    <dd>TOPMed: 0.4292%</dd>
                                    <dd>ESP AA: 1.2652%</dd>
                                    <dd>ESP EA: 0.0360%</dd>
                                    <dd>ESP All: 0.4418%</dd>
                                    <dd>ExAC AFR: 1.2721%</dd>
                                    <dd>ExAC AMR: 0.0784%</dd>
                                    <dd>ExAC NFE: 0.0522%</dd>
                                    <dd>ExAC OTH: 0.2294%</dd>
                                    <dd>ExAC SAS: 0.0182%</dd>
                                    <dd>gnomAD_E_AFR: 1.3991%</dd>
                                    <dd>gnomAD_E_AMR: 0.0984%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0102%</dd>
                                    <dd>gnomAD_E_NFE: 0.0414%</dd>
                                    <dd>gnomAD_E_OTH: 0.1838%</dd>
                                    <dd>gnomAD_E_SAS: 0.0097%</dd>
                                    <dd>gnomAD_G_AFR: 1.4223%</dd>
                                    <dd>gnomAD_G_NFE: 0.0267%</dd>
                                    <dd>gnomAD_G_OTH: 0.4073%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000277399">GPR179</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.040</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.535</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GPR179%0DMITF&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MITF</a> and phenotypic similarity 0.624 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=897">Waardenburg-Shah syndrome</a> associated with MITF.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0005214, Intestinal obstruction</dd><dd>HP:0012378, Fatigue - HP:0002027, Abdominal pain</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/614565">OMIM:614565</a> Night blindness, congenital stationary (complete), 1E, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=215">ORPHA:215</a> Congenital stationary night blindness</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.040</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.535</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.38335120C&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201283251">rs201283251</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201283251&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.885</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616987.4">GPR179:ENST00000616987.4:c.1558G&gt;T:p.(Ala520Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621958.1">GPR179:ENST00000621958.1:c.1561G&gt;T:p.(Ala521Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9661156</dd>
                                    
                                    <dd>CADD: 0.966 (14.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.1075%</dd>
                                    <dd>ESP AA: 0.3260%</dd>
                                    <dd>ESP EA: 0.0118%</dd>
                                    <dd>ESP All: 0.1174%</dd>
                                    <dd>ExAC AFR: 0.4826%</dd>
                                    <dd>ExAC AMR: 0.0169%</dd>
                                    <dd>gnomAD_E_AFR: 0.3455%</dd>
                                    <dd>gnomAD_E_AMR: 0.0156%</dd>
                                    <dd>gnomAD_E_OTH: 0.0192%</dd>
                                    <dd>gnomAD_G_AFR: 0.3450%</dd>
                                    <dd>gnomAD_G_OTH: 0.1022%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.38329428A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4398144">rs4398144</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs4398144&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=877345%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.185</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616987.4">GPR179:ENST00000616987.4:c.4141T&gt;C:p.(Cys1381Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621958.1">GPR179:ENST00000621958.1:c.4144T&gt;C:p.(Cys1382Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999988</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.995 (23.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.6190%</dd>
                                    <dd>TOPMed: 0.5989%</dd>
                                    <dd>ESP AA: 1.5679%</dd>
                                    <dd>ESP All: 0.5141%</dd>
                                    <dd>ExAC AFR: 1.6861%</dd>
                                    <dd>ExAC AMR: 0.0950%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 1.7208%</dd>
                                    <dd>gnomAD_E_AMR: 0.0923%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0730%</dd>
                                    <dd>gnomAD_E_SAS: 0.0097%</dd>
                                    <dd>gnomAD_G_AFR: 1.9486%</dd>
                                    <dd>gnomAD_G_OTH: 0.2041%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154309">DISP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.040</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.816</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.271 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=280200">Microform holoprosencephaly</a> associated with DISP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002247, Duodenal atresia</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl><dt>Phenotypic similarity 0.197 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DISP1">DISP1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0030314, absent primary palate</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0030314, absent primary palate</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DISP1%0DFOXA2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FOXA2</a> and phenotypic similarity 0.609 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=95494">Combined pituitary hormone deficiencies, genetic forms</a> associated with FOXA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002019, Constipation</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DISP1%0DFOXA2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FOXA2</a> and phenotypic similarity 0.304 to mouse mutant of FOXA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0000474, abnormal foregut morphology</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0000474, abnormal foregut morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=280200">ORPHA:280200</a> Microform holoprosencephaly</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.040</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.816</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.222992015A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=577346199">rs577346199</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs577346199&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000284476.7">DISP1:ENST00000284476.7:c.794A&gt;G:p.(His265Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.998585</dd>
                                    
                                    <dd>Polyphen2: 0.002 (B)</dd>
                                    <dd>Mutation Taster: 0.999 (P)</dd>
                                    <dd>SIFT: 0.252 (T)</dd>
                                    <dd>CADD: 0.994 (22.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.223005113T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=115615723">rs115615723</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs115615723&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=707115%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.636</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000284476.7">DISP1:ENST00000284476.7:c.3716T&gt;C:p.(Leu1239Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9662713</dd>
                                    
                                    <dd>Polyphen2: 0.000 (B)</dd>
                                    <dd>SIFT: 0.248 (T)</dd>
                                    <dd>CADD: 0.966 (14.720)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4193%</dd>
                                    <dd>TOPMed: 0.3910%</dd>
                                    <dd>ESP AA: 1.2029%</dd>
                                    <dd>ESP All: 0.4075%</dd>
                                    <dd>ExAC AFR: 1.0778%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>gnomAD_E_AFR: 1.0985%</dd>
                                    <dd>gnomAD_E_AMR: 0.0536%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_AFR: 1.2606%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000020256">ZFP64</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.035</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.519</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZFP64%0DADA2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ADA2</a> and phenotypic similarity 0.633 to <a href="http://www.omim.org/615688">Vasculitis, autoinflammation, immunodeficiency, and hematologic defects syndrome</a> associated with ADA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002040, Esophageal varix</dd><dd>HP:0012378, Fatigue - HP:0003326, Myalgia</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.035</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.519</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr20:g.52152280T&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=140887206">rs140887206</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs140887206&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.864</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000216923.5">ZFP64:ENST00000216923.5:c.1912A&gt;C:p.(Ile638Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346617.8">ZFP64:ENST00000346617.8:c.1750A&gt;C:p.(Ile584Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371515.8">ZFP64:ENST00000371515.8:c.1906A&gt;C:p.(Ile636Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361387.6">ZFP64:ENST00000361387.6:c.763+7843A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371518.6">ZFP64:ENST00000371518.6:c.763+7843A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9939744</dd>
                                    
                                    <dd>Polyphen2: 0.004 (B)</dd>
                                    <dd>SIFT: 0.091 (T)</dd>
                                    <dd>CADD: 0.994 (22.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2796%</dd>
                                    <dd>TOPMed: 0.1696%</dd>
                                    <dd>ESP AA: 0.4993%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.1768%</dd>
                                    <dd>ExAC AFR: 0.6151%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>gnomAD_E_AFR: 0.6273%</dd>
                                    <dd>gnomAD_E_AMR: 0.0208%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_AFR: 0.6757%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr20:g.52084831C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=61734611">rs61734611</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs61734611&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.173</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361387.6">ZFP64:ENST00000361387.6:c.1664G&gt;A:p.(Arg555Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371523.8">ZFP64:ENST00000371523.8:c.1007G&gt;A:p.(Arg336Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371518.6">ZFP64:ENST00000371518.6:c.1228+3561G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9824612</dd>
                                    
                                    <dd>Polyphen2: 0.003 (B)</dd>
                                    <dd>SIFT: 0.228 (T)</dd>
                                    <dd>CADD: 0.982 (17.560)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.5591%</dd>
                                    <dd>TOPMed: 0.6483%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 1.7476%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.6074%</dd>
                                    <dd>ExAC AFR: 1.9583%</dd>
                                    <dd>ExAC AMR: 0.0519%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>ExAC OTH: 0.2208%</dd>
                                    <dd>gnomAD_E_AFR: 1.9289%</dd>
                                    <dd>gnomAD_E_AMR: 0.0626%</dd>
                                    <dd>gnomAD_E_ASJ: 0.1118%</dd>
                                    <dd>gnomAD_E_NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_OTH: 0.1095%</dd>
                                    <dd>gnomAD_G_AFR: 1.7763%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000181143">MUC16</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.030</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.499</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MUC16%0DMUC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MUC2</a> and phenotypic similarity 0.677 to mouse mutant of MUC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0002957, increased intestinal adenocarcinoma incidence</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0010155, abnormal intestine physiology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.030</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.499</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.8965749A&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=557723809">rs557723809</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs557723809&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397910.8">MUC16:ENST00000397910.8:c.11021T&gt;A:p.(Ile3674Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.982 (17.400)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0103%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.8897504G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=540336252">rs540336252</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs540336252&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.001</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397910.8">MUC16:ENST00000397910.8:c.39372C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 9.2059374E-4</dd>
                                    
                                    <dd>CADD: 0.001 (0.004)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0080%</dd>
                                    <dd>ExAC AFR: 0.0510%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AFR: 0.0458%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198062">POTEH</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.028</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.494</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=POTEH%0DSCN8A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SCN8A</a> and phenotypic similarity 0.628 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=306">Benign familial infantile epilepsy</a> associated with SCN8A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002017, Nausea and vomiting</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.028</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.494</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr22:g.15695791C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=754484257">rs754484257</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs754484257&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.494</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343518.11">POTEH:ENST00000343518.11:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621704.4">POTEH:ENST00000621704.4:c.783C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.49405885</dd>
                                    
                                    <dd>CADD: 0.494 (2.959)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143631">FLG</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.026</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.768</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FLG%0DJAK2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to JAK2</a> and phenotypic similarity 0.631 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=729">Polycythemia vera</a> associated with JAK2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002863, Myelodysplasia</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FLG%0DJAK2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to JAK2</a> and phenotypic similarity 0.282 to mouse mutant of JAK2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0010296, increased hemolymphoid system tumor incidence</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/146700">OMIM:146700</a> Ichthyosis vulgaris - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.026</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.768</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.152307255G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=563762165">rs563762165</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs563762165&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368799.2">FLG:ENST00000368799.2:c.7631C&gt;G:p.(Ser2544Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.997</dd>
                                    
                                    <dd>Polyphen2: 0.883 (P)</dd>
                                    <dd>SIFT: 0.003 (D)</dd>
                                    <dd>CADD: 0.995 (22.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.152303484T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=144046718">rs144046718</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs144046718&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.542</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368799.2">FLG:ENST00000368799.2:c.11402A&gt;G:p.(His3801Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.55</dd>
                                    
                                    <dd>Polyphen2: 0.003 (B)</dd>
                                    <dd>SIFT: 0.450 (T)</dd>
                                    <dd>CADD: 0.492 (2.945)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0998%</dd>
                                    <dd>TOPMed: 0.0199%</dd>
                                    <dd>ExAC AFR: 0.0962%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AFR: 0.0850%</dd>
                                    <dd>gnomAD_E_AMR: 0.0357%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_SAS: 0.0162%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135631">RAB11FIP5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.024</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.479</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RAB11FIP5%0DKIF23&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KIF23</a> and phenotypic similarity 0.622 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98870">Congenital dyserythropoietic anemia type III</a> associated with KIF23.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002249, Melena</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0012378, Fatigue</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.024</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.479</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.73081390G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=530196356">rs530196356</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs530196356&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.935</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000486777.7">RAB11FIP5:ENST00000486777.7:c.1842C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258098.6">RAB11FIP5:ENST00000258098.6:c.1569-5208C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9706235</dd>
                                    
                                    <dd>CADD: 0.971 (15.320)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1398%</dd>
                                    <dd>TOPMed: 0.0980%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>gnomAD_G_AFR: 0.2410%</dd>
                                    <dd>gnomAD_G_NFE: 0.0267%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.73075533G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=149355459">rs149355459</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs149355459&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=708352%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.022</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258098.6">RAB11FIP5:ENST00000258098.6:c.1950C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000486777.7">RAB11FIP5:ENST00000486777.7:c.3963C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.025907934</dd>
                                    
                                    <dd>CADD: 0.026 (0.114)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2596%</dd>
                                    <dd>TOPMed: 0.1625%</dd>
                                    <dd>ESP AA: 0.3858%</dd>
                                    <dd>ESP All: 0.1307%</dd>
                                    <dd>ExAC AFR: 0.6792%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC SAS: 0.0242%</dd>
                                    <dd>gnomAD_E_AFR: 0.5703%</dd>
                                    <dd>gnomAD_E_AMR: 0.0149%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_E_SAS: 0.0260%</dd>
                                    <dd>gnomAD_G_AFR: 0.6086%</dd>
                                    <dd>gnomAD_G_OTH: 0.1022%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">INFRAME_INSERTION</span>
                    <b>chr2:g.73081095T&gt;TTCC [-/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=551578436">rs551578436</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs551578436&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.857</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000486777.7">RAB11FIP5:ENST00000486777.7:c.2134_2136dup:p.(Gly712dup)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258098.6">RAB11FIP5:ENST00000258098.6:c.1569-4914_1569-4913insGGA:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.95704633</dd>
                        
                        <dd>CADD: 0.957 (13.670)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_G_AFR: 0.3933%</dd>
                        <dd>gnomAD_G_AMR: 0.1241%</dd>
                        <dd>gnomAD_G_EAS: 0.5732%</dd>
                        <dd>gnomAD_G_FIN: 0.2341%</dd>
                        <dd>gnomAD_G_NFE: 0.4058%</dd>
                        <dd>gnomAD_G_OTH: 0.4274%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">INTERNAL_FEATURE_ELONGATION</span>
                    <b>chr2:g.73081095T&gt;CTCC [-/1:0/-:0/0]</b>
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.850</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000486777.7">RAB11FIP5:ENST00000486777.7:c.2137delAinsGGAG:p.(Arg713Gly)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258098.6">RAB11FIP5:ENST00000258098.6:c.1569-4913delAinsGGAG:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        
                        <dd>No
                            pathogenicity data
                        </dd>
                        
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000070778">PTPN21</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.022</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.465</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PTPN21%0DPTPN3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTPN3</a> and phenotypic similarity 0.713 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=70567">Cholangiocarcinoma</a> associated with PTPN3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0100574, Biliary tract neoplasm</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.022</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.465</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.88479055C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=371337531">rs371337531</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs371337531&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.761</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000328736.7">PTPN21:ENST00000328736.7:c.2376G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556564.6">PTPN21:ENST00000556564.6:c.2376G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7781293</dd>
                                    
                                    <dd>CADD: 0.778 (6.539)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0271%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.1104%</dd>
                                    <dd>gnomAD_E_AFR: 0.1313%</dd>
                                    <dd>gnomAD_E_AMR: 0.0061%</dd>
                                    <dd>gnomAD_E_OTH: 0.0187%</dd>
                                    <dd>gnomAD_G_AFR: 0.1491%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.88478968G&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=143571855">rs143571855</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs143571855&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.169</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000328736.7">PTPN21:ENST00000328736.7:c.2463C&gt;G:p.(Asp821Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556564.6">PTPN21:ENST00000556564.6:c.2463C&gt;G:p.(Asp821Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96642625</dd>
                                    
                                    <dd>Polyphen2: 0.011 (B)</dd>
                                    <dd>CADD: 0.966 (14.740)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.5990%</dd>
                                    <dd>TOPMed: 0.5081%</dd>
                                    <dd>ESP AA: 1.7703%</dd>
                                    <dd>ESP All: 0.5997%</dd>
                                    <dd>ExAC AFR: 1.8946%</dd>
                                    <dd>ExAC AMR: 0.1053%</dd>
                                    <dd>gnomAD_E_AFR: 1.9595%</dd>
                                    <dd>gnomAD_E_AMR: 0.0812%</dd>
                                    <dd>gnomAD_E_NFE: 0.0011%</dd>
                                    <dd>gnomAD_E_SAS: 0.0095%</dd>
                                    <dd>gnomAD_G_AFR: 1.7898%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000279231">AL356585.3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                                <b>chr13:g.18345097A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1465107930">rs1465107930</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1465107930&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000625196.1">AL356585.3:ENST00000625196.1:n.628+1A&gt;G:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9987411</dd>
                                    
                                    <dd>CADD: 0.999 (29.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000224435">NF1P6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                                <b>chr22:g.15631918A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=56234788">rs56234788</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs56234788&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000426025.1">NF1P6:ENST00000426025.1:n.133+1T&gt;C:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9943766</dd>
                                    
                                    <dd>CADD: 0.994 (22.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000227124">ZNF717</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.75737519G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=80089603">rs80089603</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs80089603&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000478296.5">ZNF717:ENST00000478296.5:c.1954C&gt;T:p.(Pro652Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000652011.2">ZNF717:ENST00000652011.2:c.2104C&gt;T:p.(Pro702Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000477374.5">ZNF717:ENST00000477374.5:c.277+3757C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.014 (D)</dd>
                                    <dd>CADD: 0.993 (21.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183773">AIFM3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.20974799C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=553803201">rs553803201</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs553803201&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399163.6">AIFM3:ENST00000399163.6:c.703C&gt;T:p.(Arg235Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399167.6">AIFM3:ENST00000399167.6:c.703C&gt;T:p.(Arg235Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405089.5">AIFM3:ENST00000405089.5:c.721C&gt;T:p.(Arg241Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000440238.3">AIFM3:ENST00000440238.3:c.703C&gt;T:p.(Arg235Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.998 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.999 (31.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154822">PLCL2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.17010302A&gt;G [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000432376.5">PLCL2:ENST00000432376.5:c.578A&gt;G:p.(Lys193Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615277.5">PLCL2:ENST00000615277.5:c.956A&gt;G:p.(Lys319Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.997246</dd>
                                    
                                    <dd>Polyphen2: 0.005 (B)</dd>
                                    <dd>Mutation Taster: 0.997 (P)</dd>
                                    <dd>SIFT: 0.209 (T)</dd>
                                    <dd>CADD: 0.993 (21.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211648">IGLV1-47</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.22357983A&gt;G [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390294.2">IGLV1-47:ENST00000390294.2:c.74A&gt;G:p.(Gln25Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99411154</dd>
                                    
                                    <dd>CADD: 0.994 (22.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22358090G&gt;C [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=765440546">rs765440546</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs765440546&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.993</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390294.2">IGLV1-47:ENST00000390294.2:c.181G&gt;C:p.(Gly61Arg)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9948714</dd>
                        
                        <dd>CADD: 0.995 (22.900)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>ExAC AFR: 0.0102%</dd>
                        <dd>gnomAD_E_AFR: 0.0065%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22357973G&gt;A [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=553777572">rs553777572</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs553777572&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.984</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390294.2">IGLV1-47:ENST00000390294.2:c.64G&gt;A:p.(Val22Met)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9869983</dd>
                        
                        <dd>CADD: 0.987 (18.860)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0200%</dd>
                        <dd>TOPMed: 0.0008%</dd>
                        <dd>gnomAD_E_EAS: 0.0058%</dd>
                        <dd>gnomAD_E_NFE: 0.0009%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000265190">ANXA8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.47474375C&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=977788">rs977788</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs977788&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583874.5">ANXA8:ENST00000583874.5:c.519G&gt;T:p.(Lys173Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583911.5">ANXA8:ENST00000583911.5:c.390G&gt;T:p.(Lys130Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000585281.5">ANXA8:ENST00000585281.5:c.576G&gt;T:p.(Lys192Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611843.4">ANXA8:ENST00000611843.4:c.690G&gt;T:p.(Lys230Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583448.2">ANXA8:ENST00000583448.2:c.21+9538G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9920567</dd>
                                    
                                    <dd>CADD: 0.992 (21.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211598">IGKV4-1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.991</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.991</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.88885852A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=574609872">rs574609872</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs574609872&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.991</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390243.2">IGKV4-1:ENST00000390243.2:c.62A&gt;G:p.(Asp21Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9938341</dd>
                                    
                                    <dd>CADD: 0.994 (22.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0121%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0193%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr2:g.88885860A&gt;C [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=369054972">rs369054972</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs369054972&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.944</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390243.2">IGKV4-1:ENST00000390243.2:c.70A&gt;C:p.(Met24Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9466665</dd>
                        
                        <dd>CADD: 0.947 (12.730)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>ESP EA: 0.0241%</dd>
                        <dd>ESP All: 0.0162%</dd>
                        <dd>ExAC AFR: 0.0106%</dd>
                        <dd>ExAC EAS: 0.0119%</dd>
                        <dd>ExAC NFE: 0.0031%</dd>
                        <dd>gnomAD_E_AFR: 0.0133%</dd>
                        <dd>gnomAD_E_NFE: 0.0028%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000276950">GSTT4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.983</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.983</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.23998678T&gt;C [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1177906550">rs1177906550</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1177906550&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.983</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621179.5">GSTT4:ENST00000621179.5:c.590A&gt;G:p.(Gln197Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9830956</dd>
                                    
                                    <dd>CADD: 0.983 (17.720)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                    <b>chr22:g.24005240T&gt;C [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1299755982">rs1299755982</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1299755982&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.885</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621179.5">GSTT4:ENST00000621179.5:c.112+5A&gt;G:p.?</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.89316833</dd>
                        
                        <dd>CADD: 0.893 (9.713)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_G_AFR: 0.0115%</dd>
                        <dd>gnomAD_G_EAS: 0.0617%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211974">IGHV2-70D</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.983</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.983</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.106723661A&gt;C [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.983</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390634.3">IGHV2-70D:ENST00000390634.3:c.271T&gt;G:p.(Ser91Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9828209</dd>
                                    
                                    <dd>CADD: 0.983 (17.650)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr14:g.106723657T&gt;C [0/1:0/0:0/0]</b>
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.837</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390634.3">IGHV2-70D:ENST00000390634.3:c.275A&gt;G:p.(Lys92Arg)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8366196</dd>
                        
                        <dd>CADD: 0.837 (7.868)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185842">DNAH14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.980</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.980</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.225364821C&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=577228672">rs577228672</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs577228672&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430092.5">DNAH14:ENST00000430092.5:c.11738C&gt;G:p.(Pro3913Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439375.6">DNAH14:ENST00000439375.6:c.11738C&gt;G:p.(Pro3913Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445597.6">DNAH14:ENST00000445597.6:c.9205+6169C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.991 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.007 (D)</dd>
                                    <dd>CADD: 0.995 (22.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0104%</dd>
                                    <dd>gnomAD_E_AFR: 0.0132%</dd>
                                    <dd>gnomAD_E_AMR: 0.0042%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.225258011T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=566569441">rs566569441</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs566569441&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.963</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430092.5">DNAH14:ENST00000430092.5:c.6899T&gt;C:p.(Ile2300Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439375.6">DNAH14:ENST00000439375.6:c.6899T&gt;C:p.(Ile2300Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445597.6">DNAH14:ENST00000445597.6:c.5584+5594T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9740582</dd>
                                    
                                    <dd>Polyphen2: 0.154 (B)</dd>
                                    <dd>SIFT: 0.446 (T)</dd>
                                    <dd>CADD: 0.974 (15.860)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0263%</dd>
                                    <dd>gnomAD_E_AMR: 0.0042%</dd>
                                    <dd>gnomAD_E_OTH: 0.0260%</dd>
                                    <dd>gnomAD_G_AFR: 0.0573%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.225358646A&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=182215041">rs182215041</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs182215041&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.848</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430092.5">DNAH14:ENST00000430092.5:c.11491A&gt;T:p.(Thr3831Ser)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439375.6">DNAH14:ENST00000439375.6:c.11491A&gt;T:p.(Thr3831Ser)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445597.6">DNAH14:ENST00000445597.6:c.9199A&gt;T:p.(Thr3067Ser)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8576033</dd>
                        
                        <dd>Polyphen2: 0.001 (B)</dd>
                        <dd>SIFT: 0.881 (T)</dd>
                        <dd>CADD: 0.858 (8.465)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0799%</dd>
                        <dd>TOPMed: 0.0064%</dd>
                        <dd>gnomAD_E_AFR: 0.0135%</dd>
                        <dd>gnomAD_E_AMR: 0.0089%</dd>
                        <dd>gnomAD_E_OTH: 0.0270%</dd>
                        <dd>gnomAD_G_AFR: 0.0459%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr1:g.225140842C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=116052053">rs116052053</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs116052053&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.156</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430092.5">DNAH14:ENST00000430092.5:c.4278C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439375.6">DNAH14:ENST00000439375.6:c.4278C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445597.6">DNAH14:ENST00000445597.6:c.3126C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.52564883</dd>
                        
                        <dd>CADD: 0.526 (3.239)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.6190%</dd>
                        <dd>TOPMed: 0.5033%</dd>
                        <dd>UK10K: 0.0132%</dd>
                        <dd>ESP AA: 1.5173%</dd>
                        <dd>ESP All: 0.4599%</dd>
                        <dd>ExAC AFR: 1.8241%</dd>
                        <dd>ExAC NFE: 0.0115%</dd>
                        <dd>gnomAD_E_AFR: 1.7087%</dd>
                        <dd>gnomAD_E_AMR: 0.0668%</dd>
                        <dd>gnomAD_E_NFE: 0.0087%</dd>
                        <dd>gnomAD_E_OTH: 0.0518%</dd>
                        <dd>gnomAD_G_AFR: 1.7075%</dd>
                        <dd>gnomAD_G_AMR: 0.1193%</dd>
                        <dd>gnomAD_G_NFE: 0.0067%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000244437">IGKV3-15</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.977</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.977</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.89085454A&gt;T [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.977</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390252.2">IGKV3-15:ENST00000390252.2:c.68T&gt;A:p.(Val23Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9770385</dd>
                                    
                                    <dd>CADD: 0.977 (16.390)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr2:g.89085219T&gt;G [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=762102539">rs762102539</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs762102539&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.972</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390252.2">IGKV3-15:ENST00000390252.2:c.303A&gt;C:p.(Glu101Asp)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9722668</dd>
                        
                        <dd>CADD: 0.972 (15.570)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_E_AMR: 0.0031%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr2:g.89085284C&gt;T [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=562860462">rs562860462</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs562860462&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.715</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390252.2">IGKV3-15:ENST00000390252.2:c.238G&gt;A:p.(Ala80Thr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.7189394</dd>
                        
                        <dd>CADD: 0.719 (5.512)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0399%</dd>
                        <dd>TOPMed: 0.0399%</dd>
                        <dd>gnomAD_E_EAS: 0.0058%</dd>
                        <dd>gnomAD_E_NFE: 0.0009%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000270629">NBPF14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.977</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.977</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.148548888C&gt;T [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.977</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614999.4">NBPF14:ENST00000614999.4:c.5855G&gt;A:p.(Ser1952Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581201.5">NBPF14:ENST00000581201.5:c.2585+2265G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611826.4">NBPF14:ENST00000611826.4:c.1438+3088G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621066.4">NBPF14:ENST00000621066.4:c.2522+2265G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621070.4">NBPF14:ENST00000621070.4:c.2360+2265G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614329.4">NBPF14:ENST00000614329.4:c.*708+2265G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616120.4">NBPF14:ENST00000616120.4:c.*708+2265G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620240.1">NBPF14:ENST00000620240.1:c.*708+2265G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97666544</dd>
                                    
                                    <dd>CADD: 0.977 (16.320)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.815</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.148533216C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=782519833">rs782519833</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs782519833&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.987</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581201.5">NBPF14:ENST00000581201.5:c.2618G&gt;A:p.(Arg873His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611826.4">NBPF14:ENST00000611826.4:c.1523G&gt;A:p.(Arg508His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614999.4">NBPF14:ENST00000614999.4:c.8249G&gt;A:p.(Arg2750His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621066.4">NBPF14:ENST00000621066.4:c.2555G&gt;A:p.(Arg852His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621070.4">NBPF14:ENST00000621070.4:c.2393G&gt;A:p.(Arg798His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614329.4">NBPF14:ENST00000614329.4:c.*741G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616120.4">NBPF14:ENST00000616120.4:c.*741G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620240.1">NBPF14:ENST00000620240.1:c.*741G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98714715</dd>
                                    
                                    <dd>CADD: 0.987 (18.910)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.148595576C&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1263755885">rs1263755885</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1263755885&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.643</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581201.5">NBPF14:ENST00000581201.5:c.142G&gt;T:p.(Ala48Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611826.4">NBPF14:ENST00000611826.4:c.142G&gt;T:p.(Ala48Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614999.4">NBPF14:ENST00000614999.4:c.142G&gt;T:p.(Ala48Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616120.4">NBPF14:ENST00000616120.4:c.142G&gt;T:p.(Ala48Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620240.1">NBPF14:ENST00000620240.1:c.140G&gt;T:p.(Gly47Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621070.4">NBPF14:ENST00000621070.4:c.142G&gt;T:p.(Ala48Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614329.4">NBPF14:ENST00000614329.4:c.-7+148G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8174844</dd>
                                    
                                    <dd>CADD: 0.817 (7.387)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.9477%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr1:g.148587330C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1343209236">rs1343209236</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1343209236&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.308</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581201.5">NBPF14:ENST00000581201.5:c.1062G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614329.4">NBPF14:ENST00000614329.4:c.249G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614999.4">NBPF14:ENST00000614999.4:c.1062G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616120.4">NBPF14:ENST00000616120.4:c.249G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620240.1">NBPF14:ENST00000620240.1:c.957G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621070.4">NBPF14:ENST00000621070.4:c.1062G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611826.4">NBPF14:ENST00000611826.4:c.567-2654G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621066.4">NBPF14:ENST00000621066.4:c.72-861G&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.308169</dd>
                        
                        <dd>CADD: 0.308 (1.600)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr1:g.148586448G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1360509174">rs1360509174</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1360509174&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.288</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581201.5">NBPF14:ENST00000581201.5:c.1113C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614329.4">NBPF14:ENST00000614329.4:c.300C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614999.4">NBPF14:ENST00000614999.4:c.1113C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616120.4">NBPF14:ENST00000616120.4:c.300C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620240.1">NBPF14:ENST00000620240.1:c.1008C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621066.4">NBPF14:ENST00000621066.4:c.93C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621070.4">NBPF14:ENST00000621070.4:c.1113C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611826.4">NBPF14:ENST00000611826.4:c.567-1772C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.2879672</dd>
                        
                        <dd>CADD: 0.288 (1.475)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000268320">SCGB1C2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.976</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.976</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.137603C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1239602891">rs1239602891</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1239602891&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.976</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000595228.3">SCGB1C2:ENST00000595228.3:c.13C&gt;T:p.(Arg5Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9761219</dd>
                                    
                                    <dd>CADD: 0.976 (16.220)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr17:g.138213G&gt;A [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1398701246">rs1398701246</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1398701246&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.749</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000595228.3">SCGB1C2:ENST00000595228.3:c.66G&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.74927366</dd>
                        
                        <dd>CADD: 0.749 (6.008)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000035403">VCL</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.689</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.304 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=VCL">VCL</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0000474, abnormal foregut morphology</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0000474, abnormal foregut morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=VCL%0DPFN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PFN1</a> and phenotypic similarity 0.605 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=803">Amyotrophic lateral sclerosis</a> associated with PFN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002017, Nausea and vomiting</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/611407">OMIM:611407</a> Cardiomyopathy, dilated, 1W - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/613255">OMIM:613255</a> Cardiomyopathy, hypertrophic, 15 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=154">ORPHA:154</a> Familial isolated dilated cardiomyopathy</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.689</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.74089288G&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=377289804">rs377289804</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs377289804&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000211998.9">VCL:ENST00000211998.9:c.1115G&gt;T:p.(Arg372Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372755.7">VCL:ENST00000372755.7:c.1115G&gt;T:p.(Arg372Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.212 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.089 (T)</dd>
                                    <dd>CADD: 0.996 (24.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0289%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr10:g.74095669C&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=150120464">rs150120464</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs150120464&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=54749%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.383</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000211998.9">VCL:ENST00000211998.9:c.1557C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372755.7">VCL:ENST00000372755.7:c.1557C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9457999</dd>
                                    
                                    <dd>CADD: 0.946 (12.660)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.6789%</dd>
                                    <dd>TOPMed: 0.4555%</dd>
                                    <dd>ESP AA: 1.2483%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.4383%</dd>
                                    <dd>ExAC AFR: 1.5006%</dd>
                                    <dd>ExAC AMR: 0.1037%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>ExAC OTH: 0.1101%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 1.6861%</dd>
                                    <dd>gnomAD_E_AMR: 0.0804%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.1458%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 1.5575%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104972">LILRB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.972</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.972</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54631718C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=766498638">rs766498638</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs766498638&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.972</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324602.11">LILRB1:ENST00000324602.11:c.289C&gt;T:p.(Arg97Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396315.5">LILRB1:ENST00000396315.5:c.289C&gt;T:p.(Arg97Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396317.5">LILRB1:ENST00000396317.5:c.289C&gt;T:p.(Arg97Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396327.7">LILRB1:ENST00000396327.7:c.289C&gt;T:p.(Arg97Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396331.5">LILRB1:ENST00000396331.5:c.289C&gt;T:p.(Arg97Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396332.8">LILRB1:ENST00000396332.8:c.289C&gt;T:p.(Arg97Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427581.6">LILRB1:ENST00000427581.6:c.397C&gt;T:p.(Arg133Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.974</dd>
                                    
                                    <dd>Polyphen2: 0.509 (P)</dd>
                                    <dd>SIFT: 0.026 (D)</dd>
                                    <dd>CADD: 0.968 (14.990)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.010</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54631632C&gt;T [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1061679">rs1061679</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1061679&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.010</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324602.11">LILRB1:ENST00000324602.11:c.203C&gt;T:p.(Pro68Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396315.5">LILRB1:ENST00000396315.5:c.203C&gt;T:p.(Pro68Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396317.5">LILRB1:ENST00000396317.5:c.203C&gt;T:p.(Pro68Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396327.7">LILRB1:ENST00000396327.7:c.203C&gt;T:p.(Pro68Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396331.5">LILRB1:ENST00000396331.5:c.203C&gt;T:p.(Pro68Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396332.8">LILRB1:ENST00000396332.8:c.203C&gt;T:p.(Pro68Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427581.6">LILRB1:ENST00000427581.6:c.311C&gt;T:p.(Pro104Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.009624243</dd>
                                    
                                    <dd>CADD: 0.010 (0.042)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.54631686C&gt;T [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200880414">rs200880414</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200880414&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.907</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324602.11">LILRB1:ENST00000324602.11:c.257C&gt;T:p.(Pro86Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396315.5">LILRB1:ENST00000396315.5:c.257C&gt;T:p.(Pro86Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396317.5">LILRB1:ENST00000396317.5:c.257C&gt;T:p.(Pro86Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396327.7">LILRB1:ENST00000396327.7:c.257C&gt;T:p.(Pro86Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396331.5">LILRB1:ENST00000396331.5:c.257C&gt;T:p.(Pro86Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396332.8">LILRB1:ENST00000396332.8:c.257C&gt;T:p.(Pro86Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427581.6">LILRB1:ENST00000427581.6:c.365C&gt;T:p.(Pro122Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.92</dd>
                        
                        <dd>Polyphen2: 0.119 (B)</dd>
                        <dd>SIFT: 0.080 (T)</dd>
                        <dd>CADD: 0.003 (0.015)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0799%</dd>
                        <dd>TOPMed: 0.0799%</dd>
                        <dd>ExAC AFR: 0.0961%</dd>
                        <dd>ExAC AMR: 0.0173%</dd>
                        <dd>ExAC NFE: 0.0225%</dd>
                        <dd>ExAC SAS: 0.0061%</dd>
                        <dd>gnomAD_E_AFR: 0.0850%</dd>
                        <dd>gnomAD_E_AMR: 0.0149%</dd>
                        <dd>gnomAD_E_NFE: 0.0179%</dd>
                        <dd>gnomAD_E_SAS: 0.0065%</dd>
                        <dd>gnomAD_G_AFR: 0.0804%</dd>
                        <dd>gnomAD_G_NFE: 0.0201%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr19:g.54631963C&gt;T [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=762235922">rs762235922</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs762235922&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.762</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324602.11">LILRB1:ENST00000324602.11:c.387C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396315.5">LILRB1:ENST00000396315.5:c.387C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396317.5">LILRB1:ENST00000396317.5:c.387C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396327.7">LILRB1:ENST00000396327.7:c.387C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396331.5">LILRB1:ENST00000396331.5:c.387C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396332.8">LILRB1:ENST00000396332.8:c.387C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427581.6">LILRB1:ENST00000427581.6:c.495C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.7623159</dd>
                        
                        <dd>CADD: 0.762 (6.240)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>ExAC NFE: 0.0015%</dd>
                        <dd>gnomAD_E_NFE: 0.0009%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.54631965A&gt;T [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=767704704">rs767704704</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs767704704&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.422</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324602.11">LILRB1:ENST00000324602.11:c.389A&gt;T:p.(Gln130Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396315.5">LILRB1:ENST00000396315.5:c.389A&gt;T:p.(Gln130Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396317.5">LILRB1:ENST00000396317.5:c.389A&gt;T:p.(Gln130Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396327.7">LILRB1:ENST00000396327.7:c.389A&gt;T:p.(Gln130Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396331.5">LILRB1:ENST00000396331.5:c.389A&gt;T:p.(Gln130Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396332.8">LILRB1:ENST00000396332.8:c.389A&gt;T:p.(Gln130Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427581.6">LILRB1:ENST00000427581.6:c.497A&gt;T:p.(Gln166Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.4221701</dd>
                        
                        <dd>CADD: 0.422 (2.382)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_E_NFE: 0.0009%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162702">ZNF281</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.406</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF281%0DSOX2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SOX2</a> and phenotypic similarity 0.611 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3157">Septo-optic dysplasia spectrum</a> associated with SOX2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002032, Esophageal atresia</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002019, Constipation</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF281%0DSOX2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SOX2</a> and phenotypic similarity 0.431 to mouse mutant of SOX2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0000467, abnormal esophagus morphology</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0001663, abnormal digestive system physiology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.406</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.200407111C&gt;G [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=113285043">rs113285043</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs113285043&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.406</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000294740.3">ZNF281:ENST00000294740.3:c.2595G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367352.3">ZNF281:ENST00000367352.3:c.2487G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367353.2">ZNF281:ENST00000367353.2:c.2595G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99168235</dd>
                                    
                                    <dd>CADD: 0.992 (20.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4593%</dd>
                                    <dd>TOPMed: 0.4810%</dd>
                                    <dd>ESP AA: 1.5433%</dd>
                                    <dd>ESP All: 0.5228%</dd>
                                    <dd>ExAC AFR: 1.6340%</dd>
                                    <dd>ExAC AMR: 0.0432%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 1.6795%</dd>
                                    <dd>gnomAD_E_AMR: 0.0625%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 1.4436%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000274933">TBC1D3I</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.970</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.970</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.36254409G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1228374843">rs1228374843</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1228374843&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.970</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621034.1">TBC1D3I:ENST00000621034.1:c.1195C&gt;T:p.(Arg399Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616671.4">TBC1D3I:ENST00000616671.4:c.*153C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618620.4">TBC1D3I:ENST00000618620.4:c.*153C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9703517</dd>
                                    
                                    <dd>CADD: 0.970 (15.280)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000127334">DYRK2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.402</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DYRK2%0DGLI2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GLI2</a> and phenotypic similarity 0.609 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=95494">Combined pituitary hormone deficiencies, genetic forms</a> associated with GLI2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002019, Constipation</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DYRK2%0DGLI2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GLI2</a> and phenotypic similarity 0.333 to mouse mutant of GLI2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0010884, esophagus stenosis</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0000474, abnormal foregut morphology</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DYRK2%0DGLI2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GLI2</a> and phenotypic similarity 0.236 to fish mutant of GLI2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - ZP:0012368, abnormal(ly) malformed esophagus</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.402</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr12:g.67649166C&gt;T [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200090077">rs200090077</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200090077&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=713719%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.402</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344096.3">DYRK2:ENST00000344096.3:c.33C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393555.3">DYRK2:ENST00000393555.3:c.-38C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9768261</dd>
                                    
                                    <dd>CADD: 0.977 (16.350)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3395%</dd>
                                    <dd>TOPMed: 0.3711%</dd>
                                    <dd>ESP AA: 0.5638%</dd>
                                    <dd>ESP All: 0.1898%</dd>
                                    <dd>ExAC AFR: 1.6768%</dd>
                                    <dd>ExAC AMR: 0.0545%</dd>
                                    <dd>ExAC NFE: 0.0021%</dd>
                                    <dd>gnomAD_E_AFR: 1.2373%</dd>
                                    <dd>gnomAD_E_AMR: 0.0354%</dd>
                                    <dd>gnomAD_E_NFE: 0.0024%</dd>
                                    <dd>gnomAD_G_AFR: 1.2349%</dd>
                                    <dd>gnomAD_G_AMR: 0.3916%</dd>
                                    <dd>gnomAD_G_NFE: 0.0136%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197978">GOLGA6L9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.961</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.961</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr15:g.82430167A&gt;G [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1448521888">rs1448521888</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1448521888&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.961</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618348.1">GOLGA6L9:ENST00000618348.1:c.84+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9614522</dd>
                                    
                                    <dd>CADD: 0.961 (14.140)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr15:g.82434051C&gt;G [1/1:0/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1305272775">rs1305272775</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1305272775&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.790</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618348.1">GOLGA6L9:ENST00000618348.1:c.451C&gt;G:p.(Gln151Glu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.79010606</dd>
                        
                        <dd>CADD: 0.790 (6.780)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211950">IGHV1-24</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.960</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.960</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.106276944C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=782185006">rs782185006</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs782185006&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.960</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390610.2">IGHV1-24:ENST00000390610.2:c.40G&gt;A:p.(Ala14Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9600055</dd>
                                    
                                    <dd>CADD: 0.960 (13.980)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC NFE: 0.0015%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105499">PLA2G4C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.390</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLA2G4C%0DPLA2G2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLA2G2A</a> and phenotypic similarity 0.640 to <a href="http://www.omim.org/114500">?Colorectal cancer, susceptibility to</a> associated with PLA2G2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0006716, Hereditary nonpolyposis colorectal carcinoma</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.390</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.48105392C&gt;T [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=11564522">rs11564522</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs11564522&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.390</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354276.7">PLA2G4C:ENST00000354276.7:c.61G&gt;A:p.(Glu21Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000599111.5">PLA2G4C:ENST00000599111.5:c.91G&gt;A:p.(Glu31Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000599921.6">PLA2G4C:ENST00000599921.6:c.61G&gt;A:p.(Glu21Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8604917</dd>
                                    
                                    <dd>Polyphen2: 0.009 (B)</dd>
                                    <dd>SIFT: 0.825 (T)</dd>
                                    <dd>CADD: 0.860 (8.554)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3395%</dd>
                                    <dd>TOPMed: 0.4675%</dd>
                                    <dd>ESP AA: 1.4299%</dd>
                                    <dd>ESP All: 0.4844%</dd>
                                    <dd>ExAC AFR: 1.6173%</dd>
                                    <dd>ExAC AMR: 0.0951%</dd>
                                    <dd>ExAC SAS: 0.0182%</dd>
                                    <dd>gnomAD_E_AFR: 1.5686%</dd>
                                    <dd>gnomAD_E_AMR: 0.1014%</dd>
                                    <dd>gnomAD_E_OTH: 0.0730%</dd>
                                    <dd>gnomAD_E_SAS: 0.0097%</dd>
                                    <dd>gnomAD_G_AFR: 1.4790%</dd>
                                    <dd>gnomAD_G_AMR: 0.1196%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.2037%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000277865">GOLGA6L22</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.954</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.954</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr15:g.22466164T&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=368493848">rs368493848</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs368493848&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.954</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620691.4">GOLGA6L22:ENST00000620691.4:c.1823T&gt;G:p.(Met608Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622895.1">GOLGA6L22:ENST00000622895.1:c.1772T&gt;G:p.(Met591Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.95821697</dd>
                                    
                                    <dd>CADD: 0.958 (13.790)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_NFE: 0.0078%</dd>
                                    <dd>gnomAD_E_SAS: 0.0094%</dd>
                                    <dd>gnomAD_G_NFE: 0.0290%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000153786">ZDHHC7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.951</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.010</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.951</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr16:g.84977104G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=139969155">rs139969155</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs139969155&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.951</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313732.8">ZDHHC7:ENST00000313732.8:c.741C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000564466.5">ZDHHC7:ENST00000564466.5:c.852C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.95574117</dd>
                                    
                                    <dd>CADD: 0.956 (13.540)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0231%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0349%</dd>
                                    <dd>ESP All: 0.0308%</dd>
                                    <dd>ExAC AFR: 0.0288%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC FIN: 0.0151%</dd>
                                    <dd>ExAC NFE: 0.0165%</dd>
                                    <dd>gnomAD_E_AFR: 0.0327%</dd>
                                    <dd>gnomAD_E_AMR: 0.0208%</dd>
                                    <dd>gnomAD_E_FIN: 0.0224%</dd>
                                    <dd>gnomAD_E_NFE: 0.0206%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000280411">IGHV1-69D</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.949</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.010</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.949</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.106762174C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1367563346">rs1367563346</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1367563346&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.949</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000624687.1">IGHV1-69D:ENST00000624687.1:c.271G&gt;A:p.(Ala91Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9489495</dd>
                                    
                                    <dd>CADD: 0.949 (12.920)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr14:g.106762218G&gt;A [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1470737182">rs1470737182</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1470737182&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.005</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000624687.1">IGHV1-69D:ENST00000624687.1:c.227C&gt;T:p.(Thr76Ile)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.0050528646</dd>
                        
                        <dd>CADD: 0.005 (0.022)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000110711">AIP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.302</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.601</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.604 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=963">Acromegaly</a> associated with AIP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0006767, Pituitary prolactin cell adenoma</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002829, Arthralgia</dd></dl><dl><dt>Phenotypic similarity 0.373 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=AIP">AIP</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0002041, increased pituitary adenoma incidence</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=AIP%0DRET&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RET</a> and phenotypic similarity 0.632 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=388">Hirschsprung disease</a> associated with RET.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0200008, Intestinal polyposis</dd><dd>HP:0012378, Fatigue - HP:0002027, Abdominal pain</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=AIP%0DRET&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RET</a> and phenotypic similarity 0.524 to mouse mutant of RET.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0000495, abnormal colon morphology</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0001270, distended abdomen</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=AIP%0DRET&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RET</a> and phenotypic similarity 0.301 to fish mutant of RET.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - ZP:0009740, abnormal(ly) absent posterior intestine neuron</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - ZP:0009738, abnormal(ly) disrupted digestive system process</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/102200">OMIM:102200</a> Pituitary adenoma 1, multiple types - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=963">ORPHA:963</a> Acromegaly</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=99725">ORPHA:99725</a> Pituitary gigantism</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.010</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.302</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.601</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.67490920G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4930199">rs4930199</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs4930199&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=838853%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.988</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000279146.8">AIP:ENST00000279146.8:c.920G&gt;A:p.(Arg307Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9892848</dd>
                                    
                                    <dd>CADD: 0.989 (19.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0127%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr11:g.67490891C&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=35665586">rs35665586</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs35665586&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=321678%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.215</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000279146.8">AIP:ENST00000279146.8:c.891C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9700084</dd>
                                    
                                    <dd>CADD: 0.970 (15.230)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.6190%</dd>
                                    <dd>TOPMed: 0.5407%</dd>
                                    <dd>ESP AA: 1.5938%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.5471%</dd>
                                    <dd>ExAC AFR: 1.8379%</dd>
                                    <dd>ExAC AMR: 0.0702%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_AFR: 1.9103%</dd>
                                    <dd>gnomAD_E_AMR: 0.0955%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0185%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 1.8557%</dd>
                                    <dd>gnomAD_G_AMR: 0.3589%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000175820">CCDC168</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.937</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.009</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.937</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr13:g.102741437A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=555941623">rs555941623</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs555941623&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.961</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322527.4">CCDC168:ENST00000322527.4:c.9260T&gt;C:p.(Phe3087Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98883134</dd>
                                    
                                    <dd>CADD: 0.989 (19.520)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0358%</dd>
                                    <dd>ExAC AFR: 0.1901%</dd>
                                    <dd>gnomAD_E_AFR: 0.1452%</dd>
                                    <dd>gnomAD_E_AMR: 0.0042%</dd>
                                    <dd>gnomAD_G_AFR: 0.0458%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr13:g.102730637G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=141027642">rs141027642</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs141027642&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.913</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322527.4">CCDC168:ENST00000322527.4:c.20060C&gt;T:p.(Ser6687Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9869383</dd>
                                    
                                    <dd>CADD: 0.987 (18.840)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2196%</dd>
                                    <dd>TOPMed: 0.1457%</dd>
                                    <dd>ESP AA: 0.1445%</dd>
                                    <dd>ESP All: 0.0438%</dd>
                                    <dd>ExAC AFR: 0.3157%</dd>
                                    <dd>gnomAD_E_AFR: 0.4382%</dd>
                                    <dd>gnomAD_E_AMR: 0.0417%</dd>
                                    <dd>gnomAD_E_OTH: 0.0757%</dd>
                                    <dd>gnomAD_G_AFR: 0.4123%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr13:g.102731514T&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=541214181">rs541214181</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs541214181&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.926</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322527.4">CCDC168:ENST00000322527.4:c.19183A&gt;T:p.(Ile6395Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9455497</dd>
                        
                        <dd>CADD: 0.946 (12.640)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0399%</dd>
                        <dd>TOPMed: 0.0303%</dd>
                        <dd>ExAC AFR: 0.1399%</dd>
                        <dd>gnomAD_E_AFR: 0.1447%</dd>
                        <dd>gnomAD_G_AFR: 0.0458%</dd>
                        <dd>gnomAD_G_OTH: 0.1020%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr13:g.102733676C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=192543665">rs192543665</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs192543665&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.839</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322527.4">CCDC168:ENST00000322527.4:c.17021G&gt;A:p.(Arg5674Lys)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9756219</dd>
                        
                        <dd>CADD: 0.976 (16.130)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.2995%</dd>
                        <dd>TOPMed: 0.2150%</dd>
                        <dd>ExAC AFR: 0.6724%</dd>
                        <dd>gnomAD_E_AFR: 0.7120%</dd>
                        <dd>gnomAD_E_AMR: 0.0501%</dd>
                        <dd>gnomAD_E_OTH: 0.0790%</dd>
                        <dd>gnomAD_G_AFR: 0.5622%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr13:g.102731343A&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=147982063">rs147982063</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs147982063&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.801</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322527.4">CCDC168:ENST00000322527.4:c.19354T&gt;A:p.(Ser6452Thr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9237921</dd>
                        
                        <dd>CADD: 0.924 (11.180)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.2995%</dd>
                        <dd>TOPMed: 0.2094%</dd>
                        <dd>ExAC AFR: 0.6524%</dd>
                        <dd>gnomAD_E_AFR: 0.6839%</dd>
                        <dd>gnomAD_E_AMR: 0.0501%</dd>
                        <dd>gnomAD_E_OTH: 0.0789%</dd>
                        <dd>gnomAD_G_AFR: 0.5269%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr13:g.102731262G&gt;C [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=143423314">rs143423314</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs143423314&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.789</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322527.4">CCDC168:ENST00000322527.4:c.19435C&gt;G:p.(Pro6479Ala)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.90942675</dd>
                        
                        <dd>CADD: 0.909 (10.430)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.2995%</dd>
                        <dd>TOPMed: 0.2094%</dd>
                        <dd>ExAC AFR: 0.6542%</dd>
                        <dd>gnomAD_E_AFR: 0.6839%</dd>
                        <dd>gnomAD_E_AMR: 0.0501%</dd>
                        <dd>gnomAD_E_OTH: 0.0792%</dd>
                        <dd>gnomAD_G_AFR: 0.5269%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr13:g.102734514T&gt;C [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=148723872">rs148723872</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs148723872&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.532</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322527.4">CCDC168:ENST00000322527.4:c.16183A&gt;G:p.(Met5395Val)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.624076</dd>
                        
                        <dd>CADD: 0.624 (4.249)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.3195%</dd>
                        <dd>TOPMed: 0.2429%</dd>
                        <dd>ESP AA: 0.3613%</dd>
                        <dd>ESP All: 0.1095%</dd>
                        <dd>ExAC AFR: 0.6990%</dd>
                        <dd>gnomAD_E_AFR: 0.7361%</dd>
                        <dd>gnomAD_E_AMR: 0.0543%</dd>
                        <dd>gnomAD_E_OTH: 0.0790%</dd>
                        <dd>gnomAD_G_AFR: 0.6077%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr13:g.102737769C&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=143742365">rs143742365</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs143742365&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.488</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322527.4">CCDC168:ENST00000322527.4:c.12928G&gt;C:p.(Glu4310Gln)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.95502204</dd>
                        
                        <dd>CADD: 0.955 (13.470)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.2796%</dd>
                        <dd>TOPMed: 0.2684%</dd>
                        <dd>UK10K: 0.0132%</dd>
                        <dd>ESP AA: 0.8671%</dd>
                        <dd>ESP All: 0.2628%</dd>
                        <dd>ExAC AFR: 1.5292%</dd>
                        <dd>gnomAD_E_AFR: 1.0507%</dd>
                        <dd>gnomAD_E_AMR: 0.0543%</dd>
                        <dd>gnomAD_E_NFE: 0.0053%</dd>
                        <dd>gnomAD_E_SAS: 0.0044%</dd>
                        <dd>gnomAD_G_AFR: 0.6769%</dd>
                        <dd>gnomAD_G_OTH: 0.1020%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr13:g.102743033A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=143357855">rs143357855</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs143357855&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.412</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322527.4">CCDC168:ENST00000322527.4:c.7664T&gt;C:p.(Phe2555Ser)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9749389</dd>
                        
                        <dd>CADD: 0.975 (16.010)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.3395%</dd>
                        <dd>TOPMed: 0.2971%</dd>
                        <dd>UK10K: 0.0132%</dd>
                        <dd>ExAC AFR: 1.6619%</dd>
                        <dd>gnomAD_E_AFR: 1.1499%</dd>
                        <dd>gnomAD_E_AMR: 0.0589%</dd>
                        <dd>gnomAD_E_NFE: 0.0053%</dd>
                        <dd>gnomAD_E_SAS: 0.0044%</dd>
                        <dd>gnomAD_G_AFR: 0.8137%</dd>
                        <dd>gnomAD_G_OTH: 0.1018%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr13:g.102741897T&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=150489495">rs150489495</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs150489495&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.401</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322527.4">CCDC168:ENST00000322527.4:c.8800A&gt;T:p.(Ile2934Phe)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9045007</dd>
                        
                        <dd>CADD: 0.905 (10.200)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.3395%</dd>
                        <dd>TOPMed: 0.3385%</dd>
                        <dd>ESP AA: 1.0838%</dd>
                        <dd>ESP All: 0.3285%</dd>
                        <dd>ExAC AFR: 1.6325%</dd>
                        <dd>gnomAD_E_AFR: 1.2773%</dd>
                        <dd>gnomAD_E_AMR: 0.0626%</dd>
                        <dd>gnomAD_E_NFE: 0.0053%</dd>
                        <dd>gnomAD_E_SAS: 0.0044%</dd>
                        <dd>gnomAD_G_AFR: 0.8704%</dd>
                        <dd>gnomAD_G_OTH: 0.1018%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr13:g.102748553A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=116822868">rs116822868</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs116822868&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.352</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322527.4">CCDC168:ENST00000322527.4:c.2144T&gt;C:p.(Ile715Thr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.7958732</dd>
                        
                        <dd>CADD: 0.796 (6.901)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.3395%</dd>
                        <dd>TOPMed: 0.2978%</dd>
                        <dd>ESP AA: 0.9393%</dd>
                        <dd>ESP All: 0.2848%</dd>
                        <dd>ExAC AFR: 1.6340%</dd>
                        <dd>gnomAD_E_AFR: 1.1585%</dd>
                        <dd>gnomAD_E_AMR: 0.0585%</dd>
                        <dd>gnomAD_E_NFE: 0.0053%</dd>
                        <dd>gnomAD_E_SAS: 0.0044%</dd>
                        <dd>gnomAD_G_AFR: 0.8255%</dd>
                        <dd>gnomAD_G_OTH: 0.1025%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr13:g.102743754C&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=146287893">rs146287893</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs146287893&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.000</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322527.4">CCDC168:ENST00000322527.4:c.6943G&gt;C:p.(Glu2315Gln)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 4.6038628E-4</dd>
                        
                        <dd>CADD: 0.000 (0.002)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.2995%</dd>
                        <dd>TOPMed: 0.2947%</dd>
                        <dd>UK10K: 0.0132%</dd>
                        <dd>ESP AA: 0.8671%</dd>
                        <dd>ESP All: 0.2629%</dd>
                        <dd>ExAC AFR: 1.5918%</dd>
                        <dd>gnomAD_E_AFR: 1.1199%</dd>
                        <dd>gnomAD_E_AMR: 0.0543%</dd>
                        <dd>gnomAD_E_NFE: 0.0054%</dd>
                        <dd>gnomAD_E_SAS: 0.0044%</dd>
                        <dd>gnomAD_G_AFR: 0.8131%</dd>
                        <dd>gnomAD_G_OTH: 0.1025%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000266074">BAHCC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.933</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.009</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.933</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.81463634C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=191228108">rs191228108</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs191228108&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.935</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000307745.8">BAHCC1:ENST00000307745.8:c.7644C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000584436.7">BAHCC1:ENST00000584436.7:c.7737C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9580241</dd>
                                    
                                    <dd>CADD: 0.958 (13.770)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0350%</dd>
                                    <dd>ESP AA: 0.1646%</dd>
                                    <dd>ESP All: 0.0551%</dd>
                                    <dd>ExAC AFR: 0.1371%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>gnomAD_E_AFR: 0.1313%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 0.1261%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.81447309C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537847945">rs537847945</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537847945&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.932</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000307745.8">BAHCC1:ENST00000307745.8:c.3437C&gt;T:p.(Ala1146Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000584436.7">BAHCC1:ENST00000584436.7:c.3530C&gt;T:p.(Ala1177Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.937</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.063 (T)</dd>
                                    <dd>CADD: 0.175 (0.835)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC SAS: 0.0159%</dd>
                                    <dd>gnomAD_E_AFR: 0.0100%</dd>
                                    <dd>gnomAD_E_SAS: 0.0083%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000091409">ITGA6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.361</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.403 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79403">Junctional epidermolysis bullosa-pyloric atresia syndrome</a> associated with ITGA6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0011100, Intestinal atresia</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - HP:0002017, Nausea and vomiting</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ITGA6%0DCOL5A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COL5A1</a> and phenotypic similarity 0.646 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=287">Classical Ehlers-Danlos syndrome</a> associated with COL5A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002036, Hiatus hernia</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0003326, Myalgia</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/226730">OMIM:226730</a> Epidermolysis bullosa, junctional, with pyloric stenosis - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79403">ORPHA:79403</a> Junctional epidermolysis bullosa-pyloric atresia syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.009</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.361</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.172485154A&gt;G [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=16860530">rs16860530</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs16860530&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=285274%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.361</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264107.11">ITGA6:ENST00000264107.11:c.1744A&gt;G:p.(Ile582Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409080.6">ITGA6:ENST00000409080.6:c.1744A&gt;G:p.(Ile582Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409532.5">ITGA6:ENST00000409532.5:c.1387A&gt;G:p.(Ile463Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000442250.5">ITGA6:ENST00000442250.5:c.1861A&gt;G:p.(Ile621Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000458358.5">ITGA6:ENST00000458358.5:c.1729A&gt;G:p.(Ile577Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999815</dd>
                                    
                                    <dd>Polyphen2: 0.128 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.562 (T)</dd>
                                    <dd>CADD: 0.988 (19.140)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.6190%</dd>
                                    <dd>TOPMed: 0.4436%</dd>
                                    <dd>ESP AA: 1.4526%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.4998%</dd>
                                    <dd>ExAC AFR: 1.5772%</dd>
                                    <dd>ExAC AMR: 0.1210%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>ExAC OTH: 0.2203%</dd>
                                    <dd>gnomAD_E_AFR: 1.7446%</dd>
                                    <dd>gnomAD_E_AMR: 0.1072%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_E_OTH: 0.0730%</dd>
                                    <dd>gnomAD_G_AFR: 1.4551%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000111249">CUX2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.644</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUX2%0DBCL6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BCL6</a> and phenotypic similarity 0.633 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=545">Follicular lymphoma</a> associated with BCL6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002665, Lymphoma</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0030166, Night sweats</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUX2%0DBCL6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BCL6</a> and phenotypic similarity 0.298 to mouse mutant of BCL6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0000696, abnormal Peyer's patch morphology</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0000696, abnormal Peyer's patch morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618141">OMIM:618141</a> Epileptic encephalopathy, early infantile, 67 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2382">ORPHA:2382</a> Lennox-Gastaut syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.008</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.644</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr12:g.111320436C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=539557491">rs539557491</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs539557491&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.778</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261726.11">CUX2:ENST00000261726.11:c.2427C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7798595</dd>
                                    
                                    <dd>CADD: 0.780 (6.573)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0134%</dd>
                                    <dd>gnomAD_E_AFR: 0.0075%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr12:g.111334451A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=567284017">rs567284017</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs567284017&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.510</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261726.11">CUX2:ENST00000261726.11:c.2937A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5114602</dd>
                                    
                                    <dd>CADD: 0.511 (3.111)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0106%</dd>
                                    <dd>gnomAD_E_AFR: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154358">OBSCN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.927</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.008</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.927</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                                <b>chr1:g.228288889G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=527810384">rs527810384</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs527810384&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000284548.16">OBSCN:ENST00000284548.16:c.10339+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366707.9">OBSCN:ENST00000366707.9:c.11626+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422127.5">OBSCN:ENST00000422127.5:c.10339+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000570156.6">OBSCN:ENST00000570156.6:c.11626+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000636476.2">OBSCN:ENST00000636476.2:c.10339+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000636875.1">OBSCN:ENST00000636875.1:c.10339+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000662438.1">OBSCN:ENST00000662438.1:c.10339+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.996 (23.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC AFR: 0.0102%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.228360368C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=376866724">rs376866724</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs376866724&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.856</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000284548.16">OBSCN:ENST00000284548.16:c.19476C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366707.9">OBSCN:ENST00000366707.9:c.21533-2208C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422127.5">OBSCN:ENST00000422127.5:c.18662-2208C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000570156.6">OBSCN:ENST00000570156.6:c.21533-2208C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000636476.2">OBSCN:ENST00000636476.2:c.18665-2208C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000636875.1">OBSCN:ENST00000636875.1:c.18662-2208C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000662438.1">OBSCN:ENST00000662438.1:c.20207-2208C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9148862</dd>
                                    
                                    <dd>CADD: 0.915 (10.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1597%</dd>
                                    <dd>TOPMed: 0.0980%</dd>
                                    <dd>ESP AA: 0.1403%</dd>
                                    <dd>ESP EA: 0.0118%</dd>
                                    <dd>ESP All: 0.0550%</dd>
                                    <dd>ExAC AFR: 0.3873%</dd>
                                    <dd>ExAC NFE: 0.0031%</dd>
                                    <dd>ExAC OTH: 0.1155%</dd>
                                    <dd>gnomAD_E_AFR: 0.3780%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0367%</dd>
                                    <dd>gnomAD_G_AFR: 0.3561%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.228370697G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=73099650">rs73099650</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs73099650&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.760</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366707.9">OBSCN:ENST00000366707.9:c.23056G&gt;A:p.(Glu7686Lys)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422127.5">OBSCN:ENST00000422127.5:c.20185G&gt;A:p.(Glu6729Lys)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000570156.6">OBSCN:ENST00000570156.6:c.23056G&gt;A:p.(Glu7686Lys)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000636476.2">OBSCN:ENST00000636476.2:c.20188G&gt;A:p.(Glu6730Lys)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000636875.1">OBSCN:ENST00000636875.1:c.20185G&gt;A:p.(Glu6729Lys)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000662438.1">OBSCN:ENST00000662438.1:c.21730G&gt;A:p.(Glu7244Lys)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9959262</dd>
                        
                        <dd>Polyphen2: 0.012 (B)</dd>
                        <dd>SIFT: 0.124 (T)</dd>
                        <dd>CADD: 0.996 (23.900)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.2596%</dd>
                        <dd>TOPMed: 0.2963%</dd>
                        <dd>ESP AA: 0.8136%</dd>
                        <dd>ESP All: 0.2660%</dd>
                        <dd>ExAC AFR: 0.8908%</dd>
                        <dd>ExAC AMR: 0.0173%</dd>
                        <dd>gnomAD_E_AFR: 0.8570%</dd>
                        <dd>gnomAD_E_AMR: 0.0298%</dd>
                        <dd>gnomAD_E_OTH: 0.0365%</dd>
                        <dd>gnomAD_G_AFR: 1.0108%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr1:g.228377097G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=56293188">rs56293188</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs56293188&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.001</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366707.9">OBSCN:ENST00000366707.9:c.25956G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422127.5">OBSCN:ENST00000422127.5:c.23085G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000570156.6">OBSCN:ENST00000570156.6:c.25956G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000636476.2">OBSCN:ENST00000636476.2:c.23088G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000636875.1">OBSCN:ENST00000636875.1:c.23085G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000662438.1">OBSCN:ENST00000662438.1:c.24630G&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.001840353</dd>
                        
                        <dd>CADD: 0.002 (0.008)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.2196%</dd>
                        <dd>TOPMed: 0.2843%</dd>
                        <dd>ESP AA: 0.7239%</dd>
                        <dd>ESP All: 0.2351%</dd>
                        <dd>ExAC AFR: 1.0747%</dd>
                        <dd>ExAC AMR: 0.0120%</dd>
                        <dd>gnomAD_E_AFR: 0.7588%</dd>
                        <dd>gnomAD_E_AMR: 0.0302%</dd>
                        <dd>gnomAD_E_OTH: 0.0384%</dd>
                        <dd>gnomAD_G_AFR: 0.9551%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000049759">NEDD4L</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.254</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.640</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.294 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98892">Periventricular nodular heterotopia</a> associated with NEDD4L.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002020, Gastroesophageal reflux</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - HP:0002020, Gastroesophageal reflux</dd></dl><dl><dt>Phenotypic similarity 0.348 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NEDD4L">NEDD4L</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0014206, decreased intestinal epithelial sodium ion transmembrane transport</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0014206, decreased intestinal epithelial sodium ion transmembrane transport</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NEDD4L%0DSCNN1B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SCNN1B</a> and phenotypic similarity 0.609 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=526">Liddle syndrome</a> associated with SCNN1B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002019, Constipation</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NEDD4L%0DSCNN1B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SCNN1B</a> and phenotypic similarity 0.417 to mouse mutant of SCNN1B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0001663, abnormal digestive system physiology</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0001663, abnormal digestive system physiology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617201">OMIM:617201</a> Periventricular nodular heterotopia 7 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98892">ORPHA:98892</a> Periventricular nodular heterotopia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.008</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.254</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.640</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr18:g.58256634T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=555335609">rs555335609</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs555335609&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.921</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000635997.1">NEDD4L:ENST00000635997.1:c.994T&gt;C:p.(Cys332Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000256830.13">NEDD4L:ENST00000256830.13:c.297+4580T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356462.10">NEDD4L:ENST00000356462.10:c.297+4580T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357895.9">NEDD4L:ENST00000357895.9:c.273+4580T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382850.8">NEDD4L:ENST00000382850.8:c.297+4580T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400345.8">NEDD4L:ENST00000400345.8:c.297+4580T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000586263.5">NEDD4L:ENST00000586263.5:c.273+4580T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589054.5">NEDD4L:ENST00000589054.5:c.49-134012T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431212.6">NEDD4L:ENST00000431212.6:c.-67+4580T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000435432.6">NEDD4L:ENST00000435432.6:c.-67+4580T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456173.6">NEDD4L:ENST00000456173.6:c.-67+4580T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456986.5">NEDD4L:ENST00000456986.5:c.-67+4580T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.994 (22.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2796%</dd>
                                    <dd>TOPMed: 0.1394%</dd>
                                    <dd>gnomAD_G_AFR: 0.4586%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr18:g.58256141C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=575995975">rs575995975</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs575995975&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.358</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000635997.1">NEDD4L:ENST00000635997.1:c.501C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000256830.13">NEDD4L:ENST00000256830.13:c.297+4087C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356462.10">NEDD4L:ENST00000356462.10:c.297+4087C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357895.9">NEDD4L:ENST00000357895.9:c.273+4087C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382850.8">NEDD4L:ENST00000382850.8:c.297+4087C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400345.8">NEDD4L:ENST00000400345.8:c.297+4087C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000586263.5">NEDD4L:ENST00000586263.5:c.273+4087C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589054.5">NEDD4L:ENST00000589054.5:c.49-134505C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431212.6">NEDD4L:ENST00000431212.6:c.-67+4087C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000435432.6">NEDD4L:ENST00000435432.6:c.-67+4087C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456173.6">NEDD4L:ENST00000456173.6:c.-67+4087C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456986.5">NEDD4L:ENST00000456986.5:c.-67+4087C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.40502328</dd>
                                    
                                    <dd>CADD: 0.405 (2.255)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1797%</dd>
                                    <dd>TOPMed: 0.2302%</dd>
                                    <dd>gnomAD_G_AFR: 0.6190%</dd>
                                    <dd>gnomAD_G_AMR: 0.1269%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211967">IGHV3-53</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.918</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.008</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.918</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.106592950T&gt;A [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2731154">rs2731154</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2731154&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.918</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390627.3">IGHV3-53:ENST00000390627.3:c.76A&gt;T:p.(Thr26Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9175862</dd>
                                    
                                    <dd>CADD: 0.918 (10.840)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr14:g.106593086G&gt;A [1/1:0/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2731152">rs2731152</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2731152&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.914</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390627.3">IGHV3-53:ENST00000390627.3:c.41C&gt;T:p.(Ser14Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9144933</dd>
                        
                        <dd>CADD: 0.914 (10.680)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000278263">AC135068.2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.910</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.007</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.910</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr15:g.21742639C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1169038365">rs1169038365</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1169038365&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.910</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622410.2">AC135068.2:ENST00000622410.2:c.78G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9104635</dd>
                                    
                                    <dd>CADD: 0.910 (10.480)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130844">ZNF331</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.006</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.327</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF331%0DTSR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TSR2</a> and phenotypic similarity 0.623 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=124">Blackfan-Diamond anemia</a> associated with TSR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002488, Acute leukemia</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0012378, Fatigue</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.006</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.327</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.53577454C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=10420618">rs10420618</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs10420618&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.637</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000253144.13">ZNF331:ENST00000253144.13:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000411977.6">ZNF331:ENST00000411977.6:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449416.5">ZNF331:ENST00000449416.5:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000504493.6">ZNF331:ENST00000504493.6:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000511154.5">ZNF331:ENST00000511154.5:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000511593.6">ZNF331:ENST00000511593.6:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512387.6">ZNF331:ENST00000512387.6:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513999.5">ZNF331:ENST00000513999.5:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000648122.1">ZNF331:ENST00000648122.1:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000648236.1">ZNF331:ENST00000648236.1:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000648397.1">ZNF331:ENST00000648397.1:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000648511.1">ZNF331:ENST00000648511.1:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000649326.1">ZNF331:ENST00000649326.1:c.894C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513265.5">ZNF331:ENST00000513265.5:c.*7C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8577344</dd>
                                    
                                    <dd>CADD: 0.858 (8.469)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.5591%</dd>
                                    <dd>TOPMed: 0.2684%</dd>
                                    <dd>ESP AA: 1.0667%</dd>
                                    <dd>ESP All: 0.3614%</dd>
                                    <dd>ExAC AFR: 0.9612%</dd>
                                    <dd>ExAC AMR: 0.0691%</dd>
                                    <dd>ExAC OTH: 0.1104%</dd>
                                    <dd>gnomAD_E_AFR: 0.9801%</dd>
                                    <dd>gnomAD_E_AMR: 0.0685%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_AFR: 0.9395%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.53577754C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=147705191">rs147705191</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs147705191&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.018</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000253144.13">ZNF331:ENST00000253144.13:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000411977.6">ZNF331:ENST00000411977.6:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449416.5">ZNF331:ENST00000449416.5:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000504493.6">ZNF331:ENST00000504493.6:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000511154.5">ZNF331:ENST00000511154.5:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000511593.6">ZNF331:ENST00000511593.6:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512387.6">ZNF331:ENST00000512387.6:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513999.5">ZNF331:ENST00000513999.5:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000648122.1">ZNF331:ENST00000648122.1:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000648236.1">ZNF331:ENST00000648236.1:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000648397.1">ZNF331:ENST00000648397.1:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000648511.1">ZNF331:ENST00000648511.1:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000649326.1">ZNF331:ENST00000649326.1:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.018252075</dd>
                                    
                                    <dd>CADD: 0.018 (0.080)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1198%</dd>
                                    <dd>TOPMed: 0.0502%</dd>
                                    <dd>ESP AA: 0.0908%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0384%</dd>
                                    <dd>ExAC AFR: 0.1830%</dd>
                                    <dd>ExAC AMR: 0.0260%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AFR: 0.1895%</dd>
                                    <dd>gnomAD_E_AMR: 0.0298%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_SAS: 0.0097%</dd>
                                    <dd>gnomAD_G_AFR: 0.1145%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154134">ROBO3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.006</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.321</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ROBO3%0DABL1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ABL1</a> and phenotypic similarity 0.630 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=521">Chronic myeloid leukemia</a> associated with ABL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0005547, Myeloproliferative disorder</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0004396, Poor appetite</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ROBO3%0DABL1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ABL1</a> and phenotypic similarity 0.366 to mouse mutant of ABL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0000493, rectal prolapse</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0004545, enlarged esophagus</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/607313">OMIM:607313</a> Gaze palsy, familial horizontal, with progressive scoliosis, 1 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2744">ORPHA:2744</a> Horizontal gaze palsy with progressive scoliosis</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.006</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.321</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_TRUNCATION</span>
                                <b>chr11:g.124880556GT&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=759310343">rs759310343</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs759310343&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=712682%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.488</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538940.5">ROBO3:ENST00000538940.5:c.4032del:p.(Ser1344Serfs*53)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543966.5">ROBO3:ENST00000543966.5:c.387del:p.(Ser129Serfs*53)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397801.6">ROBO3:ENST00000397801.6:c.4098del:p.(Ser1366Serfs*53)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99601895</dd>
                                    
                                    <dd>CADD: 0.996 (24.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 1.1209%</dd>
                                    <dd>gnomAD_E_AMR: 0.0368%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0513%</dd>
                                    <dd>gnomAD_G_AFR: 1.5647%</dd>
                                    <dd>gnomAD_G_OTH: 0.1029%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.124872411C&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=114572753">rs114572753</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs114572753&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=325202%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.153</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397801.6">ROBO3:ENST00000397801.6:c.1189C&gt;A:p.(Pro397Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538940.5">ROBO3:ENST00000538940.5:c.1123C&gt;A:p.(Pro375Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9952137</dd>
                                    
                                    <dd>Polyphen2: 0.464 (P)</dd>
                                    <dd>Mutation Taster: 0.781</dd>
                                    <dd>SIFT: 0.094 (T)</dd>
                                    <dd>CADD: 0.995 (23.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4992%</dd>
                                    <dd>TOPMed: 0.5997%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 1.7113%</dd>
                                    <dd>ESP All: 0.5569%</dd>
                                    <dd>ExAC AFR: 1.9388%</dd>
                                    <dd>ExAC AMR: 0.0777%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>ExAC OTH: 0.1114%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 1.9817%</dd>
                                    <dd>gnomAD_E_AMR: 0.0625%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_G_AFR: 1.7411%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000277758">FO681492.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.871</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.005</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.871</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.47757224C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1435481212">rs1435481212</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1435481212&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.948</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622861.4">FO681492.1:ENST00000622861.4:c.927G&gt;C:p.(Glu309Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000623662.3">FO681492.1:ENST00000623662.3:c.1086G&gt;C:p.(Glu362Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.94776034</dd>
                                    
                                    <dd>CADD: 0.948 (12.820)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.47762712C&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1368894589">rs1368894589</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1368894589&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.794</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000623662.3">FO681492.1:ENST00000623662.3:c.293G&gt;T:p.(Arg98Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622861.4">FO681492.1:ENST00000622861.4:c.-72G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7938896</dd>
                                    
                                    <dd>CADD: 0.794 (6.859)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198792">TMEM184B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.004</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.850</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr22:g.38272501G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=546014115">rs546014115</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs546014115&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361684.8">TMEM184B:ENST00000361684.8:c.-61C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361906.8">TMEM184B:ENST00000361906.8:c.-59+383C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8526331</dd>
                                    
                                    <dd>CADD: 0.853 (8.316)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000270775">AP000436.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.829</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.829</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr11:g.65025779C&gt;CTTTTTTTTTT [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=36027416">rs36027416</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs36027416&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.829</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000605239.1">AP000436.1:ENST00000605239.1:n.410_411insAAAAAAAAAA:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000301886.3">AP000436.1:ENST00000301886.3:c.340-1539_340-1538insTTTTTTTTTT:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8285648</dd>
                                    
                                    <dd>CADD: 0.829 (7.659)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000054654">SYNE2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.532</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.337 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98853">Autosomal dominant Emery-Dreifuss muscular dystrophy</a> associated with SYNE2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0003418, Back pain</dd><dd>HP:0002027, Abdominal pain - HP:0003418, Back pain</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SYNE2%0DNUMA1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NUMA1</a> and phenotypic similarity 0.668 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=520">Acute promyelocytic leukemia</a> associated with NUMA1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/612999">OMIM:612999</a> Emery-Dreifuss muscular dystrophy 5, autosomal dominant - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98853">ORPHA:98853</a> Autosomal dominant Emery-Dreifuss muscular dystrophy</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.532</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.64170260C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=147365925">rs147365925</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs147365925&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=336443%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.939</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344113.8">SYNE2:ENST00000344113.8:c.17033C&gt;T:p.(Thr5678Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357395.7">SYNE2:ENST00000357395.7:c.16676C&gt;T:p.(Thr5559Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358025.7">SYNE2:ENST00000358025.7:c.17033C&gt;T:p.(Thr5678Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000554584.5">SYNE2:ENST00000554584.5:c.16658C&gt;T:p.(Thr5553Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555002.5">SYNE2:ENST00000555002.5:c.6935C&gt;T:p.(Thr2312Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556275.5">SYNE2:ENST00000556275.5:c.1406+64710G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.997</dd>
                                    
                                    <dd>Polyphen2: 0.372 (B)</dd>
                                    <dd>SIFT: 0.003 (D)</dd>
                                    <dd>CADD: 0.991 (20.400)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1997%</dd>
                                    <dd>TOPMed: 0.1067%</dd>
                                    <dd>UK10K: 0.0397%</dd>
                                    <dd>ESP AA: 0.1589%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.0692%</dd>
                                    <dd>ExAC AFR: 0.2211%</dd>
                                    <dd>ExAC AMR: 0.1469%</dd>
                                    <dd>ExAC NFE: 0.0300%</dd>
                                    <dd>gnomAD_E_AFR: 0.2222%</dd>
                                    <dd>gnomAD_E_AMR: 0.1966%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0102%</dd>
                                    <dd>gnomAD_E_NFE: 0.0269%</dd>
                                    <dd>gnomAD_E_OTH: 0.1277%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 0.1488%</dd>
                                    <dd>gnomAD_G_AMR: 0.3580%</dd>
                                    <dd>gnomAD_G_NFE: 0.0466%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.64221617G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=75079588">rs75079588</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs75079588&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=464317%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.125</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344113.8">SYNE2:ENST00000344113.8:c.20034G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357395.7">SYNE2:ENST00000357395.7:c.19800G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358025.7">SYNE2:ENST00000358025.7:c.20103G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000554584.5">SYNE2:ENST00000554584.5:c.19782G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555002.5">SYNE2:ENST00000555002.5:c.10005G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000639659.1">SYNE2:ENST00000639659.1:c.669G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556275.5">SYNE2:ENST00000556275.5:c.1406+13353C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.90405995</dd>
                                    
                                    <dd>CADD: 0.904 (10.180)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.5990%</dd>
                                    <dd>TOPMed: 0.5575%</dd>
                                    <dd>ESP AA: 1.9973%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.6843%</dd>
                                    <dd>ExAC AFR: 1.8931%</dd>
                                    <dd>ExAC AMR: 0.0432%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>ExAC OTH: 0.2203%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 1.8688%</dd>
                                    <dd>gnomAD_E_AMR: 0.0655%</dd>
                                    <dd>gnomAD_E_EAS: 0.0232%</dd>
                                    <dd>gnomAD_E_OTH: 0.0911%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 1.6869%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211970">IGHV4-61</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.810</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.810</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.106639408G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=782031467">rs782031467</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs782031467&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.810</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390630.3">IGHV4-61:ENST00000390630.3:c.67C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.81024206</dd>
                                    
                                    <dd>CADD: 0.810 (7.218)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0032%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000277282">IGHV1OR21-1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.804</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.804</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr21:g.10649564C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=376900399">rs376900399</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs376900399&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.804</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622028.1">IGHV1OR21-1:ENST00000622028.1:c.189G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8055088</dd>
                                    
                                    <dd>CADD: 0.806 (7.111)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000239998">LILRA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.802</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.802</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54574809T&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=571423648">rs571423648</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs571423648&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.802</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251376.7">LILRA2:ENST00000251376.7:c.431T&gt;A:p.(Val144Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251377.7">LILRA2:ENST00000251377.7:c.431T&gt;A:p.(Val144Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391737.3">LILRA2:ENST00000391737.3:c.395T&gt;A:p.(Val132Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391738.8">LILRA2:ENST00000391738.8:c.431T&gt;A:p.(Val144Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000629481.1">LILRA2:ENST00000629481.1:c.314+265T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.802303</dd>
                                    
                                    <dd>Polyphen2: 0.005 (B)</dd>
                                    <dd>SIFT: 0.551 (T)</dd>
                                    <dd>CADD: 0.802 (7.040)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000275395">FCGBP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.790</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.116</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.39885865T&gt;C [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=777522803">rs777522803</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs777522803&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.116</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.8196A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.4791A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.11773306</dd>
                                    
                                    <dd>CADD: 0.118 (0.544)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0189%</dd>
                                    <dd>gnomAD_E_EAS: 0.0073%</dd>
                                    <dd>gnomAD_E_NFE: 0.0010%</dd>
                                    <dd>gnomAD_G_EAS: 0.0965%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.002</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.790</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.39902257T&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=587698183">rs587698183</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs587698183&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.935</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.4571A&gt;C:p.(Asn1524Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.1166A&gt;C:p.(Asn389Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.94847715</dd>
                                    
                                    <dd>CADD: 0.948 (12.880)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0998%</dd>
                                    <dd>TOPMed: 0.0998%</dd>
                                    <dd>gnomAD_E_AFR: 0.0152%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0123%</dd>
                                    <dd>gnomAD_E_NFE: 0.0011%</dd>
                                    <dd>gnomAD_G_NFE: 0.0082%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.39902691A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=587614965">rs587614965</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs587614965&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.646</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.4137T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.732T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6868157</dd>
                                    
                                    <dd>CADD: 0.687 (5.042)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2396%</dd>
                                    <dd>TOPMed: 0.2396%</dd>
                                    <dd>gnomAD_E_AFR: 0.0184%</dd>
                                    <dd>gnomAD_E_AMR: 0.0428%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0368%</dd>
                                    <dd>gnomAD_E_EAS: 0.0275%</dd>
                                    <dd>gnomAD_E_FIN: 0.0223%</dd>
                                    <dd>gnomAD_E_NFE: 0.0384%</dd>
                                    <dd>gnomAD_E_OTH: 0.0218%</dd>
                                    <dd>gnomAD_E_SAS: 0.0872%</dd>
                                    <dd>gnomAD_G_AFR: 0.0238%</dd>
                                    <dd>gnomAD_G_AMR: 0.1326%</dd>
                                    <dd>gnomAD_G_ASJ: 0.3650%</dd>
                                    <dd>gnomAD_G_FIN: 0.0623%</dd>
                                    <dd>gnomAD_G_NFE: 0.1908%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.39886005T&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2916067">rs2916067</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2916067&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.899</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.8174A&gt;C:p.(Asn2725Thr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.4769A&gt;C:p.(Asn1590Thr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.94567496</dd>
                        
                        <dd>CADD: 0.946 (12.650)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_E_AFR: 0.0477%</dd>
                        <dd>gnomAD_E_AMR: 0.0061%</dd>
                        <dd>gnomAD_G_AFR: 0.3120%</dd>
                        <dd>gnomAD_G_OTH: 0.1073%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr19:g.39902409G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=758520702">rs758520702</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs758520702&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.891</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.4419C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.1014C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.89292204</dd>
                        
                        <dd>CADD: 0.893 (9.703)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_E_AMR: 0.0089%</dd>
                        <dd>gnomAD_E_EAS: 0.0174%</dd>
                        <dd>gnomAD_E_NFE: 0.0009%</dd>
                        <dd>gnomAD_E_SAS: 0.0163%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.39885857T&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200588294">rs200588294</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200588294&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.721</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.8204A&gt;C:p.(Gln2735Pro)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.4799A&gt;C:p.(Gln1600Pro)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.7230509</dd>
                        
                        <dd>CADD: 0.723 (5.576)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0200%</dd>
                        <dd>TOPMed: 0.0200%</dd>
                        <dd>gnomAD_E_AFR: 0.0164%</dd>
                        <dd>gnomAD_E_EAS: 0.0073%</dd>
                        <dd>gnomAD_E_NFE: 0.0021%</dd>
                        <dd>gnomAD_E_SAS: 0.0141%</dd>
                        <dd>gnomAD_G_AFR: 0.0127%</dd>
                        <dd>gnomAD_G_NFE: 0.0072%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr19:g.39892073A&gt;G [1/1:0/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=757623546">rs757623546</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs757623546&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.564</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.6570T&gt;C:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.3165T&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.942456</dd>
                        
                        <dd>CADD: 0.942 (12.400)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_E_AFR: 0.0352%</dd>
                        <dd>gnomAD_E_AMR: 0.3742%</dd>
                        <dd>gnomAD_E_ASJ: 0.0134%</dd>
                        <dd>gnomAD_E_EAS: 0.0948%</dd>
                        <dd>gnomAD_E_FIN: 0.0057%</dd>
                        <dd>gnomAD_E_NFE: 0.0347%</dd>
                        <dd>gnomAD_E_OTH: 0.3897%</dd>
                        <dd>gnomAD_E_SAS: 1.3776%</dd>
                        <dd>gnomAD_G_NFE: 0.0095%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr19:g.39885790C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1292472919">rs1292472919</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1292472919&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.124</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.8271G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.4866G&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.123806536</dd>
                        
                        <dd>CADD: 0.124 (0.574)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000184313">MROH7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.778</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.002</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.778</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr1:g.54673740C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200552161">rs200552161</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200552161&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.974</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339553.9">MROH7:ENST00000339553.9:c.1735C&gt;T:p.(Arg579*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395690.6">MROH7:ENST00000395690.6:c.1735C&gt;T:p.(Arg579*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409996.5">MROH7:ENST00000409996.5:c.439C&gt;T:p.(Arg147*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421030.7">MROH7:ENST00000421030.7:c.1735C&gt;T:p.(Arg579*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000414150.6">MROH7:ENST00000414150.6:c.1735C&gt;T:p.(Arg579*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 1.000 (39.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0534%</dd>
                                    <dd>ESP AA: 0.0515%</dd>
                                    <dd>ESP EA: 0.0121%</dd>
                                    <dd>ESP All: 0.0247%</dd>
                                    <dd>ExAC AFR: 0.1734%</dd>
                                    <dd>ExAC AMR: 0.0518%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>ExAC SAS: 0.0242%</dd>
                                    <dd>gnomAD_E_AFR: 0.1504%</dd>
                                    <dd>gnomAD_E_AMR: 0.0357%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_FIN: 0.0090%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0162%</dd>
                                    <dd>gnomAD_G_AFR: 0.1032%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.54709991A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=149350120">rs149350120</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs149350120&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.582</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409996.5">MROH7:ENST00000409996.5:c.2480A&gt;G:p.(Tyr827Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421030.7">MROH7:ENST00000421030.7:c.3776A&gt;G:p.(Tyr1259Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000414150.6">MROH7:ENST00000414150.6:c.3730+915A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.952137</dd>
                                    
                                    <dd>Polyphen2: 0.006 (B)</dd>
                                    <dd>SIFT: 0.257 (T)</dd>
                                    <dd>CADD: 0.952 (13.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.5192%</dd>
                                    <dd>TOPMed: 0.3831%</dd>
                                    <dd>ESP AA: 1.1430%</dd>
                                    <dd>ESP EA: 0.0357%</dd>
                                    <dd>ESP All: 0.3998%</dd>
                                    <dd>ExAC AFR: 1.2566%</dd>
                                    <dd>ExAC AMR: 0.1211%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AFR: 1.3542%</dd>
                                    <dd>gnomAD_E_AMR: 0.0923%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_AFR: 1.3545%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000118197">DDX59</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.212</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.242 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2919">Orofaciodigital syndrome type 5</a> associated with DDX59.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002251, Aganglionic megacolon</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DDX59%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.668 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=520">Acute promyelocytic leukemia</a> associated with PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DDX59%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.436 to mouse mutant of PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0003331, increased hepatocellular carcinoma incidence</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/174300">OMIM:174300</a> Orofaciodigital syndrome V - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2919">ORPHA:2919</a> Orofaciodigital syndrome type 5</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.002</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.212</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.200663925A&gt;G [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=145473014">rs145473014</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs145473014&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=718506%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.212</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000331314.11">DDX59:ENST00000331314.11:c.966T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000447706.6">DDX59:ENST00000447706.6:c.966T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.27872646</dd>
                                    
                                    <dd>CADD: 0.279 (1.419)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3395%</dd>
                                    <dd>TOPMed: 0.2947%</dd>
                                    <dd>ESP AA: 0.7717%</dd>
                                    <dd>ESP All: 0.2614%</dd>
                                    <dd>ExAC AFR: 1.0216%</dd>
                                    <dd>ExAC AMR: 0.0607%</dd>
                                    <dd>ExAC FIN: 0.0151%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>ExAC SAS: 0.0068%</dd>
                                    <dd>gnomAD_E_AFR: 0.8808%</dd>
                                    <dd>gnomAD_E_AMR: 0.0660%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0187%</dd>
                                    <dd>gnomAD_E_SAS: 0.0034%</dd>
                                    <dd>gnomAD_G_AFR: 0.8708%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000124831">LRRFIP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.193</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRRFIP1%0DTLR3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TLR3</a> and phenotypic similarity 0.629 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1930">Herpes simplex virus encephalitis</a> associated with TLR3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0002017, Nausea and vomiting</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRRFIP1%0DTLR3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TLR3</a> and phenotypic similarity 0.247 to mouse mutant of TLR3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0002020, increased tumor incidence</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.002</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.193</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.237769984T&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=148443859">rs148443859</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs148443859&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.377</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308482.14">LRRFIP1:ENST00000308482.14:c.1501T&gt;A:p.(Leu501Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.957</dd>
                                    
                                    <dd>SIFT: 0.043 (D)</dd>
                                    <dd>CADD: 0.941 (12.290)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3794%</dd>
                                    <dd>TOPMed: 0.3369%</dd>
                                    <dd>ESP AA: 1.1480%</dd>
                                    <dd>ESP EA: 0.0140%</dd>
                                    <dd>ESP All: 0.3592%</dd>
                                    <dd>ExAC AFR: 1.7010%</dd>
                                    <dd>ExAC AMR: 0.0505%</dd>
                                    <dd>ExAC NFE: 0.0033%</dd>
                                    <dd>gnomAD_E_AFR: 1.2126%</dd>
                                    <dd>gnomAD_E_AMR: 0.0188%</dd>
                                    <dd>gnomAD_E_NFE: 0.0039%</dd>
                                    <dd>gnomAD_E_OTH: 0.0390%</dd>
                                    <dd>gnomAD_E_SAS: 0.0035%</dd>
                                    <dd>gnomAD_G_AFR: 1.1231%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.237763585C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=146819325">rs146819325</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs146819325&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.010</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000244815.9">LRRFIP1:ENST00000244815.9:c.1800C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000289175.10">LRRFIP1:ENST00000289175.10:c.1704C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392000.4">LRRFIP1:ENST00000392000.4:c.1872C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308482.14">LRRFIP1:ENST00000308482.14:c.1459+3380C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.011902034</dd>
                                    
                                    <dd>CADD: 0.012 (0.052)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3195%</dd>
                                    <dd>TOPMed: 0.2596%</dd>
                                    <dd>ESP AA: 0.6582%</dd>
                                    <dd>ESP EA: 0.0582%</dd>
                                    <dd>ESP All: 0.2615%</dd>
                                    <dd>ExAC AFR: 0.7539%</dd>
                                    <dd>ExAC AMR: 0.2448%</dd>
                                    <dd>ExAC NFE: 0.0801%</dd>
                                    <dd>ExAC OTH: 0.3371%</dd>
                                    <dd>gnomAD_E_AFR: 0.6292%</dd>
                                    <dd>gnomAD_E_AMR: 0.3370%</dd>
                                    <dd>gnomAD_E_ASJ: 0.2238%</dd>
                                    <dd>gnomAD_E_NFE: 0.0707%</dd>
                                    <dd>gnomAD_E_OTH: 0.3671%</dd>
                                    <dd>gnomAD_E_SAS: 0.0098%</dd>
                                    <dd>gnomAD_G_AFR: 0.5617%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_NFE: 0.0466%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185567">AHNAK2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.759</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.002</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.759</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.104952746G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200145279">rs200145279</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200145279&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.902</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333244.6">AHNAK2:ENST00000333244.6:c.2705C&gt;T:p.(Pro902Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557457.1">AHNAK2:ENST00000557457.1:c.-221+1135C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9175862</dd>
                                    
                                    <dd>Polyphen2: 0.058 (B)</dd>
                                    <dd>SIFT: 0.369 (T)</dd>
                                    <dd>CADD: 0.918 (10.840)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0599%</dd>
                                    <dd>ExAC AFR: 0.0313%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC FIN: 0.0302%</dd>
                                    <dd>ExAC NFE: 0.0362%</dd>
                                    <dd>ExAC SAS: 0.0242%</dd>
                                    <dd>gnomAD_E_AFR: 0.0200%</dd>
                                    <dd>gnomAD_E_AMR: 0.0269%</dd>
                                    <dd>gnomAD_E_EAS: 0.0119%</dd>
                                    <dd>gnomAD_E_FIN: 0.0628%</dd>
                                    <dd>gnomAD_E_NFE: 0.0369%</dd>
                                    <dd>gnomAD_E_OTH: 0.0550%</dd>
                                    <dd>gnomAD_E_SAS: 0.0131%</dd>
                                    <dd>gnomAD_G_AFR: 0.0467%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_FIN: 0.0860%</dd>
                                    <dd>gnomAD_G_NFE: 0.0335%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.104947648T&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=199812824">rs199812824</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs199812824&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.616</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333244.6">AHNAK2:ENST00000333244.6:c.7803A&gt;C:p.(Glu2601Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557457.1">AHNAK2:ENST00000557457.1:c.-221+6233A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.73115623</dd>
                                    
                                    <dd>Polyphen2: 0.016 (B)</dd>
                                    <dd>SIFT: 0.668 (T)</dd>
                                    <dd>CADD: 0.731 (5.705)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4193%</dd>
                                    <dd>TOPMed: 0.4193%</dd>
                                    <dd>ESP AA: 0.6937%</dd>
                                    <dd>ESP All: 0.2179%</dd>
                                    <dd>gnomAD_E_AFR: 0.7740%</dd>
                                    <dd>gnomAD_E_AMR: 0.0685%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_AFR: 0.7108%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr14:g.104942477G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=143133046">rs143133046</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs143133046&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.595</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333244.6">AHNAK2:ENST00000333244.6:c.12974C&gt;T:p.(Ala4325Val)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557457.1">AHNAK2:ENST00000557457.1:c.-220-1499C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.724</dd>
                        
                        <dd>Polyphen2: 0.329 (B)</dd>
                        <dd>SIFT: 0.276 (T)</dd>
                        <dd>CADD: 0.446 (2.568)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.3794%</dd>
                        <dd>TOPMed: 0.2501%</dd>
                        <dd>ESP AA: 0.7136%</dd>
                        <dd>ESP EA: 0.0121%</dd>
                        <dd>ESP All: 0.2377%</dd>
                        <dd>ExAC AFR: 0.8388%</dd>
                        <dd>ExAC AMR: 0.0518%</dd>
                        <dd>ExAC NFE: 0.0060%</dd>
                        <dd>ExAC SAS: 0.0121%</dd>
                        <dd>gnomAD_E_AFR: 0.7205%</dd>
                        <dd>gnomAD_E_AMR: 0.0715%</dd>
                        <dd>gnomAD_E_NFE: 0.0045%</dd>
                        <dd>gnomAD_E_OTH: 0.0365%</dd>
                        <dd>gnomAD_E_SAS: 0.0032%</dd>
                        <dd>gnomAD_G_AFR: 0.7678%</dd>
                        <dd>gnomAD_G_NFE: 0.0067%</dd>
                        <dd>gnomAD_G_OTH: 0.1022%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr14:g.104938197C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=80245452">rs80245452</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs80245452&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.509</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333244.6">AHNAK2:ENST00000333244.6:c.17254G&gt;A:p.(Gly5752Arg)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557457.1">AHNAK2:ENST00000557457.1:c.2248G&gt;A:p.(Gly750Arg)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.988</dd>
                        
                        <dd>Polyphen2: 0.295 (B)</dd>
                        <dd>SIFT: 0.012 (D)</dd>
                        <dd>CADD: 0.985 (18.270)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.4792%</dd>
                        <dd>TOPMed: 0.4619%</dd>
                        <dd>ESP AA: 0.9386%</dd>
                        <dd>ESP All: 0.3024%</dd>
                        <dd>ExAC AFR: 1.5217%</dd>
                        <dd>ExAC AMR: 0.0694%</dd>
                        <dd>ExAC EAS: 0.0349%</dd>
                        <dd>ExAC OTH: 0.1119%</dd>
                        <dd>gnomAD_E_AFR: 1.5140%</dd>
                        <dd>gnomAD_E_AMR: 0.0864%</dd>
                        <dd>gnomAD_E_EAS: 0.0290%</dd>
                        <dd>gnomAD_E_NFE: 0.0009%</dd>
                        <dd>gnomAD_E_OTH: 0.0365%</dd>
                        <dd>gnomAD_G_AFR: 1.4787%</dd>
                        <dd>gnomAD_G_AMR: 0.1193%</dd>
                        <dd>gnomAD_G_OTH: 0.1018%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr14:g.104942004C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=114164529">rs114164529</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs114164529&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.190</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333244.6">AHNAK2:ENST00000333244.6:c.13447G&gt;A:p.(Glu4483Lys)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557457.1">AHNAK2:ENST00000557457.1:c.-220-1026G&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.99</dd>
                        
                        <dd>Polyphen2: 0.990 (D)</dd>
                        <dd>SIFT: 0.031 (D)</dd>
                        <dd>CADD: 0.969 (15.070)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.5591%</dd>
                        <dd>TOPMed: 0.5591%</dd>
                        <dd>ESP AA: 1.2568%</dd>
                        <dd>ESP All: 0.4105%</dd>
                        <dd>ExAC AFR: 1.8447%</dd>
                        <dd>ExAC AMR: 0.0518%</dd>
                        <dd>ExAC EAS: 0.0116%</dd>
                        <dd>ExAC NFE: 0.0015%</dd>
                        <dd>ExAC OTH: 0.1114%</dd>
                        <dd>gnomAD_E_AFR: 1.9419%</dd>
                        <dd>gnomAD_E_AMR: 0.0923%</dd>
                        <dd>gnomAD_E_EAS: 0.0058%</dd>
                        <dd>gnomAD_E_NFE: 0.0009%</dd>
                        <dd>gnomAD_E_OTH: 0.0365%</dd>
                        <dd>gnomAD_G_AFR: 1.9350%</dd>
                        <dd>gnomAD_G_AMR: 0.2387%</dd>
                        <dd>gnomAD_G_EAS: 0.0618%</dd>
                        <dd>gnomAD_G_OTH: 0.1020%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr14:g.104944432A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=191005153">rs191005153</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs191005153&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.034</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333244.6">AHNAK2:ENST00000333244.6:c.11019T&gt;C:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557457.1">AHNAK2:ENST00000557457.1:c.-220-3454T&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.04126185</dd>
                        
                        <dd>CADD: 0.041 (0.183)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.3794%</dd>
                        <dd>TOPMed: 0.3794%</dd>
                        <dd>ESP AA: 0.6483%</dd>
                        <dd>ESP All: 0.2062%</dd>
                        <dd>ExAC AFR: 0.8584%</dd>
                        <dd>ExAC AMR: 0.0605%</dd>
                        <dd>ExAC NFE: 0.0030%</dd>
                        <dd>ExAC SAS: 0.0061%</dd>
                        <dd>gnomAD_E_AFR: 0.7270%</dd>
                        <dd>gnomAD_E_AMR: 0.0775%</dd>
                        <dd>gnomAD_E_EAS: 0.0058%</dd>
                        <dd>gnomAD_E_NFE: 0.0036%</dd>
                        <dd>gnomAD_E_OTH: 0.0365%</dd>
                        <dd>gnomAD_G_AFR: 0.8269%</dd>
                        <dd>gnomAD_G_AMR: 0.1232%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr14:g.104942932C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=202032988">rs202032988</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs202032988&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.003</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333244.6">AHNAK2:ENST00000333244.6:c.12519G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557457.1">AHNAK2:ENST00000557457.1:c.-220-1954G&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.005510986</dd>
                        
                        <dd>CADD: 0.006 (0.024)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.5990%</dd>
                        <dd>TOPMed: 0.4731%</dd>
                        <dd>UK10K: 0.0132%</dd>
                        <dd>ESP AA: 0.9893%</dd>
                        <dd>ESP All: 0.3189%</dd>
                        <dd>ExAC AFR: 1.5558%</dd>
                        <dd>ExAC AMR: 0.0259%</dd>
                        <dd>ExAC OTH: 0.2227%</dd>
                        <dd>ExAC SAS: 0.3149%</dd>
                        <dd>gnomAD_E_AFR: 1.5443%</dd>
                        <dd>gnomAD_E_AMR: 0.0715%</dd>
                        <dd>gnomAD_E_OTH: 0.0548%</dd>
                        <dd>gnomAD_E_SAS: 0.3216%</dd>
                        <dd>gnomAD_G_AFR: 1.5614%</dd>
                        <dd>gnomAD_G_AMR: 0.1193%</dd>
                        <dd>gnomAD_G_OTH: 0.1020%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000243716">NPIPB5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.732</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.732</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr16:g.22534471C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=755632125">rs755632125</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs755632125&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    <span class="label label-info">LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=390103%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.732</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424340.5">NPIPB5:ENST00000424340.5:c.1488C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000451409.5">NPIPB5:ENST00000451409.5:c.1488C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000517539.5">NPIPB5:ENST00000517539.5:c.1488C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357370.6">NPIPB5:ENST00000357370.6:c.817+23C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.74123824</dd>
                                    
                                    <dd>CADD: 0.741 (5.871)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_G_AFR: 0.0342%</dd>
                                    <dd>gnomAD_G_EAS: 0.0917%</dd>
                                    <dd>gnomAD_G_FIN: 0.0386%</dd>
                                    <dd>gnomAD_G_NFE: 0.0221%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000172969">FRG2C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.727</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.727</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.75665551A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=62247157">rs62247157</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs62247157&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.727</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308062.7">FRG2C:ENST00000308062.7:c.359A&gt;G:p.(Asn120Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000464571.1">FRG2C:ENST00000464571.1:c.356A&gt;G:p.(Asn119Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000638439.1">FRG2C:ENST00000638439.1:n.137+4469T&gt;C:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.72735345</dd>
                                    
                                    <dd>CADD: 0.727 (5.644)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000057593">F7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.143</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.231 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=327">Congenital factor VII deficiency</a> associated with F7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002239, Gastrointestinal hemorrhage</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - HP:0002239, Gastrointestinal hemorrhage</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=F7%0DBRCA2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BRCA2</a> and phenotypic similarity 0.630 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=654">Nephroblastoma</a> associated with BRCA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002896, Neoplasm of the liver</dd><dd>HP:0012378, Fatigue - HP:0002027, Abdominal pain</dd><dd>HP:0002027, Abdominal pain - HP:0002027, Abdominal pain</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=F7%0DBRCA2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BRCA2</a> and phenotypic similarity 0.388 to mouse mutant of BRCA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0002038, increased carcinoma incidence</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/227500">OMIM:227500</a> Factor VII deficiency - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=327">ORPHA:327</a> Congenital factor VII deficiency</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.143</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr13:g.113117603A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=519650">rs519650</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs519650&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=390088%5Balleleid%5D">(criteria_provided,_conflicting_interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.259</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346342.7">F7:ENST00000346342.7:c.739+7A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375581.3">F7:ENST00000375581.3:c.805+7A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541084.5">F7:ENST00000541084.5:c.553+7A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.74771</dd>
                                    
                                    <dd>CADD: 0.748 (5.981)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4992%</dd>
                                    <dd>TOPMed: 0.4992%</dd>
                                    <dd>UK10K: 1.4943%</dd>
                                    <dd>ESP AA: 0.3858%</dd>
                                    <dd>ESP EA: 1.3837%</dd>
                                    <dd>ESP All: 1.0457%</dd>
                                    <dd>ExAC AFR: 0.3874%</dd>
                                    <dd>ExAC AMR: 0.7274%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC FIN: 1.2649%</dd>
                                    <dd>ExAC NFE: 1.7915%</dd>
                                    <dd>ExAC OTH: 1.7738%</dd>
                                    <dd>ExAC SAS: 0.2271%</dd>
                                    <dd>gnomAD_E_AFR: 0.0789%</dd>
                                    <dd>gnomAD_E_AMR: 0.3953%</dd>
                                    <dd>gnomAD_E_ASJ: 0.9446%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_FIN: 0.2814%</dd>
                                    <dd>gnomAD_E_NFE: 0.5311%</dd>
                                    <dd>gnomAD_E_OTH: 0.4226%</dd>
                                    <dd>gnomAD_E_SAS: 0.1010%</dd>
                                    <dd>gnomAD_G_AFR: 0.0833%</dd>
                                    <dd>gnomAD_G_AMR: 0.3778%</dd>
                                    <dd>gnomAD_G_ASJ: 1.0204%</dd>
                                    <dd>gnomAD_G_FIN: 0.3335%</dd>
                                    <dd>gnomAD_G_NFE: 0.3362%</dd>
                                    <dd>gnomAD_G_OTH: 0.3268%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr13:g.113118759G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=145994193">rs145994193</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs145994193&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.028</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346342.7">F7:ENST00000346342.7:c.1086G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375581.3">F7:ENST00000375581.3:c.1152G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541084.5">F7:ENST00000541084.5:c.900G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.02837205</dd>
                                    
                                    <dd>CADD: 0.028 (0.125)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0111%</dd>
                                    <dd>ESP AA: 0.1135%</dd>
                                    <dd>ESP All: 0.0384%</dd>
                                    <dd>ExAC AFR: 0.0698%</dd>
                                    <dd>ExAC AMR: 0.0441%</dd>
                                    <dd>ExAC NFE: 0.0031%</dd>
                                    <dd>ExAC SAS: 0.0122%</dd>
                                    <dd>gnomAD_E_AFR: 0.0726%</dd>
                                    <dd>gnomAD_E_AMR: 0.0298%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000160767">FAM189B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.707</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.707</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.155248439C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=140120577">rs140120577</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs140120577&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.893</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000350210.6">FAM189B:ENST00000350210.6:c.1257G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361361.7">FAM189B:ENST00000361361.7:c.1545G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368368.7">FAM189B:ENST00000368368.7:c.1491G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621094.4">FAM189B:ENST00000621094.4:c.596-910G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9840412</dd>
                                    
                                    <dd>CADD: 0.984 (17.970)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1997%</dd>
                                    <dd>TOPMed: 0.1402%</dd>
                                    <dd>ESP AA: 0.4312%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.1615%</dd>
                                    <dd>ExAC AFR: 0.5195%</dd>
                                    <dd>ExAC AMR: 0.1038%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>gnomAD_E_AFR: 0.4639%</dd>
                                    <dd>gnomAD_E_AMR: 0.0774%</dd>
                                    <dd>gnomAD_E_NFE: 0.0081%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 0.4932%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.155254100A&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=141007586">rs141007586</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs141007586&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.521</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361361.7">FAM189B:ENST00000361361.7:c.377T&gt;A:p.(Phe126Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368368.7">FAM189B:ENST00000368368.7:c.320T&gt;A:p.(Phe107Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000350210.6">FAM189B:ENST00000350210.6:c.226-368T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621094.4">FAM189B:ENST00000621094.4:c.226-368T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.757 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.010 (D)</dd>
                                    <dd>CADD: 0.996 (24.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4593%</dd>
                                    <dd>TOPMed: 0.4579%</dd>
                                    <dd>ESP AA: 1.2710%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.4383%</dd>
                                    <dd>ExAC AFR: 1.4147%</dd>
                                    <dd>ExAC AMR: 0.1299%</dd>
                                    <dd>ExAC NFE: 0.0075%</dd>
                                    <dd>gnomAD_E_AFR: 1.4052%</dd>
                                    <dd>gnomAD_E_AMR: 0.1191%</dd>
                                    <dd>gnomAD_E_NFE: 0.0090%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_G_AFR: 1.5131%</dd>
                                    <dd>gnomAD_G_AMR: 0.3580%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000099814">CEP170B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.700</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.700</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.104886811G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201228019">rs201228019</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201228019&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.773</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000414716.8">CEP170B:ENST00000414716.8:c.2572G&gt;A:p.(Gly858Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453495.2">CEP170B:ENST00000453495.2:c.2575G&gt;A:p.(Gly859Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556508.5">CEP170B:ENST00000556508.5:c.2362G&gt;A:p.(Gly788Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8579962</dd>
                                    
                                    <dd>Polyphen2: 0.008 (B)</dd>
                                    <dd>SIFT: 0.520 (T)</dd>
                                    <dd>CADD: 0.858 (8.477)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2796%</dd>
                                    <dd>TOPMed: 0.1625%</dd>
                                    <dd>ESP AA: 0.3005%</dd>
                                    <dd>ESP All: 0.0976%</dd>
                                    <dd>ExAC AFR: 0.5482%</dd>
                                    <dd>ExAC AMR: 0.0262%</dd>
                                    <dd>gnomAD_E_AFR: 0.5479%</dd>
                                    <dd>gnomAD_E_AMR: 0.0477%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.5059%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.104889519G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=770807911">rs770807911</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs770807911&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.627</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453495.2">CEP170B:ENST00000453495.2:c.3747G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556508.5">CEP170B:ENST00000556508.5:c.3534G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000414716.8">CEP170B:ENST00000414716.8:c.3740-101G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.627351</dd>
                                    
                                    <dd>CADD: 0.627 (4.287)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0055%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr14:g.104884075C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201713998">rs201713998</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201713998&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.005</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000414716.8">CEP170B:ENST00000414716.8:c.1296C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453495.2">CEP170B:ENST00000453495.2:c.1299C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556508.5">CEP170B:ENST00000556508.5:c.1086C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.013720512</dd>
                        
                        <dd>CADD: 0.014 (0.060)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.5391%</dd>
                        <dd>TOPMed: 0.3472%</dd>
                        <dd>ESP AA: 1.0449%</dd>
                        <dd>ESP All: 0.3374%</dd>
                        <dd>ExAC AFR: 1.6968%</dd>
                        <dd>ExAC AMR: 0.1410%</dd>
                        <dd>gnomAD_E_AFR: 1.2330%</dd>
                        <dd>gnomAD_E_AMR: 0.0885%</dd>
                        <dd>gnomAD_E_NFE: 0.0021%</dd>
                        <dd>gnomAD_E_OTH: 0.0601%</dd>
                        <dd>gnomAD_G_AFR: 1.0930%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000107611">CUBN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.120</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUBN%0DINSR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to INSR</a> and phenotypic similarity 0.619 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=263458">Hyperinsulinism due to INSR deficiency</a> associated with INSR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0012378, Fatigue</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUBN%0DINSR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to INSR</a> and phenotypic similarity 0.312 to mouse mutant of INSR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0008537, increased susceptibility to induced colitis</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0008537, increased susceptibility to induced colitis</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/261100">OMIM:261100</a> Imerslund-Grasbeck syndrome 1 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.120</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.17043904C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=115048360">rs115048360</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs115048360&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=321523%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.153</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377833.10">CUBN:ENST00000377833.10:c.3752G&gt;C:p.(Ser1251Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98289984</dd>
                                    
                                    <dd>Polyphen2: 0.007 (B)</dd>
                                    <dd>SIFT: 0.276 (T)</dd>
                                    <dd>CADD: 0.983 (17.670)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.6390%</dd>
                                    <dd>TOPMed: 0.6005%</dd>
                                    <dd>ESP AA: 1.6341%</dd>
                                    <dd>ESP All: 0.5536%</dd>
                                    <dd>ExAC AFR: 1.9800%</dd>
                                    <dd>ExAC AMR: 0.0522%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 1.9610%</dd>
                                    <dd>gnomAD_E_AMR: 0.0746%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_G_AFR: 1.8667%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.3067%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.16835109C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=145872906">rs145872906</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs145872906&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=309849%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.087</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377833.10">CUBN:ENST00000377833.10:c.10267G&gt;A:p.(Val3423Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.11752987</dd>
                                    
                                    <dd>Polyphen2: 0.004 (B)</dd>
                                    <dd>SIFT: 1.000 (T)</dd>
                                    <dd>CADD: 0.118 (0.543)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2396%</dd>
                                    <dd>TOPMed: 0.2986%</dd>
                                    <dd>ESP AA: 0.9079%</dd>
                                    <dd>ESP All: 0.3076%</dd>
                                    <dd>ExAC AFR: 1.0671%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 1.0328%</dd>
                                    <dd>gnomAD_E_AMR: 0.0357%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_AFR: 0.9972%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197870">PRB3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.645</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.645</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_VARIANT</span>
                                <b>chr12:g.11267400A&gt;AG [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3842295">rs3842295</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs3842295&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381842.7">PRB3:ENST00000381842.7:c.722dup:p.(His242Serfs*14)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538488.2">PRB3:ENST00000538488.2:c.848dup:p.(His284Serfs*14)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9930817</dd>
                                    
                                    <dd>CADD: 0.993 (21.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.11267944T&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=879049259">rs879049259</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs879049259&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.290</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381842.7">PRB3:ENST00000381842.7:c.305A&gt;C:p.(Gln102Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538488.2">PRB3:ENST00000538488.2:c.305A&gt;C:p.(Gln102Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3403706</dd>
                                    
                                    <dd>CADD: 0.340 (1.807)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.4367%</dd>
                                    <dd>gnomAD_E_AMR: 0.0626%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0554%</dd>
                                    <dd>gnomAD_E_EAS: 0.1336%</dd>
                                    <dd>gnomAD_E_FIN: 0.0092%</dd>
                                    <dd>gnomAD_E_NFE: 0.0380%</dd>
                                    <dd>gnomAD_E_OTH: 0.1096%</dd>
                                    <dd>gnomAD_E_SAS: 0.0596%</dd>
                                    <dd>gnomAD_G_AFR: 0.1112%</dd>
                                    <dd>gnomAD_G_AMR: 0.7407%</dd>
                                    <dd>gnomAD_G_FIN: 0.4870%</dd>
                                    <dd>gnomAD_G_NFE: 0.0483%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000176809">LRRC37A3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.638</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.638</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.64894904C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1284941135">rs1284941135</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1284941135&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.638</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000319651.9">LRRC37A3:ENST00000319651.9:c.2354G&gt;A:p.(Arg785Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000584306.5">LRRC37A3:ENST00000584306.5:c.2354G&gt;A:p.(Arg785Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339474.9">LRRC37A3:ENST00000339474.9:c.117-2304G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400877.7">LRRC37A3:ENST00000400877.7:c.-277-2304G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.64199996</dd>
                                    
                                    <dd>Polyphen2: 0.006 (B)</dd>
                                    <dd>SIFT: 0.358 (T)</dd>
                                    <dd>CADD: 0.315 (1.644)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0361%</dd>
                                    <dd>gnomAD_G_AFR: 0.0437%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000263956">NBPF11</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.625</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.210</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.148106997T&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1169257878">rs1169257878</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1169257878&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.210</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.2196A&gt;C:p.(Arg732Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.2091A&gt;C:p.(Arg697Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.2196A&gt;C:p.(Arg732Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.20986736</dd>
                                    
                                    <dd>CADD: 0.210 (1.023)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.625</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.148107750T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1488492448">rs1488492448</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1488492448&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.965</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.2039A&gt;G:p.(Tyr680Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.1934A&gt;G:p.(Tyr645Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.2039A&gt;G:p.(Tyr680Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.965086</dd>
                                    
                                    <dd>CADD: 0.965 (14.570)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.148115890T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=28626270">rs28626270</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs28626270&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.285</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.1488A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.1383A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.1488A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.28533912</dd>
                                    
                                    <dd>CADD: 0.285 (1.459)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.148123948G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1209593044">rs1209593044</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1209593044&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.774</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.398C&gt;T:p.(Pro133Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.293C&gt;T:p.(Pro98Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.398C&gt;T:p.(Pro133Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.77395236</dd>
                        
                        <dd>CADD: 0.774 (6.458)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165626">BEND7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.580</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.580</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.13492727C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=150020216">rs150020216</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs150020216&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.953</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341083.7">BEND7:ENST00000341083.7:c.565G&gt;C:p.(Glu189Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378605.3">BEND7:ENST00000378605.3:c.604G&gt;C:p.(Glu202Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9908799</dd>
                                    
                                    <dd>Polyphen2: 0.844 (P)</dd>
                                    <dd>SIFT: 0.011 (D)</dd>
                                    <dd>CADD: 0.991 (20.400)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1198%</dd>
                                    <dd>TOPMed: 0.0406%</dd>
                                    <dd>ESP AA: 0.2043%</dd>
                                    <dd>ESP All: 0.0692%</dd>
                                    <dd>ExAC AFR: 0.2499%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>gnomAD_E_AFR: 0.1960%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_G_AFR: 0.1259%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr10:g.13439316G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=147487918">rs147487918</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs147487918&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.207</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341083.7">BEND7:ENST00000341083.7:c.1263C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.21367878</dd>
                                    
                                    <dd>CADD: 0.214 (1.044)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0589%</dd>
                                    <dd>ESP AA: 0.1589%</dd>
                                    <dd>ESP All: 0.0538%</dd>
                                    <dd>ExAC AFR: 0.1730%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.1830%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 0.2180%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000158486">DNAH3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.568</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.568</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.20963874G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=530510705">rs530510705</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs530510705&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261383.3">DNAH3:ENST00000261383.3:c.10010C&gt;T:p.(Thr3337Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.992</dd>
                                    
                                    <dd>Polyphen2: 0.247 (B)</dd>
                                    <dd>SIFT: 0.008 (D)</dd>
                                    <dd>CADD: 0.958 (13.740)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr16:g.20948528G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=536339888">rs536339888</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs536339888&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.147</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261383.3">DNAH3:ENST00000261383.3:c.11298C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.14748901</dd>
                                    
                                    <dd>CADD: 0.147 (0.693)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>ExAC AFR: 0.0199%</dd>
                                    <dd>ExAC EAS: 0.0118%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr16:g.21000501G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=144486513">rs144486513</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs144486513&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.803</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261383.3">DNAH3:ENST00000261383.3:c.6144C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.91469</dd>
                        
                        <dd>CADD: 0.915 (10.690)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.2396%</dd>
                        <dd>TOPMed: 0.1696%</dd>
                        <dd>ESP AA: 0.4998%</dd>
                        <dd>ESP All: 0.1692%</dd>
                        <dd>ExAC AFR: 0.6440%</dd>
                        <dd>ExAC AMR: 0.0261%</dd>
                        <dd>ExAC NFE: 0.0015%</dd>
                        <dd>gnomAD_E_AFR: 0.6215%</dd>
                        <dd>gnomAD_E_AMR: 0.0449%</dd>
                        <dd>gnomAD_E_NFE: 0.0028%</dd>
                        <dd>gnomAD_E_OTH: 0.0373%</dd>
                        <dd>gnomAD_G_AFR: 0.4360%</dd>
                        <dd>gnomAD_G_AMR: 0.1196%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000231475">IGHV4-31</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.558</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.558</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.106349375G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=61739267">rs61739267</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs61739267&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.860</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000438142.3">IGHV4-31:ENST00000438142.3:c.264C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8598478</dd>
                                    
                                    <dd>CADD: 0.860 (8.534)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.106349557A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=111274188">rs111274188</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs111274188&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.257</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000438142.3">IGHV4-31:ENST00000438142.3:c.82T&gt;C:p.(Ser28Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.25698084</dd>
                                    
                                    <dd>CADD: 0.257 (1.290)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr14:g.106349478G&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=117757529">rs117757529</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs117757529&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.011</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000438142.3">IGHV4-31:ENST00000438142.3:c.161C&gt;A:p.(Ser54Tyr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.010991573</dd>
                        
                        <dd>CADD: 0.011 (0.048)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000255837">TAS2R20</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.489</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.489</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr12:g.10997772C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=116400924">rs116400924</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs116400924&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=713325%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.638</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538986.1">TAS2R20:ENST00000538986.1:c.104G&gt;A:p.(Trp35*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536668.2">TAS2R20:ENST00000536668.2:c.110-24051G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539853.5">TAS2R20:ENST00000539853.5:c.-125-24051G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 1.000 (35.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.5391%</dd>
                                    <dd>TOPMed: 0.3775%</dd>
                                    <dd>ESP AA: 1.1121%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.3844%</dd>
                                    <dd>ExAC AFR: 1.3016%</dd>
                                    <dd>ExAC AMR: 0.0790%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>ExAC OTH: 0.1126%</dd>
                                    <dd>gnomAD_E_AFR: 1.2393%</dd>
                                    <dd>gnomAD_E_AMR: 0.0657%</dd>
                                    <dd>gnomAD_E_NFE: 0.0099%</dd>
                                    <dd>gnomAD_E_OTH: 0.0368%</dd>
                                    <dd>gnomAD_G_AFR: 0.9732%</dd>
                                    <dd>gnomAD_G_NFE: 0.0267%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.10997360C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=113758584">rs113758584</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs113758584&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.341</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538986.1">TAS2R20:ENST00000538986.1:c.516G&gt;A:p.(Met172Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536668.2">TAS2R20:ENST00000536668.2:c.110-23639G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539853.5">TAS2R20:ENST00000539853.5:c.-125-23639G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.39499998</dd>
                                    
                                    <dd>Polyphen2: 0.006 (B)</dd>
                                    <dd>SIFT: 0.605 (T)</dd>
                                    <dd>CADD: 0.127 (0.591)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.2357%</dd>
                                    <dd>ESP AA: 0.5674%</dd>
                                    <dd>ESP All: 0.1922%</dd>
                                    <dd>ExAC AFR: 0.6246%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>gnomAD_E_AFR: 0.6993%</dd>
                                    <dd>gnomAD_E_AMR: 0.0238%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_AFR: 0.5384%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000274576">IGHV2-70</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.472</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.472</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.106770689T&gt;C [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=367734033">rs367734033</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs367734033&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.472</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617374.2">IGHV2-70:ENST00000617374.2:c.246A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.47361922</dd>
                                    
                                    <dd>CADD: 0.474 (2.787)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ESP EA: 0.0239%</dd>
                                    <dd>ESP All: 0.0160%</dd>
                                    <dd>ExAC AFR: 0.0204%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC NFE: 0.0120%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0064%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000121904">CSMD2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.451</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.451</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.33625096G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=140414813">rs140414813</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs140414813&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.662</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373380.5">CSMD2:ENST00000373380.5:c.2074C&gt;T:p.(Pro692Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373381.8">CSMD2:ENST00000373381.8:c.5455C&gt;T:p.(Pro1819Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373388.6">CSMD2:ENST00000373388.6:c.5335C&gt;T:p.(Pro1779Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000619121.4">CSMD2:ENST00000619121.4:c.5335C&gt;T:p.(Pro1779Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373377.1">CSMD2:ENST00000373377.1:c.-249C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.997 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.007 (D)</dd>
                                    <dd>CADD: 0.997 (25.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4792%</dd>
                                    <dd>TOPMed: 0.3775%</dd>
                                    <dd>ESP AA: 1.0440%</dd>
                                    <dd>ESP All: 0.3537%</dd>
                                    <dd>ExAC AFR: 1.1109%</dd>
                                    <dd>ExAC AMR: 0.0865%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 1.2426%</dd>
                                    <dd>gnomAD_E_AMR: 0.0655%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 1.2517%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.33636520G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=34507149">rs34507149</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs34507149&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.240</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373380.5">CSMD2:ENST00000373380.5:c.1428C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373381.8">CSMD2:ENST00000373381.8:c.4809C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373388.6">CSMD2:ENST00000373388.6:c.4689C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000619121.4">CSMD2:ENST00000619121.4:c.4689C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7976049</dd>
                                    
                                    <dd>CADD: 0.798 (6.938)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.5791%</dd>
                                    <dd>TOPMed: 0.5670%</dd>
                                    <dd>ESP AA: 1.6341%</dd>
                                    <dd>ESP All: 0.5536%</dd>
                                    <dd>ExAC AFR: 1.8187%</dd>
                                    <dd>ExAC AMR: 0.0692%</dd>
                                    <dd>ExAC OTH: 0.1109%</dd>
                                    <dd>gnomAD_E_AFR: 1.8102%</dd>
                                    <dd>gnomAD_E_AMR: 0.0625%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_G_AFR: 1.7313%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr1:g.33636454G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=141475511">rs141475511</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs141475511&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.056</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373380.5">CSMD2:ENST00000373380.5:c.1494C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373381.8">CSMD2:ENST00000373381.8:c.4875C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373388.6">CSMD2:ENST00000373388.6:c.4755C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000619121.4">CSMD2:ENST00000619121.4:c.4755C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.18959892</dd>
                        
                        <dd>CADD: 0.190 (0.913)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.6589%</dd>
                        <dd>TOPMed: 0.5710%</dd>
                        <dd>ESP AA: 1.5887%</dd>
                        <dd>ESP All: 0.5382%</dd>
                        <dd>ExAC AFR: 1.6496%</dd>
                        <dd>ExAC AMR: 0.0953%</dd>
                        <dd>ExAC EAS: 0.0116%</dd>
                        <dd>ExAC NFE: 0.0030%</dd>
                        <dd>gnomAD_E_AFR: 1.7775%</dd>
                        <dd>gnomAD_E_AMR: 0.0923%</dd>
                        <dd>gnomAD_E_EAS: 0.0116%</dd>
                        <dd>gnomAD_E_NFE: 0.0045%</dd>
                        <dd>gnomAD_E_OTH: 0.0547%</dd>
                        <dd>gnomAD_E_SAS: 0.0032%</dd>
                        <dd>gnomAD_G_AFR: 1.8234%</dd>
                        <dd>gnomAD_G_AMR: 0.2387%</dd>
                        <dd>gnomAD_G_EAS: 0.0617%</dd>
                        <dd>gnomAD_G_NFE: 0.0067%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183421">RIPK4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.147</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.332 to <a href="http://www.omim.org/263650">Popliteal pterygium syndrome, Bartsocas-Papas type</a> associated with RIPK4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - HP:0002025, Anal stenosis</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain -</dd></dl><dl><dt>Phenotypic similarity 0.364 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RIPK4">RIPK4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0003130, anal atresia</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0003276, esophageal atresia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RIPK4%0DSLC3A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC3A1</a> and phenotypic similarity 0.619 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=163690">Hypotonia-cystinuria syndrome</a> associated with SLC3A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine -</dd><dd>HP:0012378, Fatigue - HP:0012378, Fatigue</dd><dd>HP:0002027, Abdominal pain - HP:0012378, Fatigue</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RIPK4%0DSLC3A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC3A1</a> and phenotypic similarity 0.251 to mouse mutant of SLC3A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0040275, Adenocarcinoma of the large intestine - MP:0000741, impaired contractility of urinary bladder detrusor smooth muscle</dd><dd>HP:0012378, Fatigue -</dd><dd>HP:0002027, Abdominal pain - MP:0000741, impaired contractility of urinary bladder detrusor smooth muscle</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/214350">OMIM:214350</a> CHAND syndrome - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/263650">OMIM:263650</a> Popliteal pterygium syndrome, Bartsocas-Papas type - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1234">ORPHA:1234</a> Bartsocas-Papas syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1401">ORPHA:1401</a> CHAND syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.147</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr21:g.41756723G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=558806428">rs558806428</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs558806428&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.147</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000332512.8">RIPK4:ENST00000332512.8:c.276C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000352483.3">RIPK4:ENST00000352483.3:c.276C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.14807767</dd>
                                    
                                    <dd>CADD: 0.148 (0.696)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0088%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC NFE: 0.0090%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0203%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0125%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_EAS: 0.0617%</dd>
                                    <dd>gnomAD_G_NFE: 0.0400%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188897">AC099489.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.399</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.399</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.11425166G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=565085923">rs565085923</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs565085923&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.798</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000598234.6">AC099489.1:ENST00000598234.6:c.5867C&gt;T:p.(Ala1956Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8452609</dd>
                                    
                                    <dd>CADD: 0.845 (8.104)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1597%</dd>
                                    <dd>TOPMed: 0.1155%</dd>
                                    <dd>ExAC AFR: 0.2688%</dd>
                                    <dd>gnomAD_E_AFR: 0.2762%</dd>
                                    <dd>gnomAD_G_AFR: 0.3444%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr16:g.11448940C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=140401887">rs140401887</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs140401887&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.001</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000598234.6">AC099489.1:ENST00000598234.6:c.4425G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.0013806224</dd>
                                    
                                    <dd>CADD: 0.001 (0.006)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.5391%</dd>
                                    <dd>TOPMed: 0.5447%</dd>
                                    <dd>UK10K: 0.2248%</dd>
                                    <dd>ExAC AFR: 1.4545%</dd>
                                    <dd>ExAC AMR: 1.1905%</dd>
                                    <dd>ExAC NFE: 0.5806%</dd>
                                    <dd>ExAC SAS: 0.0993%</dd>
                                    <dd>gnomAD_E_AFR: 1.1039%</dd>
                                    <dd>gnomAD_E_AMR: 0.9651%</dd>
                                    <dd>gnomAD_E_ASJ: 0.9213%</dd>
                                    <dd>gnomAD_E_NFE: 0.2840%</dd>
                                    <dd>gnomAD_E_OTH: 1.3491%</dd>
                                    <dd>gnomAD_E_SAS: 0.0362%</dd>
                                    <dd>gnomAD_G_AFR: 1.0550%</dd>
                                    <dd>gnomAD_G_AMR: 1.3126%</dd>
                                    <dd>gnomAD_G_NFE: 0.1600%</dd>
                                    <dd>gnomAD_G_OTH: 0.5102%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000215529">EFCAB8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.398</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.398</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr20:g.32912837G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=113341957">rs113341957</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs113341957&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.693</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400522.8">EFCAB8:ENST00000400522.8:c.1829G&gt;C:p.(Gly610Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99645185</dd>
                                    
                                    <dd>CADD: 0.996 (24.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4792%</dd>
                                    <dd>TOPMed: 0.3201%</dd>
                                    <dd>ESP AA: 0.6503%</dd>
                                    <dd>ESP All: 0.1971%</dd>
                                    <dd>ExAC AFR: 0.8304%</dd>
                                    <dd>ExAC NFE: 0.0106%</dd>
                                    <dd>gnomAD_E_AFR: 0.9904%</dd>
                                    <dd>gnomAD_E_AMR: 0.0877%</dd>
                                    <dd>gnomAD_E_NFE: 0.0034%</dd>
                                    <dd>gnomAD_E_OTH: 0.0253%</dd>
                                    <dd>gnomAD_G_AFR: 1.1796%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr20:g.32885622G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=139861538">rs139861538</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs139861538&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.104</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400522.8">EFCAB8:ENST00000400522.8:c.549G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.16555065</dd>
                                    
                                    <dd>CADD: 0.166 (0.786)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3994%</dd>
                                    <dd>TOPMed: 0.6060%</dd>
                                    <dd>UK10K: 1.1108%</dd>
                                    <dd>ESP AA: 0.0723%</dd>
                                    <dd>ESP EA: 1.1314%</dd>
                                    <dd>ESP All: 0.8103%</dd>
                                    <dd>ExAC AFR: 0.0931%</dd>
                                    <dd>ExAC AMR: 0.9756%</dd>
                                    <dd>ExAC FIN: 0.7082%</dd>
                                    <dd>ExAC NFE: 0.7955%</dd>
                                    <dd>ExAC OTH: 1.3043%</dd>
                                    <dd>ExAC SAS: 0.7455%</dd>
                                    <dd>gnomAD_E_AFR: 0.0655%</dd>
                                    <dd>gnomAD_E_AMR: 0.3715%</dd>
                                    <dd>gnomAD_E_ASJ: 0.2547%</dd>
                                    <dd>gnomAD_E_EAS: 0.0195%</dd>
                                    <dd>gnomAD_E_FIN: 0.5516%</dd>
                                    <dd>gnomAD_E_NFE: 0.9579%</dd>
                                    <dd>gnomAD_E_OTH: 0.9016%</dd>
                                    <dd>gnomAD_E_SAS: 0.7975%</dd>
                                    <dd>gnomAD_G_AFR: 0.2293%</dd>
                                    <dd>gnomAD_G_AMR: 0.4773%</dd>
                                    <dd>gnomAD_G_ASJ: 1.3245%</dd>
                                    <dd>gnomAD_G_FIN: 0.5438%</dd>
                                    <dd>gnomAD_G_NFE: 0.9073%</dd>
                                    <dd>gnomAD_G_OTH: 0.4073%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143971">ETAA1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.367</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.367</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.67404668A&gt;G [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=113974372">rs113974372</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs113974372&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.367</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000272342.6">ETAA1:ENST00000272342.6:c.1986A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000644028.1">ETAA1:ENST00000644028.1:c.543-568A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6753615</dd>
                                    
                                    <dd>CADD: 0.675 (4.886)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3594%</dd>
                                    <dd>TOPMed: 0.4316%</dd>
                                    <dd>ESP AA: 1.4766%</dd>
                                    <dd>ESP All: 0.5005%</dd>
                                    <dd>ExAC AFR: 1.3261%</dd>
                                    <dd>ExAC AMR: 0.0434%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>ExAC OTH: 0.1104%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AFR: 0.4743%</dd>
                                    <dd>gnomAD_E_AMR: 0.0328%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_AFR: 1.2279%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000131845">AC005261.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.338</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.338</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.57356225G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=61731266">rs61731266</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs61731266&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.364</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000282286.5">AC005261.1:ENST00000282286.5:c.356G&gt;A:p.(Arg119His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391705.7">AC005261.1:ENST00000391705.7:c.356G&gt;A:p.(Arg119His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000443917.6">AC005261.1:ENST00000443917.6:c.497G&gt;A:p.(Arg166His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000598744.1">AC005261.1:ENST00000598744.1:c.230G&gt;A:p.(Arg77His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.70630264</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.555 (T)</dd>
                                    <dd>CADD: 0.706 (5.321)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.7788%</dd>
                                    <dd>TOPMed: 0.3974%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 1.2029%</dd>
                                    <dd>ESP All: 0.4075%</dd>
                                    <dd>ExAC AFR: 1.4033%</dd>
                                    <dd>ExAC AMR: 0.0950%</dd>
                                    <dd>ExAC NFE: 0.0075%</dd>
                                    <dd>ExAC SAS: 0.0242%</dd>
                                    <dd>gnomAD_E_AFR: 1.5225%</dd>
                                    <dd>gnomAD_E_AMR: 0.1221%</dd>
                                    <dd>gnomAD_E_NFE: 0.0090%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_E_SAS: 0.0195%</dd>
                                    <dd>gnomAD_G_AFR: 1.1567%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                    <dd>gnomAD_G_OTH: 0.2041%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.57355329G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=574274358">rs574274358</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs574274358&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.313</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000443917.6">AC005261.1:ENST00000443917.6:c.219G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000282286.5">AC005261.1:ENST00000282286.5:c.161-701G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391705.7">AC005261.1:ENST00000391705.7:c.161-701G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000598744.1">AC005261.1:ENST00000598744.1:c.35-701G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.313406</dd>
                                    
                                    <dd>CADD: 0.313 (1.633)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0123%</dd>
                                    <dd>gnomAD_E_AFR: 0.0072%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000278522">POTEB3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.260</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.260</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr15:g.21439844C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=140204818">rs140204818</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs140204818&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.260</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611217.4">POTEB3:ENST00000611217.4:c.168G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.2603947</dd>
                                    
                                    <dd>CADD: 0.260 (1.310)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>

    

    <div id="about">
        <h3>About</h3>
        <p>
            The Exomizer is a Java program that functionally annotates variants from whole-exome
            sequencing data starting from a VCF file (version 4). The functional annotation code is
            based on <a href="https://github.com/charite/jannovar/">Jannovar</a> and uses
            <a href="http://genome.ucsc.edu/">UCSC</a> KnownGene transcript definitions and
            hg19 genomic coordinates
        </p>
        <p>
            Variants are prioritized according to user-defined criteria on variant frequency, pathogenicity,
            quality, inheritance pattern, and model organism phenotype data. Predicted pathogenicity data
            was extracted from the <a href="http://www.ncbi.nlm.nih.gov/pubmed/21520341">dbNSFP</a> resource.
        </p>
        <p>
            Developed by the Computational Biology and Bioinformatics group at the
            <a href="http://genetik.charite.de/">Institute for Medical Genetics and Human Genetics</a> of the
            <a href="http://www.charite.de">Charit&eacute; - Universit&auml;tsmedizin Berlin</a>, the Mouse
            Informatics Group at the <a href="http://www.sanger.ac.uk/">Sanger Institute</a> and the Smedley group at
            <a href="https://qmul.ac.uk/whri">Queen Mary University of London</a>.
        </p>


        <p>Problems, suggestions, or comments? Please <a href="mailto:peter.robinson@charite.de">let us know</a></p>
    </div>
</div>
</body>
</html>
