<!--
  ~ The Exomiser - A tool to annotate and prioritize genomic variants
  ~
  ~ Copyright (c) 2016-2019 Queen Mary University of London.
  ~ Copyright (c) 2012-2016 Charité Universitätsmedizin Berlin and Genome Research Ltd.
  ~
  ~ This program is free software: you can redistribute it and/or modify
  ~ it under the terms of the GNU Affero General Public License as
  ~ published by the Free Software Foundation, either version 3 of the
  ~ License, or (at your option) any later version.
  ~
  ~ This program is distributed in the hope that it will be useful,
  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  ~ GNU Affero General Public License for more details.
  ~
  ~ You should have received a copy of the GNU Affero General Public License
  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  -->

<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>
    <title>The Exomiser - A Tool to Annotate and Prioritise Whole-Exome Sequencing Data</title>
    <meta charset="UTF-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <style>
        /*!
 * Bootstrap v3.3.7 (http://getbootstrap.com)
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 *//*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */html{font-family:sans-serif;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block;vertical-align:baseline}audio:not([controls]){display:none;height:0}[hidden],template{display:none}a{background-color:transparent}a:active,a:hover{outline:0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:700}dfn{font-style:italic}h1{margin:.67em 0;font-size:2em}mark{color:#000;background:#ff0}small{font-size:80%}sub,sup{position:relative;font-size:75%;line-height:0;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}img{border:0}svg:not(:root){overflow:hidden}figure{margin:1em 40px}hr{height:0;-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}pre{overflow:auto}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}button,input,optgroup,select,textarea{margin:0;font:inherit;color:inherit}button{overflow:visible}button,select{text-transform:none}button,html input[type=button],input[type=reset],input[type=submit]{-webkit-appearance:button;cursor:pointer}button[disabled],html input[disabled]{cursor:default}button::-moz-focus-inner,input::-moz-focus-inner{padding:0;border:0}input{line-height:normal}input[type=checkbox],input[type=radio]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;padding:0}input[type=number]::-webkit-inner-spin-button,input[type=number]::-webkit-outer-spin-button{height:auto}input[type=search]{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;-webkit-appearance:textfield}input[type=search]::-webkit-search-cancel-button,input[type=search]::-webkit-search-decoration{-webkit-appearance:none}fieldset{padding:.35em .625em .75em;margin:0 2px;border:1px solid silver}legend{padding:0;border:0}textarea{overflow:auto}optgroup{font-weight:700}table{border-spacing:0;border-collapse:collapse}td,th{padding:0}/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */@media print{*,:after,:before{color:#000!important;text-shadow:none!important;background:0 0!important;-webkit-box-shadow:none!important;box-shadow:none!important}a,a:visited{text-decoration:underline}a[href]:after{content:" (" attr(href) ")"}abbr[title]:after{content:" (" attr(title) ")"}a[href^="javascript:"]:after,a[href^="#"]:after{content:""}blockquote,pre{border:1px solid #999;page-break-inside:avoid}thead{display:table-header-group}img,tr{page-break-inside:avoid}img{max-width:100%!important}h2,h3,p{orphans:3;widows:3}h2,h3{page-break-after:avoid}.navbar{display:none}.btn>.caret,.dropup>.btn>.caret{border-top-color:#000!important}.label{border:1px solid #000}.table{border-collapse:collapse!important}.table td,.table th{background-color:#fff!important}.table-bordered td,.table-bordered th{border:1px solid #ddd!important}}@font-face{font-family:'Glyphicons Halflings';src:url(../fonts/glyphicons-halflings-regular.eot);src:url(../fonts/glyphicons-halflings-regular.eot?#iefix) format('embedded-opentype'),url(../fonts/glyphicons-halflings-regular.woff2) format('woff2'),url(../fonts/glyphicons-halflings-regular.woff) format('woff'),url(../fonts/glyphicons-halflings-regular.ttf) format('truetype'),url(../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular) format('svg')}.glyphicon{position:relative;top:1px;display:inline-block;font-family:'Glyphicons Halflings';font-style:normal;font-weight:400;line-height:1;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.glyphicon-asterisk:before{content:"\002a"}.glyphicon-plus:before{content:"\002b"}.glyphicon-eur:before,.glyphicon-euro:before{content:"\20ac"}.glyphicon-minus:before{content:"\2212"}.glyphicon-cloud:before{content:"\2601"}.glyphicon-envelope:before{content:"\2709"}.glyphicon-pencil:before{content:"\270f"}.glyphicon-glass:before{content:"\e001"}.glyphicon-music:before{content:"\e002"}.glyphicon-search:before{content:"\e003"}.glyphicon-heart:before{content:"\e005"}.glyphicon-star:before{content:"\e006"}.glyphicon-star-empty:before{content:"\e007"}.glyphicon-user:before{content:"\e008"}.glyphicon-film:before{content:"\e009"}.glyphicon-th-large:before{content:"\e010"}.glyphicon-th:before{content:"\e011"}.glyphicon-th-list:before{content:"\e012"}.glyphicon-ok:before{content:"\e013"}.glyphicon-remove:before{content:"\e014"}.glyphicon-zoom-in:before{content:"\e015"}.glyphicon-zoom-out:before{content:"\e016"}.glyphicon-off:before{content:"\e017"}.glyphicon-signal:before{content:"\e018"}.glyphicon-cog:before{content:"\e019"}.glyphicon-trash:before{content:"\e020"}.glyphicon-home:before{content:"\e021"}.glyphicon-file:before{content:"\e022"}.glyphicon-time:before{content:"\e023"}.glyphicon-road:before{content:"\e024"}.glyphicon-download-alt:before{content:"\e025"}.glyphicon-download:before{content:"\e026"}.glyphicon-upload:before{content:"\e027"}.glyphicon-inbox:before{content:"\e028"}.glyphicon-play-circle:before{content:"\e029"}.glyphicon-repeat:before{content:"\e030"}.glyphicon-refresh:before{content:"\e031"}.glyphicon-list-alt:before{content:"\e032"}.glyphicon-lock:before{content:"\e033"}.glyphicon-flag:before{content:"\e034"}.glyphicon-headphones:before{content:"\e035"}.glyphicon-volume-off:before{content:"\e036"}.glyphicon-volume-down:before{content:"\e037"}.glyphicon-volume-up:before{content:"\e038"}.glyphicon-qrcode:before{content:"\e039"}.glyphicon-barcode:before{content:"\e040"}.glyphicon-tag:before{content:"\e041"}.glyphicon-tags:before{content:"\e042"}.glyphicon-book:before{content:"\e043"}.glyphicon-bookmark:before{content:"\e044"}.glyphicon-print:before{content:"\e045"}.glyphicon-camera:before{content:"\e046"}.glyphicon-font:before{content:"\e047"}.glyphicon-bold:before{content:"\e048"}.glyphicon-italic:before{content:"\e049"}.glyphicon-text-height:before{content:"\e050"}.glyphicon-text-width:before{content:"\e051"}.glyphicon-align-left:before{content:"\e052"}.glyphicon-align-center:before{content:"\e053"}.glyphicon-align-right:before{content:"\e054"}.glyphicon-align-justify:before{content:"\e055"}.glyphicon-list:before{content:"\e056"}.glyphicon-indent-left:before{content:"\e057"}.glyphicon-indent-right:before{content:"\e058"}.glyphicon-facetime-video:before{content:"\e059"}.glyphicon-picture:before{content:"\e060"}.glyphicon-map-marker:before{content:"\e062"}.glyphicon-adjust:before{content:"\e063"}.glyphicon-tint:before{content:"\e064"}.glyphicon-edit:before{content:"\e065"}.glyphicon-share:before{content:"\e066"}.glyphicon-check:before{content:"\e067"}.glyphicon-move:before{content:"\e068"}.glyphicon-step-backward:before{content:"\e069"}.glyphicon-fast-backward:before{content:"\e070"}.glyphicon-backward:before{content:"\e071"}.glyphicon-play:before{content:"\e072"}.glyphicon-pause:before{content:"\e073"}.glyphicon-stop:before{content:"\e074"}.glyphicon-forward:before{content:"\e075"}.glyphicon-fast-forward:before{content:"\e076"}.glyphicon-step-forward:before{content:"\e077"}.glyphicon-eject:before{content:"\e078"}.glyphicon-chevron-left:before{content:"\e079"}.glyphicon-chevron-right:before{content:"\e080"}.glyphicon-plus-sign:before{content:"\e081"}.glyphicon-minus-sign:before{content:"\e082"}.glyphicon-remove-sign:before{content:"\e083"}.glyphicon-ok-sign:before{content:"\e084"}.glyphicon-question-sign:before{content:"\e085"}.glyphicon-info-sign:before{content:"\e086"}.glyphicon-screenshot:before{content:"\e087"}.glyphicon-remove-circle:before{content:"\e088"}.glyphicon-ok-circle:before{content:"\e089"}.glyphicon-ban-circle:before{content:"\e090"}.glyphicon-arrow-left:before{content:"\e091"}.glyphicon-arrow-right:before{content:"\e092"}.glyphicon-arrow-up:before{content:"\e093"}.glyphicon-arrow-down:before{content:"\e094"}.glyphicon-share-alt:before{content:"\e095"}.glyphicon-resize-full:before{content:"\e096"}.glyphicon-resize-small:before{content:"\e097"}.glyphicon-exclamation-sign:before{content:"\e101"}.glyphicon-gift:before{content:"\e102"}.glyphicon-leaf:before{content:"\e103"}.glyphicon-fire:before{content:"\e104"}.glyphicon-eye-open:before{content:"\e105"}.glyphicon-eye-close:before{content:"\e106"}.glyphicon-warning-sign:before{content:"\e107"}.glyphicon-plane:before{content:"\e108"}.glyphicon-calendar:before{content:"\e109"}.glyphicon-random:before{content:"\e110"}.glyphicon-comment:before{content:"\e111"}.glyphicon-magnet:before{content:"\e112"}.glyphicon-chevron-up:before{content:"\e113"}.glyphicon-chevron-down:before{content:"\e114"}.glyphicon-retweet:before{content:"\e115"}.glyphicon-shopping-cart:before{content:"\e116"}.glyphicon-folder-close:before{content:"\e117"}.glyphicon-folder-open:before{content:"\e118"}.glyphicon-resize-vertical:before{content:"\e119"}.glyphicon-resize-horizontal:before{content:"\e120"}.glyphicon-hdd:before{content:"\e121"}.glyphicon-bullhorn:before{content:"\e122"}.glyphicon-bell:before{content:"\e123"}.glyphicon-certificate:before{content:"\e124"}.glyphicon-thumbs-up:before{content:"\e125"}.glyphicon-thumbs-down:before{content:"\e126"}.glyphicon-hand-right:before{content:"\e127"}.glyphicon-hand-left:before{content:"\e128"}.glyphicon-hand-up:before{content:"\e129"}.glyphicon-hand-down:before{content:"\e130"}.glyphicon-circle-arrow-right:before{content:"\e131"}.glyphicon-circle-arrow-left:before{content:"\e132"}.glyphicon-circle-arrow-up:before{content:"\e133"}.glyphicon-circle-arrow-down:before{content:"\e134"}.glyphicon-globe:before{content:"\e135"}.glyphicon-wrench:before{content:"\e136"}.glyphicon-tasks:before{content:"\e137"}.glyphicon-filter:before{content:"\e138"}.glyphicon-briefcase:before{content:"\e139"}.glyphicon-fullscreen:before{content:"\e140"}.glyphicon-dashboard:before{content:"\e141"}.glyphicon-paperclip:before{content:"\e142"}.glyphicon-heart-empty:before{content:"\e143"}.glyphicon-link:before{content:"\e144"}.glyphicon-phone:before{content:"\e145"}.glyphicon-pushpin:before{content:"\e146"}.glyphicon-usd:before{content:"\e148"}.glyphicon-gbp:before{content:"\e149"}.glyphicon-sort:before{content:"\e150"}.glyphicon-sort-by-alphabet:before{content:"\e151"}.glyphicon-sort-by-alphabet-alt:before{content:"\e152"}.glyphicon-sort-by-order:before{content:"\e153"}.glyphicon-sort-by-order-alt:before{content:"\e154"}.glyphicon-sort-by-attributes:before{content:"\e155"}.glyphicon-sort-by-attributes-alt:before{content:"\e156"}.glyphicon-unchecked:before{content:"\e157"}.glyphicon-expand:before{content:"\e158"}.glyphicon-collapse-down:before{content:"\e159"}.glyphicon-collapse-up:before{content:"\e160"}.glyphicon-log-in:before{content:"\e161"}.glyphicon-flash:before{content:"\e162"}.glyphicon-log-out:before{content:"\e163"}.glyphicon-new-window:before{content:"\e164"}.glyphicon-record:before{content:"\e165"}.glyphicon-save:before{content:"\e166"}.glyphicon-open:before{content:"\e167"}.glyphicon-saved:before{content:"\e168"}.glyphicon-import:before{content:"\e169"}.glyphicon-export:before{content:"\e170"}.glyphicon-send:before{content:"\e171"}.glyphicon-floppy-disk:before{content:"\e172"}.glyphicon-floppy-saved:before{content:"\e173"}.glyphicon-floppy-remove:before{content:"\e174"}.glyphicon-floppy-save:before{content:"\e175"}.glyphicon-floppy-open:before{content:"\e176"}.glyphicon-credit-card:before{content:"\e177"}.glyphicon-transfer:before{content:"\e178"}.glyphicon-cutlery:before{content:"\e179"}.glyphicon-header:before{content:"\e180"}.glyphicon-compressed:before{content:"\e181"}.glyphicon-earphone:before{content:"\e182"}.glyphicon-phone-alt:before{content:"\e183"}.glyphicon-tower:before{content:"\e184"}.glyphicon-stats:before{content:"\e185"}.glyphicon-sd-video:before{content:"\e186"}.glyphicon-hd-video:before{content:"\e187"}.glyphicon-subtitles:before{content:"\e188"}.glyphicon-sound-stereo:before{content:"\e189"}.glyphicon-sound-dolby:before{content:"\e190"}.glyphicon-sound-5-1:before{content:"\e191"}.glyphicon-sound-6-1:before{content:"\e192"}.glyphicon-sound-7-1:before{content:"\e193"}.glyphicon-copyright-mark:before{content:"\e194"}.glyphicon-registration-mark:before{content:"\e195"}.glyphicon-cloud-download:before{content:"\e197"}.glyphicon-cloud-upload:before{content:"\e198"}.glyphicon-tree-conifer:before{content:"\e199"}.glyphicon-tree-deciduous:before{content:"\e200"}.glyphicon-cd:before{content:"\e201"}.glyphicon-save-file:before{content:"\e202"}.glyphicon-open-file:before{content:"\e203"}.glyphicon-level-up:before{content:"\e204"}.glyphicon-copy:before{content:"\e205"}.glyphicon-paste:before{content:"\e206"}.glyphicon-alert:before{content:"\e209"}.glyphicon-equalizer:before{content:"\e210"}.glyphicon-king:before{content:"\e211"}.glyphicon-queen:before{content:"\e212"}.glyphicon-pawn:before{content:"\e213"}.glyphicon-bishop:before{content:"\e214"}.glyphicon-knight:before{content:"\e215"}.glyphicon-baby-formula:before{content:"\e216"}.glyphicon-tent:before{content:"\26fa"}.glyphicon-blackboard:before{content:"\e218"}.glyphicon-bed:before{content:"\e219"}.glyphicon-apple:before{content:"\f8ff"}.glyphicon-erase:before{content:"\e221"}.glyphicon-hourglass:before{content:"\231b"}.glyphicon-lamp:before{content:"\e223"}.glyphicon-duplicate:before{content:"\e224"}.glyphicon-piggy-bank:before{content:"\e225"}.glyphicon-scissors:before{content:"\e226"}.glyphicon-bitcoin:before{content:"\e227"}.glyphicon-btc:before{content:"\e227"}.glyphicon-xbt:before{content:"\e227"}.glyphicon-yen:before{content:"\00a5"}.glyphicon-jpy:before{content:"\00a5"}.glyphicon-ruble:before{content:"\20bd"}.glyphicon-rub:before{content:"\20bd"}.glyphicon-scale:before{content:"\e230"}.glyphicon-ice-lolly:before{content:"\e231"}.glyphicon-ice-lolly-tasted:before{content:"\e232"}.glyphicon-education:before{content:"\e233"}.glyphicon-option-horizontal:before{content:"\e234"}.glyphicon-option-vertical:before{content:"\e235"}.glyphicon-menu-hamburger:before{content:"\e236"}.glyphicon-modal-window:before{content:"\e237"}.glyphicon-oil:before{content:"\e238"}.glyphicon-grain:before{content:"\e239"}.glyphicon-sunglasses:before{content:"\e240"}.glyphicon-text-size:before{content:"\e241"}.glyphicon-text-color:before{content:"\e242"}.glyphicon-text-background:before{content:"\e243"}.glyphicon-object-align-top:before{content:"\e244"}.glyphicon-object-align-bottom:before{content:"\e245"}.glyphicon-object-align-horizontal:before{content:"\e246"}.glyphicon-object-align-left:before{content:"\e247"}.glyphicon-object-align-vertical:before{content:"\e248"}.glyphicon-object-align-right:before{content:"\e249"}.glyphicon-triangle-right:before{content:"\e250"}.glyphicon-triangle-left:before{content:"\e251"}.glyphicon-triangle-bottom:before{content:"\e252"}.glyphicon-triangle-top:before{content:"\e253"}.glyphicon-console:before{content:"\e254"}.glyphicon-superscript:before{content:"\e255"}.glyphicon-subscript:before{content:"\e256"}.glyphicon-menu-left:before{content:"\e257"}.glyphicon-menu-right:before{content:"\e258"}.glyphicon-menu-down:before{content:"\e259"}.glyphicon-menu-up:before{content:"\e260"}*{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}:after,:before{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}html{font-size:10px;-webkit-tap-highlight-color:rgba(0,0,0,0)}body{font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;line-height:1.42857143;color:#333;background-color:#fff}button,input,select,textarea{font-family:inherit;font-size:inherit;line-height:inherit}a{color:#337ab7;text-decoration:none}a:focus,a:hover{color:#23527c;text-decoration:underline}a:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}figure{margin:0}img{vertical-align:middle}.carousel-inner>.item>a>img,.carousel-inner>.item>img,.img-responsive,.thumbnail a>img,.thumbnail>img{display:block;max-width:100%;height:auto}.img-rounded{border-radius:6px}.img-thumbnail{display:inline-block;max-width:100%;height:auto;padding:4px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:all .2s ease-in-out;-o-transition:all .2s ease-in-out;transition:all .2s ease-in-out}.img-circle{border-radius:50%}hr{margin-top:20px;margin-bottom:20px;border:0;border-top:1px solid #eee}.sr-only{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0,0,0,0);border:0}.sr-only-focusable:active,.sr-only-focusable:focus{position:static;width:auto;height:auto;margin:0;overflow:visible;clip:auto}[role=button]{cursor:pointer}.h1,.h2,.h3,.h4,.h5,.h6,h1,h2,h3,h4,h5,h6{font-family:inherit;font-weight:500;line-height:1.1;color:inherit}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-weight:400;line-height:1;color:#777}.h1,.h2,.h3,h1,h2,h3{margin-top:20px;margin-bottom:10px}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small{font-size:65%}.h4,.h5,.h6,h4,h5,h6{margin-top:10px;margin-bottom:10px}.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-size:75%}.h1,h1{font-size:36px}.h2,h2{font-size:30px}.h3,h3{font-size:24px}.h4,h4{font-size:18px}.h5,h5{font-size:14px}.h6,h6{font-size:12px}p{margin:0 0 10px}.lead{margin-bottom:20px;font-size:16px;font-weight:300;line-height:1.4}@media (min-width:768px){.lead{font-size:21px}}.small,small{font-size:85%}.mark,mark{padding:.2em;background-color:#fcf8e3}.text-left{text-align:left}.text-right{text-align:right}.text-center{text-align:center}.text-justify{text-align:justify}.text-nowrap{white-space:nowrap}.text-lowercase{text-transform:lowercase}.text-uppercase{text-transform:uppercase}.text-capitalize{text-transform:capitalize}.text-muted{color:#777}.text-primary{color:#337ab7}a.text-primary:focus,a.text-primary:hover{color:#286090}.text-success{color:#3c763d}a.text-success:focus,a.text-success:hover{color:#2b542c}.text-info{color:#31708f}a.text-info:focus,a.text-info:hover{color:#245269}.text-warning{color:#8a6d3b}a.text-warning:focus,a.text-warning:hover{color:#66512c}.text-danger{color:#a94442}a.text-danger:focus,a.text-danger:hover{color:#843534}.bg-primary{color:#fff;background-color:#337ab7}a.bg-primary:focus,a.bg-primary:hover{background-color:#286090}.bg-success{background-color:#dff0d8}a.bg-success:focus,a.bg-success:hover{background-color:#c1e2b3}.bg-info{background-color:#d9edf7}a.bg-info:focus,a.bg-info:hover{background-color:#afd9ee}.bg-warning{background-color:#fcf8e3}a.bg-warning:focus,a.bg-warning:hover{background-color:#f7ecb5}.bg-danger{background-color:#f2dede}a.bg-danger:focus,a.bg-danger:hover{background-color:#e4b9b9}.page-header{padding-bottom:9px;margin:40px 0 20px;border-bottom:1px solid #eee}ol,ul{margin-top:0;margin-bottom:10px}ol ol,ol ul,ul ol,ul ul{margin-bottom:0}.list-unstyled{padding-left:0;list-style:none}.list-inline{padding-left:0;margin-left:-5px;list-style:none}.list-inline>li{display:inline-block;padding-right:5px;padding-left:5px}dl{margin-top:0;margin-bottom:20px}dd,dt{line-height:1.42857143}dt{font-weight:700}dd{margin-left:0}@media (min-width:768px){.dl-horizontal dt{float:left;width:160px;overflow:hidden;clear:left;text-align:right;text-overflow:ellipsis;white-space:nowrap}.dl-horizontal dd{margin-left:180px}}abbr[data-original-title],abbr[title]{cursor:help;border-bottom:1px dotted #777}.initialism{font-size:90%;text-transform:uppercase}blockquote{padding:10px 20px;margin:0 0 20px;font-size:17.5px;border-left:5px solid #eee}blockquote ol:last-child,blockquote p:last-child,blockquote ul:last-child{margin-bottom:0}blockquote .small,blockquote footer,blockquote small{display:block;font-size:80%;line-height:1.42857143;color:#777}blockquote .small:before,blockquote footer:before,blockquote small:before{content:'\2014 \00A0'}.blockquote-reverse,blockquote.pull-right{padding-right:15px;padding-left:0;text-align:right;border-right:5px solid #eee;border-left:0}.blockquote-reverse .small:before,.blockquote-reverse footer:before,.blockquote-reverse small:before,blockquote.pull-right .small:before,blockquote.pull-right footer:before,blockquote.pull-right small:before{content:''}.blockquote-reverse .small:after,.blockquote-reverse footer:after,.blockquote-reverse small:after,blockquote.pull-right .small:after,blockquote.pull-right footer:after,blockquote.pull-right small:after{content:'\00A0 \2014'}address{margin-bottom:20px;font-style:normal;line-height:1.42857143}code,kbd,pre,samp{font-family:Menlo,Monaco,Consolas,"Courier New",monospace}code{padding:2px 4px;font-size:90%;color:#c7254e;background-color:#f9f2f4;border-radius:4px}kbd{padding:2px 4px;font-size:90%;color:#fff;background-color:#333;border-radius:3px;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.25);box-shadow:inset 0 -1px 0 rgba(0,0,0,.25)}kbd kbd{padding:0;font-size:100%;font-weight:700;-webkit-box-shadow:none;box-shadow:none}pre{display:block;padding:9.5px;margin:0 0 10px;font-size:13px;line-height:1.42857143;color:#333;word-break:break-all;word-wrap:break-word;background-color:#f5f5f5;border:1px solid #ccc;border-radius:4px}pre code{padding:0;font-size:inherit;color:inherit;white-space:pre-wrap;background-color:transparent;border-radius:0}.pre-scrollable{max-height:340px;overflow-y:scroll}.container{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}@media (min-width:768px){.container{width:750px}}@media (min-width:992px){.container{width:970px}}@media (min-width:1200px){.container{width:1170px}}.container-fluid{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}.row{margin-right:-15px;margin-left:-15px}.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9,.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9,.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9,.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{position:relative;min-height:1px;padding-right:15px;padding-left:15px}.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{float:left}.col-xs-12{width:100%}.col-xs-11{width:91.66666667%}.col-xs-10{width:83.33333333%}.col-xs-9{width:75%}.col-xs-8{width:66.66666667%}.col-xs-7{width:58.33333333%}.col-xs-6{width:50%}.col-xs-5{width:41.66666667%}.col-xs-4{width:33.33333333%}.col-xs-3{width:25%}.col-xs-2{width:16.66666667%}.col-xs-1{width:8.33333333%}.col-xs-pull-12{right:100%}.col-xs-pull-11{right:91.66666667%}.col-xs-pull-10{right:83.33333333%}.col-xs-pull-9{right:75%}.col-xs-pull-8{right:66.66666667%}.col-xs-pull-7{right:58.33333333%}.col-xs-pull-6{right:50%}.col-xs-pull-5{right:41.66666667%}.col-xs-pull-4{right:33.33333333%}.col-xs-pull-3{right:25%}.col-xs-pull-2{right:16.66666667%}.col-xs-pull-1{right:8.33333333%}.col-xs-pull-0{right:auto}.col-xs-push-12{left:100%}.col-xs-push-11{left:91.66666667%}.col-xs-push-10{left:83.33333333%}.col-xs-push-9{left:75%}.col-xs-push-8{left:66.66666667%}.col-xs-push-7{left:58.33333333%}.col-xs-push-6{left:50%}.col-xs-push-5{left:41.66666667%}.col-xs-push-4{left:33.33333333%}.col-xs-push-3{left:25%}.col-xs-push-2{left:16.66666667%}.col-xs-push-1{left:8.33333333%}.col-xs-push-0{left:auto}.col-xs-offset-12{margin-left:100%}.col-xs-offset-11{margin-left:91.66666667%}.col-xs-offset-10{margin-left:83.33333333%}.col-xs-offset-9{margin-left:75%}.col-xs-offset-8{margin-left:66.66666667%}.col-xs-offset-7{margin-left:58.33333333%}.col-xs-offset-6{margin-left:50%}.col-xs-offset-5{margin-left:41.66666667%}.col-xs-offset-4{margin-left:33.33333333%}.col-xs-offset-3{margin-left:25%}.col-xs-offset-2{margin-left:16.66666667%}.col-xs-offset-1{margin-left:8.33333333%}.col-xs-offset-0{margin-left:0}@media (min-width:768px){.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9{float:left}.col-sm-12{width:100%}.col-sm-11{width:91.66666667%}.col-sm-10{width:83.33333333%}.col-sm-9{width:75%}.col-sm-8{width:66.66666667%}.col-sm-7{width:58.33333333%}.col-sm-6{width:50%}.col-sm-5{width:41.66666667%}.col-sm-4{width:33.33333333%}.col-sm-3{width:25%}.col-sm-2{width:16.66666667%}.col-sm-1{width:8.33333333%}.col-sm-pull-12{right:100%}.col-sm-pull-11{right:91.66666667%}.col-sm-pull-10{right:83.33333333%}.col-sm-pull-9{right:75%}.col-sm-pull-8{right:66.66666667%}.col-sm-pull-7{right:58.33333333%}.col-sm-pull-6{right:50%}.col-sm-pull-5{right:41.66666667%}.col-sm-pull-4{right:33.33333333%}.col-sm-pull-3{right:25%}.col-sm-pull-2{right:16.66666667%}.col-sm-pull-1{right:8.33333333%}.col-sm-pull-0{right:auto}.col-sm-push-12{left:100%}.col-sm-push-11{left:91.66666667%}.col-sm-push-10{left:83.33333333%}.col-sm-push-9{left:75%}.col-sm-push-8{left:66.66666667%}.col-sm-push-7{left:58.33333333%}.col-sm-push-6{left:50%}.col-sm-push-5{left:41.66666667%}.col-sm-push-4{left:33.33333333%}.col-sm-push-3{left:25%}.col-sm-push-2{left:16.66666667%}.col-sm-push-1{left:8.33333333%}.col-sm-push-0{left:auto}.col-sm-offset-12{margin-left:100%}.col-sm-offset-11{margin-left:91.66666667%}.col-sm-offset-10{margin-left:83.33333333%}.col-sm-offset-9{margin-left:75%}.col-sm-offset-8{margin-left:66.66666667%}.col-sm-offset-7{margin-left:58.33333333%}.col-sm-offset-6{margin-left:50%}.col-sm-offset-5{margin-left:41.66666667%}.col-sm-offset-4{margin-left:33.33333333%}.col-sm-offset-3{margin-left:25%}.col-sm-offset-2{margin-left:16.66666667%}.col-sm-offset-1{margin-left:8.33333333%}.col-sm-offset-0{margin-left:0}}@media (min-width:992px){.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9{float:left}.col-md-12{width:100%}.col-md-11{width:91.66666667%}.col-md-10{width:83.33333333%}.col-md-9{width:75%}.col-md-8{width:66.66666667%}.col-md-7{width:58.33333333%}.col-md-6{width:50%}.col-md-5{width:41.66666667%}.col-md-4{width:33.33333333%}.col-md-3{width:25%}.col-md-2{width:16.66666667%}.col-md-1{width:8.33333333%}.col-md-pull-12{right:100%}.col-md-pull-11{right:91.66666667%}.col-md-pull-10{right:83.33333333%}.col-md-pull-9{right:75%}.col-md-pull-8{right:66.66666667%}.col-md-pull-7{right:58.33333333%}.col-md-pull-6{right:50%}.col-md-pull-5{right:41.66666667%}.col-md-pull-4{right:33.33333333%}.col-md-pull-3{right:25%}.col-md-pull-2{right:16.66666667%}.col-md-pull-1{right:8.33333333%}.col-md-pull-0{right:auto}.col-md-push-12{left:100%}.col-md-push-11{left:91.66666667%}.col-md-push-10{left:83.33333333%}.col-md-push-9{left:75%}.col-md-push-8{left:66.66666667%}.col-md-push-7{left:58.33333333%}.col-md-push-6{left:50%}.col-md-push-5{left:41.66666667%}.col-md-push-4{left:33.33333333%}.col-md-push-3{left:25%}.col-md-push-2{left:16.66666667%}.col-md-push-1{left:8.33333333%}.col-md-push-0{left:auto}.col-md-offset-12{margin-left:100%}.col-md-offset-11{margin-left:91.66666667%}.col-md-offset-10{margin-left:83.33333333%}.col-md-offset-9{margin-left:75%}.col-md-offset-8{margin-left:66.66666667%}.col-md-offset-7{margin-left:58.33333333%}.col-md-offset-6{margin-left:50%}.col-md-offset-5{margin-left:41.66666667%}.col-md-offset-4{margin-left:33.33333333%}.col-md-offset-3{margin-left:25%}.col-md-offset-2{margin-left:16.66666667%}.col-md-offset-1{margin-left:8.33333333%}.col-md-offset-0{margin-left:0}}@media (min-width:1200px){.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9{float:left}.col-lg-12{width:100%}.col-lg-11{width:91.66666667%}.col-lg-10{width:83.33333333%}.col-lg-9{width:75%}.col-lg-8{width:66.66666667%}.col-lg-7{width:58.33333333%}.col-lg-6{width:50%}.col-lg-5{width:41.66666667%}.col-lg-4{width:33.33333333%}.col-lg-3{width:25%}.col-lg-2{width:16.66666667%}.col-lg-1{width:8.33333333%}.col-lg-pull-12{right:100%}.col-lg-pull-11{right:91.66666667%}.col-lg-pull-10{right:83.33333333%}.col-lg-pull-9{right:75%}.col-lg-pull-8{right:66.66666667%}.col-lg-pull-7{right:58.33333333%}.col-lg-pull-6{right:50%}.col-lg-pull-5{right:41.66666667%}.col-lg-pull-4{right:33.33333333%}.col-lg-pull-3{right:25%}.col-lg-pull-2{right:16.66666667%}.col-lg-pull-1{right:8.33333333%}.col-lg-pull-0{right:auto}.col-lg-push-12{left:100%}.col-lg-push-11{left:91.66666667%}.col-lg-push-10{left:83.33333333%}.col-lg-push-9{left:75%}.col-lg-push-8{left:66.66666667%}.col-lg-push-7{left:58.33333333%}.col-lg-push-6{left:50%}.col-lg-push-5{left:41.66666667%}.col-lg-push-4{left:33.33333333%}.col-lg-push-3{left:25%}.col-lg-push-2{left:16.66666667%}.col-lg-push-1{left:8.33333333%}.col-lg-push-0{left:auto}.col-lg-offset-12{margin-left:100%}.col-lg-offset-11{margin-left:91.66666667%}.col-lg-offset-10{margin-left:83.33333333%}.col-lg-offset-9{margin-left:75%}.col-lg-offset-8{margin-left:66.66666667%}.col-lg-offset-7{margin-left:58.33333333%}.col-lg-offset-6{margin-left:50%}.col-lg-offset-5{margin-left:41.66666667%}.col-lg-offset-4{margin-left:33.33333333%}.col-lg-offset-3{margin-left:25%}.col-lg-offset-2{margin-left:16.66666667%}.col-lg-offset-1{margin-left:8.33333333%}.col-lg-offset-0{margin-left:0}}table{background-color:transparent}caption{padding-top:8px;padding-bottom:8px;color:#777;text-align:left}th{text-align:left}.table{width:100%;max-width:100%;margin-bottom:20px}.table>tbody>tr>td,.table>tbody>tr>th,.table>tfoot>tr>td,.table>tfoot>tr>th,.table>thead>tr>td,.table>thead>tr>th{padding:8px;line-height:1.42857143;vertical-align:top;border-top:1px solid #ddd}.table>thead>tr>th{vertical-align:bottom;border-bottom:2px solid #ddd}.table>caption+thead>tr:first-child>td,.table>caption+thead>tr:first-child>th,.table>colgroup+thead>tr:first-child>td,.table>colgroup+thead>tr:first-child>th,.table>thead:first-child>tr:first-child>td,.table>thead:first-child>tr:first-child>th{border-top:0}.table>tbody+tbody{border-top:2px solid #ddd}.table .table{background-color:#fff}.table-condensed>tbody>tr>td,.table-condensed>tbody>tr>th,.table-condensed>tfoot>tr>td,.table-condensed>tfoot>tr>th,.table-condensed>thead>tr>td,.table-condensed>thead>tr>th{padding:5px}.table-bordered{border:1px solid #ddd}.table-bordered>tbody>tr>td,.table-bordered>tbody>tr>th,.table-bordered>tfoot>tr>td,.table-bordered>tfoot>tr>th,.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border:1px solid #ddd}.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border-bottom-width:2px}.table-striped>tbody>tr:nth-of-type(odd){background-color:#f9f9f9}.table-hover>tbody>tr:hover{background-color:#f5f5f5}table col[class*=col-]{position:static;display:table-column;float:none}table td[class*=col-],table th[class*=col-]{position:static;display:table-cell;float:none}.table>tbody>tr.active>td,.table>tbody>tr.active>th,.table>tbody>tr>td.active,.table>tbody>tr>th.active,.table>tfoot>tr.active>td,.table>tfoot>tr.active>th,.table>tfoot>tr>td.active,.table>tfoot>tr>th.active,.table>thead>tr.active>td,.table>thead>tr.active>th,.table>thead>tr>td.active,.table>thead>tr>th.active{background-color:#f5f5f5}.table-hover>tbody>tr.active:hover>td,.table-hover>tbody>tr.active:hover>th,.table-hover>tbody>tr:hover>.active,.table-hover>tbody>tr>td.active:hover,.table-hover>tbody>tr>th.active:hover{background-color:#e8e8e8}.table>tbody>tr.success>td,.table>tbody>tr.success>th,.table>tbody>tr>td.success,.table>tbody>tr>th.success,.table>tfoot>tr.success>td,.table>tfoot>tr.success>th,.table>tfoot>tr>td.success,.table>tfoot>tr>th.success,.table>thead>tr.success>td,.table>thead>tr.success>th,.table>thead>tr>td.success,.table>thead>tr>th.success{background-color:#dff0d8}.table-hover>tbody>tr.success:hover>td,.table-hover>tbody>tr.success:hover>th,.table-hover>tbody>tr:hover>.success,.table-hover>tbody>tr>td.success:hover,.table-hover>tbody>tr>th.success:hover{background-color:#d0e9c6}.table>tbody>tr.info>td,.table>tbody>tr.info>th,.table>tbody>tr>td.info,.table>tbody>tr>th.info,.table>tfoot>tr.info>td,.table>tfoot>tr.info>th,.table>tfoot>tr>td.info,.table>tfoot>tr>th.info,.table>thead>tr.info>td,.table>thead>tr.info>th,.table>thead>tr>td.info,.table>thead>tr>th.info{background-color:#d9edf7}.table-hover>tbody>tr.info:hover>td,.table-hover>tbody>tr.info:hover>th,.table-hover>tbody>tr:hover>.info,.table-hover>tbody>tr>td.info:hover,.table-hover>tbody>tr>th.info:hover{background-color:#c4e3f3}.table>tbody>tr.warning>td,.table>tbody>tr.warning>th,.table>tbody>tr>td.warning,.table>tbody>tr>th.warning,.table>tfoot>tr.warning>td,.table>tfoot>tr.warning>th,.table>tfoot>tr>td.warning,.table>tfoot>tr>th.warning,.table>thead>tr.warning>td,.table>thead>tr.warning>th,.table>thead>tr>td.warning,.table>thead>tr>th.warning{background-color:#fcf8e3}.table-hover>tbody>tr.warning:hover>td,.table-hover>tbody>tr.warning:hover>th,.table-hover>tbody>tr:hover>.warning,.table-hover>tbody>tr>td.warning:hover,.table-hover>tbody>tr>th.warning:hover{background-color:#faf2cc}.table>tbody>tr.danger>td,.table>tbody>tr.danger>th,.table>tbody>tr>td.danger,.table>tbody>tr>th.danger,.table>tfoot>tr.danger>td,.table>tfoot>tr.danger>th,.table>tfoot>tr>td.danger,.table>tfoot>tr>th.danger,.table>thead>tr.danger>td,.table>thead>tr.danger>th,.table>thead>tr>td.danger,.table>thead>tr>th.danger{background-color:#f2dede}.table-hover>tbody>tr.danger:hover>td,.table-hover>tbody>tr.danger:hover>th,.table-hover>tbody>tr:hover>.danger,.table-hover>tbody>tr>td.danger:hover,.table-hover>tbody>tr>th.danger:hover{background-color:#ebcccc}.table-responsive{min-height:.01%;overflow-x:auto}@media screen and (max-width:767px){.table-responsive{width:100%;margin-bottom:15px;overflow-y:hidden;-ms-overflow-style:-ms-autohiding-scrollbar;border:1px solid #ddd}.table-responsive>.table{margin-bottom:0}.table-responsive>.table>tbody>tr>td,.table-responsive>.table>tbody>tr>th,.table-responsive>.table>tfoot>tr>td,.table-responsive>.table>tfoot>tr>th,.table-responsive>.table>thead>tr>td,.table-responsive>.table>thead>tr>th{white-space:nowrap}.table-responsive>.table-bordered{border:0}.table-responsive>.table-bordered>tbody>tr>td:first-child,.table-responsive>.table-bordered>tbody>tr>th:first-child,.table-responsive>.table-bordered>tfoot>tr>td:first-child,.table-responsive>.table-bordered>tfoot>tr>th:first-child,.table-responsive>.table-bordered>thead>tr>td:first-child,.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.table-responsive>.table-bordered>tbody>tr>td:last-child,.table-responsive>.table-bordered>tbody>tr>th:last-child,.table-responsive>.table-bordered>tfoot>tr>td:last-child,.table-responsive>.table-bordered>tfoot>tr>th:last-child,.table-responsive>.table-bordered>thead>tr>td:last-child,.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.table-responsive>.table-bordered>tbody>tr:last-child>td,.table-responsive>.table-bordered>tbody>tr:last-child>th,.table-responsive>.table-bordered>tfoot>tr:last-child>td,.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}}fieldset{min-width:0;padding:0;margin:0;border:0}legend{display:block;width:100%;padding:0;margin-bottom:20px;font-size:21px;line-height:inherit;color:#333;border:0;border-bottom:1px solid #e5e5e5}label{display:inline-block;max-width:100%;margin-bottom:5px;font-weight:700}input[type=search]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}input[type=checkbox],input[type=radio]{margin:4px 0 0;margin-top:1px\9;line-height:normal}input[type=file]{display:block}input[type=range]{display:block;width:100%}select[multiple],select[size]{height:auto}input[type=file]:focus,input[type=checkbox]:focus,input[type=radio]:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}output{display:block;padding-top:7px;font-size:14px;line-height:1.42857143;color:#555}.form-control{display:block;width:100%;height:34px;padding:6px 12px;font-size:14px;line-height:1.42857143;color:#555;background-color:#fff;background-image:none;border:1px solid #ccc;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075);-webkit-transition:border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;-o-transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s;transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s}.form-control:focus{border-color:#66afe9;outline:0;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6);box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6)}.form-control::-moz-placeholder{color:#999;opacity:1}.form-control:-ms-input-placeholder{color:#999}.form-control::-webkit-input-placeholder{color:#999}.form-control::-ms-expand{background-color:transparent;border:0}.form-control[disabled],.form-control[readonly],fieldset[disabled] .form-control{background-color:#eee;opacity:1}.form-control[disabled],fieldset[disabled] .form-control{cursor:not-allowed}textarea.form-control{height:auto}input[type=search]{-webkit-appearance:none}@media screen and (-webkit-min-device-pixel-ratio:0){input[type=date].form-control,input[type=time].form-control,input[type=datetime-local].form-control,input[type=month].form-control{line-height:34px}.input-group-sm input[type=date],.input-group-sm input[type=time],.input-group-sm input[type=datetime-local],.input-group-sm input[type=month],input[type=date].input-sm,input[type=time].input-sm,input[type=datetime-local].input-sm,input[type=month].input-sm{line-height:30px}.input-group-lg input[type=date],.input-group-lg input[type=time],.input-group-lg input[type=datetime-local],.input-group-lg input[type=month],input[type=date].input-lg,input[type=time].input-lg,input[type=datetime-local].input-lg,input[type=month].input-lg{line-height:46px}}.form-group{margin-bottom:15px}.checkbox,.radio{position:relative;display:block;margin-top:10px;margin-bottom:10px}.checkbox label,.radio label{min-height:20px;padding-left:20px;margin-bottom:0;font-weight:400;cursor:pointer}.checkbox input[type=checkbox],.checkbox-inline input[type=checkbox],.radio input[type=radio],.radio-inline input[type=radio]{position:absolute;margin-top:4px\9;margin-left:-20px}.checkbox+.checkbox,.radio+.radio{margin-top:-5px}.checkbox-inline,.radio-inline{position:relative;display:inline-block;padding-left:20px;margin-bottom:0;font-weight:400;vertical-align:middle;cursor:pointer}.checkbox-inline+.checkbox-inline,.radio-inline+.radio-inline{margin-top:0;margin-left:10px}fieldset[disabled] input[type=checkbox],fieldset[disabled] input[type=radio],input[type=checkbox].disabled,input[type=checkbox][disabled],input[type=radio].disabled,input[type=radio][disabled]{cursor:not-allowed}.checkbox-inline.disabled,.radio-inline.disabled,fieldset[disabled] .checkbox-inline,fieldset[disabled] .radio-inline{cursor:not-allowed}.checkbox.disabled label,.radio.disabled label,fieldset[disabled] .checkbox label,fieldset[disabled] .radio label{cursor:not-allowed}.form-control-static{min-height:34px;padding-top:7px;padding-bottom:7px;margin-bottom:0}.form-control-static.input-lg,.form-control-static.input-sm{padding-right:0;padding-left:0}.input-sm{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-sm{height:30px;line-height:30px}select[multiple].input-sm,textarea.input-sm{height:auto}.form-group-sm .form-control{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.form-group-sm select.form-control{height:30px;line-height:30px}.form-group-sm select[multiple].form-control,.form-group-sm textarea.form-control{height:auto}.form-group-sm .form-control-static{height:30px;min-height:32px;padding:6px 10px;font-size:12px;line-height:1.5}.input-lg{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-lg{height:46px;line-height:46px}select[multiple].input-lg,textarea.input-lg{height:auto}.form-group-lg .form-control{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.form-group-lg select.form-control{height:46px;line-height:46px}.form-group-lg select[multiple].form-control,.form-group-lg textarea.form-control{height:auto}.form-group-lg .form-control-static{height:46px;min-height:38px;padding:11px 16px;font-size:18px;line-height:1.3333333}.has-feedback{position:relative}.has-feedback .form-control{padding-right:42.5px}.form-control-feedback{position:absolute;top:0;right:0;z-index:2;display:block;width:34px;height:34px;line-height:34px;text-align:center;pointer-events:none}.form-group-lg .form-control+.form-control-feedback,.input-group-lg+.form-control-feedback,.input-lg+.form-control-feedback{width:46px;height:46px;line-height:46px}.form-group-sm .form-control+.form-control-feedback,.input-group-sm+.form-control-feedback,.input-sm+.form-control-feedback{width:30px;height:30px;line-height:30px}.has-success .checkbox,.has-success .checkbox-inline,.has-success .control-label,.has-success .help-block,.has-success .radio,.has-success .radio-inline,.has-success.checkbox label,.has-success.checkbox-inline label,.has-success.radio label,.has-success.radio-inline label{color:#3c763d}.has-success .form-control{border-color:#3c763d;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-success .form-control:focus{border-color:#2b542c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168}.has-success .input-group-addon{color:#3c763d;background-color:#dff0d8;border-color:#3c763d}.has-success .form-control-feedback{color:#3c763d}.has-warning .checkbox,.has-warning .checkbox-inline,.has-warning .control-label,.has-warning .help-block,.has-warning .radio,.has-warning .radio-inline,.has-warning.checkbox label,.has-warning.checkbox-inline label,.has-warning.radio label,.has-warning.radio-inline label{color:#8a6d3b}.has-warning .form-control{border-color:#8a6d3b;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-warning .form-control:focus{border-color:#66512c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b}.has-warning .input-group-addon{color:#8a6d3b;background-color:#fcf8e3;border-color:#8a6d3b}.has-warning .form-control-feedback{color:#8a6d3b}.has-error .checkbox,.has-error .checkbox-inline,.has-error .control-label,.has-error .help-block,.has-error .radio,.has-error .radio-inline,.has-error.checkbox label,.has-error.checkbox-inline label,.has-error.radio label,.has-error.radio-inline label{color:#a94442}.has-error .form-control{border-color:#a94442;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-error .form-control:focus{border-color:#843534;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483}.has-error .input-group-addon{color:#a94442;background-color:#f2dede;border-color:#a94442}.has-error .form-control-feedback{color:#a94442}.has-feedback label~.form-control-feedback{top:25px}.has-feedback label.sr-only~.form-control-feedback{top:0}.help-block{display:block;margin-top:5px;margin-bottom:10px;color:#737373}@media (min-width:768px){.form-inline .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.form-inline .form-control{display:inline-block;width:auto;vertical-align:middle}.form-inline .form-control-static{display:inline-block}.form-inline .input-group{display:inline-table;vertical-align:middle}.form-inline .input-group .form-control,.form-inline .input-group .input-group-addon,.form-inline .input-group .input-group-btn{width:auto}.form-inline .input-group>.form-control{width:100%}.form-inline .control-label{margin-bottom:0;vertical-align:middle}.form-inline .checkbox,.form-inline .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.form-inline .checkbox label,.form-inline .radio label{padding-left:0}.form-inline .checkbox input[type=checkbox],.form-inline .radio input[type=radio]{position:relative;margin-left:0}.form-inline .has-feedback .form-control-feedback{top:0}}.form-horizontal .checkbox,.form-horizontal .checkbox-inline,.form-horizontal .radio,.form-horizontal .radio-inline{padding-top:7px;margin-top:0;margin-bottom:0}.form-horizontal .checkbox,.form-horizontal .radio{min-height:27px}.form-horizontal .form-group{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.form-horizontal .control-label{padding-top:7px;margin-bottom:0;text-align:right}}.form-horizontal .has-feedback .form-control-feedback{right:15px}@media (min-width:768px){.form-horizontal .form-group-lg .control-label{padding-top:11px;font-size:18px}}@media (min-width:768px){.form-horizontal .form-group-sm .control-label{padding-top:6px;font-size:12px}}.btn{display:inline-block;padding:6px 12px;margin-bottom:0;font-size:14px;font-weight:400;line-height:1.42857143;text-align:center;white-space:nowrap;vertical-align:middle;-ms-touch-action:manipulation;touch-action:manipulation;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;background-image:none;border:1px solid transparent;border-radius:4px}.btn.active.focus,.btn.active:focus,.btn.focus,.btn:active.focus,.btn:active:focus,.btn:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}.btn.focus,.btn:focus,.btn:hover{color:#333;text-decoration:none}.btn.active,.btn:active{background-image:none;outline:0;-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn.disabled,.btn[disabled],fieldset[disabled] .btn{cursor:not-allowed;filter:alpha(opacity=65);-webkit-box-shadow:none;box-shadow:none;opacity:.65}a.btn.disabled,fieldset[disabled] a.btn{pointer-events:none}.btn-default{color:#333;background-color:#fff;border-color:#ccc}.btn-default.focus,.btn-default:focus{color:#333;background-color:#e6e6e6;border-color:#8c8c8c}.btn-default:hover{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active.focus,.btn-default.active:focus,.btn-default.active:hover,.btn-default:active.focus,.btn-default:active:focus,.btn-default:active:hover,.open>.dropdown-toggle.btn-default.focus,.open>.dropdown-toggle.btn-default:focus,.open>.dropdown-toggle.btn-default:hover{color:#333;background-color:#d4d4d4;border-color:#8c8c8c}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{background-image:none}.btn-default.disabled.focus,.btn-default.disabled:focus,.btn-default.disabled:hover,.btn-default[disabled].focus,.btn-default[disabled]:focus,.btn-default[disabled]:hover,fieldset[disabled] .btn-default.focus,fieldset[disabled] .btn-default:focus,fieldset[disabled] .btn-default:hover{background-color:#fff;border-color:#ccc}.btn-default .badge{color:#fff;background-color:#333}.btn-primary{color:#fff;background-color:#337ab7;border-color:#2e6da4}.btn-primary.focus,.btn-primary:focus{color:#fff;background-color:#286090;border-color:#122b40}.btn-primary:hover{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active.focus,.btn-primary.active:focus,.btn-primary.active:hover,.btn-primary:active.focus,.btn-primary:active:focus,.btn-primary:active:hover,.open>.dropdown-toggle.btn-primary.focus,.open>.dropdown-toggle.btn-primary:focus,.open>.dropdown-toggle.btn-primary:hover{color:#fff;background-color:#204d74;border-color:#122b40}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{background-image:none}.btn-primary.disabled.focus,.btn-primary.disabled:focus,.btn-primary.disabled:hover,.btn-primary[disabled].focus,.btn-primary[disabled]:focus,.btn-primary[disabled]:hover,fieldset[disabled] .btn-primary.focus,fieldset[disabled] .btn-primary:focus,fieldset[disabled] .btn-primary:hover{background-color:#337ab7;border-color:#2e6da4}.btn-primary .badge{color:#337ab7;background-color:#fff}.btn-success{color:#fff;background-color:#5cb85c;border-color:#4cae4c}.btn-success.focus,.btn-success:focus{color:#fff;background-color:#449d44;border-color:#255625}.btn-success:hover{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active.focus,.btn-success.active:focus,.btn-success.active:hover,.btn-success:active.focus,.btn-success:active:focus,.btn-success:active:hover,.open>.dropdown-toggle.btn-success.focus,.open>.dropdown-toggle.btn-success:focus,.open>.dropdown-toggle.btn-success:hover{color:#fff;background-color:#398439;border-color:#255625}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{background-image:none}.btn-success.disabled.focus,.btn-success.disabled:focus,.btn-success.disabled:hover,.btn-success[disabled].focus,.btn-success[disabled]:focus,.btn-success[disabled]:hover,fieldset[disabled] .btn-success.focus,fieldset[disabled] .btn-success:focus,fieldset[disabled] .btn-success:hover{background-color:#5cb85c;border-color:#4cae4c}.btn-success .badge{color:#5cb85c;background-color:#fff}.btn-info{color:#fff;background-color:#5bc0de;border-color:#46b8da}.btn-info.focus,.btn-info:focus{color:#fff;background-color:#31b0d5;border-color:#1b6d85}.btn-info:hover{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active.focus,.btn-info.active:focus,.btn-info.active:hover,.btn-info:active.focus,.btn-info:active:focus,.btn-info:active:hover,.open>.dropdown-toggle.btn-info.focus,.open>.dropdown-toggle.btn-info:focus,.open>.dropdown-toggle.btn-info:hover{color:#fff;background-color:#269abc;border-color:#1b6d85}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{background-image:none}.btn-info.disabled.focus,.btn-info.disabled:focus,.btn-info.disabled:hover,.btn-info[disabled].focus,.btn-info[disabled]:focus,.btn-info[disabled]:hover,fieldset[disabled] .btn-info.focus,fieldset[disabled] .btn-info:focus,fieldset[disabled] .btn-info:hover{background-color:#5bc0de;border-color:#46b8da}.btn-info .badge{color:#5bc0de;background-color:#fff}.btn-warning{color:#fff;background-color:#f0ad4e;border-color:#eea236}.btn-warning.focus,.btn-warning:focus{color:#fff;background-color:#ec971f;border-color:#985f0d}.btn-warning:hover{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active.focus,.btn-warning.active:focus,.btn-warning.active:hover,.btn-warning:active.focus,.btn-warning:active:focus,.btn-warning:active:hover,.open>.dropdown-toggle.btn-warning.focus,.open>.dropdown-toggle.btn-warning:focus,.open>.dropdown-toggle.btn-warning:hover{color:#fff;background-color:#d58512;border-color:#985f0d}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{background-image:none}.btn-warning.disabled.focus,.btn-warning.disabled:focus,.btn-warning.disabled:hover,.btn-warning[disabled].focus,.btn-warning[disabled]:focus,.btn-warning[disabled]:hover,fieldset[disabled] .btn-warning.focus,fieldset[disabled] .btn-warning:focus,fieldset[disabled] .btn-warning:hover{background-color:#f0ad4e;border-color:#eea236}.btn-warning .badge{color:#f0ad4e;background-color:#fff}.btn-danger{color:#fff;background-color:#d9534f;border-color:#d43f3a}.btn-danger.focus,.btn-danger:focus{color:#fff;background-color:#c9302c;border-color:#761c19}.btn-danger:hover{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active.focus,.btn-danger.active:focus,.btn-danger.active:hover,.btn-danger:active.focus,.btn-danger:active:focus,.btn-danger:active:hover,.open>.dropdown-toggle.btn-danger.focus,.open>.dropdown-toggle.btn-danger:focus,.open>.dropdown-toggle.btn-danger:hover{color:#fff;background-color:#ac2925;border-color:#761c19}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{background-image:none}.btn-danger.disabled.focus,.btn-danger.disabled:focus,.btn-danger.disabled:hover,.btn-danger[disabled].focus,.btn-danger[disabled]:focus,.btn-danger[disabled]:hover,fieldset[disabled] .btn-danger.focus,fieldset[disabled] .btn-danger:focus,fieldset[disabled] .btn-danger:hover{background-color:#d9534f;border-color:#d43f3a}.btn-danger .badge{color:#d9534f;background-color:#fff}.btn-link{font-weight:400;color:#337ab7;border-radius:0}.btn-link,.btn-link.active,.btn-link:active,.btn-link[disabled],fieldset[disabled] .btn-link{background-color:transparent;-webkit-box-shadow:none;box-shadow:none}.btn-link,.btn-link:active,.btn-link:focus,.btn-link:hover{border-color:transparent}.btn-link:focus,.btn-link:hover{color:#23527c;text-decoration:underline;background-color:transparent}.btn-link[disabled]:focus,.btn-link[disabled]:hover,fieldset[disabled] .btn-link:focus,fieldset[disabled] .btn-link:hover{color:#777;text-decoration:none}.btn-group-lg>.btn,.btn-lg{padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.btn-group-sm>.btn,.btn-sm{padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.btn-group-xs>.btn,.btn-xs{padding:1px 5px;font-size:12px;line-height:1.5;border-radius:3px}.btn-block{display:block;width:100%}.btn-block+.btn-block{margin-top:5px}input[type=button].btn-block,input[type=reset].btn-block,input[type=submit].btn-block{width:100%}.fade{opacity:0;-webkit-transition:opacity .15s linear;-o-transition:opacity .15s linear;transition:opacity .15s linear}.fade.in{opacity:1}.collapse{display:none}.collapse.in{display:block}tr.collapse.in{display:table-row}tbody.collapse.in{display:table-row-group}.collapsing{position:relative;height:0;overflow:hidden;-webkit-transition-timing-function:ease;-o-transition-timing-function:ease;transition-timing-function:ease;-webkit-transition-duration:.35s;-o-transition-duration:.35s;transition-duration:.35s;-webkit-transition-property:height,visibility;-o-transition-property:height,visibility;transition-property:height,visibility}.caret{display:inline-block;width:0;height:0;margin-left:2px;vertical-align:middle;border-top:4px dashed;border-top:4px solid\9;border-right:4px solid transparent;border-left:4px solid transparent}.dropdown,.dropup{position:relative}.dropdown-toggle:focus{outline:0}.dropdown-menu{position:absolute;top:100%;left:0;z-index:1000;display:none;float:left;min-width:160px;padding:5px 0;margin:2px 0 0;font-size:14px;text-align:left;list-style:none;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.15);border-radius:4px;-webkit-box-shadow:0 6px 12px rgba(0,0,0,.175);box-shadow:0 6px 12px rgba(0,0,0,.175)}.dropdown-menu.pull-right{right:0;left:auto}.dropdown-menu .divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.dropdown-menu>li>a{display:block;padding:3px 20px;clear:both;font-weight:400;line-height:1.42857143;color:#333;white-space:nowrap}.dropdown-menu>li>a:focus,.dropdown-menu>li>a:hover{color:#262626;text-decoration:none;background-color:#f5f5f5}.dropdown-menu>.active>a,.dropdown-menu>.active>a:focus,.dropdown-menu>.active>a:hover{color:#fff;text-decoration:none;background-color:#337ab7;outline:0}.dropdown-menu>.disabled>a,.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{color:#777}.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{text-decoration:none;cursor:not-allowed;background-color:transparent;background-image:none;filter:progid:DXImageTransform.Microsoft.gradient(enabled=false)}.open>.dropdown-menu{display:block}.open>a{outline:0}.dropdown-menu-right{right:0;left:auto}.dropdown-menu-left{right:auto;left:0}.dropdown-header{display:block;padding:3px 20px;font-size:12px;line-height:1.42857143;color:#777;white-space:nowrap}.dropdown-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:990}.pull-right>.dropdown-menu{right:0;left:auto}.dropup .caret,.navbar-fixed-bottom .dropdown .caret{content:"";border-top:0;border-bottom:4px dashed;border-bottom:4px solid\9}.dropup .dropdown-menu,.navbar-fixed-bottom .dropdown .dropdown-menu{top:auto;bottom:100%;margin-bottom:2px}@media (min-width:768px){.navbar-right .dropdown-menu{right:0;left:auto}.navbar-right .dropdown-menu-left{right:auto;left:0}}.btn-group,.btn-group-vertical{position:relative;display:inline-block;vertical-align:middle}.btn-group-vertical>.btn,.btn-group>.btn{position:relative;float:left}.btn-group-vertical>.btn.active,.btn-group-vertical>.btn:active,.btn-group-vertical>.btn:focus,.btn-group-vertical>.btn:hover,.btn-group>.btn.active,.btn-group>.btn:active,.btn-group>.btn:focus,.btn-group>.btn:hover{z-index:2}.btn-group .btn+.btn,.btn-group .btn+.btn-group,.btn-group .btn-group+.btn,.btn-group .btn-group+.btn-group{margin-left:-1px}.btn-toolbar{margin-left:-5px}.btn-toolbar .btn,.btn-toolbar .btn-group,.btn-toolbar .input-group{float:left}.btn-toolbar>.btn,.btn-toolbar>.btn-group,.btn-toolbar>.input-group{margin-left:5px}.btn-group>.btn:not(:first-child):not(:last-child):not(.dropdown-toggle){border-radius:0}.btn-group>.btn:first-child{margin-left:0}.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn:last-child:not(:first-child),.btn-group>.dropdown-toggle:not(:first-child){border-top-left-radius:0;border-bottom-left-radius:0}.btn-group>.btn-group{float:left}.btn-group>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-bottom-left-radius:0}.btn-group .dropdown-toggle:active,.btn-group.open .dropdown-toggle{outline:0}.btn-group>.btn+.dropdown-toggle{padding-right:8px;padding-left:8px}.btn-group>.btn-lg+.dropdown-toggle{padding-right:12px;padding-left:12px}.btn-group.open .dropdown-toggle{-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn-group.open .dropdown-toggle.btn-link{-webkit-box-shadow:none;box-shadow:none}.btn .caret{margin-left:0}.btn-lg .caret{border-width:5px 5px 0;border-bottom-width:0}.dropup .btn-lg .caret{border-width:0 5px 5px}.btn-group-vertical>.btn,.btn-group-vertical>.btn-group,.btn-group-vertical>.btn-group>.btn{display:block;float:none;width:100%;max-width:100%}.btn-group-vertical>.btn-group>.btn{float:none}.btn-group-vertical>.btn+.btn,.btn-group-vertical>.btn+.btn-group,.btn-group-vertical>.btn-group+.btn,.btn-group-vertical>.btn-group+.btn-group{margin-top:-1px;margin-left:0}.btn-group-vertical>.btn:not(:first-child):not(:last-child){border-radius:0}.btn-group-vertical>.btn:first-child:not(:last-child){border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn:last-child:not(:first-child){border-top-left-radius:0;border-top-right-radius:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}.btn-group-vertical>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group-vertical>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group-vertical>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-top-right-radius:0}.btn-group-justified{display:table;width:100%;table-layout:fixed;border-collapse:separate}.btn-group-justified>.btn,.btn-group-justified>.btn-group{display:table-cell;float:none;width:1%}.btn-group-justified>.btn-group .btn{width:100%}.btn-group-justified>.btn-group .dropdown-menu{left:auto}[data-toggle=buttons]>.btn input[type=checkbox],[data-toggle=buttons]>.btn input[type=radio],[data-toggle=buttons]>.btn-group>.btn input[type=checkbox],[data-toggle=buttons]>.btn-group>.btn input[type=radio]{position:absolute;clip:rect(0,0,0,0);pointer-events:none}.input-group{position:relative;display:table;border-collapse:separate}.input-group[class*=col-]{float:none;padding-right:0;padding-left:0}.input-group .form-control{position:relative;z-index:2;float:left;width:100%;margin-bottom:0}.input-group .form-control:focus{z-index:3}.input-group-lg>.form-control,.input-group-lg>.input-group-addon,.input-group-lg>.input-group-btn>.btn{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-group-lg>.form-control,select.input-group-lg>.input-group-addon,select.input-group-lg>.input-group-btn>.btn{height:46px;line-height:46px}select[multiple].input-group-lg>.form-control,select[multiple].input-group-lg>.input-group-addon,select[multiple].input-group-lg>.input-group-btn>.btn,textarea.input-group-lg>.form-control,textarea.input-group-lg>.input-group-addon,textarea.input-group-lg>.input-group-btn>.btn{height:auto}.input-group-sm>.form-control,.input-group-sm>.input-group-addon,.input-group-sm>.input-group-btn>.btn{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-group-sm>.form-control,select.input-group-sm>.input-group-addon,select.input-group-sm>.input-group-btn>.btn{height:30px;line-height:30px}select[multiple].input-group-sm>.form-control,select[multiple].input-group-sm>.input-group-addon,select[multiple].input-group-sm>.input-group-btn>.btn,textarea.input-group-sm>.form-control,textarea.input-group-sm>.input-group-addon,textarea.input-group-sm>.input-group-btn>.btn{height:auto}.input-group .form-control,.input-group-addon,.input-group-btn{display:table-cell}.input-group .form-control:not(:first-child):not(:last-child),.input-group-addon:not(:first-child):not(:last-child),.input-group-btn:not(:first-child):not(:last-child){border-radius:0}.input-group-addon,.input-group-btn{width:1%;white-space:nowrap;vertical-align:middle}.input-group-addon{padding:6px 12px;font-size:14px;font-weight:400;line-height:1;color:#555;text-align:center;background-color:#eee;border:1px solid #ccc;border-radius:4px}.input-group-addon.input-sm{padding:5px 10px;font-size:12px;border-radius:3px}.input-group-addon.input-lg{padding:10px 16px;font-size:18px;border-radius:6px}.input-group-addon input[type=checkbox],.input-group-addon input[type=radio]{margin-top:0}.input-group .form-control:first-child,.input-group-addon:first-child,.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group>.btn,.input-group-btn:first-child>.dropdown-toggle,.input-group-btn:last-child>.btn-group:not(:last-child)>.btn,.input-group-btn:last-child>.btn:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.input-group-addon:first-child{border-right:0}.input-group .form-control:last-child,.input-group-addon:last-child,.input-group-btn:first-child>.btn-group:not(:first-child)>.btn,.input-group-btn:first-child>.btn:not(:first-child),.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group>.btn,.input-group-btn:last-child>.dropdown-toggle{border-top-left-radius:0;border-bottom-left-radius:0}.input-group-addon:last-child{border-left:0}.input-group-btn{position:relative;font-size:0;white-space:nowrap}.input-group-btn>.btn{position:relative}.input-group-btn>.btn+.btn{margin-left:-1px}.input-group-btn>.btn:active,.input-group-btn>.btn:focus,.input-group-btn>.btn:hover{z-index:2}.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group{margin-right:-1px}.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group{z-index:2;margin-left:-1px}.nav{padding-left:0;margin-bottom:0;list-style:none}.nav>li{position:relative;display:block}.nav>li>a{position:relative;display:block;padding:10px 15px}.nav>li>a:focus,.nav>li>a:hover{text-decoration:none;background-color:#eee}.nav>li.disabled>a{color:#777}.nav>li.disabled>a:focus,.nav>li.disabled>a:hover{color:#777;text-decoration:none;cursor:not-allowed;background-color:transparent}.nav .open>a,.nav .open>a:focus,.nav .open>a:hover{background-color:#eee;border-color:#337ab7}.nav .nav-divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.nav>li>a>img{max-width:none}.nav-tabs{border-bottom:1px solid #ddd}.nav-tabs>li{float:left;margin-bottom:-1px}.nav-tabs>li>a{margin-right:2px;line-height:1.42857143;border:1px solid transparent;border-radius:4px 4px 0 0}.nav-tabs>li>a:hover{border-color:#eee #eee #ddd}.nav-tabs>li.active>a,.nav-tabs>li.active>a:focus,.nav-tabs>li.active>a:hover{color:#555;cursor:default;background-color:#fff;border:1px solid #ddd;border-bottom-color:transparent}.nav-tabs.nav-justified{width:100%;border-bottom:0}.nav-tabs.nav-justified>li{float:none}.nav-tabs.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-tabs.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-tabs.nav-justified>li{display:table-cell;width:1%}.nav-tabs.nav-justified>li>a{margin-bottom:0}}.nav-tabs.nav-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs.nav-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border-bottom-color:#fff}}.nav-pills>li{float:left}.nav-pills>li>a{border-radius:4px}.nav-pills>li+li{margin-left:2px}.nav-pills>li.active>a,.nav-pills>li.active>a:focus,.nav-pills>li.active>a:hover{color:#fff;background-color:#337ab7}.nav-stacked>li{float:none}.nav-stacked>li+li{margin-top:2px;margin-left:0}.nav-justified{width:100%}.nav-justified>li{float:none}.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-justified>li{display:table-cell;width:1%}.nav-justified>li>a{margin-bottom:0}}.nav-tabs-justified{border-bottom:0}.nav-tabs-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border-bottom-color:#fff}}.tab-content>.tab-pane{display:none}.tab-content>.active{display:block}.nav-tabs .dropdown-menu{margin-top:-1px;border-top-left-radius:0;border-top-right-radius:0}.navbar{position:relative;min-height:50px;margin-bottom:20px;border:1px solid transparent}@media (min-width:768px){.navbar{border-radius:4px}}@media (min-width:768px){.navbar-header{float:left}}.navbar-collapse{padding-right:15px;padding-left:15px;overflow-x:visible;-webkit-overflow-scrolling:touch;border-top:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1)}.navbar-collapse.in{overflow-y:auto}@media (min-width:768px){.navbar-collapse{width:auto;border-top:0;-webkit-box-shadow:none;box-shadow:none}.navbar-collapse.collapse{display:block!important;height:auto!important;padding-bottom:0;overflow:visible!important}.navbar-collapse.in{overflow-y:visible}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse,.navbar-static-top .navbar-collapse{padding-right:0;padding-left:0}}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:340px}@media (max-device-width:480px) and (orientation:landscape){.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:200px}}.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:0;margin-left:0}}.navbar-static-top{z-index:1000;border-width:0 0 1px}@media (min-width:768px){.navbar-static-top{border-radius:0}}.navbar-fixed-bottom,.navbar-fixed-top{position:fixed;right:0;left:0;z-index:1030}@media (min-width:768px){.navbar-fixed-bottom,.navbar-fixed-top{border-radius:0}}.navbar-fixed-top{top:0;border-width:0 0 1px}.navbar-fixed-bottom{bottom:0;margin-bottom:0;border-width:1px 0 0}.navbar-brand{float:left;height:50px;padding:15px 15px;font-size:18px;line-height:20px}.navbar-brand:focus,.navbar-brand:hover{text-decoration:none}.navbar-brand>img{display:block}@media (min-width:768px){.navbar>.container .navbar-brand,.navbar>.container-fluid .navbar-brand{margin-left:-15px}}.navbar-toggle{position:relative;float:right;padding:9px 10px;margin-top:8px;margin-right:15px;margin-bottom:8px;background-color:transparent;background-image:none;border:1px solid transparent;border-radius:4px}.navbar-toggle:focus{outline:0}.navbar-toggle .icon-bar{display:block;width:22px;height:2px;border-radius:1px}.navbar-toggle .icon-bar+.icon-bar{margin-top:4px}@media (min-width:768px){.navbar-toggle{display:none}}.navbar-nav{margin:7.5px -15px}.navbar-nav>li>a{padding-top:10px;padding-bottom:10px;line-height:20px}@media (max-width:767px){.navbar-nav .open .dropdown-menu{position:static;float:none;width:auto;margin-top:0;background-color:transparent;border:0;-webkit-box-shadow:none;box-shadow:none}.navbar-nav .open .dropdown-menu .dropdown-header,.navbar-nav .open .dropdown-menu>li>a{padding:5px 15px 5px 25px}.navbar-nav .open .dropdown-menu>li>a{line-height:20px}.navbar-nav .open .dropdown-menu>li>a:focus,.navbar-nav .open .dropdown-menu>li>a:hover{background-image:none}}@media (min-width:768px){.navbar-nav{float:left;margin:0}.navbar-nav>li{float:left}.navbar-nav>li>a{padding-top:15px;padding-bottom:15px}}.navbar-form{padding:10px 15px;margin-top:8px;margin-right:-15px;margin-bottom:8px;margin-left:-15px;border-top:1px solid transparent;border-bottom:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1)}@media (min-width:768px){.navbar-form .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.navbar-form .form-control{display:inline-block;width:auto;vertical-align:middle}.navbar-form .form-control-static{display:inline-block}.navbar-form .input-group{display:inline-table;vertical-align:middle}.navbar-form .input-group .form-control,.navbar-form .input-group .input-group-addon,.navbar-form .input-group .input-group-btn{width:auto}.navbar-form .input-group>.form-control{width:100%}.navbar-form .control-label{margin-bottom:0;vertical-align:middle}.navbar-form .checkbox,.navbar-form .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.navbar-form .checkbox label,.navbar-form .radio label{padding-left:0}.navbar-form .checkbox input[type=checkbox],.navbar-form .radio input[type=radio]{position:relative;margin-left:0}.navbar-form .has-feedback .form-control-feedback{top:0}}@media (max-width:767px){.navbar-form .form-group{margin-bottom:5px}.navbar-form .form-group:last-child{margin-bottom:0}}@media (min-width:768px){.navbar-form{width:auto;padding-top:0;padding-bottom:0;margin-right:0;margin-left:0;border:0;-webkit-box-shadow:none;box-shadow:none}}.navbar-nav>li>.dropdown-menu{margin-top:0;border-top-left-radius:0;border-top-right-radius:0}.navbar-fixed-bottom .navbar-nav>li>.dropdown-menu{margin-bottom:0;border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.navbar-btn{margin-top:8px;margin-bottom:8px}.navbar-btn.btn-sm{margin-top:10px;margin-bottom:10px}.navbar-btn.btn-xs{margin-top:14px;margin-bottom:14px}.navbar-text{margin-top:15px;margin-bottom:15px}@media (min-width:768px){.navbar-text{float:left;margin-right:15px;margin-left:15px}}@media (min-width:768px){.navbar-left{float:left!important}.navbar-right{float:right!important;margin-right:-15px}.navbar-right~.navbar-right{margin-right:0}}.navbar-default{background-color:#f8f8f8;border-color:#e7e7e7}.navbar-default .navbar-brand{color:#777}.navbar-default .navbar-brand:focus,.navbar-default .navbar-brand:hover{color:#5e5e5e;background-color:transparent}.navbar-default .navbar-text{color:#777}.navbar-default .navbar-nav>li>a{color:#777}.navbar-default .navbar-nav>li>a:focus,.navbar-default .navbar-nav>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav>.active>a,.navbar-default .navbar-nav>.active>a:focus,.navbar-default .navbar-nav>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav>.disabled>a,.navbar-default .navbar-nav>.disabled>a:focus,.navbar-default .navbar-nav>.disabled>a:hover{color:#ccc;background-color:transparent}.navbar-default .navbar-toggle{border-color:#ddd}.navbar-default .navbar-toggle:focus,.navbar-default .navbar-toggle:hover{background-color:#ddd}.navbar-default .navbar-toggle .icon-bar{background-color:#888}.navbar-default .navbar-collapse,.navbar-default .navbar-form{border-color:#e7e7e7}.navbar-default .navbar-nav>.open>a,.navbar-default .navbar-nav>.open>a:focus,.navbar-default .navbar-nav>.open>a:hover{color:#555;background-color:#e7e7e7}@media (max-width:767px){.navbar-default .navbar-nav .open .dropdown-menu>li>a{color:#777}.navbar-default .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav .open .dropdown-menu>.active>a,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#ccc;background-color:transparent}}.navbar-default .navbar-link{color:#777}.navbar-default .navbar-link:hover{color:#333}.navbar-default .btn-link{color:#777}.navbar-default .btn-link:focus,.navbar-default .btn-link:hover{color:#333}.navbar-default .btn-link[disabled]:focus,.navbar-default .btn-link[disabled]:hover,fieldset[disabled] .navbar-default .btn-link:focus,fieldset[disabled] .navbar-default .btn-link:hover{color:#ccc}.navbar-inverse{background-color:#222;border-color:#080808}.navbar-inverse .navbar-brand{color:#9d9d9d}.navbar-inverse .navbar-brand:focus,.navbar-inverse .navbar-brand:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-text{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a:focus,.navbar-inverse .navbar-nav>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav>.active>a,.navbar-inverse .navbar-nav>.active>a:focus,.navbar-inverse .navbar-nav>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav>.disabled>a,.navbar-inverse .navbar-nav>.disabled>a:focus,.navbar-inverse .navbar-nav>.disabled>a:hover{color:#444;background-color:transparent}.navbar-inverse .navbar-toggle{border-color:#333}.navbar-inverse .navbar-toggle:focus,.navbar-inverse .navbar-toggle:hover{background-color:#333}.navbar-inverse .navbar-toggle .icon-bar{background-color:#fff}.navbar-inverse .navbar-collapse,.navbar-inverse .navbar-form{border-color:#101010}.navbar-inverse .navbar-nav>.open>a,.navbar-inverse .navbar-nav>.open>a:focus,.navbar-inverse .navbar-nav>.open>a:hover{color:#fff;background-color:#080808}@media (max-width:767px){.navbar-inverse .navbar-nav .open .dropdown-menu>.dropdown-header{border-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu .divider{background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#444;background-color:transparent}}.navbar-inverse .navbar-link{color:#9d9d9d}.navbar-inverse .navbar-link:hover{color:#fff}.navbar-inverse .btn-link{color:#9d9d9d}.navbar-inverse .btn-link:focus,.navbar-inverse .btn-link:hover{color:#fff}.navbar-inverse .btn-link[disabled]:focus,.navbar-inverse .btn-link[disabled]:hover,fieldset[disabled] .navbar-inverse .btn-link:focus,fieldset[disabled] .navbar-inverse .btn-link:hover{color:#444}.breadcrumb{padding:8px 15px;margin-bottom:20px;list-style:none;background-color:#f5f5f5;border-radius:4px}.breadcrumb>li{display:inline-block}.breadcrumb>li+li:before{padding:0 5px;color:#ccc;content:"/\00a0"}.breadcrumb>.active{color:#777}.pagination{display:inline-block;padding-left:0;margin:20px 0;border-radius:4px}.pagination>li{display:inline}.pagination>li>a,.pagination>li>span{position:relative;float:left;padding:6px 12px;margin-left:-1px;line-height:1.42857143;color:#337ab7;text-decoration:none;background-color:#fff;border:1px solid #ddd}.pagination>li:first-child>a,.pagination>li:first-child>span{margin-left:0;border-top-left-radius:4px;border-bottom-left-radius:4px}.pagination>li:last-child>a,.pagination>li:last-child>span{border-top-right-radius:4px;border-bottom-right-radius:4px}.pagination>li>a:focus,.pagination>li>a:hover,.pagination>li>span:focus,.pagination>li>span:hover{z-index:2;color:#23527c;background-color:#eee;border-color:#ddd}.pagination>.active>a,.pagination>.active>a:focus,.pagination>.active>a:hover,.pagination>.active>span,.pagination>.active>span:focus,.pagination>.active>span:hover{z-index:3;color:#fff;cursor:default;background-color:#337ab7;border-color:#337ab7}.pagination>.disabled>a,.pagination>.disabled>a:focus,.pagination>.disabled>a:hover,.pagination>.disabled>span,.pagination>.disabled>span:focus,.pagination>.disabled>span:hover{color:#777;cursor:not-allowed;background-color:#fff;border-color:#ddd}.pagination-lg>li>a,.pagination-lg>li>span{padding:10px 16px;font-size:18px;line-height:1.3333333}.pagination-lg>li:first-child>a,.pagination-lg>li:first-child>span{border-top-left-radius:6px;border-bottom-left-radius:6px}.pagination-lg>li:last-child>a,.pagination-lg>li:last-child>span{border-top-right-radius:6px;border-bottom-right-radius:6px}.pagination-sm>li>a,.pagination-sm>li>span{padding:5px 10px;font-size:12px;line-height:1.5}.pagination-sm>li:first-child>a,.pagination-sm>li:first-child>span{border-top-left-radius:3px;border-bottom-left-radius:3px}.pagination-sm>li:last-child>a,.pagination-sm>li:last-child>span{border-top-right-radius:3px;border-bottom-right-radius:3px}.pager{padding-left:0;margin:20px 0;text-align:center;list-style:none}.pager li{display:inline}.pager li>a,.pager li>span{display:inline-block;padding:5px 14px;background-color:#fff;border:1px solid #ddd;border-radius:15px}.pager li>a:focus,.pager li>a:hover{text-decoration:none;background-color:#eee}.pager .next>a,.pager .next>span{float:right}.pager .previous>a,.pager .previous>span{float:left}.pager .disabled>a,.pager .disabled>a:focus,.pager .disabled>a:hover,.pager .disabled>span{color:#777;cursor:not-allowed;background-color:#fff}.label{display:inline;padding:.2em .6em .3em;font-size:75%;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:baseline;border-radius:.25em}a.label:focus,a.label:hover{color:#fff;text-decoration:none;cursor:pointer}.label:empty{display:none}.btn .label{position:relative;top:-1px}.label-default{background-color:#777}.label-default[href]:focus,.label-default[href]:hover{background-color:#5e5e5e}.label-primary{background-color:#337ab7}.label-primary[href]:focus,.label-primary[href]:hover{background-color:#286090}.label-success{background-color:#5cb85c}.label-success[href]:focus,.label-success[href]:hover{background-color:#449d44}.label-info{background-color:#5bc0de}.label-info[href]:focus,.label-info[href]:hover{background-color:#31b0d5}.label-warning{background-color:#f0ad4e}.label-warning[href]:focus,.label-warning[href]:hover{background-color:#ec971f}.label-danger{background-color:#d9534f}.label-danger[href]:focus,.label-danger[href]:hover{background-color:#c9302c}.badge{display:inline-block;min-width:10px;padding:3px 7px;font-size:12px;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:middle;background-color:#777;border-radius:10px}.badge:empty{display:none}.btn .badge{position:relative;top:-1px}.btn-group-xs>.btn .badge,.btn-xs .badge{top:0;padding:1px 5px}a.badge:focus,a.badge:hover{color:#fff;text-decoration:none;cursor:pointer}.list-group-item.active>.badge,.nav-pills>.active>a>.badge{color:#337ab7;background-color:#fff}.list-group-item>.badge{float:right}.list-group-item>.badge+.badge{margin-right:5px}.nav-pills>li>a>.badge{margin-left:3px}.jumbotron{padding-top:30px;padding-bottom:30px;margin-bottom:30px;color:inherit;background-color:#eee}.jumbotron .h1,.jumbotron h1{color:inherit}.jumbotron p{margin-bottom:15px;font-size:21px;font-weight:200}.jumbotron>hr{border-top-color:#d5d5d5}.container .jumbotron,.container-fluid .jumbotron{padding-right:15px;padding-left:15px;border-radius:6px}.jumbotron .container{max-width:100%}@media screen and (min-width:768px){.jumbotron{padding-top:48px;padding-bottom:48px}.container .jumbotron,.container-fluid .jumbotron{padding-right:60px;padding-left:60px}.jumbotron .h1,.jumbotron h1{font-size:63px}}.thumbnail{display:block;padding:4px;margin-bottom:20px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:border .2s ease-in-out;-o-transition:border .2s ease-in-out;transition:border .2s ease-in-out}.thumbnail a>img,.thumbnail>img{margin-right:auto;margin-left:auto}a.thumbnail.active,a.thumbnail:focus,a.thumbnail:hover{border-color:#337ab7}.thumbnail .caption{padding:9px;color:#333}.alert{padding:15px;margin-bottom:20px;border:1px solid transparent;border-radius:4px}.alert h4{margin-top:0;color:inherit}.alert .alert-link{font-weight:700}.alert>p,.alert>ul{margin-bottom:0}.alert>p+p{margin-top:5px}.alert-dismissable,.alert-dismissible{padding-right:35px}.alert-dismissable .close,.alert-dismissible .close{position:relative;top:-2px;right:-21px;color:inherit}.alert-success{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.alert-success hr{border-top-color:#c9e2b3}.alert-success .alert-link{color:#2b542c}.alert-info{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.alert-info hr{border-top-color:#a6e1ec}.alert-info .alert-link{color:#245269}.alert-warning{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.alert-warning hr{border-top-color:#f7e1b5}.alert-warning .alert-link{color:#66512c}.alert-danger{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.alert-danger hr{border-top-color:#e4b9c0}.alert-danger .alert-link{color:#843534}@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@-o-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}.progress{height:20px;margin-bottom:20px;overflow:hidden;background-color:#f5f5f5;border-radius:4px;-webkit-box-shadow:inset 0 1px 2px rgba(0,0,0,.1);box-shadow:inset 0 1px 2px rgba(0,0,0,.1)}.progress-bar{float:left;width:0;height:100%;font-size:12px;line-height:20px;color:#fff;text-align:center;background-color:#337ab7;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);-webkit-transition:width .6s ease;-o-transition:width .6s ease;transition:width .6s ease}.progress-bar-striped,.progress-striped .progress-bar{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);-webkit-background-size:40px 40px;background-size:40px 40px}.progress-bar.active,.progress.active .progress-bar{-webkit-animation:progress-bar-stripes 2s linear infinite;-o-animation:progress-bar-stripes 2s linear infinite;animation:progress-bar-stripes 2s linear infinite}.progress-bar-success{background-color:#5cb85c}.progress-striped .progress-bar-success{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-info{background-color:#5bc0de}.progress-striped .progress-bar-info{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-warning{background-color:#f0ad4e}.progress-striped .progress-bar-warning{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-danger{background-color:#d9534f}.progress-striped .progress-bar-danger{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.media{margin-top:15px}.media:first-child{margin-top:0}.media,.media-body{overflow:hidden;zoom:1}.media-body{width:10000px}.media-object{display:block}.media-object.img-thumbnail{max-width:none}.media-right,.media>.pull-right{padding-left:10px}.media-left,.media>.pull-left{padding-right:10px}.media-body,.media-left,.media-right{display:table-cell;vertical-align:top}.media-middle{vertical-align:middle}.media-bottom{vertical-align:bottom}.media-heading{margin-top:0;margin-bottom:5px}.media-list{padding-left:0;list-style:none}.list-group{padding-left:0;margin-bottom:20px}.list-group-item{position:relative;display:block;padding:10px 15px;margin-bottom:-1px;background-color:#fff;border:1px solid #ddd}.list-group-item:first-child{border-top-left-radius:4px;border-top-right-radius:4px}.list-group-item:last-child{margin-bottom:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}a.list-group-item,button.list-group-item{color:#555}a.list-group-item .list-group-item-heading,button.list-group-item .list-group-item-heading{color:#333}a.list-group-item:focus,a.list-group-item:hover,button.list-group-item:focus,button.list-group-item:hover{color:#555;text-decoration:none;background-color:#f5f5f5}button.list-group-item{width:100%;text-align:left}.list-group-item.disabled,.list-group-item.disabled:focus,.list-group-item.disabled:hover{color:#777;cursor:not-allowed;background-color:#eee}.list-group-item.disabled .list-group-item-heading,.list-group-item.disabled:focus .list-group-item-heading,.list-group-item.disabled:hover .list-group-item-heading{color:inherit}.list-group-item.disabled .list-group-item-text,.list-group-item.disabled:focus .list-group-item-text,.list-group-item.disabled:hover .list-group-item-text{color:#777}.list-group-item.active,.list-group-item.active:focus,.list-group-item.active:hover{z-index:2;color:#fff;background-color:#337ab7;border-color:#337ab7}.list-group-item.active .list-group-item-heading,.list-group-item.active .list-group-item-heading>.small,.list-group-item.active .list-group-item-heading>small,.list-group-item.active:focus .list-group-item-heading,.list-group-item.active:focus .list-group-item-heading>.small,.list-group-item.active:focus .list-group-item-heading>small,.list-group-item.active:hover .list-group-item-heading,.list-group-item.active:hover .list-group-item-heading>.small,.list-group-item.active:hover .list-group-item-heading>small{color:inherit}.list-group-item.active .list-group-item-text,.list-group-item.active:focus .list-group-item-text,.list-group-item.active:hover .list-group-item-text{color:#c7ddef}.list-group-item-success{color:#3c763d;background-color:#dff0d8}a.list-group-item-success,button.list-group-item-success{color:#3c763d}a.list-group-item-success .list-group-item-heading,button.list-group-item-success .list-group-item-heading{color:inherit}a.list-group-item-success:focus,a.list-group-item-success:hover,button.list-group-item-success:focus,button.list-group-item-success:hover{color:#3c763d;background-color:#d0e9c6}a.list-group-item-success.active,a.list-group-item-success.active:focus,a.list-group-item-success.active:hover,button.list-group-item-success.active,button.list-group-item-success.active:focus,button.list-group-item-success.active:hover{color:#fff;background-color:#3c763d;border-color:#3c763d}.list-group-item-info{color:#31708f;background-color:#d9edf7}a.list-group-item-info,button.list-group-item-info{color:#31708f}a.list-group-item-info .list-group-item-heading,button.list-group-item-info .list-group-item-heading{color:inherit}a.list-group-item-info:focus,a.list-group-item-info:hover,button.list-group-item-info:focus,button.list-group-item-info:hover{color:#31708f;background-color:#c4e3f3}a.list-group-item-info.active,a.list-group-item-info.active:focus,a.list-group-item-info.active:hover,button.list-group-item-info.active,button.list-group-item-info.active:focus,button.list-group-item-info.active:hover{color:#fff;background-color:#31708f;border-color:#31708f}.list-group-item-warning{color:#8a6d3b;background-color:#fcf8e3}a.list-group-item-warning,button.list-group-item-warning{color:#8a6d3b}a.list-group-item-warning .list-group-item-heading,button.list-group-item-warning .list-group-item-heading{color:inherit}a.list-group-item-warning:focus,a.list-group-item-warning:hover,button.list-group-item-warning:focus,button.list-group-item-warning:hover{color:#8a6d3b;background-color:#faf2cc}a.list-group-item-warning.active,a.list-group-item-warning.active:focus,a.list-group-item-warning.active:hover,button.list-group-item-warning.active,button.list-group-item-warning.active:focus,button.list-group-item-warning.active:hover{color:#fff;background-color:#8a6d3b;border-color:#8a6d3b}.list-group-item-danger{color:#a94442;background-color:#f2dede}a.list-group-item-danger,button.list-group-item-danger{color:#a94442}a.list-group-item-danger .list-group-item-heading,button.list-group-item-danger .list-group-item-heading{color:inherit}a.list-group-item-danger:focus,a.list-group-item-danger:hover,button.list-group-item-danger:focus,button.list-group-item-danger:hover{color:#a94442;background-color:#ebcccc}a.list-group-item-danger.active,a.list-group-item-danger.active:focus,a.list-group-item-danger.active:hover,button.list-group-item-danger.active,button.list-group-item-danger.active:focus,button.list-group-item-danger.active:hover{color:#fff;background-color:#a94442;border-color:#a94442}.list-group-item-heading{margin-top:0;margin-bottom:5px}.list-group-item-text{margin-bottom:0;line-height:1.3}.panel{margin-bottom:20px;background-color:#fff;border:1px solid transparent;border-radius:4px;-webkit-box-shadow:0 1px 1px rgba(0,0,0,.05);box-shadow:0 1px 1px rgba(0,0,0,.05)}.panel-body{padding:15px}.panel-heading{padding:10px 15px;border-bottom:1px solid transparent;border-top-left-radius:3px;border-top-right-radius:3px}.panel-heading>.dropdown .dropdown-toggle{color:inherit}.panel-title{margin-top:0;margin-bottom:0;font-size:16px;color:inherit}.panel-title>.small,.panel-title>.small>a,.panel-title>a,.panel-title>small,.panel-title>small>a{color:inherit}.panel-footer{padding:10px 15px;background-color:#f5f5f5;border-top:1px solid #ddd;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.list-group,.panel>.panel-collapse>.list-group{margin-bottom:0}.panel>.list-group .list-group-item,.panel>.panel-collapse>.list-group .list-group-item{border-width:1px 0;border-radius:0}.panel>.list-group:first-child .list-group-item:first-child,.panel>.panel-collapse>.list-group:first-child .list-group-item:first-child{border-top:0;border-top-left-radius:3px;border-top-right-radius:3px}.panel>.list-group:last-child .list-group-item:last-child,.panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{border-bottom:0;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.panel-heading+.panel-collapse>.list-group .list-group-item:first-child{border-top-left-radius:0;border-top-right-radius:0}.panel-heading+.list-group .list-group-item:first-child{border-top-width:0}.list-group+.panel-footer{border-top-width:0}.panel>.panel-collapse>.table,.panel>.table,.panel>.table-responsive>.table{margin-bottom:0}.panel>.panel-collapse>.table caption,.panel>.table caption,.panel>.table-responsive>.table caption{padding-right:15px;padding-left:15px}.panel>.table-responsive:first-child>.table:first-child,.panel>.table:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child,.panel>.table:first-child>thead:first-child>tr:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table:first-child>thead:first-child>tr:first-child th:first-child{border-top-left-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table:first-child>thead:first-child>tr:first-child th:last-child{border-top-right-radius:3px}.panel>.table-responsive:last-child>.table:last-child,.panel>.table:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:first-child{border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:last-child{border-bottom-right-radius:3px}.panel>.panel-body+.table,.panel>.panel-body+.table-responsive,.panel>.table+.panel-body,.panel>.table-responsive+.panel-body{border-top:1px solid #ddd}.panel>.table>tbody:first-child>tr:first-child td,.panel>.table>tbody:first-child>tr:first-child th{border-top:0}.panel>.table-bordered,.panel>.table-responsive>.table-bordered{border:0}.panel>.table-bordered>tbody>tr>td:first-child,.panel>.table-bordered>tbody>tr>th:first-child,.panel>.table-bordered>tfoot>tr>td:first-child,.panel>.table-bordered>tfoot>tr>th:first-child,.panel>.table-bordered>thead>tr>td:first-child,.panel>.table-bordered>thead>tr>th:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:first-child,.panel>.table-responsive>.table-bordered>thead>tr>td:first-child,.panel>.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.panel>.table-bordered>tbody>tr>td:last-child,.panel>.table-bordered>tbody>tr>th:last-child,.panel>.table-bordered>tfoot>tr>td:last-child,.panel>.table-bordered>tfoot>tr>th:last-child,.panel>.table-bordered>thead>tr>td:last-child,.panel>.table-bordered>thead>tr>th:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:last-child,.panel>.table-responsive>.table-bordered>thead>tr>td:last-child,.panel>.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.panel>.table-bordered>tbody>tr:first-child>td,.panel>.table-bordered>tbody>tr:first-child>th,.panel>.table-bordered>thead>tr:first-child>td,.panel>.table-bordered>thead>tr:first-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>th,.panel>.table-responsive>.table-bordered>thead>tr:first-child>td,.panel>.table-responsive>.table-bordered>thead>tr:first-child>th{border-bottom:0}.panel>.table-bordered>tbody>tr:last-child>td,.panel>.table-bordered>tbody>tr:last-child>th,.panel>.table-bordered>tfoot>tr:last-child>td,.panel>.table-bordered>tfoot>tr:last-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>th,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>td,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}.panel>.table-responsive{margin-bottom:0;border:0}.panel-group{margin-bottom:20px}.panel-group .panel{margin-bottom:0;border-radius:4px}.panel-group .panel+.panel{margin-top:5px}.panel-group .panel-heading{border-bottom:0}.panel-group .panel-heading+.panel-collapse>.list-group,.panel-group .panel-heading+.panel-collapse>.panel-body{border-top:1px solid #ddd}.panel-group .panel-footer{border-top:0}.panel-group .panel-footer+.panel-collapse .panel-body{border-bottom:1px solid #ddd}.panel-default{border-color:#ddd}.panel-default>.panel-heading{color:#333;background-color:#f5f5f5;border-color:#ddd}.panel-default>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ddd}.panel-default>.panel-heading .badge{color:#f5f5f5;background-color:#333}.panel-default>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ddd}.panel-primary{border-color:#337ab7}.panel-primary>.panel-heading{color:#fff;background-color:#337ab7;border-color:#337ab7}.panel-primary>.panel-heading+.panel-collapse>.panel-body{border-top-color:#337ab7}.panel-primary>.panel-heading .badge{color:#337ab7;background-color:#fff}.panel-primary>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#337ab7}.panel-success{border-color:#d6e9c6}.panel-success>.panel-heading{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.panel-success>.panel-heading+.panel-collapse>.panel-body{border-top-color:#d6e9c6}.panel-success>.panel-heading .badge{color:#dff0d8;background-color:#3c763d}.panel-success>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#d6e9c6}.panel-info{border-color:#bce8f1}.panel-info>.panel-heading{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.panel-info>.panel-heading+.panel-collapse>.panel-body{border-top-color:#bce8f1}.panel-info>.panel-heading .badge{color:#d9edf7;background-color:#31708f}.panel-info>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#bce8f1}.panel-warning{border-color:#faebcc}.panel-warning>.panel-heading{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.panel-warning>.panel-heading+.panel-collapse>.panel-body{border-top-color:#faebcc}.panel-warning>.panel-heading .badge{color:#fcf8e3;background-color:#8a6d3b}.panel-warning>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#faebcc}.panel-danger{border-color:#ebccd1}.panel-danger>.panel-heading{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.panel-danger>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ebccd1}.panel-danger>.panel-heading .badge{color:#f2dede;background-color:#a94442}.panel-danger>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ebccd1}.embed-responsive{position:relative;display:block;height:0;padding:0;overflow:hidden}.embed-responsive .embed-responsive-item,.embed-responsive embed,.embed-responsive iframe,.embed-responsive object,.embed-responsive video{position:absolute;top:0;bottom:0;left:0;width:100%;height:100%;border:0}.embed-responsive-16by9{padding-bottom:56.25%}.embed-responsive-4by3{padding-bottom:75%}.well{min-height:20px;padding:19px;margin-bottom:20px;background-color:#f5f5f5;border:1px solid #e3e3e3;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.05);box-shadow:inset 0 1px 1px rgba(0,0,0,.05)}.well blockquote{border-color:#ddd;border-color:rgba(0,0,0,.15)}.well-lg{padding:24px;border-radius:6px}.well-sm{padding:9px;border-radius:3px}.close{float:right;font-size:21px;font-weight:700;line-height:1;color:#000;text-shadow:0 1px 0 #fff;filter:alpha(opacity=20);opacity:.2}.close:focus,.close:hover{color:#000;text-decoration:none;cursor:pointer;filter:alpha(opacity=50);opacity:.5}button.close{-webkit-appearance:none;padding:0;cursor:pointer;background:0 0;border:0}.modal-open{overflow:hidden}.modal{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1050;display:none;overflow:hidden;-webkit-overflow-scrolling:touch;outline:0}.modal.fade .modal-dialog{-webkit-transition:-webkit-transform .3s ease-out;-o-transition:-o-transform .3s ease-out;transition:transform .3s ease-out;-webkit-transform:translate(0,-25%);-ms-transform:translate(0,-25%);-o-transform:translate(0,-25%);transform:translate(0,-25%)}.modal.in .modal-dialog{-webkit-transform:translate(0,0);-ms-transform:translate(0,0);-o-transform:translate(0,0);transform:translate(0,0)}.modal-open .modal{overflow-x:hidden;overflow-y:auto}.modal-dialog{position:relative;width:auto;margin:10px}.modal-content{position:relative;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #999;border:1px solid rgba(0,0,0,.2);border-radius:6px;outline:0;-webkit-box-shadow:0 3px 9px rgba(0,0,0,.5);box-shadow:0 3px 9px rgba(0,0,0,.5)}.modal-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1040;background-color:#000}.modal-backdrop.fade{filter:alpha(opacity=0);opacity:0}.modal-backdrop.in{filter:alpha(opacity=50);opacity:.5}.modal-header{padding:15px;border-bottom:1px solid #e5e5e5}.modal-header .close{margin-top:-2px}.modal-title{margin:0;line-height:1.42857143}.modal-body{position:relative;padding:15px}.modal-footer{padding:15px;text-align:right;border-top:1px solid #e5e5e5}.modal-footer .btn+.btn{margin-bottom:0;margin-left:5px}.modal-footer .btn-group .btn+.btn{margin-left:-1px}.modal-footer .btn-block+.btn-block{margin-left:0}.modal-scrollbar-measure{position:absolute;top:-9999px;width:50px;height:50px;overflow:scroll}@media (min-width:768px){.modal-dialog{width:600px;margin:30px auto}.modal-content{-webkit-box-shadow:0 5px 15px rgba(0,0,0,.5);box-shadow:0 5px 15px rgba(0,0,0,.5)}.modal-sm{width:300px}}@media (min-width:992px){.modal-lg{width:900px}}.tooltip{position:absolute;z-index:1070;display:block;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:12px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;filter:alpha(opacity=0);opacity:0;line-break:auto}.tooltip.in{filter:alpha(opacity=90);opacity:.9}.tooltip.top{padding:5px 0;margin-top:-3px}.tooltip.right{padding:0 5px;margin-left:3px}.tooltip.bottom{padding:5px 0;margin-top:3px}.tooltip.left{padding:0 5px;margin-left:-3px}.tooltip-inner{max-width:200px;padding:3px 8px;color:#fff;text-align:center;background-color:#000;border-radius:4px}.tooltip-arrow{position:absolute;width:0;height:0;border-color:transparent;border-style:solid}.tooltip.top .tooltip-arrow{bottom:0;left:50%;margin-left:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-left .tooltip-arrow{right:5px;bottom:0;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-right .tooltip-arrow{bottom:0;left:5px;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.right .tooltip-arrow{top:50%;left:0;margin-top:-5px;border-width:5px 5px 5px 0;border-right-color:#000}.tooltip.left .tooltip-arrow{top:50%;right:0;margin-top:-5px;border-width:5px 0 5px 5px;border-left-color:#000}.tooltip.bottom .tooltip-arrow{top:0;left:50%;margin-left:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-left .tooltip-arrow{top:0;right:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-right .tooltip-arrow{top:0;left:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.popover{position:absolute;top:0;left:0;z-index:1060;display:none;max-width:276px;padding:1px;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.2);border-radius:6px;-webkit-box-shadow:0 5px 10px rgba(0,0,0,.2);box-shadow:0 5px 10px rgba(0,0,0,.2);line-break:auto}.popover.top{margin-top:-10px}.popover.right{margin-left:10px}.popover.bottom{margin-top:10px}.popover.left{margin-left:-10px}.popover-title{padding:8px 14px;margin:0;font-size:14px;background-color:#f7f7f7;border-bottom:1px solid #ebebeb;border-radius:5px 5px 0 0}.popover-content{padding:9px 14px}.popover>.arrow,.popover>.arrow:after{position:absolute;display:block;width:0;height:0;border-color:transparent;border-style:solid}.popover>.arrow{border-width:11px}.popover>.arrow:after{content:"";border-width:10px}.popover.top>.arrow{bottom:-11px;left:50%;margin-left:-11px;border-top-color:#999;border-top-color:rgba(0,0,0,.25);border-bottom-width:0}.popover.top>.arrow:after{bottom:1px;margin-left:-10px;content:" ";border-top-color:#fff;border-bottom-width:0}.popover.right>.arrow{top:50%;left:-11px;margin-top:-11px;border-right-color:#999;border-right-color:rgba(0,0,0,.25);border-left-width:0}.popover.right>.arrow:after{bottom:-10px;left:1px;content:" ";border-right-color:#fff;border-left-width:0}.popover.bottom>.arrow{top:-11px;left:50%;margin-left:-11px;border-top-width:0;border-bottom-color:#999;border-bottom-color:rgba(0,0,0,.25)}.popover.bottom>.arrow:after{top:1px;margin-left:-10px;content:" ";border-top-width:0;border-bottom-color:#fff}.popover.left>.arrow{top:50%;right:-11px;margin-top:-11px;border-right-width:0;border-left-color:#999;border-left-color:rgba(0,0,0,.25)}.popover.left>.arrow:after{right:1px;bottom:-10px;content:" ";border-right-width:0;border-left-color:#fff}.carousel{position:relative}.carousel-inner{position:relative;width:100%;overflow:hidden}.carousel-inner>.item{position:relative;display:none;-webkit-transition:.6s ease-in-out left;-o-transition:.6s ease-in-out left;transition:.6s ease-in-out left}.carousel-inner>.item>a>img,.carousel-inner>.item>img{line-height:1}@media all and (transform-3d),(-webkit-transform-3d){.carousel-inner>.item{-webkit-transition:-webkit-transform .6s ease-in-out;-o-transition:-o-transform .6s ease-in-out;transition:transform .6s ease-in-out;-webkit-backface-visibility:hidden;backface-visibility:hidden;-webkit-perspective:1000px;perspective:1000px}.carousel-inner>.item.active.right,.carousel-inner>.item.next{left:0;-webkit-transform:translate3d(100%,0,0);transform:translate3d(100%,0,0)}.carousel-inner>.item.active.left,.carousel-inner>.item.prev{left:0;-webkit-transform:translate3d(-100%,0,0);transform:translate3d(-100%,0,0)}.carousel-inner>.item.active,.carousel-inner>.item.next.left,.carousel-inner>.item.prev.right{left:0;-webkit-transform:translate3d(0,0,0);transform:translate3d(0,0,0)}}.carousel-inner>.active,.carousel-inner>.next,.carousel-inner>.prev{display:block}.carousel-inner>.active{left:0}.carousel-inner>.next,.carousel-inner>.prev{position:absolute;top:0;width:100%}.carousel-inner>.next{left:100%}.carousel-inner>.prev{left:-100%}.carousel-inner>.next.left,.carousel-inner>.prev.right{left:0}.carousel-inner>.active.left{left:-100%}.carousel-inner>.active.right{left:100%}.carousel-control{position:absolute;top:0;bottom:0;left:0;width:15%;font-size:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6);background-color:rgba(0,0,0,0);filter:alpha(opacity=50);opacity:.5}.carousel-control.left{background-image:-webkit-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.5)),to(rgba(0,0,0,.0001)));background-image:linear-gradient(to right,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#80000000', endColorstr='#00000000', GradientType=1);background-repeat:repeat-x}.carousel-control.right{right:0;left:auto;background-image:-webkit-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.0001)),to(rgba(0,0,0,.5)));background-image:linear-gradient(to right,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#00000000', endColorstr='#80000000', GradientType=1);background-repeat:repeat-x}.carousel-control:focus,.carousel-control:hover{color:#fff;text-decoration:none;filter:alpha(opacity=90);outline:0;opacity:.9}.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{position:absolute;top:50%;z-index:5;display:inline-block;margin-top:-10px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{left:50%;margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{right:50%;margin-right:-10px}.carousel-control .icon-next,.carousel-control .icon-prev{width:20px;height:20px;font-family:serif;line-height:1}.carousel-control .icon-prev:before{content:'\2039'}.carousel-control .icon-next:before{content:'\203a'}.carousel-indicators{position:absolute;bottom:10px;left:50%;z-index:15;width:60%;padding-left:0;margin-left:-30%;text-align:center;list-style:none}.carousel-indicators li{display:inline-block;width:10px;height:10px;margin:1px;text-indent:-999px;cursor:pointer;background-color:#000\9;background-color:rgba(0,0,0,0);border:1px solid #fff;border-radius:10px}.carousel-indicators .active{width:12px;height:12px;margin:0;background-color:#fff}.carousel-caption{position:absolute;right:15%;bottom:20px;left:15%;z-index:10;padding-top:20px;padding-bottom:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6)}.carousel-caption .btn{text-shadow:none}@media screen and (min-width:768px){.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{width:30px;height:30px;margin-top:-10px;font-size:30px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{margin-right:-10px}.carousel-caption{right:20%;left:20%;padding-bottom:30px}.carousel-indicators{bottom:20px}}.btn-group-vertical>.btn-group:after,.btn-group-vertical>.btn-group:before,.btn-toolbar:after,.btn-toolbar:before,.clearfix:after,.clearfix:before,.container-fluid:after,.container-fluid:before,.container:after,.container:before,.dl-horizontal dd:after,.dl-horizontal dd:before,.form-horizontal .form-group:after,.form-horizontal .form-group:before,.modal-footer:after,.modal-footer:before,.modal-header:after,.modal-header:before,.nav:after,.nav:before,.navbar-collapse:after,.navbar-collapse:before,.navbar-header:after,.navbar-header:before,.navbar:after,.navbar:before,.pager:after,.pager:before,.panel-body:after,.panel-body:before,.row:after,.row:before{display:table;content:" "}.btn-group-vertical>.btn-group:after,.btn-toolbar:after,.clearfix:after,.container-fluid:after,.container:after,.dl-horizontal dd:after,.form-horizontal .form-group:after,.modal-footer:after,.modal-header:after,.nav:after,.navbar-collapse:after,.navbar-header:after,.navbar:after,.pager:after,.panel-body:after,.row:after{clear:both}.center-block{display:block;margin-right:auto;margin-left:auto}.pull-right{float:right!important}.pull-left{float:left!important}.hide{display:none!important}.show{display:block!important}.invisible{visibility:hidden}.text-hide{font:0/0 a;color:transparent;text-shadow:none;background-color:transparent;border:0}.hidden{display:none!important}.affix{position:fixed}@-ms-viewport{width:device-width}.visible-lg,.visible-md,.visible-sm,.visible-xs{display:none!important}.visible-lg-block,.visible-lg-inline,.visible-lg-inline-block,.visible-md-block,.visible-md-inline,.visible-md-inline-block,.visible-sm-block,.visible-sm-inline,.visible-sm-inline-block,.visible-xs-block,.visible-xs-inline,.visible-xs-inline-block{display:none!important}@media (max-width:767px){.visible-xs{display:block!important}table.visible-xs{display:table!important}tr.visible-xs{display:table-row!important}td.visible-xs,th.visible-xs{display:table-cell!important}}@media (max-width:767px){.visible-xs-block{display:block!important}}@media (max-width:767px){.visible-xs-inline{display:inline!important}}@media (max-width:767px){.visible-xs-inline-block{display:inline-block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm{display:block!important}table.visible-sm{display:table!important}tr.visible-sm{display:table-row!important}td.visible-sm,th.visible-sm{display:table-cell!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-block{display:block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline{display:inline!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline-block{display:inline-block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md{display:block!important}table.visible-md{display:table!important}tr.visible-md{display:table-row!important}td.visible-md,th.visible-md{display:table-cell!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-block{display:block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline{display:inline!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline-block{display:inline-block!important}}@media (min-width:1200px){.visible-lg{display:block!important}table.visible-lg{display:table!important}tr.visible-lg{display:table-row!important}td.visible-lg,th.visible-lg{display:table-cell!important}}@media (min-width:1200px){.visible-lg-block{display:block!important}}@media (min-width:1200px){.visible-lg-inline{display:inline!important}}@media (min-width:1200px){.visible-lg-inline-block{display:inline-block!important}}@media (max-width:767px){.hidden-xs{display:none!important}}@media (min-width:768px) and (max-width:991px){.hidden-sm{display:none!important}}@media (min-width:992px) and (max-width:1199px){.hidden-md{display:none!important}}@media (min-width:1200px){.hidden-lg{display:none!important}}.visible-print{display:none!important}@media print{.visible-print{display:block!important}table.visible-print{display:table!important}tr.visible-print{display:table-row!important}td.visible-print,th.visible-print{display:table-cell!important}}.visible-print-block{display:none!important}@media print{.visible-print-block{display:block!important}}.visible-print-inline{display:none!important}@media print{.visible-print-inline{display:inline!important}}.visible-print-inline-block{display:none!important}@media print{.visible-print-inline-block{display:inline-block!important}}@media print{.hidden-print{display:none!important}}
        /*# sourceMappingURL=bootstrap.min.css.map */
    </style>
    <!--<link rel="stylesheet" type="text/css" media="all" href="../../resources/bootstrap-3.2.0/css/bootstrap.min.css" th:href="@{resources/bootstrap-3.2.0/css/bootstrap.min.css}"/>-->
</head>
<body>
<div id="content" class="container">
    <h2>
        Exomiser Analysis Results for
        proband
    </h2>
    <ul class="nav nav-pills" id="tabs">
        <li><a href="#settings" role="tab" data-toggle="tab"><b>Analysis Settings</b></a></li>
        <li><a href="#filtering" role="tab" data-toggle="tab"><b>Filtering Summary</b></a></li>
        <li><a href="#distribution" role="tab" data-toggle="tab"><b>Variant Type Distribution</b></a></li>
        <li><a href="#prioritisation" role="tab" data-toggle="tab"><b>Prioritised Genes</b></a></li>
        
        <li><a href="#about" role="tab" data-toggle="tab"><b>About</b></a></li>
    </ul>
    <div class="panel panel-default" id="settings">
        <div class="panel-heading">
            <h3>Analysis Settings</h3>
        </div>
        <div class="panel-body"><p>Settings used for this analysis:</p>
            <pre>---
vcf: &quot;/mnt/scratch/Public/TRAINING/GenomeAnalysisModule/CaseInformation/CaseFiles/Case10/Case10_Merged.vcf.gz&quot;
genomeAssembly: &quot;HG38&quot;
pedigree:
  individuals:
  - familyId: &quot;FAM&quot;
    id: &quot;proband&quot;
    motherId: &quot;mother&quot;
    fatherId: &quot;father&quot;
    sex: &quot;MALE&quot;
    status: &quot;AFFECTED&quot;
  - familyId: &quot;FAM&quot;
    id: &quot;mother&quot;
    motherId: &quot;&quot;
    fatherId: &quot;&quot;
    sex: &quot;FEMALE&quot;
    status: &quot;AFFECTED&quot;
  - familyId: &quot;FAM&quot;
    id: &quot;father&quot;
    motherId: &quot;&quot;
    fatherId: &quot;&quot;
    sex: &quot;MALE&quot;
    status: &quot;UNAFFECTED&quot;
proband: &quot;proband&quot;
hpoIds:
- &quot;HP:0001249&quot;
- &quot;HP:0004322&quot;
- &quot;HP:0000252&quot;
- &quot;HP:0000325&quot;
- &quot;HP:0000400&quot;
- &quot;HP:0000316&quot;
- &quot;HP:0000637&quot;
- &quot;HP:0001572&quot;
inheritanceModes:
  maxFreq: 2.0
  empty: false
  definedModes:
  - &quot;AUTOSOMAL_DOMINANT&quot;
  - &quot;AUTOSOMAL_RECESSIVE&quot;
  - &quot;X_RECESSIVE&quot;
  - &quot;X_DOMINANT&quot;
  - &quot;MITOCHONDRIAL&quot;
  definedSubModes:
  - &quot;AUTOSOMAL_DOMINANT&quot;
  - &quot;AUTOSOMAL_RECESSIVE_COMP_HET&quot;
  - &quot;AUTOSOMAL_RECESSIVE_HOM_ALT&quot;
  - &quot;X_RECESSIVE_COMP_HET&quot;
  - &quot;X_RECESSIVE_HOM_ALT&quot;
  - &quot;X_DOMINANT&quot;
  - &quot;MITOCHONDRIAL&quot;
analysisMode: &quot;PASS_ONLY&quot;
frequencySources:
- &quot;THOUSAND_GENOMES&quot;
- &quot;TOPMED&quot;
- &quot;UK10K&quot;
- &quot;ESP_AFRICAN_AMERICAN&quot;
- &quot;ESP_EUROPEAN_AMERICAN&quot;
- &quot;ESP_ALL&quot;
- &quot;EXAC_AFRICAN_INC_AFRICAN_AMERICAN&quot;
- &quot;EXAC_AMERICAN&quot;
- &quot;EXAC_EAST_ASIAN&quot;
- &quot;EXAC_FINNISH&quot;
- &quot;EXAC_NON_FINNISH_EUROPEAN&quot;
- &quot;EXAC_OTHER&quot;
- &quot;EXAC_SOUTH_ASIAN&quot;
- &quot;GNOMAD_E_AFR&quot;
- &quot;GNOMAD_E_AMR&quot;
- &quot;GNOMAD_E_ASJ&quot;
- &quot;GNOMAD_E_EAS&quot;
- &quot;GNOMAD_E_FIN&quot;
- &quot;GNOMAD_E_NFE&quot;
- &quot;GNOMAD_E_OTH&quot;
- &quot;GNOMAD_E_SAS&quot;
- &quot;GNOMAD_G_AFR&quot;
- &quot;GNOMAD_G_AMR&quot;
- &quot;GNOMAD_G_ASJ&quot;
- &quot;GNOMAD_G_EAS&quot;
- &quot;GNOMAD_G_FIN&quot;
- &quot;GNOMAD_G_NFE&quot;
- &quot;GNOMAD_G_OTH&quot;
- &quot;GNOMAD_G_SAS&quot;
pathogenicitySources:
- &quot;POLYPHEN&quot;
- &quot;MUTATION_TASTER&quot;
- &quot;SIFT&quot;
- &quot;CADD&quot;
analysisSteps:
- offTargetVariantTypes:
  - &quot;CODING_TRANSCRIPT_INTRON_VARIANT&quot;
  - &quot;FIVE_PRIME_UTR_EXON_VARIANT&quot;
  - &quot;THREE_PRIME_UTR_EXON_VARIANT&quot;
  - &quot;FIVE_PRIME_UTR_INTRON_VARIANT&quot;
  - &quot;THREE_PRIME_UTR_INTRON_VARIANT&quot;
  - &quot;NON_CODING_TRANSCRIPT_EXON_VARIANT&quot;
  - &quot;NON_CODING_TRANSCRIPT_INTRON_VARIANT&quot;
  - &quot;UPSTREAM_GENE_VARIANT&quot;
  - &quot;DOWNSTREAM_GENE_VARIANT&quot;
  - &quot;INTERGENIC_VARIANT&quot;
  - &quot;REGULATORY_REGION_VARIANT&quot;
- decoratedFilter:
    maxFreq: 2.0
- decoratedFilter:
    keepNonPathogenic: true
- compatibleModes:
  - &quot;AUTOSOMAL_DOMINANT&quot;
  - &quot;AUTOSOMAL_RECESSIVE&quot;
  - &quot;X_RECESSIVE&quot;
  - &quot;X_DOMINANT&quot;
  - &quot;MITOCHONDRIAL&quot;
- priorityType: &quot;OMIM_PRIORITY&quot;
- priorityType: &quot;HIPHIVE_PRIORITY&quot;
---
outputContributingVariantsOnly: false
outputPrefix: &quot;Case10_exomiser&quot;
outputFormats:
- &quot;HTML&quot;
- &quot;VCF&quot;
- &quot;TSV_GENE&quot;
- &quot;TSV_VARIANT&quot;
- &quot;JSON&quot;
numGenes: 0
</pre>
        </div>
    </div>

    <div class="panel panel-default" id="filtering">
        <div class="panel-heading">
            <h3>Filtering Summary</h3>
        </div>
        <div class="panel-body">
            <table class="table">
                <thead>
                <tr>
                    <th>Filter</th>
                    <th>Report</th>
                    <th>Passed filter</th>
                    <th>Failed filter</th>
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td><b>Variant effect</b></td>
                    <td>
                        <ul>Removed variants with effects of type: [CODING_TRANSCRIPT_INTRON_VARIANT, FIVE_PRIME_UTR_EXON_VARIANT, THREE_PRIME_UTR_EXON_VARIANT, FIVE_PRIME_UTR_INTRON_VARIANT, THREE_PRIME_UTR_INTRON_VARIANT, NON_CODING_TRANSCRIPT_EXON_VARIANT, NON_CODING_TRANSCRIPT_INTRON_VARIANT, UPSTREAM_GENE_VARIANT, DOWNSTREAM_GENE_VARIANT, INTERGENIC_VARIANT, REGULATORY_REGION_VARIANT]</ul>
                    </td>
                    <td>325</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Frequency</b></td>
                    <td>
                        <ul>Variants filtered for maximum allele frequency of 2.00%</ul>
                    </td>
                    <td>325</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Pathogenicity</b></td>
                    <td>
                        <ul>Retained all non-pathogenic variants of all types. Scoring was applied, but the filter passed all variants.</ul>
                    </td>
                    <td>325</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Inheritance</b></td>
                    <td>
                        <ul>Genes filtered for compatibility with AUTOSOMAL_DOMINANT, AUTOSOMAL_RECESSIVE, X_RECESSIVE, X_DOMINANT, MITOCHONDRIAL inheritance.</ul>
                    </td>
                    <td>278</td>
                    <td>0</td>
                </tr>
                </tbody>
            </table>
        </div>
    </div>
    <div class="panel panel-default" id="distribution">
        <div class="panel-heading">
            <h3>Variant Type Distribution</h3>
        </div>
        <div class="panel-body">
            <table class="table-striped table-condensed">
                <thead>
                <tr>
                    <th>Variant Type</th>
                    
                        <th>proband</th>
                    
                        <th>mother</th>
                    
                        <th>father</th>
                    
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td>FRAMESHIFT_ELONGATION</td>
                    
                        <td>2</td>
                    
                        <td>2</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_TRUNCATION</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_VARIANT</td>
                    
                        <td>2</td>
                    
                        <td>2</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INTERNAL_FEATURE_ELONGATION</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FEATURE_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>MNV</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>STOP_GAINED</td>
                    
                        <td>4</td>
                    
                        <td>4</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>STOP_LOST</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>START_LOST</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_ACCEPTOR_VARIANT</td>
                    
                        <td>3</td>
                    
                        <td>3</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_DONOR_VARIANT</td>
                    
                        <td>2</td>
                    
                        <td>2</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>MISSENSE_VARIANT</td>
                    
                        <td>186</td>
                    
                        <td>186</td>
                    
                        <td>18</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_INSERTION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_INSERTION</td>
                    
                        <td>3</td>
                    
                        <td>3</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_DELETION</td>
                    
                        <td>2</td>
                    
                        <td>2</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_DELETION</td>
                    
                        <td>5</td>
                    
                        <td>5</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_REGION_VARIANT</td>
                    
                        <td>23</td>
                    
                        <td>22</td>
                    
                        <td>2</td>
                    
                </tr>
                <tr>
                    <td>STOP_RETAINED_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INITIATOR_CODON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SYNONYMOUS_VARIANT</td>
                    
                        <td>90</td>
                    
                        <td>90</td>
                    
                        <td>8</td>
                    
                </tr>
                <tr>
                    <td>CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>UPSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DOWNSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INTERGENIC_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>REGULATORY_REGION_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                </tbody>
            </table>
        </div>
    </div>

    

    <div class="panel panel-default" id="prioritisation">
        <div class="panel-heading">
            <h3>Prioritised Genes</h3>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167522">ANKRD11</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.997</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.969</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.969 to <a href="http://www.omim.org/148050">KBG syndrome</a> associated with ANKRD11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0001572, Macrodontia</dd></dl><dl><dt>Phenotypic similarity 0.541 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ANKRD11">ANKRD11</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000441, increased cranium width</dd><dd>HP:0000325, Triangular face - MP:0030069, broad face</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000102, abnormal nasal bone morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANKRD11%0DNCOA3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NCOA3</a> and phenotypic similarity 0.627 to mouse mutant of NCOA3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001289, persistence of hyaloid vascular system</dd><dd>HP:0000637, Long palpebral fissure - MP:0001289, persistence of hyaloid vascular system</dd><dd>HP:0001572, Macrodontia - MP:0002100, abnormal tooth morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/148050">OMIM:148050</a> KBG syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2332">ORPHA:2332</a> KBG syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.997</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.969</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_ELONGATION</span>
                                <b>chr16:g.89280859T&gt;TG [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1163944246">rs1163944246</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1163944246&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-danger">PATHOGENIC</span>
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=645137%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                <span class="label label-danger">WHITELIST VARIANT</span>
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000301030.10">ANKRD11:ENST00000301030.10:c.5682dup:p.(Arg1895Glnfs*55)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378330.7">ANKRD11:ENST00000378330.7:c.5682dup:p.(Arg1895Glnfs*55)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000642600.1">ANKRD11:ENST00000642600.1:c.5682dup:p.(Arg1895Glnfs*55)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99636924</dd>
                                    
                                    <dd>CADD: 0.996 (24.400)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000147044">CASK</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.987</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.823</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.704 to <a href="http://www.omim.org/300749">Mental retardation and microcephaly with pontine and cerebellar hypoplasia</a> associated with CASK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002342, Intellectual disability, moderate</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000253, Progressive microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000286, Epicanthus</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Phenotypic similarity 0.823 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CASK">CASK</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001505, hunched posture</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0000433, microcephaly</dd><dd>HP:0000325, Triangular face - MP:0000448, pointed snout</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001290, delayed eyelid opening</dd><dd>HP:0000637, Long palpebral fissure - MP:0001290, delayed eyelid opening</dd><dd>HP:0001572, Macrodontia - MP:0000455, abnormal maxilla morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/300422">OMIM:300422</a> Mental retardation, with or without nystagmus - X-linked</dd>
<dd><a href="http://www.omim.org/entry/300749">OMIM:300749</a> Mental retardation and microcephaly with pontine and cerebellar hypoplasia - X-linked dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=163937">ORPHA:163937</a> X-linked intellectual disability, Najm type</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1934">ORPHA:1934</a> Early infantile epileptic encephalopathy</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.987</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.823</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr23:g.41569717A&gt;G [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=781207535">rs781207535</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs781207535&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378154.2">CASK:ENST00000378154.2:c.1533T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378158.6">CASK:ENST00000378158.6:c.1515T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378163.6">CASK:ENST00000378163.6:c.1533T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378166.9">CASK:ENST00000378166.9:c.1515T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378179.8">CASK:ENST00000378179.8:c.378T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421587.7">CASK:ENST00000421587.7:c.1533T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000442742.7">CASK:ENST00000442742.7:c.1515T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000644219.1">CASK:ENST00000644219.1:c.1515T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000644347.1">CASK:ENST00000644347.1:c.1515T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000645566.1">CASK:ENST00000645566.1:c.1533T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000645986.1">CASK:ENST00000645986.1:c.1551T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000646087.1">CASK:ENST00000646087.1:c.1533T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000646120.1">CASK:ENST00000646120.1:c.1551T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9922375</dd>
                                    
                                    <dd>CADD: 0.992 (21.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0265%</dd>
                                    <dd>TOPMed: 0.0265%</dd>
                                    <dd>ExAC AFR: 0.0118%</dd>
                                    <dd>gnomAD_E_AFR: 0.0083%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000120008">WDR11</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.977</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.760</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.512 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=432">Normosmic congenital hypogonadotropic hypogonadism</a> associated with WDR11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0000002, Abnormality of body height</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000164, Abnormality of the dentition</dd></dl><dl><dt>Phenotypic similarity 0.760 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=WDR11">WDR11</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0000547, short limbs</dd><dd>HP:0000252, Microcephaly - MP:0000433, microcephaly</dd><dd>HP:0000325, Triangular face - MP:0003743, abnormal facial morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0006197, ocular hypotelorism</dd><dd>HP:0000637, Long palpebral fissure - MP:0001286, abnormal eye development</dd><dd>HP:0001572, Macrodontia - MP:0004592, small mandible</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/614858">OMIM:614858</a> Hypogonadotropic hypogonadism 14 with or without anosmia - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=432">ORPHA:432</a> Normosmic congenital hypogonadotropic hypogonadism</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=478">ORPHA:478</a> Kallmann syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.977</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.760</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.120865202C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=562880069">rs562880069</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs562880069&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263461.11">WDR11:ENST00000263461.11:c.869C&gt;T:p.(Pro290Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.997 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.004 (D)</dd>
                                    <dd>CADD: 0.998 (26.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000110921">MVK</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.973</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.753</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.986</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.753 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=29">Mevalonic aciduria</a> associated with MVK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia - HP:0000368, Low-set, posteriorly rotated ears</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000494, Downslanted palpebral fissures</dd><dd>HP:0001572, Macrodontia - HP:0000239, Large fontanelles</dd></dl><dl><dt>Phenotypic similarity 0.393 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MVK">MVK</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0003232, abnormal forebrain development</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001297, microphthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0001297, microphthalmia</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MVK%0DPDSS2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PDSS2</a> and phenotypic similarity 0.225 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=255249">Leigh syndrome with nephrotic syndrome</a> associated with PDSS2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001511, Intrauterine growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MVK%0DPDSS2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PDSS2</a> and phenotypic similarity 0.632 to mouse mutant of PDSS2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001304, cataract</dd><dd>HP:0000637, Long palpebral fissure - MP:0001304, cataract</dd><dd>HP:0001572, Macrodontia - MP:0002100, abnormal tooth morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/175900">OMIM:175900</a> Porokeratosis 3, multiple types - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/260920">OMIM:260920</a> Hyper-IgD syndrome - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/610377">OMIM:610377</a> Mevalonic aciduria - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=29">ORPHA:29</a> Mevalonic aciduria</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=343">ORPHA:343</a> Hyperimmunoglobulinemia D with periodic fever</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=735">ORPHA:735</a> Porokeratosis of Mibelli</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79152">ORPHA:79152</a> Disseminated superficial actinic porokeratosis</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.973</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.753</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.986</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.109581547A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=138818135">rs138818135</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs138818135&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.986</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000228510.8">MVK:ENST00000228510.8:c.524A&gt;G:p.(Asn175Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539575.4">MVK:ENST00000539575.4:c.524A&gt;G:p.(Asn175Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392727.7">MVK:ENST00000392727.7:c.371+1601A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000625889.2">MVK:ENST00000625889.2:c.371+1601A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000629016.2">MVK:ENST00000629016.2:c.227-4475A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539696.5">MVK:ENST00000539696.5:c.-92+7674A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.975 (16.060)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0998%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0674%</dd>
                                    <dd>gnomAD_E_AFR: 0.0458%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171634">BPTF</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.947</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.685</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.986</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.685 to <a href="http://www.omim.org/617755">Neurodevelopmental disorder with dysmorphic facies and distal limb anomalies</a> associated with BPTF.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Phenotypic similarity 0.246 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=BPTF">BPTF</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=BPTF%0DWDR73&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to WDR73</a> and phenotypic similarity 0.708 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2065">Galloway-Mowat syndrome</a> associated with WDR73.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/617755">OMIM:617755</a> Neurodevelopmental disorder with dysmorphic facies and distal limb anomalies - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.947</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.685</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.986</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.67912707G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=531901425">rs531901425</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs531901425&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.986</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000306378.11">BPTF:ENST00000306378.11:c.4823G&gt;A:p.(Gly1608Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000321892.8">BPTF:ENST00000321892.8:c.5201G&gt;A:p.(Gly1734Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000342579.8">BPTF:ENST00000342579.8:c.4784G&gt;A:p.(Gly1595Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000644067.1">BPTF:ENST00000644067.1:c.4784G&gt;A:p.(Gly1595Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9883855</dd>
                                    
                                    <dd>Polyphen2: 0.004 (B)</dd>
                                    <dd>SIFT: 0.179 (T)</dd>
                                    <dd>CADD: 0.988 (19.350)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0099%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000083857">FAT1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.947</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.699</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.969</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.699 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FAT1">FAT1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0005157, holoprosencephaly</dd><dd>HP:0000325, Triangular face - MP:0006290, proboscis</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005163, cyclopia</dd><dd>HP:0000637, Long palpebral fissure - MP:0001293, anophthalmia</dd><dd>HP:0001572, Macrodontia - MP:0006290, proboscis</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FAT1%0DDCHS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DCHS1</a> and phenotypic similarity 0.681 to <a href="http://www.omim.org/601390">Van Maldergem syndrome 1</a> associated with DCHS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0001510, Growth delay</dd><dd>HP:0000252, Microcephaly - HP:0002079, Hypoplasia of the corpus callosum</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0008551, Microtia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000689, Dental malocclusion</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FAT1%0DDCHS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DCHS1</a> and phenotypic similarity 0.367 to mouse mutant of DCHS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0000031, abnormal cochlea morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.947</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.699</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.969</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.186600230C&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=375087606">rs375087606</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs375087606&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.969</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441802.7">FAT1:ENST00000441802.7:c.11771G&gt;T:p.(Arg3924Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614102.4">FAT1:ENST00000614102.4:c.11777G&gt;T:p.(Arg3926Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9804566</dd>
                                    
                                    <dd>Polyphen2: 0.512 (P)</dd>
                                    <dd>SIFT: 0.073 (T)</dd>
                                    <dd>CADD: 0.980 (17.090)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0135%</dd>
                                    <dd>ESP AA: 0.0755%</dd>
                                    <dd>ESP All: 0.0244%</dd>
                                    <dd>ExAC AFR: 0.0620%</dd>
                                    <dd>ExAC AMR: 0.0263%</dd>
                                    <dd>gnomAD_E_AFR: 0.0589%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_AFR: 0.0802%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000166341">DCHS1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.946</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.681</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.988</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.681 to <a href="http://www.omim.org/601390">Van Maldergem syndrome 1</a> associated with DCHS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0001510, Growth delay</dd><dd>HP:0000252, Microcephaly - HP:0002079, Hypoplasia of the corpus callosum</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0008551, Microtia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000689, Dental malocclusion</dd></dl><dl><dt>Phenotypic similarity 0.367 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DCHS1">DCHS1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0000031, abnormal cochlea morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DCHS1%0DFAT4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FAT4</a> and phenotypic similarity 0.681 to <a href="http://www.omim.org/615546">Van Maldergem syndrome 2</a> associated with FAT4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0001510, Growth delay</dd><dd>HP:0000252, Microcephaly - HP:0002079, Hypoplasia of the corpus callosum</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0008551, Microtia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000689, Dental malocclusion</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DCHS1%0DFAT4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FAT4</a> and phenotypic similarity 0.333 to mouse mutant of FAT4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0000031, abnormal cochlea morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/601390">OMIM:601390</a> Van Maldergem syndrome 1 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/607829">OMIM:607829</a> Mitral valve prolapse 2 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=314679">ORPHA:314679</a> Cerebrofacioarticular syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.946</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.681</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.988</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.6630440C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=557911084">rs557911084</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs557911084&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.988</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000299441.5">DCHS1:ENST00000299441.5:c.4354G&gt;C:p.(Gly1452Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.998</dd>
                                    
                                    <dd>Polyphen2: 0.992 (D)</dd>
                                    <dd>Mutation Taster: 0.988 (P)</dd>
                                    <dd>SIFT: 0.002 (D)</dd>
                                    <dd>CADD: 0.994 (22.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0127%</dd>
                                    <dd>gnomAD_E_AFR: 0.0330%</dd>
                                    <dd>gnomAD_E_AMR: 0.0046%</dd>
                                    <dd>gnomAD_G_AFR: 0.0703%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165370">GPR101</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.927</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.644</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.644 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=963">Acromegaly</a> associated with GPR101.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0000098, Tall stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0100540, Palpebral edema</dd><dd>HP:0001572, Macrodontia - HP:0000687, Widely spaced teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GPR101%0DLPAR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LPAR1</a> and phenotypic similarity 0.900 to mouse mutant of LPAR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001255, decreased body height</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000445, short snout</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - MP:0001300, ocular hypertelorism</dd><dd>HP:0001572, Macrodontia - MP:0004831, long incisors</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/300943">OMIM:300943</a> Pituitary adenoma 2, GH-secreting - X-linked</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=963">ORPHA:963</a> Acromegaly</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.927</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.644</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.137031256G&gt;C [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=767263132">rs767263132</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs767263132&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298110.1">GPR101:ENST00000298110.1:c.419C&gt;G:p.(Pro140Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000651716.1">GPR101:ENST00000651716.1:c.419C&gt;G:p.(Pro140Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.997953</dd>
                                    
                                    <dd>Polyphen2: 0.990 (D)</dd>
                                    <dd>Mutation Taster: 0.998 (P)</dd>
                                    <dd>SIFT: 0.011 (D)</dd>
                                    <dd>CADD: 0.998 (26.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0265%</dd>
                                    <dd>ExAC AFR: 0.0120%</dd>
                                    <dd>gnomAD_E_AFR: 0.0082%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000110711">AIP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.923</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.644</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.988</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.644 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=963">Acromegaly</a> associated with AIP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0000098, Tall stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0100540, Palpebral edema</dd><dd>HP:0001572, Macrodontia - HP:0000687, Widely spaced teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=AIP%0DAFF3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AFF3</a> and phenotypic similarity 0.614 to mouse mutant of AFF3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000443, abnormal snout morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0002100, abnormal tooth morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/102200">OMIM:102200</a> Pituitary adenoma 1, multiple types - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=963">ORPHA:963</a> Acromegaly</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=99725">ORPHA:99725</a> Pituitary gigantism</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.923</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.644</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.988</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.67490920G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4930199">rs4930199</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs4930199&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=838853%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.988</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000279146.8">AIP:ENST00000279146.8:c.920G&gt;A:p.(Arg307Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9892848</dd>
                                    
                                    <dd>CADD: 0.989 (19.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0127%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000102081">FMR1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.918</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.628</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.628 to <a href="http://www.omim.org/300624">Fragile X syndrome</a> associated with FMR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002342, Intellectual disability, moderate</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000303, Mandibular prognathia</dd></dl><dl><dt>Phenotypic similarity 0.434 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FMR1">FMR1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001463, abnormal spatial learning</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0012144, decreased b wave amplitude</dd><dd>HP:0000637, Long palpebral fissure - MP:0012144, decreased b wave amplitude</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.517 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FMR1%0DZNF81&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ZNF81</a> and phenotypic similarity 0.750 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with ZNF81.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002187, Intellectual disability, profound</dd><dd>HP:0004322, Short stature - HP:0001518, Small for gestational age</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000684, Delayed eruption of teeth</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/300623">OMIM:300623</a> Fragile X tremor/ataxia syndrome - X-linked dominant</dd>
<dd><a href="http://www.omim.org/entry/300624">OMIM:300624</a> Fragile X syndrome - X-linked dominant</dd>
<dd><a href="http://www.omim.org/entry/311360">OMIM:311360</a> Premature ovarian failure 1 - X-linked</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=261483">ORPHA:261483</a> Xq27.3q28 duplication syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=908">ORPHA:908</a> Fragile X syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=908">ORPHA:908</a> Fragile X syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93256">ORPHA:93256</a> Fragile X-associated tremor/ataxia syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.918</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.628</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_ACCEPTOR_VARIANT</span>
                                <b>chr23:g.147912051C&gt;CGGCGGCGGCGGA [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1441792007">rs1441792007</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1441792007&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621987.4">FMR1:ENST00000621987.4:c.-127-2_-127-1insGGCGGCGGCGGA:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000218200.12">FMR1:ENST00000218200.12:c.-118_-117insAGGCGGCGGCGG:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000334557.10">FMR1:ENST00000334557.10:c.-118_-117insAGGCGGCGGCGG:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370471.7">FMR1:ENST00000370471.7:c.-118_-117insAGGCGGCGGCGG:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370475.8">FMR1:ENST00000370475.8:c.-118_-117insAGGCGGCGGCGG:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370477.5">FMR1:ENST00000370477.5:c.-118_-117insAGGCGGCGGCGG:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439526.6">FMR1:ENST00000439526.6:c.-118_-117insAGGCGGCGGCGG:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000440235.6">FMR1:ENST00000440235.6:c.-118_-117insAGGCGGCGGCGG:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621453.4">FMR1:ENST00000621453.4:c.-118_-117insAGGCGGCGGCGG:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99241424</dd>
                                    
                                    <dd>CADD: 0.992 (21.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000087266">SH3BP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.900</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.610</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.506 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=184">Cherubism</a> associated with SH3BP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000520, Proptosis</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0006482, Abnormality of dental morphology</dd></dl><dl><dt>Phenotypic similarity 0.610 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SH3BP2">SH3BP2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000454, abnormal jaw morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005251, blepharitis</dd><dd>HP:0000637, Long palpebral fissure - MP:0005251, blepharitis</dd><dd>HP:0001572, Macrodontia - MP:0030466, alveolar process atrophy</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SH3BP2%0DBCR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BCR</a> and phenotypic similarity 0.679 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=261330">Distal 22q11.2 microdeletion syndrome</a> associated with BCR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000363, Abnormality of earlobe</dd><dd>HP:0000316, Hypertelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000637, Long palpebral fissure - HP:0000581, Blepharophimosis</dd><dd>HP:0001572, Macrodontia - HP:0000307, Pointed chin</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/118400">OMIM:118400</a> Cherubism - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=184">ORPHA:184</a> Cherubism</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.900</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.610</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.2830056C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=529543434">rs529543434</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs529543434&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    <span class="label label-info">LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=293283%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356331.9">SH3BP2:ENST00000356331.9:c.1150C&gt;T:p.(Arg384Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000435136.8">SH3BP2:ENST00000435136.8:c.1234C&gt;T:p.(Arg412Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000503393.8">SH3BP2:ENST00000503393.8:c.1150C&gt;T:p.(Arg384Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000511747.6">SH3BP2:ENST00000511747.6:c.1321C&gt;T:p.(Arg441Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999</dd>
                                    
                                    <dd>Polyphen2: 0.857 (P)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.994 (22.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0114%</dd>
                                    <dd>ExAC FIN: 0.0155%</dd>
                                    <dd>gnomAD_E_AFR: 0.0142%</dd>
                                    <dd>gnomAD_E_FIN: 0.0094%</dd>
                                    <dd>gnomAD_E_NFE: 0.0028%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000097007">ABL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.898</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.607</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.548 to <a href="http://www.omim.org/617602">Congenital heart defects and skeletal malformations syndrome</a> associated with ABL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dl><dt>Phenotypic similarity 0.607 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ABL1">ABL1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0008525, decreased cranium height</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001304, cataract</dd><dd>HP:0000637, Long palpebral fissure - MP:0001302, eyelids open at birth</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ABL1%0DSH3BP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SH3BP2</a> and phenotypic similarity 0.506 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=184">Cherubism</a> associated with SH3BP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000520, Proptosis</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0006482, Abnormality of dental morphology</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ABL1%0DSH3BP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SH3BP2</a> and phenotypic similarity 0.610 to mouse mutant of SH3BP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000454, abnormal jaw morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005251, blepharitis</dd><dd>HP:0000637, Long palpebral fissure - MP:0005251, blepharitis</dd><dd>HP:0001572, Macrodontia - MP:0030466, alveolar process atrophy</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/608232">OMIM:608232</a> Leukemia, Philadelphia chromosome-positive, resistant to imatinib - somatic</dd>
<dd><a href="http://www.omim.org/entry/617602">OMIM:617602</a> Congenital heart defects and skeletal malformations syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=521">ORPHA:521</a> Chronic myeloid leukemia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.898</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.607</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.130880108A&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=541225290">rs541225290</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs541225290&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000318560.6">ABL1:ENST00000318560.6:c.1464A&gt;C:p.(Glu488Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372348.7">ABL1:ENST00000372348.7:c.1521A&gt;C:p.(Glu507Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999999</dd>
                                    
                                    <dd>Polyphen2: 0.121 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.101 (T)</dd>
                                    <dd>CADD: 0.995 (22.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100014">SPECC1L</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.883</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.592</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.592 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1519">Hypertelorism, Teebi type</a> associated with SPECC1L.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000248, Brachycephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0011039, Abnormality of the helix</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia - HP:0006288, Advanced eruption of teeth</dd></dl><dl><dt>Phenotypic similarity 0.294 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=SPECC1L">SPECC1L</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - ZP:0001708, abnormal(ly) malformed mandibular arch skeleton</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/145410">OMIM:145410</a> Opitz GBBB syndrome, type II - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/145420">OMIM:145420</a> Hypertelorism, Teebi type - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/600251">OMIM:600251</a> ?Facial clefting, oblique, 1 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1519">ORPHA:1519</a> Hypertelorism, Teebi type</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.883</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.592</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.24321519C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=529314817">rs529314817</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs529314817&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000314328.14">SPECC1L:ENST00000314328.14:c.539C&gt;G:p.(Ala180Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000437398.5">SPECC1L:ENST00000437398.5:c.539C&gt;G:p.(Ala180Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541492.1">SPECC1L:ENST00000541492.1:c.539C&gt;G:p.(Ala180Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000651059.1">SPECC1L:ENST00000651059.1:c.596C&gt;G:p.(Ala199Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358654.2">SPECC1L:ENST00000358654.2:c.539C&gt;G:p.(Ala180Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.018 (D)</dd>
                                    <dd>CADD: 0.995 (22.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000155657">TTN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.859</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.571</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.496 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=169186">Autosomal recessive centronuclear myopathy</a> associated with TTN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001256, Intellectual disability, mild</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000411, Protruding ear</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000602, Ophthalmoplegia</dd><dd>HP:0001572, Macrodontia - HP:0000278, Retrognathia</dd></dl><dl><dt>Phenotypic similarity 0.571 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TTN">TTN</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000433, microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.332 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=TTN">TTN</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - ZP:0000054, abnormal(ly) decreased size head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/600334">OMIM:600334</a> Tibial muscular dystrophy, tardive - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/603689">OMIM:603689</a> Myopathy, myofibrillar, 9, with early respiratory failure - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/604145">OMIM:604145</a> Cardiomyopathy, dilated, 1G - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/608807">OMIM:608807</a> Muscular dystrophy, limb-girdle, autosomal recessive 10 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/611705">OMIM:611705</a> Salih myopathy - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/613765">OMIM:613765</a> Cardiomyopathy, familial hypertrophic, 9 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=154">ORPHA:154</a> Familial isolated dilated cardiomyopathy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=169186">ORPHA:169186</a> Autosomal recessive centronuclear myopathy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=178464">ORPHA:178464</a> Hereditary myopathy with early respiratory failure</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=324604">ORPHA:324604</a> Classic multiminicore myopathy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=609">ORPHA:609</a> Tibial muscular dystrophy</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.859</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.571</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.178570509G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=549335677">rs549335677</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs549335677&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000342175.11">TTN:ENST00000342175.11:c.49004C&gt;A:p.(Pro16335His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000342992.10">TTN:ENST00000342992.10:c.67919C&gt;A:p.(Pro22640His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359218.10">TTN:ENST00000359218.10:c.48803C&gt;A:p.(Pro16268His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000460472.6">TTN:ENST00000460472.6:c.48428C&gt;A:p.(Pro16143His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589042.5">TTN:ENST00000589042.5:c.75623C&gt;A:p.(Pro25208His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000591111.5">TTN:ENST00000591111.5:c.70700C&gt;A:p.(Pro23567His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615779.5">TTN:ENST00000615779.5:c.70700C&gt;A:p.(Pro23567His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.967 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.999 (33.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0103%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr2:g.178802247G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=528853682">rs528853682</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs528853682&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                <div class="col-sm-12">
                    <b>ClinVar: </b>
                    
                        
                        
                        
                        
                        
                        
                        <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                        <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=193734%5Balleleid%5D">(criteria_provided,_conflicting_interpretations)</a>
                    
                </div>
                <div class="col-sm-6">
                    <b>Variant score: 0.966</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000342175.11">TTN:ENST00000342175.11:c.186C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000342992.10">TTN:ENST00000342992.10:c.186C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359218.10">TTN:ENST00000359218.10:c.186C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360870.9">TTN:ENST00000360870.9:c.186C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000460472.6">TTN:ENST00000460472.6:c.186C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589042.5">TTN:ENST00000589042.5:c.186C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000591111.5">TTN:ENST00000591111.5:c.186C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615779.5">TTN:ENST00000615779.5:c.186C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9718162</dd>
                        
                        <dd>CADD: 0.972 (15.500)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0399%</dd>
                        <dd>TOPMed: 0.0040%</dd>
                        <dd>ExAC AFR: 0.0192%</dd>
                        <dd>gnomAD_E_AFR: 0.0131%</dd>
                        <dd>gnomAD_E_FIN: 0.0045%</dd>
                        <dd>gnomAD_E_NFE: 0.0018%</dd>
                        <dd>gnomAD_E_SAS: 0.0032%</dd>
                        <dd>gnomAD_G_AFR: 0.0229%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr2:g.178740006G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=571473566">rs571473566</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs571473566&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                <div class="col-sm-12">
                    <b>ClinVar: </b>
                    
                        
                        
                        <span class="label label-info">LIKELY_BENIGN</span>
                        
                        
                        
                        
                        <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=499549%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                    
                </div>
                <div class="col-sm-6">
                    <b>Variant score: 0.828</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000342175.11">TTN:ENST00000342175.11:c.12714C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359218.10">TTN:ENST00000359218.10:c.12513C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000460472.6">TTN:ENST00000460472.6:c.12138C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589042.5">TTN:ENST00000589042.5:c.13227C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000591111.5">TTN:ENST00000591111.5:c.12276C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615779.5">TTN:ENST00000615779.5:c.12276C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000342992.10">TTN:ENST00000342992.10:c.10361-1646C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8333137</dd>
                        
                        <dd>CADD: 0.833 (7.781)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0200%</dd>
                        <dd>TOPMed: 0.0064%</dd>
                        <dd>ExAC AFR: 0.0204%</dd>
                        <dd>ExAC EAS: 0.0117%</dd>
                        <dd>gnomAD_E_AFR: 0.0131%</dd>
                        <dd>gnomAD_E_EAS: 0.0116%</dd>
                        <dd>gnomAD_G_AFR: 0.0458%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000156983">BRPF1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.837</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.685</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.849</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.685 to <a href="http://www.omim.org/617333">Intellectual developmental disorder with dysmorphic facies and ptosis</a> associated with BRPF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000377, Abnormality of the pinna</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000581, Blepharophimosis</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=BRPF1%0DKAT6B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KAT6B</a> and phenotypic similarity 0.680 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3047">Blepharophimosis-intellectual disability syndrome, SBBYS type</a> associated with KAT6B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0003510, Severe short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0009738, Abnormality of the antihelix</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000278, Retrognathia</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=BRPF1%0DKAT6B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KAT6B</a> and phenotypic similarity 0.752 to mouse mutant of KAT6B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0000435, shortened head</dd><dd>HP:0000325, Triangular face - MP:0000088, short mandible</dd><dd>HP:0000400, Macrotia - MP:0000023, abnormal ear position</dd><dd>HP:0000316, Hypertelorism - MP:0001297, microphthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0008969, abnormal nasolacrimal duct morphology</dd><dd>HP:0001572, Macrodontia - MP:0000088, short mandible</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/617333">OMIM:617333</a> Intellectual developmental disorder with dysmorphic facies and ptosis - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.837</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.685</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.849</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr3:g.9743021C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=373441695">rs373441695</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs373441695&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.849</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000383829.7">BRPF1:ENST00000383829.7:c.2079C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424362.6">BRPF1:ENST00000424362.6:c.2061C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000433861.6">BRPF1:ENST00000433861.6:c.2061C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457855.1">BRPF1:ENST00000457855.1:c.2061C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8521232</dd>
                                    
                                    <dd>CADD: 0.852 (8.301)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000082068">WDR70</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.815</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.546</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.991</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.546 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=WDR70%0DCPLANE1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPLANE1</a> and phenotypic similarity 0.609 to <a href="http://www.omim.org/277170">Orofaciodigital syndrome VI</a> associated with CPLANE1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0001320, Cerebellar vermis hypoplasia</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000286, Epicanthus</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Proximity score 0.546 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=WDR70%0DCPLANE1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CPLANE1</a> and phenotypic similarity 0.541 to mouse mutant of CPLANE1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0000774, decreased brain size</dd><dd>HP:0000325, Triangular face - MP:0000762, abnormal tongue morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000460, mandible hypoplasia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.815</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.546</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.991</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr5:g.37703059G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=572793662">rs572793662</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs572793662&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.991</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265107.9">WDR70:ENST00000265107.9:c.1388G&gt;A:p.(Arg463Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99323916</dd>
                                    
                                    <dd>Polyphen2: 0.000 (B)</dd>
                                    <dd>Mutation Taster: 0.648</dd>
                                    <dd>SIFT: 1.000 (T)</dd>
                                    <dd>CADD: 0.993 (21.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000056558">TRAF1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.804</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.578</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.946</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.578 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRAF1%0DTRIM37&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRIM37</a> and phenotypic similarity 0.731 to <a href="http://www.omim.org/253250">Mulibrey nanism</a> associated with TRIM37.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000268, Dolichocephaly</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000678, Dental crowding</dd></dl><dl><dt>Proximity score 0.578 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRAF1%0DTRIM37&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRIM37</a> and phenotypic similarity 0.447 to mouse mutant of TRIM37.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0005352, small cranium</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.804</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.578</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.946</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr9:g.120909365G&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=569958951">rs569958951</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs569958951&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.946</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373887.8">TRAF1:ENST00000373887.8:c.897C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540010.1">TRAF1:ENST00000540010.1:c.897C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000546084.5">TRAF1:ENST00000546084.5:c.531C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9489495</dd>
                                    
                                    <dd>CADD: 0.949 (12.920)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169548">ZNF280A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.791</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.528</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.528 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF280A%0DQRICH1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to QRICH1</a> and phenotypic similarity 0.633 to <a href="http://www.omim.org/617982">Ververi-Brady syndrome</a> associated with QRICH1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001256, Intellectual disability, mild</dd><dd>HP:0004322, Short stature - HP:0001511, Intrauterine growth retardation</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000378, Cupped ear</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.528 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF280A%0DQRICH1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to QRICH1</a> and phenotypic similarity 0.322 to mouse mutant of QRICH1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000111, cleft palate</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.791</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.528</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.22514169T&gt;G [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361666">rs361666</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361666&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.1462A&gt;C:p.(Ser488Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9939744</dd>
                                    
                                    <dd>CADD: 0.994 (22.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22514173C&gt;G [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361762">rs361762</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361762&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.994</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.1458G&gt;C:p.(Leu486Phe)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9939744</dd>
                        
                        <dd>CADD: 0.994 (22.200)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22514799C&gt;T [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=362003">rs362003</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs362003&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.994</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.832G&gt;A:p.(Asp278Asn)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.99354345</dd>
                        
                        <dd>CADD: 0.994 (21.900)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr22:g.22514452A&gt;G [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=362173">rs362173</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs362173&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.976</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.1179T&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9757897</dd>
                        
                        <dd>CADD: 0.976 (16.160)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22515418A&gt;C [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361959">rs361959</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361959&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.972</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.213T&gt;G:p.(Asn71Lys)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.97168607</dd>
                        
                        <dd>CADD: 0.972 (15.480)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22515221G&gt;T [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361580">rs361580</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361580&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.938</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.410C&gt;A:p.(Ser137Tyr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9379131</dd>
                        
                        <dd>CADD: 0.938 (12.070)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr22:g.22514449A&gt;G [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361737">rs361737</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361737&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.919</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.1182T&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9187169</dd>
                        
                        <dd>CADD: 0.919 (10.900)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22514894C&gt;T [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=362132">rs362132</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs362132&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.905</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.737G&gt;A:p.(Ser246Asn)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.90493953</dd>
                        
                        <dd>CADD: 0.905 (10.220)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22514885G&gt;C [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=362124">rs362124</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs362124&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.824</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.746C&gt;G:p.(Ala249Gly)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8244504</dd>
                        
                        <dd>CADD: 0.824 (7.556)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22515224C&gt;T [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=362011">rs362011</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs362011&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.533</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.407G&gt;A:p.(Ser136Asn)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.5326954</dd>
                        
                        <dd>CADD: 0.533 (3.304)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr22:g.22515214G&gt;A [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361611">rs361611</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361611&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.030</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.417C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.030383468</dd>
                        
                        <dd>CADD: 0.030 (0.134)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000111331">OAS3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.791</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.527</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.995</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.527 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OAS3%0DSMOC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMOC1</a> and phenotypic similarity 0.830 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1106">Microphthalmia with limb anomalies</a> associated with SMOC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002342, Intellectual disability, moderate</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0002139, Arrhinencephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0009748, Large earlobe</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000581, Blepharophimosis</dd><dd>HP:0001572, Macrodontia - HP:0001572, Macrodontia</dd></dl><dl><dt>Proximity score 0.527 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OAS3%0DSMOC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMOC1</a> and phenotypic similarity 0.566 to mouse mutant of SMOC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001406, abnormal gait</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000434, megacephaly</dd><dd>HP:0000325, Triangular face - MP:0000443, abnormal snout morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000443, abnormal snout morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.791</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.527</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.995</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.112938696A&gt;C [0/1:0/1:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.995</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000228928.12">OAS3:ENST00000228928.12:c.166A&gt;C:p.(Lys56Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000548514.1">OAS3:ENST00000548514.1:c.166A&gt;C:p.(Lys56Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000551007.1">OAS3:ENST00000551007.1:c.166A&gt;C:p.(Lys56Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9952137</dd>
                                    
                                    <dd>Polyphen2: 0.933 (P)</dd>
                                    <dd>SIFT: 0.111 (T)</dd>
                                    <dd>CADD: 0.995 (23.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000110880">CORO1C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.780</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.523</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.523 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CORO1C">CORO1C</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000783, abnormal forebrain morphology</dd><dd>HP:0000325, Triangular face - MP:0009532, decreased parotid gland size</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0003686, abnormal eye muscle morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0003686, abnormal eye muscle morphology</dd><dd>HP:0001572, Macrodontia - MP:0005274, abnormal viscerocranium morphology</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CORO1C%0DACTG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTG1</a> and phenotypic similarity 0.833 to <a href="http://www.omim.org/614583">Baraitser-Winter syndrome 2</a> associated with ACTG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0005484, Postnatal microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000377, Abnormality of the pinna</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000307, Pointed chin</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CORO1C%0DACTG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTG1</a> and phenotypic similarity 0.314 to mouse mutant of ACTG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0006325, impaired hearing</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.780</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.523</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                                <b>chr12:g.108648686GTTC&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=752584961">rs752584961</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs752584961&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261401.7">CORO1C:ENST00000261401.7:c.1221_1223del:p.(Lys407del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000420959.6">CORO1C:ENST00000420959.6:c.1380_1382del:p.(Lys460del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421578.6">CORO1C:ENST00000421578.6:c.906_908del:p.(Lys302del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541050.5">CORO1C:ENST00000541050.5:c.1221_1223del:p.(Lys407del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000549772.5">CORO1C:ENST00000549772.5:c.1239_1241del:p.(Lys413del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9945046</dd>
                                    
                                    <dd>CADD: 0.995 (22.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000048052">HDAC9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.773</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.516</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.516 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=HDAC9%0DMECP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MECP2</a> and phenotypic similarity 0.750 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with MECP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002187, Intellectual disability, profound</dd><dd>HP:0004322, Short stature - HP:0001518, Small for gestational age</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000684, Delayed eruption of teeth</dd></dl><dl><dt>Proximity score 0.516 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=HDAC9%0DMECP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MECP2</a> and phenotypic similarity 0.656 to mouse mutant of MECP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0003312, abnormal locomotor coordination</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000454, abnormal jaw morphology</dd><dd>HP:0000400, Macrotia - MP:0001963, abnormal hearing physiology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0002100, abnormal tooth morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.773</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.516</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.18634653G&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=552180292">rs552180292</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs552180292&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000401921.5">HDAC9:ENST00000401921.5:c.691G&gt;C:p.(Gly231Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405010.7">HDAC9:ENST00000405010.7:c.814G&gt;C:p.(Gly272Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000406072.5">HDAC9:ENST00000406072.5:c.775G&gt;C:p.(Gly259Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000406451.8">HDAC9:ENST00000406451.8:c.814G&gt;C:p.(Gly272Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000417496.6">HDAC9:ENST00000417496.6:c.808G&gt;C:p.(Gly270Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428307.6">HDAC9:ENST00000428307.6:c.682G&gt;C:p.(Gly228Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000432645.6">HDAC9:ENST00000432645.6:c.814G&gt;C:p.(Gly272Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441542.6">HDAC9:ENST00000441542.6:c.823G&gt;C:p.(Gly275Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456174.6">HDAC9:ENST00000456174.6:c.730G&gt;C:p.(Gly244Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524023.1">HDAC9:ENST00000524023.1:c.583G&gt;C:p.(Gly195Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622668.4">HDAC9:ENST00000622668.4:c.691G&gt;C:p.(Gly231Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.946 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.021 (D)</dd>
                                    <dd>CADD: 0.996 (23.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0257%</dd>
                                    <dd>gnomAD_E_AFR: 0.0080%</dd>
                                    <dd>gnomAD_E_AMR: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000259207">ITGB3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.766</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.512</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.259 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ITGB3">ITGB3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0009660, abnormal induced retinal neovascularization</dd><dd>HP:0000637, Long palpebral fissure - MP:0009660, abnormal induced retinal neovascularization</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.512 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ITGB3%0DFOXC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FOXC2</a> and phenotypic similarity 0.392 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=33001">Lymphedema-distichiasis syndrome</a> associated with FOXC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000656, Ectropion</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.512 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ITGB3%0DFOXC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FOXC2</a> and phenotypic similarity 0.640 to mouse mutant of FOXC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0001788, periorbital edema</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0010030, abnormal orbit morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0001344, blepharoptosis</dd><dd>HP:0001572, Macrodontia - MP:0001788, periorbital edema</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/187800">OMIM:187800</a> Bleeding disorder, platelet-type, 16, autosomal dominant - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/273800">OMIM:273800</a> Glanzmann thrombasthenia - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=849">ORPHA:849</a> Glanzmann thrombasthenia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.766</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.512</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.47292305A&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=145103142">rs145103142</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs145103142&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000559488.6">ITGB3:ENST00000559488.6:c.1427A&gt;T:p.(Asn476Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000560629.1">ITGB3:ENST00000560629.1:c.1392A&gt;T:p.(Gln464His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999432</dd>
                                    
                                    <dd>Polyphen2: 0.152 (B)</dd>
                                    <dd>Mutation Taster: 0.999 (P)</dd>
                                    <dd>SIFT: 0.054 (D)</dd>
                                    <dd>CADD: 0.984 (18.050)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000153234">NR4A2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.762</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.509</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.344 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NR4A2">NR4A2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0009746, enhanced behavioral response to xenobiotic</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NR4A2%0DPITX3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PITX3</a> and phenotypic similarity 0.322 to <a href="http://www.omim.org/610623">Cataract 11, syndromic, autosomal recessive</a> associated with PITX3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NR4A2%0DPITX3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PITX3</a> and phenotypic similarity 0.709 to mouse mutant of PITX3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0020543, decreased substantia nigra size</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000637, Long palpebral fissure - MP:0013453, enlarged lacrimal gland</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NR4A2%0DPITX3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PITX3</a> and phenotypic similarity 0.388 to fish mutant of PITX3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - ZP:0000054, abnormal(ly) decreased size head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - ZP:0001052, abnormal(ly) morphology mandibular arch skeleton</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.762</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.509</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.156329642G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=569649814">rs569649814</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs569649814&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339562.9">NR4A2:ENST00000339562.9:c.545C&gt;A:p.(Pro182His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409108.6">NR4A2:ENST00000409108.6:c.545C&gt;A:p.(Pro182His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409572.5">NR4A2:ENST00000409572.5:c.545C&gt;A:p.(Pro182His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000426264.5">NR4A2:ENST00000426264.5:c.356C&gt;A:p.(Pro119His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000429376.5">NR4A2:ENST00000429376.5:c.356C&gt;A:p.(Pro119His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.790 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.998 (28.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0100%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000141401">IMPA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.762</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.509</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=IMPA2%0DBPNT2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BPNT2</a> and phenotypic similarity 0.458 to <a href="http://www.omim.org/614078">Chondrodysplasia with joint dislocations, GPAPP type</a> associated with BPNT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000520, Proptosis</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=IMPA2%0DIMPAD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IMPAD1</a> and phenotypic similarity 0.663 to mouse mutant of IMPAD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0000547, short limbs</dd><dd>HP:0000252, Microcephaly - MP:0000435, shortened head</dd><dd>HP:0000325, Triangular face - MP:0009890, cleft secondary palate</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0009890, cleft secondary palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.762</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.509</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr18:g.12014271T&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=534318865">rs534318865</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs534318865&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000269159.8">IMPA2:ENST00000269159.8:c.388T&gt;A:p.(Phe130Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000588927.5">IMPA2:ENST00000588927.5:c.-180T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589238.5">IMPA2:ENST00000589238.5:c.-180T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.496 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.472 (T)</dd>
                                    <dd>CADD: 0.996 (24.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0101%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182957">SPATA13</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.759</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.508</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.508 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPATA13%0DSH3PXD2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SH3PXD2B</a> and phenotypic similarity 0.613 to <a href="http://www.omim.org/249420">Frank-ter Haar syndrome</a> associated with SH3PXD2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001510, Growth delay</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0020206, Simple ear</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000494, Downslanted palpebral fissures</dd><dd>HP:0001572, Macrodontia - HP:0000689, Dental malocclusion</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPATA13%0DSH3PXD2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SH3PXD2B</a> and phenotypic similarity 0.853 to mouse mutant of SH3PXD2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - MP:0001312, abnormal cornea morphology</dd><dd>HP:0001572, Macrodontia - MP:0002100, abnormal tooth morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.759</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_ELONGATION</span>
                                <b>chr13:g.24278791G&gt;GA [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=749470297">rs749470297</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs749470297&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343003.10">SPATA13:ENST00000343003.10:c.99dup:p.(Cys34Metfs*7)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382095.8">SPATA13:ENST00000382095.8:c.290-5344_290-5343insA:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382108.8">SPATA13:ENST00000382108.8:c.2165-5344_2165-5343insA:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399949.6">SPATA13:ENST00000399949.6:c.56-5344_56-5343insA:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409126.5">SPATA13:ENST00000409126.5:c.56-5344_56-5343insA:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424834.6">SPATA13:ENST00000424834.6:c.2165-5344_2165-5343insA:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382141.4">SPATA13:ENST00000382141.4:c.1799-5344_1799-5343insA:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.12097746</dd>
                                    
                                    <dd>CADD: 0.121 (0.560)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0226%</dd>
                                    <dd>gnomAD_E_AMR: 0.0033%</dd>
                                    <dd>gnomAD_E_FIN: 0.0051%</dd>
                                    <dd>gnomAD_E_SAS: 0.0037%</dd>
                                    <dd>gnomAD_G_AFR: 0.0116%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167470">MIDN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.757</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.322 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MIDN">MIDN</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000111, cleft palate</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000111, cleft palate</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MIDN%0DTP53BP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TP53BP2</a> and phenotypic similarity 0.703 to mouse mutant of TP53BP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0003742, narrow head</dd><dd>HP:0000325, Triangular face - MP:0000111, cleft palate</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001293, anophthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0001293, anophthalmia</dd><dd>HP:0001572, Macrodontia - MP:0002639, micrognathia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.757</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.1257024G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=561185140">rs561185140</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs561185140&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000300952.6">MIDN:ENST00000300952.6:c.1159G&gt;A:p.(Ala387Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000591446.6">MIDN:ENST00000591446.6:c.1159G&gt;A:p.(Ala387Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999996</dd>
                                    
                                    <dd>Polyphen2: 0.535 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.015 (D)</dd>
                                    <dd>CADD: 0.997 (24.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0117%</dd>
                                    <dd>ExAC AMR: 0.0088%</dd>
                                    <dd>gnomAD_E_AFR: 0.0070%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000253797">UTP14C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.755</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.995</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.159 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=UTP14C">UTP14C</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0001750, increased circulating follicle stimulating hormone level</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=UTP14C%0DWDR4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to WDR4</a> and phenotypic similarity 0.708 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2065">Galloway-Mowat syndrome</a> associated with WDR4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=UTP14C%0DWDR4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to WDR4</a> and phenotypic similarity 0.204 to mouse mutant of WDR4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000913, abnormal brain development</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.755</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.995</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr13:g.52029556A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=535940779">rs535940779</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs535940779&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.995</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000521776.2">UTP14C:ENST00000521776.2:c.752A&gt;G:p.(Tyr251Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000521508.2">UTP14C:ENST00000521508.2:c.*966A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000523764.1">UTP14C:ENST00000523764.1:c.*1171A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000649340.1">UTP14C:ENST00000649340.1:c.*1171A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.983 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.996 (24.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>gnomAD_E_AFR: 0.0262%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_G_AFR: 0.0343%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000126602">TRAP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.753</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.373 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TRAP1">TRAP1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0002092, abnormal eye morphology</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRAP1%0DEXT2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EXT2</a> and phenotypic similarity 0.571 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=466926">Seizures-scoliosis-macrocephaly syndrome</a> associated with EXT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001256, Intellectual disability, mild</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000356, Abnormality of the outer ear</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRAP1%0DEXT2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EXT2</a> and phenotypic similarity 0.715 to fish mutant of EXT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - ZP:0001246, abnormal(ly) morphology inner ear</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - ZP:0007219, abnormal(ly) increased thickness ceratobranchial 5 tooth</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.753</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.3663442A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=549753520">rs549753520</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs549753520&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000246957.10">TRAP1:ENST00000246957.10:c.1690T&gt;C:p.(Phe564Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538171.5">TRAP1:ENST00000538171.5:c.1531T&gt;C:p.(Phe511Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000575671.5">TRAP1:ENST00000575671.5:c.1063T&gt;C:p.(Phe355Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000407479.5">TRAP1:ENST00000407479.5:c.*5489A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.002 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.802 (T)</dd>
                                    <dd>CADD: 0.995 (23.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0399%</dd>
                                    <dd>ExAC AFR: 0.0193%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000274391">TPTE</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.752</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.508</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.508 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TPTE%0DSPRTN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SPRTN</a> and phenotypic similarity 0.652 to <a href="http://www.omim.org/616200">Ruijs-Aalfs syndrome</a> associated with SPRTN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TPTE%0DSPRTN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SPRTN</a> and phenotypic similarity 0.351 to mouse mutant of SPRTN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001304, cataract</dd><dd>HP:0000637, Long palpebral fissure - MP:0001304, cataract</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.752</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr21:g.10569454G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1810856">rs1810856</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1810856&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427445.6">TPTE:ENST00000427445.6:c.470G&gt;A:p.(Arg157Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612746.1">TPTE:ENST00000612746.1:c.170G&gt;A:p.(Arg57Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618007.4">TPTE:ENST00000618007.4:c.584G&gt;A:p.(Arg195Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622113.4">TPTE:ENST00000622113.4:c.530G&gt;A:p.(Arg177Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9922375</dd>
                                    
                                    <dd>CADD: 0.992 (21.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                    <b>chr21:g.10569526TAAG&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=113444703">rs113444703</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs113444703&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.992</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427445.6">TPTE:ENST00000427445.6:c.546_548del:p.(Arg184del)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612746.1">TPTE:ENST00000612746.1:c.246_248del:p.(Arg84del)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618007.4">TPTE:ENST00000618007.4:c.660_662del:p.(Arg222del)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622113.4">TPTE:ENST00000622113.4:c.606_608del:p.(Arg204del)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9922375</dd>
                        
                        <dd>CADD: 0.992 (21.100)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000074755">ZZEF1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.752</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.515</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.984</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.515 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ZZEF1">ZZEF1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0030610, absent teeth</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0030610, absent teeth</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.752</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.515</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.984</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.4059224T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=531836331">rs531836331</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs531836331&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.984</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381638.7">ZZEF1:ENST00000381638.7:c.4950A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98705804</dd>
                                    
                                    <dd>CADD: 0.987 (18.880)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0031%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0035%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000113522">RAD50</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.751</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.497 to <a href="http://www.omim.org/613078">Nijmegen breakage syndrome-like disorder</a> associated with RAD50.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.192 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RAD50">RAD50</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0001891, hydrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RAD50%0DXRCC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to XRCC4</a> and phenotypic similarity 0.755 to <a href="http://www.omim.org/616541">Short stature, microcephaly, and endocrine dysfunction</a> associated with XRCC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000692, Misalignment of teeth</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RAD50%0DXRCC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to XRCC4</a> and phenotypic similarity 0.298 to mouse mutant of XRCC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0004201, fetal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0000788, abnormal cerebral cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/613078">OMIM:613078</a> Nijmegen breakage syndrome-like disorder - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=145">ORPHA:145</a> Hereditary breast and ovarian cancer syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.751</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr5:g.132587565C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=555367432">rs555367432</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs555367432&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=182529%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378823.8">RAD50:ENST00000378823.8:c.760C&gt;T:p.(Arg254Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000638452.2">RAD50:ENST00000638452.2:c.463C&gt;T:p.(Arg155Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000638568.2">RAD50:ENST00000638568.2:c.463C&gt;T:p.(Arg155Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000640655.2">RAD50:ENST00000640655.2:c.463C&gt;T:p.(Arg155Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999966</dd>
                                    
                                    <dd>Polyphen2: 0.004 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.996 (24.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0056%</dd>
                                    <dd>ExAC AFR: 0.0312%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.0197%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.0114%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000133962">CATSPERB</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.751</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CATSPERB%0DCACNA1G&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CACNA1G</a> and phenotypic similarity 0.673 to <a href="http://www.omim.org/618087">Spinocerebellar ataxia 42, early-onset, severe, with neurodevelopmental deficits</a> associated with CACNA1G.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0040080, Anteverted ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0045025, Narrow palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000303, Mandibular prognathia</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CATSPERB%0DCACNA1G&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CACNA1G</a> and phenotypic similarity 0.337 to mouse mutant of CACNA1G.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001406, abnormal gait</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0004097, abnormal cerebellar cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.751</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.91723081A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=147068286">rs147068286</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs147068286&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000256343.8">CATSPERB:ENST00000256343.8:c.277T&gt;C:p.(Tyr93His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.990 (D)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.996 (23.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0079%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000173156">RHOD</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.751</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RHOD%0DOPHN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OPHN1</a> and phenotypic similarity 0.673 to <a href="http://www.omim.org/300486">Mental retardation, X-linked, with cerebellar hypoplasia and distinctive facial appearance</a> associated with OPHN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000303, Mandibular prognathia</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RHOD%0DOPHN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OPHN1</a> and phenotypic similarity 0.379 to mouse mutant of OPHN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001463, abnormal spatial learning</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0006254, thin cerebral cortex</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.751</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.67070494C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4930409">rs4930409</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs4930409&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308831.7">RHOD:ENST00000308831.7:c.400C&gt;T:p.(Arg134Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000532559.1">RHOD:ENST00000532559.1:c.202C&gt;T:p.(Arg68Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9987977</dd>
                                    
                                    <dd>CADD: 0.999 (29.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0120%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188833">ENTPD8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.749</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.999</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ENTPD8%0DPOLR3A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLR3A</a> and phenotypic similarity 0.746 to <a href="http://www.omim.org/264090">Wiedemann-Rautenstrauch syndrome</a> associated with POLR3A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia - HP:0000387, Absent earlobe</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000656, Ectropion</dd><dd>HP:0001572, Macrodontia - HP:0000668, Hypodontia</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ENTPD8%0DPOLR3A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLR3A</a> and phenotypic similarity 0.268 to mouse mutant of POLR3A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001406, abnormal gait</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.749</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.999</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_VARIANT</span>
                                <b>chr9:g.137436969CG&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=761018753">rs761018753</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs761018753&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.999</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344119.6">ENTPD8:ENST00000344119.6:c.454del:p.(Arg152Glyfs*102)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371506.6">ENTPD8:ENST00000371506.6:c.454del:p.(Arg152Glyfs*102)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000472938.5">ENTPD8:ENST00000472938.5:c.454del:p.(Arg152Glyfs*102)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99369043</dd>
                                    
                                    <dd>CADD: 0.994 (22.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC AFR: 0.0098%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000206384">COL6A6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.748</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=COL6A6%0DITGA11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ITGA11</a> and phenotypic similarity 0.715 to mouse mutant of ITGA11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0002100, abnormal tooth morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.748</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.130586601T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=531102788">rs531102788</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs531102788&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358511.10">COL6A6:ENST00000358511.10:c.4066T&gt;C:p.(Tyr1356His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.997 (D)</dd>
                                    <dd>Mutation Taster: 0.980 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.998 (26.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0102%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000166035">LIPC</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.748</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LIPC%0DACSL4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACSL4</a> and phenotypic similarity 0.750 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with ACSL4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002187, Intellectual disability, profound</dd><dd>HP:0004322, Short stature - HP:0001518, Small for gestational age</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000684, Delayed eruption of teeth</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LIPC%0DACSL4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACSL4</a> and phenotypic similarity 0.243 to mouse mutant of ACSL4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0011967, increased or absent threshold for auditory brainstem response</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/125853">OMIM:125853</a> Diabetes mellitus, noninsulin-dependent (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/614025">OMIM:614025</a> Hepatic lipase deficiency - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=140905">ORPHA:140905</a> Hyperlipidemia due to hepatic triacylglycerol lipase deficiency</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.748</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr15:g.58565217C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=188801316">rs188801316</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs188801316&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000414170.7">LIPC:ENST00000414170.7:c.1427C&gt;T:p.(Ser476Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000299022.10">LIPC:ENST00000299022.10:c.1388+1494C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356113.10">LIPC:ENST00000356113.10:c.1388+1494C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000433326.2">LIPC:ENST00000433326.2:c.1205+1494C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.878 (9.149)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>gnomAD_E_AFR: 0.0172%</dd>
                                    <dd>gnomAD_E_AMR: 0.0127%</dd>
                                    <dd>gnomAD_E_NFE: 0.0022%</dd>
                                    <dd>gnomAD_E_OTH: 0.0573%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000175224">ATG13</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.747</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.257 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ATG13">ATG13</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0010865, prenatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ATG13%0DTBC1D32&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TBC1D32</a> and phenotypic similarity 0.698 to mouse mutant of TBC1D32.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0008797, facial cleft</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001293, anophthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0001293, anophthalmia</dd><dd>HP:0001572, Macrodontia - MP:0005170, cleft upper lip</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.747</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.46645943C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=371039787">rs371039787</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs371039787&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359513.8">ATG13:ENST00000359513.8:c.224C&gt;T:p.(Ala75Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524625.5">ATG13:ENST00000524625.5:c.224C&gt;T:p.(Ala75Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000526508.5">ATG13:ENST00000526508.5:c.224C&gt;T:p.(Ala75Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000528494.5">ATG13:ENST00000528494.5:c.224C&gt;T:p.(Ala75Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529655.5">ATG13:ENST00000529655.5:c.224C&gt;T:p.(Ala75Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530500.5">ATG13:ENST00000530500.5:c.-14C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.580 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.036 (D)</dd>
                                    <dd>CADD: 0.994 (22.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000121281">ADCY7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.747</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADCY7%0DGNAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNAS</a> and phenotypic similarity 0.529 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=562">McCune-Albright syndrome</a> associated with GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0000098, Tall stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADCY7%0DGNAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNAS</a> and phenotypic similarity 0.636 to mouse mutant of GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001504, abnormal posture</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0004097, abnormal cerebellar cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0002177, abnormal outer ear morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.747</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.50315443G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=554372990">rs554372990</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs554372990&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000254235.7">ADCY7:ENST00000254235.7:c.3181G&gt;A:p.(Glu1061Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394697.7">ADCY7:ENST00000394697.7:c.3181G&gt;A:p.(Glu1061Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000673801.1">ADCY7:ENST00000673801.1:c.3181G&gt;A:p.(Glu1061Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.679 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.115 (T)</dd>
                                    <dd>CADD: 0.999 (33.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105656">ELL</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.746</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ELL%0DVPS25&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VPS25</a> and phenotypic similarity 0.684 to mouse mutant of VPS25.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0000547, short limbs</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000445, short snout</dd><dd>HP:0000400, Macrotia - MP:0002177, abnormal outer ear morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000460, mandible hypoplasia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.746</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.18521998C&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=373066896">rs373066896</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs373066896&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262809.9">ELL:ENST00000262809.9:c.58G&gt;T:p.(Gly20Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999998</dd>
                                    
                                    <dd>Polyphen2: 0.983 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.005 (D)</dd>
                                    <dd>CADD: 0.999 (29.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0111%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0415%</dd>
                                    <dd>gnomAD_E_AFR: 0.0482%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105613">MAST1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.746</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.398 to <a href="http://www.omim.org/618273">Mega-corpus-callosum syndrome with cerebellar hypoplasia and cortical malformations</a> associated with MAST1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0002365, Hypoplasia of the brainstem</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.323 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MAST1">MAST1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000851, cerebellum hypoplasia</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAST1%0DPTEN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTEN</a> and phenotypic similarity 0.735 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=744">Proteus syndrome</a> associated with PTEN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0001519, Disproportionate tall stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAST1%0DPTEN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTEN</a> and phenotypic similarity 0.450 to mouse mutant of PTEN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000434, megacephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAST1%0DPTEN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTEN</a> and phenotypic similarity 0.260 to fish mutant of PTEN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - ZP:0003725, abnormal(ly) disrupted semicircular canal formation</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/618273">OMIM:618273</a> Mega-corpus-callosum syndrome with cerebellar hypoplasia and cortical malformations - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.746</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.12864851C&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=557249365">rs557249365</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs557249365&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251472.9">MAST1:ENST00000251472.9:c.1409C&gt;A:p.(Pro470His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.996 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.995 (22.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000152990">ADGRA3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.745</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADGRA3%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.608 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=950">Acrodysostosis</a> associated with PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000248, Brachycephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000286, Epicanthus</dd><dd>HP:0001572, Macrodontia - HP:0010807, Open bite</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADGRA3%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.246 to mouse mutant of PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.745</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.22392611A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=576433232">rs576433232</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs576433232&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000334304.10">ADGRA3:ENST00000334304.10:c.2561T&gt;C:p.(Val854Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999984</dd>
                                    
                                    <dd>Polyphen2: 0.846 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.197 (T)</dd>
                                    <dd>CADD: 0.995 (23.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000264343">NOTCH2NLA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.745</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.998</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NOTCH2NLA%0DENPP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ENPP1</a> and phenotypic similarity 0.492 to <a href="http://www.omim.org/613312">Hypophosphatemic rickets, autosomal recessive, 2</a> associated with ENPP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NOTCH2NLA%0DENPP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ENPP1</a> and phenotypic similarity 0.629 to mouse mutant of ENPP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0009393, abnormal resting posture</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0004665, abnormal stapedial artery morphology</dd><dd>HP:0000325, Triangular face - MP:0030533, abnormal snout skin morphology</dd><dd>HP:0000400, Macrotia - MP:0006018, abnormal tympanic membrane morphology</dd><dd>HP:0000316, Hypertelorism - MP:0001325, abnormal retina morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0001325, abnormal retina morphology</dd><dd>HP:0001572, Macrodontia - MP:0030533, abnormal snout skin morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.745</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.998</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.146156771C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=76765512">rs76765512</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs76765512&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.998</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000362074.7">NOTCH2NLA:ENST00000362074.7:c.343G&gt;A:p.(Ala115Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579793.6">NOTCH2NLA:ENST00000579793.6:c.343G&gt;A:p.(Ala115Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9978122</dd>
                                    
                                    <dd>SIFT: 0.093 (T)</dd>
                                    <dd>CADD: 0.998 (26.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143578">CREB3L4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.745</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.995</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CREB3L4%0DHCFC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HCFC1</a> and phenotypic similarity 0.750 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with HCFC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002187, Intellectual disability, profound</dd><dd>HP:0004322, Short stature - HP:0001518, Small for gestational age</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000684, Delayed eruption of teeth</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.745</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.995</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.153968569C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=374617097">rs374617097</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs374617097&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.995</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000271889.8">CREB3L4:ENST00000271889.8:c.44C&gt;T:p.(Pro15Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368600.7">CREB3L4:ENST00000368600.7:c.44C&gt;T:p.(Pro15Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368601.5">CREB3L4:ENST00000368601.5:c.44C&gt;T:p.(Pro15Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368603.5">CREB3L4:ENST00000368603.5:c.44C&gt;T:p.(Pro15Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368607.8">CREB3L4:ENST00000368607.8:c.44C&gt;T:p.(Pro15Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000648921.1">CREB3L4:ENST00000648921.1:c.193-4866G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.997 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.004 (D)</dd>
                                    <dd>CADD: 0.995 (22.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0088%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0386%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0327%</dd>
                                    <dd>gnomAD_E_AMR: 0.0208%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171195">MUC7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.745</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MUC7%0DAHSG&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AHSG</a> and phenotypic similarity 0.681 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2850">Alopecia-intellectual disability syndrome</a> associated with AHSG.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0100840, Aplasia/Hypoplasia of the eyebrow</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MUC7%0DAHSG&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AHSG</a> and phenotypic similarity 0.340 to mouse mutant of AHSG.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0002764, short tibia</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0011967, increased or absent threshold for auditory brainstem response</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/600807">OMIM:600807</a> Asthma, protection against (susceptibility)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.745</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr4:g.70480811C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=546576850">rs546576850</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs546576850&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000304887.6">MUC7:ENST00000304887.6:c.67C&gt;T:p.(Arg23*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413702.5">MUC7:ENST00000413702.5:c.67C&gt;T:p.(Arg23*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456088.5">MUC7:ENST00000456088.5:c.67C&gt;T:p.(Arg23*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.999 (33.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000181143">MUC16</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.745</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MUC16%0DAHSG&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AHSG</a> and phenotypic similarity 0.681 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2850">Alopecia-intellectual disability syndrome</a> associated with AHSG.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0100840, Aplasia/Hypoplasia of the eyebrow</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MUC16%0DAHSG&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AHSG</a> and phenotypic similarity 0.340 to mouse mutant of AHSG.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0002764, short tibia</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0011967, increased or absent threshold for auditory brainstem response</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.745</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.8965749A&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=557723809">rs557723809</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs557723809&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397910.8">MUC16:ENST00000397910.8:c.11021T&gt;A:p.(Ile3674Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.982 (17.400)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0103%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000108852">MPP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.745</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MPP2%0DDLG3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DLG3</a> and phenotypic similarity 0.750 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with DLG3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002187, Intellectual disability, profound</dd><dd>HP:0004322, Short stature - HP:0001518, Small for gestational age</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000684, Delayed eruption of teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MPP2%0DDLG3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DLG3</a> and phenotypic similarity 0.294 to mouse mutant of DLG3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001463, abnormal spatial learning</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.745</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.43883280G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=562847000">rs562847000</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs562847000&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000269095.8">MPP2:ENST00000269095.8:c.226C&gt;T:p.(Arg76Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377184.7">MPP2:ENST00000377184.7:c.277C&gt;T:p.(Arg93Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000461854.5">MPP2:ENST00000461854.5:c.298C&gt;T:p.(Arg100Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000518766.5">MPP2:ENST00000518766.5:c.361C&gt;T:p.(Arg121Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000523501.5">MPP2:ENST00000523501.5:c.193C&gt;T:p.(Arg65Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536246.5">MPP2:ENST00000536246.5:c.193C&gt;T:p.(Arg65Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612133.4">MPP2:ENST00000612133.4:c.226C&gt;T:p.(Arg76Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622681.4">MPP2:ENST00000622681.4:c.193C&gt;T:p.(Arg65Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000625880.2">MPP2:ENST00000625880.2:c.289C&gt;T:p.(Arg97Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000520305.5">MPP2:ENST00000520305.5:c.-192C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999999</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.013 (D)</dd>
                                    <dd>CADD: 0.997 (24.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0105%</dd>
                                    <dd>gnomAD_E_AFR: 0.0133%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163879">DNALI1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.745</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNALI1%0DDCTN5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DCTN5</a> and phenotypic similarity 0.744 to mouse mutant of DCTN5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000433, microcephaly</dd><dd>HP:0000325, Triangular face - MP:0000452, abnormal mouth morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0002092, abnormal eye morphology</dd><dd>HP:0001572, Macrodontia - MP:0002639, micrognathia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.745</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.37559455G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=557499297">rs557499297</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs557499297&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000296218.8">DNALI1:ENST00000296218.8:c.422G&gt;T:p.(Gly141Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000652629.1">DNALI1:ENST00000652629.1:c.356G&gt;T:p.(Gly119Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.997 (25.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000122477">LRRC39</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.744</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRRC39%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.608 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=950">Acrodysostosis</a> associated with PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000248, Brachycephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000286, Epicanthus</dd><dd>HP:0001572, Macrodontia - HP:0010807, Open bite</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRRC39%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.246 to mouse mutant of PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.744</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.100159348A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537725710">rs537725710</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537725710&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000342895.7">LRRC39:ENST00000342895.7:c.287T&gt;C:p.(Leu96Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370137.5">LRRC39:ENST00000370137.5:c.287T&gt;C:p.(Leu96Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370138.1">LRRC39:ENST00000370138.1:c.287T&gt;C:p.(Leu96Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620882.4">LRRC39:ENST00000620882.4:c.287T&gt;C:p.(Leu96Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999474</dd>
                                    
                                    <dd>Polyphen2: 0.478 (P)</dd>
                                    <dd>Mutation Taster: 0.999 (P)</dd>
                                    <dd>SIFT: 0.262 (T)</dd>
                                    <dd>CADD: 0.994 (22.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000107731">UNC5B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.744</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=UNC5B%0DIQSEC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IQSEC2</a> and phenotypic similarity 0.750 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with IQSEC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002187, Intellectual disability, profound</dd><dd>HP:0004322, Short stature - HP:0001518, Small for gestational age</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000684, Delayed eruption of teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=UNC5B%0DIQSEC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IQSEC2</a> and phenotypic similarity 0.382 to mouse mutant of IQSEC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0008428, abnormal spatial working memory</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0012459, increased dentate gyrus size</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.744</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.71296632T&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=145620917">rs145620917</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs145620917&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335350.10">UNC5B:ENST00000335350.10:c.2380T&gt;G:p.(Phe794Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373192.4">UNC5B:ENST00000373192.4:c.2347T&gt;G:p.(Phe783Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.999 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 1.000 (34.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000160796">NBEAL2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.744</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NBEAL2%0DITGB1BP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ITGB1BP1</a> and phenotypic similarity 0.704 to mouse mutant of ITGB1BP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0030189, broad snout</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0002750, exophthalmos</dd><dd>HP:0000637, Long palpebral fissure - MP:0002750, exophthalmos</dd><dd>HP:0001572, Macrodontia - MP:0030189, broad snout</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/139090">OMIM:139090</a> Gray platelet syndrome - autosomal dominant/recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=721">ORPHA:721</a> Gray platelet syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.744</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.46989374C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=549465321">rs549465321</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs549465321&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000450053.8">NBEAL2:ENST00000450053.8:c.466C&gt;T:p.(Arg156Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000651747.1">NBEAL2:ENST00000651747.1:c.445C&gt;T:p.(Arg149Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.083 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.143 (T)</dd>
                                    <dd>CADD: 0.994 (22.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC NFE: 0.0053%</dd>
                                    <dd>ExAC SAS: 0.0112%</dd>
                                    <dd>gnomAD_E_EAS: 0.0078%</dd>
                                    <dd>gnomAD_E_NFE: 0.0013%</dd>
                                    <dd>gnomAD_E_SAS: 0.0080%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000168090">COPS6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.744</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.995</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.380 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=COPS6">COPS6</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - ZP:0000054, abnormal(ly) decreased size head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - ZP:0001965, abnormal(ly) decreased size mandibular arch skeleton</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=COPS6%0DERCC8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ERCC8</a> and phenotypic similarity 0.604 to <a href="http://www.omim.org/216400">Cockayne syndrome, type A</a> associated with ERCC8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0008850, Severe postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000377, Abnormality of the pinna</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000685, Hypoplasia of teeth</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=COPS6%0DERCC8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ERCC8</a> and phenotypic similarity 0.335 to mouse mutant of ERCC8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0004736, abnormal distortion product otoacoustic emission</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.744</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.995</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                                <b>chr7:g.100088992TGGCGGCGGC&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=571882095">rs571882095</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs571882095&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.995</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000303904.8">COPS6:ENST00000303904.8:c.15_23del:p.(Ala8_Ala10del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000418625.5">COPS6:ENST00000418625.5:c.15_23del:p.(Ala8_Ala10del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99475193</dd>
                                    
                                    <dd>CADD: 0.995 (22.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165643">SOHLH1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.744</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.357 to <a href="http://www.omim.org/617690">Ovarian dysgenesis 5</a> associated with SOHLH1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SOHLH1%0DSSBP3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SSBP3</a> and phenotypic similarity 0.609 to mouse mutant of SSBP3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000433, microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/617690">OMIM:617690</a> Ovarian dysgenesis 5 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/618115">OMIM:618115</a> Spermatogenic failure 32 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=399805">ORPHA:399805</a> Male infertility with azoospermia or oligozoospermia due to single gene mutation</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.744</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.135698445G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=572151479">rs572151479</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs572151479&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298466.9">SOHLH1:ENST00000298466.9:c.229C&gt;T:p.(Arg77Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000425225.2">SOHLH1:ENST00000425225.2:c.229C&gt;T:p.(Arg77Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.998 (D)</dd>
                                    <dd>Mutation Taster: 0.852</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.999 (29.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135905">DOCK10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.744</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DOCK10%0DOPHN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OPHN1</a> and phenotypic similarity 0.673 to <a href="http://www.omim.org/300486">Mental retardation, X-linked, with cerebellar hypoplasia and distinctive facial appearance</a> associated with OPHN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000303, Mandibular prognathia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DOCK10%0DOPHN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OPHN1</a> and phenotypic similarity 0.379 to mouse mutant of OPHN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001463, abnormal spatial learning</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0006254, thin cerebral cortex</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.744</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.224982315C&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=574101643">rs574101643</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs574101643&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000645028.1">DOCK10:ENST00000645028.1:c.54G&gt;T:p.(Gln18His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258390.11">DOCK10:ENST00000258390.11:c.124-50647G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.990 (20.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000102001">CACNA1F</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.744</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.155 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=215">Congenital stationary night blindness</a> associated with CACNA1F.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0007766, Optic disc hypoplasia</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.455 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CACNA1F">CACNA1F</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005253, abnormal eye physiology</dd><dd>HP:0000637, Long palpebral fissure - MP:0005253, abnormal eye physiology</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CACNA1F%0DCACNA1G&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CACNA1G</a> and phenotypic similarity 0.673 to <a href="http://www.omim.org/618087">Spinocerebellar ataxia 42, early-onset, severe, with neurodevelopmental deficits</a> associated with CACNA1G.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0040080, Anteverted ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0045025, Narrow palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000303, Mandibular prognathia</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CACNA1F%0DCACNA1G&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CACNA1G</a> and phenotypic similarity 0.337 to mouse mutant of CACNA1G.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001406, abnormal gait</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0004097, abnormal cerebellar cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/300071">OMIM:300071</a> Night blindness, congenital stationary (incomplete), 2A, X-linked - X-linked</dd>
<dd><a href="http://www.omim.org/entry/300476">OMIM:300476</a> Cone-rod dystrophy, X-linked, 3 - X-linked recessive</dd>
<dd><a href="http://www.omim.org/entry/300600">OMIM:300600</a> Aland Island eye disease - X-linked</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=178333">ORPHA:178333</a> Åland Islands eye disease</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1872">ORPHA:1872</a> Cone rod dystrophy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=215">ORPHA:215</a> Congenital stationary night blindness</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.744</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.49230276A&gt;G [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=202029187">rs202029187</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs202029187&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=850152%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000323022.9">CACNA1F:ENST00000323022.9:c.761T&gt;C:p.(Ile254Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376251.5">CACNA1F:ENST00000376251.5:c.566T&gt;C:p.(Ile189Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376265.2">CACNA1F:ENST00000376265.2:c.761T&gt;C:p.(Ile254Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.991 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.998 (28.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0530%</dd>
                                    <dd>TOPMed: 0.0287%</dd>
                                    <dd>ESP AA: 0.0261%</dd>
                                    <dd>ESP All: 0.0095%</dd>
                                    <dd>ExAC AFR: 0.0457%</dd>
                                    <dd>ExAC AMR: 0.0147%</dd>
                                    <dd>ExAC NFE: 0.0059%</dd>
                                    <dd>ExAC SAS: 0.0144%</dd>
                                    <dd>gnomAD_E_AFR: 0.0347%</dd>
                                    <dd>gnomAD_E_AMR: 0.0077%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_SAS: 0.0114%</dd>
                                    <dd>gnomAD_G_FIN: 0.0377%</dd>
                                    <dd>gnomAD_G_NFE: 0.0191%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000181786">ACTL9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.743</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACTL9%0DACTG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTG1</a> and phenotypic similarity 0.833 to <a href="http://www.omim.org/614583">Baraitser-Winter syndrome 2</a> associated with ACTG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0005484, Postnatal microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000377, Abnormality of the pinna</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000307, Pointed chin</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACTL9%0DACTG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTG1</a> and phenotypic similarity 0.314 to mouse mutant of ACTG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0006325, impaired hearing</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.711</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.978</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.8698376G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=534855327">rs534855327</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs534855327&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.978</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324436.5">ACTL9:ENST00000324436.5:c.326C&gt;A:p.(Thr109Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612068.1">ACTL9:ENST00000612068.1:c.326C&gt;A:p.(Thr109Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98107654</dd>
                                    
                                    <dd>SIFT: 0.305 (T)</dd>
                                    <dd>CADD: 0.981 (17.230)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>gnomAD_E_AFR: 0.0071%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.743</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.8698550G&gt;A [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=10410943">rs10410943</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs10410943&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324436.5">ACTL9:ENST00000324436.5:c.152C&gt;T:p.(Ala51Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612068.1">ACTL9:ENST00000612068.1:c.152C&gt;T:p.(Ala51Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99628466</dd>
                                    
                                    <dd>CADD: 0.996 (24.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000066248">NGEF</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.742</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NGEF%0DOPHN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OPHN1</a> and phenotypic similarity 0.673 to <a href="http://www.omim.org/300486">Mental retardation, X-linked, with cerebellar hypoplasia and distinctive facial appearance</a> associated with OPHN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000303, Mandibular prognathia</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NGEF%0DOPHN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OPHN1</a> and phenotypic similarity 0.379 to mouse mutant of OPHN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001463, abnormal spatial learning</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0006254, thin cerebral cortex</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.742</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.232920489G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=559278273">rs559278273</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs559278273&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264051.8">NGEF:ENST00000264051.8:c.623C&gt;T:p.(Ser208Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373552.8">NGEF:ENST00000373552.8:c.347C&gt;T:p.(Ser116Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409079.1">NGEF:ENST00000409079.1:c.347C&gt;T:p.(Ser116Phe)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.995</dd>
                                    
                                    <dd>Polyphen2: 0.770 (P)</dd>
                                    <dd>Mutation Taster: 0.693</dd>
                                    <dd>SIFT: 0.005 (D)</dd>
                                    <dd>CADD: 0.994 (22.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130513">GDF15</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.740</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.252 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GDF15">GDF15</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001405, impaired coordination</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GDF15%0DDLK1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DLK1</a> and phenotypic similarity 0.691 to mouse mutant of DLK1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005287, narrow eye opening</dd><dd>HP:0000637, Long palpebral fissure - MP:0005287, narrow eye opening</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.740</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.18388679G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537315987">rs537315987</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537315987&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000252809.3">GDF15:ENST00000252809.3:c.671G&gt;T:p.(Gly224Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 1.000 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.999 (28.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0048%</dd>
                                    <dd>ExAC AFR: 0.0253%</dd>
                                    <dd>gnomAD_E_AFR: 0.0209%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000095951">HIVEP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.740</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=HIVEP1%0DBMP4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BMP4</a> and phenotypic similarity 0.650 to <a href="http://www.omim.org/607932">Microphthalmia, syndromic 6</a> associated with BMP4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001508, Failure to thrive</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000278, Retrognathia</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=HIVEP1%0DBMP4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BMP4</a> and phenotypic similarity 0.568 to mouse mutant of BMP4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0001333, absent optic nerve</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0000042, abnormal organ of Corti morphology</dd><dd>HP:0000316, Hypertelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0002092, abnormal eye morphology</dd><dd>HP:0001572, Macrodontia - MP:0004380, short frontal bone</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.740</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr6:g.12163565G&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=374937963">rs374937963</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs374937963&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379388.7">HIVEP1:ENST00000379388.7:c.7261G&gt;C:p.(Ala2421Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541134.5">HIVEP1:ENST00000541134.5:c.7258G&gt;C:p.(Ala2420Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000627968.2">HIVEP1:ENST00000627968.2:c.856G&gt;C:p.(Ala286Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.003 (D)</dd>
                                    <dd>CADD: 0.996 (24.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ESP AA: 0.0239%</dd>
                                    <dd>ESP All: 0.0079%</dd>
                                    <dd>ExAC AFR: 0.0408%</dd>
                                    <dd>gnomAD_E_AFR: 0.0327%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000174125">TLR1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.740</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TLR1%0DCHUK&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CHUK</a> and phenotypic similarity 0.479 to <a href="http://www.omim.org/613630">Cocoon syndrome</a> associated with CHUK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0009892, Anotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0009939, Mandibular aplasia</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TLR1%0DCHUK&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CHUK</a> and phenotypic similarity 0.824 to mouse mutant of CHUK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0000547, short limbs</dd><dd>HP:0000252, Microcephaly - MP:0000435, shortened head</dd><dd>HP:0000325, Triangular face - MP:0000445, short snout</dd><dd>HP:0000400, Macrotia - MP:0003142, anotia</dd><dd>HP:0000316, Hypertelorism - MP:0005193, abnormal anterior eye segment morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0001340, abnormal eyelid morphology</dd><dd>HP:0001572, Macrodontia - MP:0005358, abnormal incisor morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.740</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.38797820T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=535191236">rs535191236</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs535191236&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308979.7">TLR1:ENST00000308979.7:c.1012A&gt;G:p.(Met338Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000502213.6">TLR1:ENST00000502213.6:c.1012A&gt;G:p.(Met338Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.996</dd>
                                    
                                    <dd>Polyphen2: 0.206 (B)</dd>
                                    <dd>Mutation Taster: 0.518</dd>
                                    <dd>SIFT: 0.004 (D)</dd>
                                    <dd>CADD: 0.993 (21.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0098%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169122">FAM110B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.740</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.247 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FAM110B">FAM110B</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0005316, abnormal response to tactile stimuli</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FAM110B%0DGJA1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GJA1</a> and phenotypic similarity 0.783 to <a href="http://www.omim.org/257850">Oculodentodigital dysplasia, autosomal recessive</a> associated with GJA1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000248, Brachycephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0009748, Large earlobe</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000675, Macrodontia of permanent maxillary central incisor</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FAM110B%0DGJA1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GJA1</a> and phenotypic similarity 0.705 to mouse mutant of GJA1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0005352, small cranium</dd><dd>HP:0000325, Triangular face - MP:0030190, small snout</dd><dd>HP:0000400, Macrotia - MP:0006020, decreased tympanic ring size</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000116, abnormal tooth development</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.740</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr8:g.58146520C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=556320056">rs556320056</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs556320056&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361488.7">FAM110B:ENST00000361488.7:c.290C&gt;T:p.(Thr97Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999979</dd>
                                    
                                    <dd>Polyphen2: 0.895 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.008 (D)</dd>
                                    <dd>CADD: 0.996 (23.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0201%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0199%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104901">DKKL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.739</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DKKL1%0DPORCN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PORCN</a> and phenotypic similarity 0.623 to <a href="http://www.omim.org/305600">Focal dermal hypoplasia</a> associated with PORCN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000377, Abnormality of the pinna</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000689, Dental malocclusion</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DKKL1%0DPORCN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PORCN</a> and phenotypic similarity 0.544 to mouse mutant of PORCN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0012132, abnormal midbrain-hindbrain boundary morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005262, coloboma</dd><dd>HP:0000637, Long palpebral fissure - MP:0009264, failure of eyelid fusion</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.739</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.49374826G&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=144902139">rs144902139</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs144902139&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000221498.7">DKKL1:ENST00000221498.7:c.527G&gt;C:p.(Arg176Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000594268.1">DKKL1:ENST00000594268.1:c.101G&gt;C:p.(Arg34Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.997</dd>
                                    
                                    <dd>Polyphen2: 0.899 (P)</dd>
                                    <dd>SIFT: 0.003 (D)</dd>
                                    <dd>CADD: 0.992 (21.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0100%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000155760">FZD7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.739</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.988</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FZD7%0DROR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ROR2</a> and phenotypic similarity 0.824 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1507">Autosomal recessive Robinow syndrome</a> associated with ROR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0008873, Disproportionate short-limb short stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000368, Low-set, posteriorly rotated ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0011069, Increased number of teeth</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FZD7%0DROR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ROR2</a> and phenotypic similarity 0.883 to mouse mutant of ROR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0004355, short radius</dd><dd>HP:0000252, Microcephaly - MP:0000441, increased cranium width</dd><dd>HP:0000325, Triangular face - MP:0004726, abnormal nasal capsule morphology</dd><dd>HP:0000400, Macrotia - MP:0030321, abnormal tegmen tympani morphology</dd><dd>HP:0000316, Hypertelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - MP:0001300, ocular hypertelorism</dd><dd>HP:0001572, Macrodontia - MP:0000454, abnormal jaw morphology</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FZD7%0DROR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ROR2</a> and phenotypic similarity 0.332 to fish mutant of ROR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - ZP:0000054, abnormal(ly) decreased size head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.739</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.988</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.202034788C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=538318441">rs538318441</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs538318441&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.988</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000286201.2">FZD7:ENST00000286201.2:c.141C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99066746</dd>
                                    
                                    <dd>CADD: 0.991 (20.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197410">DCHS2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.738</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DCHS2%0DFAT4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FAT4</a> and phenotypic similarity 0.681 to <a href="http://www.omim.org/615546">Van Maldergem syndrome 2</a> associated with FAT4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0001510, Growth delay</dd><dd>HP:0000252, Microcephaly - HP:0002079, Hypoplasia of the corpus callosum</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0008551, Microtia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000689, Dental malocclusion</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DCHS2%0DFAT4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FAT4</a> and phenotypic similarity 0.333 to mouse mutant of FAT4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0000031, abnormal cochlea morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.738</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.154490161C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=539141275">rs539141275</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs539141275&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339452.2">DCHS2:ENST00000339452.2:c.1195G&gt;A:p.(Val399Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357232.10">DCHS2:ENST00000357232.10:c.1195G&gt;A:p.(Val399Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.997</dd>
                                    
                                    <dd>SIFT: 0.003 (D)</dd>
                                    <dd>CADD: 0.996 (24.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>gnomAD_E_AFR: 0.0149%</dd>
                                    <dd>gnomAD_E_AMR: 0.0126%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000106052">TAX1BP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.736</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.282 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TAX1BP1">TAX1BP1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0008489, slow postnatal weight gain</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TAX1BP1%0DTRAF6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRAF6</a> and phenotypic similarity 0.457 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1810">Autosomal dominant hypohidrotic ectodermal dysplasia</a> associated with TRAF6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0006482, Abnormality of dental morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TAX1BP1%0DTRAF6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRAF6</a> and phenotypic similarity 0.635 to mouse mutant of TRAF6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000121, failure of tooth eruption</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0013385, abnormal Meibomian gland development</dd><dd>HP:0000637, Long palpebral fissure - MP:0013385, abnormal Meibomian gland development</dd><dd>HP:0001572, Macrodontia - MP:0000121, failure of tooth eruption</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.736</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                                <b>chr7:g.27769713TGTTAAA&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=761547882">rs761547882</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs761547882&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265393.10">TAX1BP1:ENST00000265393.10:c.497_502del:p.(Lys166_Leu167del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396319.6">TAX1BP1:ENST00000396319.6:c.497_502del:p.(Lys166_Leu167del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409980.5">TAX1BP1:ENST00000409980.5:c.497_502del:p.(Lys166_Leu167del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000433216.6">TAX1BP1:ENST00000433216.6:c.26_31del:p.(Lys9_Leu10del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543117.5">TAX1BP1:ENST00000543117.5:c.497_502del:p.(Lys166_Leu167del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9930817</dd>
                                    
                                    <dd>CADD: 0.993 (21.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC AFR: 0.0099%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000101825">MXRA5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.736</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MXRA5%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.608 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=950">Acrodysostosis</a> associated with PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000248, Brachycephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000286, Epicanthus</dd><dd>HP:0001572, Macrodontia - HP:0010807, Open bite</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MXRA5%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.246 to mouse mutant of PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.736</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.3310306C&gt;A [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=777090401">rs777090401</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs777090401&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000217939.7">MXRA5:ENST00000217939.7:c.7897G&gt;T:p.(Val2633Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9957342</dd>
                                    
                                    <dd>Polyphen2: 0.791 (P)</dd>
                                    <dd>SIFT: 0.040 (D)</dd>
                                    <dd>CADD: 0.996 (23.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0265%</dd>
                                    <dd>TOPMed: 0.0265%</dd>
                                    <dd>gnomAD_E_AFR: 0.0083%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000180113">TDRD6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.736</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TDRD6%0DFMR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FMR1</a> and phenotypic similarity 0.628 to <a href="http://www.omim.org/300624">Fragile X syndrome</a> associated with FMR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002342, Intellectual disability, moderate</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000303, Mandibular prognathia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TDRD6%0DFMR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FMR1</a> and phenotypic similarity 0.434 to mouse mutant of FMR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001463, abnormal spatial learning</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0012144, decreased b wave amplitude</dd><dd>HP:0000637, Long palpebral fissure - MP:0012144, decreased b wave amplitude</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.736</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr6:g.46693641C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201037130">rs201037130</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201037130&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000316081.11">TDRD6:ENST00000316081.11:c.5513C&gt;T:p.(Pro1838Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544460.5">TDRD6:ENST00000544460.5:c.5513C&gt;T:p.(Pro1838Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.948 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.992 (21.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>UK10K: 0.0529%</dd>
                                    <dd>ExAC AFR: 0.0099%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr6:g.46690180A&gt;G [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537390407">rs537390407</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537390407&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.091</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000316081.11">TDRD6:ENST00000316081.11:c.2052A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544460.5">TDRD6:ENST00000544460.5:c.2052A&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.09155214</dd>
                        
                        <dd>CADD: 0.092 (0.417)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0200%</dd>
                        <dd>TOPMed: 0.0016%</dd>
                        <dd>ExAC AFR: 0.0101%</dd>
                        <dd>gnomAD_E_AFR: 0.0133%</dd>
                        <dd>gnomAD_E_AMR: 0.0030%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000095203">EPB41L4B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.736</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=EPB41L4B%0DGNAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNAS</a> and phenotypic similarity 0.529 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=562">McCune-Albright syndrome</a> associated with GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0000098, Tall stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=EPB41L4B%0DGNAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNAS</a> and phenotypic similarity 0.636 to mouse mutant of GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001504, abnormal posture</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0004097, abnormal cerebellar cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0002177, abnormal outer ear morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.736</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.109217071G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=536573534">rs536573534</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs536573534&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374566.7">EPB41L4B:ENST00000374566.7:c.1484C&gt;T:p.(Pro495Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999</dd>
                                    
                                    <dd>Polyphen2: 0.410 (B)</dd>
                                    <dd>Mutation Taster: 0.996 (P)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.996 (23.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>ExAC AFR: 0.0102%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_G_AFR: 0.0458%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000121377">TAS2R7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.736</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TAS2R7%0DGNAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNAS</a> and phenotypic similarity 0.529 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=562">McCune-Albright syndrome</a> associated with GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0000098, Tall stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TAS2R7%0DGNAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNAS</a> and phenotypic similarity 0.636 to mouse mutant of GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001504, abnormal posture</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0004097, abnormal cerebellar cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0002177, abnormal outer ear morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.736</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.10801705T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=569111567">rs569111567</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs569111567&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000240687.2">TAS2R7:ENST00000240687.2:c.866A&gt;G:p.(Asn289Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9951022</dd>
                                    
                                    <dd>Polyphen2: 0.764 (P)</dd>
                                    <dd>SIFT: 0.041 (D)</dd>
                                    <dd>CADD: 0.995 (23.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104517">UBR5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.734</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.283 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=UBR5">UBR5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0000913, abnormal brain development</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=UBR5%0DGTF2E2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GTF2E2</a> and phenotypic similarity 0.624 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=33364">Trichothiodystrophy</a> associated with GTF2E2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001511, Intrauterine growth retardation</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000411, Protruding ear</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000656, Ectropion</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.734</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr8:g.102280462C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=376712641">rs376712641</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs376712641&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000220959.8">UBR5:ENST00000220959.8:c.5933G&gt;A:p.(Arg1978Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000520539.6">UBR5:ENST00000520539.6:c.5933G&gt;A:p.(Arg1978Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000521922.5">UBR5:ENST00000521922.5:c.5915G&gt;A:p.(Arg1972Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999998</dd>
                                    
                                    <dd>Polyphen2: 0.614 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.171 (T)</dd>
                                    <dd>CADD: 0.995 (23.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0159%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0770%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 0.0719%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000122674">CCZ1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.734</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.991</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CCZ1%0DGDI1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GDI1</a> and phenotypic similarity 0.750 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with GDI1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002187, Intellectual disability, profound</dd><dd>HP:0004322, Short stature - HP:0001518, Small for gestational age</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000684, Delayed eruption of teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CCZ1%0DGDI1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GDI1</a> and phenotypic similarity 0.366 to mouse mutant of GDI1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001454, abnormal cued conditioning behavior</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0002761, abnormal hippocampal mossy fiber morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.734</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.991</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.5905205A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=369436541">rs369436541</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs369436541&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.991</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000325974.9">CCZ1:ENST00000325974.9:c.634A&gt;G:p.(Arg212Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000628813.2">CCZ1:ENST00000628813.2:c.*303A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99979</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.105 (T)</dd>
                                    <dd>CADD: 0.995 (22.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0151%</dd>
                                    <dd>ESP AA: 0.0234%</dd>
                                    <dd>ESP All: 0.0078%</dd>
                                    <dd>gnomAD_E_AFR: 0.0499%</dd>
                                    <dd>gnomAD_E_AMR: 0.0153%</dd>
                                    <dd>gnomAD_G_AFR: 0.0258%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000147256">ARHGAP36</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.733</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARHGAP36%0DFGD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FGD1</a> and phenotypic similarity 0.634 to <a href="http://www.omim.org/305400">Mental retardation, X-linked syndromic 16</a> associated with FGD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001256, Intellectual disability, mild</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0009748, Large earlobe</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia - HP:0000668, Hypodontia</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARHGAP36%0DFGD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FGD1</a> and phenotypic similarity 0.385 to mouse mutant of FGD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001297, microphthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0001297, microphthalmia</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.733</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.131083902G&gt;A [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=148485340">rs148485340</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs148485340&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000276211.10">ARHGAP36:ENST00000276211.10:c.488G&gt;A:p.(Arg163Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370921.1">ARHGAP36:ENST00000370921.1:c.80G&gt;A:p.(Arg27Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370922.5">ARHGAP36:ENST00000370922.5:c.452G&gt;A:p.(Arg151Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000639280.1">ARHGAP36:ENST00000639280.1:c.80G&gt;A:p.(Arg27Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99601895</dd>
                                    
                                    <dd>Polyphen2: 0.986 (D)</dd>
                                    <dd>SIFT: 0.006 (D)</dd>
                                    <dd>CADD: 0.996 (24.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0265%</dd>
                                    <dd>TOPMed: 0.0151%</dd>
                                    <dd>ESP AA: 0.0522%</dd>
                                    <dd>ESP All: 0.0189%</dd>
                                    <dd>ExAC AFR: 0.0473%</dd>
                                    <dd>gnomAD_E_AFR: 0.0408%</dd>
                                    <dd>gnomAD_G_AFR: 0.0338%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162881">OXER1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.733</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.990</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OXER1%0DLPAR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LPAR1</a> and phenotypic similarity 0.900 to mouse mutant of LPAR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001255, decreased body height</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000445, short snout</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - MP:0001300, ocular hypertelorism</dd><dd>HP:0001572, Macrodontia - MP:0004831, long incisors</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.733</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.990</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.42763588G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=541716654">rs541716654</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs541716654&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378661.3">OXER1:ENST00000378661.3:c.592C&gt;T:p.(His198Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.993</dd>
                                    
                                    <dd>Polyphen2: 0.989 (D)</dd>
                                    <dd>SIFT: 0.007 (D)</dd>
                                    <dd>CADD: 0.993 (21.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0225%</dd>
                                    <dd>gnomAD_E_AFR: 0.0077%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136488">CSH1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.733</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.304 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CSH1">CSH1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001938, delayed sexual maturation</dd><dd>HP:0000252, Microcephaly - MP:0005361, small pituitary gland</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSH1%0DLIFR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LIFR</a> and phenotypic similarity 0.630 to <a href="http://www.omim.org/601559">Stuve-Wiedemann syndrome/Schwartz-Jampel type 2 syndrome</a> associated with LIFR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000682, Abnormality of dental enamel</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSH1%0DLIFR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LIFR</a> and phenotypic similarity 0.211 to mouse mutant of LIFR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0002152, abnormal brain morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.733</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr17:g.63895426G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2320124">rs2320124</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2320124&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329882.8">CSH1:ENST00000329882.8:c.503C&gt;A:p.(Ser168*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000316193.13">CSH1:ENST00000316193.13:c.456+47C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453363.7">CSH1:ENST00000453363.7:c.172-207C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000610991.1">CSH1:ENST00000610991.1:c.189+47C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.003 (0.011)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0799%</dd>
                                    <dd>ESP AA: 0.0228%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AFR: 0.0291%</dd>
                                    <dd>ExAC EAS: 0.0578%</dd>
                                    <dd>ExAC NFE: 0.0330%</dd>
                                    <dd>ExAC SAS: 0.0485%</dd>
                                    <dd>gnomAD_E_AFR: 0.0459%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0754%</dd>
                                    <dd>gnomAD_E_NFE: 0.0313%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0357%</dd>
                                    <dd>gnomAD_G_AFR: 0.0117%</dd>
                                    <dd>gnomAD_G_EAS: 0.0618%</dd>
                                    <dd>gnomAD_G_NFE: 0.0402%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000125734">GPR108</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.732</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.990</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.296 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GPR108">GPR108</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0002764, short tibia</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GPR108%0DGNAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNAS</a> and phenotypic similarity 0.529 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=562">McCune-Albright syndrome</a> associated with GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0000098, Tall stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GPR108%0DGNAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNAS</a> and phenotypic similarity 0.636 to mouse mutant of GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001504, abnormal posture</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0004097, abnormal cerebellar cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0002177, abnormal outer ear morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.732</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.990</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                                <b>chr19:g.6736988A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=541645357">rs541645357</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs541645357&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430424.8">GPR108:ENST00000430424.8:c.-607+2T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264080.11">GPR108:ENST00000264080.11:c.121-277T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8240051</dd>
                                    
                                    <dd>CADD: 0.824 (7.545)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0080%</dd>
                                    <dd>gnomAD_G_AFR: 0.0688%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204120">GIGYF2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.732</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.990</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.270 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=411602">Hereditary late-onset Parkinson disease</a> associated with GIGYF2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001824, Weight loss</dd><dd>HP:0000252, Microcephaly - HP:0002120, Cerebral cortical atrophy</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.252 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GIGYF2">GIGYF2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001405, impaired coordination</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GIGYF2%0DDCHS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DCHS1</a> and phenotypic similarity 0.681 to <a href="http://www.omim.org/601390">Van Maldergem syndrome 1</a> associated with DCHS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0001510, Growth delay</dd><dd>HP:0000252, Microcephaly - HP:0002079, Hypoplasia of the corpus callosum</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0008551, Microtia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000689, Dental malocclusion</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GIGYF2%0DDCHS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DCHS1</a> and phenotypic similarity 0.367 to mouse mutant of DCHS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0000031, abnormal cochlea morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/607688">OMIM:607688</a> Parkinson disease 11 (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=411602">ORPHA:411602</a> Hereditary late-onset Parkinson disease</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.732</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.990</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.232815714C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=552038398">rs552038398</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs552038398&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373563.8">GIGYF2:ENST00000373563.8:c.2185C&gt;G:p.(Leu729Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409196.7">GIGYF2:ENST00000409196.7:c.2167C&gt;G:p.(Leu723Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409451.7">GIGYF2:ENST00000409451.7:c.2248C&gt;G:p.(Leu750Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409480.5">GIGYF2:ENST00000409480.5:c.2251C&gt;G:p.(Leu751Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409547.5">GIGYF2:ENST00000409547.5:c.2185C&gt;G:p.(Leu729Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000629305.2">GIGYF2:ENST00000629305.2:c.2251C&gt;G:p.(Leu751Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99369043</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>Mutation Taster: 0.993 (P)</dd>
                                    <dd>SIFT: 0.057 (D)</dd>
                                    <dd>CADD: 0.994 (22.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC AFR: 0.0295%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr2:g.232794920A&gt;G [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=547560862">rs547560862</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs547560862&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.946</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373563.8">GIGYF2:ENST00000373563.8:c.1455A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409196.7">GIGYF2:ENST00000409196.7:c.1437A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409451.7">GIGYF2:ENST00000409451.7:c.1518A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409480.5">GIGYF2:ENST00000409480.5:c.1521A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409547.5">GIGYF2:ENST00000409547.5:c.1455A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000629305.2">GIGYF2:ENST00000629305.2:c.1521A&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9489495</dd>
                        
                        <dd>CADD: 0.949 (12.920)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0200%</dd>
                        <dd>TOPMed: 0.0200%</dd>
                        <dd>ExAC AFR: 0.0096%</dd>
                        <dd>gnomAD_E_AFR: 0.0065%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000240583">AQP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.732</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.337 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=AQP1">AQP1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0005202, lethargy</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=AQP1%0DTHBS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to THBS1</a> and phenotypic similarity 0.745 to mouse mutant of THBS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001312, abnormal cornea morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0005287, narrow eye opening</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.732</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.30922616T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=371391537">rs371391537</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs371391537&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311813.10">AQP1:ENST00000311813.10:c.602T&gt;C:p.(Val201Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409611.1">AQP1:ENST00000409611.1:c.449T&gt;C:p.(Val150Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409899.5">AQP1:ENST00000409899.5:c.257T&gt;C:p.(Val86Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441328.6">AQP1:ENST00000441328.6:c.353T&gt;C:p.(Val118Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000652696.1">AQP1:ENST00000652696.1:c.602T&gt;C:p.(Val201Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000509504.2">AQP1:ENST00000509504.2:c.1140T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9969097</dd>
                                    
                                    <dd>Polyphen2: 0.312 (B)</dd>
                                    <dd>Mutation Taster: 0.978 (P)</dd>
                                    <dd>SIFT: 0.047 (D)</dd>
                                    <dd>CADD: 0.997 (25.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0480%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_AFR: 0.0457%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000142207">URB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.729</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=URB1%0DGRHL2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GRHL2</a> and phenotypic similarity 0.483 to <a href="http://www.omim.org/616029">Ectodermal dysplasia/short stature syndrome</a> associated with GRHL2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000668, Hypodontia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=URB1%0DGRHL2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GRHL2</a> and phenotypic similarity 0.703 to mouse mutant of GRHL2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000914, exencephaly</dd><dd>HP:0000325, Triangular face - MP:0008797, facial cleft</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005262, coloboma</dd><dd>HP:0000637, Long palpebral fissure - MP:0005262, coloboma</dd><dd>HP:0001572, Macrodontia - MP:0008797, facial cleft</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=URB1%0DGRHL2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GRHL2</a> and phenotypic similarity 0.182 to fish mutant of GRHL2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - ZP:0008189, abnormal(ly) absent otolith</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.729</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr21:g.32341485A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=566281589">rs566281589</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs566281589&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382751.4">URB1:ENST00000382751.4:c.4297T&gt;C:p.(Phe1433Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9948714</dd>
                                    
                                    <dd>Polyphen2: 0.108 (B)</dd>
                                    <dd>Mutation Taster: 0.942 (P)</dd>
                                    <dd>SIFT: 0.271 (T)</dd>
                                    <dd>CADD: 0.995 (22.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_E_AFR: 0.0394%</dd>
                                    <dd>gnomAD_G_AFR: 0.0458%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000091656">ZFHX4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.727</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.986</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.331 to <a href="http://www.omim.org/178300">?Ptosis, congenital</a> associated with ZFHX4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0007970, Congenital ptosis</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZFHX4%0DCHD4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CHD4</a> and phenotypic similarity 0.688 to <a href="http://www.omim.org/617159">Sifrim-Hitz-Weiss syndrome</a> associated with CHD4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000378, Cupped ear</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/178300">OMIM:178300</a> ?Ptosis, congenital (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.727</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.986</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr8:g.76849695G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=368967335">rs368967335</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs368967335&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.986</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000518282.5">ZFHX4:ENST00000518282.5:c.3751G&gt;A:p.(Glu1251Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000651372.2">ZFHX4:ENST00000651372.2:c.3829G&gt;A:p.(Glu1277Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.003 (D)</dd>
                                    <dd>CADD: 0.998 (27.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0998%</dd>
                                    <dd>TOPMed: 0.0159%</dd>
                                    <dd>ESP AA: 0.0498%</dd>
                                    <dd>ESP All: 0.0161%</dd>
                                    <dd>ExAC AFR: 0.0844%</dd>
                                    <dd>ExAC AMR: 0.0090%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0720%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0573%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000106714">CNTNAP3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.727</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.509</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.977</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.294 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CNTNAP3">CNTNAP3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0002804, abnormal motor learning</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CNTNAP3%0DL1CAM&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to L1CAM</a> and phenotypic similarity 0.503 to <a href="http://www.omim.org/303350">MASA syndrome</a> associated with L1CAM.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CNTNAP3%0DL1CAM&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to L1CAM</a> and phenotypic similarity 0.680 to mouse mutant of L1CAM.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0002733, abnormal thermal nociception</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0011380, enlarged brain ventricles</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0006198, enophthalmos</dd><dd>HP:0000637, Long palpebral fissure - MP:0006198, enophthalmos</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.727</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.509</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.977</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.39078800G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1479645696">rs1479645696</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1479645696&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.977</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000297668.10">CNTNAP3:ENST00000297668.10:c.3563C&gt;T:p.(Pro1188Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377656.6">CNTNAP3:ENST00000377656.6:c.3320C&gt;T:p.(Pro1107Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9769856</dd>
                                    
                                    <dd>Polyphen2: 0.002 (B)</dd>
                                    <dd>SIFT: 0.184 (T)</dd>
                                    <dd>CADD: 0.977 (16.380)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000155897">ADCY8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.726</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.986</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.268 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ADCY8">ADCY8</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0008531, increased chemical nociceptive threshold</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADCY8%0DCAMK2G&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAMK2G</a> and phenotypic similarity 0.751 to <a href="http://www.omim.org/618522">Mental retardation, autosomal dominant 59</a> associated with CAMK2G.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0003502, Mild short stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000411, Protruding ear</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADCY8%0DCAMK2G&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAMK2G</a> and phenotypic similarity 0.238 to fish mutant of CAMK2G.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - ZP:0012134, abnormal(ly) decreased length otic vesicle</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.726</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.986</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr8:g.130780760C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=372297753">rs372297753</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs372297753&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.986</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000286355.10">ADCY8:ENST00000286355.10:c.3386G&gt;A:p.(Arg1129Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377928.7">ADCY8:ENST00000377928.7:c.2993G&gt;A:p.(Arg998Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9952137</dd>
                                    
                                    <dd>Polyphen2: 0.651 (P)</dd>
                                    <dd>SIFT: 0.042 (D)</dd>
                                    <dd>CADD: 0.995 (23.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0120%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0384%</dd>
                                    <dd>ExAC EAS: 0.0347%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.0523%</dd>
                                    <dd>gnomAD_E_AMR: 0.0149%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0102%</dd>
                                    <dd>gnomAD_E_EAS: 0.0290%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 0.0687%</dd>
                                    <dd>gnomAD_G_EAS: 0.0617%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163661">PTX3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.726</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.987</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PTX3%0DIRX5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRX5</a> and phenotypic similarity 0.588 to <a href="http://www.omim.org/611174">Hamamy syndrome</a> associated with IRX5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000248, Brachycephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0005338, Sparse lateral eyebrow</dd><dd>HP:0001572, Macrodontia - HP:0000689, Dental malocclusion</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PTX3%0DIRX5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRX5</a> and phenotypic similarity 0.653 to mouse mutant of IRX5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005287, narrow eye opening</dd><dd>HP:0000637, Long palpebral fissure - MP:0005287, narrow eye opening</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.726</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.987</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.157442896G&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=149891511">rs149891511</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs149891511&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.987</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295927.4">PTX3:ENST00000295927.4:c.1063G&gt;C:p.(Glu355Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000362010.7">PTX3:ENST00000362010.7:c.530-14408C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392832.6">PTX3:ENST00000392832.6:c.530-14408C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392833.6">PTX3:ENST00000392833.6:c.530-14408C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99498814</dd>
                                    
                                    <dd>Polyphen2: 0.498 (P)</dd>
                                    <dd>SIFT: 0.169 (T)</dd>
                                    <dd>CADD: 0.995 (23.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0582%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>gnomAD_E_AFR: 0.0523%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130939">UBE4B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.722</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.980</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.328 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=UBE4B">UBE4B</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001406, abnormal gait</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000846, abnormal medulla oblongata morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=UBE4B%0DUSP9X&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to USP9X</a> and phenotypic similarity 0.750 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with USP9X.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002187, Intellectual disability, profound</dd><dd>HP:0004322, Short stature - HP:0001518, Small for gestational age</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000684, Delayed eruption of teeth</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.722</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.980</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.10130513G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=541907403">rs541907403</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs541907403&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.980</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000253251.12">UBE4B:ENST00000253251.12:c.1322G&gt;A:p.(Arg441Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343090.11">UBE4B:ENST00000343090.11:c.1709G&gt;A:p.(Arg570Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000672724.1">UBE4B:ENST00000672724.1:c.1862G&gt;A:p.(Arg621Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98872805</dd>
                                    
                                    <dd>Polyphen2: 0.005 (B)</dd>
                                    <dd>SIFT: 0.094 (T)</dd>
                                    <dd>CADD: 0.989 (19.480)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC NFE: 0.0075%</dd>
                                    <dd>ExAC SAS: 0.0666%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_NFE: 0.0090%</dd>
                                    <dd>gnomAD_E_SAS: 0.0455%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000131469">RPL27</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.717</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.980</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.369 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=124">Blackfan-Diamond anemia</a> associated with RPL27.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RPL27%0DEIF2S3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EIF2S3</a> and phenotypic similarity 0.693 to <a href="http://www.omim.org/300148">MEHMO syndrome</a> associated with EIF2S3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0003561, Birth length less than 3rd percentile</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000687, Widely spaced teeth</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/617408">OMIM:617408</a> ?Diamond-Blackfan anemia 16 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=124">ORPHA:124</a> Blackfan-Diamond anemia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.717</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.980</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.42998783G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=149080883">rs149080883</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs149080883&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.980</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000253788.11">RPL27:ENST00000253788.11:c.33G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589037.5">RPL27:ENST00000589037.5:c.33G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589913.6">RPL27:ENST00000589913.6:c.33G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99323916</dd>
                                    
                                    <dd>CADD: 0.993 (21.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0175%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0673%</dd>
                                    <dd>gnomAD_E_AFR: 0.0916%</dd>
                                    <dd>gnomAD_G_AFR: 0.0572%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163481">RNF25</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.714</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.981</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.328 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RNF25">RNF25</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0011275, abnormal behavioral response to light</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0011275, abnormal behavioral response to light</dd><dd>HP:0000637, Long palpebral fissure - MP:0011275, abnormal behavioral response to light</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RNF25%0DDVL3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DVL3</a> and phenotypic similarity 0.820 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3107">Autosomal dominant Robinow syndrome</a> associated with DVL3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0003510, Severe short stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0011069, Increased number of teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RNF25%0DDVL3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DVL3</a> and phenotypic similarity 0.713 to mouse mutant of DVL3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000445, short snout</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0006198, enophthalmos</dd><dd>HP:0000637, Long palpebral fissure - MP:0006198, enophthalmos</dd><dd>HP:0001572, Macrodontia - MP:0002639, micrognathia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.714</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.981</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.218664093C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=139628113">rs139628113</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs139628113&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.981</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295704.7">RNF25:ENST00000295704.7:c.1244G&gt;A:p.(Arg415Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99475193</dd>
                                    
                                    <dd>Polyphen2: 0.005 (B)</dd>
                                    <dd>SIFT: 0.335 (T)</dd>
                                    <dd>CADD: 0.995 (22.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0998%</dd>
                                    <dd>TOPMed: 0.0080%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0500%</dd>
                                    <dd>gnomAD_E_AFR: 0.0416%</dd>
                                    <dd>gnomAD_E_AMR: 0.0146%</dd>
                                    <dd>gnomAD_E_EAS: 0.0201%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000072071">ADGRL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.707</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.977</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADGRL1%0DUGDH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UGDH</a> and phenotypic similarity 0.665 to <a href="http://www.omim.org/618792">Epileptic encephalopathy, early infantile, 84</a> associated with UGDH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0009748, Large earlobe</dd><dd>HP:0000316, Hypertelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000307, Pointed chin</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADGRL1%0DUGDH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UGDH</a> and phenotypic similarity 0.246 to mouse mutant of UGDH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADGRL1%0DUGDH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UGDH</a> and phenotypic similarity 0.358 to fish mutant of UGDH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - ZP:0001719, abnormal(ly) decreased length head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - ZP:0001249, abnormal(ly) aplastic semicircular canal</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.707</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.977</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.14160665C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=565414527">rs565414527</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs565414527&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.977</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340736.10">ADGRL1:ENST00000340736.10:c.1557G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361434.7">ADGRL1:ENST00000361434.7:c.1542G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9796764</dd>
                                    
                                    <dd>CADD: 0.980 (16.920)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0106%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182149">IST1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.704</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.975</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=IST1%0DVPS25&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VPS25</a> and phenotypic similarity 0.684 to mouse mutant of VPS25.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0000547, short limbs</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000445, short snout</dd><dd>HP:0000400, Macrotia - MP:0002177, abnormal outer ear morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000460, mandible hypoplasia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.704</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.975</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_INSERTION</span>
                                <b>chr16:g.71922626C&gt;CATGCCT [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=28701631">rs28701631</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs28701631&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.975</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329908.12">IST1:ENST00000329908.12:c.711_716dup:p.(Met238_Pro239dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378798.9">IST1:ENST00000378798.9:c.711_716dup:p.(Met238_Pro239dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378799.11">IST1:ENST00000378799.11:c.711_716dup:p.(Met238_Pro239dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000535424.5">IST1:ENST00000535424.5:c.750_755dup:p.(Met251_Pro252dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538850.5">IST1:ENST00000538850.5:c.267_272dup:p.(Met90_Pro91dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541571.6">IST1:ENST00000541571.6:c.711_716dup:p.(Met238_Pro239dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544564.5">IST1:ENST00000544564.5:c.711_716dup:p.(Met238_Pro239dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000606369.5">IST1:ENST00000606369.5:c.267_272dup:p.(Met90_Pro91dup)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9754529</dd>
                                    
                                    <dd>CADD: 0.975 (16.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0067%</dd>
                                    <dd>gnomAD_E_NFE: 0.0038%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000110768">GTF2H1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.695</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.515</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.954</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.515 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GTF2H1%0DGTF2H5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GTF2H5</a> and phenotypic similarity 0.624 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=33364">Trichothiodystrophy</a> associated with GTF2H5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001511, Intrauterine growth retardation</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000411, Protruding ear</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000656, Ectropion</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.695</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.515</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.954</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.18347942C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=370398806">rs370398806</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs370398806&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.954</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524753.4">GTF2H1:ENST00000524753.4:c.464C&gt;T:p.(Pro155Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265963.9">GTF2H1:ENST00000265963.9:c.1053+23C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453096.6">GTF2H1:ENST00000453096.6:c.1053+23C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530496.6">GTF2H1:ENST00000530496.6:c.117+23C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000534641.5">GTF2H1:ENST00000534641.5:c.705+23C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.962</dd>
                                    
                                    <dd>SIFT: 0.038 (D)</dd>
                                    <dd>CADD: 0.423 (2.388)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0088%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0324%</dd>
                                    <dd>ExAC EAS: 0.0418%</dd>
                                    <dd>ExAC NFE: 0.0017%</dd>
                                    <dd>gnomAD_E_AFR: 0.0328%</dd>
                                    <dd>gnomAD_E_EAS: 0.0350%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0573%</dd>
                                    <dd>gnomAD_G_EAS: 0.0617%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000248905">FMN1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.689</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.968</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.380 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FMN1">FMN1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0000547, short limbs</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FMN1%0DSIX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SIX1</a> and phenotypic similarity 0.461 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=107">BOR syndrome</a> associated with SIX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0008572, External ear malformation</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0011481, Abnormal lacrimal duct morphology</dd><dd>HP:0001572, Macrodontia - HP:0000278, Retrognathia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FMN1%0DSIX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SIX1</a> and phenotypic similarity 0.686 to mouse mutant of SIX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0002233, abnormal nose morphology</dd><dd>HP:0000400, Macrotia - MP:0000026, abnormal inner ear morphology</dd><dd>HP:0000316, Hypertelorism - MP:0001302, eyelids open at birth</dd><dd>HP:0000637, Long palpebral fissure - MP:0001302, eyelids open at birth</dd><dd>HP:0001572, Macrodontia - MP:0002639, micrognathia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FMN1%0DSIX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SIX1</a> and phenotypic similarity 0.241 to fish mutant of SIX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - ZP:0006179, abnormal(ly) aplastic medial rectus</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.689</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.968</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr15:g.33154488G&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=536196985">rs536196985</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs536196985&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.968</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000320930.7">FMN1:ENST00000320930.7:c.427C&gt;G:p.(Leu143Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000561249.5">FMN1:ENST00000561249.5:c.427C&gt;G:p.(Leu143Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616417.5">FMN1:ENST00000616417.5:c.427C&gt;G:p.(Leu143Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9703517</dd>
                                    
                                    <dd>SIFT: 0.108 (T)</dd>
                                    <dd>CADD: 0.970 (15.280)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>gnomAD_E_AMR: 0.0042%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000129467">ADCY4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.674</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.959</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADCY4%0DGNAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNAS</a> and phenotypic similarity 0.529 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=562">McCune-Albright syndrome</a> associated with GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0000098, Tall stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADCY4%0DGNAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNAS</a> and phenotypic similarity 0.636 to mouse mutant of GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001504, abnormal posture</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0004097, abnormal cerebellar cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0002177, abnormal outer ear morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.674</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.959</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.24329096C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=377492744">rs377492744</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs377492744&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.959</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000310677.8">ADCY4:ENST00000310677.8:c.1489G&gt;A:p.(Gly497Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000418030.6">ADCY4:ENST00000418030.6:c.1489G&gt;A:p.(Gly497Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000554068.6">ADCY4:ENST00000554068.6:c.1489G&gt;A:p.(Gly497Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9658807</dd>
                                    
                                    <dd>CADD: 0.966 (14.670)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0481%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.0523%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000121579">NAA50</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.669</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.954</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.385 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NAA50">NAA50</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001297, microphthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0001297, microphthalmia</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NAA50%0DNAA10&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NAA10</a> and phenotypic similarity 0.691 to <a href="http://www.omim.org/300855">Ogden syndrome</a> associated with NAA10.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000520, Proptosis</dd><dd>HP:0000637, Long palpebral fissure - HP:0000494, Downslanted palpebral fissures</dd><dd>HP:0001572, Macrodontia - HP:0000164, Abnormality of the dentition</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NAA50%0DNAA10&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NAA10</a> and phenotypic similarity 0.332 to fish mutant of NAA10.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - ZP:0000054, abnormal(ly) decreased size head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.669</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.954</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr3:g.113724080C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=565002331">rs565002331</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs565002331&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.954</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000240922.8">NAA50:ENST00000240922.8:c.24G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000477813.5">NAA50:ENST00000477813.5:c.24G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000493900.5">NAA50:ENST00000493900.5:c.21G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000497255.1">NAA50:ENST00000497255.1:c.21G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616174.1">NAA50:ENST00000616174.1:c.24G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000630058.1">NAA50:ENST00000630058.1:c.21G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000493454.5">NAA50:ENST00000493454.5:c.-199G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000497525.5">NAA50:ENST00000497525.5:c.-199G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.956549</dd>
                                    
                                    <dd>CADD: 0.957 (13.620)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000125447">GGA3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.661</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.950</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.282 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GGA3">GGA3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0008489, slow postnatal weight gain</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GGA3%0DAP1S2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AP1S2</a> and phenotypic similarity 0.768 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=85329">X-linked intellectual disability-hypotonia-facial dysmorphism-aggressive behavior syndrome</a> associated with AP1S2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002187, Intellectual disability, profound</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia - HP:0000411, Protruding ear</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000331, Short chin</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GGA3%0DAP1S2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AP1S2</a> and phenotypic similarity 0.302 to mouse mutant of AP1S2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0012307, impaired spatial learning</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.661</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.950</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr17:g.75261540C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=553317835">rs553317835</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs553317835&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.950</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537686.5">GGA3:ENST00000537686.5:c.40+8G&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538886.5">GGA3:ENST00000538886.5:c.40+8G&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000578348.5">GGA3:ENST00000578348.5:c.-228+742G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000582486.5">GGA3:ENST00000582486.5:c.-177+742G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000582717.5">GGA3:ENST00000582717.5:c.-177+742G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9522471</dd>
                                    
                                    <dd>CADD: 0.952 (13.210)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000090263">MRPS33</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.656</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.951</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MRPS33%0DNDUFS4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NDUFS4</a> and phenotypic similarity 0.494 to <a href="http://www.omim.org/252010">Mitochondrial complex I deficiency, nuclear type 1</a> associated with NDUFS4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001510, Growth delay</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MRPS33%0DNDUFS4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NDUFS4</a> and phenotypic similarity 0.627 to mouse mutant of NDUFS4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001504, abnormal posture</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0001915, intracranial hemorrhage</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001304, cataract</dd><dd>HP:0000637, Long palpebral fissure - MP:0001344, blepharoptosis</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.656</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.951</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.141010459C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=372380228">rs372380228</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs372380228&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.951</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324787.10">MRPS33:ENST00000324787.10:c.175G&gt;A:p.(Ala59Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393008.7">MRPS33:ENST00000393008.7:c.175G&gt;A:p.(Ala59Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000467334.1">MRPS33:ENST00000467334.1:c.145G&gt;A:p.(Ala49Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000469351.1">MRPS33:ENST00000469351.1:c.175G&gt;A:p.(Ala59Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000496958.5">MRPS33:ENST00000496958.5:c.175G&gt;A:p.(Ala59Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9597283</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.411 (T)</dd>
                                    <dd>CADD: 0.960 (13.950)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0088%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0288%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC EAS: 0.0231%</dd>
                                    <dd>gnomAD_E_AFR: 0.0457%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_EAS: 0.0617%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000076067">RBMS2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.650</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.948</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RBMS2%0DRBMX&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RBMX</a> and phenotypic similarity 0.658 to <a href="http://www.omim.org/300238">?Mental retardation, X-linked, syndromic 11, Shashi type</a> associated with RBMX.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002342, Intellectual disability, moderate</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0045025, Narrow palpebral fissure</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.650</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.948</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr12:g.56571847C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=564788503">rs564788503</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs564788503&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.948</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262031.10">RBMS2:ENST00000262031.10:c.534C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000550726.5">RBMS2:ENST00000550726.5:c.159C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000552247.6">RBMS2:ENST00000552247.6:c.534C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542360.1">RBMS2:ENST00000542360.1:c.9+1857C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.95102215</dd>
                                    
                                    <dd>CADD: 0.951 (13.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0108%</dd>
                                    <dd>gnomAD_E_AFR: 0.0072%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000149050">ZNF214</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.647</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.947</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF214%0DSIN3A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SIN3A</a> and phenotypic similarity 0.727 to <a href="http://www.omim.org/613406">Witteveen-Kolk syndrome</a> associated with SIN3A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000582, Upslanted palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000687, Widely spaced teeth</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.647</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.947</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.7000985C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537720412">rs537720412</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537720412&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.947</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000278314.5">ZNF214:ENST00000278314.5:c.698G&gt;A:p.(Arg233Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536068.5">ZNF214:ENST00000536068.5:c.698G&gt;A:p.(Arg233Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.94964993</dd>
                                    
                                    <dd>SIFT: 0.771 (T)</dd>
                                    <dd>CADD: 0.950 (12.980)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0098%</dd>
                                    <dd>ExAC FIN: 0.0152%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0195%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000215182">MUC5AC</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.636</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.514</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.926</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.514 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MUC5AC">MUC5AC</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005253, abnormal eye physiology</dd><dd>HP:0000637, Long palpebral fissure - MP:0012164, abnormal conjunctiva goblet cell morphology</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.636</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.514</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.926</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.1190555C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1449827908">rs1449827908</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1449827908&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.926</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.12410C&gt;T:p.(Thr4137Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9255268</dd>
                                    
                                    <dd>CADD: 0.926 (11.280)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1186303C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1191022121">rs1191022121</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1191022121&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.895</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.8158C&gt;T:p.(Pro2720Ser)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8948038</dd>
                        
                        <dd>CADD: 0.895 (9.780)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1191269C&gt;G [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1409088375">rs1409088375</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1409088375&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.887</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.13124C&gt;G:p.(Thr4375Ser)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8871244</dd>
                        
                        <dd>CADD: 0.887 (9.474)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1190388T&gt;C [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1243776629">rs1243776629</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1243776629&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.683</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.12243T&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.68311626</dd>
                        
                        <dd>CADD: 0.683 (4.991)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1182975C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1473448879">rs1473448879</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1473448879&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.014</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.4830C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.014401555</dd>
                        
                        <dd>CADD: 0.014 (0.063)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000117569">PTBP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.632</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.934</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.260 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PTBP2">PTBP2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0008458, abnormal cortical ventricular zone morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PTBP2%0DRBM10&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RBM10</a> and phenotypic similarity 0.668 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2886">TARP syndrome</a> associated with RBM10.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0001508, Failure to thrive</dd><dd>HP:0000252, Microcephaly - HP:0001321, Cerebellar hypoplasia</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000395, Prominent antihelix</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0009085, Alveolar ridge overgrowth</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.632</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.934</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.96813330C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=376479747">rs376479747</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs376479747&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.934</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370197.5">PTBP2:ENST00000370197.5:c.1539C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370198.5">PTBP2:ENST00000370198.5:c.1536C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000426398.2">PTBP2:ENST00000426398.2:c.1521C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000609116.5">PTBP2:ENST00000609116.5:c.1524C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9367588</dd>
                                    
                                    <dd>CADD: 0.937 (11.990)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AFR: 0.0197%</dd>
                                    <dd>gnomAD_E_AMR: 0.0031%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000267680">ZNF224</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.630</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.938</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF224%0DTET1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TET1</a> and phenotypic similarity 0.846 to mouse mutant of TET1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0001890, anencephaly</dd><dd>HP:0000325, Triangular face - MP:0000108, midline facial cleft</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - MP:0001300, ocular hypertelorism</dd><dd>HP:0001572, Macrodontia - MP:0008818, abnormal interfrontal bone morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.630</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.938</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.44108217A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=577703786">rs577703786</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs577703786&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.938</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336976.10">ZNF224:ENST00000336976.10:c.2057A&gt;G:p.(Asp686Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9408438</dd>
                                    
                                    <dd>Polyphen2: 0.000 (B)</dd>
                                    <dd>SIFT: 1.000 (T)</dd>
                                    <dd>CADD: 0.941 (12.280)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0100%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000174672">BRSK2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.627</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.937</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=BRSK2%0DTET1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TET1</a> and phenotypic similarity 0.846 to mouse mutant of TET1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0001890, anencephaly</dd><dd>HP:0000325, Triangular face - MP:0000108, midline facial cleft</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - MP:0001300, ocular hypertelorism</dd><dd>HP:0001572, Macrodontia - MP:0008818, abnormal interfrontal bone morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.627</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.937</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr11:g.1445915T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=557553708">rs557553708</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs557553708&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.937</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308219.13">BRSK2:ENST00000308219.13:c.1226+8T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382179.5">BRSK2:ENST00000382179.5:c.1364+8T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000526678.5">BRSK2:ENST00000526678.5:c.1226+8T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000528710.5">BRSK2:ENST00000528710.5:c.1046+8T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000528841.6">BRSK2:ENST00000528841.6:c.1226+8T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000531197.5">BRSK2:ENST00000531197.5:c.1226+8T&gt;C:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9396051</dd>
                                    
                                    <dd>CADD: 0.940 (12.190)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0120%</dd>
                                    <dd>gnomAD_E_AFR: 0.0069%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154305">MIA3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.625</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.457</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.986</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.457 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MIA3">MIA3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0000547, short limbs</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000445, short snout</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000445, short snout</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.625</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.457</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.986</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.222627966G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=530146277">rs530146277</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs530146277&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.986</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344922.10">MIA3:ENST00000344922.10:c.746G&gt;T:p.(Ser249Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98827803</dd>
                                    
                                    <dd>Polyphen2: 0.074 (B)</dd>
                                    <dd>SIFT: 0.082 (T)</dd>
                                    <dd>CADD: 0.988 (19.310)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0104%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000137845">ADAM10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.621</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.509</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.925</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.247 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ADAM10">ADAM10</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0003232, abnormal forebrain development</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADAM10%0DDLK1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DLK1</a> and phenotypic similarity 0.691 to mouse mutant of DLK1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005287, narrow eye opening</dd><dd>HP:0000637, Long palpebral fissure - MP:0005287, narrow eye opening</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/615537">OMIM:615537</a> Reticulate acropigmentation of Kitamura - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.621</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.509</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.925</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr15:g.58643937G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201371720">rs201371720</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201371720&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.925</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000260408.8">ADAM10:ENST00000260408.8:c.777C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000402627.5">ADAM10:ENST00000402627.5:c.56-3107C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000561288.1">ADAM10:ENST00000561288.1:c.56-46419C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.92969275</dd>
                                    
                                    <dd>CADD: 0.930 (11.530)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>ExAC AMR: 0.0259%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100628">ASB2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.617</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.933</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.287 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ASB2">ASB2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0010097, abnormal retinal blood vessel morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0010097, abnormal retinal blood vessel morphology</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ASB2%0DPHGDH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PHGDH</a> and phenotypic similarity 0.691 to <a href="http://www.omim.org/256520">Neu-Laxova syndrome 1</a> associated with PHGDH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001511, Intrauterine growth retardation</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0011224, Ablepharon</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ASB2%0DPHGDH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PHGDH</a> and phenotypic similarity 0.467 to mouse mutant of PHGDH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0011496, abnormal head size</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0002092, abnormal eye morphology</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.617</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.933</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.93947374T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=548080899">rs548080899</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs548080899&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.933</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000315988.8">ASB2:ENST00000315988.8:c.883A&gt;G:p.(Ile295Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555019.6">ASB2:ENST00000555019.6:c.1027A&gt;G:p.(Ile343Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.93558306</dd>
                                    
                                    <dd>Polyphen2: 0.006 (B)</dd>
                                    <dd>SIFT: 0.935 (T)</dd>
                                    <dd>CADD: 0.936 (11.910)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171206">TRIM8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.560</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.907</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.478 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1934">Early infantile epileptic encephalopathy</a> associated with TRIM8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0001508, Failure to thrive</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRIM8%0DLIFR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LIFR</a> and phenotypic similarity 0.630 to <a href="http://www.omim.org/601559">Stuve-Wiedemann syndrome/Schwartz-Jampel type 2 syndrome</a> associated with LIFR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000682, Abnormality of dental enamel</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRIM8%0DLIFR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LIFR</a> and phenotypic similarity 0.211 to mouse mutant of LIFR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0002152, abnormal brain morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1934">ORPHA:1934</a> Early infantile epileptic encephalopathy</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.560</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.907</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr10:g.102657216C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=372270090">rs372270090</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs372270090&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.907</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000643721.1">TRIM8:ENST00000643721.1:c.1518C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302424.12">TRIM8:ENST00000302424.12:c.*697C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9100503</dd>
                                    
                                    <dd>CADD: 0.910 (10.460)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0195%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0149%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198062">POTEH</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.551</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.903</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=POTEH%0DL1CAM&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to L1CAM</a> and phenotypic similarity 0.503 to <a href="http://www.omim.org/303350">MASA syndrome</a> associated with L1CAM.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=POTEH%0DL1CAM&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to L1CAM</a> and phenotypic similarity 0.680 to mouse mutant of L1CAM.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0002733, abnormal thermal nociception</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0011380, enlarged brain ventricles</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0006198, enophthalmos</dd><dd>HP:0000637, Long palpebral fissure - MP:0006198, enophthalmos</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.551</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.903</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.15700206G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2845218">rs2845218</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2845218&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.903</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343518.11">POTEH:ENST00000343518.11:c.1157G&gt;A:p.(Arg386His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621704.4">POTEH:ENST00000621704.4:c.1046G&gt;A:p.(Arg349His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.90317225</dd>
                                    
                                    <dd>CADD: 0.903 (10.140)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169174">PCSK9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.543</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.899</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PCSK9%0DSP6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SP6</a> and phenotypic similarity 0.795 to mouse mutant of SP6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0002100, abnormal tooth morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/603776">OMIM:603776</a> Hypercholesterolemia, familial, 3 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/603776">OMIM:603776</a> Low density lipoprotein cholesterol level QTL 1 (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=391665">ORPHA:391665</a> Homozygous familial hypercholesterolemia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.543</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.899</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.55061484C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=374856617">rs374856617</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs374856617&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    <span class="label label-info">LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=616194%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.899</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302118.5">PCSK9:ENST00000302118.5:c.1791C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000673903.1">PCSK9:ENST00000673903.1:c.1416C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9036171</dd>
                                    
                                    <dd>CADD: 0.904 (10.160)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0111%</dd>
                                    <dd>ESP AA: 0.0228%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0155%</dd>
                                    <dd>ExAC EAS: 0.0375%</dd>
                                    <dd>ExAC NFE: 0.0025%</dd>
                                    <dd>gnomAD_E_AFR: 0.0073%</dd>
                                    <dd>gnomAD_E_AMR: 0.0031%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0107%</dd>
                                    <dd>gnomAD_E_EAS: 0.0123%</dd>
                                    <dd>gnomAD_E_NFE: 0.0010%</dd>
                                    <dd>gnomAD_G_AFR: 0.0114%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143126">CELSR2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.542</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.899</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.251 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CELSR2">CELSR2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000825, dilated lateral ventricles</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CELSR2%0DTP53BP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TP53BP2</a> and phenotypic similarity 0.703 to mouse mutant of TP53BP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0003742, narrow head</dd><dd>HP:0000325, Triangular face - MP:0000111, cleft palate</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001293, anophthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0001293, anophthalmia</dd><dd>HP:0001572, Macrodontia - MP:0002639, micrognathia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.542</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.899</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.109269559G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=530273076">rs530273076</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs530273076&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.899</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000271332.4">CELSR2:ENST00000271332.4:c.6948G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9011447</dd>
                                    
                                    <dd>CADD: 0.901 (10.050)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000180537">RNF182</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.537</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.897</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RNF182%0DATP6V1E1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ATP6V1E1</a> and phenotypic similarity 0.761 to <a href="http://www.omim.org/617402">Cutis laxa, autosomal recessive, type IIC</a> associated with ATP6V1E1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000621, Entropion</dd><dd>HP:0001572, Macrodontia - HP:0000678, Dental crowding</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RNF182%0DATP6V1E1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ATP6V1E1</a> and phenotypic similarity 0.421 to fish mutant of ATP6V1E1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - ZP:0000054, abnormal(ly) decreased size head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - ZP:0000993, abnormal(ly) decreased size inner ear</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - ZP:0002155, abnormal(ly) shape mandibular arch skeleton</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.537</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.897</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr6:g.13977233G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=542059808">rs542059808</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs542059808&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.897</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000488300.6">RNF182:ENST00000488300.6:c.114G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537388.1">RNF182:ENST00000537388.1:c.114G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537663.5">RNF182:ENST00000537663.5:c.114G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544682.5">RNF182:ENST00000544682.5:c.114G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9022763</dd>
                                    
                                    <dd>CADD: 0.902 (10.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_G_AFR: 0.0114%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000144674">GOLGA4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.536</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.896</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GOLGA4%0DTBC1D20&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TBC1D20</a> and phenotypic similarity 0.668 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2510">Micro syndrome</a> associated with TBC1D20.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0010864, Intellectual disability, severe</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GOLGA4%0DTBC1D20&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TBC1D20</a> and phenotypic similarity 0.423 to mouse mutant of TBC1D20.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0001303, abnormal lens morphology</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.536</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.896</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr3:g.37326260C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=561876768">rs561876768</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs561876768&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.896</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356847.8">GOLGA4:ENST00000356847.8:c.4440C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361924.6">GOLGA4:ENST00000361924.6:c.4374C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000444882.5">GOLGA4:ENST00000444882.5:c.478-39820C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8985388</dd>
                                    
                                    <dd>CADD: 0.899 (9.937)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0102%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000108306">FBXL20</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.535</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.896</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FBXL20%0DFBXW7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FBXW7</a> and phenotypic similarity 0.602 to mouse mutant of FBXW7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000111, cleft palate</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001302, eyelids open at birth</dd><dd>HP:0000637, Long palpebral fissure - MP:0001302, eyelids open at birth</dd><dd>HP:0001572, Macrodontia - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.535</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.896</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.39299049C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=542198565">rs542198565</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs542198565&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.896</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264658.11">FBXL20:ENST00000264658.11:c.270G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394294.7">FBXL20:ENST00000394294.7:c.270G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000577399.5">FBXL20:ENST00000577399.5:c.276G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583610.5">FBXL20:ENST00000583610.5:c.270G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.89884204</dd>
                                    
                                    <dd>CADD: 0.899 (9.950)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000147246">HTR2C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.534</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.895</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.376 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=HTR2C">HTR2C</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0009746, enhanced behavioral response to xenobiotic</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=HTR2C%0DGNAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNAS</a> and phenotypic similarity 0.529 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=562">McCune-Albright syndrome</a> associated with GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0000098, Tall stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=HTR2C%0DGNAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GNAS</a> and phenotypic similarity 0.636 to mouse mutant of GNAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001504, abnormal posture</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0004097, abnormal cerebellar cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0002177, abnormal outer ear morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.534</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.895</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.114731326C&gt;G [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=6318">rs6318</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs6318&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.895</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000276198.6">HTR2C:ENST00000276198.6:c.68C&gt;G:p.(Ser23Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371950.3">HTR2C:ENST00000371950.3:c.68C&gt;G:p.(Ser23Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371951.5">HTR2C:ENST00000371951.5:c.68C&gt;G:p.(Ser23Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.89468265</dd>
                                    
                                    <dd>CADD: 0.895 (9.775)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135750">KCNK1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.522</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.889</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KCNK1%0DKCNK9&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KCNK9</a> and phenotypic similarity 0.734 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=166108">Intellectual disability, Birk-Barel type</a> associated with KCNK9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000268, Dolichocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000411, Protruding ear</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0002553, Highly arched eyebrow</dd><dd>HP:0001572, Macrodontia - HP:0011081, Incisor macrodontia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.522</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.889</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.233666821C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=563157083">rs563157083</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs563157083&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.889</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366621.8">KCNK1:ENST00000366621.8:c.582C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.89617544</dd>
                                    
                                    <dd>CADD: 0.896 (9.837)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0096%</dd>
                                    <dd>ExAC AFR: 0.0384%</dd>
                                    <dd>gnomAD_E_AFR: 0.0457%</dd>
                                    <dd>gnomAD_G_AFR: 0.0573%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165115">KIF27</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.509</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.883</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.397 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=KIF27">KIF27</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0001917, intraventricular hemorrhage</dd><dd>HP:0000325, Triangular face - MP:0002236, abnormal internal nares morphology</dd><dd>HP:0000400, Macrotia - MP:0001850, increased susceptibility to otitis media</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0002236, abnormal internal nares morphology</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIF27%0DGLI3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GLI3</a> and phenotypic similarity 0.624 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=672">Pallister-Hall syndrome</a> associated with GLI3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0030021, Auricular tag</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia - HP:0000308, Microretrognathia</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIF27%0DGLI3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GLI3</a> and phenotypic similarity 0.724 to mouse mutant of GLI3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0002764, short tibia</dd><dd>HP:0000252, Microcephaly - MP:0002152, abnormal brain morphology</dd><dd>HP:0000325, Triangular face - MP:0000111, cleft palate</dd><dd>HP:0000400, Macrotia - MP:0000023, abnormal ear position</dd><dd>HP:0000316, Hypertelorism - MP:0001286, abnormal eye development</dd><dd>HP:0000637, Long palpebral fissure - MP:0001286, abnormal eye development</dd><dd>HP:0001572, Macrodontia - MP:0000116, abnormal tooth development</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.509</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.883</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr9:g.83888520A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=570677869">rs570677869</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs570677869&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.883</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000297814.7">KIF27:ENST00000297814.7:c.2052T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000334204.6">KIF27:ENST00000334204.6:c.2052T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376347.1">KIF27:ENST00000376347.1:c.225T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413982.5">KIF27:ENST00000413982.5:c.2052T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8850259</dd>
                                    
                                    <dd>CADD: 0.885 (9.394)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0048%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>gnomAD_E_AFR: 0.0139%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000101639">CEP192</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.878</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.328 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CEP192">CEP192</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0011275, abnormal behavioral response to light</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0011275, abnormal behavioral response to light</dd><dd>HP:0000637, Long palpebral fissure - MP:0011275, abnormal behavioral response to light</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CEP192%0DCEP57&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CEP57</a> and phenotypic similarity 0.791 to <a href="http://www.omim.org/614114">Mosaic variegated aneuploidy syndrome 2</a> associated with CEP57.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001256, Intellectual disability, mild</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.878</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr18:g.13068129G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=566949334">rs566949334</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs566949334&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.878</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000325971.12">CEP192:ENST00000325971.12:c.3225G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000506447.5">CEP192:ENST00000506447.5:c.4650G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430049.6">CEP192:ENST00000430049.6:c.-82G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.88068366</dd>
                                    
                                    <dd>CADD: 0.881 (9.233)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000142197">DOP1B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.499</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.880</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DOP1B%0DUBE2T&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UBE2T</a> and phenotypic similarity 0.695 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with UBE2T.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0008572, External ear malformation</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.499</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.880</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr21:g.36281505A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=149665876">rs149665876</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs149665876&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.880</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399151.3">DOP1B:ENST00000399151.3:c.6054A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8839086</dd>
                                    
                                    <dd>CADD: 0.884 (9.352)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0048%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0290%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000048392">RRM2B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.497</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.879</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.436 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=298">Mitochondrial neurogastrointestinal encephalomyopathy</a> associated with RRM2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004326, Cachexia</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.370 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RRM2B">RRM2B</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0003793, abnormal submandibular gland morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0003793, abnormal submandibular gland morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RRM2B%0DTLK2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TLK2</a> and phenotypic similarity 0.661 to <a href="http://www.omim.org/618050">Mental retardation, autosomal dominant 57</a> associated with TLK2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000358, Posteriorly rotated ears</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000581, Blepharophimosis</dd><dd>HP:0001572, Macrodontia - HP:0400000, Tall chin</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/612075">OMIM:612075</a> Mitochondrial DNA depletion syndrome 8A (encephalomyopathic type with renal tubulopathy) - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/613077">OMIM:613077</a> Progressive external ophthalmoplegia with mitochondrial DNA deletions, autosomal dominant 5 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=254892">ORPHA:254892</a> Autosomal dominant progressive external ophthalmoplegia</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=298">ORPHA:298</a> Mitochondrial neurogastrointestinal encephalomyopathy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=329336">ORPHA:329336</a> Adult-onset chronic progressive external ophthalmoplegia with mitochondrial myopathy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=480">ORPHA:480</a> Kearns-Sayre syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.497</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.879</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr8:g.102218950G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=542652118">rs542652118</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs542652118&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.879</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251810.8">RRM2B:ENST00000251810.8:c.551-3C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395912.6">RRM2B:ENST00000395912.6:c.395-3C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621845.1">RRM2B:ENST00000621845.1:c.389-3C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000519317.5">RRM2B:ENST00000519317.5:c.49-4792C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000519962.5">RRM2B:ENST00000519962.5:c.49-10665C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8816141</dd>
                                    
                                    <dd>CADD: 0.882 (9.267)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0098%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000177084">POLE</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.497</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.544</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.830</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.544 to <a href="http://www.omim.org/618336">IMAGE-I syndrome</a> associated with POLE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0003510, Severe short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0008551, Microtia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=POLE%0DXRCC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to XRCC4</a> and phenotypic similarity 0.755 to <a href="http://www.omim.org/616541">Short stature, microcephaly, and endocrine dysfunction</a> associated with XRCC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000692, Misalignment of teeth</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=POLE%0DXRCC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to XRCC4</a> and phenotypic similarity 0.298 to mouse mutant of XRCC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0004201, fetal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0000788, abnormal cerebral cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/615083">OMIM:615083</a> Colorectal cancer, susceptibility to, 12 (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/615139">OMIM:615139</a> FILS syndrome - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/618336">OMIM:618336</a> IMAGE-I syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=447877">ORPHA:447877</a> Polymerase proofreading-related adenomatous polyposis</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=85173">ORPHA:85173</a> IMAGe syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.497</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.544</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.830</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr12:g.132635887C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=562946055">rs562946055</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs562946055&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=476796%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.830</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000320574.10">POLE:ENST00000320574.10:c.5811+5G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000535270.5">POLE:ENST00000535270.5:c.5730+5G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8343086</dd>
                                    
                                    <dd>CADD: 0.834 (7.807)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC AFR: 0.0290%</dd>
                                    <dd>ExAC SAS: 0.0071%</dd>
                                    <dd>gnomAD_E_AFR: 0.0262%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000055609">KMT2C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.478</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.563</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.502 to <a href="http://www.omim.org/617768">Kleefstra syndrome 2</a> associated with KMT2C.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0001510, Growth delay</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000574, Thick eyebrow</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.563 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=KMT2C">KMT2C</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001290, delayed eyelid opening</dd><dd>HP:0000637, Long palpebral fissure - MP:0001290, delayed eyelid opening</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KMT2C%0DPLOD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLOD2</a> and phenotypic similarity 0.617 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2771">Bruck syndrome</a> associated with PLOD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/617768">OMIM:617768</a> Kleefstra syndrome 2 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.478</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.563</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr7:g.152178015T&gt;TA [0/1:0/-:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=746018833">rs746018833</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs746018833&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262189.11">KMT2C:ENST00000262189.11:c.7443-6_7443-5insT:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355193.6">KMT2C:ENST00000355193.6:c.7443-6_7443-5insT:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.17889255</dd>
                                    
                                    <dd>CADD: 0.179 (0.856)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000159307">SCUBE1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.469</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.453</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.921</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.453 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SCUBE1">SCUBE1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000783, abnormal forebrain morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0002856, abnormal vestibular ganglion morphology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0004377, small frontal bone</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.469</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.453</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.921</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr22:g.43258235G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=376771972">rs376771972</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs376771972&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.921</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000290460.7">SCUBE1:ENST00000290460.7:c.711C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360835.9">SCUBE1:ENST00000360835.9:c.711C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615096.4">SCUBE1:ENST00000615096.4:c.711C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9273894</dd>
                                    
                                    <dd>CADD: 0.927 (11.390)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0231%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0482%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 0.0327%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185722">ANKFY1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.467</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.866</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.337 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ANKFY1">ANKFY1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001407, short stride length</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000774, decreased brain size</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANKFY1%0DRAD21&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAD21</a> and phenotypic similarity 0.685 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=199">Cornelia de Lange syndrome</a> associated with RAD21.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0010864, Intellectual disability, severe</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia - HP:0000687, Widely spaced teeth</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANKFY1%0DRAD21&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAD21</a> and phenotypic similarity 0.332 to fish mutant of RAD21.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - ZP:0000054, abnormal(ly) decreased size head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.467</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.866</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.4263644C&gt;T [0/1:0/1:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.866</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000570535.5">ANKFY1:ENST00000570535.5:c.87G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341657.9">ANKFY1:ENST00000341657.9:c.10+288G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000574367.5">ANKFY1:ENST00000574367.5:c.10+288G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8657235</dd>
                                    
                                    <dd>CADD: 0.866 (8.720)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135631">RAB11FIP5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.452</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.509</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RAB11FIP5%0DEHD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EHD1</a> and phenotypic similarity 0.778 to mouse mutant of EHD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000120, malocclusion</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001304, cataract</dd><dd>HP:0000637, Long palpebral fissure - MP:0003671, abnormal eyelid aperture</dd><dd>HP:0001572, Macrodontia - MP:0000120, malocclusion</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.452</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.509</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.850</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">INTERNAL_FEATURE_ELONGATION</span>
                                <b>chr2:g.73081095T&gt;CTCC [-/1:0/-:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000486777.7">RAB11FIP5:ENST00000486777.7:c.2137delAinsGGAG:p.(Arg713Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258098.6">RAB11FIP5:ENST00000258098.6:c.1569-4913delAinsGGAG:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000164828">SUN1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.433</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.371</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.371 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SUN1">SUN1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0011967, increased or absent threshold for auditory brainstem response</dd><dd>HP:0000316, Hypertelorism - MP:0011965, decreased total retina thickness</dd><dd>HP:0000637, Long palpebral fissure - MP:0011965, decreased total retina thickness</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.433</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.371</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.843371G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537431677">rs537431677</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537431677&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389574.7">SUN1:ENST00000389574.7:c.359G&gt;A:p.(Gly120Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000401592.6">SUN1:ENST00000401592.6:c.509G&gt;A:p.(Gly170Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000403868.5">SUN1:ENST00000403868.5:c.509G&gt;A:p.(Gly170Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405266.5">SUN1:ENST00000405266.5:c.509G&gt;A:p.(Gly170Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000425407.6">SUN1:ENST00000425407.6:c.359G&gt;A:p.(Gly120Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457378.6">SUN1:ENST00000457378.6:c.572G&gt;A:p.(Gly191Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000452783.6">SUN1:ENST00000452783.6:c.451+1241G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 1.000 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.996 (23.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0105%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143631">FLG</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.422</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.769</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.542</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.769 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FLG">FLG</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0000019, thick ears</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FLG%0DUSP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to USP1</a> and phenotypic similarity 0.743 to mouse mutant of USP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000443, abnormal snout morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0003671, abnormal eyelid aperture</dd><dd>HP:0000637, Long palpebral fissure - MP:0003671, abnormal eyelid aperture</dd><dd>HP:0001572, Macrodontia - MP:0000443, abnormal snout morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/146700">OMIM:146700</a> Ichthyosis vulgaris - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.422</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.769</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.542</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.152303484T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=144046718">rs144046718</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs144046718&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.542</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368799.2">FLG:ENST00000368799.2:c.11402A&gt;G:p.(His3801Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.55</dd>
                                    
                                    <dd>Polyphen2: 0.003 (B)</dd>
                                    <dd>SIFT: 0.450 (T)</dd>
                                    <dd>CADD: 0.492 (2.945)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0998%</dd>
                                    <dd>TOPMed: 0.0199%</dd>
                                    <dd>ExAC AFR: 0.0962%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AFR: 0.0850%</dd>
                                    <dd>gnomAD_E_AMR: 0.0357%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_SAS: 0.0162%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000069018">TRPC7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.411</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.842</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.365 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TRPC7">TRPC7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005253, abnormal eye physiology</dd><dd>HP:0000637, Long palpebral fissure - MP:0005253, abnormal eye physiology</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRPC7%0DGRM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GRM1</a> and phenotypic similarity 0.483 to <a href="http://www.omim.org/614831">Spinocerebellar ataxia, autosomal recessive 13</a> associated with GRM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0007068, Inferior vermis hypoplasia</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRPC7%0DGRM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GRM1</a> and phenotypic similarity 0.678 to mouse mutant of GRM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001393, ataxia</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005287, narrow eye opening</dd><dd>HP:0000637, Long palpebral fissure - MP:0005287, narrow eye opening</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.411</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.842</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr5:g.136251719C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=577436964">rs577436964</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs577436964&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.842</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000352189.8">TRPC7:ENST00000352189.8:c.1161G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378459.7">TRPC7:ENST00000378459.7:c.1326G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000502753.4">TRPC7:ENST00000502753.4:c.1344G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513104.5">TRPC7:ENST00000513104.5:c.1509G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.843937</dd>
                                    
                                    <dd>CADD: 0.844 (8.067)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0104%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204179">PTPN20</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.400</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.357</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.998</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.357 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PTPN20">PTPN20</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0001303, abnormal lens morphology</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.400</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.357</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.998</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.46984553G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1187251387">rs1187251387</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1187251387&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.998</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374218.6">PTPN20:ENST00000374218.6:c.664G&gt;A:p.(Val222Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374339.4">PTPN20:ENST00000374339.4:c.907G&gt;A:p.(Val303Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395721.6">PTPN20:ENST00000395721.6:c.664G&gt;A:p.(Val222Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395727.6">PTPN20:ENST00000395727.6:c.331G&gt;A:p.(Val111Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505814.5">PTPN20:ENST00000505814.5:c.664G&gt;A:p.(Val222Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000506080.5">PTPN20:ENST00000506080.5:c.664G&gt;A:p.(Val222Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000511769.5">PTPN20:ENST00000511769.5:c.880G&gt;A:p.(Val294Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374342.6">PTPN20:ENST00000374342.6:c.98-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374346.7">PTPN20:ENST00000374346.7:c.584-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395722.7">PTPN20:ENST00000395722.7:c.98-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395725.7">PTPN20:ENST00000395725.7:c.341-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000502254.5">PTPN20:ENST00000502254.5:c.341-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000502705.5">PTPN20:ENST00000502705.5:c.493-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000503851.5">PTPN20:ENST00000503851.5:c.314-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000508602.5">PTPN20:ENST00000508602.5:c.341-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000508715.5">PTPN20:ENST00000508715.5:c.98-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000509599.5">PTPN20:ENST00000509599.5:c.584-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000509774.5">PTPN20:ENST00000509774.5:c.557-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000509900.5">PTPN20:ENST00000509900.5:c.341-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513156.5">PTPN20:ENST00000513156.5:c.466-2787G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513159.5">PTPN20:ENST00000513159.5:c.98-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513266.5">PTPN20:ENST00000513266.5:c.250-2787G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513756.5">PTPN20:ENST00000513756.5:c.98-15359G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.998</dd>
                                    
                                    <dd>SIFT: 0.002 (D)</dd>
                                    <dd>CADD: 0.998 (26.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187475">H1-6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.382</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.828</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=H1-6%0DKDM6A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KDM6A</a> and phenotypic similarity 0.816 to <a href="http://www.omim.org/300867">Kabuki syndrome 2</a> associated with KDM6A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000689, Dental malocclusion</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=H1-6%0DKDM6A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KDM6A</a> and phenotypic similarity 0.288 to mouse mutant of KDM6A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.382</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.828</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr6:g.26107866G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=539911407">rs539911407</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs539911407&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.828</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000338379.5">H1-6:ENST00000338379.5:c.228C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8307222</dd>
                                    
                                    <dd>CADD: 0.831 (7.714)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000254521">SIGLEC12</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.371</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.822</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SIGLEC12%0DEGR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EGR2</a> and phenotypic similarity 0.638 to mouse mutant of EGR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0002764, short tibia</dd><dd>HP:0000252, Microcephaly - MP:0000933, abnormal rhombomere morphology</dd><dd>HP:0000325, Triangular face - MP:0000458, abnormal mandible morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001290, delayed eyelid opening</dd><dd>HP:0000637, Long palpebral fissure - MP:0001290, delayed eyelid opening</dd><dd>HP:0001572, Macrodontia - MP:0000458, abnormal mandible morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.371</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.822</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.51501431G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=535238404">rs535238404</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs535238404&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.822</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000291707.8">SIGLEC12:ENST00000291707.8:c.303C&gt;A:p.(Asn101Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8256588</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 1.000 (T)</dd>
                                    <dd>CADD: 0.826 (7.586)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0056%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000116539">ASH1L</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.363</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.517</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.322 to <a href="http://www.omim.org/617796">Mental retardation, autosomal dominant 52</a> associated with ASH1L.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.517 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ASH1L">ASH1L</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0006000, abnormal corneal epithelium morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0005251, blepharitis</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ASH1L%0DPLOD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLOD2</a> and phenotypic similarity 0.617 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2771">Bruck syndrome</a> associated with PLOD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/617796">OMIM:617796</a> Mental retardation, autosomal dominant 52 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.363</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.517</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr1:g.155520359CA&gt;C [0/1:-/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=57127431">rs57127431</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs57127431&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000548830.1">ASH1L:ENST00000548830.1:c.421-4del:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368346.7">ASH1L:ENST00000368346.7:c.420+740del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392403.8">ASH1L:ENST00000392403.8:c.420+740del:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.03172213</dd>
                                    
                                    <dd>CADD: 0.032 (0.140)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000011422">PLAUR</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.352</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.357</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.977</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.357 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PLAUR">PLAUR</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001417, decreased exploration in new environment</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000788, abnormal cerebral cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.352</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.357</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.977</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">INFRAME_DELETION</span>
                                <b>chr19:g.43655493CGTT&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=750890590">rs750890590</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs750890590&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.977</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339082.7">PLAUR:ENST00000339082.7:c.550_552del:p.(Asn184del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340093.8">PLAUR:ENST00000340093.8:c.550_552del:p.(Asn184del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000601723.5">PLAUR:ENST00000601723.5:c.550_552del:p.(Asn184del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000221264.8">PLAUR:ENST00000221264.8:c.472+983_472+985del:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9785711</dd>
                                    
                                    <dd>CADD: 0.979 (16.690)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187741">FANCA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.347</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.348</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.985</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.695 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with FANCA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0008572, External ear malformation</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Phenotypic similarity 0.394 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FANCA">FANCA</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001297, microphthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0001297, microphthalmia</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FANCA%0DPALB2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PALB2</a> and phenotypic similarity 0.695 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with PALB2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0008572, External ear malformation</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FANCA%0DPALB2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PALB2</a> and phenotypic similarity 0.177 to mouse mutant of PALB2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000914, exencephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/227650">OMIM:227650</a> Fanconi anemia, complementation group A - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">ORPHA:84</a> Fanconi anemia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.347</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.348</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.985</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.89773334C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=140785340">rs140785340</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs140785340&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.985</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389301.8">FANCA:ENST00000389301.8:c.1951G&gt;A:p.(Gly651Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568369.5">FANCA:ENST00000568369.5:c.1951G&gt;A:p.(Gly651Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9938341</dd>
                                    
                                    <dd>Polyphen2: 0.556 (P)</dd>
                                    <dd>SIFT: 0.044 (D)</dd>
                                    <dd>CADD: 0.994 (22.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0056%</dd>
                                    <dd>ESP AA: 0.0493%</dd>
                                    <dd>ESP All: 0.0167%</dd>
                                    <dd>ExAC AFR: 0.0362%</dd>
                                    <dd>gnomAD_E_AFR: 0.0652%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165030">NFIL3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.332</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.804</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.379 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NFIL3">NFIL3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001312, abnormal cornea morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0001312, abnormal cornea morphology</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NFIL3%0DMESP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MESP1</a> and phenotypic similarity 0.610 to mouse mutant of MESP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0000433, microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.332</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.804</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr9:g.91410690A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=144584421">rs144584421</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs144584421&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.804</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000297689.4">NFIL3:ENST00000297689.4:c.45T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8057773</dd>
                                    
                                    <dd>CADD: 0.806 (7.117)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0099%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000132326">PER2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.327</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.509</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.792</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.386 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PER2">PER2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0009754, enhanced behavioral response to cocaine</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000619, salivary gland epithelial hyperplasia</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000619, salivary gland epithelial hyperplasia</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PER2%0DNONO&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NONO</a> and phenotypic similarity 0.734 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=466791">Macrocephaly-intellectual disability-left ventricular non compaction syndrome</a> associated with NONO.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001256, Intellectual disability, mild</dd><dd>HP:0004322, Short stature - HP:0001533, Slender build</dd><dd>HP:0000252, Microcephaly - HP:0004482, Relative macrocephaly</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000582, Upslanted palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000687, Widely spaced teeth</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/604348">OMIM:604348</a> ?Advanced sleep phase syndrome, familial, 1 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.327</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.509</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.792</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr2:g.238273064G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=373082143">rs373082143</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs373082143&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.792</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000254657.8">PER2:ENST00000254657.8:c.570+6C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6742383</dd>
                                    
                                    <dd>CADD: 0.674 (4.871)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0088%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0290%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_G_AFR: 0.0687%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000070413">DGCR2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.322</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DGCR2%0DUFD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UFD1</a> and phenotypic similarity 0.653 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=567">22q11.2 deletion syndrome</a> associated with UFD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001256, Intellectual disability, mild</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000385, Small earlobe</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000506, Telecanthus</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DGCR2%0DUFD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UFD1</a> and phenotypic similarity 0.386 to fish mutant of UFD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - ZP:0000054, abnormal(ly) decreased size head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - ZP:0002276, abnormal(ly) increased thickness mandibular arch skeleton</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.322</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr22:g.19039127G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=565271987">rs565271987</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs565271987&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263196.12">DGCR2:ENST00000263196.12:c.1397-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537045.5">DGCR2:ENST00000537045.5:c.1274-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545799.5">DGCR2:ENST00000545799.5:c.1397-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.80207527</dd>
                                    
                                    <dd>CADD: 0.802 (7.035)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0101%</dd>
                                    <dd>gnomAD_E_AFR: 0.0198%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000006607">FARP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.319</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.797</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FARP2%0DRAP1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAP1A</a> and phenotypic similarity 0.709 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2322">Kabuki syndrome</a> associated with RAP1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0007655, Eversion of lateral third of lower eyelids</dd><dd>HP:0001572, Macrodontia - HP:0000691, Microdontia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.319</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.797</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr2:g.241463894C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537031260">rs537031260</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537031260&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.797</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264042.8">FARP2:ENST00000264042.8:c.1812-5C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373287.8">FARP2:ENST00000373287.8:c.1812-5C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000627550.2">FARP2:ENST00000627550.2:c.1812-5C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.20329052</dd>
                                    
                                    <dd>CADD: 0.203 (0.987)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ExAC AFR: 0.0206%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_AFR: 0.0262%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000079432">CIC</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.318</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.798</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.322 to <a href="http://www.omim.org/617600">Mental retardation, autosomal dominant 45</a> associated with CIC.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CIC%0DRBM10&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RBM10</a> and phenotypic similarity 0.668 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2886">TARP syndrome</a> associated with RBM10.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0001508, Failure to thrive</dd><dd>HP:0000252, Microcephaly - HP:0001321, Cerebellar hypoplasia</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000395, Prominent antihelix</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0012745, Short palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0009085, Alveolar ridge overgrowth</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/617600">OMIM:617600</a> Mental retardation, autosomal dominant 45 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.318</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.798</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.42287929G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=374479285">rs374479285</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs374479285&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.798</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000160740.7">CIC:ENST00000160740.7:c.885G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000572681.6">CIC:ENST00000572681.6:c.3612G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000575354.6">CIC:ENST00000575354.6:c.885G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8087947</dd>
                                    
                                    <dd>CADD: 0.809 (7.185)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0319%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0931%</dd>
                                    <dd>ExAC AMR: 0.0380%</dd>
                                    <dd>gnomAD_E_AFR: 0.0940%</dd>
                                    <dd>gnomAD_E_AMR: 0.0158%</dd>
                                    <dd>gnomAD_E_NFE: 0.0010%</dd>
                                    <dd>gnomAD_G_AFR: 0.0917%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000134216">CHIA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.318</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.798</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CHIA%0DGUSB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GUSB</a> and phenotypic similarity 0.560 to <a href="http://www.omim.org/253220">Mucopolysaccharidosis VII</a> associated with GUSB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000574, Thick eyebrow</dd><dd>HP:0001572, Macrodontia - HP:0000687, Widely spaced teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CHIA%0DGUSB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GUSB</a> and phenotypic similarity 0.640 to mouse mutant of GUSB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0000547, short limbs</dd><dd>HP:0000252, Microcephaly - MP:0000435, shortened head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0001967, deafness</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.318</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.798</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr1:g.111315876C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=530580847">rs530580847</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs530580847&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.798</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000353665.10">CHIA:ENST00000353665.10:c.-4+4C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343320.10">CHIA:ENST00000343320.10:c.480+441C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369740.6">CHIA:ENST00000369740.6:c.480+441C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430615.1">CHIA:ENST00000430615.1:c.156+441C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000451398.6">CHIA:ENST00000451398.6:c.-4+1280C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483391.5">CHIA:ENST00000483391.5:c.-4+1280C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.14493328</dd>
                                    
                                    <dd>CADD: 0.145 (0.680)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0056%</dd>
                                    <dd>gnomAD_E_AMR: 0.0181%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000177000">MTHFR</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.312</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.672</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.601</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.384 to <a href="http://www.omim.org/236250">Homocystinuria due to MTHFR deficiency</a> associated with MTHFR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.672 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MTHFR">MTHFR</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0000852, small cerebellum</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0002750, exophthalmos</dd><dd>HP:0000637, Long palpebral fissure - MP:0002750, exophthalmos</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MTHFR%0DCBS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CBS</a> and phenotypic similarity 0.513 to <a href="http://www.omim.org/236200">Homocystinuria, B6-responsive and nonresponsive types</a> associated with CBS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0000098, Tall stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000678, Dental crowding</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MTHFR%0DCBS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CBS</a> and phenotypic similarity 0.661 to mouse mutant of CBS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0003743, abnormal facial morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001286, abnormal eye development</dd><dd>HP:0000637, Long palpebral fissure - MP:0001290, delayed eyelid opening</dd><dd>HP:0001572, Macrodontia - MP:0003743, abnormal facial morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/181500">OMIM:181500</a> Schizophrenia, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/188050">OMIM:188050</a> Thromboembolism, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/236250">OMIM:236250</a> Homocystinuria due to MTHFR deficiency - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/601634">OMIM:601634</a> Neural tube defects, susceptibility to (susceptibility)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.312</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.672</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.601</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.11790605T&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=532267156">rs532267156</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs532267156&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.601</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000641407.1">MTHFR:ENST00000641407.1:c.1864A&gt;T:p.(Asn622Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376583.7">MTHFR:ENST00000376583.7:c.*75A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376585.6">MTHFR:ENST00000376585.6:c.*75A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376590.9">MTHFR:ENST00000376590.9:c.*75A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376592.6">MTHFR:ENST00000376592.6:c.*75A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423400.7">MTHFR:ENST00000423400.7:c.*75A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.60417795</dd>
                                    
                                    <dd>CADD: 0.604 (4.025)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0399%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000111325">OGFOD2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.309</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.793</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OGFOD2%0DPSIP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PSIP1</a> and phenotypic similarity 0.762 to mouse mutant of PSIP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001505, hunched posture</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000435, shortened head</dd><dd>HP:0000325, Triangular face - MP:0030069, broad face</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005251, blepharitis</dd><dd>HP:0000637, Long palpebral fissure - MP:0005251, blepharitis</dd><dd>HP:0001572, Macrodontia - MP:0000454, abnormal jaw morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.309</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.793</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr12:g.122977982G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=535663785">rs535663785</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs535663785&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.793</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538755.5">OGFOD2:ENST00000538755.5:c.-281G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000228922.11">OGFOD2:ENST00000228922.11:c.404-460G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397389.6">OGFOD2:ENST00000397389.6:c.224-460G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454694.6">OGFOD2:ENST00000454694.6:c.-89-460G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536150.5">OGFOD2:ENST00000536150.5:c.-89-460G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538628.5">OGFOD2:ENST00000538628.5:c.-206-343G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545317.5">OGFOD2:ENST00000545317.5:c.-206-343G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545612.5">OGFOD2:ENST00000545612.5:c.-171-378G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542678.5">OGFOD2:ENST00000542678.5:c.-88+916C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.77818036</dd>
                                    
                                    <dd>CADD: 0.778 (6.540)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0143%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000103047">TANGO6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.294</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.785</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.373 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TANGO6">TANGO6</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0002092, abnormal eye morphology</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TANGO6%0DNCOA3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NCOA3</a> and phenotypic similarity 0.627 to mouse mutant of NCOA3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001289, persistence of hyaloid vascular system</dd><dd>HP:0000637, Long palpebral fissure - MP:0001289, persistence of hyaloid vascular system</dd><dd>HP:0001572, Macrodontia - MP:0002100, abnormal tooth morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.294</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.785</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr16:g.68867138A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=377669154">rs377669154</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs377669154&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.785</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261778.2">TANGO6:ENST00000261778.2:c.912A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7878221</dd>
                                    
                                    <dd>CADD: 0.788 (6.733)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0056%</dd>
                                    <dd>ESP EA: 0.0121%</dd>
                                    <dd>ESP All: 0.0083%</dd>
                                    <dd>ExAC AFR: 0.0204%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000111249">CUX2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.279</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.778</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.322 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2382">Lennox-Gastaut syndrome</a> associated with CUX2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.299 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CUX2">CUX2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000788, abnormal cerebral cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUX2%0DGBA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GBA</a> and phenotypic similarity 0.751 to <a href="http://www.omim.org/608013">Gaucher disease, perinatal lethal</a> associated with GBA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004325, Decreased body weight</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia - HP:0008551, Microtia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000278, Retrognathia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUX2%0DGBA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GBA</a> and phenotypic similarity 0.249 to mouse mutant of GBA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0004077, abnormal striatum morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/618141">OMIM:618141</a> Epileptic encephalopathy, early infantile, 67 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2382">ORPHA:2382</a> Lennox-Gastaut syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.279</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.778</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr12:g.111320436C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=539557491">rs539557491</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs539557491&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.778</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261726.11">CUX2:ENST00000261726.11:c.2427C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7798595</dd>
                                    
                                    <dd>CADD: 0.780 (6.573)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0134%</dd>
                                    <dd>gnomAD_E_AFR: 0.0075%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196365">LONP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.262</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.298</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.596 to <a href="http://www.omim.org/600373">CODAS syndrome</a> associated with LONP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0002682, Broad skull</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0009901, Crumpled ear</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia - HP:0006297, Hypoplasia of dental enamel</dd></dl><dl><dt>Phenotypic similarity 0.246 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LONP1">LONP1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LONP1%0DTBX22&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TBX22</a> and phenotypic similarity 0.621 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=921">Abruzzo-Erickson syndrome</a> associated with TBX22.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000286, Epicanthus</dd><dd>HP:0001572, Macrodontia - HP:0000272, Malar flattening</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LONP1%0DTBX22&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TBX22</a> and phenotypic similarity 0.528 to mouse mutant of TBX22.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0009652, abnormal palatal rugae morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000098, abnormal vomer bone morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/600373">OMIM:600373</a> CODAS syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1458">ORPHA:1458</a> CODAS syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.262</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.298</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.5692167C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=573698675">rs573698675</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs573698675&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360614.8">LONP1:ENST00000360614.8:c.2745G&gt;C:p.(Glu915Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540670.6">LONP1:ENST00000540670.6:c.2157G&gt;C:p.(Glu719Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000585374.5">LONP1:ENST00000585374.5:c.2403G&gt;C:p.(Glu801Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000590729.5">LONP1:ENST00000590729.5:c.2355G&gt;C:p.(Glu785Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000593119.5">LONP1:ENST00000593119.5:c.2553G&gt;C:p.(Glu851Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999838</dd>
                                    
                                    <dd>Polyphen2: 0.010 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.982 (T)</dd>
                                    <dd>CADD: 0.992 (21.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154529">CNTNAP3B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.258</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.294</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.999</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.294 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CNTNAP3B">CNTNAP3B</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0002804, abnormal motor learning</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.258</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.294</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.999</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.41964577A&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=76032838">rs76032838</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs76032838&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.999</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341990.8">CNTNAP3B:ENST00000341990.8:c.1717T&gt;G:p.(Cys573Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377561.6">CNTNAP3B:ENST00000377561.6:c.1717T&gt;G:p.(Cys573Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612828.4">CNTNAP3B:ENST00000612828.4:c.1717T&gt;G:p.(Cys573Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617422.4">CNTNAP3B:ENST00000617422.4:c.1478-3685T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99871176</dd>
                                    
                                    <dd>CADD: 0.999 (28.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr9:g.41929326G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2935342">rs2935342</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2935342&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.994</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377561.6">CNTNAP3B:ENST00000377561.6:c.2356C&gt;T:p.(Arg786Cys)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612828.4">CNTNAP3B:ENST00000612828.4:c.2353C&gt;T:p.(Arg785Cys)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9939744</dd>
                        
                        <dd>CADD: 0.994 (22.200)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr9:g.41906917G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1329237256">rs1329237256</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1329237256&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.913</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377561.6">CNTNAP3B:ENST00000377561.6:c.3291C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612828.4">CNTNAP3B:ENST00000612828.4:c.3048C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.91250163</dd>
                        
                        <dd>CADD: 0.913 (10.580)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr9:g.41991685C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=62555056">rs62555056</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs62555056&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.081</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341990.8">CNTNAP3B:ENST00000341990.8:c.1258G&gt;A:p.(Val420Ile)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377561.6">CNTNAP3B:ENST00000377561.6:c.1258G&gt;A:p.(Val420Ile)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612828.4">CNTNAP3B:ENST00000612828.4:c.1258G&gt;A:p.(Val420Ile)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617422.4">CNTNAP3B:ENST00000617422.4:c.1258G&gt;A:p.(Val420Ile)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.081455946</dd>
                        
                        <dd>CADD: 0.081 (0.369)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104529">EEF1D</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.241</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.750</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=EEF1D%0DHBS1L&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HBS1L</a> and phenotypic similarity 0.707 to mouse mutant of HBS1L.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0003743, abnormal facial morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001314, corneal opacity</dd><dd>HP:0000637, Long palpebral fissure - MP:0001314, corneal opacity</dd><dd>HP:0001572, Macrodontia - MP:0002100, abnormal tooth morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.241</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.750</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr8:g.143589326G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=374067636">rs374067636</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs374067636&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.750</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423316.6">EEF1D:ENST00000423316.6:c.756C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000442189.6">EEF1D:ENST00000442189.6:c.756C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000532741.5">EEF1D:ENST00000532741.5:c.906C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618139.2">EEF1D:ENST00000618139.2:c.756C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317198.10">EEF1D:ENST00000317198.10:c.-7-2474C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395119.7">EEF1D:ENST00000395119.7:c.-7-2474C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000419152.6">EEF1D:ENST00000419152.6:c.-4-2477C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524624.5">EEF1D:ENST00000524624.5:c.-7-2474C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000526838.5">EEF1D:ENST00000526838.5:c.-7-2474C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000528610.5">EEF1D:ENST00000528610.5:c.-7-2474C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529272.5">EEF1D:ENST00000529272.5:c.-4-2477C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000531621.5">EEF1D:ENST00000531621.5:c.-7-2474C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000532400.1">EEF1D:ENST00000532400.1:c.-7-2474C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.75237185</dd>
                                    
                                    <dd>CADD: 0.752 (6.062)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ExAC NFE: 0.0263%</dd>
                                    <dd>gnomAD_E_NFE: 0.0099%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0134%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000151779">NBAS</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.229</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.281</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.561 to <a href="http://www.omim.org/614800">Short stature, optic nerve atrophy, and Pelger-Huet anomaly</a> associated with NBAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000248, Brachycephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000574, Thick eyebrow</dd><dd>HP:0001572, Macrodontia - HP:0002057, Prominent glabella</dd></dl><dl><dt>Phenotypic similarity 0.375 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NBAS">NBAS</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001289, persistence of hyaloid vascular system</dd><dd>HP:0000637, Long palpebral fissure - MP:0001289, persistence of hyaloid vascular system</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NBAS%0DGPKOW&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GPKOW</a> and phenotypic similarity 0.626 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2570">Lethal intrauterine growth restriction-cortical malformation-congenital contractures syndrome</a> associated with GPKOW.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001511, Intrauterine growth retardation</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000637, Long palpebral fissure - HP:0000581, Blepharophimosis</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614800">OMIM:614800</a> Short stature, optic nerve atrophy, and Pelger-Huet anomaly - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/616483">OMIM:616483</a> Infantile liver failure syndrome 2 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.229</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.281</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.15417640T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=574433313">rs574433313</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs574433313&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000281513.10">NBAS:ENST00000281513.10:c.2650A&gt;G:p.(Asn884Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.003 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 1.000 (T)</dd>
                                    <dd>CADD: 0.984 (18.080)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000016082">ISL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.227</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.294</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.981</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.587 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ISL1">ISL1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0030419, excessive cerumen</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.332 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=ISL1">ISL1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - ZP:0000054, abnormal(ly) decreased size head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ISL1%0DRLIM&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RLIM</a> and phenotypic similarity 0.627 to <a href="http://www.omim.org/300978">Tonne-Kalscheuer syndrome</a> associated with RLIM.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000494, Downslanted palpebral fissures</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93930">ORPHA:93930</a> Bladder exstrophy</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.227</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.294</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.981</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr5:g.51387733G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=544000199">rs544000199</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs544000199&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.981</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000230658.12">ISL1:ENST00000230658.12:c.462G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000511384.1">ISL1:ENST00000511384.1:c.462G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98366946</dd>
                                    
                                    <dd>CADD: 0.984 (17.870)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0106%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000039139">DNAH5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.207</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.268</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.171 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with DNAH5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0011109, Chronic sinusitis</dd></dl><dl><dt>Phenotypic similarity 0.536 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DNAH5">DNAH5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001406, abnormal gait</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0000439, enlarged cranium</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0001963, abnormal hearing physiology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAH5%0DDCTN5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DCTN5</a> and phenotypic similarity 0.744 to mouse mutant of DCTN5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000433, microcephaly</dd><dd>HP:0000325, Triangular face - MP:0000452, abnormal mouth morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0002092, abnormal eye morphology</dd><dd>HP:0001572, Macrodontia - MP:0002639, micrognathia</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/608644">OMIM:608644</a> Ciliary dyskinesia, primary, 3, with or without situs inversus - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">ORPHA:244</a> Primary ciliary dyskinesia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.207</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.268</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr5:g.13754206C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=558861943">rs558861943</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs558861943&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265104.4">DNAH5:ENST00000265104.4:c.10552G&gt;C:p.(Val3518Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.000 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.010 (D)</dd>
                                    <dd>CADD: 0.993 (21.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196177">ACADSB</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.182</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.384 to <a href="http://www.omim.org/610006">2-methylbutyrylglycinuria</a> associated with ACADSB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.478 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ACADSB">ACADSB</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000452, abnormal mouth morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0001303, abnormal lens morphology</dd><dd>HP:0001572, Macrodontia - MP:0000452, abnormal mouth morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACADSB%0DSLC25A21&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC25A21</a> and phenotypic similarity 0.230 to <a href="http://www.omim.org/618811">?Mitochondrial DNA depletion syndrome 18</a> associated with SLC25A21.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001508, Failure to thrive</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACADSB%0DSLC25A21&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC25A21</a> and phenotypic similarity 0.634 to mouse mutant of SLC25A21.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000443, abnormal snout morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0002100, abnormal tooth morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/610006">OMIM:610006</a> 2-methylbutyrylglycinuria - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.182</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                                <b>chr10:g.123051188T&gt;TAA [0/1:-/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=10571424">rs10571424</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs10571424&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358776.7">ACADSB:ENST00000358776.7:c.1128+2_1128+3insAA:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368869.8">ACADSB:ENST00000368869.8:c.822+2_822+3insAA:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.987782</dd>
                                    
                                    <dd>CADD: 0.988 (19.130)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000168769">TET2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.181</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.235 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=729">Polycythemia vera</a> associated with TET2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001824, Weight loss</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TET2%0DASXL1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ASXL1</a> and phenotypic similarity 0.626 to <a href="http://www.omim.org/605039">Bohring-Opitz syndrome</a> associated with ASXL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002187, Intellectual disability, profound</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000582, Upslanted palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000187, Broad alveolar ridges</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TET2%0DASXL1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ASXL1</a> and phenotypic similarity 0.545 to mouse mutant of ASXL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000111, cleft palate</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001293, anophthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0001293, anophthalmia</dd><dd>HP:0001572, Macrodontia - MP:0000460, mandible hypoplasia</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614286">OMIM:614286</a> Myelodysplastic syndrome, somatic - somatic</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3318">ORPHA:3318</a> Essential thrombocythemia</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=729">ORPHA:729</a> Polycythemia vera</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=824">ORPHA:824</a> Primary myelofibrosis</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.181</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.105275647C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=550187530">rs550187530</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs550187530&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380013.8">TET2:ENST00000380013.8:c.5137C&gt;T:p.(Pro1713Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513237.5">TET2:ENST00000513237.5:c.5200C&gt;T:p.(Pro1734Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540549.5">TET2:ENST00000540549.5:c.5137C&gt;T:p.(Pro1713Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.686 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.006 (D)</dd>
                                    <dd>CADD: 0.995 (23.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0048%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167306">MYO5B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.180</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.276 to <a href="http://www.omim.org/251850">Microvillus inclusion disease</a> associated with MYO5B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001510, Growth delay</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MYO5B%0DTNNI2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TNNI2</a> and phenotypic similarity 0.686 to <a href="http://www.omim.org/601680">Arthrogryposis, distal, type 2B1</a> associated with TNNI2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000494, Downslanted palpebral fissures</dd><dd>HP:0001572, Macrodontia - HP:0000303, Mandibular prognathia</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MYO5B%0DTNNI2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TNNI2</a> and phenotypic similarity 0.321 to mouse mutant of TNNI2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0004355, short radius</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/251850">OMIM:251850</a> Microvillus inclusion disease - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2290">ORPHA:2290</a> Microvillus inclusion disease</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.180</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr18:g.49902674G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=370548179">rs370548179</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs370548179&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=879325%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000285039.12">MYO5B:ENST00000285039.12:c.2731C&gt;T:p.(Arg911Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.971 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 1.000 (34.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ESP EA: 0.0120%</dd>
                                    <dd>ESP All: 0.0081%</dd>
                                    <dd>ExAC AFR: 0.0205%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000124198">ARFGEF2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.178</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.422 to <a href="http://www.omim.org/608097">Periventricular heterotopia with microcephaly</a> associated with ARFGEF2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0010864, Intellectual disability, severe</dd><dd>HP:0004322, Short stature - HP:0001508, Failure to thrive</dd><dd>HP:0000252, Microcephaly - HP:0000253, Progressive microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.265 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ARFGEF2">ARFGEF2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000788, abnormal cerebral cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARFGEF2%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.608 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=950">Acrodysostosis</a> associated with PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000248, Brachycephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000286, Epicanthus</dd><dd>HP:0001572, Macrodontia - HP:0010807, Open bite</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARFGEF2%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.246 to mouse mutant of PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/608097">OMIM:608097</a> Periventricular heterotopia with microcephaly - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98892">ORPHA:98892</a> Periventricular nodular heterotopia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.178</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr20:g.49023166C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=540964686">rs540964686</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs540964686&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371917.5">ARFGEF2:ENST00000371917.5:c.4740C&gt;G:p.(His1580Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999992</dd>
                                    
                                    <dd>Polyphen2: 0.084 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.252 (T)</dd>
                                    <dd>CADD: 0.994 (22.400)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000133805">AMPD3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.178</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=AMPD3%0DXPNPEP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to XPNPEP1</a> and phenotypic similarity 0.633 to mouse mutant of XPNPEP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0000433, microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=45">ORPHA:45</a> Adenosine monophosphate deaminase deficiency</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.178</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr11:g.10505804C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=368883763">rs368883763</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs368883763&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396553.6">AMPD3:ENST00000396553.6:c.2224C&gt;T:p.(Arg742*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396554.7">AMPD3:ENST00000396554.7:c.2251C&gt;T:p.(Arg751*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000444303.6">AMPD3:ENST00000444303.6:c.1747C&gt;T:p.(Arg583*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000528723.5">AMPD3:ENST00000528723.5:c.2245C&gt;T:p.(Arg749*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529507.5">AMPD3:ENST00000529507.5:c.2224C&gt;T:p.(Arg742*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 1.000 (42.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0193%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000107249">GLIS3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.177</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.419 to <a href="http://www.omim.org/610199">Diabetes mellitus, neonatal, with congenital hypothyroidism</a> associated with GLIS3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001511, Intrauterine growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000286, Epicanthus</dd><dd>HP:0001572, Macrodontia - HP:0000260, Wide anterior fontanel</dd></dl><dl><dt>Phenotypic similarity 0.288 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GLIS3">GLIS3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GLIS3%0DSUFU&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SUFU</a> and phenotypic similarity 0.595 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=377">Gorlin syndrome</a> associated with SUFU.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000248, Brachycephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000506, Telecanthus</dd><dd>HP:0001572, Macrodontia - HP:0000670, Carious teeth</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GLIS3%0DSUFU&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SUFU</a> and phenotypic similarity 0.673 to mouse mutant of SUFU.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000433, microcephaly</dd><dd>HP:0000325, Triangular face - MP:0000111, cleft palate</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0002639, micrognathia</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GLIS3%0DSUFU&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SUFU</a> and phenotypic similarity 0.355 to fish mutant of SUFU.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - ZP:0002909, abnormal(ly) dwarf-like whole organism</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - ZP:0001246, abnormal(ly) morphology inner ear</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/610199">OMIM:610199</a> Diabetes mellitus, neonatal, with congenital hypothyroidism - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.177</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.4286337G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=114010705">rs114010705</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs114010705&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381971.8">GLIS3:ENST00000381971.8:c.89C&gt;A:p.(Ala30Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9957342</dd>
                                    
                                    <dd>Polyphen2: 0.068 (B)</dd>
                                    <dd>SIFT: 0.019 (D)</dd>
                                    <dd>CADD: 0.996 (23.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC AFR: 0.0102%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170113">NIPA1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.175</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.256</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.988</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.512 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NIPA1%0DSPART&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SPART</a> and phenotypic similarity 0.640 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=101000">Autosomal recessive spastic paraplegia type 20</a> associated with SPART.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000494, Downslanted palpebral fissures</dd><dd>HP:0001572, Macrodontia - HP:0011094, Overbite</dd></dl><dl><dt>Proximity score 0.512 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NIPA1%0DSPART&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SPART</a> and phenotypic similarity 0.319 to mouse mutant of SPART.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001406, abnormal gait</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0009434, paraparesis</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/600363">OMIM:600363</a> Spastic paraplegia 6, autosomal dominant - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=100988">ORPHA:100988</a> Autosomal dominant spastic paraplegia type 6</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.175</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.256</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.988</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                                <b>chr15:g.22786677AGCG&gt;A [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=531550505">rs531550505</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs531550505&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=190659%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.988</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000337435.9">NIPA1:ENST00000337435.9:c.45_47del:p.(Ala16del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000437912.6">NIPA1:ENST00000437912.6:c.-48+12365_-48+12367del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000561183.5">NIPA1:ENST00000561183.5:c.-48+430_-48+432del:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9883319</dd>
                                    
                                    <dd>CADD: 0.988 (19.330)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000113971">NPHP3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.175</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.280</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.962</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.357 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3156">Senior-Loken syndrome</a> associated with NPHP3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.559 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NPHP3">NPHP3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000120, malocclusion</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000120, malocclusion</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/208540">OMIM:208540</a> Renal-hepatic-pancreatic dysplasia 1 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/267010">OMIM:267010</a> Meckel syndrome 7 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/604387">OMIM:604387</a> Nephronophthisis 3 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3032">ORPHA:3032</a> NPHP3-related Meckel-like syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3156">ORPHA:3156</a> Senior-Loken syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.175</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.280</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.962</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.132713217T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=372145755">rs372145755</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs372145755&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=250948%5Balleleid%5D">(criteria_provided,_conflicting_interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.962</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000337331.10">NPHP3:ENST00000337331.10:c.1027A&gt;G:p.(Ile343Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000471702.2">NPHP3:ENST00000471702.2:c.1027A&gt;G:p.(Ile343Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9657232</dd>
                                    
                                    <dd>Polyphen2: 0.000 (B)</dd>
                                    <dd>SIFT: 1.000 (T)</dd>
                                    <dd>CADD: 0.966 (14.650)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0316%</dd>
                                    <dd>gnomAD_E_AFR: 0.0265%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000151835">SACS</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.173</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.322 to <a href="http://www.omim.org/270550">Spastic ataxia, Charlevoix-Saguenay type</a> associated with SACS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.337 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SACS">SACS</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001525, impaired balance</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000877, abnormal Purkinje cell morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SACS%0DCCL13&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCL13</a> and phenotypic similarity 0.630 to mouse mutant of CCL13.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0002805, abnormal conditioned taste aversion behavior</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000122, premature tooth eruption</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0006187, retinal deposits</dd><dd>HP:0000637, Long palpebral fissure - MP:0006187, retinal deposits</dd><dd>HP:0001572, Macrodontia - MP:0000122, premature tooth eruption</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/270550">OMIM:270550</a> Spastic ataxia, Charlevoix-Saguenay type - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98">ORPHA:98</a> Autosomal recessive spastic ataxia of Charlevoix-Saguenay</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.173</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr13:g.23355545A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=149444716">rs149444716</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs149444716&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382292.9">SACS:ENST00000382292.9:c.1067T&gt;C:p.(Ile356Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423156.2">SACS:ENST00000423156.2:c.1067T&gt;C:p.(Ile356Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000455470.6">SACS:ENST00000455470.6:c.1067T&gt;C:p.(Ile356Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000402364.1">SACS:ENST00000402364.1:c.-1184T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.039 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.056 (D)</dd>
                                    <dd>CADD: 0.995 (22.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0104%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0289%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165138">ANKS6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.171</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.246 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ANKS6">ANKS6</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0004201, fetal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANKS6%0DB9D2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to B9D2</a> and phenotypic similarity 0.587 to <a href="http://www.omim.org/614175">Joubert syndrome 34</a> associated with B9D2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0002323, Anencephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000377, Abnormality of the pinna</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0045025, Narrow palpebral fissure</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANKS6%0DB9D2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to B9D2</a> and phenotypic similarity 0.655 to mouse mutant of B9D2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0004268, abnormal optic stalk morphology</dd><dd>HP:0000325, Triangular face - MP:0000764, abnormal tongue epithelium morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0003078, aphakia</dd><dd>HP:0000637, Long palpebral fissure - MP:0003078, aphakia</dd><dd>HP:0001572, Macrodontia - MP:0005358, abnormal incisor morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/615382">OMIM:615382</a> Nephronophthisis 16 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.171</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.98790535C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=561600690">rs561600690</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs561600690&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000353234.5">ANKS6:ENST00000353234.5:c.431G&gt;C:p.(Gly144Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375019.6">ANKS6:ENST00000375019.6:c.-41-5659G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.998 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.020 (D)</dd>
                                    <dd>CADD: 0.997 (25.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0104%</dd>
                                    <dd>ExAC AFR: 0.0541%</dd>
                                    <dd>gnomAD_E_AFR: 0.0596%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_G_AFR: 0.0458%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000085185">BCORL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.169</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.991</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.470 to <a href="http://www.omim.org/301029">Shukla-Vernon syndrome</a> associated with BCORL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000494, Downslanted palpebral fissures</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=BCORL1%0DANKRD11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANKRD11</a> and phenotypic similarity 0.969 to <a href="http://www.omim.org/148050">KBG syndrome</a> associated with ANKRD11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0001572, Macrodontia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=BCORL1%0DANKRD11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANKRD11</a> and phenotypic similarity 0.541 to mouse mutant of ANKRD11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000441, increased cranium width</dd><dd>HP:0000325, Triangular face - MP:0030069, broad face</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000102, abnormal nasal bone morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/301029">OMIM:301029</a> Shukla-Vernon syndrome - X-linked recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.169</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.991</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.130015791C&gt;A [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=770210399">rs770210399</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs770210399&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.991</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000218147.11">BCORL1:ENST00000218147.11:c.3019C&gt;A:p.(Pro1007Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540052.5">BCORL1:ENST00000540052.5:c.3019C&gt;A:p.(Pro1007Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999</dd>
                                    
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.995 (22.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0530%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ExAC AFR: 0.0587%</dd>
                                    <dd>gnomAD_E_AFR: 0.0490%</dd>
                                    <dd>gnomAD_E_AMR: 0.0038%</dd>
                                    <dd>gnomAD_E_OTH: 0.0246%</dd>
                                    <dd>gnomAD_G_AFR: 0.0339%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136381">IREB2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.163</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.986</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.400 to <a href="http://www.omim.org/618451">Neurodegeneration, early-onset, with choreoathetoid movements and microcytic anemia</a> associated with IREB2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.416 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=IREB2">IREB2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001504, abnormal posture</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000832, abnormal thalamus morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0004742, abnormal vestibular system physiology</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=IREB2%0DOTULIN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OTULIN</a> and phenotypic similarity 0.230 to <a href="http://www.omim.org/617099">Autoinflammation, panniculitis, and dermatosis syndrome</a> associated with OTULIN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001508, Failure to thrive</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=IREB2%0DOTULIN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OTULIN</a> and phenotypic similarity 0.663 to mouse mutant of OTULIN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000433, microcephaly</dd><dd>HP:0000325, Triangular face - MP:0002639, micrognathia</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0002639, micrognathia</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618451">OMIM:618451</a> Neurodegeneration, early-onset, with choreoathetoid movements and microcytic anemia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.163</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.986</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr15:g.78490435G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=564994305">rs564994305</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs564994305&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.986</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258886.13">IREB2:ENST00000258886.13:c.2090G&gt;A:p.(Arg697Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9942456</dd>
                                    
                                    <dd>Polyphen2: 0.009 (B)</dd>
                                    <dd>Mutation Taster: 0.689</dd>
                                    <dd>SIFT: 0.440 (T)</dd>
                                    <dd>CADD: 0.994 (22.400)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC SAS: 0.0200%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_SAS: 0.0108%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_EAS: 0.0617%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000090889">KIF4A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.161</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.254</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.980</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.322 to <a href="http://www.omim.org/300923">?Mental retardation, X-linked 100</a> associated with KIF4A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIF4A%0DCIT&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CIT</a> and phenotypic similarity 0.689 to <a href="http://www.omim.org/617090">Microcephaly 17, primary, autosomal recessive</a> associated with CIT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIF4A%0DCIT&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CIT</a> and phenotypic similarity 0.392 to mouse mutant of CIT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0002906, increased susceptibility to pharmacologically induced seizures</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0008540, abnormal cerebral hemisphere morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/300923">OMIM:300923</a> ?Mental retardation, X-linked 100 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.161</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.254</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.980</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr23:g.70395673T&gt;C [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=780556114">rs780556114</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs780556114&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.980</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374403.4">KIF4A:ENST00000374403.4:c.2235T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9832891</dd>
                                    
                                    <dd>CADD: 0.983 (17.770)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0265%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC AFR: 0.0118%</dd>
                                    <dd>gnomAD_E_AFR: 0.0082%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000095485">CWF19L1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.160</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.983</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.396 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=453521">Autosomal recessive cerebellar ataxia due to CWF19L1 deficiency</a> associated with CWF19L1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0002342, Intellectual disability, moderate</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0040196, Mild microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000664, Synophrys</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CWF19L1%0DCWC27&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CWC27</a> and phenotypic similarity 0.688 to <a href="http://www.omim.org/250410">Retinitis pigmentosa with or without skeletal anomalies</a> associated with CWC27.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000494, Downslanted palpebral fissures</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CWF19L1%0DCWC27&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CWC27</a> and phenotypic similarity 0.398 to mouse mutant of CWC27.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0012144, decreased b wave amplitude</dd><dd>HP:0000637, Long palpebral fissure - MP:0012144, decreased b wave amplitude</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/616127">OMIM:616127</a> Spinocerebellar ataxia, autosomal recessive 17 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=453521">ORPHA:453521</a> Autosomal recessive cerebellar ataxia due to CWF19L1 deficiency</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.160</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.983</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">INFRAME_DELETION</span>
                                <b>chr10:g.100245826GAGA&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=779118978">rs779118978</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs779118978&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.983</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354105.10">CWF19L1:ENST00000354105.10:c.934_936del:p.(Ser312del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9870282</dd>
                                    
                                    <dd>CADD: 0.987 (18.870)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>ESP EA: 0.0121%</dd>
                                    <dd>ESP All: 0.0080%</dd>
                                    <dd>ExAC AFR: 0.0288%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AFR: 0.0261%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000239998">LILRA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.160</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.245</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.245 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LILRA2">LILRA2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000805, abnormal visual cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.160</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.245</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54574799C&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1304061217">rs1304061217</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1304061217&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251376.7">LILRA2:ENST00000251376.7:c.421C&gt;A:p.(Leu141Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251377.7">LILRA2:ENST00000251377.7:c.421C&gt;A:p.(Leu141Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391737.3">LILRA2:ENST00000391737.3:c.385C&gt;A:p.(Leu129Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391738.8">LILRA2:ENST00000391738.8:c.421C&gt;A:p.(Leu141Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000629481.1">LILRA2:ENST00000629481.1:c.314+255C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98906046</dd>
                                    
                                    <dd>Polyphen2: 0.719 (P)</dd>
                                    <dd>SIFT: 0.031 (D)</dd>
                                    <dd>CADD: 0.989 (19.610)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr19:g.54574792C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=142262609">rs142262609</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs142262609&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.001</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251376.7">LILRA2:ENST00000251376.7:c.414C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251377.7">LILRA2:ENST00000251377.7:c.414C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391737.3">LILRA2:ENST00000391737.3:c.378C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391738.8">LILRA2:ENST00000391738.8:c.414C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000629481.1">LILRA2:ENST00000629481.1:c.314+248C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.0011506081</dd>
                        
                        <dd>CADD: 0.001 (0.005)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>ESP EA: 0.0349%</dd>
                        <dd>ESP All: 0.0231%</dd>
                        <dd>ExAC AFR: 0.0096%</dd>
                        <dd>ExAC AMR: 0.0691%</dd>
                        <dd>ExAC NFE: 0.0090%</dd>
                        <dd>ExAC SAS: 0.0363%</dd>
                        <dd>gnomAD_E_AFR: 0.0065%</dd>
                        <dd>gnomAD_E_AMR: 0.0417%</dd>
                        <dd>gnomAD_E_NFE: 0.0161%</dd>
                        <dd>gnomAD_E_SAS: 0.0162%</dd>
                        <dd>gnomAD_G_AFR: 0.0230%</dd>
                        <dd>gnomAD_G_FIN: 0.0287%</dd>
                        <dd>gnomAD_G_NFE: 0.0067%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000008710">PKD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.159</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.699</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.405 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PKD1">PKD1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0002639, micrognathia</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0002639, micrognathia</dd></dl><dl><dt>Phenotypic similarity 0.294 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=PKD1">PKD1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - ZP:0001708, abnormal(ly) malformed mandibular arch skeleton</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKD1%0DKIF7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KIF7</a> and phenotypic similarity 0.621 to <a href="http://www.omim.org/200990">Acrocallosal syndrome</a> associated with KIF7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0001510, Growth delay</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000377, Abnormality of the pinna</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000494, Downslanted palpebral fissures</dd><dd>HP:0001572, Macrodontia - HP:0000685, Hypoplasia of teeth</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKD1%0DKIF7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KIF7</a> and phenotypic similarity 0.463 to mouse mutant of KIF7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0001891, hydrocephaly</dd><dd>HP:0000325, Triangular face - MP:0005170, cleft upper lip</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001293, anophthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0001293, anophthalmia</dd><dd>HP:0001572, Macrodontia - MP:0005170, cleft upper lip</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKD1%0DKIF7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KIF7</a> and phenotypic similarity 0.332 to fish mutant of KIF7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - ZP:0000054, abnormal(ly) decreased size head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/173900">OMIM:173900</a> Polycystic kidney disease 1 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=730">ORPHA:730</a> Autosomal dominant polycystic kidney disease</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.159</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.699</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.2108565G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=564434419">rs564434419</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs564434419&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.699</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262304.9">PKD1:ENST00000262304.9:c.6602C&gt;T:p.(Pro2201Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423118.5">PKD1:ENST00000423118.5:c.6602C&gt;T:p.(Pro2201Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.70077354</dd>
                                    
                                    <dd>Polyphen2: 0.004 (B)</dd>
                                    <dd>SIFT: 0.516 (T)</dd>
                                    <dd>CADD: 0.701 (5.240)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>gnomAD_E_AMR: 0.0072%</dd>
                                    <dd>gnomAD_E_NFE: 0.0013%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000175106">TVP23C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.126</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.670</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TVP23C%0DROR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ROR2</a> and phenotypic similarity 0.824 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1507">Autosomal recessive Robinow syndrome</a> associated with ROR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0008873, Disproportionate short-limb short stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000368, Low-set, posteriorly rotated ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000637, Long palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0011069, Increased number of teeth</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TVP23C%0DROR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ROR2</a> and phenotypic similarity 0.883 to mouse mutant of ROR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0004355, short radius</dd><dd>HP:0000252, Microcephaly - MP:0000441, increased cranium width</dd><dd>HP:0000325, Triangular face - MP:0004726, abnormal nasal capsule morphology</dd><dd>HP:0000400, Macrotia - MP:0030321, abnormal tegmen tympani morphology</dd><dd>HP:0000316, Hypertelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - MP:0001300, ocular hypertelorism</dd><dd>HP:0001572, Macrodontia - MP:0000454, abnormal jaw morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TVP23C%0DROR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ROR2</a> and phenotypic similarity 0.332 to fish mutant of ROR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - ZP:0000054, abnormal(ly) decreased size head</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.126</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.670</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.15503044A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=375535999">rs375535999</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs375535999&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.670</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000225576.7">TVP23C:ENST00000225576.7:c.651T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000522212.6">TVP23C:ENST00000522212.6:c.462+42741T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.67250854</dd>
                                    
                                    <dd>CADD: 0.673 (4.848)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000266074">BAHCC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.125</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.268</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.932</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.268 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=BAHCC1">BAHCC1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001406, abnormal gait</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.125</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.268</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.932</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.81447309C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537847945">rs537847945</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537847945&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.932</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000307745.8">BAHCC1:ENST00000307745.8:c.3437C&gt;T:p.(Ala1146Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000584436.7">BAHCC1:ENST00000584436.7:c.3530C&gt;T:p.(Ala1177Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.937</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.063 (T)</dd>
                                    <dd>CADD: 0.175 (0.835)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC SAS: 0.0159%</dd>
                                    <dd>gnomAD_E_AFR: 0.0100%</dd>
                                    <dd>gnomAD_E_SAS: 0.0083%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000126870">WDR60</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.124</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.951</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.419 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93271">Short rib-polydactyly syndrome, Verma-Naumoff type</a> associated with DYNC2I1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0008873, Disproportionate short-limb short stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000286, Epicanthus</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Phenotypic similarity 0.246 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=WDR60">WDR60</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=WDR60%0DDCTN5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DCTN5</a> and phenotypic similarity 0.744 to mouse mutant of DCTN5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000433, microcephaly</dd><dd>HP:0000325, Triangular face - MP:0000452, abnormal mouth morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000637, Long palpebral fissure - MP:0002092, abnormal eye morphology</dd><dd>HP:0001572, Macrodontia - MP:0002639, micrognathia</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/615503">OMIM:615503</a> Short-rib thoracic dysplasia 8 with or without polydactyly - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=474">ORPHA:474</a> Jeune syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93271">ORPHA:93271</a> Short rib-polydactyly syndrome, Verma-Naumoff type</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.124</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.951</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.158871557G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=541525679">rs541525679</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs541525679&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.951</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000407559.8">WDR60:ENST00000407559.8:c.485G&gt;T:p.(Arg162Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.95926195</dd>
                                    
                                    <dd>Polyphen2: 0.103 (B)</dd>
                                    <dd>SIFT: 0.094 (T)</dd>
                                    <dd>CADD: 0.959 (13.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>ExAC SAS: 0.0155%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_OTH: 0.0604%</dd>
                                    <dd>gnomAD_E_SAS: 0.0139%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169836">TACR3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.115</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.256</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.935</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.512 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=432">Normosmic congenital hypogonadotropic hypogonadism</a> associated with TACR3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0000002, Abnormality of body height</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000164, Abnormality of the dentition</dd></dl><dl><dt>Phenotypic similarity 0.332 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TACR3">TACR3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001463, abnormal spatial learning</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TACR3%0DGRM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GRM1</a> and phenotypic similarity 0.483 to <a href="http://www.omim.org/614831">Spinocerebellar ataxia, autosomal recessive 13</a> associated with GRM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0007068, Inferior vermis hypoplasia</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TACR3%0DGRM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GRM1</a> and phenotypic similarity 0.678 to mouse mutant of GRM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001393, ataxia</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0005287, narrow eye opening</dd><dd>HP:0000637, Long palpebral fissure - MP:0005287, narrow eye opening</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614840">OMIM:614840</a> Hypogonadotropic hypogonadism 11 with or without anosmia - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=432">ORPHA:432</a> Normosmic congenital hypogonadotropic hypogonadism</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=478">ORPHA:478</a> Kallmann syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.115</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.256</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.935</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.103719587G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=538867830">rs538867830</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs538867830&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.935</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000304883.3">TACR3:ENST00000304883.3:c.89C&gt;T:p.(Ala30Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.94138616</dd>
                                    
                                    <dd>Polyphen2: 0.000 (B)</dd>
                                    <dd>SIFT: 0.989 (T)</dd>
                                    <dd>CADD: 0.941 (12.320)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0096%</dd>
                                    <dd>ExAC AFR: 0.0300%</dd>
                                    <dd>gnomAD_E_AFR: 0.0198%</dd>
                                    <dd>gnomAD_G_AFR: 0.0459%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000115840">SLC25A12</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.103</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.928</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.413 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SLC25A12">SLC25A12</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001525, impaired balance</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0012468, decreased striatum size</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC25A12%0DSLC25A21&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC25A21</a> and phenotypic similarity 0.230 to <a href="http://www.omim.org/618811">?Mitochondrial DNA depletion syndrome 18</a> associated with SLC25A21.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001508, Failure to thrive</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC25A12%0DSLC25A21&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC25A21</a> and phenotypic similarity 0.634 to mouse mutant of SLC25A21.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000443, abnormal snout morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0002100, abnormal tooth morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/612949">OMIM:612949</a> Epileptic encephalopathy, early infantile, 39 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.103</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.928</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.171837264G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=138459110">rs138459110</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs138459110&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    <span class="label label-info">LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=683421%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.928</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422440.7">SLC25A12:ENST00000422440.7:c.469C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.93690425</dd>
                                    
                                    <dd>CADD: 0.937 (12.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0127%</dd>
                                    <dd>ESP AA: 0.0681%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000286219">NOTCH2NLC</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.101</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.184</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.184 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2289">Neuronal intranuclear inclusion disease</a> associated with NOTCH2NLC.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000602, Ophthalmoplegia</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2289">ORPHA:2289</a> Neuronal intranuclear inclusion disease</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.101</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.184</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr1:g.149430945C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1431569926">rs1431569926</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1431569926&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000650865.1">NOTCH2NLC:ENST00000650865.1:c.139C&gt;T:p.(Arg47*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621744.4">NOTCH2NLC:ENST00000621744.4:c.-959C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000578189.1">NOTCH2NLC:ENST00000578189.1:c.-33C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000652191.1">NOTCH2NLC:ENST00000652191.1:c.-33C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98465383</dd>
                                    
                                    <dd>CADD: 0.985 (18.140)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.037</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.092</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.988</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_INSERTION</span>
                                <b>chr1:g.149390802A&gt;AGGCGGCGGCGGCGGCGGCGGCGGCGGC [1/1:1/1:-/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1179469287">rs1179469287</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1179469287&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.988</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000650865.1">NOTCH2NLC:ENST00000650865.1:c.18_44dup:p.(Gly10_Gly18dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621744.4">NOTCH2NLC:ENST00000621744.4:c.-1088_-1062dup:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000652191.1">NOTCH2NLC:ENST00000652191.1:c.-162_-136dup:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9876973</dd>
                                    
                                    <dd>CADD: 0.988 (19.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000151849">CENPJ</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.087</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.343</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.803</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.619 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=808">Seckel syndrome</a> associated with CENPJ.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000387, Absent earlobe</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000494, Downslanted palpebral fissures</dd><dd>HP:0001572, Macrodontia - HP:0009804, Reduced number of teeth</dd></dl><dl><dt>Phenotypic similarity 0.686 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CENPJ">CENPJ</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0005352, small cranium</dd><dd>HP:0000325, Triangular face - MP:0030048, sloping forehead</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001293, anophthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0005176, eyelids fail to open</dd><dd>HP:0001572, Macrodontia - MP:0030048, sloping forehead</dd></dl><dl><dt>Proximity score 0.523 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CENPJ%0DWDR62&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to WDR62</a> and phenotypic similarity 0.530 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2512">Autosomal recessive primary microcephaly</a> associated with WDR62.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0010864, Intellectual disability, severe</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000582, Upslanted palpebral fissure</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.523 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CENPJ%0DWDR62&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to WDR62</a> and phenotypic similarity 0.622 to mouse mutant of WDR62.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0004201, fetal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0000433, microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/608393">OMIM:608393</a> Microcephaly 6, primary, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/613676">OMIM:613676</a> ?Seckel syndrome 4 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2512">ORPHA:2512</a> Autosomal recessive primary microcephaly</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=808">ORPHA:808</a> Seckel syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.087</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.343</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.803</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr13:g.24906781T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=574582015">rs574582015</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs574582015&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.803</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381884.9">CENPJ:ENST00000381884.9:c.1257A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545981.5">CENPJ:ENST00000545981.5:c.1257A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616936.4">CENPJ:ENST00000616936.4:c.1257A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.804746</dd>
                                    
                                    <dd>CADD: 0.805 (7.094)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000140386">SCAPER</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.086</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.177</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.990</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.353 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">Retinitis pigmentosa</a> associated with SCAPER.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000602, Ophthalmoplegia</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.243 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SCAPER">SCAPER</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0011967, increased or absent threshold for auditory brainstem response</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618195">OMIM:618195</a> Intellectual developmental disorder and retinitis pigmentosa - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">ORPHA:791</a> Retinitis pigmentosa</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.086</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.177</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.990</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr15:g.76434286T&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=561831300">rs561831300</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs561831300&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324767.11">SCAPER:ENST00000324767.11:c.3103A&gt;C:p.(Thr1035Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538941.6">SCAPER:ENST00000538941.6:c.2365A&gt;C:p.(Thr789Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000563290.6">SCAPER:ENST00000563290.6:c.3103A&gt;C:p.(Thr1035Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9929205</dd>
                                    
                                    <dd>SIFT: 0.176 (T)</dd>
                                    <dd>CADD: 0.993 (21.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0105%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000174917">MICOS13</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.086</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.187</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.978</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.374 to <a href="http://www.omim.org/618329">Combined oxidative phosphorylation deficiency 37</a> associated with MICOS13.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001508, Failure to thrive</dd><dd>HP:0000252, Microcephaly - HP:0005484, Postnatal microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618329">OMIM:618329</a> Combined oxidative phosphorylation deficiency 37 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=67047">ORPHA:67047</a> 3-methylglutaconic aciduria type 3</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.086</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.187</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.978</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr19:g.5679399G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=371995866">rs371995866</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs371995866&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.978</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309324.8">MICOS13:ENST00000309324.8:c.208-3C&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000587589.1">MICOS13:ENST00000587589.1:c.208-3C&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000587950.5">MICOS13:ENST00000587950.5:c.274-3C&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000590389.5">MICOS13:ENST00000590389.5:c.274-3C&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579649.5">MICOS13:ENST00000579649.5:c.-60+4427G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9898141</dd>
                                    
                                    <dd>CADD: 0.990 (19.920)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0151%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0824%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_AFR: 0.0660%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000155085">AK9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.077</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.169</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.985</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.339 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98913">Postsynaptic congenital myasthenic syndromes</a> associated with AK9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98913">ORPHA:98913</a> Postsynaptic congenital myasthenic syndromes</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.077</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.169</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.985</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr6:g.109514298C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=372572064">rs372572064</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs372572064&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.985</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424296.6">AK9:ENST00000424296.6:c.4205G&gt;A:p.(Arg1402His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9971816</dd>
                                    
                                    <dd>Polyphen2: 0.791 (P)</dd>
                                    <dd>SIFT: 0.123 (T)</dd>
                                    <dd>CADD: 0.997 (25.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0096%</dd>
                                    <dd>ESP AA: 0.0723%</dd>
                                    <dd>ESP All: 0.0219%</dd>
                                    <dd>ExAC AFR: 0.0892%</dd>
                                    <dd>gnomAD_E_AFR: 0.0255%</dd>
                                    <dd>gnomAD_E_AMR: 0.0125%</dd>
                                    <dd>gnomAD_E_NFE: 0.0051%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100722">ZC3H14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.075</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.238</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.905</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.475 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=88616">Autosomal recessive non-syndromic intellectual disability</a> associated with ZC3H14.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001256, Intellectual disability, mild</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.338 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ZC3H14">ZC3H14</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0008428, abnormal spatial working memory</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0008535, enlarged lateral ventricles</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617125">OMIM:617125</a> Mental retardation, autosomal recessive 56 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=88616">ORPHA:88616</a> Autosomal recessive non-syndromic intellectual disability</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.075</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.238</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.905</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.88572079G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=553660137">rs553660137</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs553660137&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.905</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251038.10">ZC3H14:ENST00000251038.10:c.285G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302216.12">ZC3H14:ENST00000302216.12:c.285G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336693.8">ZC3H14:ENST00000336693.8:c.183G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393514.9">ZC3H14:ENST00000393514.9:c.285G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555755.5">ZC3H14:ENST00000555755.5:c.285G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556945.5">ZC3H14:ENST00000556945.5:c.285G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557607.5">ZC3H14:ENST00000557607.5:c.-181G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.90984285</dd>
                                    
                                    <dd>CADD: 0.910 (10.450)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000142655">PEX14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.069</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.315</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.807</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.630 to <a href="http://www.omim.org/614887">Peroxisome biogenesis disorder 13A (Zellweger)</a> associated with PEX14.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000268, Dolichocephaly</dd><dd>HP:0000325, Triangular face - HP:0000325, Triangular face</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Proximity score 0.512 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PEX14%0DPEX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PEX1</a> and phenotypic similarity 0.604 to <a href="http://www.omim.org/214100">Peroxisome biogenesis disorder 1A (Zellweger)</a> associated with PEX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0006887, Intellectual disability, progressive</dd><dd>HP:0004322, Short stature - HP:0001508, Failure to thrive</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0011039, Abnormality of the helix</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000582, Upslanted palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Proximity score 0.512 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PEX14%0DPEX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PEX1</a> and phenotypic similarity 0.426 to mouse mutant of PEX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001732, postnatal growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0004021, abnormal rod electrophysiology</dd><dd>HP:0000637, Long palpebral fissure - MP:0004021, abnormal rod electrophysiology</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614887">OMIM:614887</a> Peroxisome biogenesis disorder 13A (Zellweger) - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=44">ORPHA:44</a> Neonatal adrenoleukodystrophy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=772">ORPHA:772</a> Infantile Refsum disease</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=912">ORPHA:912</a> Zellweger syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.069</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.315</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.807</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.10629798C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=560264914">rs560264914</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs560264914&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=929935%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.807</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356607.9">PEX14:ENST00000356607.9:c.945C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.80949783</dd>
                                    
                                    <dd>CADD: 0.809 (7.201)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ExAC AFR: 0.0181%</dd>
                                    <dd>gnomAD_E_AFR: 0.0162%</dd>
                                    <dd>gnomAD_E_NFE: 0.0021%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198838">RYR3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.060</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.580</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.332 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RYR3">RYR3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001463, abnormal spatial learning</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RYR3%0DCACNA1S&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CACNA1S</a> and phenotypic similarity 0.235 to <a href="http://www.omim.org/188580">Thyrotoxic periodic paralysis, susceptibility to, 1</a> associated with CACNA1S.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001824, Weight loss</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RYR3%0DCACNA1S&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CACNA1S</a> and phenotypic similarity 0.717 to mouse mutant of CACNA1S.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0004343, small scapula</dd><dd>HP:0000252, Microcephaly - MP:0000435, shortened head</dd><dd>HP:0000325, Triangular face - MP:0000762, abnormal tongue morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000458, abnormal mandible morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.060</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.580</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr15:g.33644456T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=374218298">rs374218298</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs374218298&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.580</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389232.9">RYR3:ENST00000389232.9:c.3702T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000415757.7">RYR3:ENST00000415757.7:c.3702T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622037.1">RYR3:ENST00000622037.1:c.3702T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000634418.1">RYR3:ENST00000634418.1:c.3702T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000634891.1">RYR3:ENST00000634891.1:c.3702T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.58590496</dd>
                                    
                                    <dd>CADD: 0.586 (3.829)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0120%</dd>
                                    <dd>ESP AA: 0.0255%</dd>
                                    <dd>ESP All: 0.0082%</dd>
                                    <dd>ExAC AFR: 0.0317%</dd>
                                    <dd>ExAC AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_AFR: 0.0720%</dd>
                                    <dd>gnomAD_E_AMR: 0.0209%</dd>
                                    <dd>gnomAD_G_AFR: 0.0688%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000244482">LILRA6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.057</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.576</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.245 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LILRA6">LILRA6</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000805, abnormal visual cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LILRA6%0DBCR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BCR</a> and phenotypic similarity 0.679 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=261330">Distal 22q11.2 microdeletion syndrome</a> associated with BCR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000363, Abnormality of earlobe</dd><dd>HP:0000316, Hypertelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000637, Long palpebral fissure - HP:0000581, Blepharophimosis</dd><dd>HP:0001572, Macrodontia - HP:0000307, Pointed chin</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.057</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.576</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54240307A&gt;G [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=76090367">rs76090367</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs76090367&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.576</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245621.6">LILRA6:ENST00000245621.6:c.1225T&gt;C:p.(Phe409Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396365.6">LILRA6:ENST00000396365.6:c.1225T&gt;C:p.(Phe409Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.86113274</dd>
                                    
                                    <dd>SIFT: 0.292 (T)</dd>
                                    <dd>CADD: 0.861 (8.574)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC AFR: 0.8850%</dd>
                                    <dd>ExAC AMR: 0.1472%</dd>
                                    <dd>ExAC EAS: 0.0462%</dd>
                                    <dd>ExAC FIN: 0.2272%</dd>
                                    <dd>ExAC NFE: 0.2917%</dd>
                                    <dd>ExAC OTH: 0.1114%</dd>
                                    <dd>ExAC SAS: 0.9340%</dd>
                                    <dd>gnomAD_E_AFR: 0.6951%</dd>
                                    <dd>gnomAD_E_AMR: 0.2229%</dd>
                                    <dd>gnomAD_E_ASJ: 0.2153%</dd>
                                    <dd>gnomAD_E_EAS: 0.0291%</dd>
                                    <dd>gnomAD_E_FIN: 0.0688%</dd>
                                    <dd>gnomAD_E_NFE: 0.2065%</dd>
                                    <dd>gnomAD_E_OTH: 0.4403%</dd>
                                    <dd>gnomAD_E_SAS: 0.5318%</dd>
                                    <dd>gnomAD_G_AFR: 1.2383%</dd>
                                    <dd>gnomAD_G_AMR: 0.1238%</dd>
                                    <dd>gnomAD_G_FIN: 0.2179%</dd>
                                    <dd>gnomAD_G_NFE: 0.4992%</dd>
                                    <dd>gnomAD_G_OTH: 0.8949%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.54240306A&gt;T [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3193485">rs3193485</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs3193485&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.441</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245621.6">LILRA6:ENST00000245621.6:c.1226T&gt;A:p.(Phe409Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396365.6">LILRA6:ENST00000396365.6:c.1226T&gt;A:p.(Phe409Tyr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.799</dd>
                        
                        <dd>SIFT: 0.201 (T)</dd>
                        <dd>CADD: 0.041 (0.180)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>ExAC AFR: 0.9634%</dd>
                        <dd>ExAC AMR: 0.1385%</dd>
                        <dd>ExAC EAS: 0.0231%</dd>
                        <dd>ExAC FIN: 0.2270%</dd>
                        <dd>ExAC NFE: 0.2855%</dd>
                        <dd>ExAC OTH: 0.1109%</dd>
                        <dd>ExAC SAS: 0.9213%</dd>
                        <dd>gnomAD_E_AFR: 0.7290%</dd>
                        <dd>gnomAD_E_AMR: 0.2258%</dd>
                        <dd>gnomAD_E_ASJ: 0.2258%</dd>
                        <dd>gnomAD_E_EAS: 0.0174%</dd>
                        <dd>gnomAD_E_FIN: 0.0687%</dd>
                        <dd>gnomAD_E_NFE: 0.2063%</dd>
                        <dd>gnomAD_E_OTH: 0.4399%</dd>
                        <dd>gnomAD_E_SAS: 0.5231%</dd>
                        <dd>gnomAD_G_AFR: 1.4618%</dd>
                        <dd>gnomAD_G_AMR: 0.1241%</dd>
                        <dd>gnomAD_G_FIN: 0.2171%</dd>
                        <dd>gnomAD_G_NFE: 0.5058%</dd>
                        <dd>gnomAD_G_OTH: 0.8929%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000173258">ZNF483</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.047</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.552</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF483%0DNSD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NSD1</a> and phenotypic similarity 0.719 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3447">Weaver syndrome</a> associated with NSD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0000098, Tall stature</dd><dd>HP:0000252, Microcephaly - HP:0000256, Macrocephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000494, Downslanted palpebral fissures</dd><dd>HP:0001572, Macrodontia - HP:0000278, Retrognathia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.047</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.552</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.111527847C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=546868484">rs546868484</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs546868484&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.552</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374374.3">ZNF483:ENST00000374374.3:c.452C&gt;T:p.(Ala151Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309235.6">ZNF483:ENST00000309235.6:c.412+40C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355824.7">ZNF483:ENST00000355824.7:c.412+40C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358151.8">ZNF483:ENST00000358151.8:c.412+40C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.554</dd>
                                    
                                    <dd>Polyphen2: 0.004 (B)</dd>
                                    <dd>SIFT: 0.446 (T)</dd>
                                    <dd>CADD: 0.364 (1.967)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0103%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183117">CSMD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.042</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.541</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSMD1%0DTAPT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TAPT1</a> and phenotypic similarity 0.690 to <a href="http://www.omim.org/616897">Osteochondrodysplasia, complex lethal, Symoens-Barnes-Gistelinck type</a> associated with TAPT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001518, Small for gestational age</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0002265, Large fleshy ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000506, Telecanthus</dd><dd>HP:0001572, Macrodontia - HP:0000347, Micrognathia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSMD1%0DTAPT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TAPT1</a> and phenotypic similarity 0.322 to mouse mutant of TAPT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000111, cleft palate</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.042</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.541</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr8:g.3348079C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=569558476">rs569558476</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs569558476&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.541</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400186.7">CSMD1:ENST00000400186.7:c.3390G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000520002.5">CSMD1:ENST00000520002.5:c.3390G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537824.2">CSMD1:ENST00000537824.2:c.2973G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602557.5">CSMD1:ENST00000602557.5:c.3390G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602723.5">CSMD1:ENST00000602723.5:c.3390G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000635120.2">CSMD1:ENST00000635120.2:c.3387G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5439631</dd>
                                    
                                    <dd>CADD: 0.544 (3.410)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ExAC AFR: 0.0339%</dd>
                                    <dd>ExAC EAS: 0.0133%</dd>
                                    <dd>ExAC NFE: 0.0069%</dd>
                                    <dd>gnomAD_E_AFR: 0.0329%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154118">JPH3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.037</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.280</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.776</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.280 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98934">Huntington disease-like 2</a> associated with JPH3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001824, Weight loss</dd><dd>HP:0000252, Microcephaly - HP:0002120, Cerebral cortical atrophy</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Phenotypic similarity 0.252 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=JPH3">JPH3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001405, impaired coordination</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/606438">OMIM:606438</a> Huntington disease-like 2 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98934">ORPHA:98934</a> Huntington disease-like 2</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.037</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.280</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.776</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr16:g.87644703C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=571536520">rs571536520</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs571536520&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.776</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000284262.3">JPH3:ENST00000284262.3:c.828C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.77980876</dd>
                                    
                                    <dd>CADD: 0.780 (6.572)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ExAC AFR: 0.0390%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC EAS: 0.0117%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>ExAC SAS: 0.0303%</dd>
                                    <dd>gnomAD_E_AFR: 0.0328%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_SAS: 0.0325%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000225805">DEFB131B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.034</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.243</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.808</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.243 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DEFB131B">DEFB131B</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - MP:0011967, increased or absent threshold for auditory brainstem response</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.034</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.243</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.808</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.71884554A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=558100327">rs558100327</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs558100327&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.808</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530210.1">DEFB131B:ENST00000530210.1:c.206A&gt;G:p.(Lys69Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000528511.6">DEFB131B:ENST00000528511.6:c.*61T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.81280255</dd>
                                    
                                    <dd>CADD: 0.813 (7.277)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0088%</dd>
                                    <dd>ExAC AFR: 0.0152%</dd>
                                    <dd>gnomAD_E_AFR: 0.0070%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000142619">PADI3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/191480">OMIM:191480</a> Uncombable hair syndrome - autosomal dominant/recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1410">ORPHA:1410</a> Uncombable hair syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.17280779G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=34097903">rs34097903</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs34097903&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-danger">PATHOGENIC</span>
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=590375%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                <span class="label label-danger">WHITELIST VARIANT</span>
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375460.3">PADI3:ENST00000375460.3:c.1744G&gt;A:p.(Ala582Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99988</dd>
                                    
                                    <dd>Polyphen2: 0.545 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.067 (T)</dd>
                                    <dd>CADD: 0.999 (28.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.8586%</dd>
                                    <dd>TOPMed: 0.6339%</dd>
                                    <dd>ESP AA: 1.9973%</dd>
                                    <dd>ESP All: 0.6766%</dd>
                                    <dd>ExAC AFR: 2.2876%</dd>
                                    <dd>ExAC AMR: 0.0345%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>gnomAD_E_AFR: 2.2086%</dd>
                                    <dd>gnomAD_E_AMR: 0.0864%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_G_AFR: 2.4295%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197870">PRB3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_VARIANT</span>
                                <b>chr12:g.11267400A&gt;AG [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3842295">rs3842295</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs3842295&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381842.7">PRB3:ENST00000381842.7:c.722dup:p.(His242Serfs*14)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538488.2">PRB3:ENST00000538488.2:c.848dup:p.(His284Serfs*14)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9930817</dd>
                                    
                                    <dd>CADD: 0.993 (21.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000155307">SAMSN1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_TRUNCATION</span>
                                <b>chr21:g.14583737TC&gt;T [0/1:0/1:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000647101.1">SAMSN1:ENST00000647101.1:c.504del:p.(Gly168Glyfs*3)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000274279">SPATA31E3P</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_ACCEPTOR_VARIANT</span>
                                <b>chr15:g.22402131G&gt;A [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4966360">rs4966360</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs4966360&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613133.1">SPATA31E3P:ENST00000613133.1:n.201-2G&gt;A:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96548563</dd>
                                    
                                    <dd>CADD: 0.965 (14.620)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000011105">TSPAN9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.3278443G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=545632699">rs545632699</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs545632699&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000011898.10">TSPAN9:ENST00000011898.10:c.86G&gt;A:p.(Gly29Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000407263.2">TSPAN9:ENST00000407263.2:c.86G&gt;A:p.(Gly29Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537971.5">TSPAN9:ENST00000537971.5:c.86G&gt;A:p.(Gly29Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 1.000 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.999 (29.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000124574">ABCC10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr6:g.43433047G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=543789317">rs543789317</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs543789317&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000244533.7">ABCC10:ENST00000244533.7:c.938G&gt;A:p.(Arg313Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372530.9">ABCC10:ENST00000372530.9:c.1067G&gt;A:p.(Arg356Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999996</dd>
                                    
                                    <dd>Polyphen2: 0.992 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.004 (D)</dd>
                                    <dd>CADD: 0.995 (22.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000227124">ZNF717</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.75738211C&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=111315075">rs111315075</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs111315075&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000478296.5">ZNF717:ENST00000478296.5:c.1262G&gt;T:p.(Arg421Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000652011.2">ZNF717:ENST00000652011.2:c.1412G&gt;T:p.(Arg471Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000477374.5">ZNF717:ENST00000477374.5:c.277+3065G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.030 (D)</dd>
                                    <dd>CADD: 0.990 (20.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0238%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.75737791C&gt;A [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2918517">rs2918517</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2918517&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000478296.5">ZNF717:ENST00000478296.5:c.1682G&gt;T:p.(Arg561Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000652011.2">ZNF717:ENST00000652011.2:c.1832G&gt;T:p.(Arg611Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000477374.5">ZNF717:ENST00000477374.5:c.277+3485G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999999</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.084 (T)</dd>
                                    <dd>CADD: 0.879 (9.180)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0399%</dd>
                                    <dd>gnomAD_E_AFR: 0.0152%</dd>
                                    <dd>gnomAD_E_EAS: 0.0111%</dd>
                                    <dd>gnomAD_E_OTH: 0.0275%</dd>
                                    <dd>gnomAD_G_AFR: 0.0132%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr3:g.75737580T&gt;C [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=17028812">rs17028812</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs17028812&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.922</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000478296.5">ZNF717:ENST00000478296.5:c.1893A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000652011.2">ZNF717:ENST00000652011.2:c.2043A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000477374.5">ZNF717:ENST00000477374.5:c.277+3696A&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.92273194</dd>
                        
                        <dd>CADD: 0.923 (11.120)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_E_AMR: 0.0047%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr3:g.75739007G&gt;C [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3009004">rs3009004</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs3009004&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.849</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000478296.5">ZNF717:ENST00000478296.5:c.466C&gt;G:p.(Leu156Val)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000652011.2">ZNF717:ENST00000652011.2:c.616C&gt;G:p.(Leu206Val)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000477374.5">ZNF717:ENST00000477374.5:c.277+2269C&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8492699</dd>
                        
                        <dd>SIFT: 0.259 (T)</dd>
                        <dd>CADD: 0.849 (8.218)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr3:g.75737477G&gt;C [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3009023">rs3009023</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs3009023&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.836</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000478296.5">ZNF717:ENST00000478296.5:c.1996C&gt;G:p.(Gln666Glu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000652011.2">ZNF717:ENST00000652011.2:c.2146C&gt;G:p.(Gln716Glu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000477374.5">ZNF717:ENST00000477374.5:c.277+3799C&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.84263796</dd>
                        
                        <dd>SIFT: 0.791 (T)</dd>
                        <dd>CADD: 0.843 (8.031)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0599%</dd>
                        <dd>TOPMed: 0.0599%</dd>
                        <dd>gnomAD_E_AMR: 0.0045%</dd>
                        <dd>gnomAD_E_NFE: 0.0018%</dd>
                        <dd>gnomAD_E_SAS: 0.0057%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000275896">PRSS2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/167800">OMIM:167800</a> Pancreatitis, chronic, protection against (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=676">ORPHA:676</a> Hereditary chronic pancreatitis</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.142773976G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=930218821">rs930218821</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs930218821&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539842.5">PRSS2:ENST00000539842.5:c.512G&gt;A:p.(Cys171Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000632805.1">PRSS2:ENST00000632805.1:c.509G&gt;A:p.(Cys170Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000632998.1">PRSS2:ENST00000632998.1:c.512G&gt;A:p.(Cys171Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000633969.1">PRSS2:ENST00000633969.1:c.554G&gt;A:p.(Cys185Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99645185</dd>
                                    
                                    <dd>CADD: 0.996 (24.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr7:g.142772158C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=368679390">rs368679390</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs368679390&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.464</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539842.5">PRSS2:ENST00000539842.5:c.150C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000632805.1">PRSS2:ENST00000632805.1:c.150C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000632998.1">PRSS2:ENST00000632998.1:c.150C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000633969.1">PRSS2:ENST00000633969.1:c.150C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.46370965</dd>
                        
                        <dd>CADD: 0.464 (2.706)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000236446">CT47B1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.120875270C&gt;T [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=746190277">rs746190277</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs746190277&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371311.4">CT47B1:ENST00000371311.4:c.401G&gt;A:p.(Arg134His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.984 (D)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.993 (21.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0265%</dd>
                                    <dd>TOPMed: 0.0056%</dd>
                                    <dd>ExAC AFR: 0.0256%</dd>
                                    <dd>gnomAD_E_AFR: 0.0082%</dd>
                                    <dd>gnomAD_E_EAS: 0.0078%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000000003">TSPAN6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.100630826A&gt;G [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=778356735">rs778356735</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs778356735&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373020.9">TSPAN6:ENST00000373020.9:c.710T&gt;C:p.(Ile237Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612152.4">TSPAN6:ENST00000612152.4:c.362T&gt;C:p.(Ile121Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.999 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.998 (26.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0265%</dd>
                                    <dd>TOPMed: 0.0265%</dd>
                                    <dd>ExAC AFR: 0.0125%</dd>
                                    <dd>gnomAD_E_AFR: 0.0082%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000224051">CPTP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.1327615C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=370811590">rs370811590</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs370811590&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343938.9">CPTP:ENST00000343938.9:c.497C&gt;T:p.(Pro166Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 1.000 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.997 (24.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0143%</dd>
                                    <dd>ESP AA: 0.0228%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0225%</dd>
                                    <dd>ExAC AMR: 0.0278%</dd>
                                    <dd>gnomAD_E_AFR: 0.0134%</dd>
                                    <dd>gnomAD_E_AMR: 0.0090%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162398">LEXM</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.54807991C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=141936308">rs141936308</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs141936308&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358193.7">LEXM:ENST00000358193.7:c.460C&gt;T:p.(Arg154Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371273.3">LEXM:ENST00000371273.3:c.460C&gt;T:p.(Arg154Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.998 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.996 (23.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0297%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>ExAC SAS: 0.0076%</dd>
                                    <dd>gnomAD_E_AFR: 0.0207%</dd>
                                    <dd>gnomAD_E_AMR: 0.0188%</dd>
                                    <dd>gnomAD_E_NFE: 0.0019%</dd>
                                    <dd>gnomAD_E_SAS: 0.0077%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162039">MEIOB</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.995</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617706">OMIM:617706</a> ?Spermatogenic failure 22 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=399805">ORPHA:399805</a> Male infertility with azoospermia or oligozoospermia due to single gene mutation</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.995</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.1844874C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=575464897">rs575464897</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs575464897&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.995</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000325962.7">MEIOB:ENST00000325962.7:c.868G&gt;C:p.(Glu290Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397344.7">MEIOB:ENST00000397344.7:c.868G&gt;C:p.(Glu290Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000470044.5">MEIOB:ENST00000470044.5:c.247G&gt;C:p.(Glu83Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.998036</dd>
                                    
                                    <dd>Polyphen2: 0.255 (B)</dd>
                                    <dd>Mutation Taster: 0.998 (P)</dd>
                                    <dd>SIFT: 0.082 (T)</dd>
                                    <dd>CADD: 0.996 (24.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0107%</dd>
                                    <dd>ExAC AMR: 0.0182%</dd>
                                    <dd>gnomAD_E_AFR: 0.0067%</dd>
                                    <dd>gnomAD_E_AMR: 0.0099%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204572">KRTAP5-10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.71565817G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=528994893">rs528994893</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs528994893&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398531.3">KRTAP5-10:ENST00000398531.3:c.230G&gt;A:p.(Cys77Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.008 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.979 (16.820)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0599%</dd>
                                    <dd>ExAC AFR: 0.0222%</dd>
                                    <dd>gnomAD_E_AFR: 0.0135%</dd>
                                    <dd>gnomAD_G_AFR: 0.0128%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162460">TMEM82</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.15744398G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=527833067">rs527833067</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs527833067&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375782.2">TMEM82:ENST00000375782.2:c.575G&gt;A:p.(Gly192Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9942456</dd>
                                    
                                    <dd>Polyphen2: 0.590 (P)</dd>
                                    <dd>SIFT: 0.035 (D)</dd>
                                    <dd>CADD: 0.994 (22.400)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_E_SAS: 0.0038%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000244219">TMEM225B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.990</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.990</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">START_LOST</span>
                                <b>chr7:g.99604390T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=566399754">rs566399754</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs566399754&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431679.6">TMEM225B:ENST00000431679.6:c.2T&gt;C:p.0?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453227.5">TMEM225B:ENST00000453227.5:c.2T&gt;C:p.0?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000455905.1">TMEM225B:ENST00000455905.1:c.219T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98584205</dd>
                                    
                                    <dd>CADD: 0.986 (18.490)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0135%</dd>
                                    <dd>gnomAD_E_AFR: 0.0162%</dd>
                                    <dd>gnomAD_E_AMR: 0.0126%</dd>
                                    <dd>gnomAD_G_AFR: 0.0688%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178104">PDE4DIP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.990</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.990</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.149026790C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=111420757">rs111420757</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs111420757&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369354.7">PDE4DIP:ENST00000369354.7:c.6401C&gt;G:p.(Ser2134Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369356.8">PDE4DIP:ENST00000369356.8:c.6401C&gt;G:p.(Ser2134Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524974.5">PDE4DIP:ENST00000524974.5:c.6656C&gt;G:p.(Ser2219Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000585156.5">PDE4DIP:ENST00000585156.5:c.6809C&gt;G:p.(Ser2270Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618462.4">PDE4DIP:ENST00000618462.4:c.6083C&gt;G:p.(Ser2028Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98967236</dd>
                                    
                                    <dd>SIFT: 0.025 (D)</dd>
                                    <dd>CADD: 0.990 (19.860)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.982</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.149029906G&gt;A [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3851872">rs3851872</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs3851872&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.982</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369354.7">PDE4DIP:ENST00000369354.7:c.6933G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369356.8">PDE4DIP:ENST00000369356.8:c.6933G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524974.5">PDE4DIP:ENST00000524974.5:c.7188G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000585156.5">PDE4DIP:ENST00000585156.5:c.7341G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618462.4">PDE4DIP:ENST00000618462.4:c.6615G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9820527</dd>
                                    
                                    <dd>CADD: 0.982 (17.460)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr1:g.149029915C&gt;T [1/1:1/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=833192">rs833192</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs833192&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.953</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369354.7">PDE4DIP:ENST00000369354.7:c.6942C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369356.8">PDE4DIP:ENST00000369356.8:c.6942C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524974.5">PDE4DIP:ENST00000524974.5:c.7197C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000585156.5">PDE4DIP:ENST00000585156.5:c.7350C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618462.4">PDE4DIP:ENST00000618462.4:c.6624C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.95301056</dd>
                        
                        <dd>CADD: 0.953 (13.280)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170871">KIAA0232</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.990</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.990</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.6862102G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=368241178">rs368241178</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs368241178&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000307659.6">KIAA0232:ENST00000307659.6:c.1720G&gt;A:p.(Val574Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000425103.5">KIAA0232:ENST00000425103.5:c.1720G&gt;A:p.(Val574Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9930817</dd>
                                    
                                    <dd>Polyphen2: 0.057 (B)</dd>
                                    <dd>Mutation Taster: 0.905</dd>
                                    <dd>SIFT: 0.161 (T)</dd>
                                    <dd>CADD: 0.993 (21.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>ESP AA: 0.0251%</dd>
                                    <dd>ESP All: 0.0081%</dd>
                                    <dd>ExAC AFR: 0.0204%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000101220">C20orf27</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.990</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.990</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr20:g.3754495G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=560977252">rs560977252</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs560977252&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000217195.12">C20orf27:ENST00000217195.12:c.401C&gt;T:p.(Ser134Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379772.3">C20orf27:ENST00000379772.3:c.326C&gt;T:p.(Ser109Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9922375</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.656 (T)</dd>
                                    <dd>CADD: 0.992 (21.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000158486">DNAH3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.20963874G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=530510705">rs530510705</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs530510705&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261383.3">DNAH3:ENST00000261383.3:c.10010C&gt;T:p.(Thr3337Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.992</dd>
                                    
                                    <dd>Polyphen2: 0.247 (B)</dd>
                                    <dd>SIFT: 0.008 (D)</dd>
                                    <dd>CADD: 0.958 (13.740)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000080200">CRYBG3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.97875483A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=547892297">rs547892297</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs547892297&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389622.7">CRYBG3:ENST00000389622.7:c.4289A&gt;G:p.(Asp1430Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99148864</dd>
                                    
                                    <dd>CADD: 0.991 (20.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000276975">HYDIN2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.988</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.988</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_ACCEPTOR_VARIANT</span>
                                <b>chr1:g.146819489G&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1305599324">rs1305599324</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1305599324&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.988</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614586.3">HYDIN2:ENST00000614586.3:n.8546-1G&gt;C:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99553317</dd>
                                    
                                    <dd>CADD: 0.996 (23.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0836%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000041515">MYO16</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.986</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.986</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr13:g.109141025C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=527776649">rs527776649</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs527776649&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.986</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356711.7">MYO16:ENST00000356711.7:c.4747C&gt;T:p.(Pro1583Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457511.7">MYO16:ENST00000457511.7:c.4813C&gt;T:p.(Pro1605Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9948714</dd>
                                    
                                    <dd>Polyphen2: 0.039 (B)</dd>
                                    <dd>SIFT: 0.033 (D)</dd>
                                    <dd>CADD: 0.995 (22.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>gnomAD_G_EAS: 0.0618%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000268043">NBPF12</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.986</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.986</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.146975709G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1163302609">rs1163302609</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1163302609&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.986</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611443.4">NBPF12:ENST00000611443.4:c.1937G&gt;A:p.(Arg646Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613714.2">NBPF12:ENST00000613714.2:c.1124G&gt;A:p.(Arg375Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617614.4">NBPF12:ENST00000617614.4:c.1124G&gt;A:p.(Arg375Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617844.4">NBPF12:ENST00000617844.4:c.1937G&gt;A:p.(Arg646Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617931.4">NBPF12:ENST00000617931.4:c.1937G&gt;A:p.(Arg646Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439206.6">NBPF12:ENST00000439206.6:c.567-1757G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579935.8">NBPF12:ENST00000579935.8:c.1307-1220G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617864.3">NBPF12:ENST00000617864.3:c.494-1220G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98574394</dd>
                                    
                                    <dd>CADD: 0.986 (18.460)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.146984164A&gt;G [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1433305376">rs1433305376</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1433305376&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.969</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439206.6">NBPF12:ENST00000439206.6:c.1019A&gt;G:p.(Asp340Gly)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579935.8">NBPF12:ENST00000579935.8:c.1832A&gt;G:p.(Asp611Gly)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611443.4">NBPF12:ENST00000611443.4:c.2645A&gt;G:p.(Asp882Gly)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613714.2">NBPF12:ENST00000613714.2:c.1832A&gt;G:p.(Asp611Gly)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617614.4">NBPF12:ENST00000617614.4:c.1832A&gt;G:p.(Asp611Gly)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617844.4">NBPF12:ENST00000617844.4:c.2645A&gt;G:p.(Asp882Gly)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617931.4">NBPF12:ENST00000617931.4:c.2645A&gt;G:p.(Asp882Gly)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617864.3">NBPF12:ENST00000617864.3:c.855A&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9782229</dd>
                        
                        <dd>CADD: 0.978 (16.620)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_E_AFR: 0.0694%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.146977540G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1301699973">rs1301699973</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1301699973&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.960</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439206.6">NBPF12:ENST00000439206.6:c.641G&gt;A:p.(Ser214Asn)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579935.8">NBPF12:ENST00000579935.8:c.1454G&gt;A:p.(Ser485Asn)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611443.4">NBPF12:ENST00000611443.4:c.2267G&gt;A:p.(Ser756Asn)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613714.2">NBPF12:ENST00000613714.2:c.1454G&gt;A:p.(Ser485Asn)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617614.4">NBPF12:ENST00000617614.4:c.1454G&gt;A:p.(Ser485Asn)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617844.4">NBPF12:ENST00000617844.4:c.2267G&gt;A:p.(Ser756Asn)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617864.3">NBPF12:ENST00000617864.3:c.641G&gt;A:p.(Ser214Asn)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617931.4">NBPF12:ENST00000617931.4:c.2267G&gt;A:p.(Ser756Asn)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.96293193</dd>
                        
                        <dd>CADD: 0.963 (14.310)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>TOPMed: 0.0215%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.146977638G&gt;C [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1403260761">rs1403260761</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1403260761&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.959</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439206.6">NBPF12:ENST00000439206.6:c.739G&gt;C:p.(Val247Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579935.8">NBPF12:ENST00000579935.8:c.1552G&gt;C:p.(Val518Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611443.4">NBPF12:ENST00000611443.4:c.2365G&gt;C:p.(Val789Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613714.2">NBPF12:ENST00000613714.2:c.1552G&gt;C:p.(Val518Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617614.4">NBPF12:ENST00000617614.4:c.1552G&gt;C:p.(Val518Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617844.4">NBPF12:ENST00000617844.4:c.2365G&gt;C:p.(Val789Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617864.3">NBPF12:ENST00000617864.3:c.739G&gt;C:p.(Val247Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617931.4">NBPF12:ENST00000617931.4:c.2365G&gt;C:p.(Val789Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9591681</dd>
                        
                        <dd>CADD: 0.959 (13.890)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.146972801G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=587667641">rs587667641</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs587667641&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.947</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611443.4">NBPF12:ENST00000611443.4:c.1642G&gt;A:p.(Gly548Ser)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613714.2">NBPF12:ENST00000613714.2:c.829G&gt;A:p.(Gly277Ser)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617614.4">NBPF12:ENST00000617614.4:c.829G&gt;A:p.(Gly277Ser)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617844.4">NBPF12:ENST00000617844.4:c.1642G&gt;A:p.(Gly548Ser)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617931.4">NBPF12:ENST00000617931.4:c.1642G&gt;A:p.(Gly548Ser)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439206.6">NBPF12:ENST00000439206.6:c.566+2082G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579935.8">NBPF12:ENST00000579935.8:c.1306+3205G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617864.3">NBPF12:ENST00000617864.3:c.493+3205G&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9597283</dd>
                        
                        <dd>CADD: 0.960 (13.950)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>TOPMed: 0.0295%</dd>
                        <dd>gnomAD_E_AFR: 0.0959%</dd>
                        <dd>gnomAD_E_EAS: 0.0193%</dd>
                        <dd>gnomAD_E_NFE: 0.0019%</dd>
                        <dd>gnomAD_G_AFR: 0.0774%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.146977635C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1320739646">rs1320739646</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1320739646&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.887</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439206.6">NBPF12:ENST00000439206.6:c.736C&gt;T:p.(His246Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579935.8">NBPF12:ENST00000579935.8:c.1549C&gt;T:p.(His517Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611443.4">NBPF12:ENST00000611443.4:c.2362C&gt;T:p.(His788Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613714.2">NBPF12:ENST00000613714.2:c.1549C&gt;T:p.(His517Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617614.4">NBPF12:ENST00000617614.4:c.1549C&gt;T:p.(His517Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617844.4">NBPF12:ENST00000617844.4:c.2362C&gt;T:p.(His788Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617864.3">NBPF12:ENST00000617864.3:c.736C&gt;T:p.(His246Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617931.4">NBPF12:ENST00000617931.4:c.2362C&gt;T:p.(His788Tyr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8872543</dd>
                        
                        <dd>CADD: 0.887 (9.479)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000206043">C18orf63</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.985</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.985</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr18:g.74342282A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=548499134">rs548499134</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs548499134&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.985</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579455.2">C18orf63:ENST00000579455.2:c.750A&gt;G:p.(Ile250Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9959262</dd>
                                    
                                    <dd>CADD: 0.996 (23.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0135%</dd>
                                    <dd>gnomAD_E_AMR: 0.0084%</dd>
                                    <dd>gnomAD_G_AFR: 0.0458%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135835">KIAA1614</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.981</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.981</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.180938567G&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=368897390">rs368897390</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs368897390&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.981</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367587.1">KIAA1614:ENST00000367587.1:c.1637G&gt;C:p.(Gly546Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367588.9">KIAA1614:ENST00000367588.9:c.2774G&gt;C:p.(Gly925Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9925869</dd>
                                    
                                    <dd>Polyphen2: 0.513 (P)</dd>
                                    <dd>SIFT: 0.103 (T)</dd>
                                    <dd>CADD: 0.993 (21.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>ESP AA: 0.0466%</dd>
                                    <dd>ESP All: 0.0156%</dd>
                                    <dd>ExAC AFR: 0.0615%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>gnomAD_E_AFR: 0.0523%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000272031">ANKRD34A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.980</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.980</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.145961559G&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=142103666">rs142103666</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs142103666&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.980</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000606888.3">ANKRD34A:ENST00000606888.3:c.201C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000625258.1">ANKRD34A:ENST00000625258.1:c.-29-15525G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9910875</dd>
                                    
                                    <dd>CADD: 0.991 (20.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0621%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204510">PRAMEF7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.973</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.973</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.12920307T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1832597">rs1832597</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1832597&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.973</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000330881.6">PRAMEF7:ENST00000330881.6:c.1319T&gt;C:p.(Leu440Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616979.4">PRAMEF7:ENST00000616979.4:c.1319T&gt;C:p.(Leu440Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9733314</dd>
                                    
                                    <dd>CADD: 0.973 (15.740)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.005</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.878</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.12920302C&gt;G [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=9328936">rs9328936</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs9328936&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.878</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000330881.6">PRAMEF7:ENST00000330881.6:c.1314C&gt;G:p.(Asn438Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616979.4">PRAMEF7:ENST00000616979.4:c.1314C&gt;G:p.(Asn438Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.878073</dd>
                                    
                                    <dd>CADD: 0.878 (9.139)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000269713">NBPF9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.972</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.972</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.149055882A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201908311">rs201908311</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201908311&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.972</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000584027.8">NBPF9:ENST00000584027.8:c.3110T&gt;C:p.(Met1037Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000610300.4">NBPF9:ENST00000610300.4:c.2603T&gt;C:p.(Met868Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613595.4">NBPF9:ENST00000613595.4:c.2603T&gt;C:p.(Met868Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613969.4">NBPF9:ENST00000613969.4:c.2603T&gt;C:p.(Met868Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615421.4">NBPF9:ENST00000615421.4:c.3110T&gt;C:p.(Met1037Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621074.4">NBPF9:ENST00000621074.4:c.2378T&gt;C:p.(Met793Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621645.4">NBPF9:ENST00000621645.4:c.2720T&gt;C:p.(Met907Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483630.5">NBPF9:ENST00000483630.5:c.*726T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9715554</dd>
                                    
                                    <dd>CADD: 0.972 (15.460)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.149062099A&gt;C [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1227241524">rs1227241524</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1227241524&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.182</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000584027.8">NBPF9:ENST00000584027.8:c.2245T&gt;G:p.(Leu749Val)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000610300.4">NBPF9:ENST00000610300.4:c.2245T&gt;G:p.(Leu749Val)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613595.4">NBPF9:ENST00000613595.4:c.2245T&gt;G:p.(Leu749Val)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615421.4">NBPF9:ENST00000615421.4:c.2245T&gt;G:p.(Leu749Val)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621645.4">NBPF9:ENST00000621645.4:c.1855T&gt;G:p.(Leu619Val)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613969.4">NBPF9:ENST00000613969.4:c.2078+763T&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621074.4">NBPF9:ENST00000621074.4:c.1853+2332T&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483630.5">NBPF9:ENST00000483630.5:c.*201+763T&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.18191206</dd>
                        
                        <dd>CADD: 0.182 (0.872)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183598">H3C13</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.971</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.971</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.149813550G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=374345320">rs374345320</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs374345320&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.971</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000331491.1">H3C13:ENST00000331491.1:c.132C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97905886</dd>
                                    
                                    <dd>CADD: 0.979 (16.790)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0056%</dd>
                                    <dd>ESP AA: 0.0322%</dd>
                                    <dd>ESP All: 0.0098%</dd>
                                    <dd>ExAC AFR: 0.0567%</dd>
                                    <dd>gnomAD_E_AFR: 0.0410%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204176">SYT15</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.969</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.969</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.46584599G&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3127785">rs3127785</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs3127785&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.969</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374321.9">SYT15:ENST00000374321.9:c.927G&gt;C:p.(Glu309Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374323.8">SYT15:ENST00000374323.8:c.1086G&gt;C:p.(Glu362Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000503753.5">SYT15:ENST00000503753.5:c.927G&gt;C:p.(Glu309Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9690258</dd>
                                    
                                    <dd>SIFT: 0.487 (T)</dd>
                                    <dd>CADD: 0.969 (15.090)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000179965">ZNF771</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.966</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.966</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr16:g.30418295G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=571216265">rs571216265</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs571216265&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.966</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000319296.10">ZNF771:ENST00000319296.10:c.882G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000434417.1">ZNF771:ENST00000434417.1:c.882G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000566625.2">ZNF771:ENST00000566625.2:c.141+10101G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97393847</dd>
                                    
                                    <dd>CADD: 0.974 (15.840)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>gnomAD_E_AFR: 0.0487%</dd>
                                    <dd>gnomAD_E_AMR: 0.0051%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000151632">AKR1C2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.965</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614279">OMIM:614279</a> 46XY sex reversal 8 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.965</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.4995342G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1332625400">rs1332625400</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1332625400&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.965</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380753.8">AKR1C2:ENST00000380753.8:c.823C&gt;A:p.(Gln275Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421196.7">AKR1C2:ENST00000421196.7:c.745C&gt;A:p.(Gln249Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96540606</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>CADD: 0.965 (14.610)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000263956">NBPF11</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.965</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.965</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.148107750T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1488492448">rs1488492448</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1488492448&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.965</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.2039A&gt;G:p.(Tyr680Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.1934A&gt;G:p.(Tyr645Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.2039A&gt;G:p.(Tyr680Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.965086</dd>
                                    
                                    <dd>CADD: 0.965 (14.570)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.148123948G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1209593044">rs1209593044</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1209593044&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.774</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.398C&gt;T:p.(Pro133Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.293C&gt;T:p.(Pro98Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.398C&gt;T:p.(Pro133Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.77395236</dd>
                        
                        <dd>CADD: 0.774 (6.458)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143552">NUP210L</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.963</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.963</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.154054312T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=375097363">rs375097363</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs375097363&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.963</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000271854.3">NUP210L:ENST00000271854.3:c.3399A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368553.5">NUP210L:ENST00000368553.5:c.198A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368559.7">NUP210L:ENST00000368559.7:c.3399A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97007734</dd>
                                    
                                    <dd>CADD: 0.970 (15.240)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0088%</dd>
                                    <dd>ESP AA: 0.0255%</dd>
                                    <dd>ESP All: 0.0082%</dd>
                                    <dd>ExAC AFR: 0.0510%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0458%</dd>
                                    <dd>gnomAD_E_AMR: 0.0209%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185842">DNAH14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.963</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.963</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.225258011T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=566569441">rs566569441</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs566569441&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.963</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430092.5">DNAH14:ENST00000430092.5:c.6899T&gt;C:p.(Ile2300Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439375.6">DNAH14:ENST00000439375.6:c.6899T&gt;C:p.(Ile2300Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445597.6">DNAH14:ENST00000445597.6:c.5584+5594T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9740582</dd>
                                    
                                    <dd>Polyphen2: 0.154 (B)</dd>
                                    <dd>SIFT: 0.446 (T)</dd>
                                    <dd>CADD: 0.974 (15.860)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0263%</dd>
                                    <dd>gnomAD_E_AMR: 0.0042%</dd>
                                    <dd>gnomAD_E_OTH: 0.0260%</dd>
                                    <dd>gnomAD_G_AFR: 0.0573%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.225358646A&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=182215041">rs182215041</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs182215041&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.848</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430092.5">DNAH14:ENST00000430092.5:c.11491A&gt;T:p.(Thr3831Ser)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439375.6">DNAH14:ENST00000439375.6:c.11491A&gt;T:p.(Thr3831Ser)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445597.6">DNAH14:ENST00000445597.6:c.9199A&gt;T:p.(Thr3067Ser)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8576033</dd>
                        
                        <dd>Polyphen2: 0.001 (B)</dd>
                        <dd>SIFT: 0.881 (T)</dd>
                        <dd>CADD: 0.858 (8.465)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0799%</dd>
                        <dd>TOPMed: 0.0064%</dd>
                        <dd>gnomAD_E_AFR: 0.0135%</dd>
                        <dd>gnomAD_E_AMR: 0.0089%</dd>
                        <dd>gnomAD_E_OTH: 0.0270%</dd>
                        <dd>gnomAD_G_AFR: 0.0459%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000144026">ZNF514</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.963</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.963</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.95149792C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=145283605">rs145283605</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs145283605&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.963</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295208.7">ZNF514:ENST00000295208.7:c.693G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000411425.1">ZNF514:ENST00000411425.1:c.693G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96993923</dd>
                                    
                                    <dd>CADD: 0.970 (15.220)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0207%</dd>
                                    <dd>ESP EA: 0.0349%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AFR: 0.0193%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0102%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0143%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_E_SAS: 0.0130%</dd>
                                    <dd>gnomAD_G_AFR: 0.0114%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000251369">ZNF550</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.961</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.961</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.57547629G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=536070861">rs536070861</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs536070861&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.961</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000325134.9">ZNF550:ENST00000325134.9:c.615C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457177.5">ZNF550:ENST00000457177.5:c.615C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000506609.2">ZNF550:ENST00000506609.2:c.492C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000594943.1">ZNF550:ENST00000594943.1:c.34-4756G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602149.1">ZNF550:ENST00000602149.1:c.200-4756G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000601415.5">ZNF550:ENST00000601415.5:c.250+4998C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9641078</dd>
                                    
                                    <dd>CADD: 0.964 (14.450)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211659">IGLV3-25</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.961</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.961</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.22686799C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=443102">rs443102</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs443102&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.961</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390305.2">IGLV3-25:ENST00000390305.2:c.31C&gt;T:p.(Leu11Phe)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96064496</dd>
                                    
                                    <dd>CADD: 0.961 (14.050)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000237700">PRAMEF33</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.959</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.959</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.13306679A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1229614832">rs1229614832</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1229614832&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.959</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000437300.2">PRAMEF33:ENST00000437300.2:c.329A&gt;G:p.(Glu110Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.958885</dd>
                                    
                                    <dd>CADD: 0.959 (13.860)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165457">FOLR2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.955</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.955</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr11:g.72216929A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=371443707">rs371443707</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs371443707&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.955</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298223.11">FOLR2:ENST00000298223.11:c.-25+4A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449475.6">FOLR2:ENST00000449475.6:c.27+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454954.6">FOLR2:ENST00000454954.6:c.27+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000619261.4">FOLR2:ENST00000619261.4:c.27+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9599133</dd>
                                    
                                    <dd>CADD: 0.960 (13.970)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0088%</dd>
                                    <dd>ESP AA: 0.0251%</dd>
                                    <dd>ESP All: 0.0081%</dd>
                                    <dd>ExAC AFR: 0.0106%</dd>
                                    <dd>gnomAD_E_AFR: 0.0067%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000223705">NSUN5P1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.952</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.952</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr7:g.75416415T&gt;C [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=7050">rs7050</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs7050&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.952</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421140.6">NSUN5P1:ENST00000421140.6:n.1423T&gt;C:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422386.5">NSUN5P1:ENST00000422386.5:n.600T&gt;C:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428392.1">NSUN5P1:ENST00000428392.1:n.1190T&gt;C:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457988.5">NSUN5P1:ENST00000457988.5:n.1464T&gt;C:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9524665</dd>
                                    
                                    <dd>CADD: 0.952 (13.230)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135636">DYSF</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.952</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/253601">OMIM:253601</a> Muscular dystrophy, limb-girdle, autosomal recessive 2 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/254130">OMIM:254130</a> Miyoshi muscular dystrophy 1 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/606768">OMIM:606768</a> Myopathy, distal, with anterior tibial onset - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=178400">ORPHA:178400</a> Distal myopathy with anterior tibial onset</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=268">ORPHA:268</a> Dysferlin-related  limb-girdle muscular dystrophy R2</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=45448">ORPHA:45448</a> Miyoshi myopathy</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.952</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.71570663G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=529077809">rs529077809</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs529077809&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.952</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258104.7">DYSF:ENST00000258104.7:c.3096G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394120.6">DYSF:ENST00000394120.6:c.3099G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409366.5">DYSF:ENST00000409366.5:c.3099G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409582.7">DYSF:ENST00000409582.7:c.3147G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409651.5">DYSF:ENST00000409651.5:c.3192G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409744.5">DYSF:ENST00000409744.5:c.3057G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409762.5">DYSF:ENST00000409762.5:c.3147G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000410020.8">DYSF:ENST00000410020.8:c.3150G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000410041.1">DYSF:ENST00000410041.1:c.3150G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413539.6">DYSF:ENST00000413539.6:c.3189G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000429174.6">DYSF:ENST00000429174.6:c.3096G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.95481443</dd>
                                    
                                    <dd>CADD: 0.955 (13.450)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0100%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000277758">FO681492.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.948</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.010</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.948</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.47757224C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1435481212">rs1435481212</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1435481212&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.948</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622861.4">FO681492.1:ENST00000622861.4:c.927G&gt;C:p.(Glu309Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000623662.3">FO681492.1:ENST00000623662.3:c.1086G&gt;C:p.(Glu362Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.94776034</dd>
                                    
                                    <dd>CADD: 0.948 (12.820)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000179397">CATSPERE</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.939</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.009</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.939</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.244552652T&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=567394296">rs567394296</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs567394296&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.939</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366531.7">CATSPERE:ENST00000366531.7:c.414T&gt;G:p.(His138Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366533.8">CATSPERE:ENST00000366533.8:c.867T&gt;G:p.(His289Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366534.9">CATSPERE:ENST00000366534.9:c.867T&gt;G:p.(His289Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.944</dd>
                                    
                                    <dd>SIFT: 0.056 (D)</dd>
                                    <dd>CADD: 0.798 (6.953)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104972">LILRB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.245</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.661</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.245 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LILRB1">LILRB1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000805, abnormal visual cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.009</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.245</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.661</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.54631288C&gt;A [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1985501">rs1985501</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1985501&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.661</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324602.11">LILRB1:ENST00000324602.11:c.52C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396315.5">LILRB1:ENST00000396315.5:c.52C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396317.5">LILRB1:ENST00000396317.5:c.52C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396327.7">LILRB1:ENST00000396327.7:c.52C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396331.5">LILRB1:ENST00000396331.5:c.52C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396332.8">LILRB1:ENST00000396332.8:c.52C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427581.6">LILRB1:ENST00000427581.6:c.160C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6609217</dd>
                                    
                                    <dd>CADD: 0.661 (4.697)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000274471">AC242376.2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.938</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.009</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.938</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr15:g.23310185C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=9672843">rs9672843</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs9672843&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.938</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000619355.1">AC242376.2:ENST00000619355.1:n.130-3C&gt;G:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.93762654</dd>
                                    
                                    <dd>CADD: 0.938 (12.050)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211647">IGLV5-48</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.937</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.009</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.937</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.22353209G&gt;C [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201334749">rs201334749</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201334749&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.937</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390293.1">IGLV5-48:ENST00000390293.1:c.145G&gt;C:p.(Val49Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9370494</dd>
                                    
                                    <dd>CADD: 0.937 (12.010)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000270339">PPIAL4H</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.930</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.009</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.930</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.146344597C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1423108907">rs1423108907</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1423108907&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.930</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000584068.3">PPIAL4H:ENST00000584068.3:c.119G&gt;A:p.(Ser40Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000643391.1">PPIAL4H:ENST00000643391.1:n.376-16386G&gt;A:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9295307</dd>
                                    
                                    <dd>CADD: 0.930 (11.520)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000146410">MTFR2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.929</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.009</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.929</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr6:g.136241618C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=544618242">rs544618242</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs544618242&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.929</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000420702.6">MTFR2:ENST00000420702.6:c.340G&gt;C:p.(Val114Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000451457.6">MTFR2:ENST00000451457.6:c.340G&gt;C:p.(Val114Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.93513656</dd>
                                    
                                    <dd>Polyphen2: 0.024 (B)</dd>
                                    <dd>SIFT: 0.222 (T)</dd>
                                    <dd>CADD: 0.935 (11.880)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>ExAC AFR: 0.0289%</dd>
                                    <dd>gnomAD_E_AFR: 0.0327%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_AFR: 0.0458%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000229391">HLA-DRB6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.918</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.008</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.918</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr6:g.32553378G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=79811264">rs79811264</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs79811264&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.918</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000411500.5">HLA-DRB6:ENST00000411500.5:n.873C&gt;A:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000437183.5">HLA-DRB6:ENST00000437183.5:n.964C&gt;A:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000437650.2">HLA-DRB6:ENST00000437650.2:n.686C&gt;A:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9181535</dd>
                                    
                                    <dd>CADD: 0.918 (10.870)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000265190">ANXA8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.917</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.008</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.917</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.47478635G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1250105976">rs1250105976</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1250105976&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.917</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583874.5">ANXA8:ENST00000583874.5:c.219C&gt;A:p.(Asn73Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611843.4">ANXA8:ENST00000611843.4:c.219C&gt;A:p.(Asn73Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000585281.5">ANXA8:ENST00000585281.5:c.113-8C&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583448.2">ANXA8:ENST00000583448.2:c.21+5278C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583911.5">ANXA8:ENST00000583911.5:c.22-1441C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9168236</dd>
                                    
                                    <dd>CADD: 0.917 (10.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188559">RALGAPA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.913</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.007</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.913</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr20:g.20583047T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=369189295">rs369189295</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs369189295&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.913</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000202677.12">RALGAPA2:ENST00000202677.12:c.2707+3A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.92326385</dd>
                                    
                                    <dd>CADD: 0.923 (11.150)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0120%</dd>
                                    <dd>ESP AA: 0.0780%</dd>
                                    <dd>ESP All: 0.0248%</dd>
                                    <dd>ExAC AFR: 0.0617%</dd>
                                    <dd>ExAC AMR: 0.0266%</dd>
                                    <dd>gnomAD_E_AFR: 0.0589%</dd>
                                    <dd>gnomAD_E_AMR: 0.0239%</dd>
                                    <dd>gnomAD_E_OTH: 0.0184%</dd>
                                    <dd>gnomAD_G_AFR: 0.0458%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135205">CCDC146</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.909</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.007</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.909</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.77278968A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=367747203">rs367747203</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs367747203&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.909</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000285871.5">CCDC146:ENST00000285871.5:c.1561A&gt;G:p.(Ile521Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.921657</dd>
                                    
                                    <dd>Polyphen2: 0.006 (B)</dd>
                                    <dd>SIFT: 0.133 (T)</dd>
                                    <dd>CADD: 0.922 (11.060)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0104%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0995%</dd>
                                    <dd>ExAC AMR: 0.0176%</dd>
                                    <dd>gnomAD_E_AFR: 0.0658%</dd>
                                    <dd>gnomAD_E_AMR: 0.0121%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196196">HRCT1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.908</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.007</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.908</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_INSERTION</span>
                                <b>chr9:g.35906589A&gt;ACCACCACCACCACCC [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1554690900">rs1554690900</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1554690900&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.908</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354323.3">HRCT1:ENST00000354323.3:c.308_322dup:p.(His103_His107dup)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9075302</dd>
                                    
                                    <dd>CADD: 0.908 (10.340)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130052">STARD8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.337</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=STARD8%0DFGD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FGD1</a> and phenotypic similarity 0.634 to <a href="http://www.omim.org/305400">Mental retardation, X-linked syndromic 16</a> associated with FGD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001256, Intellectual disability, mild</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0009748, Large earlobe</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000508, Ptosis</dd><dd>HP:0001572, Macrodontia - HP:0000668, Hypodontia</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=STARD8%0DFGD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FGD1</a> and phenotypic similarity 0.385 to mouse mutant of FGD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0003984, embryonic growth retardation</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001297, microphthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0001297, microphthalmia</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.007</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.337</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr23:g.68718096C&gt;T [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=143103482">rs143103482</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs143103482&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.337</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000252336.10">STARD8:ENST00000252336.10:c.942C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374597.3">STARD8:ENST00000374597.3:c.942C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374599.7">STARD8:ENST00000374599.7:c.1182C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3397628</dd>
                                    
                                    <dd>CADD: 0.340 (1.803)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0265%</dd>
                                    <dd>TOPMed: 0.0111%</dd>
                                    <dd>ESP AA: 0.0261%</dd>
                                    <dd>ESP EA: 0.0149%</dd>
                                    <dd>ESP All: 0.0189%</dd>
                                    <dd>ExAC AFR: 0.0126%</dd>
                                    <dd>ExAC NFE: 0.0068%</dd>
                                    <dd>gnomAD_E_AFR: 0.0083%</dd>
                                    <dd>gnomAD_E_NFE: 0.0101%</dd>
                                    <dd>gnomAD_G_AFR: 0.0516%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000106683">LIMK1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.006</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.739</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.061</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.739 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=904">Williams syndrome</a> associated with LIMK1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure - HP:0000581, Blepharophimosis</dd><dd>HP:0001572, Macrodontia - HP:0000691, Microdontia</dd></dl><dl><dt>Phenotypic similarity 0.403 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LIMK1">LIMK1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0002063, abnormal learning/memory/conditioning</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000807, abnormal hippocampus morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=904">ORPHA:904</a> Williams syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.006</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.739</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.061</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr7:g.74107037G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=573783723">rs573783723</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs573783723&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.061</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336180.7">LIMK1:ENST00000336180.7:c.909G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000418310.5">LIMK1:ENST00000418310.5:c.999G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538333.3">LIMK1:ENST00000538333.3:c.807G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.061357975</dd>
                                    
                                    <dd>CADD: 0.061 (0.275)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>ExAC AFR: 0.0212%</dd>
                                    <dd>ExAC NFE: 0.0037%</dd>
                                    <dd>gnomAD_E_AFR: 0.0076%</dd>
                                    <dd>gnomAD_E_AMR: 0.0064%</dd>
                                    <dd>gnomAD_E_NFE: 0.0031%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188573">FBLL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.874</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.005</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.874</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr5:g.168529618G&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1044478393">rs1044478393</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1044478393&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.874</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000338333.5">FBLL1:ENST00000338333.5:c.114G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.873701</dd>
                                    
                                    <dd>CADD: 0.874 (8.986)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0016%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169894">MUC3A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.861</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.005</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.861</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.100966478C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=114167943">rs114167943</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs114167943&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.861</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.9704C&gt;T:p.(Ala3235Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.9612-174C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8609087</dd>
                                    
                                    <dd>CADD: 0.861 (8.567)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr7:g.100957266C&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1374178955">rs1374178955</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1374178955&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.134</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.5487C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.5487C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.13423502</dd>
                        
                        <dd>CADD: 0.134 (0.626)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196557">CACNA1H</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.290</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.322 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=64280">Childhood absence epilepsy</a> associated with CACNA1H.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CACNA1H%0DCACNA1G&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CACNA1G</a> and phenotypic similarity 0.673 to <a href="http://www.omim.org/618087">Spinocerebellar ataxia 42, early-onset, severe, with neurodevelopmental deficits</a> associated with CACNA1G.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0040080, Anteverted ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0045025, Narrow palpebral fissure</dd><dd>HP:0001572, Macrodontia - HP:0000303, Mandibular prognathia</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CACNA1H%0DCACNA1G&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CACNA1G</a> and phenotypic similarity 0.337 to mouse mutant of CACNA1G.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001406, abnormal gait</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0004097, abnormal cerebellar cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/617027">OMIM:617027</a> Hyperaldosteronism, familial, type IV - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=64280">ORPHA:64280</a> Childhood absence epilepsy</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.005</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.290</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr16:g.1207796C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=564292386">rs564292386</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs564292386&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.290</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000348261.11">CACNA1H:ENST00000348261.11:c.3090C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358590.8">CACNA1H:ENST00000358590.8:c.3090C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000565831.6">CACNA1H:ENST00000565831.6:c.3090C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000638323.1">CACNA1H:ENST00000638323.1:c.3051C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.29042226</dd>
                                    
                                    <dd>CADD: 0.290 (1.490)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000231475">IGHV4-31</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.860</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.005</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.860</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.106349375G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=61739267">rs61739267</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs61739267&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.860</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000438142.3">IGHV4-31:ENST00000438142.3:c.264C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8598478</dd>
                                    
                                    <dd>CADD: 0.860 (8.534)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr14:g.106349478G&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=117757529">rs117757529</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs117757529&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.011</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000438142.3">IGHV4-31:ENST00000438142.3:c.161C&gt;A:p.(Ser54Tyr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.010991573</dd>
                        
                        <dd>CADD: 0.011 (0.048)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135898">GPR55</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.290</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GPR55%0DLPAR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LPAR1</a> and phenotypic similarity 0.900 to mouse mutant of LPAR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - MP:0001255, decreased body height</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0000445, short snout</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - MP:0001300, ocular hypertelorism</dd><dd>HP:0001572, Macrodontia - MP:0004831, long incisors</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.004</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.290</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.230910510G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=375661190">rs375661190</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs375661190&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.290</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392039.2">GPR55:ENST00000392039.2:c.453C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392040.5">GPR55:ENST00000392040.5:c.453C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622008.4">GPR55:ENST00000622008.4:c.453C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000650999.1">GPR55:ENST00000650999.1:c.453C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.29123867</dd>
                                    
                                    <dd>CADD: 0.291 (1.495)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0048%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0289%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000172748">ZNF596</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.820</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.820</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr8:g.261159A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=547973615">rs547973615</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs547973615&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.820</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000640035.1">ZNF596:ENST00000640035.1:c.506+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.82457155</dd>
                                    
                                    <dd>CADD: 0.825 (7.559)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000278522">POTEB3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.808</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.808</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr15:g.21439885T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2605913">rs2605913</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2605913&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.808</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611217.4">POTEB3:ENST00000611217.4:c.127A&gt;G:p.(Met43Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.80782366</dd>
                                    
                                    <dd>CADD: 0.808 (7.163)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197953">AADACL2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.806</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.806</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.151734172C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=556814703">rs556814703</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs556814703&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.806</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356517.4">AADACL2:ENST00000356517.4:c.137C&gt;T:p.(Thr46Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.81041676</dd>
                                    
                                    <dd>Polyphen2: 0.002 (B)</dd>
                                    <dd>SIFT: 0.605 (T)</dd>
                                    <dd>CADD: 0.810 (7.222)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0099%</dd>
                                    <dd>ExAC AMR: 0.0183%</dd>
                                    <dd>ExAC NFE: 0.0032%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000235961">PNMA6A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.805</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.805</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.153073226C&gt;T [1/1:1/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201232151">rs201232151</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201232151&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.805</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421798.4">PNMA6A:ENST00000421798.4:c.164C&gt;T:p.(Ala55Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8051053</dd>
                                    
                                    <dd>SIFT: 0.936 (T)</dd>
                                    <dd>CADD: 0.805 (7.102)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr23:g.153073517G&gt;A [1/1:1/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1347594472">rs1347594472</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1347594472&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.624</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421798.4">PNMA6A:ENST00000421798.4:c.455G&gt;A:p.(Cys152Tyr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.6244221</dd>
                        
                        <dd>SIFT: 1.000 (T)</dd>
                        <dd>CADD: 0.624 (4.253)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000273763">AC007318.3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr2:g.65207203CAA&gt;C [1/1:1/1:-/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=34639949">rs34639949</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs34639949&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000650256.1">AC007318.3:ENST00000650256.1:n.196-9_196-8del:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.19942874</dd>
                                    
                                    <dd>CADD: 0.199 (0.966)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000285215">AC241377.5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr1:g.120973340T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1401211209">rs1401211209</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1401211209&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000642174.1">AC241377.5:ENST00000642174.1:n.518-7T&gt;C:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000649989.1">AC241377.5:ENST00000649989.1:n.727-7T&gt;C:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.60861874</dd>
                                    
                                    <dd>CADD: 0.609 (4.074)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105136">ZNF419</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.795</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.795</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr19:g.57490143A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=368169880">rs368169880</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs368169880&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.795</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000221735.11">ZNF419:ENST00000221735.11:c.34-4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000347466.10">ZNF419:ENST00000347466.10:c.34-4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424930.6">ZNF419:ENST00000424930.6:c.34-4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000518999.5">ZNF419:ENST00000518999.5:c.34-4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354197.8">ZNF419:ENST00000354197.8:c.34-1325A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000415379.6">ZNF419:ENST00000415379.6:c.34-1328A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000426954.6">ZNF419:ENST00000426954.6:c.34-1325A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000442920.6">ZNF419:ENST00000442920.6:c.34-1328A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000520540.5">ZNF419:ENST00000520540.5:c.34-1325A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000601674.6">ZNF419:ENST00000601674.6:c.34-1328A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3473205</dd>
                                    
                                    <dd>CADD: 0.347 (1.853)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>ESP EA: 0.0120%</dd>
                                    <dd>ESP All: 0.0080%</dd>
                                    <dd>ExAC AFR: 0.0106%</dd>
                                    <dd>ExAC AMR: 0.0275%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_AMR: 0.0418%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000176155">CCDC57</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.793</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.002</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.793</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.82178574G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=573980729">rs573980729</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs573980729&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.793</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389641.9">CCDC57:ENST00000389641.9:c.1406C&gt;T:p.(Ala469Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392343.3">CCDC57:ENST00000392343.3:c.1406C&gt;T:p.(Ala469Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392347.5">CCDC57:ENST00000392347.5:c.1406C&gt;T:p.(Ala469Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000665763.1">CCDC57:ENST00000665763.1:c.1406C&gt;T:p.(Ala469Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.79474205</dd>
                                    
                                    <dd>Polyphen2: 0.009 (B)</dd>
                                    <dd>SIFT: 0.259 (T)</dd>
                                    <dd>CADD: 0.795 (6.877)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AFR: 0.0103%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0102%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182885">ADGRG3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.790</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.002</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.790</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr16:g.57684382G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=531431822">rs531431822</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs531431822&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.790</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333493.9">ADGRG3:ENST00000333493.9:c.1163-8G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000450388.7">ADGRG3:ENST00000450388.7:c.803-8G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.06760484</dd>
                                    
                                    <dd>CADD: 0.068 (0.304)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0159%</dd>
                                    <dd>ExAC AFR: 0.0577%</dd>
                                    <dd>gnomAD_E_AFR: 0.0850%</dd>
                                    <dd>gnomAD_G_AFR: 0.0459%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198573">SPANXC</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.766</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.002</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.766</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.141241609C&gt;G [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3208371">rs3208371</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs3208371&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.766</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358993.3">SPANXC:ENST00000358993.3:c.202G&gt;C:p.(Val68Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.76611626</dd>
                                    
                                    <dd>CADD: 0.766 (6.310)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000090975">PITPNM2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.166</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PITPNM2%0DCHRNG&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CHRNG</a> and phenotypic similarity 0.622 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2990">Autosomal recessive multiple pterygium syndrome</a> associated with CHRNG.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0004322, Short stature</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000369, Low-set ears</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000506, Telecanthus</dd><dd>HP:0001572, Macrodontia - HP:0000307, Pointed chin</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PITPNM2%0DCHRNG&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CHRNG</a> and phenotypic similarity 0.274 to mouse mutant of CHRNG.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001512, trunk curl</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.166</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr12:g.123005367G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=375136096">rs375136096</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs375136096&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.166</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000280562.9">PITPNM2:ENST00000280562.9:c.825C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000320201.8">PITPNM2:ENST00000320201.8:c.825C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542749.5">PITPNM2:ENST00000542749.5:c.825C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000546049.5">PITPNM2:ENST00000546049.5:c.825C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1674698</dd>
                                    
                                    <dd>CADD: 0.167 (0.796)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0288%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0327%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183067">IGSF5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.155</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=IGSF5%0DTCOF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TCOF1</a> and phenotypic similarity 0.618 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=861">Treacher-Collins syndrome</a> associated with TCOF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature - HP:0001508, Failure to thrive</dd><dd>HP:0000252, Microcephaly - HP:0000248, Brachycephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0008551, Microtia</dd><dd>HP:0000316, Hypertelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000637, Long palpebral fissure - HP:0000625, Eyelid coloboma</dd><dd>HP:0001572, Macrodontia - HP:0006482, Abnormality of dental morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=IGSF5%0DTCOF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TCOF1</a> and phenotypic similarity 0.780 to mouse mutant of TCOF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - MP:0001504, abnormal posture</dd><dd>HP:0004322, Short stature - MP:0010865, prenatal growth retardation</dd><dd>HP:0000252, Microcephaly - MP:0000783, abnormal forebrain morphology</dd><dd>HP:0000325, Triangular face - MP:0012523, abnormal upper lip morphology</dd><dd>HP:0000400, Macrotia - MP:0003735, cup-shaped ears</dd><dd>HP:0000316, Hypertelorism - MP:0001293, anophthalmia</dd><dd>HP:0000637, Long palpebral fissure - MP:0001293, anophthalmia</dd><dd>HP:0001572, Macrodontia - MP:0004592, small mandible</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.155</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr21:g.39765824G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=555437974">rs555437974</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs555437974&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.155</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380588.4">IGSF5:ENST00000380588.4:c.390G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.15530485</dd>
                                    
                                    <dd>CADD: 0.155 (0.733)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000266714">MYO15B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.708</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.708</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.75611914C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=557233957">rs557233957</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs557233957&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.708</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000610510.4">MYO15B:ENST00000610510.4:c.4482C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000633867.1">MYO15B:ENST00000633867.1:c.1356C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000642007.1">MYO15B:ENST00000642007.1:c.90C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7122601</dd>
                                    
                                    <dd>CADD: 0.712 (5.410)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>gnomAD_E_AFR: 0.0323%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000227234">SPANXB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.703</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.703</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.141003560C&gt;G [1/1:1/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=878856926">rs878856926</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs878856926&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.703</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449283.2">SPANXB1:ENST00000449283.2:c.220C&gt;G:p.(Leu74Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7029018</dd>
                                    
                                    <dd>CADD: 0.703 (5.271)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211900">IGHJ6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.634</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.634</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.105863241G&gt;A [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2338628">rs2338628</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2338628&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.634</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390560.2">IGHJ6:ENST00000390560.2:c.18C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.69689023</dd>
                                    
                                    <dd>CADD: 0.697 (5.184)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0885%</dd>
                                    <dd>gnomAD_E_AMR: 0.1604%</dd>
                                    <dd>gnomAD_E_ASJ: 0.4987%</dd>
                                    <dd>gnomAD_E_EAS: 0.0845%</dd>
                                    <dd>gnomAD_E_FIN: 0.0091%</dd>
                                    <dd>gnomAD_E_NFE: 0.1313%</dd>
                                    <dd>gnomAD_E_OTH: 0.0963%</dd>
                                    <dd>gnomAD_E_SAS: 0.0335%</dd>
                                    <dd>gnomAD_G_AMR: 0.5102%</dd>
                                    <dd>gnomAD_G_NFE: 0.0473%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000099814">CEP170B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.627</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.627</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.104889519G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=770807911">rs770807911</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs770807911&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.627</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453495.2">CEP170B:ENST00000453495.2:c.3747G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556508.5">CEP170B:ENST00000556508.5:c.3534G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000414716.8">CEP170B:ENST00000414716.8:c.3740-101G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.627351</dd>
                                    
                                    <dd>CADD: 0.627 (4.287)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0055%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182330">PRAMEF8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.625</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.625</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.13281806C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1891607">rs1891607</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1891607&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.625</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357367.6">PRAMEF8:ENST00000357367.6:c.990G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614831.1">PRAMEF8:ENST00000614831.1:c.990G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621519.4">PRAMEF8:ENST00000621519.4:c.990G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6254585</dd>
                                    
                                    <dd>CADD: 0.625 (4.265)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000075234">TTC38</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.619</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.619</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr22:g.46289855C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=530056986">rs530056986</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs530056986&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.619</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381031.8">TTC38:ENST00000381031.8:c.1272C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.62120867</dd>
                                    
                                    <dd>CADD: 0.621 (4.216)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0048%</dd>
                                    <dd>ExAC AFR: 0.0306%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>ExAC FIN: 0.0151%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0298%</dd>
                                    <dd>gnomAD_E_FIN: 0.0135%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000181381">DDX60L</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.024</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DDX60L%0DHBS1L&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HBS1L</a> and phenotypic similarity 0.707 to mouse mutant of HBS1L.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly -</dd><dd>HP:0000325, Triangular face - MP:0003743, abnormal facial morphology</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism - MP:0001314, corneal opacity</dd><dd>HP:0000637, Long palpebral fissure - MP:0001314, corneal opacity</dd><dd>HP:0001572, Macrodontia - MP:0002100, abnormal tooth morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.024</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr4:g.168400879A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=543289709">rs543289709</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs543289709&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.024</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000260184.11">DDX60L:ENST00000260184.11:c.3438T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505890.5">DDX60L:ENST00000505890.5:c.3438T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000511577.5">DDX60L:ENST00000511577.5:c.3438T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.024336636</dd>
                                    
                                    <dd>CADD: 0.024 (0.107)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC AFR: 0.0309%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.0458%</dd>
                                    <dd>gnomAD_E_SAS: 0.0098%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000282936">AC004706.3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.575</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.575</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.6636886G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=943537383">rs943537383</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs943537383&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.575</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000634965.2">AC004706.3:ENST00000634965.2:c.3431C&gt;T:p.(Pro1144Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361413.8">AC004706.3:ENST00000361413.8:c.-68-1715C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000572370.5">AC004706.3:ENST00000572370.5:c.-824+3751C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5778177</dd>
                                    
                                    <dd>CADD: 0.578 (3.745)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0183%</dd>
                                    <dd>gnomAD_G_AFR: 0.0345%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000147113">DIPK2B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.574</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.574</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr23:g.45157856C&gt;G [1/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=774573891">rs774573891</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs774573891&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.574</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398000.7">DIPK2B:ENST00000398000.7:c.531G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5762595</dd>
                                    
                                    <dd>CADD: 0.576 (3.729)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0265%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163395">IGFN1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.005</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=IGFN1%0DPRDX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRDX1</a> and phenotypic similarity 0.649 to <a href="http://www.omim.org/277400">Methylmalonic aciduria and homocystinuria, cblC type, digenic</a> associated with PRDX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability - HP:0001249, Intellectual disability</dd><dd>HP:0004322, Short stature - HP:0001508, Failure to thrive</dd><dd>HP:0000252, Microcephaly - HP:0000252, Microcephaly</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia - HP:0000400, Macrotia</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.005</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.201225837C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=545610095">rs545610095</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs545610095&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.005</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295591.12">IGFN1:ENST00000295591.12:c.3129C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335211.9">IGFN1:ENST00000335211.9:c.10500C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.0045945644</dd>
                                    
                                    <dd>CADD: 0.005 (0.020)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0056%</dd>
                                    <dd>ExAC AFR: 0.0209%</dd>
                                    <dd>gnomAD_E_AFR: 0.0198%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188771">PLET1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.514</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.514</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.112260462A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=368109344">rs368109344</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs368109344&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.514</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000338832.4">PLET1:ENST00000338832.4:c.128T&gt;C:p.(Ile43Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.519382</dd>
                                    
                                    <dd>Polyphen2: 0.002 (B)</dd>
                                    <dd>SIFT: 0.519 (T)</dd>
                                    <dd>CADD: 0.519 (3.182)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ESP AA: 0.0723%</dd>
                                    <dd>ESP All: 0.0219%</dd>
                                    <dd>ExAC AFR: 0.0455%</dd>
                                    <dd>gnomAD_E_AFR: 0.0257%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000126226">PCID2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.432</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.432</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr13:g.113197091C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=557392526">rs557392526</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs557392526&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.432</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000246505.9">PCID2:ENST00000246505.9:c.353G&gt;C:p.(Ser118Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622406.4">PCID2:ENST00000622406.4:c.353G&gt;C:p.(Ser118Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000337344.9">PCID2:ENST00000337344.9:c.266+87G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375457.2">PCID2:ENST00000375457.2:c.260+87G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375459.5">PCID2:ENST00000375459.5:c.260+87G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375477.5">PCID2:ENST00000375477.5:c.266+87G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375479.6">PCID2:ENST00000375479.6:c.266+87G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.436</dd>
                                    
                                    <dd>SIFT: 0.564 (T)</dd>
                                    <dd>CADD: 0.323 (1.693)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0088%</dd>
                                    <dd>ExAC AFR: 0.0577%</dd>
                                    <dd>gnomAD_E_AFR: 0.0392%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167613">LAIR1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.424</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.424</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54360069C&gt;G [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=190462445">rs190462445</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs190462445&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.424</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391742.7">LAIR1:ENST00000391742.7:c.368G&gt;C:p.(Ser123Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391743.7">LAIR1:ENST00000391743.7:c.314G&gt;C:p.(Ser105Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000434277.6">LAIR1:ENST00000434277.6:c.365G&gt;C:p.(Ser122Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000348231.8">LAIR1:ENST00000348231.8:c.364+847G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000474878.5">LAIR1:ENST00000474878.5:c.361+847G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4244275</dd>
                                    
                                    <dd>CADD: 0.424 (2.399)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000279804">PRAMEF18</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.388</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.388</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.13223890C&gt;T [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=56066665">rs56066665</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs56066665&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.388</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000624297.2">PRAMEF18:ENST00000624297.2:c.882G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.38835418</dd>
                                    
                                    <dd>CADD: 0.388 (2.135)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105609">LILRB5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.245</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.039</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.245 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LILRB5">LILRB5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001249, Intellectual disability -</dd><dd>HP:0004322, Short stature -</dd><dd>HP:0000252, Microcephaly - MP:0000805, abnormal visual cortex morphology</dd><dd>HP:0000325, Triangular face -</dd><dd>HP:0000400, Macrotia -</dd><dd>HP:0000316, Hypertelorism -</dd><dd>HP:0000637, Long palpebral fissure -</dd><dd>HP:0001572, Macrodontia -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.245</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.039</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.54256077C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=144716655">rs144716655</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs144716655&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.039</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000316219.9">LILRB5:ENST00000316219.9:c.621G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449561.3">LILRB5:ENST00000449561.3:c.621G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000345866.10">LILRB5:ENST00000345866.10:c.355+412G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.039936423</dd>
                                    
                                    <dd>CADD: 0.040 (0.177)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0199%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0806%</dd>
                                    <dd>ExAC NFE: 0.0047%</dd>
                                    <dd>gnomAD_E_AFR: 0.0755%</dd>
                                    <dd>gnomAD_E_NFE: 0.0030%</dd>
                                    <dd>gnomAD_G_AFR: 0.0692%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr19:g.54256254G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=141325399">rs141325399</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs141325399&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.006</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000316219.9">LILRB5:ENST00000316219.9:c.444C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449561.3">LILRB5:ENST00000449561.3:c.444C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000345866.10">LILRB5:ENST00000345866.10:c.355+235C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.006426513</dd>
                        
                        <dd>CADD: 0.006 (0.028)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0799%</dd>
                        <dd>TOPMed: 0.0183%</dd>
                        <dd>ESP AA: 0.0454%</dd>
                        <dd>ESP All: 0.0154%</dd>
                        <dd>ExAC AFR: 0.0772%</dd>
                        <dd>ExAC NFE: 0.0030%</dd>
                        <dd>gnomAD_E_AFR: 0.0785%</dd>
                        <dd>gnomAD_E_AMR: 0.0089%</dd>
                        <dd>gnomAD_E_NFE: 0.0036%</dd>
                        <dd>gnomAD_G_AFR: 0.0689%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000131845">AC005261.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.313</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.313</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.57355329G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=574274358">rs574274358</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs574274358&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.313</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000443917.6">AC005261.1:ENST00000443917.6:c.219G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000282286.5">AC005261.1:ENST00000282286.5:c.161-701G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391705.7">AC005261.1:ENST00000391705.7:c.161-701G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000598744.1">AC005261.1:ENST00000598744.1:c.35-701G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.313406</dd>
                                    
                                    <dd>CADD: 0.313 (1.633)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AFR: 0.0123%</dd>
                                    <dd>gnomAD_E_AFR: 0.0072%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000270629">NBPF14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.308</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.308</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.148587330C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1343209236">rs1343209236</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1343209236&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.308</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581201.5">NBPF14:ENST00000581201.5:c.1062G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614329.4">NBPF14:ENST00000614329.4:c.249G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614999.4">NBPF14:ENST00000614999.4:c.1062G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616120.4">NBPF14:ENST00000616120.4:c.249G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620240.1">NBPF14:ENST00000620240.1:c.957G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621070.4">NBPF14:ENST00000621070.4:c.1062G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611826.4">NBPF14:ENST00000611826.4:c.567-2654G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621066.4">NBPF14:ENST00000621066.4:c.72-861G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.308169</dd>
                                    
                                    <dd>CADD: 0.308 (1.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr1:g.148586448G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1360509174">rs1360509174</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1360509174&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.288</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581201.5">NBPF14:ENST00000581201.5:c.1113C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614329.4">NBPF14:ENST00000614329.4:c.300C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614999.4">NBPF14:ENST00000614999.4:c.1113C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616120.4">NBPF14:ENST00000616120.4:c.300C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620240.1">NBPF14:ENST00000620240.1:c.1008C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621066.4">NBPF14:ENST00000621066.4:c.93C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621070.4">NBPF14:ENST00000621070.4:c.1113C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611826.4">NBPF14:ENST00000611826.4:c.567-1772C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.2879672</dd>
                        
                        <dd>CADD: 0.288 (1.475)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000261732">AL031708.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.271</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.271</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.1647048C&gt;G [0/1:0/1:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.271</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454337.1">AL031708.1:ENST00000454337.1:c.295C&gt;G:p.(Leu99Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000293925.9">AL031708.1:ENST00000293925.9:c.828-5448C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397412.8">AL031708.1:ENST00000397412.8:c.828-5448C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.27087831</dd>
                                    
                                    <dd>CADD: 0.271 (1.372)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000267740">AC024592.3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.253</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.253</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.5896588C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=371542591">rs371542591</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs371542591&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.253</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000586349.5">AC024592.3:ENST00000586349.5:c.248G&gt;A:p.(Arg83His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308961.4">AC024592.3:ENST00000308961.4:c.191-13G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000418389.6">AC024592.3:ENST00000418389.6:c.191-13G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000592634.5">AC024592.3:ENST00000592634.5:c.191-13G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.25337934</dd>
                                    
                                    <dd>CADD: 0.253 (1.269)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ESP AA: 0.0231%</dd>
                                    <dd>ESP All: 0.0078%</dd>
                                    <dd>ExAC SAS: 0.0124%</dd>
                                    <dd>gnomAD_E_NFE: 0.0031%</dd>
                                    <dd>gnomAD_E_SAS: 0.0042%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000128383">APOBEC3A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.189</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.189</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr22:g.38961569C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201179078">rs201179078</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201179078&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.189</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000249116.7">APOBEC3A:ENST00000249116.7:c.357C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000402255.5">APOBEC3A:ENST00000402255.5:c.357C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618553.1">APOBEC3A:ENST00000618553.1:c.357C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1894123</dd>
                                    
                                    <dd>CADD: 0.189 (0.912)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ESP AA: 0.0235%</dd>
                                    <dd>ESP All: 0.0081%</dd>
                                    <dd>ExAC AFR: 0.0203%</dd>
                                    <dd>ExAC NFE: 0.0065%</dd>
                                    <dd>gnomAD_E_AFR: 0.0135%</dd>
                                    <dd>gnomAD_E_NFE: 0.0076%</dd>
                                    <dd>gnomAD_G_AFR: 0.0272%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000274209">ANTXRL</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.113</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.113</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.46330066C&gt;G [1/1:1/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=7091749">rs7091749</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs7091749&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.113</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622632.4">ANTXRL:ENST00000622632.4:c.1658C&gt;G:p.(Pro553Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620264.4">ANTXRL:ENST00000620264.4:c.1878C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.11284399</dd>
                                    
                                    <dd>CADD: 0.113 (0.520)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170954">ZNF415</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.015</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.015</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.53122407G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=566752458">rs566752458</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs566752458&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.015</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000500065.8">ZNF415:ENST00000500065.8:c.12C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000595813.5">ZNF415:ENST00000595813.5:c.12C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000599261.5">ZNF415:ENST00000599261.5:c.12C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000243643.8">ZNF415:ENST00000243643.8:c.15+255C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421033.5">ZNF415:ENST00000421033.5:c.15+255C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000594011.5">ZNF415:ENST00000594011.5:c.15+255C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000595193.5">ZNF415:ENST00000595193.5:c.15+255C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000597503.5">ZNF415:ENST00000597503.5:c.15+255C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000597748.5">ZNF415:ENST00000597748.5:c.15+255C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000600574.5">ZNF415:ENST00000600574.5:c.15+255C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000601215.5">ZNF415:ENST00000601215.5:c.308+255C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000601493.5">ZNF415:ENST00000601493.5:c.-555+255C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.0150821805</dd>
                                    
                                    <dd>CADD: 0.015 (0.066)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0183%</dd>
                                    <dd>ExAC SAS: 0.0128%</dd>
                                    <dd>gnomAD_E_AFR: 0.0268%</dd>
                                    <dd>gnomAD_E_NFE: 0.0035%</dd>
                                    <dd>gnomAD_E_OTH: 0.0266%</dd>
                                    <dd>gnomAD_E_SAS: 0.0131%</dd>
                                    <dd>gnomAD_G_AFR: 0.0917%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000179698">WDR97</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.015</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.015</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr8:g.144113842C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=529678124">rs529678124</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs529678124&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.015</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000323662.9">WDR97:ENST00000323662.9:c.3369C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.01462847</dd>
                                    
                                    <dd>CADD: 0.015 (0.064)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0127%</dd>
                                    <dd>gnomAD_E_AFR: 0.0323%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165935">SMCO2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.013</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.013</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr12:g.27470643G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=373506540">rs373506540</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs373506540&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.013</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298876.8">SMCO2:ENST00000298876.8:c.12G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000416383.5">SMCO2:ENST00000416383.5:c.12G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000535986.1">SMCO2:ENST00000535986.1:c.12G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.013038993</dd>
                                    
                                    <dd>CADD: 0.013 (0.057)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ESP AA: 0.0723%</dd>
                                    <dd>ESP All: 0.0219%</dd>
                                    <dd>ExAC SAS: 0.0126%</dd>
                                    <dd>gnomAD_E_AFR: 0.0132%</dd>
                                    <dd>gnomAD_E_AMR: 0.0125%</dd>
                                    <dd>gnomAD_E_NFE: 0.0017%</dd>
                                    <dd>gnomAD_E_SAS: 0.0044%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000263464">PPIAL4C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.010</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.010</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.149583931G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1247433378">rs1247433378</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1247433378&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.010</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000585245.2">PPIAL4C:ENST00000585245.2:c.34G&gt;A:p.(Val12Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.0103081465</dd>
                                    
                                    <dd>CADD: 0.010 (0.045)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000275342">PRAG1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.001</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.001</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr8:g.8377860C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200291133">rs200291133</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200291133&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.001</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615670.5">PRAG1:ENST00000615670.5:c.549G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622241.1">PRAG1:ENST00000622241.1:c.549G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.0011506081</dd>
                                    
                                    <dd>CADD: 0.001 (0.005)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0088%</dd>
                                    <dd>ESP AA: 0.0252%</dd>
                                    <dd>ESP All: 0.0082%</dd>
                                    <dd>ExAC AFR: 0.0510%</dd>
                                    <dd>gnomAD_E_AFR: 0.0458%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>

    

    <div id="about">
        <h3>About</h3>
        <p>
            The Exomizer is a Java program that functionally annotates variants from whole-exome
            sequencing data starting from a VCF file (version 4). The functional annotation code is
            based on <a href="https://github.com/charite/jannovar/">Jannovar</a> and uses
            <a href="http://genome.ucsc.edu/">UCSC</a> KnownGene transcript definitions and
            hg19 genomic coordinates
        </p>
        <p>
            Variants are prioritized according to user-defined criteria on variant frequency, pathogenicity,
            quality, inheritance pattern, and model organism phenotype data. Predicted pathogenicity data
            was extracted from the <a href="http://www.ncbi.nlm.nih.gov/pubmed/21520341">dbNSFP</a> resource.
        </p>
        <p>
            Developed by the Computational Biology and Bioinformatics group at the
            <a href="http://genetik.charite.de/">Institute for Medical Genetics and Human Genetics</a> of the
            <a href="http://www.charite.de">Charit&eacute; - Universit&auml;tsmedizin Berlin</a>, the Mouse
            Informatics Group at the <a href="http://www.sanger.ac.uk/">Sanger Institute</a> and the Smedley group at
            <a href="https://qmul.ac.uk/whri">Queen Mary University of London</a>.
        </p>


        <p>Problems, suggestions, or comments? Please <a href="mailto:peter.robinson@charite.de">let us know</a></p>
    </div>
</div>
</body>
</html>
