<!--
  ~ The Exomiser - A tool to annotate and prioritize genomic variants
  ~
  ~ Copyright (c) 2016-2019 Queen Mary University of London.
  ~ Copyright (c) 2012-2016 Charité Universitätsmedizin Berlin and Genome Research Ltd.
  ~
  ~ This program is free software: you can redistribute it and/or modify
  ~ it under the terms of the GNU Affero General Public License as
  ~ published by the Free Software Foundation, either version 3 of the
  ~ License, or (at your option) any later version.
  ~
  ~ This program is distributed in the hope that it will be useful,
  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  ~ GNU Affero General Public License for more details.
  ~
  ~ You should have received a copy of the GNU Affero General Public License
  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  -->

<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>
    <title>The Exomiser - A Tool to Annotate and Prioritise Whole-Exome Sequencing Data</title>
    <meta charset="UTF-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <style>
        /*!
 * Bootstrap v3.3.7 (http://getbootstrap.com)
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 *//*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */html{font-family:sans-serif;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block;vertical-align:baseline}audio:not([controls]){display:none;height:0}[hidden],template{display:none}a{background-color:transparent}a:active,a:hover{outline:0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:700}dfn{font-style:italic}h1{margin:.67em 0;font-size:2em}mark{color:#000;background:#ff0}small{font-size:80%}sub,sup{position:relative;font-size:75%;line-height:0;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}img{border:0}svg:not(:root){overflow:hidden}figure{margin:1em 40px}hr{height:0;-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}pre{overflow:auto}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}button,input,optgroup,select,textarea{margin:0;font:inherit;color:inherit}button{overflow:visible}button,select{text-transform:none}button,html input[type=button],input[type=reset],input[type=submit]{-webkit-appearance:button;cursor:pointer}button[disabled],html input[disabled]{cursor:default}button::-moz-focus-inner,input::-moz-focus-inner{padding:0;border:0}input{line-height:normal}input[type=checkbox],input[type=radio]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;padding:0}input[type=number]::-webkit-inner-spin-button,input[type=number]::-webkit-outer-spin-button{height:auto}input[type=search]{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;-webkit-appearance:textfield}input[type=search]::-webkit-search-cancel-button,input[type=search]::-webkit-search-decoration{-webkit-appearance:none}fieldset{padding:.35em .625em .75em;margin:0 2px;border:1px solid silver}legend{padding:0;border:0}textarea{overflow:auto}optgroup{font-weight:700}table{border-spacing:0;border-collapse:collapse}td,th{padding:0}/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */@media print{*,:after,:before{color:#000!important;text-shadow:none!important;background:0 0!important;-webkit-box-shadow:none!important;box-shadow:none!important}a,a:visited{text-decoration:underline}a[href]:after{content:" (" attr(href) ")"}abbr[title]:after{content:" (" attr(title) ")"}a[href^="javascript:"]:after,a[href^="#"]:after{content:""}blockquote,pre{border:1px solid #999;page-break-inside:avoid}thead{display:table-header-group}img,tr{page-break-inside:avoid}img{max-width:100%!important}h2,h3,p{orphans:3;widows:3}h2,h3{page-break-after:avoid}.navbar{display:none}.btn>.caret,.dropup>.btn>.caret{border-top-color:#000!important}.label{border:1px solid #000}.table{border-collapse:collapse!important}.table td,.table th{background-color:#fff!important}.table-bordered td,.table-bordered th{border:1px solid #ddd!important}}@font-face{font-family:'Glyphicons Halflings';src:url(../fonts/glyphicons-halflings-regular.eot);src:url(../fonts/glyphicons-halflings-regular.eot?#iefix) format('embedded-opentype'),url(../fonts/glyphicons-halflings-regular.woff2) format('woff2'),url(../fonts/glyphicons-halflings-regular.woff) format('woff'),url(../fonts/glyphicons-halflings-regular.ttf) format('truetype'),url(../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular) format('svg')}.glyphicon{position:relative;top:1px;display:inline-block;font-family:'Glyphicons Halflings';font-style:normal;font-weight:400;line-height:1;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.glyphicon-asterisk:before{content:"\002a"}.glyphicon-plus:before{content:"\002b"}.glyphicon-eur:before,.glyphicon-euro:before{content:"\20ac"}.glyphicon-minus:before{content:"\2212"}.glyphicon-cloud:before{content:"\2601"}.glyphicon-envelope:before{content:"\2709"}.glyphicon-pencil:before{content:"\270f"}.glyphicon-glass:before{content:"\e001"}.glyphicon-music:before{content:"\e002"}.glyphicon-search:before{content:"\e003"}.glyphicon-heart:before{content:"\e005"}.glyphicon-star:before{content:"\e006"}.glyphicon-star-empty:before{content:"\e007"}.glyphicon-user:before{content:"\e008"}.glyphicon-film:before{content:"\e009"}.glyphicon-th-large:before{content:"\e010"}.glyphicon-th:before{content:"\e011"}.glyphicon-th-list:before{content:"\e012"}.glyphicon-ok:before{content:"\e013"}.glyphicon-remove:before{content:"\e014"}.glyphicon-zoom-in:before{content:"\e015"}.glyphicon-zoom-out:before{content:"\e016"}.glyphicon-off:before{content:"\e017"}.glyphicon-signal:before{content:"\e018"}.glyphicon-cog:before{content:"\e019"}.glyphicon-trash:before{content:"\e020"}.glyphicon-home:before{content:"\e021"}.glyphicon-file:before{content:"\e022"}.glyphicon-time:before{content:"\e023"}.glyphicon-road:before{content:"\e024"}.glyphicon-download-alt:before{content:"\e025"}.glyphicon-download:before{content:"\e026"}.glyphicon-upload:before{content:"\e027"}.glyphicon-inbox:before{content:"\e028"}.glyphicon-play-circle:before{content:"\e029"}.glyphicon-repeat:before{content:"\e030"}.glyphicon-refresh:before{content:"\e031"}.glyphicon-list-alt:before{content:"\e032"}.glyphicon-lock:before{content:"\e033"}.glyphicon-flag:before{content:"\e034"}.glyphicon-headphones:before{content:"\e035"}.glyphicon-volume-off:before{content:"\e036"}.glyphicon-volume-down:before{content:"\e037"}.glyphicon-volume-up:before{content:"\e038"}.glyphicon-qrcode:before{content:"\e039"}.glyphicon-barcode:before{content:"\e040"}.glyphicon-tag:before{content:"\e041"}.glyphicon-tags:before{content:"\e042"}.glyphicon-book:before{content:"\e043"}.glyphicon-bookmark:before{content:"\e044"}.glyphicon-print:before{content:"\e045"}.glyphicon-camera:before{content:"\e046"}.glyphicon-font:before{content:"\e047"}.glyphicon-bold:before{content:"\e048"}.glyphicon-italic:before{content:"\e049"}.glyphicon-text-height:before{content:"\e050"}.glyphicon-text-width:before{content:"\e051"}.glyphicon-align-left:before{content:"\e052"}.glyphicon-align-center:before{content:"\e053"}.glyphicon-align-right:before{content:"\e054"}.glyphicon-align-justify:before{content:"\e055"}.glyphicon-list:before{content:"\e056"}.glyphicon-indent-left:before{content:"\e057"}.glyphicon-indent-right:before{content:"\e058"}.glyphicon-facetime-video:before{content:"\e059"}.glyphicon-picture:before{content:"\e060"}.glyphicon-map-marker:before{content:"\e062"}.glyphicon-adjust:before{content:"\e063"}.glyphicon-tint:before{content:"\e064"}.glyphicon-edit:before{content:"\e065"}.glyphicon-share:before{content:"\e066"}.glyphicon-check:before{content:"\e067"}.glyphicon-move:before{content:"\e068"}.glyphicon-step-backward:before{content:"\e069"}.glyphicon-fast-backward:before{content:"\e070"}.glyphicon-backward:before{content:"\e071"}.glyphicon-play:before{content:"\e072"}.glyphicon-pause:before{content:"\e073"}.glyphicon-stop:before{content:"\e074"}.glyphicon-forward:before{content:"\e075"}.glyphicon-fast-forward:before{content:"\e076"}.glyphicon-step-forward:before{content:"\e077"}.glyphicon-eject:before{content:"\e078"}.glyphicon-chevron-left:before{content:"\e079"}.glyphicon-chevron-right:before{content:"\e080"}.glyphicon-plus-sign:before{content:"\e081"}.glyphicon-minus-sign:before{content:"\e082"}.glyphicon-remove-sign:before{content:"\e083"}.glyphicon-ok-sign:before{content:"\e084"}.glyphicon-question-sign:before{content:"\e085"}.glyphicon-info-sign:before{content:"\e086"}.glyphicon-screenshot:before{content:"\e087"}.glyphicon-remove-circle:before{content:"\e088"}.glyphicon-ok-circle:before{content:"\e089"}.glyphicon-ban-circle:before{content:"\e090"}.glyphicon-arrow-left:before{content:"\e091"}.glyphicon-arrow-right:before{content:"\e092"}.glyphicon-arrow-up:before{content:"\e093"}.glyphicon-arrow-down:before{content:"\e094"}.glyphicon-share-alt:before{content:"\e095"}.glyphicon-resize-full:before{content:"\e096"}.glyphicon-resize-small:before{content:"\e097"}.glyphicon-exclamation-sign:before{content:"\e101"}.glyphicon-gift:before{content:"\e102"}.glyphicon-leaf:before{content:"\e103"}.glyphicon-fire:before{content:"\e104"}.glyphicon-eye-open:before{content:"\e105"}.glyphicon-eye-close:before{content:"\e106"}.glyphicon-warning-sign:before{content:"\e107"}.glyphicon-plane:before{content:"\e108"}.glyphicon-calendar:before{content:"\e109"}.glyphicon-random:before{content:"\e110"}.glyphicon-comment:before{content:"\e111"}.glyphicon-magnet:before{content:"\e112"}.glyphicon-chevron-up:before{content:"\e113"}.glyphicon-chevron-down:before{content:"\e114"}.glyphicon-retweet:before{content:"\e115"}.glyphicon-shopping-cart:before{content:"\e116"}.glyphicon-folder-close:before{content:"\e117"}.glyphicon-folder-open:before{content:"\e118"}.glyphicon-resize-vertical:before{content:"\e119"}.glyphicon-resize-horizontal:before{content:"\e120"}.glyphicon-hdd:before{content:"\e121"}.glyphicon-bullhorn:before{content:"\e122"}.glyphicon-bell:before{content:"\e123"}.glyphicon-certificate:before{content:"\e124"}.glyphicon-thumbs-up:before{content:"\e125"}.glyphicon-thumbs-down:before{content:"\e126"}.glyphicon-hand-right:before{content:"\e127"}.glyphicon-hand-left:before{content:"\e128"}.glyphicon-hand-up:before{content:"\e129"}.glyphicon-hand-down:before{content:"\e130"}.glyphicon-circle-arrow-right:before{content:"\e131"}.glyphicon-circle-arrow-left:before{content:"\e132"}.glyphicon-circle-arrow-up:before{content:"\e133"}.glyphicon-circle-arrow-down:before{content:"\e134"}.glyphicon-globe:before{content:"\e135"}.glyphicon-wrench:before{content:"\e136"}.glyphicon-tasks:before{content:"\e137"}.glyphicon-filter:before{content:"\e138"}.glyphicon-briefcase:before{content:"\e139"}.glyphicon-fullscreen:before{content:"\e140"}.glyphicon-dashboard:before{content:"\e141"}.glyphicon-paperclip:before{content:"\e142"}.glyphicon-heart-empty:before{content:"\e143"}.glyphicon-link:before{content:"\e144"}.glyphicon-phone:before{content:"\e145"}.glyphicon-pushpin:before{content:"\e146"}.glyphicon-usd:before{content:"\e148"}.glyphicon-gbp:before{content:"\e149"}.glyphicon-sort:before{content:"\e150"}.glyphicon-sort-by-alphabet:before{content:"\e151"}.glyphicon-sort-by-alphabet-alt:before{content:"\e152"}.glyphicon-sort-by-order:before{content:"\e153"}.glyphicon-sort-by-order-alt:before{content:"\e154"}.glyphicon-sort-by-attributes:before{content:"\e155"}.glyphicon-sort-by-attributes-alt:before{content:"\e156"}.glyphicon-unchecked:before{content:"\e157"}.glyphicon-expand:before{content:"\e158"}.glyphicon-collapse-down:before{content:"\e159"}.glyphicon-collapse-up:before{content:"\e160"}.glyphicon-log-in:before{content:"\e161"}.glyphicon-flash:before{content:"\e162"}.glyphicon-log-out:before{content:"\e163"}.glyphicon-new-window:before{content:"\e164"}.glyphicon-record:before{content:"\e165"}.glyphicon-save:before{content:"\e166"}.glyphicon-open:before{content:"\e167"}.glyphicon-saved:before{content:"\e168"}.glyphicon-import:before{content:"\e169"}.glyphicon-export:before{content:"\e170"}.glyphicon-send:before{content:"\e171"}.glyphicon-floppy-disk:before{content:"\e172"}.glyphicon-floppy-saved:before{content:"\e173"}.glyphicon-floppy-remove:before{content:"\e174"}.glyphicon-floppy-save:before{content:"\e175"}.glyphicon-floppy-open:before{content:"\e176"}.glyphicon-credit-card:before{content:"\e177"}.glyphicon-transfer:before{content:"\e178"}.glyphicon-cutlery:before{content:"\e179"}.glyphicon-header:before{content:"\e180"}.glyphicon-compressed:before{content:"\e181"}.glyphicon-earphone:before{content:"\e182"}.glyphicon-phone-alt:before{content:"\e183"}.glyphicon-tower:before{content:"\e184"}.glyphicon-stats:before{content:"\e185"}.glyphicon-sd-video:before{content:"\e186"}.glyphicon-hd-video:before{content:"\e187"}.glyphicon-subtitles:before{content:"\e188"}.glyphicon-sound-stereo:before{content:"\e189"}.glyphicon-sound-dolby:before{content:"\e190"}.glyphicon-sound-5-1:before{content:"\e191"}.glyphicon-sound-6-1:before{content:"\e192"}.glyphicon-sound-7-1:before{content:"\e193"}.glyphicon-copyright-mark:before{content:"\e194"}.glyphicon-registration-mark:before{content:"\e195"}.glyphicon-cloud-download:before{content:"\e197"}.glyphicon-cloud-upload:before{content:"\e198"}.glyphicon-tree-conifer:before{content:"\e199"}.glyphicon-tree-deciduous:before{content:"\e200"}.glyphicon-cd:before{content:"\e201"}.glyphicon-save-file:before{content:"\e202"}.glyphicon-open-file:before{content:"\e203"}.glyphicon-level-up:before{content:"\e204"}.glyphicon-copy:before{content:"\e205"}.glyphicon-paste:before{content:"\e206"}.glyphicon-alert:before{content:"\e209"}.glyphicon-equalizer:before{content:"\e210"}.glyphicon-king:before{content:"\e211"}.glyphicon-queen:before{content:"\e212"}.glyphicon-pawn:before{content:"\e213"}.glyphicon-bishop:before{content:"\e214"}.glyphicon-knight:before{content:"\e215"}.glyphicon-baby-formula:before{content:"\e216"}.glyphicon-tent:before{content:"\26fa"}.glyphicon-blackboard:before{content:"\e218"}.glyphicon-bed:before{content:"\e219"}.glyphicon-apple:before{content:"\f8ff"}.glyphicon-erase:before{content:"\e221"}.glyphicon-hourglass:before{content:"\231b"}.glyphicon-lamp:before{content:"\e223"}.glyphicon-duplicate:before{content:"\e224"}.glyphicon-piggy-bank:before{content:"\e225"}.glyphicon-scissors:before{content:"\e226"}.glyphicon-bitcoin:before{content:"\e227"}.glyphicon-btc:before{content:"\e227"}.glyphicon-xbt:before{content:"\e227"}.glyphicon-yen:before{content:"\00a5"}.glyphicon-jpy:before{content:"\00a5"}.glyphicon-ruble:before{content:"\20bd"}.glyphicon-rub:before{content:"\20bd"}.glyphicon-scale:before{content:"\e230"}.glyphicon-ice-lolly:before{content:"\e231"}.glyphicon-ice-lolly-tasted:before{content:"\e232"}.glyphicon-education:before{content:"\e233"}.glyphicon-option-horizontal:before{content:"\e234"}.glyphicon-option-vertical:before{content:"\e235"}.glyphicon-menu-hamburger:before{content:"\e236"}.glyphicon-modal-window:before{content:"\e237"}.glyphicon-oil:before{content:"\e238"}.glyphicon-grain:before{content:"\e239"}.glyphicon-sunglasses:before{content:"\e240"}.glyphicon-text-size:before{content:"\e241"}.glyphicon-text-color:before{content:"\e242"}.glyphicon-text-background:before{content:"\e243"}.glyphicon-object-align-top:before{content:"\e244"}.glyphicon-object-align-bottom:before{content:"\e245"}.glyphicon-object-align-horizontal:before{content:"\e246"}.glyphicon-object-align-left:before{content:"\e247"}.glyphicon-object-align-vertical:before{content:"\e248"}.glyphicon-object-align-right:before{content:"\e249"}.glyphicon-triangle-right:before{content:"\e250"}.glyphicon-triangle-left:before{content:"\e251"}.glyphicon-triangle-bottom:before{content:"\e252"}.glyphicon-triangle-top:before{content:"\e253"}.glyphicon-console:before{content:"\e254"}.glyphicon-superscript:before{content:"\e255"}.glyphicon-subscript:before{content:"\e256"}.glyphicon-menu-left:before{content:"\e257"}.glyphicon-menu-right:before{content:"\e258"}.glyphicon-menu-down:before{content:"\e259"}.glyphicon-menu-up:before{content:"\e260"}*{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}:after,:before{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}html{font-size:10px;-webkit-tap-highlight-color:rgba(0,0,0,0)}body{font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;line-height:1.42857143;color:#333;background-color:#fff}button,input,select,textarea{font-family:inherit;font-size:inherit;line-height:inherit}a{color:#337ab7;text-decoration:none}a:focus,a:hover{color:#23527c;text-decoration:underline}a:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}figure{margin:0}img{vertical-align:middle}.carousel-inner>.item>a>img,.carousel-inner>.item>img,.img-responsive,.thumbnail a>img,.thumbnail>img{display:block;max-width:100%;height:auto}.img-rounded{border-radius:6px}.img-thumbnail{display:inline-block;max-width:100%;height:auto;padding:4px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:all .2s ease-in-out;-o-transition:all .2s ease-in-out;transition:all .2s ease-in-out}.img-circle{border-radius:50%}hr{margin-top:20px;margin-bottom:20px;border:0;border-top:1px solid #eee}.sr-only{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0,0,0,0);border:0}.sr-only-focusable:active,.sr-only-focusable:focus{position:static;width:auto;height:auto;margin:0;overflow:visible;clip:auto}[role=button]{cursor:pointer}.h1,.h2,.h3,.h4,.h5,.h6,h1,h2,h3,h4,h5,h6{font-family:inherit;font-weight:500;line-height:1.1;color:inherit}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-weight:400;line-height:1;color:#777}.h1,.h2,.h3,h1,h2,h3{margin-top:20px;margin-bottom:10px}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small{font-size:65%}.h4,.h5,.h6,h4,h5,h6{margin-top:10px;margin-bottom:10px}.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-size:75%}.h1,h1{font-size:36px}.h2,h2{font-size:30px}.h3,h3{font-size:24px}.h4,h4{font-size:18px}.h5,h5{font-size:14px}.h6,h6{font-size:12px}p{margin:0 0 10px}.lead{margin-bottom:20px;font-size:16px;font-weight:300;line-height:1.4}@media (min-width:768px){.lead{font-size:21px}}.small,small{font-size:85%}.mark,mark{padding:.2em;background-color:#fcf8e3}.text-left{text-align:left}.text-right{text-align:right}.text-center{text-align:center}.text-justify{text-align:justify}.text-nowrap{white-space:nowrap}.text-lowercase{text-transform:lowercase}.text-uppercase{text-transform:uppercase}.text-capitalize{text-transform:capitalize}.text-muted{color:#777}.text-primary{color:#337ab7}a.text-primary:focus,a.text-primary:hover{color:#286090}.text-success{color:#3c763d}a.text-success:focus,a.text-success:hover{color:#2b542c}.text-info{color:#31708f}a.text-info:focus,a.text-info:hover{color:#245269}.text-warning{color:#8a6d3b}a.text-warning:focus,a.text-warning:hover{color:#66512c}.text-danger{color:#a94442}a.text-danger:focus,a.text-danger:hover{color:#843534}.bg-primary{color:#fff;background-color:#337ab7}a.bg-primary:focus,a.bg-primary:hover{background-color:#286090}.bg-success{background-color:#dff0d8}a.bg-success:focus,a.bg-success:hover{background-color:#c1e2b3}.bg-info{background-color:#d9edf7}a.bg-info:focus,a.bg-info:hover{background-color:#afd9ee}.bg-warning{background-color:#fcf8e3}a.bg-warning:focus,a.bg-warning:hover{background-color:#f7ecb5}.bg-danger{background-color:#f2dede}a.bg-danger:focus,a.bg-danger:hover{background-color:#e4b9b9}.page-header{padding-bottom:9px;margin:40px 0 20px;border-bottom:1px solid #eee}ol,ul{margin-top:0;margin-bottom:10px}ol ol,ol ul,ul ol,ul ul{margin-bottom:0}.list-unstyled{padding-left:0;list-style:none}.list-inline{padding-left:0;margin-left:-5px;list-style:none}.list-inline>li{display:inline-block;padding-right:5px;padding-left:5px}dl{margin-top:0;margin-bottom:20px}dd,dt{line-height:1.42857143}dt{font-weight:700}dd{margin-left:0}@media (min-width:768px){.dl-horizontal dt{float:left;width:160px;overflow:hidden;clear:left;text-align:right;text-overflow:ellipsis;white-space:nowrap}.dl-horizontal dd{margin-left:180px}}abbr[data-original-title],abbr[title]{cursor:help;border-bottom:1px dotted #777}.initialism{font-size:90%;text-transform:uppercase}blockquote{padding:10px 20px;margin:0 0 20px;font-size:17.5px;border-left:5px solid #eee}blockquote ol:last-child,blockquote p:last-child,blockquote ul:last-child{margin-bottom:0}blockquote .small,blockquote footer,blockquote small{display:block;font-size:80%;line-height:1.42857143;color:#777}blockquote .small:before,blockquote footer:before,blockquote small:before{content:'\2014 \00A0'}.blockquote-reverse,blockquote.pull-right{padding-right:15px;padding-left:0;text-align:right;border-right:5px solid #eee;border-left:0}.blockquote-reverse .small:before,.blockquote-reverse footer:before,.blockquote-reverse small:before,blockquote.pull-right .small:before,blockquote.pull-right footer:before,blockquote.pull-right small:before{content:''}.blockquote-reverse .small:after,.blockquote-reverse footer:after,.blockquote-reverse small:after,blockquote.pull-right .small:after,blockquote.pull-right footer:after,blockquote.pull-right small:after{content:'\00A0 \2014'}address{margin-bottom:20px;font-style:normal;line-height:1.42857143}code,kbd,pre,samp{font-family:Menlo,Monaco,Consolas,"Courier New",monospace}code{padding:2px 4px;font-size:90%;color:#c7254e;background-color:#f9f2f4;border-radius:4px}kbd{padding:2px 4px;font-size:90%;color:#fff;background-color:#333;border-radius:3px;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.25);box-shadow:inset 0 -1px 0 rgba(0,0,0,.25)}kbd kbd{padding:0;font-size:100%;font-weight:700;-webkit-box-shadow:none;box-shadow:none}pre{display:block;padding:9.5px;margin:0 0 10px;font-size:13px;line-height:1.42857143;color:#333;word-break:break-all;word-wrap:break-word;background-color:#f5f5f5;border:1px solid #ccc;border-radius:4px}pre code{padding:0;font-size:inherit;color:inherit;white-space:pre-wrap;background-color:transparent;border-radius:0}.pre-scrollable{max-height:340px;overflow-y:scroll}.container{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}@media (min-width:768px){.container{width:750px}}@media (min-width:992px){.container{width:970px}}@media (min-width:1200px){.container{width:1170px}}.container-fluid{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}.row{margin-right:-15px;margin-left:-15px}.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9,.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9,.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9,.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{position:relative;min-height:1px;padding-right:15px;padding-left:15px}.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{float:left}.col-xs-12{width:100%}.col-xs-11{width:91.66666667%}.col-xs-10{width:83.33333333%}.col-xs-9{width:75%}.col-xs-8{width:66.66666667%}.col-xs-7{width:58.33333333%}.col-xs-6{width:50%}.col-xs-5{width:41.66666667%}.col-xs-4{width:33.33333333%}.col-xs-3{width:25%}.col-xs-2{width:16.66666667%}.col-xs-1{width:8.33333333%}.col-xs-pull-12{right:100%}.col-xs-pull-11{right:91.66666667%}.col-xs-pull-10{right:83.33333333%}.col-xs-pull-9{right:75%}.col-xs-pull-8{right:66.66666667%}.col-xs-pull-7{right:58.33333333%}.col-xs-pull-6{right:50%}.col-xs-pull-5{right:41.66666667%}.col-xs-pull-4{right:33.33333333%}.col-xs-pull-3{right:25%}.col-xs-pull-2{right:16.66666667%}.col-xs-pull-1{right:8.33333333%}.col-xs-pull-0{right:auto}.col-xs-push-12{left:100%}.col-xs-push-11{left:91.66666667%}.col-xs-push-10{left:83.33333333%}.col-xs-push-9{left:75%}.col-xs-push-8{left:66.66666667%}.col-xs-push-7{left:58.33333333%}.col-xs-push-6{left:50%}.col-xs-push-5{left:41.66666667%}.col-xs-push-4{left:33.33333333%}.col-xs-push-3{left:25%}.col-xs-push-2{left:16.66666667%}.col-xs-push-1{left:8.33333333%}.col-xs-push-0{left:auto}.col-xs-offset-12{margin-left:100%}.col-xs-offset-11{margin-left:91.66666667%}.col-xs-offset-10{margin-left:83.33333333%}.col-xs-offset-9{margin-left:75%}.col-xs-offset-8{margin-left:66.66666667%}.col-xs-offset-7{margin-left:58.33333333%}.col-xs-offset-6{margin-left:50%}.col-xs-offset-5{margin-left:41.66666667%}.col-xs-offset-4{margin-left:33.33333333%}.col-xs-offset-3{margin-left:25%}.col-xs-offset-2{margin-left:16.66666667%}.col-xs-offset-1{margin-left:8.33333333%}.col-xs-offset-0{margin-left:0}@media (min-width:768px){.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9{float:left}.col-sm-12{width:100%}.col-sm-11{width:91.66666667%}.col-sm-10{width:83.33333333%}.col-sm-9{width:75%}.col-sm-8{width:66.66666667%}.col-sm-7{width:58.33333333%}.col-sm-6{width:50%}.col-sm-5{width:41.66666667%}.col-sm-4{width:33.33333333%}.col-sm-3{width:25%}.col-sm-2{width:16.66666667%}.col-sm-1{width:8.33333333%}.col-sm-pull-12{right:100%}.col-sm-pull-11{right:91.66666667%}.col-sm-pull-10{right:83.33333333%}.col-sm-pull-9{right:75%}.col-sm-pull-8{right:66.66666667%}.col-sm-pull-7{right:58.33333333%}.col-sm-pull-6{right:50%}.col-sm-pull-5{right:41.66666667%}.col-sm-pull-4{right:33.33333333%}.col-sm-pull-3{right:25%}.col-sm-pull-2{right:16.66666667%}.col-sm-pull-1{right:8.33333333%}.col-sm-pull-0{right:auto}.col-sm-push-12{left:100%}.col-sm-push-11{left:91.66666667%}.col-sm-push-10{left:83.33333333%}.col-sm-push-9{left:75%}.col-sm-push-8{left:66.66666667%}.col-sm-push-7{left:58.33333333%}.col-sm-push-6{left:50%}.col-sm-push-5{left:41.66666667%}.col-sm-push-4{left:33.33333333%}.col-sm-push-3{left:25%}.col-sm-push-2{left:16.66666667%}.col-sm-push-1{left:8.33333333%}.col-sm-push-0{left:auto}.col-sm-offset-12{margin-left:100%}.col-sm-offset-11{margin-left:91.66666667%}.col-sm-offset-10{margin-left:83.33333333%}.col-sm-offset-9{margin-left:75%}.col-sm-offset-8{margin-left:66.66666667%}.col-sm-offset-7{margin-left:58.33333333%}.col-sm-offset-6{margin-left:50%}.col-sm-offset-5{margin-left:41.66666667%}.col-sm-offset-4{margin-left:33.33333333%}.col-sm-offset-3{margin-left:25%}.col-sm-offset-2{margin-left:16.66666667%}.col-sm-offset-1{margin-left:8.33333333%}.col-sm-offset-0{margin-left:0}}@media (min-width:992px){.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9{float:left}.col-md-12{width:100%}.col-md-11{width:91.66666667%}.col-md-10{width:83.33333333%}.col-md-9{width:75%}.col-md-8{width:66.66666667%}.col-md-7{width:58.33333333%}.col-md-6{width:50%}.col-md-5{width:41.66666667%}.col-md-4{width:33.33333333%}.col-md-3{width:25%}.col-md-2{width:16.66666667%}.col-md-1{width:8.33333333%}.col-md-pull-12{right:100%}.col-md-pull-11{right:91.66666667%}.col-md-pull-10{right:83.33333333%}.col-md-pull-9{right:75%}.col-md-pull-8{right:66.66666667%}.col-md-pull-7{right:58.33333333%}.col-md-pull-6{right:50%}.col-md-pull-5{right:41.66666667%}.col-md-pull-4{right:33.33333333%}.col-md-pull-3{right:25%}.col-md-pull-2{right:16.66666667%}.col-md-pull-1{right:8.33333333%}.col-md-pull-0{right:auto}.col-md-push-12{left:100%}.col-md-push-11{left:91.66666667%}.col-md-push-10{left:83.33333333%}.col-md-push-9{left:75%}.col-md-push-8{left:66.66666667%}.col-md-push-7{left:58.33333333%}.col-md-push-6{left:50%}.col-md-push-5{left:41.66666667%}.col-md-push-4{left:33.33333333%}.col-md-push-3{left:25%}.col-md-push-2{left:16.66666667%}.col-md-push-1{left:8.33333333%}.col-md-push-0{left:auto}.col-md-offset-12{margin-left:100%}.col-md-offset-11{margin-left:91.66666667%}.col-md-offset-10{margin-left:83.33333333%}.col-md-offset-9{margin-left:75%}.col-md-offset-8{margin-left:66.66666667%}.col-md-offset-7{margin-left:58.33333333%}.col-md-offset-6{margin-left:50%}.col-md-offset-5{margin-left:41.66666667%}.col-md-offset-4{margin-left:33.33333333%}.col-md-offset-3{margin-left:25%}.col-md-offset-2{margin-left:16.66666667%}.col-md-offset-1{margin-left:8.33333333%}.col-md-offset-0{margin-left:0}}@media (min-width:1200px){.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9{float:left}.col-lg-12{width:100%}.col-lg-11{width:91.66666667%}.col-lg-10{width:83.33333333%}.col-lg-9{width:75%}.col-lg-8{width:66.66666667%}.col-lg-7{width:58.33333333%}.col-lg-6{width:50%}.col-lg-5{width:41.66666667%}.col-lg-4{width:33.33333333%}.col-lg-3{width:25%}.col-lg-2{width:16.66666667%}.col-lg-1{width:8.33333333%}.col-lg-pull-12{right:100%}.col-lg-pull-11{right:91.66666667%}.col-lg-pull-10{right:83.33333333%}.col-lg-pull-9{right:75%}.col-lg-pull-8{right:66.66666667%}.col-lg-pull-7{right:58.33333333%}.col-lg-pull-6{right:50%}.col-lg-pull-5{right:41.66666667%}.col-lg-pull-4{right:33.33333333%}.col-lg-pull-3{right:25%}.col-lg-pull-2{right:16.66666667%}.col-lg-pull-1{right:8.33333333%}.col-lg-pull-0{right:auto}.col-lg-push-12{left:100%}.col-lg-push-11{left:91.66666667%}.col-lg-push-10{left:83.33333333%}.col-lg-push-9{left:75%}.col-lg-push-8{left:66.66666667%}.col-lg-push-7{left:58.33333333%}.col-lg-push-6{left:50%}.col-lg-push-5{left:41.66666667%}.col-lg-push-4{left:33.33333333%}.col-lg-push-3{left:25%}.col-lg-push-2{left:16.66666667%}.col-lg-push-1{left:8.33333333%}.col-lg-push-0{left:auto}.col-lg-offset-12{margin-left:100%}.col-lg-offset-11{margin-left:91.66666667%}.col-lg-offset-10{margin-left:83.33333333%}.col-lg-offset-9{margin-left:75%}.col-lg-offset-8{margin-left:66.66666667%}.col-lg-offset-7{margin-left:58.33333333%}.col-lg-offset-6{margin-left:50%}.col-lg-offset-5{margin-left:41.66666667%}.col-lg-offset-4{margin-left:33.33333333%}.col-lg-offset-3{margin-left:25%}.col-lg-offset-2{margin-left:16.66666667%}.col-lg-offset-1{margin-left:8.33333333%}.col-lg-offset-0{margin-left:0}}table{background-color:transparent}caption{padding-top:8px;padding-bottom:8px;color:#777;text-align:left}th{text-align:left}.table{width:100%;max-width:100%;margin-bottom:20px}.table>tbody>tr>td,.table>tbody>tr>th,.table>tfoot>tr>td,.table>tfoot>tr>th,.table>thead>tr>td,.table>thead>tr>th{padding:8px;line-height:1.42857143;vertical-align:top;border-top:1px solid #ddd}.table>thead>tr>th{vertical-align:bottom;border-bottom:2px solid #ddd}.table>caption+thead>tr:first-child>td,.table>caption+thead>tr:first-child>th,.table>colgroup+thead>tr:first-child>td,.table>colgroup+thead>tr:first-child>th,.table>thead:first-child>tr:first-child>td,.table>thead:first-child>tr:first-child>th{border-top:0}.table>tbody+tbody{border-top:2px solid #ddd}.table .table{background-color:#fff}.table-condensed>tbody>tr>td,.table-condensed>tbody>tr>th,.table-condensed>tfoot>tr>td,.table-condensed>tfoot>tr>th,.table-condensed>thead>tr>td,.table-condensed>thead>tr>th{padding:5px}.table-bordered{border:1px solid #ddd}.table-bordered>tbody>tr>td,.table-bordered>tbody>tr>th,.table-bordered>tfoot>tr>td,.table-bordered>tfoot>tr>th,.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border:1px solid #ddd}.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border-bottom-width:2px}.table-striped>tbody>tr:nth-of-type(odd){background-color:#f9f9f9}.table-hover>tbody>tr:hover{background-color:#f5f5f5}table col[class*=col-]{position:static;display:table-column;float:none}table td[class*=col-],table th[class*=col-]{position:static;display:table-cell;float:none}.table>tbody>tr.active>td,.table>tbody>tr.active>th,.table>tbody>tr>td.active,.table>tbody>tr>th.active,.table>tfoot>tr.active>td,.table>tfoot>tr.active>th,.table>tfoot>tr>td.active,.table>tfoot>tr>th.active,.table>thead>tr.active>td,.table>thead>tr.active>th,.table>thead>tr>td.active,.table>thead>tr>th.active{background-color:#f5f5f5}.table-hover>tbody>tr.active:hover>td,.table-hover>tbody>tr.active:hover>th,.table-hover>tbody>tr:hover>.active,.table-hover>tbody>tr>td.active:hover,.table-hover>tbody>tr>th.active:hover{background-color:#e8e8e8}.table>tbody>tr.success>td,.table>tbody>tr.success>th,.table>tbody>tr>td.success,.table>tbody>tr>th.success,.table>tfoot>tr.success>td,.table>tfoot>tr.success>th,.table>tfoot>tr>td.success,.table>tfoot>tr>th.success,.table>thead>tr.success>td,.table>thead>tr.success>th,.table>thead>tr>td.success,.table>thead>tr>th.success{background-color:#dff0d8}.table-hover>tbody>tr.success:hover>td,.table-hover>tbody>tr.success:hover>th,.table-hover>tbody>tr:hover>.success,.table-hover>tbody>tr>td.success:hover,.table-hover>tbody>tr>th.success:hover{background-color:#d0e9c6}.table>tbody>tr.info>td,.table>tbody>tr.info>th,.table>tbody>tr>td.info,.table>tbody>tr>th.info,.table>tfoot>tr.info>td,.table>tfoot>tr.info>th,.table>tfoot>tr>td.info,.table>tfoot>tr>th.info,.table>thead>tr.info>td,.table>thead>tr.info>th,.table>thead>tr>td.info,.table>thead>tr>th.info{background-color:#d9edf7}.table-hover>tbody>tr.info:hover>td,.table-hover>tbody>tr.info:hover>th,.table-hover>tbody>tr:hover>.info,.table-hover>tbody>tr>td.info:hover,.table-hover>tbody>tr>th.info:hover{background-color:#c4e3f3}.table>tbody>tr.warning>td,.table>tbody>tr.warning>th,.table>tbody>tr>td.warning,.table>tbody>tr>th.warning,.table>tfoot>tr.warning>td,.table>tfoot>tr.warning>th,.table>tfoot>tr>td.warning,.table>tfoot>tr>th.warning,.table>thead>tr.warning>td,.table>thead>tr.warning>th,.table>thead>tr>td.warning,.table>thead>tr>th.warning{background-color:#fcf8e3}.table-hover>tbody>tr.warning:hover>td,.table-hover>tbody>tr.warning:hover>th,.table-hover>tbody>tr:hover>.warning,.table-hover>tbody>tr>td.warning:hover,.table-hover>tbody>tr>th.warning:hover{background-color:#faf2cc}.table>tbody>tr.danger>td,.table>tbody>tr.danger>th,.table>tbody>tr>td.danger,.table>tbody>tr>th.danger,.table>tfoot>tr.danger>td,.table>tfoot>tr.danger>th,.table>tfoot>tr>td.danger,.table>tfoot>tr>th.danger,.table>thead>tr.danger>td,.table>thead>tr.danger>th,.table>thead>tr>td.danger,.table>thead>tr>th.danger{background-color:#f2dede}.table-hover>tbody>tr.danger:hover>td,.table-hover>tbody>tr.danger:hover>th,.table-hover>tbody>tr:hover>.danger,.table-hover>tbody>tr>td.danger:hover,.table-hover>tbody>tr>th.danger:hover{background-color:#ebcccc}.table-responsive{min-height:.01%;overflow-x:auto}@media screen and (max-width:767px){.table-responsive{width:100%;margin-bottom:15px;overflow-y:hidden;-ms-overflow-style:-ms-autohiding-scrollbar;border:1px solid #ddd}.table-responsive>.table{margin-bottom:0}.table-responsive>.table>tbody>tr>td,.table-responsive>.table>tbody>tr>th,.table-responsive>.table>tfoot>tr>td,.table-responsive>.table>tfoot>tr>th,.table-responsive>.table>thead>tr>td,.table-responsive>.table>thead>tr>th{white-space:nowrap}.table-responsive>.table-bordered{border:0}.table-responsive>.table-bordered>tbody>tr>td:first-child,.table-responsive>.table-bordered>tbody>tr>th:first-child,.table-responsive>.table-bordered>tfoot>tr>td:first-child,.table-responsive>.table-bordered>tfoot>tr>th:first-child,.table-responsive>.table-bordered>thead>tr>td:first-child,.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.table-responsive>.table-bordered>tbody>tr>td:last-child,.table-responsive>.table-bordered>tbody>tr>th:last-child,.table-responsive>.table-bordered>tfoot>tr>td:last-child,.table-responsive>.table-bordered>tfoot>tr>th:last-child,.table-responsive>.table-bordered>thead>tr>td:last-child,.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.table-responsive>.table-bordered>tbody>tr:last-child>td,.table-responsive>.table-bordered>tbody>tr:last-child>th,.table-responsive>.table-bordered>tfoot>tr:last-child>td,.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}}fieldset{min-width:0;padding:0;margin:0;border:0}legend{display:block;width:100%;padding:0;margin-bottom:20px;font-size:21px;line-height:inherit;color:#333;border:0;border-bottom:1px solid #e5e5e5}label{display:inline-block;max-width:100%;margin-bottom:5px;font-weight:700}input[type=search]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}input[type=checkbox],input[type=radio]{margin:4px 0 0;margin-top:1px\9;line-height:normal}input[type=file]{display:block}input[type=range]{display:block;width:100%}select[multiple],select[size]{height:auto}input[type=file]:focus,input[type=checkbox]:focus,input[type=radio]:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}output{display:block;padding-top:7px;font-size:14px;line-height:1.42857143;color:#555}.form-control{display:block;width:100%;height:34px;padding:6px 12px;font-size:14px;line-height:1.42857143;color:#555;background-color:#fff;background-image:none;border:1px solid #ccc;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075);-webkit-transition:border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;-o-transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s;transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s}.form-control:focus{border-color:#66afe9;outline:0;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6);box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6)}.form-control::-moz-placeholder{color:#999;opacity:1}.form-control:-ms-input-placeholder{color:#999}.form-control::-webkit-input-placeholder{color:#999}.form-control::-ms-expand{background-color:transparent;border:0}.form-control[disabled],.form-control[readonly],fieldset[disabled] .form-control{background-color:#eee;opacity:1}.form-control[disabled],fieldset[disabled] .form-control{cursor:not-allowed}textarea.form-control{height:auto}input[type=search]{-webkit-appearance:none}@media screen and (-webkit-min-device-pixel-ratio:0){input[type=date].form-control,input[type=time].form-control,input[type=datetime-local].form-control,input[type=month].form-control{line-height:34px}.input-group-sm input[type=date],.input-group-sm input[type=time],.input-group-sm input[type=datetime-local],.input-group-sm input[type=month],input[type=date].input-sm,input[type=time].input-sm,input[type=datetime-local].input-sm,input[type=month].input-sm{line-height:30px}.input-group-lg input[type=date],.input-group-lg input[type=time],.input-group-lg input[type=datetime-local],.input-group-lg input[type=month],input[type=date].input-lg,input[type=time].input-lg,input[type=datetime-local].input-lg,input[type=month].input-lg{line-height:46px}}.form-group{margin-bottom:15px}.checkbox,.radio{position:relative;display:block;margin-top:10px;margin-bottom:10px}.checkbox label,.radio label{min-height:20px;padding-left:20px;margin-bottom:0;font-weight:400;cursor:pointer}.checkbox input[type=checkbox],.checkbox-inline input[type=checkbox],.radio input[type=radio],.radio-inline input[type=radio]{position:absolute;margin-top:4px\9;margin-left:-20px}.checkbox+.checkbox,.radio+.radio{margin-top:-5px}.checkbox-inline,.radio-inline{position:relative;display:inline-block;padding-left:20px;margin-bottom:0;font-weight:400;vertical-align:middle;cursor:pointer}.checkbox-inline+.checkbox-inline,.radio-inline+.radio-inline{margin-top:0;margin-left:10px}fieldset[disabled] input[type=checkbox],fieldset[disabled] input[type=radio],input[type=checkbox].disabled,input[type=checkbox][disabled],input[type=radio].disabled,input[type=radio][disabled]{cursor:not-allowed}.checkbox-inline.disabled,.radio-inline.disabled,fieldset[disabled] .checkbox-inline,fieldset[disabled] .radio-inline{cursor:not-allowed}.checkbox.disabled label,.radio.disabled label,fieldset[disabled] .checkbox label,fieldset[disabled] .radio label{cursor:not-allowed}.form-control-static{min-height:34px;padding-top:7px;padding-bottom:7px;margin-bottom:0}.form-control-static.input-lg,.form-control-static.input-sm{padding-right:0;padding-left:0}.input-sm{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-sm{height:30px;line-height:30px}select[multiple].input-sm,textarea.input-sm{height:auto}.form-group-sm .form-control{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.form-group-sm select.form-control{height:30px;line-height:30px}.form-group-sm select[multiple].form-control,.form-group-sm textarea.form-control{height:auto}.form-group-sm .form-control-static{height:30px;min-height:32px;padding:6px 10px;font-size:12px;line-height:1.5}.input-lg{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-lg{height:46px;line-height:46px}select[multiple].input-lg,textarea.input-lg{height:auto}.form-group-lg .form-control{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.form-group-lg select.form-control{height:46px;line-height:46px}.form-group-lg select[multiple].form-control,.form-group-lg textarea.form-control{height:auto}.form-group-lg .form-control-static{height:46px;min-height:38px;padding:11px 16px;font-size:18px;line-height:1.3333333}.has-feedback{position:relative}.has-feedback .form-control{padding-right:42.5px}.form-control-feedback{position:absolute;top:0;right:0;z-index:2;display:block;width:34px;height:34px;line-height:34px;text-align:center;pointer-events:none}.form-group-lg .form-control+.form-control-feedback,.input-group-lg+.form-control-feedback,.input-lg+.form-control-feedback{width:46px;height:46px;line-height:46px}.form-group-sm .form-control+.form-control-feedback,.input-group-sm+.form-control-feedback,.input-sm+.form-control-feedback{width:30px;height:30px;line-height:30px}.has-success .checkbox,.has-success .checkbox-inline,.has-success .control-label,.has-success .help-block,.has-success .radio,.has-success .radio-inline,.has-success.checkbox label,.has-success.checkbox-inline label,.has-success.radio label,.has-success.radio-inline label{color:#3c763d}.has-success .form-control{border-color:#3c763d;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-success .form-control:focus{border-color:#2b542c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168}.has-success .input-group-addon{color:#3c763d;background-color:#dff0d8;border-color:#3c763d}.has-success .form-control-feedback{color:#3c763d}.has-warning .checkbox,.has-warning .checkbox-inline,.has-warning .control-label,.has-warning .help-block,.has-warning .radio,.has-warning .radio-inline,.has-warning.checkbox label,.has-warning.checkbox-inline label,.has-warning.radio label,.has-warning.radio-inline label{color:#8a6d3b}.has-warning .form-control{border-color:#8a6d3b;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-warning .form-control:focus{border-color:#66512c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b}.has-warning .input-group-addon{color:#8a6d3b;background-color:#fcf8e3;border-color:#8a6d3b}.has-warning .form-control-feedback{color:#8a6d3b}.has-error .checkbox,.has-error .checkbox-inline,.has-error .control-label,.has-error .help-block,.has-error .radio,.has-error .radio-inline,.has-error.checkbox label,.has-error.checkbox-inline label,.has-error.radio label,.has-error.radio-inline label{color:#a94442}.has-error .form-control{border-color:#a94442;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-error .form-control:focus{border-color:#843534;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483}.has-error .input-group-addon{color:#a94442;background-color:#f2dede;border-color:#a94442}.has-error .form-control-feedback{color:#a94442}.has-feedback label~.form-control-feedback{top:25px}.has-feedback label.sr-only~.form-control-feedback{top:0}.help-block{display:block;margin-top:5px;margin-bottom:10px;color:#737373}@media (min-width:768px){.form-inline .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.form-inline .form-control{display:inline-block;width:auto;vertical-align:middle}.form-inline .form-control-static{display:inline-block}.form-inline .input-group{display:inline-table;vertical-align:middle}.form-inline .input-group .form-control,.form-inline .input-group .input-group-addon,.form-inline .input-group .input-group-btn{width:auto}.form-inline .input-group>.form-control{width:100%}.form-inline .control-label{margin-bottom:0;vertical-align:middle}.form-inline .checkbox,.form-inline .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.form-inline .checkbox label,.form-inline .radio label{padding-left:0}.form-inline .checkbox input[type=checkbox],.form-inline .radio input[type=radio]{position:relative;margin-left:0}.form-inline .has-feedback .form-control-feedback{top:0}}.form-horizontal .checkbox,.form-horizontal .checkbox-inline,.form-horizontal .radio,.form-horizontal .radio-inline{padding-top:7px;margin-top:0;margin-bottom:0}.form-horizontal .checkbox,.form-horizontal .radio{min-height:27px}.form-horizontal .form-group{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.form-horizontal .control-label{padding-top:7px;margin-bottom:0;text-align:right}}.form-horizontal .has-feedback .form-control-feedback{right:15px}@media (min-width:768px){.form-horizontal .form-group-lg .control-label{padding-top:11px;font-size:18px}}@media (min-width:768px){.form-horizontal .form-group-sm .control-label{padding-top:6px;font-size:12px}}.btn{display:inline-block;padding:6px 12px;margin-bottom:0;font-size:14px;font-weight:400;line-height:1.42857143;text-align:center;white-space:nowrap;vertical-align:middle;-ms-touch-action:manipulation;touch-action:manipulation;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;background-image:none;border:1px solid transparent;border-radius:4px}.btn.active.focus,.btn.active:focus,.btn.focus,.btn:active.focus,.btn:active:focus,.btn:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}.btn.focus,.btn:focus,.btn:hover{color:#333;text-decoration:none}.btn.active,.btn:active{background-image:none;outline:0;-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn.disabled,.btn[disabled],fieldset[disabled] .btn{cursor:not-allowed;filter:alpha(opacity=65);-webkit-box-shadow:none;box-shadow:none;opacity:.65}a.btn.disabled,fieldset[disabled] a.btn{pointer-events:none}.btn-default{color:#333;background-color:#fff;border-color:#ccc}.btn-default.focus,.btn-default:focus{color:#333;background-color:#e6e6e6;border-color:#8c8c8c}.btn-default:hover{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active.focus,.btn-default.active:focus,.btn-default.active:hover,.btn-default:active.focus,.btn-default:active:focus,.btn-default:active:hover,.open>.dropdown-toggle.btn-default.focus,.open>.dropdown-toggle.btn-default:focus,.open>.dropdown-toggle.btn-default:hover{color:#333;background-color:#d4d4d4;border-color:#8c8c8c}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{background-image:none}.btn-default.disabled.focus,.btn-default.disabled:focus,.btn-default.disabled:hover,.btn-default[disabled].focus,.btn-default[disabled]:focus,.btn-default[disabled]:hover,fieldset[disabled] .btn-default.focus,fieldset[disabled] .btn-default:focus,fieldset[disabled] .btn-default:hover{background-color:#fff;border-color:#ccc}.btn-default .badge{color:#fff;background-color:#333}.btn-primary{color:#fff;background-color:#337ab7;border-color:#2e6da4}.btn-primary.focus,.btn-primary:focus{color:#fff;background-color:#286090;border-color:#122b40}.btn-primary:hover{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active.focus,.btn-primary.active:focus,.btn-primary.active:hover,.btn-primary:active.focus,.btn-primary:active:focus,.btn-primary:active:hover,.open>.dropdown-toggle.btn-primary.focus,.open>.dropdown-toggle.btn-primary:focus,.open>.dropdown-toggle.btn-primary:hover{color:#fff;background-color:#204d74;border-color:#122b40}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{background-image:none}.btn-primary.disabled.focus,.btn-primary.disabled:focus,.btn-primary.disabled:hover,.btn-primary[disabled].focus,.btn-primary[disabled]:focus,.btn-primary[disabled]:hover,fieldset[disabled] .btn-primary.focus,fieldset[disabled] .btn-primary:focus,fieldset[disabled] .btn-primary:hover{background-color:#337ab7;border-color:#2e6da4}.btn-primary .badge{color:#337ab7;background-color:#fff}.btn-success{color:#fff;background-color:#5cb85c;border-color:#4cae4c}.btn-success.focus,.btn-success:focus{color:#fff;background-color:#449d44;border-color:#255625}.btn-success:hover{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active.focus,.btn-success.active:focus,.btn-success.active:hover,.btn-success:active.focus,.btn-success:active:focus,.btn-success:active:hover,.open>.dropdown-toggle.btn-success.focus,.open>.dropdown-toggle.btn-success:focus,.open>.dropdown-toggle.btn-success:hover{color:#fff;background-color:#398439;border-color:#255625}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{background-image:none}.btn-success.disabled.focus,.btn-success.disabled:focus,.btn-success.disabled:hover,.btn-success[disabled].focus,.btn-success[disabled]:focus,.btn-success[disabled]:hover,fieldset[disabled] .btn-success.focus,fieldset[disabled] .btn-success:focus,fieldset[disabled] .btn-success:hover{background-color:#5cb85c;border-color:#4cae4c}.btn-success .badge{color:#5cb85c;background-color:#fff}.btn-info{color:#fff;background-color:#5bc0de;border-color:#46b8da}.btn-info.focus,.btn-info:focus{color:#fff;background-color:#31b0d5;border-color:#1b6d85}.btn-info:hover{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active.focus,.btn-info.active:focus,.btn-info.active:hover,.btn-info:active.focus,.btn-info:active:focus,.btn-info:active:hover,.open>.dropdown-toggle.btn-info.focus,.open>.dropdown-toggle.btn-info:focus,.open>.dropdown-toggle.btn-info:hover{color:#fff;background-color:#269abc;border-color:#1b6d85}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{background-image:none}.btn-info.disabled.focus,.btn-info.disabled:focus,.btn-info.disabled:hover,.btn-info[disabled].focus,.btn-info[disabled]:focus,.btn-info[disabled]:hover,fieldset[disabled] .btn-info.focus,fieldset[disabled] .btn-info:focus,fieldset[disabled] .btn-info:hover{background-color:#5bc0de;border-color:#46b8da}.btn-info .badge{color:#5bc0de;background-color:#fff}.btn-warning{color:#fff;background-color:#f0ad4e;border-color:#eea236}.btn-warning.focus,.btn-warning:focus{color:#fff;background-color:#ec971f;border-color:#985f0d}.btn-warning:hover{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active.focus,.btn-warning.active:focus,.btn-warning.active:hover,.btn-warning:active.focus,.btn-warning:active:focus,.btn-warning:active:hover,.open>.dropdown-toggle.btn-warning.focus,.open>.dropdown-toggle.btn-warning:focus,.open>.dropdown-toggle.btn-warning:hover{color:#fff;background-color:#d58512;border-color:#985f0d}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{background-image:none}.btn-warning.disabled.focus,.btn-warning.disabled:focus,.btn-warning.disabled:hover,.btn-warning[disabled].focus,.btn-warning[disabled]:focus,.btn-warning[disabled]:hover,fieldset[disabled] .btn-warning.focus,fieldset[disabled] .btn-warning:focus,fieldset[disabled] .btn-warning:hover{background-color:#f0ad4e;border-color:#eea236}.btn-warning .badge{color:#f0ad4e;background-color:#fff}.btn-danger{color:#fff;background-color:#d9534f;border-color:#d43f3a}.btn-danger.focus,.btn-danger:focus{color:#fff;background-color:#c9302c;border-color:#761c19}.btn-danger:hover{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active.focus,.btn-danger.active:focus,.btn-danger.active:hover,.btn-danger:active.focus,.btn-danger:active:focus,.btn-danger:active:hover,.open>.dropdown-toggle.btn-danger.focus,.open>.dropdown-toggle.btn-danger:focus,.open>.dropdown-toggle.btn-danger:hover{color:#fff;background-color:#ac2925;border-color:#761c19}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{background-image:none}.btn-danger.disabled.focus,.btn-danger.disabled:focus,.btn-danger.disabled:hover,.btn-danger[disabled].focus,.btn-danger[disabled]:focus,.btn-danger[disabled]:hover,fieldset[disabled] .btn-danger.focus,fieldset[disabled] .btn-danger:focus,fieldset[disabled] .btn-danger:hover{background-color:#d9534f;border-color:#d43f3a}.btn-danger .badge{color:#d9534f;background-color:#fff}.btn-link{font-weight:400;color:#337ab7;border-radius:0}.btn-link,.btn-link.active,.btn-link:active,.btn-link[disabled],fieldset[disabled] .btn-link{background-color:transparent;-webkit-box-shadow:none;box-shadow:none}.btn-link,.btn-link:active,.btn-link:focus,.btn-link:hover{border-color:transparent}.btn-link:focus,.btn-link:hover{color:#23527c;text-decoration:underline;background-color:transparent}.btn-link[disabled]:focus,.btn-link[disabled]:hover,fieldset[disabled] .btn-link:focus,fieldset[disabled] .btn-link:hover{color:#777;text-decoration:none}.btn-group-lg>.btn,.btn-lg{padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.btn-group-sm>.btn,.btn-sm{padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.btn-group-xs>.btn,.btn-xs{padding:1px 5px;font-size:12px;line-height:1.5;border-radius:3px}.btn-block{display:block;width:100%}.btn-block+.btn-block{margin-top:5px}input[type=button].btn-block,input[type=reset].btn-block,input[type=submit].btn-block{width:100%}.fade{opacity:0;-webkit-transition:opacity .15s linear;-o-transition:opacity .15s linear;transition:opacity .15s linear}.fade.in{opacity:1}.collapse{display:none}.collapse.in{display:block}tr.collapse.in{display:table-row}tbody.collapse.in{display:table-row-group}.collapsing{position:relative;height:0;overflow:hidden;-webkit-transition-timing-function:ease;-o-transition-timing-function:ease;transition-timing-function:ease;-webkit-transition-duration:.35s;-o-transition-duration:.35s;transition-duration:.35s;-webkit-transition-property:height,visibility;-o-transition-property:height,visibility;transition-property:height,visibility}.caret{display:inline-block;width:0;height:0;margin-left:2px;vertical-align:middle;border-top:4px dashed;border-top:4px solid\9;border-right:4px solid transparent;border-left:4px solid transparent}.dropdown,.dropup{position:relative}.dropdown-toggle:focus{outline:0}.dropdown-menu{position:absolute;top:100%;left:0;z-index:1000;display:none;float:left;min-width:160px;padding:5px 0;margin:2px 0 0;font-size:14px;text-align:left;list-style:none;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.15);border-radius:4px;-webkit-box-shadow:0 6px 12px rgba(0,0,0,.175);box-shadow:0 6px 12px rgba(0,0,0,.175)}.dropdown-menu.pull-right{right:0;left:auto}.dropdown-menu .divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.dropdown-menu>li>a{display:block;padding:3px 20px;clear:both;font-weight:400;line-height:1.42857143;color:#333;white-space:nowrap}.dropdown-menu>li>a:focus,.dropdown-menu>li>a:hover{color:#262626;text-decoration:none;background-color:#f5f5f5}.dropdown-menu>.active>a,.dropdown-menu>.active>a:focus,.dropdown-menu>.active>a:hover{color:#fff;text-decoration:none;background-color:#337ab7;outline:0}.dropdown-menu>.disabled>a,.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{color:#777}.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{text-decoration:none;cursor:not-allowed;background-color:transparent;background-image:none;filter:progid:DXImageTransform.Microsoft.gradient(enabled=false)}.open>.dropdown-menu{display:block}.open>a{outline:0}.dropdown-menu-right{right:0;left:auto}.dropdown-menu-left{right:auto;left:0}.dropdown-header{display:block;padding:3px 20px;font-size:12px;line-height:1.42857143;color:#777;white-space:nowrap}.dropdown-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:990}.pull-right>.dropdown-menu{right:0;left:auto}.dropup .caret,.navbar-fixed-bottom .dropdown .caret{content:"";border-top:0;border-bottom:4px dashed;border-bottom:4px solid\9}.dropup .dropdown-menu,.navbar-fixed-bottom .dropdown .dropdown-menu{top:auto;bottom:100%;margin-bottom:2px}@media (min-width:768px){.navbar-right .dropdown-menu{right:0;left:auto}.navbar-right .dropdown-menu-left{right:auto;left:0}}.btn-group,.btn-group-vertical{position:relative;display:inline-block;vertical-align:middle}.btn-group-vertical>.btn,.btn-group>.btn{position:relative;float:left}.btn-group-vertical>.btn.active,.btn-group-vertical>.btn:active,.btn-group-vertical>.btn:focus,.btn-group-vertical>.btn:hover,.btn-group>.btn.active,.btn-group>.btn:active,.btn-group>.btn:focus,.btn-group>.btn:hover{z-index:2}.btn-group .btn+.btn,.btn-group .btn+.btn-group,.btn-group .btn-group+.btn,.btn-group .btn-group+.btn-group{margin-left:-1px}.btn-toolbar{margin-left:-5px}.btn-toolbar .btn,.btn-toolbar .btn-group,.btn-toolbar .input-group{float:left}.btn-toolbar>.btn,.btn-toolbar>.btn-group,.btn-toolbar>.input-group{margin-left:5px}.btn-group>.btn:not(:first-child):not(:last-child):not(.dropdown-toggle){border-radius:0}.btn-group>.btn:first-child{margin-left:0}.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn:last-child:not(:first-child),.btn-group>.dropdown-toggle:not(:first-child){border-top-left-radius:0;border-bottom-left-radius:0}.btn-group>.btn-group{float:left}.btn-group>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-bottom-left-radius:0}.btn-group .dropdown-toggle:active,.btn-group.open .dropdown-toggle{outline:0}.btn-group>.btn+.dropdown-toggle{padding-right:8px;padding-left:8px}.btn-group>.btn-lg+.dropdown-toggle{padding-right:12px;padding-left:12px}.btn-group.open .dropdown-toggle{-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn-group.open .dropdown-toggle.btn-link{-webkit-box-shadow:none;box-shadow:none}.btn .caret{margin-left:0}.btn-lg .caret{border-width:5px 5px 0;border-bottom-width:0}.dropup .btn-lg .caret{border-width:0 5px 5px}.btn-group-vertical>.btn,.btn-group-vertical>.btn-group,.btn-group-vertical>.btn-group>.btn{display:block;float:none;width:100%;max-width:100%}.btn-group-vertical>.btn-group>.btn{float:none}.btn-group-vertical>.btn+.btn,.btn-group-vertical>.btn+.btn-group,.btn-group-vertical>.btn-group+.btn,.btn-group-vertical>.btn-group+.btn-group{margin-top:-1px;margin-left:0}.btn-group-vertical>.btn:not(:first-child):not(:last-child){border-radius:0}.btn-group-vertical>.btn:first-child:not(:last-child){border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn:last-child:not(:first-child){border-top-left-radius:0;border-top-right-radius:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}.btn-group-vertical>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group-vertical>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group-vertical>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-top-right-radius:0}.btn-group-justified{display:table;width:100%;table-layout:fixed;border-collapse:separate}.btn-group-justified>.btn,.btn-group-justified>.btn-group{display:table-cell;float:none;width:1%}.btn-group-justified>.btn-group .btn{width:100%}.btn-group-justified>.btn-group .dropdown-menu{left:auto}[data-toggle=buttons]>.btn input[type=checkbox],[data-toggle=buttons]>.btn input[type=radio],[data-toggle=buttons]>.btn-group>.btn input[type=checkbox],[data-toggle=buttons]>.btn-group>.btn input[type=radio]{position:absolute;clip:rect(0,0,0,0);pointer-events:none}.input-group{position:relative;display:table;border-collapse:separate}.input-group[class*=col-]{float:none;padding-right:0;padding-left:0}.input-group .form-control{position:relative;z-index:2;float:left;width:100%;margin-bottom:0}.input-group .form-control:focus{z-index:3}.input-group-lg>.form-control,.input-group-lg>.input-group-addon,.input-group-lg>.input-group-btn>.btn{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-group-lg>.form-control,select.input-group-lg>.input-group-addon,select.input-group-lg>.input-group-btn>.btn{height:46px;line-height:46px}select[multiple].input-group-lg>.form-control,select[multiple].input-group-lg>.input-group-addon,select[multiple].input-group-lg>.input-group-btn>.btn,textarea.input-group-lg>.form-control,textarea.input-group-lg>.input-group-addon,textarea.input-group-lg>.input-group-btn>.btn{height:auto}.input-group-sm>.form-control,.input-group-sm>.input-group-addon,.input-group-sm>.input-group-btn>.btn{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-group-sm>.form-control,select.input-group-sm>.input-group-addon,select.input-group-sm>.input-group-btn>.btn{height:30px;line-height:30px}select[multiple].input-group-sm>.form-control,select[multiple].input-group-sm>.input-group-addon,select[multiple].input-group-sm>.input-group-btn>.btn,textarea.input-group-sm>.form-control,textarea.input-group-sm>.input-group-addon,textarea.input-group-sm>.input-group-btn>.btn{height:auto}.input-group .form-control,.input-group-addon,.input-group-btn{display:table-cell}.input-group .form-control:not(:first-child):not(:last-child),.input-group-addon:not(:first-child):not(:last-child),.input-group-btn:not(:first-child):not(:last-child){border-radius:0}.input-group-addon,.input-group-btn{width:1%;white-space:nowrap;vertical-align:middle}.input-group-addon{padding:6px 12px;font-size:14px;font-weight:400;line-height:1;color:#555;text-align:center;background-color:#eee;border:1px solid #ccc;border-radius:4px}.input-group-addon.input-sm{padding:5px 10px;font-size:12px;border-radius:3px}.input-group-addon.input-lg{padding:10px 16px;font-size:18px;border-radius:6px}.input-group-addon input[type=checkbox],.input-group-addon input[type=radio]{margin-top:0}.input-group .form-control:first-child,.input-group-addon:first-child,.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group>.btn,.input-group-btn:first-child>.dropdown-toggle,.input-group-btn:last-child>.btn-group:not(:last-child)>.btn,.input-group-btn:last-child>.btn:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.input-group-addon:first-child{border-right:0}.input-group .form-control:last-child,.input-group-addon:last-child,.input-group-btn:first-child>.btn-group:not(:first-child)>.btn,.input-group-btn:first-child>.btn:not(:first-child),.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group>.btn,.input-group-btn:last-child>.dropdown-toggle{border-top-left-radius:0;border-bottom-left-radius:0}.input-group-addon:last-child{border-left:0}.input-group-btn{position:relative;font-size:0;white-space:nowrap}.input-group-btn>.btn{position:relative}.input-group-btn>.btn+.btn{margin-left:-1px}.input-group-btn>.btn:active,.input-group-btn>.btn:focus,.input-group-btn>.btn:hover{z-index:2}.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group{margin-right:-1px}.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group{z-index:2;margin-left:-1px}.nav{padding-left:0;margin-bottom:0;list-style:none}.nav>li{position:relative;display:block}.nav>li>a{position:relative;display:block;padding:10px 15px}.nav>li>a:focus,.nav>li>a:hover{text-decoration:none;background-color:#eee}.nav>li.disabled>a{color:#777}.nav>li.disabled>a:focus,.nav>li.disabled>a:hover{color:#777;text-decoration:none;cursor:not-allowed;background-color:transparent}.nav .open>a,.nav .open>a:focus,.nav .open>a:hover{background-color:#eee;border-color:#337ab7}.nav .nav-divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.nav>li>a>img{max-width:none}.nav-tabs{border-bottom:1px solid #ddd}.nav-tabs>li{float:left;margin-bottom:-1px}.nav-tabs>li>a{margin-right:2px;line-height:1.42857143;border:1px solid transparent;border-radius:4px 4px 0 0}.nav-tabs>li>a:hover{border-color:#eee #eee #ddd}.nav-tabs>li.active>a,.nav-tabs>li.active>a:focus,.nav-tabs>li.active>a:hover{color:#555;cursor:default;background-color:#fff;border:1px solid #ddd;border-bottom-color:transparent}.nav-tabs.nav-justified{width:100%;border-bottom:0}.nav-tabs.nav-justified>li{float:none}.nav-tabs.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-tabs.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-tabs.nav-justified>li{display:table-cell;width:1%}.nav-tabs.nav-justified>li>a{margin-bottom:0}}.nav-tabs.nav-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs.nav-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border-bottom-color:#fff}}.nav-pills>li{float:left}.nav-pills>li>a{border-radius:4px}.nav-pills>li+li{margin-left:2px}.nav-pills>li.active>a,.nav-pills>li.active>a:focus,.nav-pills>li.active>a:hover{color:#fff;background-color:#337ab7}.nav-stacked>li{float:none}.nav-stacked>li+li{margin-top:2px;margin-left:0}.nav-justified{width:100%}.nav-justified>li{float:none}.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-justified>li{display:table-cell;width:1%}.nav-justified>li>a{margin-bottom:0}}.nav-tabs-justified{border-bottom:0}.nav-tabs-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border-bottom-color:#fff}}.tab-content>.tab-pane{display:none}.tab-content>.active{display:block}.nav-tabs .dropdown-menu{margin-top:-1px;border-top-left-radius:0;border-top-right-radius:0}.navbar{position:relative;min-height:50px;margin-bottom:20px;border:1px solid transparent}@media (min-width:768px){.navbar{border-radius:4px}}@media (min-width:768px){.navbar-header{float:left}}.navbar-collapse{padding-right:15px;padding-left:15px;overflow-x:visible;-webkit-overflow-scrolling:touch;border-top:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1)}.navbar-collapse.in{overflow-y:auto}@media (min-width:768px){.navbar-collapse{width:auto;border-top:0;-webkit-box-shadow:none;box-shadow:none}.navbar-collapse.collapse{display:block!important;height:auto!important;padding-bottom:0;overflow:visible!important}.navbar-collapse.in{overflow-y:visible}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse,.navbar-static-top .navbar-collapse{padding-right:0;padding-left:0}}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:340px}@media (max-device-width:480px) and (orientation:landscape){.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:200px}}.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:0;margin-left:0}}.navbar-static-top{z-index:1000;border-width:0 0 1px}@media (min-width:768px){.navbar-static-top{border-radius:0}}.navbar-fixed-bottom,.navbar-fixed-top{position:fixed;right:0;left:0;z-index:1030}@media (min-width:768px){.navbar-fixed-bottom,.navbar-fixed-top{border-radius:0}}.navbar-fixed-top{top:0;border-width:0 0 1px}.navbar-fixed-bottom{bottom:0;margin-bottom:0;border-width:1px 0 0}.navbar-brand{float:left;height:50px;padding:15px 15px;font-size:18px;line-height:20px}.navbar-brand:focus,.navbar-brand:hover{text-decoration:none}.navbar-brand>img{display:block}@media (min-width:768px){.navbar>.container .navbar-brand,.navbar>.container-fluid .navbar-brand{margin-left:-15px}}.navbar-toggle{position:relative;float:right;padding:9px 10px;margin-top:8px;margin-right:15px;margin-bottom:8px;background-color:transparent;background-image:none;border:1px solid transparent;border-radius:4px}.navbar-toggle:focus{outline:0}.navbar-toggle .icon-bar{display:block;width:22px;height:2px;border-radius:1px}.navbar-toggle .icon-bar+.icon-bar{margin-top:4px}@media (min-width:768px){.navbar-toggle{display:none}}.navbar-nav{margin:7.5px -15px}.navbar-nav>li>a{padding-top:10px;padding-bottom:10px;line-height:20px}@media (max-width:767px){.navbar-nav .open .dropdown-menu{position:static;float:none;width:auto;margin-top:0;background-color:transparent;border:0;-webkit-box-shadow:none;box-shadow:none}.navbar-nav .open .dropdown-menu .dropdown-header,.navbar-nav .open .dropdown-menu>li>a{padding:5px 15px 5px 25px}.navbar-nav .open .dropdown-menu>li>a{line-height:20px}.navbar-nav .open .dropdown-menu>li>a:focus,.navbar-nav .open .dropdown-menu>li>a:hover{background-image:none}}@media (min-width:768px){.navbar-nav{float:left;margin:0}.navbar-nav>li{float:left}.navbar-nav>li>a{padding-top:15px;padding-bottom:15px}}.navbar-form{padding:10px 15px;margin-top:8px;margin-right:-15px;margin-bottom:8px;margin-left:-15px;border-top:1px solid transparent;border-bottom:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1)}@media (min-width:768px){.navbar-form .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.navbar-form .form-control{display:inline-block;width:auto;vertical-align:middle}.navbar-form .form-control-static{display:inline-block}.navbar-form .input-group{display:inline-table;vertical-align:middle}.navbar-form .input-group .form-control,.navbar-form .input-group .input-group-addon,.navbar-form .input-group .input-group-btn{width:auto}.navbar-form .input-group>.form-control{width:100%}.navbar-form .control-label{margin-bottom:0;vertical-align:middle}.navbar-form .checkbox,.navbar-form .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.navbar-form .checkbox label,.navbar-form .radio label{padding-left:0}.navbar-form .checkbox input[type=checkbox],.navbar-form .radio input[type=radio]{position:relative;margin-left:0}.navbar-form .has-feedback .form-control-feedback{top:0}}@media (max-width:767px){.navbar-form .form-group{margin-bottom:5px}.navbar-form .form-group:last-child{margin-bottom:0}}@media (min-width:768px){.navbar-form{width:auto;padding-top:0;padding-bottom:0;margin-right:0;margin-left:0;border:0;-webkit-box-shadow:none;box-shadow:none}}.navbar-nav>li>.dropdown-menu{margin-top:0;border-top-left-radius:0;border-top-right-radius:0}.navbar-fixed-bottom .navbar-nav>li>.dropdown-menu{margin-bottom:0;border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.navbar-btn{margin-top:8px;margin-bottom:8px}.navbar-btn.btn-sm{margin-top:10px;margin-bottom:10px}.navbar-btn.btn-xs{margin-top:14px;margin-bottom:14px}.navbar-text{margin-top:15px;margin-bottom:15px}@media (min-width:768px){.navbar-text{float:left;margin-right:15px;margin-left:15px}}@media (min-width:768px){.navbar-left{float:left!important}.navbar-right{float:right!important;margin-right:-15px}.navbar-right~.navbar-right{margin-right:0}}.navbar-default{background-color:#f8f8f8;border-color:#e7e7e7}.navbar-default .navbar-brand{color:#777}.navbar-default .navbar-brand:focus,.navbar-default .navbar-brand:hover{color:#5e5e5e;background-color:transparent}.navbar-default .navbar-text{color:#777}.navbar-default .navbar-nav>li>a{color:#777}.navbar-default .navbar-nav>li>a:focus,.navbar-default .navbar-nav>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav>.active>a,.navbar-default .navbar-nav>.active>a:focus,.navbar-default .navbar-nav>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav>.disabled>a,.navbar-default .navbar-nav>.disabled>a:focus,.navbar-default .navbar-nav>.disabled>a:hover{color:#ccc;background-color:transparent}.navbar-default .navbar-toggle{border-color:#ddd}.navbar-default .navbar-toggle:focus,.navbar-default .navbar-toggle:hover{background-color:#ddd}.navbar-default .navbar-toggle .icon-bar{background-color:#888}.navbar-default .navbar-collapse,.navbar-default .navbar-form{border-color:#e7e7e7}.navbar-default .navbar-nav>.open>a,.navbar-default .navbar-nav>.open>a:focus,.navbar-default .navbar-nav>.open>a:hover{color:#555;background-color:#e7e7e7}@media (max-width:767px){.navbar-default .navbar-nav .open .dropdown-menu>li>a{color:#777}.navbar-default .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav .open .dropdown-menu>.active>a,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#ccc;background-color:transparent}}.navbar-default .navbar-link{color:#777}.navbar-default .navbar-link:hover{color:#333}.navbar-default .btn-link{color:#777}.navbar-default .btn-link:focus,.navbar-default .btn-link:hover{color:#333}.navbar-default .btn-link[disabled]:focus,.navbar-default .btn-link[disabled]:hover,fieldset[disabled] .navbar-default .btn-link:focus,fieldset[disabled] .navbar-default .btn-link:hover{color:#ccc}.navbar-inverse{background-color:#222;border-color:#080808}.navbar-inverse .navbar-brand{color:#9d9d9d}.navbar-inverse .navbar-brand:focus,.navbar-inverse .navbar-brand:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-text{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a:focus,.navbar-inverse .navbar-nav>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav>.active>a,.navbar-inverse .navbar-nav>.active>a:focus,.navbar-inverse .navbar-nav>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav>.disabled>a,.navbar-inverse .navbar-nav>.disabled>a:focus,.navbar-inverse .navbar-nav>.disabled>a:hover{color:#444;background-color:transparent}.navbar-inverse .navbar-toggle{border-color:#333}.navbar-inverse .navbar-toggle:focus,.navbar-inverse .navbar-toggle:hover{background-color:#333}.navbar-inverse .navbar-toggle .icon-bar{background-color:#fff}.navbar-inverse .navbar-collapse,.navbar-inverse .navbar-form{border-color:#101010}.navbar-inverse .navbar-nav>.open>a,.navbar-inverse .navbar-nav>.open>a:focus,.navbar-inverse .navbar-nav>.open>a:hover{color:#fff;background-color:#080808}@media (max-width:767px){.navbar-inverse .navbar-nav .open .dropdown-menu>.dropdown-header{border-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu .divider{background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#444;background-color:transparent}}.navbar-inverse .navbar-link{color:#9d9d9d}.navbar-inverse .navbar-link:hover{color:#fff}.navbar-inverse .btn-link{color:#9d9d9d}.navbar-inverse .btn-link:focus,.navbar-inverse .btn-link:hover{color:#fff}.navbar-inverse .btn-link[disabled]:focus,.navbar-inverse .btn-link[disabled]:hover,fieldset[disabled] .navbar-inverse .btn-link:focus,fieldset[disabled] .navbar-inverse .btn-link:hover{color:#444}.breadcrumb{padding:8px 15px;margin-bottom:20px;list-style:none;background-color:#f5f5f5;border-radius:4px}.breadcrumb>li{display:inline-block}.breadcrumb>li+li:before{padding:0 5px;color:#ccc;content:"/\00a0"}.breadcrumb>.active{color:#777}.pagination{display:inline-block;padding-left:0;margin:20px 0;border-radius:4px}.pagination>li{display:inline}.pagination>li>a,.pagination>li>span{position:relative;float:left;padding:6px 12px;margin-left:-1px;line-height:1.42857143;color:#337ab7;text-decoration:none;background-color:#fff;border:1px solid #ddd}.pagination>li:first-child>a,.pagination>li:first-child>span{margin-left:0;border-top-left-radius:4px;border-bottom-left-radius:4px}.pagination>li:last-child>a,.pagination>li:last-child>span{border-top-right-radius:4px;border-bottom-right-radius:4px}.pagination>li>a:focus,.pagination>li>a:hover,.pagination>li>span:focus,.pagination>li>span:hover{z-index:2;color:#23527c;background-color:#eee;border-color:#ddd}.pagination>.active>a,.pagination>.active>a:focus,.pagination>.active>a:hover,.pagination>.active>span,.pagination>.active>span:focus,.pagination>.active>span:hover{z-index:3;color:#fff;cursor:default;background-color:#337ab7;border-color:#337ab7}.pagination>.disabled>a,.pagination>.disabled>a:focus,.pagination>.disabled>a:hover,.pagination>.disabled>span,.pagination>.disabled>span:focus,.pagination>.disabled>span:hover{color:#777;cursor:not-allowed;background-color:#fff;border-color:#ddd}.pagination-lg>li>a,.pagination-lg>li>span{padding:10px 16px;font-size:18px;line-height:1.3333333}.pagination-lg>li:first-child>a,.pagination-lg>li:first-child>span{border-top-left-radius:6px;border-bottom-left-radius:6px}.pagination-lg>li:last-child>a,.pagination-lg>li:last-child>span{border-top-right-radius:6px;border-bottom-right-radius:6px}.pagination-sm>li>a,.pagination-sm>li>span{padding:5px 10px;font-size:12px;line-height:1.5}.pagination-sm>li:first-child>a,.pagination-sm>li:first-child>span{border-top-left-radius:3px;border-bottom-left-radius:3px}.pagination-sm>li:last-child>a,.pagination-sm>li:last-child>span{border-top-right-radius:3px;border-bottom-right-radius:3px}.pager{padding-left:0;margin:20px 0;text-align:center;list-style:none}.pager li{display:inline}.pager li>a,.pager li>span{display:inline-block;padding:5px 14px;background-color:#fff;border:1px solid #ddd;border-radius:15px}.pager li>a:focus,.pager li>a:hover{text-decoration:none;background-color:#eee}.pager .next>a,.pager .next>span{float:right}.pager .previous>a,.pager .previous>span{float:left}.pager .disabled>a,.pager .disabled>a:focus,.pager .disabled>a:hover,.pager .disabled>span{color:#777;cursor:not-allowed;background-color:#fff}.label{display:inline;padding:.2em .6em .3em;font-size:75%;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:baseline;border-radius:.25em}a.label:focus,a.label:hover{color:#fff;text-decoration:none;cursor:pointer}.label:empty{display:none}.btn .label{position:relative;top:-1px}.label-default{background-color:#777}.label-default[href]:focus,.label-default[href]:hover{background-color:#5e5e5e}.label-primary{background-color:#337ab7}.label-primary[href]:focus,.label-primary[href]:hover{background-color:#286090}.label-success{background-color:#5cb85c}.label-success[href]:focus,.label-success[href]:hover{background-color:#449d44}.label-info{background-color:#5bc0de}.label-info[href]:focus,.label-info[href]:hover{background-color:#31b0d5}.label-warning{background-color:#f0ad4e}.label-warning[href]:focus,.label-warning[href]:hover{background-color:#ec971f}.label-danger{background-color:#d9534f}.label-danger[href]:focus,.label-danger[href]:hover{background-color:#c9302c}.badge{display:inline-block;min-width:10px;padding:3px 7px;font-size:12px;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:middle;background-color:#777;border-radius:10px}.badge:empty{display:none}.btn .badge{position:relative;top:-1px}.btn-group-xs>.btn .badge,.btn-xs .badge{top:0;padding:1px 5px}a.badge:focus,a.badge:hover{color:#fff;text-decoration:none;cursor:pointer}.list-group-item.active>.badge,.nav-pills>.active>a>.badge{color:#337ab7;background-color:#fff}.list-group-item>.badge{float:right}.list-group-item>.badge+.badge{margin-right:5px}.nav-pills>li>a>.badge{margin-left:3px}.jumbotron{padding-top:30px;padding-bottom:30px;margin-bottom:30px;color:inherit;background-color:#eee}.jumbotron .h1,.jumbotron h1{color:inherit}.jumbotron p{margin-bottom:15px;font-size:21px;font-weight:200}.jumbotron>hr{border-top-color:#d5d5d5}.container .jumbotron,.container-fluid .jumbotron{padding-right:15px;padding-left:15px;border-radius:6px}.jumbotron .container{max-width:100%}@media screen and (min-width:768px){.jumbotron{padding-top:48px;padding-bottom:48px}.container .jumbotron,.container-fluid .jumbotron{padding-right:60px;padding-left:60px}.jumbotron .h1,.jumbotron h1{font-size:63px}}.thumbnail{display:block;padding:4px;margin-bottom:20px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:border .2s ease-in-out;-o-transition:border .2s ease-in-out;transition:border .2s ease-in-out}.thumbnail a>img,.thumbnail>img{margin-right:auto;margin-left:auto}a.thumbnail.active,a.thumbnail:focus,a.thumbnail:hover{border-color:#337ab7}.thumbnail .caption{padding:9px;color:#333}.alert{padding:15px;margin-bottom:20px;border:1px solid transparent;border-radius:4px}.alert h4{margin-top:0;color:inherit}.alert .alert-link{font-weight:700}.alert>p,.alert>ul{margin-bottom:0}.alert>p+p{margin-top:5px}.alert-dismissable,.alert-dismissible{padding-right:35px}.alert-dismissable .close,.alert-dismissible .close{position:relative;top:-2px;right:-21px;color:inherit}.alert-success{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.alert-success hr{border-top-color:#c9e2b3}.alert-success .alert-link{color:#2b542c}.alert-info{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.alert-info hr{border-top-color:#a6e1ec}.alert-info .alert-link{color:#245269}.alert-warning{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.alert-warning hr{border-top-color:#f7e1b5}.alert-warning .alert-link{color:#66512c}.alert-danger{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.alert-danger hr{border-top-color:#e4b9c0}.alert-danger .alert-link{color:#843534}@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@-o-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}.progress{height:20px;margin-bottom:20px;overflow:hidden;background-color:#f5f5f5;border-radius:4px;-webkit-box-shadow:inset 0 1px 2px rgba(0,0,0,.1);box-shadow:inset 0 1px 2px rgba(0,0,0,.1)}.progress-bar{float:left;width:0;height:100%;font-size:12px;line-height:20px;color:#fff;text-align:center;background-color:#337ab7;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);-webkit-transition:width .6s ease;-o-transition:width .6s ease;transition:width .6s ease}.progress-bar-striped,.progress-striped .progress-bar{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);-webkit-background-size:40px 40px;background-size:40px 40px}.progress-bar.active,.progress.active .progress-bar{-webkit-animation:progress-bar-stripes 2s linear infinite;-o-animation:progress-bar-stripes 2s linear infinite;animation:progress-bar-stripes 2s linear infinite}.progress-bar-success{background-color:#5cb85c}.progress-striped .progress-bar-success{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-info{background-color:#5bc0de}.progress-striped .progress-bar-info{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-warning{background-color:#f0ad4e}.progress-striped .progress-bar-warning{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-danger{background-color:#d9534f}.progress-striped .progress-bar-danger{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.media{margin-top:15px}.media:first-child{margin-top:0}.media,.media-body{overflow:hidden;zoom:1}.media-body{width:10000px}.media-object{display:block}.media-object.img-thumbnail{max-width:none}.media-right,.media>.pull-right{padding-left:10px}.media-left,.media>.pull-left{padding-right:10px}.media-body,.media-left,.media-right{display:table-cell;vertical-align:top}.media-middle{vertical-align:middle}.media-bottom{vertical-align:bottom}.media-heading{margin-top:0;margin-bottom:5px}.media-list{padding-left:0;list-style:none}.list-group{padding-left:0;margin-bottom:20px}.list-group-item{position:relative;display:block;padding:10px 15px;margin-bottom:-1px;background-color:#fff;border:1px solid #ddd}.list-group-item:first-child{border-top-left-radius:4px;border-top-right-radius:4px}.list-group-item:last-child{margin-bottom:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}a.list-group-item,button.list-group-item{color:#555}a.list-group-item .list-group-item-heading,button.list-group-item .list-group-item-heading{color:#333}a.list-group-item:focus,a.list-group-item:hover,button.list-group-item:focus,button.list-group-item:hover{color:#555;text-decoration:none;background-color:#f5f5f5}button.list-group-item{width:100%;text-align:left}.list-group-item.disabled,.list-group-item.disabled:focus,.list-group-item.disabled:hover{color:#777;cursor:not-allowed;background-color:#eee}.list-group-item.disabled .list-group-item-heading,.list-group-item.disabled:focus .list-group-item-heading,.list-group-item.disabled:hover .list-group-item-heading{color:inherit}.list-group-item.disabled .list-group-item-text,.list-group-item.disabled:focus .list-group-item-text,.list-group-item.disabled:hover .list-group-item-text{color:#777}.list-group-item.active,.list-group-item.active:focus,.list-group-item.active:hover{z-index:2;color:#fff;background-color:#337ab7;border-color:#337ab7}.list-group-item.active .list-group-item-heading,.list-group-item.active .list-group-item-heading>.small,.list-group-item.active .list-group-item-heading>small,.list-group-item.active:focus .list-group-item-heading,.list-group-item.active:focus .list-group-item-heading>.small,.list-group-item.active:focus .list-group-item-heading>small,.list-group-item.active:hover .list-group-item-heading,.list-group-item.active:hover .list-group-item-heading>.small,.list-group-item.active:hover .list-group-item-heading>small{color:inherit}.list-group-item.active .list-group-item-text,.list-group-item.active:focus .list-group-item-text,.list-group-item.active:hover .list-group-item-text{color:#c7ddef}.list-group-item-success{color:#3c763d;background-color:#dff0d8}a.list-group-item-success,button.list-group-item-success{color:#3c763d}a.list-group-item-success .list-group-item-heading,button.list-group-item-success .list-group-item-heading{color:inherit}a.list-group-item-success:focus,a.list-group-item-success:hover,button.list-group-item-success:focus,button.list-group-item-success:hover{color:#3c763d;background-color:#d0e9c6}a.list-group-item-success.active,a.list-group-item-success.active:focus,a.list-group-item-success.active:hover,button.list-group-item-success.active,button.list-group-item-success.active:focus,button.list-group-item-success.active:hover{color:#fff;background-color:#3c763d;border-color:#3c763d}.list-group-item-info{color:#31708f;background-color:#d9edf7}a.list-group-item-info,button.list-group-item-info{color:#31708f}a.list-group-item-info .list-group-item-heading,button.list-group-item-info .list-group-item-heading{color:inherit}a.list-group-item-info:focus,a.list-group-item-info:hover,button.list-group-item-info:focus,button.list-group-item-info:hover{color:#31708f;background-color:#c4e3f3}a.list-group-item-info.active,a.list-group-item-info.active:focus,a.list-group-item-info.active:hover,button.list-group-item-info.active,button.list-group-item-info.active:focus,button.list-group-item-info.active:hover{color:#fff;background-color:#31708f;border-color:#31708f}.list-group-item-warning{color:#8a6d3b;background-color:#fcf8e3}a.list-group-item-warning,button.list-group-item-warning{color:#8a6d3b}a.list-group-item-warning .list-group-item-heading,button.list-group-item-warning .list-group-item-heading{color:inherit}a.list-group-item-warning:focus,a.list-group-item-warning:hover,button.list-group-item-warning:focus,button.list-group-item-warning:hover{color:#8a6d3b;background-color:#faf2cc}a.list-group-item-warning.active,a.list-group-item-warning.active:focus,a.list-group-item-warning.active:hover,button.list-group-item-warning.active,button.list-group-item-warning.active:focus,button.list-group-item-warning.active:hover{color:#fff;background-color:#8a6d3b;border-color:#8a6d3b}.list-group-item-danger{color:#a94442;background-color:#f2dede}a.list-group-item-danger,button.list-group-item-danger{color:#a94442}a.list-group-item-danger .list-group-item-heading,button.list-group-item-danger .list-group-item-heading{color:inherit}a.list-group-item-danger:focus,a.list-group-item-danger:hover,button.list-group-item-danger:focus,button.list-group-item-danger:hover{color:#a94442;background-color:#ebcccc}a.list-group-item-danger.active,a.list-group-item-danger.active:focus,a.list-group-item-danger.active:hover,button.list-group-item-danger.active,button.list-group-item-danger.active:focus,button.list-group-item-danger.active:hover{color:#fff;background-color:#a94442;border-color:#a94442}.list-group-item-heading{margin-top:0;margin-bottom:5px}.list-group-item-text{margin-bottom:0;line-height:1.3}.panel{margin-bottom:20px;background-color:#fff;border:1px solid transparent;border-radius:4px;-webkit-box-shadow:0 1px 1px rgba(0,0,0,.05);box-shadow:0 1px 1px rgba(0,0,0,.05)}.panel-body{padding:15px}.panel-heading{padding:10px 15px;border-bottom:1px solid transparent;border-top-left-radius:3px;border-top-right-radius:3px}.panel-heading>.dropdown .dropdown-toggle{color:inherit}.panel-title{margin-top:0;margin-bottom:0;font-size:16px;color:inherit}.panel-title>.small,.panel-title>.small>a,.panel-title>a,.panel-title>small,.panel-title>small>a{color:inherit}.panel-footer{padding:10px 15px;background-color:#f5f5f5;border-top:1px solid #ddd;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.list-group,.panel>.panel-collapse>.list-group{margin-bottom:0}.panel>.list-group .list-group-item,.panel>.panel-collapse>.list-group .list-group-item{border-width:1px 0;border-radius:0}.panel>.list-group:first-child .list-group-item:first-child,.panel>.panel-collapse>.list-group:first-child .list-group-item:first-child{border-top:0;border-top-left-radius:3px;border-top-right-radius:3px}.panel>.list-group:last-child .list-group-item:last-child,.panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{border-bottom:0;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.panel-heading+.panel-collapse>.list-group .list-group-item:first-child{border-top-left-radius:0;border-top-right-radius:0}.panel-heading+.list-group .list-group-item:first-child{border-top-width:0}.list-group+.panel-footer{border-top-width:0}.panel>.panel-collapse>.table,.panel>.table,.panel>.table-responsive>.table{margin-bottom:0}.panel>.panel-collapse>.table caption,.panel>.table caption,.panel>.table-responsive>.table caption{padding-right:15px;padding-left:15px}.panel>.table-responsive:first-child>.table:first-child,.panel>.table:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child,.panel>.table:first-child>thead:first-child>tr:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table:first-child>thead:first-child>tr:first-child th:first-child{border-top-left-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table:first-child>thead:first-child>tr:first-child th:last-child{border-top-right-radius:3px}.panel>.table-responsive:last-child>.table:last-child,.panel>.table:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:first-child{border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:last-child{border-bottom-right-radius:3px}.panel>.panel-body+.table,.panel>.panel-body+.table-responsive,.panel>.table+.panel-body,.panel>.table-responsive+.panel-body{border-top:1px solid #ddd}.panel>.table>tbody:first-child>tr:first-child td,.panel>.table>tbody:first-child>tr:first-child th{border-top:0}.panel>.table-bordered,.panel>.table-responsive>.table-bordered{border:0}.panel>.table-bordered>tbody>tr>td:first-child,.panel>.table-bordered>tbody>tr>th:first-child,.panel>.table-bordered>tfoot>tr>td:first-child,.panel>.table-bordered>tfoot>tr>th:first-child,.panel>.table-bordered>thead>tr>td:first-child,.panel>.table-bordered>thead>tr>th:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:first-child,.panel>.table-responsive>.table-bordered>thead>tr>td:first-child,.panel>.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.panel>.table-bordered>tbody>tr>td:last-child,.panel>.table-bordered>tbody>tr>th:last-child,.panel>.table-bordered>tfoot>tr>td:last-child,.panel>.table-bordered>tfoot>tr>th:last-child,.panel>.table-bordered>thead>tr>td:last-child,.panel>.table-bordered>thead>tr>th:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:last-child,.panel>.table-responsive>.table-bordered>thead>tr>td:last-child,.panel>.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.panel>.table-bordered>tbody>tr:first-child>td,.panel>.table-bordered>tbody>tr:first-child>th,.panel>.table-bordered>thead>tr:first-child>td,.panel>.table-bordered>thead>tr:first-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>th,.panel>.table-responsive>.table-bordered>thead>tr:first-child>td,.panel>.table-responsive>.table-bordered>thead>tr:first-child>th{border-bottom:0}.panel>.table-bordered>tbody>tr:last-child>td,.panel>.table-bordered>tbody>tr:last-child>th,.panel>.table-bordered>tfoot>tr:last-child>td,.panel>.table-bordered>tfoot>tr:last-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>th,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>td,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}.panel>.table-responsive{margin-bottom:0;border:0}.panel-group{margin-bottom:20px}.panel-group .panel{margin-bottom:0;border-radius:4px}.panel-group .panel+.panel{margin-top:5px}.panel-group .panel-heading{border-bottom:0}.panel-group .panel-heading+.panel-collapse>.list-group,.panel-group .panel-heading+.panel-collapse>.panel-body{border-top:1px solid #ddd}.panel-group .panel-footer{border-top:0}.panel-group .panel-footer+.panel-collapse .panel-body{border-bottom:1px solid #ddd}.panel-default{border-color:#ddd}.panel-default>.panel-heading{color:#333;background-color:#f5f5f5;border-color:#ddd}.panel-default>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ddd}.panel-default>.panel-heading .badge{color:#f5f5f5;background-color:#333}.panel-default>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ddd}.panel-primary{border-color:#337ab7}.panel-primary>.panel-heading{color:#fff;background-color:#337ab7;border-color:#337ab7}.panel-primary>.panel-heading+.panel-collapse>.panel-body{border-top-color:#337ab7}.panel-primary>.panel-heading .badge{color:#337ab7;background-color:#fff}.panel-primary>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#337ab7}.panel-success{border-color:#d6e9c6}.panel-success>.panel-heading{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.panel-success>.panel-heading+.panel-collapse>.panel-body{border-top-color:#d6e9c6}.panel-success>.panel-heading .badge{color:#dff0d8;background-color:#3c763d}.panel-success>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#d6e9c6}.panel-info{border-color:#bce8f1}.panel-info>.panel-heading{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.panel-info>.panel-heading+.panel-collapse>.panel-body{border-top-color:#bce8f1}.panel-info>.panel-heading .badge{color:#d9edf7;background-color:#31708f}.panel-info>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#bce8f1}.panel-warning{border-color:#faebcc}.panel-warning>.panel-heading{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.panel-warning>.panel-heading+.panel-collapse>.panel-body{border-top-color:#faebcc}.panel-warning>.panel-heading .badge{color:#fcf8e3;background-color:#8a6d3b}.panel-warning>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#faebcc}.panel-danger{border-color:#ebccd1}.panel-danger>.panel-heading{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.panel-danger>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ebccd1}.panel-danger>.panel-heading .badge{color:#f2dede;background-color:#a94442}.panel-danger>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ebccd1}.embed-responsive{position:relative;display:block;height:0;padding:0;overflow:hidden}.embed-responsive .embed-responsive-item,.embed-responsive embed,.embed-responsive iframe,.embed-responsive object,.embed-responsive video{position:absolute;top:0;bottom:0;left:0;width:100%;height:100%;border:0}.embed-responsive-16by9{padding-bottom:56.25%}.embed-responsive-4by3{padding-bottom:75%}.well{min-height:20px;padding:19px;margin-bottom:20px;background-color:#f5f5f5;border:1px solid #e3e3e3;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.05);box-shadow:inset 0 1px 1px rgba(0,0,0,.05)}.well blockquote{border-color:#ddd;border-color:rgba(0,0,0,.15)}.well-lg{padding:24px;border-radius:6px}.well-sm{padding:9px;border-radius:3px}.close{float:right;font-size:21px;font-weight:700;line-height:1;color:#000;text-shadow:0 1px 0 #fff;filter:alpha(opacity=20);opacity:.2}.close:focus,.close:hover{color:#000;text-decoration:none;cursor:pointer;filter:alpha(opacity=50);opacity:.5}button.close{-webkit-appearance:none;padding:0;cursor:pointer;background:0 0;border:0}.modal-open{overflow:hidden}.modal{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1050;display:none;overflow:hidden;-webkit-overflow-scrolling:touch;outline:0}.modal.fade .modal-dialog{-webkit-transition:-webkit-transform .3s ease-out;-o-transition:-o-transform .3s ease-out;transition:transform .3s ease-out;-webkit-transform:translate(0,-25%);-ms-transform:translate(0,-25%);-o-transform:translate(0,-25%);transform:translate(0,-25%)}.modal.in .modal-dialog{-webkit-transform:translate(0,0);-ms-transform:translate(0,0);-o-transform:translate(0,0);transform:translate(0,0)}.modal-open .modal{overflow-x:hidden;overflow-y:auto}.modal-dialog{position:relative;width:auto;margin:10px}.modal-content{position:relative;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #999;border:1px solid rgba(0,0,0,.2);border-radius:6px;outline:0;-webkit-box-shadow:0 3px 9px rgba(0,0,0,.5);box-shadow:0 3px 9px rgba(0,0,0,.5)}.modal-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1040;background-color:#000}.modal-backdrop.fade{filter:alpha(opacity=0);opacity:0}.modal-backdrop.in{filter:alpha(opacity=50);opacity:.5}.modal-header{padding:15px;border-bottom:1px solid #e5e5e5}.modal-header .close{margin-top:-2px}.modal-title{margin:0;line-height:1.42857143}.modal-body{position:relative;padding:15px}.modal-footer{padding:15px;text-align:right;border-top:1px solid #e5e5e5}.modal-footer .btn+.btn{margin-bottom:0;margin-left:5px}.modal-footer .btn-group .btn+.btn{margin-left:-1px}.modal-footer .btn-block+.btn-block{margin-left:0}.modal-scrollbar-measure{position:absolute;top:-9999px;width:50px;height:50px;overflow:scroll}@media (min-width:768px){.modal-dialog{width:600px;margin:30px auto}.modal-content{-webkit-box-shadow:0 5px 15px rgba(0,0,0,.5);box-shadow:0 5px 15px rgba(0,0,0,.5)}.modal-sm{width:300px}}@media (min-width:992px){.modal-lg{width:900px}}.tooltip{position:absolute;z-index:1070;display:block;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:12px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;filter:alpha(opacity=0);opacity:0;line-break:auto}.tooltip.in{filter:alpha(opacity=90);opacity:.9}.tooltip.top{padding:5px 0;margin-top:-3px}.tooltip.right{padding:0 5px;margin-left:3px}.tooltip.bottom{padding:5px 0;margin-top:3px}.tooltip.left{padding:0 5px;margin-left:-3px}.tooltip-inner{max-width:200px;padding:3px 8px;color:#fff;text-align:center;background-color:#000;border-radius:4px}.tooltip-arrow{position:absolute;width:0;height:0;border-color:transparent;border-style:solid}.tooltip.top .tooltip-arrow{bottom:0;left:50%;margin-left:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-left .tooltip-arrow{right:5px;bottom:0;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-right .tooltip-arrow{bottom:0;left:5px;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.right .tooltip-arrow{top:50%;left:0;margin-top:-5px;border-width:5px 5px 5px 0;border-right-color:#000}.tooltip.left .tooltip-arrow{top:50%;right:0;margin-top:-5px;border-width:5px 0 5px 5px;border-left-color:#000}.tooltip.bottom .tooltip-arrow{top:0;left:50%;margin-left:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-left .tooltip-arrow{top:0;right:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-right .tooltip-arrow{top:0;left:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.popover{position:absolute;top:0;left:0;z-index:1060;display:none;max-width:276px;padding:1px;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.2);border-radius:6px;-webkit-box-shadow:0 5px 10px rgba(0,0,0,.2);box-shadow:0 5px 10px rgba(0,0,0,.2);line-break:auto}.popover.top{margin-top:-10px}.popover.right{margin-left:10px}.popover.bottom{margin-top:10px}.popover.left{margin-left:-10px}.popover-title{padding:8px 14px;margin:0;font-size:14px;background-color:#f7f7f7;border-bottom:1px solid #ebebeb;border-radius:5px 5px 0 0}.popover-content{padding:9px 14px}.popover>.arrow,.popover>.arrow:after{position:absolute;display:block;width:0;height:0;border-color:transparent;border-style:solid}.popover>.arrow{border-width:11px}.popover>.arrow:after{content:"";border-width:10px}.popover.top>.arrow{bottom:-11px;left:50%;margin-left:-11px;border-top-color:#999;border-top-color:rgba(0,0,0,.25);border-bottom-width:0}.popover.top>.arrow:after{bottom:1px;margin-left:-10px;content:" ";border-top-color:#fff;border-bottom-width:0}.popover.right>.arrow{top:50%;left:-11px;margin-top:-11px;border-right-color:#999;border-right-color:rgba(0,0,0,.25);border-left-width:0}.popover.right>.arrow:after{bottom:-10px;left:1px;content:" ";border-right-color:#fff;border-left-width:0}.popover.bottom>.arrow{top:-11px;left:50%;margin-left:-11px;border-top-width:0;border-bottom-color:#999;border-bottom-color:rgba(0,0,0,.25)}.popover.bottom>.arrow:after{top:1px;margin-left:-10px;content:" ";border-top-width:0;border-bottom-color:#fff}.popover.left>.arrow{top:50%;right:-11px;margin-top:-11px;border-right-width:0;border-left-color:#999;border-left-color:rgba(0,0,0,.25)}.popover.left>.arrow:after{right:1px;bottom:-10px;content:" ";border-right-width:0;border-left-color:#fff}.carousel{position:relative}.carousel-inner{position:relative;width:100%;overflow:hidden}.carousel-inner>.item{position:relative;display:none;-webkit-transition:.6s ease-in-out left;-o-transition:.6s ease-in-out left;transition:.6s ease-in-out left}.carousel-inner>.item>a>img,.carousel-inner>.item>img{line-height:1}@media all and (transform-3d),(-webkit-transform-3d){.carousel-inner>.item{-webkit-transition:-webkit-transform .6s ease-in-out;-o-transition:-o-transform .6s ease-in-out;transition:transform .6s ease-in-out;-webkit-backface-visibility:hidden;backface-visibility:hidden;-webkit-perspective:1000px;perspective:1000px}.carousel-inner>.item.active.right,.carousel-inner>.item.next{left:0;-webkit-transform:translate3d(100%,0,0);transform:translate3d(100%,0,0)}.carousel-inner>.item.active.left,.carousel-inner>.item.prev{left:0;-webkit-transform:translate3d(-100%,0,0);transform:translate3d(-100%,0,0)}.carousel-inner>.item.active,.carousel-inner>.item.next.left,.carousel-inner>.item.prev.right{left:0;-webkit-transform:translate3d(0,0,0);transform:translate3d(0,0,0)}}.carousel-inner>.active,.carousel-inner>.next,.carousel-inner>.prev{display:block}.carousel-inner>.active{left:0}.carousel-inner>.next,.carousel-inner>.prev{position:absolute;top:0;width:100%}.carousel-inner>.next{left:100%}.carousel-inner>.prev{left:-100%}.carousel-inner>.next.left,.carousel-inner>.prev.right{left:0}.carousel-inner>.active.left{left:-100%}.carousel-inner>.active.right{left:100%}.carousel-control{position:absolute;top:0;bottom:0;left:0;width:15%;font-size:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6);background-color:rgba(0,0,0,0);filter:alpha(opacity=50);opacity:.5}.carousel-control.left{background-image:-webkit-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.5)),to(rgba(0,0,0,.0001)));background-image:linear-gradient(to right,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#80000000', endColorstr='#00000000', GradientType=1);background-repeat:repeat-x}.carousel-control.right{right:0;left:auto;background-image:-webkit-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.0001)),to(rgba(0,0,0,.5)));background-image:linear-gradient(to right,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#00000000', endColorstr='#80000000', GradientType=1);background-repeat:repeat-x}.carousel-control:focus,.carousel-control:hover{color:#fff;text-decoration:none;filter:alpha(opacity=90);outline:0;opacity:.9}.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{position:absolute;top:50%;z-index:5;display:inline-block;margin-top:-10px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{left:50%;margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{right:50%;margin-right:-10px}.carousel-control .icon-next,.carousel-control .icon-prev{width:20px;height:20px;font-family:serif;line-height:1}.carousel-control .icon-prev:before{content:'\2039'}.carousel-control .icon-next:before{content:'\203a'}.carousel-indicators{position:absolute;bottom:10px;left:50%;z-index:15;width:60%;padding-left:0;margin-left:-30%;text-align:center;list-style:none}.carousel-indicators li{display:inline-block;width:10px;height:10px;margin:1px;text-indent:-999px;cursor:pointer;background-color:#000\9;background-color:rgba(0,0,0,0);border:1px solid #fff;border-radius:10px}.carousel-indicators .active{width:12px;height:12px;margin:0;background-color:#fff}.carousel-caption{position:absolute;right:15%;bottom:20px;left:15%;z-index:10;padding-top:20px;padding-bottom:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6)}.carousel-caption .btn{text-shadow:none}@media screen and (min-width:768px){.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{width:30px;height:30px;margin-top:-10px;font-size:30px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{margin-right:-10px}.carousel-caption{right:20%;left:20%;padding-bottom:30px}.carousel-indicators{bottom:20px}}.btn-group-vertical>.btn-group:after,.btn-group-vertical>.btn-group:before,.btn-toolbar:after,.btn-toolbar:before,.clearfix:after,.clearfix:before,.container-fluid:after,.container-fluid:before,.container:after,.container:before,.dl-horizontal dd:after,.dl-horizontal dd:before,.form-horizontal .form-group:after,.form-horizontal .form-group:before,.modal-footer:after,.modal-footer:before,.modal-header:after,.modal-header:before,.nav:after,.nav:before,.navbar-collapse:after,.navbar-collapse:before,.navbar-header:after,.navbar-header:before,.navbar:after,.navbar:before,.pager:after,.pager:before,.panel-body:after,.panel-body:before,.row:after,.row:before{display:table;content:" "}.btn-group-vertical>.btn-group:after,.btn-toolbar:after,.clearfix:after,.container-fluid:after,.container:after,.dl-horizontal dd:after,.form-horizontal .form-group:after,.modal-footer:after,.modal-header:after,.nav:after,.navbar-collapse:after,.navbar-header:after,.navbar:after,.pager:after,.panel-body:after,.row:after{clear:both}.center-block{display:block;margin-right:auto;margin-left:auto}.pull-right{float:right!important}.pull-left{float:left!important}.hide{display:none!important}.show{display:block!important}.invisible{visibility:hidden}.text-hide{font:0/0 a;color:transparent;text-shadow:none;background-color:transparent;border:0}.hidden{display:none!important}.affix{position:fixed}@-ms-viewport{width:device-width}.visible-lg,.visible-md,.visible-sm,.visible-xs{display:none!important}.visible-lg-block,.visible-lg-inline,.visible-lg-inline-block,.visible-md-block,.visible-md-inline,.visible-md-inline-block,.visible-sm-block,.visible-sm-inline,.visible-sm-inline-block,.visible-xs-block,.visible-xs-inline,.visible-xs-inline-block{display:none!important}@media (max-width:767px){.visible-xs{display:block!important}table.visible-xs{display:table!important}tr.visible-xs{display:table-row!important}td.visible-xs,th.visible-xs{display:table-cell!important}}@media (max-width:767px){.visible-xs-block{display:block!important}}@media (max-width:767px){.visible-xs-inline{display:inline!important}}@media (max-width:767px){.visible-xs-inline-block{display:inline-block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm{display:block!important}table.visible-sm{display:table!important}tr.visible-sm{display:table-row!important}td.visible-sm,th.visible-sm{display:table-cell!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-block{display:block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline{display:inline!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline-block{display:inline-block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md{display:block!important}table.visible-md{display:table!important}tr.visible-md{display:table-row!important}td.visible-md,th.visible-md{display:table-cell!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-block{display:block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline{display:inline!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline-block{display:inline-block!important}}@media (min-width:1200px){.visible-lg{display:block!important}table.visible-lg{display:table!important}tr.visible-lg{display:table-row!important}td.visible-lg,th.visible-lg{display:table-cell!important}}@media (min-width:1200px){.visible-lg-block{display:block!important}}@media (min-width:1200px){.visible-lg-inline{display:inline!important}}@media (min-width:1200px){.visible-lg-inline-block{display:inline-block!important}}@media (max-width:767px){.hidden-xs{display:none!important}}@media (min-width:768px) and (max-width:991px){.hidden-sm{display:none!important}}@media (min-width:992px) and (max-width:1199px){.hidden-md{display:none!important}}@media (min-width:1200px){.hidden-lg{display:none!important}}.visible-print{display:none!important}@media print{.visible-print{display:block!important}table.visible-print{display:table!important}tr.visible-print{display:table-row!important}td.visible-print,th.visible-print{display:table-cell!important}}.visible-print-block{display:none!important}@media print{.visible-print-block{display:block!important}}.visible-print-inline{display:none!important}@media print{.visible-print-inline{display:inline!important}}.visible-print-inline-block{display:none!important}@media print{.visible-print-inline-block{display:inline-block!important}}@media print{.hidden-print{display:none!important}}
        /*# sourceMappingURL=bootstrap.min.css.map */
    </style>
    <!--<link rel="stylesheet" type="text/css" media="all" href="../../resources/bootstrap-3.2.0/css/bootstrap.min.css" th:href="@{resources/bootstrap-3.2.0/css/bootstrap.min.css}"/>-->
</head>
<body>
<div id="content" class="container">
    <h2>
        Exomiser Analysis Results for
        proband
    </h2>
    <ul class="nav nav-pills" id="tabs">
        <li><a href="#settings" role="tab" data-toggle="tab"><b>Analysis Settings</b></a></li>
        <li><a href="#filtering" role="tab" data-toggle="tab"><b>Filtering Summary</b></a></li>
        <li><a href="#distribution" role="tab" data-toggle="tab"><b>Variant Type Distribution</b></a></li>
        <li><a href="#prioritisation" role="tab" data-toggle="tab"><b>Prioritised Genes</b></a></li>
        
        <li><a href="#about" role="tab" data-toggle="tab"><b>About</b></a></li>
    </ul>
    <div class="panel panel-default" id="settings">
        <div class="panel-heading">
            <h3>Analysis Settings</h3>
        </div>
        <div class="panel-body"><p>Settings used for this analysis:</p>
            <pre>---
vcf: &quot;/mnt/scratch/Public/TRAINING/GenomeAnalysisModule/CaseInformation/CaseFiles/Case3/Case3_Merged.vcf.gz&quot;
genomeAssembly: &quot;HG38&quot;
pedigree:
  individuals:
  - familyId: &quot;FAM&quot;
    id: &quot;proband&quot;
    motherId: &quot;mother&quot;
    fatherId: &quot;father&quot;
    sex: &quot;FEMALE&quot;
    status: &quot;AFFECTED&quot;
  - familyId: &quot;FAM&quot;
    id: &quot;mother&quot;
    motherId: &quot;&quot;
    fatherId: &quot;&quot;
    sex: &quot;FEMALE&quot;
    status: &quot;UNAFFECTED&quot;
  - familyId: &quot;FAM&quot;
    id: &quot;father&quot;
    motherId: &quot;&quot;
    fatherId: &quot;&quot;
    sex: &quot;MALE&quot;
    status: &quot;UNAFFECTED&quot;
proband: &quot;proband&quot;
hpoIds:
- &quot;HP:0000964&quot;
- &quot;HP:0001047&quot;
- &quot;HP:0001581&quot;
- &quot;HP:0004386&quot;
- &quot;HP:0002090&quot;
- &quot;HP:0002205&quot;
inheritanceModes:
  maxFreq: 2.0
  empty: false
  definedModes:
  - &quot;AUTOSOMAL_DOMINANT&quot;
  - &quot;AUTOSOMAL_RECESSIVE&quot;
  - &quot;X_RECESSIVE&quot;
  - &quot;X_DOMINANT&quot;
  - &quot;MITOCHONDRIAL&quot;
  definedSubModes:
  - &quot;AUTOSOMAL_DOMINANT&quot;
  - &quot;AUTOSOMAL_RECESSIVE_COMP_HET&quot;
  - &quot;AUTOSOMAL_RECESSIVE_HOM_ALT&quot;
  - &quot;X_RECESSIVE_COMP_HET&quot;
  - &quot;X_RECESSIVE_HOM_ALT&quot;
  - &quot;X_DOMINANT&quot;
  - &quot;MITOCHONDRIAL&quot;
analysisMode: &quot;PASS_ONLY&quot;
frequencySources:
- &quot;THOUSAND_GENOMES&quot;
- &quot;TOPMED&quot;
- &quot;UK10K&quot;
- &quot;ESP_AFRICAN_AMERICAN&quot;
- &quot;ESP_EUROPEAN_AMERICAN&quot;
- &quot;ESP_ALL&quot;
- &quot;EXAC_AFRICAN_INC_AFRICAN_AMERICAN&quot;
- &quot;EXAC_AMERICAN&quot;
- &quot;EXAC_EAST_ASIAN&quot;
- &quot;EXAC_FINNISH&quot;
- &quot;EXAC_NON_FINNISH_EUROPEAN&quot;
- &quot;EXAC_OTHER&quot;
- &quot;EXAC_SOUTH_ASIAN&quot;
- &quot;GNOMAD_E_AFR&quot;
- &quot;GNOMAD_E_AMR&quot;
- &quot;GNOMAD_E_ASJ&quot;
- &quot;GNOMAD_E_EAS&quot;
- &quot;GNOMAD_E_FIN&quot;
- &quot;GNOMAD_E_NFE&quot;
- &quot;GNOMAD_E_OTH&quot;
- &quot;GNOMAD_E_SAS&quot;
- &quot;GNOMAD_G_AFR&quot;
- &quot;GNOMAD_G_AMR&quot;
- &quot;GNOMAD_G_ASJ&quot;
- &quot;GNOMAD_G_EAS&quot;
- &quot;GNOMAD_G_FIN&quot;
- &quot;GNOMAD_G_NFE&quot;
- &quot;GNOMAD_G_OTH&quot;
- &quot;GNOMAD_G_SAS&quot;
pathogenicitySources:
- &quot;POLYPHEN&quot;
- &quot;MUTATION_TASTER&quot;
- &quot;SIFT&quot;
- &quot;CADD&quot;
analysisSteps:
- offTargetVariantTypes:
  - &quot;CODING_TRANSCRIPT_INTRON_VARIANT&quot;
  - &quot;FIVE_PRIME_UTR_EXON_VARIANT&quot;
  - &quot;THREE_PRIME_UTR_EXON_VARIANT&quot;
  - &quot;FIVE_PRIME_UTR_INTRON_VARIANT&quot;
  - &quot;THREE_PRIME_UTR_INTRON_VARIANT&quot;
  - &quot;NON_CODING_TRANSCRIPT_EXON_VARIANT&quot;
  - &quot;NON_CODING_TRANSCRIPT_INTRON_VARIANT&quot;
  - &quot;UPSTREAM_GENE_VARIANT&quot;
  - &quot;DOWNSTREAM_GENE_VARIANT&quot;
  - &quot;INTERGENIC_VARIANT&quot;
  - &quot;REGULATORY_REGION_VARIANT&quot;
- decoratedFilter:
    maxFreq: 2.0
- decoratedFilter:
    keepNonPathogenic: true
- compatibleModes:
  - &quot;AUTOSOMAL_DOMINANT&quot;
  - &quot;AUTOSOMAL_RECESSIVE&quot;
  - &quot;X_RECESSIVE&quot;
  - &quot;X_DOMINANT&quot;
  - &quot;MITOCHONDRIAL&quot;
- priorityType: &quot;OMIM_PRIORITY&quot;
- priorityType: &quot;HIPHIVE_PRIORITY&quot;
---
outputContributingVariantsOnly: false
outputPrefix: &quot;Case3_exomiser&quot;
outputFormats:
- &quot;HTML&quot;
- &quot;VCF&quot;
- &quot;TSV_GENE&quot;
- &quot;TSV_VARIANT&quot;
- &quot;JSON&quot;
numGenes: 0
</pre>
        </div>
    </div>

    <div class="panel panel-default" id="filtering">
        <div class="panel-heading">
            <h3>Filtering Summary</h3>
        </div>
        <div class="panel-body">
            <table class="table">
                <thead>
                <tr>
                    <th>Filter</th>
                    <th>Report</th>
                    <th>Passed filter</th>
                    <th>Failed filter</th>
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td><b>Variant effect</b></td>
                    <td>
                        <ul>Removed variants with effects of type: [CODING_TRANSCRIPT_INTRON_VARIANT, FIVE_PRIME_UTR_EXON_VARIANT, THREE_PRIME_UTR_EXON_VARIANT, FIVE_PRIME_UTR_INTRON_VARIANT, THREE_PRIME_UTR_INTRON_VARIANT, NON_CODING_TRANSCRIPT_EXON_VARIANT, NON_CODING_TRANSCRIPT_INTRON_VARIANT, UPSTREAM_GENE_VARIANT, DOWNSTREAM_GENE_VARIANT, INTERGENIC_VARIANT, REGULATORY_REGION_VARIANT]</ul>
                    </td>
                    <td>161</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Frequency</b></td>
                    <td>
                        <ul>Variants filtered for maximum allele frequency of 2.00%</ul>
                    </td>
                    <td>161</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Pathogenicity</b></td>
                    <td>
                        <ul>Retained all non-pathogenic variants of all types. Scoring was applied, but the filter passed all variants.</ul>
                    </td>
                    <td>161</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Inheritance</b></td>
                    <td>
                        <ul>Genes filtered for compatibility with AUTOSOMAL_DOMINANT, AUTOSOMAL_RECESSIVE, X_RECESSIVE, X_DOMINANT, MITOCHONDRIAL inheritance.</ul>
                    </td>
                    <td>90</td>
                    <td>0</td>
                </tr>
                </tbody>
            </table>
        </div>
    </div>
    <div class="panel panel-default" id="distribution">
        <div class="panel-heading">
            <h3>Variant Type Distribution</h3>
        </div>
        <div class="panel-body">
            <table class="table-striped table-condensed">
                <thead>
                <tr>
                    <th>Variant Type</th>
                    
                        <th>proband</th>
                    
                        <th>mother</th>
                    
                        <th>father</th>
                    
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td>FRAMESHIFT_ELONGATION</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_VARIANT</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INTERNAL_FEATURE_ELONGATION</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FEATURE_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>MNV</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>STOP_GAINED</td>
                    
                        <td>4</td>
                    
                        <td>2</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>STOP_LOST</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>START_LOST</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_ACCEPTOR_VARIANT</td>
                    
                        <td>2</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_DONOR_VARIANT</td>
                    
                        <td>2</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>MISSENSE_VARIANT</td>
                    
                        <td>97</td>
                    
                        <td>32</td>
                    
                        <td>39</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_INSERTION</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_INSERTION</td>
                    
                        <td>2</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_DELETION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_DELETION</td>
                    
                        <td>3</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_REGION_VARIANT</td>
                    
                        <td>10</td>
                    
                        <td>3</td>
                    
                        <td>3</td>
                    
                </tr>
                <tr>
                    <td>STOP_RETAINED_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INITIATOR_CODON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SYNONYMOUS_VARIANT</td>
                    
                        <td>37</td>
                    
                        <td>16</td>
                    
                        <td>14</td>
                    
                </tr>
                <tr>
                    <td>CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>UPSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DOWNSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INTERGENIC_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>REGULATORY_REGION_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                </tbody>
            </table>
        </div>
    </div>

    

    <div class="panel panel-default" id="prioritisation">
        <div class="panel-heading">
            <h3>Prioritised Genes</h3>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196218">RYR1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.974</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.750</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.750 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98905">Congenital multicore myopathy with external ophthalmoplegia</a> associated with RYR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0002090, Pneumonia</dd><dd>HP:0001047, Atopic dermatitis - HP:0002090, Pneumonia</dd><dd>HP:0001581, Recurrent skin infections - HP:0002090, Pneumonia</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002090, Pneumonia</dd><dd>HP:0002090, Pneumonia - HP:0002090, Pneumonia</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002205, Recurrent respiratory infections</dd></dl><dl><dt>Phenotypic similarity 0.296 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RYR1">RYR1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0001191, abnormal skin condition</dd><dd>HP:0001047, Atopic dermatitis - MP:0001191, abnormal skin condition</dd><dd>HP:0001581, Recurrent skin infections - MP:0001191, abnormal skin condition</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0009890, cleft secondary palate</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RYR1%0DPSEN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PSEN1</a> and phenotypic similarity 0.717 to <a href="http://www.omim.org/613737">?Acne inversa, familial, 3</a> associated with PSEN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0012322, Perifolliculitis</dd><dd>HP:0001047, Atopic dermatitis - HP:0012322, Perifolliculitis</dd><dd>HP:0001581, Recurrent skin infections - HP:0011132, Chronic furunculosis</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0012322, Perifolliculitis</dd><dd>HP:0002090, Pneumonia - HP:0012322, Perifolliculitis</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0011132, Chronic furunculosis</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RYR1%0DPSEN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PSEN1</a> and phenotypic similarity 0.376 to mouse mutant of PSEN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0005421, loose skin</dd><dd>HP:0001047, Atopic dermatitis - MP:0005421, loose skin</dd><dd>HP:0001581, Recurrent skin infections - MP:0005421, loose skin</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia - MP:0002270, abnormal pulmonary alveolus morphology</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0002270, abnormal pulmonary alveolus morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/117000">OMIM:117000</a> Central core disease - autosomal dominant/recessive</dd>
<dd><a href="http://www.omim.org/entry/145600">OMIM:145600</a> King-Denborough syndrome - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/145600">OMIM:145600</a> Malignant hyperthermia susceptibility 1 (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/255320">OMIM:255320</a> Minicore myopathy with external ophthalmoplegia - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=169186">ORPHA:169186</a> Autosomal recessive centronuclear myopathy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=169189">ORPHA:169189</a> Autosomal dominant centronuclear myopathy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=178145">ORPHA:178145</a> Moderate multiminicore disease with hand involvement</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=423">ORPHA:423</a> Malignant hyperthermia of anesthesia</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=597">ORPHA:597</a> Central core disease</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98905">ORPHA:98905</a> Congenital multicore myopathy with external ophthalmoplegia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.974</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.750</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.38496416G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537994744">rs537994744</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537994744&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=468693%5Balleleid%5D">(criteria_provided,_conflicting_interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.995</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355481.8">RYR1:ENST00000355481.8:c.6671G&gt;A:p.(Arg2224His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359596.8">RYR1:ENST00000359596.8:c.6671G&gt;A:p.(Arg2224His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9981379</dd>
                                    
                                    <dd>Polyphen2: 0.903 (P)</dd>
                                    <dd>Mutation Taster: 0.988 (P)</dd>
                                    <dd>SIFT: 0.124 (T)</dd>
                                    <dd>CADD: 0.998 (27.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.38517416G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=553187437">rs553187437</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs553187437&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=847631%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355481.8">RYR1:ENST00000355481.8:c.9743G&gt;A:p.(Arg3248Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359596.8">RYR1:ENST00000359596.8:c.9743G&gt;A:p.(Arg3248Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9943766</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.588 (T)</dd>
                                    <dd>CADD: 0.994 (22.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000172175">MALT1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.891</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.597</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.597 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=52417">MALT lymphoma</a> associated with MALT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0012123, Posterior uveitis</dd><dd>HP:0001047, Atopic dermatitis - HP:0012123, Posterior uveitis</dd><dd>HP:0001581, Recurrent skin infections - HP:0012123, Posterior uveitis</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0012123, Posterior uveitis</dd><dd>HP:0002090, Pneumonia - HP:0002205, Recurrent respiratory infections</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002205, Recurrent respiratory infections</dd></dl><dl><dt>Proximity score 0.524 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MALT1%0DCARD9&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CARD9</a> and phenotypic similarity 0.638 to <a href="http://www.omim.org/212050">Candidiasis, familial, 2, autosomal recessive</a> associated with CARD9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0009098, Chronic oral candidiasis</dd><dd>HP:0001047, Atopic dermatitis - HP:0009098, Chronic oral candidiasis</dd><dd>HP:0001581, Recurrent skin infections - HP:0009098, Chronic oral candidiasis</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0001287, Meningitis</dd><dd>HP:0002090, Pneumonia - HP:0001287, Meningitis</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0012203, Onychomycosis</dd></dl><dl><dt>Proximity score 0.524 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MALT1%0DCARD9&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CARD9</a> and phenotypic similarity 0.377 to mouse mutant of CARD9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0002412, increased susceptibility to bacterial infection</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/615468">OMIM:615468</a> Immunodeficiency 12 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=52417">ORPHA:52417</a> MALT lymphoma</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.891</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.597</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_ELONGATION</span>
                                <b>chr18:g.58700552T&gt;TC [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1602300615">rs1602300615</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1602300615&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-danger">PATHOGENIC</span>
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=647356%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                <span class="label label-danger">WHITELIST VARIANT</span>
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000649217.2">MALT1:ENST00000649217.2:c.611dup:p.(Gln205Thrfs*16)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000345724.7">MALT1:ENST00000345724.7:c.611dup:p.(Gln205Thrfs*16)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000057294">PKP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.785</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.519</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.192 to <a href="http://www.omim.org/609040">Arrhythmogenic right ventricular dysplasia 9</a> associated with PKP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0002617, Dilatation</dd><dd>HP:0001047, Atopic dermatitis - HP:0002617, Dilatation</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl><dt>Proximity score 0.519 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKP2%0DDSC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DSC1</a> and phenotypic similarity 0.617 to mouse mutant of DSC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0001194, dermatitis</dd><dd>HP:0001047, Atopic dermatitis - MP:0001194, dermatitis</dd><dd>HP:0001581, Recurrent skin infections - MP:0001194, dermatitis</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0001194, dermatitis</dd><dd>HP:0002090, Pneumonia - MP:0001194, dermatitis</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/609040">OMIM:609040</a> Arrhythmogenic right ventricular dysplasia 9 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=130">ORPHA:130</a> Brugada syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.785</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.519</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.32896545T&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=886049324">rs886049324</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs886049324&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=332170%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000070846.10">PKP2:ENST00000070846.10:c.187A&gt;C:p.(Thr63Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340811.8">PKP2:ENST00000340811.8:c.187A&gt;C:p.(Thr63Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99993</dd>
                                    
                                    <dd>Polyphen2: 0.834 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.998 (26.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr12:g.32896551G&gt;T [0/1:0/0:0/0]</b>
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.995</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000070846.10">PKP2:ENST00000070846.10:c.181C&gt;A:p.(Gln61Lys)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340811.8">PKP2:ENST00000340811.8:c.181C&gt;A:p.(Gln61Lys)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.99498814</dd>
                        
                        <dd>Polyphen2: 0.104 (B)</dd>
                        <dd>Mutation Taster: 0.994 (P)</dd>
                        <dd>SIFT: 0.018 (D)</dd>
                        <dd>CADD: 0.995 (23.000)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000161956">SENP3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.763</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SENP3%0DKANSL1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KANSL1</a> and phenotypic similarity 0.681 to <a href="http://www.omim.org/610443">Koolen-De Vries syndrome</a> associated with KANSL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0000964, Eczema</dd><dd>HP:0001047, Atopic dermatitis - HP:0000964, Eczema</dd><dd>HP:0001581, Recurrent skin infections - HP:0000964, Eczema</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0000964, Eczema</dd><dd>HP:0002090, Pneumonia - HP:0000964, Eczema</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0001601, Laryngomalacia</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SENP3%0DKANSL1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KANSL1</a> and phenotypic similarity 0.229 to mouse mutant of KANSL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0002075, abnormal coat/hair pigmentation</dd><dd>HP:0001047, Atopic dermatitis - MP:0002075, abnormal coat/hair pigmentation</dd><dd>HP:0001581, Recurrent skin infections - MP:0002075, abnormal coat/hair pigmentation</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.763</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_ACCEPTOR_VARIANT</span>
                                <b>chr17:g.7571876TATATATATATATATATATATATATATATATATAAAA&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1245768632">rs1245768632</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1245768632&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000429205.6">SENP3:ENST00000429205.6:c.*387+7_*391del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614237.1">SENP3:ENST00000614237.1:c.1511+398_1512-372del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000321337.12">SENP3:ENST00000321337.12:c.*401_*436del:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9614522</dd>
                                    
                                    <dd>CADD: 0.961 (14.140)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000091622">PITPNM3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.761</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.999</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PITPNM3%0DCCL18&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCL18</a> and phenotypic similarity 0.637 to mouse mutant of CCL18.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0009481, cecum inflammation</dd><dd>HP:0001047, Atopic dermatitis - MP:0009481, cecum inflammation</dd><dd>HP:0001581, Recurrent skin infections - MP:0009481, cecum inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0009481, cecum inflammation</dd><dd>HP:0002090, Pneumonia - MP:0009481, cecum inflammation</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0002411, decreased susceptibility to bacterial infection</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/600977">OMIM:600977</a> Cone-rod dystrophy 5 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1872">ORPHA:1872</a> Cone rod dystrophy</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.761</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.999</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.6455641G&gt;C [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.999</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262483.13">PITPNM3:ENST00000262483.13:c.2622C&gt;G:p.(Phe874Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421306.7">PITPNM3:ENST00000421306.7:c.2514C&gt;G:p.(Phe838Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99920565</dd>
                                    
                                    <dd>Polyphen2: 0.936 (P)</dd>
                                    <dd>Mutation Taster: 0.983 (P)</dd>
                                    <dd>SIFT: 0.046 (D)</dd>
                                    <dd>CADD: 0.999 (31.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000161021">MAML1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.752</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAML1%0DE2F3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to E2F3</a> and phenotypic similarity 0.719 to mouse mutant of E2F3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0001861, lung inflammation</dd><dd>HP:0001047, Atopic dermatitis - MP:0001861, lung inflammation</dd><dd>HP:0001581, Recurrent skin infections - MP:0001861, lung inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0001861, lung inflammation</dd><dd>HP:0002090, Pneumonia - MP:0001861, lung inflammation</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0001861, lung inflammation</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.752</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_VARIANT</span>
                                <b>chr5:g.179733384G&gt;GC [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000292599.4">MAML1:ENST00000292599.4:c.274dup:p.(Leu92Profs*5)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000174225">ARL13A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.749</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARL13A%0DCOG6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COG6</a> and phenotypic similarity 0.626 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=363523">Hypohidrosis-enamel hypoplasia-palmoplantar keratoderma-intellectual disability syndrome</a> associated with COG6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0012115, Hepatitis</dd><dd>HP:0001047, Atopic dermatitis - HP:0012115, Hepatitis</dd><dd>HP:0001581, Recurrent skin infections - HP:0012115, Hepatitis</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0012115, Hepatitis</dd><dd>HP:0002090, Pneumonia - HP:0002205, Recurrent respiratory infections</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002205, Recurrent respiratory infections</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARL13A%0DCOG6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COG6</a> and phenotypic similarity 0.229 to mouse mutant of COG6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0002075, abnormal coat/hair pigmentation</dd><dd>HP:0001047, Atopic dermatitis - MP:0002075, abnormal coat/hair pigmentation</dd><dd>HP:0001581, Recurrent skin infections - MP:0002075, abnormal coat/hair pigmentation</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.749</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_ACCEPTOR_VARIANT</span>
                                <b>chr23:g.100985665A&gt;T [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000450049.7">ARL13A:ENST00000450049.7:c.131-2A&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9929205</dd>
                                    
                                    <dd>CADD: 0.993 (21.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000142599">RERE</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.748</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.999</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.331 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1606">1p36 deletion syndrome</a> associated with RERE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0001743, Abnormality of the spleen</dd><dd>HP:0001047, Atopic dermatitis - HP:0001743, Abnormality of the spleen</dd><dd>HP:0001581, Recurrent skin infections - HP:0001743, Abnormality of the spleen</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002020, Gastroesophageal reflux</dd><dd>HP:0002090, Pneumonia - HP:0000457, Depressed nasal ridge</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0000457, Depressed nasal ridge</dd></dl><dl><dt>Phenotypic similarity 0.275 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RERE">RERE</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia - MP:0000484, abnormal pulmonary artery morphology</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0000484, abnormal pulmonary artery morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RERE%0DBCL11B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BCL11B</a> and phenotypic similarity 0.607 to <a href="http://www.omim.org/617237">Immunodeficiency 49</a> associated with BCL11B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0011123, Inflammatory abnormality of the skin</dd><dd>HP:0001047, Atopic dermatitis - HP:0011123, Inflammatory abnormality of the skin</dd><dd>HP:0001581, Recurrent skin infections - HP:0011123, Inflammatory abnormality of the skin</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0011123, Inflammatory abnormality of the skin</dd><dd>HP:0002090, Pneumonia - HP:0011123, Inflammatory abnormality of the skin</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0004415, Pulmonary artery stenosis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RERE%0DBCL11B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BCL11B</a> and phenotypic similarity 0.409 to mouse mutant of BCL11B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0002796, impaired skin barrier function</dd><dd>HP:0001047, Atopic dermatitis - MP:0002796, impaired skin barrier function</dd><dd>HP:0001581, Recurrent skin infections - MP:0002796, impaired skin barrier function</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0001436, abnormal suckling behavior</dd><dd>HP:0002090, Pneumonia - MP:0001436, abnormal suckling behavior</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/616975">OMIM:616975</a> Neurodevelopmental disorder with or without anomalies of the brain, eye, or heart - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1606">ORPHA:1606</a> 1p36 deletion syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=494344">ORPHA:494344</a> RERE-related neurodevelopmental syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.748</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.999</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.8358208G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1349845013">rs1349845013</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1349845013&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.999</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000337907.7">RERE:ENST00000337907.7:c.4327C&gt;T:p.(Pro1443Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377464.5">RERE:ENST00000377464.5:c.3523C&gt;T:p.(Pro1175Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400908.7">RERE:ENST00000400908.7:c.4327C&gt;T:p.(Pro1443Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000476556.5">RERE:ENST00000476556.5:c.2665C&gt;T:p.(Pro889Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400907.6">RERE:ENST00000400907.6:c.1541-2609C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.933 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.006 (D)</dd>
                                    <dd>CADD: 0.999 (33.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_SAS: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000164199">ADGRV1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.748</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.510</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.988</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.510 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADGRV1%0DCDH23&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDH23</a> and phenotypic similarity 0.721 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=96253">Cushing disease</a> associated with CDH23.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0001061, Acne</dd><dd>HP:0001047, Atopic dermatitis - HP:0001061, Acne</dd><dd>HP:0001581, Recurrent skin infections - HP:0001581, Recurrent skin infections</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0001061, Acne</dd><dd>HP:0002090, Pneumonia - HP:0001061, Acne</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0012203, Onychomycosis</dd></dl><dl><dt>Proximity score 0.510 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADGRV1%0DCDH23&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDH23</a> and phenotypic similarity 0.246 to mouse mutant of CDH23.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0003866, abnormal defecation</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/604352">OMIM:604352</a> ?Febrile seizures, familial, 4 (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/605472">OMIM:605472</a> Usher syndrome, type 2C - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=231178">ORPHA:231178</a> Usher syndrome type 2</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=36387">ORPHA:36387</a> Generalized epilepsy with febrile seizures-plus</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.748</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.510</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.988</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr5:g.90708864G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=547397177">rs547397177</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs547397177&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=229341%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405460.9">ADGRV1:ENST00000405460.9:c.8779G&gt;A:p.(Val2927Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99628466</dd>
                                    
                                    <dd>Polyphen2: 0.566 (P)</dd>
                                    <dd>Mutation Taster: 0.975 (P)</dd>
                                    <dd>SIFT: 0.098 (T)</dd>
                                    <dd>CADD: 0.996 (24.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0096%</dd>
                                    <dd>UK10K: 0.0264%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr5:g.90629418G&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200789563">rs200789563</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200789563&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=55455%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.982</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405460.9">ADGRV1:ENST00000405460.9:c.1718G&gt;T:p.(Gly573Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.976 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.004 (D)</dd>
                                    <dd>CADD: 0.996 (23.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0342%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 0.0536%</dd>
                                    <dd>ESP All: 0.0168%</dd>
                                    <dd>ExAC AFR: 0.0715%</dd>
                                    <dd>ExAC AMR: 0.0696%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>gnomAD_E_AFR: 0.0393%</dd>
                                    <dd>gnomAD_E_AMR: 0.1222%</dd>
                                    <dd>gnomAD_E_NFE: 0.0152%</dd>
                                    <dd>gnomAD_E_OTH: 0.1096%</dd>
                                    <dd>gnomAD_G_AFR: 0.1033%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.1022%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr5:g.90716722G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200792658">rs200792658</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200792658&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                <div class="col-sm-12">
                    <b>ClinVar: </b>
                    
                        
                        
                        
                        
                        
                        
                        <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                        <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=55569%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                    
                </div>
                <div class="col-sm-6">
                    <b>Variant score: 0.982</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405460.9">ADGRV1:ENST00000405460.9:c.9440G&gt;A:p.(Arg3147Gln)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.99981</dd>
                        
                        <dd>Polyphen2: 0.008 (B)</dd>
                        <dd>Mutation Taster: 1.000 (P)</dd>
                        <dd>SIFT: 0.095 (T)</dd>
                        <dd>CADD: 0.992 (21.100)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0399%</dd>
                        <dd>TOPMed: 0.0366%</dd>
                        <dd>ESP AA: 0.0551%</dd>
                        <dd>ESP All: 0.0170%</dd>
                        <dd>ExAC AFR: 0.0717%</dd>
                        <dd>ExAC AMR: 0.0779%</dd>
                        <dd>ExAC NFE: 0.0120%</dd>
                        <dd>gnomAD_E_AFR: 0.0393%</dd>
                        <dd>gnomAD_E_AMR: 0.1252%</dd>
                        <dd>gnomAD_E_NFE: 0.0153%</dd>
                        <dd>gnomAD_E_OTH: 0.1096%</dd>
                        <dd>gnomAD_G_AFR: 0.1031%</dd>
                        <dd>gnomAD_G_NFE: 0.0133%</dd>
                        <dd>gnomAD_G_OTH: 0.1018%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000125498">KIR2DL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.747</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIR2DL1%0DHLA-C&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HLA-C</a> and phenotypic similarity 0.603 to <a href="http://www.omim.org/177900">Psoriasis susceptibility 1</a> associated with HLA-C.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0003765, Psoriasiform dermatitis</dd><dd>HP:0001047, Atopic dermatitis - HP:0003765, Psoriasiform dermatitis</dd><dd>HP:0001581, Recurrent skin infections - HP:0003765, Psoriasiform dermatitis</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0003765, Psoriasiform dermatitis</dd><dd>HP:0002090, Pneumonia - HP:0003765, Psoriasiform dermatitis</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0001369, Arthritis</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.747</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54773492A&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=150190837">rs150190837</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs150190837&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000291633.7">KIR2DL1:ENST00000291633.7:c.230A&gt;T:p.(His77Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336077.11">KIR2DL1:ENST00000336077.11:c.230A&gt;T:p.(His77Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.998</dd>
                                    
                                    <dd>Polyphen2: 0.147 (B)</dd>
                                    <dd>SIFT: 0.002 (D)</dd>
                                    <dd>CADD: 0.782 (6.616)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC AFR: 0.0098%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_NFE: 0.0028%</dd>
                                    <dd>gnomAD_G_AFR: 0.0131%</dd>
                                    <dd>gnomAD_G_NFE: 0.0077%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183117">CSMD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.739</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSMD1%0DHOXA5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HOXA5</a> and phenotypic similarity 0.687 to mouse mutant of HOXA5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0001861, lung inflammation</dd><dd>HP:0001047, Atopic dermatitis - MP:0001861, lung inflammation</dd><dd>HP:0001581, Recurrent skin infections - MP:0001861, lung inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0001861, lung inflammation</dd><dd>HP:0002090, Pneumonia - MP:0001861, lung inflammation</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0001861, lung inflammation</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.739</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr8:g.3409566G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=746714804">rs746714804</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs746714804&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400186.7">CSMD1:ENST00000400186.7:c.1604C&gt;T:p.(Ala535Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000520002.5">CSMD1:ENST00000520002.5:c.1604C&gt;T:p.(Ala535Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537824.2">CSMD1:ENST00000537824.2:c.1187C&gt;T:p.(Ala396Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602557.5">CSMD1:ENST00000602557.5:c.1604C&gt;T:p.(Ala535Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602723.5">CSMD1:ENST00000602723.5:c.1604C&gt;T:p.(Ala535Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000635120.2">CSMD1:ENST00000635120.2:c.1601C&gt;T:p.(Ala534Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9942456</dd>
                                    
                                    <dd>Polyphen2: 0.994 (D)</dd>
                                    <dd>SIFT: 0.373 (T)</dd>
                                    <dd>CADD: 0.994 (22.400)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197653">DNAH10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.715</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.981</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAH10%0DDNAH9&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DNAH9</a> and phenotypic similarity 0.701 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with DNAH9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0002090, Pneumonia</dd><dd>HP:0001047, Atopic dermatitis - HP:0002090, Pneumonia</dd><dd>HP:0001581, Recurrent skin infections - HP:0002090, Pneumonia</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002090, Pneumonia</dd><dd>HP:0002090, Pneumonia - HP:0002090, Pneumonia</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002205, Recurrent respiratory infections</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.715</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.981</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.123870461G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1348986936">rs1348986936</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1348986936&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.981</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409039.8">DNAH10:ENST00000409039.8:c.7444G&gt;A:p.(Glu2482Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000638045.1">DNAH10:ENST00000638045.1:c.7261G&gt;A:p.(Glu2421Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000673944.1">DNAH10:ENST00000673944.1:c.7615G&gt;A:p.(Glu2539Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9811201</dd>
                                    
                                    <dd>Polyphen2: 0.009 (B)</dd>
                                    <dd>Mutation Taster: 0.922</dd>
                                    <dd>CADD: 0.981 (17.240)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000124140">SLC12A5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.704</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.975</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.158 to <a href="http://www.omim.org/616645">Epileptic encephalopathy, early infantile, 34</a> associated with SLC12A5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0003781, Excessive salivation</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl><dt>Phenotypic similarity 0.317 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SLC12A5">SLC12A5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia - MP:0001177, atelectasis</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0001177, atelectasis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC12A5%0DGLRB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GLRB</a> and phenotypic similarity 0.652 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3197">Hereditary hyperekplexia</a> associated with GLRB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0100633, Esophagitis</dd><dd>HP:0001047, Atopic dermatitis - HP:0100633, Esophagitis</dd><dd>HP:0001581, Recurrent skin infections - HP:0100633, Esophagitis</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0100633, Esophagitis</dd><dd>HP:0002090, Pneumonia - HP:0100633, Esophagitis</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0100633, Esophagitis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC12A5%0DGLRB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GLRB</a> and phenotypic similarity 0.210 to mouse mutant of GLRB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0000743, muscle spasm</dd><dd>HP:0001047, Atopic dermatitis - MP:0000743, muscle spasm</dd><dd>HP:0001581, Recurrent skin infections - MP:0000743, muscle spasm</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0000743, muscle spasm</dd><dd>HP:0002090, Pneumonia - MP:0000743, muscle spasm</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/616645">OMIM:616645</a> Epileptic encephalopathy, early infantile, 34 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/616685">OMIM:616685</a> Epilepsy, idiopathic generalized, susceptibility to, 14 (susceptibility)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.704</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.975</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">INFRAME_INSERTION</span>
                                <b>chr20:g.46022943G&gt;GGGA [1/1:-/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=34923327">rs34923327</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs34923327&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.975</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000626701.1">SLC12A5:ENST00000626701.1:c.253_255dup:p.(Gly85dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454036.6">SLC12A5:ENST00000454036.6:c.121+1057_121+1058insGGA:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000628272.1">SLC12A5:ENST00000628272.1:c.122-426_122-425insGGA:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97499657</dd>
                                    
                                    <dd>CADD: 0.975 (16.020)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170748">RBMXL2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.704</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.975</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RBMXL2%0DSIRT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SIRT1</a> and phenotypic similarity 0.727 to mouse mutant of SIRT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0001861, lung inflammation</dd><dd>HP:0001047, Atopic dermatitis - MP:0001861, lung inflammation</dd><dd>HP:0001581, Recurrent skin infections - MP:0001861, lung inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0001861, lung inflammation</dd><dd>HP:0002090, Pneumonia - MP:0001861, lung inflammation</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0001861, lung inflammation</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.704</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.975</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.7089431G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=776539189">rs776539189</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs776539189&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.975</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000306904.7">RBMXL2:ENST00000306904.7:c.311G&gt;A:p.(Arg104His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9753396</dd>
                                    
                                    <dd>Polyphen2: 0.114 (B)</dd>
                                    <dd>SIFT: 0.091 (T)</dd>
                                    <dd>CADD: 0.975 (16.080)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183091">NEB</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.652</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.948</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.329 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=171430">Severe congenital nemaline myopathy</a> associated with NEB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002015, Dysphagia</dd><dd>HP:0002090, Pneumonia - HP:0002089, Pulmonary hypoplasia</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002089, Pulmonary hypoplasia</dd></dl><dl><dt>Phenotypic similarity 0.237 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NEB">NEB</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0001436, abnormal suckling behavior</dd><dd>HP:0001047, Atopic dermatitis - MP:0001436, abnormal suckling behavior</dd><dd>HP:0001581, Recurrent skin infections - MP:0001436, abnormal suckling behavior</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0001436, abnormal suckling behavior</dd><dd>HP:0002090, Pneumonia - MP:0001436, abnormal suckling behavior</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NEB%0DLRBA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRBA</a> and phenotypic similarity 0.669 to <a href="http://www.omim.org/614700">Immunodeficiency, common variable, 8, with autoimmunity</a> associated with LRBA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0002090, Pneumonia</dd><dd>HP:0001047, Atopic dermatitis - HP:0002090, Pneumonia</dd><dd>HP:0001581, Recurrent skin infections - HP:0002090, Pneumonia</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0005263, Gastritis</dd><dd>HP:0002090, Pneumonia - HP:0002090, Pneumonia</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002090, Pneumonia</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NEB%0DLRBA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRBA</a> and phenotypic similarity 0.597 to mouse mutant of LRBA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0008537, increased susceptibility to induced colitis</dd><dd>HP:0001047, Atopic dermatitis - MP:0008537, increased susceptibility to induced colitis</dd><dd>HP:0001581, Recurrent skin infections - MP:0008537, increased susceptibility to induced colitis</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0008537, increased susceptibility to induced colitis</dd><dd>HP:0002090, Pneumonia - MP:0008537, increased susceptibility to induced colitis</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/256030">OMIM:256030</a> Nemaline myopathy 2, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=171430">ORPHA:171430</a> Severe congenital nemaline myopathy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=171433">ORPHA:171433</a> Intermediate nemaline myopathy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=171436">ORPHA:171436</a> Typical nemaline myopathy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=171439">ORPHA:171439</a> Childhood-onset nemaline myopathy</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.652</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.948</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.151666287G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200545007">rs200545007</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200545007&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=85033%5Balleleid%5D">(criteria_provided,_conflicting_interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.951</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000172853.14">NEB:ENST00000172853.14:c.4834C&gt;T:p.(Arg1612Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397345.7">NEB:ENST00000397345.7:c.4834C&gt;T:p.(Arg1612Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409198.5">NEB:ENST00000409198.5:c.4834C&gt;T:p.(Arg1612Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427231.6">NEB:ENST00000427231.6:c.4834C&gt;T:p.(Arg1612Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000603639.5">NEB:ENST00000603639.5:c.4834C&gt;T:p.(Arg1612Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000604864.5">NEB:ENST00000604864.5:c.4834C&gt;T:p.(Arg1612Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618972.4">NEB:ENST00000618972.4:c.4834C&gt;T:p.(Arg1612Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999998</dd>
                                    
                                    <dd>Polyphen2: 0.946 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.003 (D)</dd>
                                    <dd>CADD: 0.998 (26.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.1075%</dd>
                                    <dd>UK10K: 0.0529%</dd>
                                    <dd>ESP AA: 0.2527%</dd>
                                    <dd>ESP EA: 0.0480%</dd>
                                    <dd>ESP All: 0.1139%</dd>
                                    <dd>ExAC AFR: 0.2448%</dd>
                                    <dd>ExAC AMR: 0.1210%</dd>
                                    <dd>ExAC NFE: 0.0390%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.2747%</dd>
                                    <dd>gnomAD_E_AMR: 0.1549%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0439%</dd>
                                    <dd>gnomAD_E_OTH: 0.1277%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 0.3095%</dd>
                                    <dd>gnomAD_G_AMR: 0.1199%</dd>
                                    <dd>gnomAD_G_NFE: 0.0200%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.151493411G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=571935637">rs571935637</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs571935637&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=628501%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.945</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000172853.14">NEB:ENST00000172853.14:c.19139C&gt;T:p.(Pro6380Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397345.7">NEB:ENST00000397345.7:c.24707C&gt;T:p.(Pro8236Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409198.5">NEB:ENST00000409198.5:c.19139C&gt;T:p.(Pro6380Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427231.6">NEB:ENST00000427231.6:c.24707C&gt;T:p.(Pro8236Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000603639.5">NEB:ENST00000603639.5:c.24707C&gt;T:p.(Pro8236Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000604864.5">NEB:ENST00000604864.5:c.24707C&gt;T:p.(Pro8236Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618972.4">NEB:ENST00000618972.4:c.24812C&gt;T:p.(Pro8271Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.904 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.680 (T)</dd>
                                    <dd>CADD: 0.998 (26.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>ExAC AMR: 0.0443%</dd>
                                    <dd>ExAC FIN: 0.0153%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>ExAC OTH: 0.3417%</dd>
                                    <dd>ExAC SAS: 0.0442%</dd>
                                    <dd>gnomAD_E_AMR: 0.0276%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0619%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr2:g.151662303G&gt;A [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=139963368">rs139963368</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs139963368&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                <div class="col-sm-12">
                    <b>ClinVar: </b>
                    
                        
                        <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                        
                        
                        
                        
                        
                        <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=250274%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                    
                </div>
                <div class="col-sm-6">
                    <b>Variant score: 0.309</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000172853.14">NEB:ENST00000172853.14:c.5802C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397345.7">NEB:ENST00000397345.7:c.5802C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409198.5">NEB:ENST00000409198.5:c.5802C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427231.6">NEB:ENST00000427231.6:c.5802C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000603639.5">NEB:ENST00000603639.5:c.5802C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000604864.5">NEB:ENST00000604864.5:c.5802C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618972.4">NEB:ENST00000618972.4:c.5802C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.85405135</dd>
                        
                        <dd>CADD: 0.854 (8.358)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.6589%</dd>
                        <dd>TOPMed: 0.5041%</dd>
                        <dd>UK10K: 0.1190%</dd>
                        <dd>ESP AA: 1.4301%</dd>
                        <dd>ESP EA: 0.1585%</dd>
                        <dd>ESP All: 0.5543%</dd>
                        <dd>ExAC AFR: 1.1842%</dd>
                        <dd>ExAC AMR: 0.3123%</dd>
                        <dd>ExAC FIN: 0.0151%</dd>
                        <dd>ExAC NFE: 0.1380%</dd>
                        <dd>ExAC OTH: 0.1114%</dd>
                        <dd>ExAC SAS: 0.2548%</dd>
                        <dd>gnomAD_E_AFR: 1.3356%</dd>
                        <dd>gnomAD_E_AMR: 0.3935%</dd>
                        <dd>gnomAD_E_ASJ: 0.0203%</dd>
                        <dd>gnomAD_E_FIN: 0.0090%</dd>
                        <dd>gnomAD_E_NFE: 0.1311%</dd>
                        <dd>gnomAD_E_OTH: 0.5124%</dd>
                        <dd>gnomAD_E_SAS: 0.2471%</dd>
                        <dd>gnomAD_G_AFR: 1.7435%</dd>
                        <dd>gnomAD_G_AMR: 0.4773%</dd>
                        <dd>gnomAD_G_FIN: 0.0287%</dd>
                        <dd>gnomAD_G_NFE: 0.0666%</dd>
                        <dd>gnomAD_G_OTH: 0.2037%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000225190">PLEKHM1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.625</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.933</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.493 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=210110">Intermediate osteopetrosis</a> associated with PLEKHM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0002754, Osteomyelitis</dd><dd>HP:0001047, Atopic dermatitis - HP:0002754, Osteomyelitis</dd><dd>HP:0001581, Recurrent skin infections - HP:0002754, Osteomyelitis</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002754, Osteomyelitis</dd><dd>HP:0002090, Pneumonia - HP:0002754, Osteomyelitis</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0031035, Chronic infection</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLEKHM1%0DTCIRG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TCIRG1</a> and phenotypic similarity 0.693 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=486">Autosomal dominant severe congenital neutropenia</a> associated with TCIRG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0001581, Recurrent skin infections</dd><dd>HP:0001047, Atopic dermatitis - HP:0001581, Recurrent skin infections</dd><dd>HP:0001581, Recurrent skin infections - HP:0001581, Recurrent skin infections</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0001581, Recurrent skin infections</dd><dd>HP:0002090, Pneumonia - HP:0002090, Pneumonia</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0005425, Recurrent sinopulmonary infections</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLEKHM1%0DTCIRG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TCIRG1</a> and phenotypic similarity 0.238 to mouse mutant of TCIRG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0000696, abnormal Peyer's patch morphology</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/611497">OMIM:611497</a> ?Osteopetrosis, autosomal recessive 6 (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/618107">OMIM:618107</a> Osteopetrosis, autosomal dominant 3 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=210110">ORPHA:210110</a> Intermediate osteopetrosis</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.625</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.933</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.45458400G&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=144621291">rs144621291</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs144621291&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.984</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430334.8">PLEKHM1:ENST00000430334.8:c.1348C&gt;A:p.(Pro450Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999177</dd>
                                    
                                    <dd>Polyphen2: 0.989 (D)</dd>
                                    <dd>Mutation Taster: 0.999 (P)</dd>
                                    <dd>SIFT: 0.009 (D)</dd>
                                    <dd>CADD: 0.998 (26.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0390%</dd>
                                    <dd>ESP EA: 0.1047%</dd>
                                    <dd>ESP All: 0.0692%</dd>
                                    <dd>ExAC AMR: 0.0262%</dd>
                                    <dd>ExAC NFE: 0.0563%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0536%</dd>
                                    <dd>gnomAD_E_NFE: 0.0638%</dd>
                                    <dd>gnomAD_E_OTH: 0.0730%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 0.0114%</dd>
                                    <dd>gnomAD_G_NFE: 0.0733%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.45468545G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=564130886">rs564130886</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs564130886&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.882</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430334.8">PLEKHM1:ENST00000430334.8:c.972C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.88486695</dd>
                                    
                                    <dd>CADD: 0.885 (9.388)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0048%</dd>
                                    <dd>ExAC AMR: 0.0259%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196862">RGPD4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.606</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.921</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.216 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RGPD4">RGPD4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia - MP:0008714, increased lung carcinoma incidence</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0008714, increased lung carcinoma incidence</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RGPD4%0DTNPO3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TNPO3</a> and phenotypic similarity 0.660 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=186">Primary biliary cholangitis</a> associated with TNPO3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0004386, Gastrointestinal inflammation</dd><dd>HP:0001047, Atopic dermatitis - HP:0004386, Gastrointestinal inflammation</dd><dd>HP:0001581, Recurrent skin infections - HP:0004386, Gastrointestinal inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0004386, Gastrointestinal inflammation</dd><dd>HP:0002090, Pneumonia - HP:0004386, Gastrointestinal inflammation</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0012203, Onychomycosis</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.606</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.921</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                                <b>chr2:g.107872584AAAC&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=770558604">rs770558604</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs770558604&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.984</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000408999.3">RGPD4:ENST00000408999.3:c.4589_4591del:p.(Thr1530del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98775387</dd>
                                    
                                    <dd>CADD: 0.988 (19.120)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AMR: 0.0242%</dd>
                                    <dd>ExAC NFE: 0.0040%</dd>
                                    <dd>gnomAD_E_AMR: 0.0151%</dd>
                                    <dd>gnomAD_E_NFE: 0.0019%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.107871274C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=555634733">rs555634733</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs555634733&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.857</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000408999.3">RGPD4:ENST00000408999.3:c.3270C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8638242</dd>
                                    
                                    <dd>CADD: 0.864 (8.659)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0104%</dd>
                                    <dd>ExAC EAS: 0.0465%</dd>
                                    <dd>ExAC NFE: 0.0078%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_EAS: 0.0348%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000274391">TPTE</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.587</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.918</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TPTE%0DPIK3CD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PIK3CD</a> and phenotypic similarity 0.535 to <a href="http://www.omim.org/615513">Immunodeficiency 14</a> associated with PIK3CD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0002721, Immunodeficiency</dd><dd>HP:0001047, Atopic dermatitis - HP:0002721, Immunodeficiency</dd><dd>HP:0001581, Recurrent skin infections - HP:0002721, Immunodeficiency</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002721, Immunodeficiency</dd><dd>HP:0002090, Pneumonia - HP:0005425, Recurrent sinopulmonary infections</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0005425, Recurrent sinopulmonary infections</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TPTE%0DPIK3CD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PIK3CD</a> and phenotypic similarity 0.675 to mouse mutant of PIK3CD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0001858, intestinal inflammation</dd><dd>HP:0001047, Atopic dermatitis - MP:0001858, intestinal inflammation</dd><dd>HP:0001581, Recurrent skin infections - MP:0001858, intestinal inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0001858, intestinal inflammation</dd><dd>HP:0002090, Pneumonia - MP:0001858, intestinal inflammation</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.587</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.918</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr21:g.10577453T&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=6582917">rs6582917</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs6582917&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.918</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427445.6">TPTE:ENST00000427445.6:c.682-7T&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612746.1">TPTE:ENST00000612746.1:c.382-7T&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618007.4">TPTE:ENST00000618007.4:c.796-7T&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622113.4">TPTE:ENST00000622113.4:c.742-7T&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.91834176</dd>
                                    
                                    <dd>CADD: 0.918 (10.880)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130538">OR11H1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.546</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.901</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR11H1%0DPTPN6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTPN6</a> and phenotypic similarity 0.834 to mouse mutant of PTPN6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0004947, skin inflammation</dd><dd>HP:0001047, Atopic dermatitis - MP:0004947, skin inflammation</dd><dd>HP:0001581, Recurrent skin infections - MP:0004947, skin inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0004947, skin inflammation</dd><dd>HP:0002090, Pneumonia - MP:0001862, interstitial pneumonia</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0001862, interstitial pneumonia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR11H1%0DPTPN6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTPN6</a> and phenotypic similarity 0.505 to fish mutant of PTPN6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - ZP:0010820, abnormal(ly) increased process quality inflammatory response</dd><dd>HP:0001047, Atopic dermatitis - ZP:0010820, abnormal(ly) increased process quality inflammatory response</dd><dd>HP:0001581, Recurrent skin infections - ZP:0010820, abnormal(ly) increased process quality inflammatory response</dd><dd>HP:0004386, Gastrointestinal inflammation - ZP:0010820, abnormal(ly) increased process quality inflammatory response</dd><dd>HP:0002090, Pneumonia - ZP:0010820, abnormal(ly) increased process quality inflammatory response</dd><dd>HP:0002205, Recurrent respiratory infections - ZP:0010820, abnormal(ly) increased process quality inflammatory response</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.546</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.901</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.15528179G&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2259944">rs2259944</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2259944&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.901</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000252835.5">OR11H1:ENST00000252835.5:c.21G&gt;T:p.(Gln7His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9011447</dd>
                                    
                                    <dd>SIFT: 0.173 (T)</dd>
                                    <dd>CADD: 0.901 (10.050)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178175">ZNF366</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.520</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.890</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.220 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ZNF366">ZNF366</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia - MP:0000445, short snout</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0000445, short snout</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF366%0DFOXP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FOXP1</a> and phenotypic similarity 0.713 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=391372">Intellectual disability-severe speech delay-mild dysmorphism syndrome</a> associated with FOXP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0001581, Recurrent skin infections</dd><dd>HP:0001047, Atopic dermatitis - HP:0001581, Recurrent skin infections</dd><dd>HP:0001581, Recurrent skin infections - HP:0001581, Recurrent skin infections</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0001581, Recurrent skin infections</dd><dd>HP:0002090, Pneumonia - HP:0001581, Recurrent skin infections</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002788, Recurrent upper respiratory tract infections</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.520</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.890</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr5:g.72461003G&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=755788457">rs755788457</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs755788457&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.890</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000318442.6">ZNF366:ENST00000318442.6:c.494C&gt;A:p.(Thr165Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.894</dd>
                                    
                                    <dd>Polyphen2: 0.018 (B)</dd>
                                    <dd>SIFT: 0.106 (T)</dd>
                                    <dd>CADD: 0.860 (8.539)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_EAS: 0.0348%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198019">FCGR1B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.519</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.532</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.853</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.532 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FCGR1B">FCGR1B</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0002531, abnormal type I hypersensitivity reaction</dd><dd>HP:0001047, Atopic dermatitis - MP:0002531, abnormal type I hypersensitivity reaction</dd><dd>HP:0001581, Recurrent skin infections - MP:0002531, abnormal type I hypersensitivity reaction</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0002531, abnormal type I hypersensitivity reaction</dd><dd>HP:0002090, Pneumonia - MP:0002531, abnormal type I hypersensitivity reaction</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0002412, increased susceptibility to bacterial infection</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FCGR1B%0DIFNG&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IFNG</a> and phenotypic similarity 0.420 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=88">Idiopathic aplastic anemia</a> associated with IFNG.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0002719, Recurrent infections</dd><dd>HP:0001047, Atopic dermatitis - HP:0002719, Recurrent infections</dd><dd>HP:0001581, Recurrent skin infections - HP:0002719, Recurrent infections</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002719, Recurrent infections</dd><dd>HP:0002090, Pneumonia - HP:0002719, Recurrent infections</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002719, Recurrent infections</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FCGR1B%0DIFNG&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IFNG</a> and phenotypic similarity 0.650 to mouse mutant of IFNG.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0001873, stomach inflammation</dd><dd>HP:0001047, Atopic dermatitis - MP:0001873, stomach inflammation</dd><dd>HP:0001581, Recurrent skin infections - MP:0001873, stomach inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0001873, stomach inflammation</dd><dd>HP:0002090, Pneumonia - MP:0001873, stomach inflammation</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0002412, increased susceptibility to bacterial infection</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FCGR1B%0DIFNG&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IFNG</a> and phenotypic similarity 0.226 to fish mutant of IFNG.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - ZP:0006610, abnormal(ly) disrupted defense response to bacterium</dd><dd>HP:0001047, Atopic dermatitis - ZP:0006610, abnormal(ly) disrupted defense response to bacterium</dd><dd>HP:0001581, Recurrent skin infections - ZP:0006610, abnormal(ly) disrupted defense response to bacterium</dd><dd>HP:0004386, Gastrointestinal inflammation - ZP:0006610, abnormal(ly) disrupted defense response to bacterium</dd><dd>HP:0002090, Pneumonia - ZP:0006610, abnormal(ly) disrupted defense response to bacterium</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.519</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.532</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.853</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.121088716A&gt;G [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201922049">rs201922049</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201922049&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.853</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369384.9">FCGR1B:ENST00000369384.9:c.115A&gt;G:p.(Ile39Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616817.4">FCGR1B:ENST00000616817.4:c.118A&gt;G:p.(Ile40Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000623603.3">FCGR1B:ENST00000623603.3:c.115A&gt;G:p.(Ile39Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369383.8">FCGR1B:ENST00000369383.8:c.31+1291A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8531412</dd>
                                    
                                    <dd>CADD: 0.853 (8.331)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000215182">MUC5AC</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.485</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.410</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.976</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.410 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MUC5AC">MUC5AC</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0003449, abnormal intestinal goblet cell morphology</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0005027, increased susceptibility to parasitic infection</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.485</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.410</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.976</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.1188935G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1462651135">rs1462651135</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1462651135&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.976</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.10790G&gt;A:p.(Arg3597Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9760668</dd>
                                    
                                    <dd>CADD: 0.976 (16.210)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1190453C&gt;T [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1487348067">rs1487348067</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1487348067&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.960</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.12308C&gt;T:p.(Thr4103Ile)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9600975</dd>
                        
                        <dd>CADD: 0.960 (13.990)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                    <b>chr11:g.1190500CCCAGCACAACCTCTGCCCCTACAA&gt;C [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1456780827">rs1456780827</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1456780827&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.939</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.12387_12410del:p.(Ser4131_Thr4138del)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.938765</dd>
                        
                        <dd>CADD: 0.939 (12.130)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1185767C&gt;A [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1469470921">rs1469470921</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1469470921&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.757</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.7622C&gt;A:p.(Thr2541Asn)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.75666755</dd>
                        
                        <dd>CADD: 0.757 (6.138)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1188765C&gt;T [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1370100366">rs1370100366</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1370100366&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.059</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.10620C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.058977544</dd>
                        
                        <dd>CADD: 0.059 (0.264)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1191333C&gt;A [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1292198170">rs1292198170</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1292198170&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.004</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.13188C&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.0043653846</dd>
                        
                        <dd>CADD: 0.004 (0.019)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1190067T&gt;C [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1330848007">rs1330848007</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1330848007&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.001</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.11922T&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 9.2059374E-4</dd>
                        
                        <dd>CADD: 0.001 (0.004)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000215454">KRTAP10-4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.421</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.847</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KRTAP10-4%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.584 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=342">Familial Mediterranean fever</a> associated with MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0001055, Erysipelas</dd><dd>HP:0001047, Atopic dermatitis - HP:0001055, Erysipelas</dd><dd>HP:0001581, Recurrent skin infections - HP:0001055, Erysipelas</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002586, Peritonitis</dd><dd>HP:0002090, Pneumonia - HP:0001055, Erysipelas</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002102, Pleuritis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KRTAP10-4%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.728 to mouse mutant of MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0004947, skin inflammation</dd><dd>HP:0001047, Atopic dermatitis - MP:0004947, skin inflammation</dd><dd>HP:0001581, Recurrent skin infections - MP:0004947, skin inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0004947, skin inflammation</dd><dd>HP:0002090, Pneumonia - MP:0004947, skin inflammation</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.421</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.847</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr21:g.44573974T&gt;C [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=437778">rs437778</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs437778&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.847</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400374.4">KRTAP10-4:ENST00000400374.4:c.216T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616689.2">KRTAP10-4:ENST00000616689.2:c.183T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622352.3">KRTAP10-4:ENST00000622352.3:c.231T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000323084.9">KRTAP10-4:ENST00000323084.9:c.83-5969A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.84650296</dd>
                                    
                                    <dd>CADD: 0.847 (8.139)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162727">OR2M5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.413</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.843</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR2M5%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.584 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=342">Familial Mediterranean fever</a> associated with MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0001055, Erysipelas</dd><dd>HP:0001047, Atopic dermatitis - HP:0001055, Erysipelas</dd><dd>HP:0001581, Recurrent skin infections - HP:0001055, Erysipelas</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002586, Peritonitis</dd><dd>HP:0002090, Pneumonia - HP:0001055, Erysipelas</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002102, Pleuritis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR2M5%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.728 to mouse mutant of MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0004947, skin inflammation</dd><dd>HP:0001047, Atopic dermatitis - MP:0004947, skin inflammation</dd><dd>HP:0001581, Recurrent skin infections - MP:0004947, skin inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0004947, skin inflammation</dd><dd>HP:0002090, Pneumonia - MP:0004947, skin inflammation</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.413</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.843</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr1:g.248145805C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=41305578">rs41305578</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs41305578&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.985</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366476.1">OR2M5:ENST00000366476.1:c.658C&gt;T:p.(Arg220*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999997</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 1.000 (35.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ExAC AMR: 0.0605%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AMR: 0.0387%</dd>
                                    <dd>gnomAD_E_NFE: 0.0063%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.248145920T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=146014040">rs146014040</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs146014040&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.700</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366476.1">OR2M5:ENST00000366476.1:c.773T&gt;C:p.(Met258Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.71509504</dd>
                                    
                                    <dd>Polyphen2: 0.232 (B)</dd>
                                    <dd>SIFT: 1.000 (T)</dd>
                                    <dd>CADD: 0.715 (5.453)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0717%</dd>
                                    <dd>UK10K: 0.1190%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.1279%</dd>
                                    <dd>ESP All: 0.0923%</dd>
                                    <dd>ExAC AMR: 0.1126%</dd>
                                    <dd>ExAC NFE: 0.0809%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0626%</dd>
                                    <dd>gnomAD_E_NFE: 0.0985%</dd>
                                    <dd>gnomAD_E_OTH: 0.1459%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0867%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000261150">EPPK1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.362</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.346</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.346 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=EPPK1">EPPK1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0003453, abnormal keratinocyte physiology</dd><dd>HP:0001047, Atopic dermatitis - MP:0003453, abnormal keratinocyte physiology</dd><dd>HP:0001581, Recurrent skin infections - MP:0003453, abnormal keratinocyte physiology</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.362</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.346</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr8:g.143861963T&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1262552886">rs1262552886</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1262552886&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568225.2">EPPK1:ENST00000568225.2:c.11216A&gt;C:p.(Glu3739Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615648.2">EPPK1:ENST00000615648.2:c.11291A&gt;C:p.(Glu3764Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9930817</dd>
                                    
                                    <dd>CADD: 0.993 (21.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.111</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.346</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.829</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr8:g.143865718G&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1345882883">rs1345882883</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1345882883&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.969</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568225.2">EPPK1:ENST00000568225.2:c.7461C&gt;A:p.(Asp2487Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615648.2">EPPK1:ENST00000615648.2:c.7536C&gt;A:p.(Asp2512Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9694508</dd>
                                    
                                    <dd>CADD: 0.969 (15.150)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr8:g.143866453C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201634698">rs201634698</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201634698&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.688</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568225.2">EPPK1:ENST00000568225.2:c.6726G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615648.2">EPPK1:ENST00000615648.2:c.6801G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.82681906</dd>
                                    
                                    <dd>CADD: 0.827 (7.615)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ESP AA: 0.0465%</dd>
                                    <dd>ESP EA: 0.4599%</dd>
                                    <dd>ESP All: 0.3208%</dd>
                                    <dd>ExAC AFR: 0.1606%</dd>
                                    <dd>ExAC AMR: 0.6770%</dd>
                                    <dd>ExAC EAS: 0.2042%</dd>
                                    <dd>ExAC FIN: 0.0761%</dd>
                                    <dd>ExAC NFE: 0.7025%</dd>
                                    <dd>ExAC OTH: 0.8083%</dd>
                                    <dd>ExAC SAS: 0.1038%</dd>
                                    <dd>gnomAD_E_AFR: 0.0067%</dd>
                                    <dd>gnomAD_E_AMR: 0.0394%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0831%</dd>
                                    <dd>gnomAD_E_EAS: 0.0236%</dd>
                                    <dd>gnomAD_E_FIN: 0.0091%</dd>
                                    <dd>gnomAD_E_NFE: 0.1034%</dd>
                                    <dd>gnomAD_E_OTH: 0.0188%</dd>
                                    <dd>gnomAD_E_SAS: 0.0131%</dd>
                                    <dd>gnomAD_G_NFE: 0.0135%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr8:g.143866876C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=553724064">rs553724064</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs553724064&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.186</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568225.2">EPPK1:ENST00000568225.2:c.6303G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615648.2">EPPK1:ENST00000615648.2:c.6378G&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.18660778</dd>
                        
                        <dd>CADD: 0.187 (0.897)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0200%</dd>
                        <dd>TOPMed: 0.0200%</dd>
                        <dd>ExAC NFE: 0.0015%</dd>
                        <dd>gnomAD_E_NFE: 0.0009%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000159289">GOLGA6A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.346</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.363</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.967</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.363 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GOLGA6A">GOLGA6A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia - MP:0001175, abnormal lung morphology</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0001175, abnormal lung morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.346</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.363</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.967</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr15:g.74074168G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=62005667">rs62005667</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs62005667&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.967</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000290438.3">GOLGA6A:ENST00000290438.3:c.1405C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96711487</dd>
                                    
                                    <dd>CADD: 0.967 (14.830)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000145439">CBR4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.322</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CBR4%0DGUSB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GUSB</a> and phenotypic similarity 0.603 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=584">Mucopolysaccharidosis type 7</a> associated with GUSB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0012115, Hepatitis</dd><dd>HP:0001047, Atopic dermatitis - HP:0012115, Hepatitis</dd><dd>HP:0001581, Recurrent skin infections - HP:0012115, Hepatitis</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0012115, Hepatitis</dd><dd>HP:0002090, Pneumonia - HP:0002205, Recurrent respiratory infections</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002205, Recurrent respiratory infections</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CBR4%0DGUSB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GUSB</a> and phenotypic similarity 0.220 to mouse mutant of GUSB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia - MP:0000445, short snout</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0000445, short snout</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.322</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr4:g.169002212CA&gt;C [1/1:-/1:-/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=60552620">rs60552620</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs60552620&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000306193.8">CBR4:ENST00000306193.8:c.401-8del:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000504480.5">CBR4:ENST00000504480.5:c.401-8del:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.17794669</dd>
                                    
                                    <dd>CADD: 0.178 (0.851)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000138161">CUZD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.211</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.488</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.750</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.488 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CUZD1">CUZD1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0004761, increased susceptibility to induced pancreatitis</dd><dd>HP:0001047, Atopic dermatitis - MP:0004761, increased susceptibility to induced pancreatitis</dd><dd>HP:0001581, Recurrent skin infections - MP:0004761, increased susceptibility to induced pancreatitis</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0004761, increased susceptibility to induced pancreatitis</dd><dd>HP:0002090, Pneumonia - MP:0004761, increased susceptibility to induced pancreatitis</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.211</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.488</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.750</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.122837549G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=558554477">rs558554477</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs558554477&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.968</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392790.6">CUZD1:ENST00000392790.6:c.454C&gt;G:p.(Pro152Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368904.6">CUZD1:ENST00000368904.6:c.454C&gt;G:p.(Pro152Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97021484</dd>
                                    
                                    <dd>Polyphen2: 0.053 (B)</dd>
                                    <dd>SIFT: 0.181 (T)</dd>
                                    <dd>CADD: 0.970 (15.260)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0010%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.122836314A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=36212072">rs36212072</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs36212072&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.533</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392790.6">CUZD1:ENST00000392790.6:c.854T&gt;C:p.(Ile285Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368904.6">CUZD1:ENST00000368904.6:c.*15T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.971 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.046 (D)</dd>
                                    <dd>CADD: 0.995 (23.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2995%</dd>
                                    <dd>TOPMed: 0.5352%</dd>
                                    <dd>UK10K: 0.8992%</dd>
                                    <dd>ESP AA: 0.1362%</dd>
                                    <dd>ESP EA: 0.9419%</dd>
                                    <dd>ESP All: 0.6690%</dd>
                                    <dd>ExAC AFR: 0.1457%</dd>
                                    <dd>ExAC AMR: 0.1314%</dd>
                                    <dd>ExAC FIN: 1.4594%</dd>
                                    <dd>ExAC NFE: 0.8575%</dd>
                                    <dd>ExAC OTH: 0.4454%</dd>
                                    <dd>ExAC SAS: 0.1102%</dd>
                                    <dd>gnomAD_E_AFR: 0.1577%</dd>
                                    <dd>gnomAD_E_AMR: 0.1342%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0968%</dd>
                                    <dd>gnomAD_E_FIN: 1.4599%</dd>
                                    <dd>gnomAD_E_NFE: 0.8616%</dd>
                                    <dd>gnomAD_E_OTH: 0.5691%</dd>
                                    <dd>gnomAD_E_SAS: 0.0823%</dd>
                                    <dd>gnomAD_G_AFR: 0.1840%</dd>
                                    <dd>gnomAD_G_FIN: 1.4925%</dd>
                                    <dd>gnomAD_G_NFE: 0.8019%</dd>
                                    <dd>gnomAD_G_OTH: 0.6160%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000237700">PRAMEF33</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.197</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.259</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.259 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PRAMEF33">PRAMEF33</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0002060, abnormal skin morphology</dd><dd>HP:0001047, Atopic dermatitis - MP:0002060, abnormal skin morphology</dd><dd>HP:0001581, Recurrent skin infections - MP:0002060, abnormal skin morphology</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.197</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.259</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr1:g.13307108G&gt;A [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1343154326">rs1343154326</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1343154326&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000437300.2">PRAMEF33:ENST00000437300.2:c.758G&gt;A:p.(Trp253*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99498814</dd>
                                    
                                    <dd>CADD: 0.995 (23.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000244482">LILRA6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.154</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.688</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LILRA6%0DFCGR2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FCGR2B</a> and phenotypic similarity 0.587 to <a href="http://www.omim.org/152700">Systemic lupus erythematosus, susceptibility to</a> associated with FCGR2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0025300, Malar rash</dd><dd>HP:0001047, Atopic dermatitis - HP:0025300, Malar rash</dd><dd>HP:0001581, Recurrent skin infections - HP:0025300, Malar rash</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0025300, Malar rash</dd><dd>HP:0002090, Pneumonia - HP:0025300, Malar rash</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002102, Pleuritis</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LILRA6%0DFCGR2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FCGR2B</a> and phenotypic similarity 0.730 to mouse mutant of FCGR2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0001864, vasculitis</dd><dd>HP:0001047, Atopic dermatitis - MP:0001864, vasculitis</dd><dd>HP:0001581, Recurrent skin infections - MP:0001846, increased inflammatory response</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0001869, pancreas inflammation</dd><dd>HP:0002090, Pneumonia - MP:0001861, lung inflammation</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0001861, lung inflammation</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.154</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.688</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54242117T&gt;C [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=11673178">rs11673178</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs11673178&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.688</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245621.6">LILRA6:ENST00000245621.6:c.264A&gt;G:p.(Ile88Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396365.6">LILRA6:ENST00000396365.6:c.264A&gt;G:p.(Ile88Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6876079</dd>
                                    
                                    <dd>CADD: 0.688 (5.053)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169894">MUC3A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.131</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.249</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.959</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.249 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MUC3A">MUC3A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0000367, abnormal coat/ hair morphology</dd><dd>HP:0001047, Atopic dermatitis - MP:0000367, abnormal coat/ hair morphology</dd><dd>HP:0001581, Recurrent skin infections - MP:0000367, abnormal coat/ hair morphology</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.249</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.033</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr7:g.100957296C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1219920358">rs1219920358</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1219920358&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.033</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.5517C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.5517C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.03283626</dd>
                                    
                                    <dd>CADD: 0.033 (0.145)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.131</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.249</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.959</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.100965345C&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=73714276">rs73714276</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs73714276&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.975</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.9446C&gt;A:p.(Ala3149Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.9446C&gt;A:p.(Ala3149Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9745903</dd>
                                    
                                    <dd>CADD: 0.975 (15.950)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.100953370G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1490623108">rs1490623108</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1490623108&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.944</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.1591G&gt;A:p.(Ala531Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.1591G&gt;A:p.(Ala531Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.94440955</dd>
                                    
                                    <dd>CADD: 0.944 (12.550)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                    <b>chr7:g.100965350A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=75592954">rs75592954</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs75592954&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.964</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.9448+3A&gt;G:p.?</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.9448+3A&gt;G:p.?</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9644369</dd>
                        
                        <dd>CADD: 0.964 (14.490)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                    <b>chr7:g.100957124TAGG&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1422404475">rs1422404475</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1422404475&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.864</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.5346_5348del:p.(Gly1783del)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.5346_5348del:p.(Gly1783del)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8639808</dd>
                        
                        <dd>CADD: 0.864 (8.664)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr7:g.100960786C&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2904852">rs2904852</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2904852&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.602</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.8901C&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.8901C&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.6018928</dd>
                        
                        <dd>CADD: 0.602 (4.000)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr7:g.100957242A&gt;C [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1243555383">rs1243555383</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1243555383&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.331</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.5463A&gt;C:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.5463A&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.33104026</dd>
                        
                        <dd>CADD: 0.331 (1.746)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr7:g.100953372A&gt;T [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1260789698">rs1260789698</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1260789698&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.024</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.1593A&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.1593A&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.024111927</dd>
                        
                        <dd>CADD: 0.024 (0.106)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr7:g.100953371C&gt;G [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=36187473">rs36187473</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs36187473&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.000</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.1592C&gt;G:p.(Ala531Gly)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.1592C&gt;G:p.(Ala531Gly)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 2.3025274E-4</dd>
                        
                        <dd>CADD: 0.000 (0.001)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000128829">EIF2AK4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.109</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.193</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.295 to <a href="http://www.omim.org/234810">Pulmonary venoocclusive disease 2</a> associated with EIF2AK4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0005954, Pulmonary capillary hemangiomatosis</dd><dd>HP:0001047, Atopic dermatitis - HP:0005954, Pulmonary capillary hemangiomatosis</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia - HP:0005954, Pulmonary capillary hemangiomatosis</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0005954, Pulmonary capillary hemangiomatosis</dd></dl><dl><dt>Phenotypic similarity 0.386 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=EIF2AK4">EIF2AK4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0020916, increased susceptibility to Herpesvirales infection</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/234810">OMIM:234810</a> Pulmonary venoocclusive disease 2 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.109</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.193</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr15:g.39976546T&gt;A [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263791.10">EIF2AK4:ENST00000263791.10:c.1951T&gt;A:p.(Tyr651Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.993 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.999 (33.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000277322">GOLGA6L6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.096</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.637</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GOLGA6L6%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.584 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=342">Familial Mediterranean fever</a> associated with MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0001055, Erysipelas</dd><dd>HP:0001047, Atopic dermatitis - HP:0001055, Erysipelas</dd><dd>HP:0001581, Recurrent skin infections - HP:0001055, Erysipelas</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002586, Peritonitis</dd><dd>HP:0002090, Pneumonia - HP:0001055, Erysipelas</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002102, Pleuritis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GOLGA6L6%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.728 to mouse mutant of MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0004947, skin inflammation</dd><dd>HP:0001047, Atopic dermatitis - MP:0004947, skin inflammation</dd><dd>HP:0001581, Recurrent skin infections - MP:0004947, skin inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0004947, skin inflammation</dd><dd>HP:0002090, Pneumonia - MP:0004947, skin inflammation</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.096</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.637</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr15:g.20535161T&gt;C [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1254577349">rs1254577349</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1254577349&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.637</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000619213.1">GOLGA6L6:ENST00000619213.1:c.1273A&gt;G:p.(Lys425Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.64428675</dd>
                                    
                                    <dd>CADD: 0.644 (4.489)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0358%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0808%</dd>
                                    <dd>gnomAD_E_EAS: 0.0723%</dd>
                                    <dd>gnomAD_E_NFE: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000260458">KCNJ18</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.054</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.119</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.237 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79102">Thyrotoxic periodic paralysis</a> associated with KCNJ18.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0001265, Hyporeflexia</dd><dd>HP:0001047, Atopic dermatitis - HP:0001265, Hyporeflexia</dd><dd>HP:0001581, Recurrent skin infections - HP:0001265, Hyporeflexia</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002019, Constipation</dd><dd>HP:0002090, Pneumonia - HP:0001265, Hyporeflexia</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/613239">OMIM:613239</a> Thyrotoxic periodic paralysis, susceptibility to, 2 (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79102">ORPHA:79102</a> Thyrotoxic periodic paralysis</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.054</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.119</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.21702902G&gt;A [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1371547709">rs1371547709</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1371547709&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000567955.3">KCNJ18:ENST00000567955.3:c.116G&gt;A:p.(Arg39Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999999</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.073 (T)</dd>
                                    <dd>CADD: 0.997 (24.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr17:g.21702905G&gt;A [1/1:0/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1267282483">rs1267282483</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1267282483&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 1.000</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000567955.3">KCNJ18:ENST00000567955.3:c.119G&gt;A:p.(Arg40His)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.999968</dd>
                        
                        <dd>Mutation Taster: 1.000 (P)</dd>
                        <dd>SIFT: 0.063 (T)</dd>
                        <dd>CADD: 0.994 (22.200)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr17:g.21703531A&gt;G [1/1:0/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1182398045">rs1182398045</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1182398045&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.994</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000567955.3">KCNJ18:ENST00000567955.3:c.745A&gt;G:p.(Ile249Val)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9939744</dd>
                        
                        <dd>SIFT: 0.579 (T)</dd>
                        <dd>CADD: 0.994 (22.200)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136274">NACAD</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.049</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.558</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NACAD%0DSRP54&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SRP54</a> and phenotypic similarity 0.693 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=486">Autosomal dominant severe congenital neutropenia</a> associated with SRP54.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0001581, Recurrent skin infections</dd><dd>HP:0001047, Atopic dermatitis - HP:0001581, Recurrent skin infections</dd><dd>HP:0001581, Recurrent skin infections - HP:0001581, Recurrent skin infections</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0001581, Recurrent skin infections</dd><dd>HP:0002090, Pneumonia - HP:0002090, Pneumonia</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0005425, Recurrent sinopulmonary infections</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.049</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.558</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.45084344A&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200528171">rs200528171</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200528171&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.558</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000490531.3">NACAD:ENST00000490531.3:c.1836T&gt;A:p.(Asp612Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.55799997</dd>
                                    
                                    <dd>Polyphen2: 0.072 (B)</dd>
                                    <dd>SIFT: 0.442 (T)</dd>
                                    <dd>CADD: 0.235 (1.162)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171116">HSFX1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.035</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.521</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=HSFX1%0DSPAG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SPAG1</a> and phenotypic similarity 0.701 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with SPAG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0002090, Pneumonia</dd><dd>HP:0001047, Atopic dermatitis - HP:0002090, Pneumonia</dd><dd>HP:0001581, Recurrent skin infections - HP:0002090, Pneumonia</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002090, Pneumonia</dd><dd>HP:0002090, Pneumonia - HP:0002090, Pneumonia</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002205, Recurrent respiratory infections</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.035</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.521</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr23:g.149774897C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1287372104">rs1287372104</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1287372104&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.521</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370416.4">HSFX1:ENST00000370416.4:c.216C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5205978</dd>
                                    
                                    <dd>CADD: 0.521 (3.193)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188234">AGAP4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.033</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.514</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=AGAP4%0DNR3C1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NR3C1</a> and phenotypic similarity 0.617 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=786">Generalized glucocorticoid resistance syndrome</a> associated with NR3C1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0001061, Acne</dd><dd>HP:0001047, Atopic dermatitis - HP:0001061, Acne</dd><dd>HP:0001581, Recurrent skin infections - HP:0001061, Acne</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0001061, Acne</dd><dd>HP:0002090, Pneumonia - HP:0001061, Acne</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0001061, Acne</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=AGAP4%0DNR3C1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NR3C1</a> and phenotypic similarity 0.412 to mouse mutant of NR3C1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0001575, cyanosis</dd><dd>HP:0001047, Atopic dermatitis - MP:0001575, cyanosis</dd><dd>HP:0001581, Recurrent skin infections - MP:0001575, cyanosis</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0000609, abnormal liver physiology</dd><dd>HP:0002090, Pneumonia - MP:0001177, atelectasis</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0001177, atelectasis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=AGAP4%0DNR3C1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NR3C1</a> and phenotypic similarity 0.176 to fish mutant of NR3C1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema -</dd><dd>HP:0001047, Atopic dermatitis -</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation - ZP:0005132, abnormal(ly) decreased width intestine</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.033</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.514</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr10:g.45826797G&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=77275891">rs77275891</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs77275891&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.514</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311652.9">AGAP4:ENST00000311652.9:c.1110C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000448048.6">AGAP4:ENST00000448048.6:c.1110C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616763.4">AGAP4:ENST00000616763.4:c.1179C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618171.4">AGAP4:ENST00000618171.4:c.1179C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.51437616</dd>
                                    
                                    <dd>CADD: 0.514 (3.137)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000133401">PDZD2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.020</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.459</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PDZD2%0DNCKAP1L&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NCKAP1L</a> and phenotypic similarity 0.743 to mouse mutant of NCKAP1L.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0001861, lung inflammation</dd><dd>HP:0001047, Atopic dermatitis - MP:0001861, lung inflammation</dd><dd>HP:0001581, Recurrent skin infections - MP:0001861, lung inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0001861, lung inflammation</dd><dd>HP:0002090, Pneumonia - MP:0001861, lung inflammation</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0001861, lung inflammation</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.020</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.459</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr5:g.32090272A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=150841463">rs150841463</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs150841463&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.664</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000438447.2">PDZD2:ENST00000438447.2:c.6824A&gt;G:p.(Gln2275Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.342 (B)</dd>
                                    <dd>Mutation Taster: 0.701</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.995 (22.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.1433%</dd>
                                    <dd>UK10K: 0.2116%</dd>
                                    <dd>ESP EA: 0.1977%</dd>
                                    <dd>ESP All: 0.1307%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>ExAC AMR: 0.6651%</dd>
                                    <dd>ExAC NFE: 0.2354%</dd>
                                    <dd>ExAC OTH: 0.6608%</dd>
                                    <dd>ExAC SAS: 0.0545%</dd>
                                    <dd>gnomAD_E_AFR: 0.0327%</dd>
                                    <dd>gnomAD_E_AMR: 0.3812%</dd>
                                    <dd>gnomAD_E_ASJ: 1.2487%</dd>
                                    <dd>gnomAD_E_NFE: 0.1576%</dd>
                                    <dd>gnomAD_E_OTH: 0.4012%</dd>
                                    <dd>gnomAD_E_SAS: 0.0682%</dd>
                                    <dd>gnomAD_G_AFR: 0.0458%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_ASJ: 0.3311%</dd>
                                    <dd>gnomAD_G_NFE: 0.1000%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr5:g.32074356G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=115304584">rs115304584</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs115304584&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.255</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000438447.2">PDZD2:ENST00000438447.2:c.3250G&gt;A:p.(Ala1084Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999</dd>
                                    
                                    <dd>Polyphen2: 0.025 (B)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.994 (22.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.6789%</dd>
                                    <dd>TOPMed: 1.0740%</dd>
                                    <dd>UK10K: 1.7456%</dd>
                                    <dd>ESP AA: 0.2724%</dd>
                                    <dd>ESP EA: 1.8721%</dd>
                                    <dd>ESP All: 1.3302%</dd>
                                    <dd>ExAC AFR: 0.2980%</dd>
                                    <dd>ExAC AMR: 0.6138%</dd>
                                    <dd>ExAC FIN: 0.5443%</dd>
                                    <dd>ExAC NFE: 1.4244%</dd>
                                    <dd>ExAC OTH: 0.5519%</dd>
                                    <dd>ExAC SAS: 0.2544%</dd>
                                    <dd>gnomAD_E_AFR: 0.3071%</dd>
                                    <dd>gnomAD_E_AMR: 0.7177%</dd>
                                    <dd>gnomAD_E_ASJ: 1.1881%</dd>
                                    <dd>gnomAD_E_FIN: 0.4306%</dd>
                                    <dd>gnomAD_E_NFE: 1.4996%</dd>
                                    <dd>gnomAD_E_OTH: 0.9300%</dd>
                                    <dd>gnomAD_E_SAS: 0.2956%</dd>
                                    <dd>gnomAD_G_AFR: 0.3896%</dd>
                                    <dd>gnomAD_G_AMR: 0.5967%</dd>
                                    <dd>gnomAD_G_FIN: 0.3438%</dd>
                                    <dd>gnomAD_G_NFE: 1.4678%</dd>
                                    <dd>gnomAD_G_OTH: 0.8163%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183638">RP1L1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.020</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.151</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.151 to <a href="http://www.omim.org/618826">Retinitis pigmentosa 88</a> associated with RP1L1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0007843, Attenuation of retinal blood vessels</dd><dd>HP:0001047, Atopic dermatitis - HP:0007843, Attenuation of retinal blood vessels</dd><dd>HP:0001581, Recurrent skin infections -</dd><dd>HP:0004386, Gastrointestinal inflammation -</dd><dd>HP:0002090, Pneumonia -</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/613587">OMIM:613587</a> Occult macular dystrophy - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/618826">OMIM:618826</a> Retinitis pigmentosa 88 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.020</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.151</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.850</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_INSERTION</span>
                                <b>chr8:g.10610127T&gt;TCCTCTAACTGCACCCTCTCTTCTTGCAGCCCTTCTATTACTTTAGTCC [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=369606728">rs369606728</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs369606728&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=490785%5Balleleid%5D">(criteria_provided,_conflicting_interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382483.4">RP1L1:ENST00000382483.4:c.3970_3971insGGACTAAAGTAATAGAAGGGCTGCAAGAAGAGAGGGTGCAGTTAGAGG:p.(Glu1323_Glu1324insGlyThrLysValIleGluGlyLeuGlnGluGluArgValGlnLeuGlu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5439631</dd>
                                    
                                    <dd>CADD: 0.544 (3.410)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000282870">FRG1DP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                                <b>chr20:g.29092559T&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1457105972">rs1457105972</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1457105972&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000635586.1">FRG1DP:ENST00000635586.1:n.311+2T&gt;G:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99786204</dd>
                                    
                                    <dd>CADD: 0.998 (26.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000284662">OR4F16</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr1:g.686625A&gt;AG [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1239584546">rs1239584546</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1239584546&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000332831.4">OR4F16:ENST00000332831.4:c.29dup:p.(Glu11*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000634337.2">OR4F16:ENST00000634337.2:n.160+12301_160+12302insC:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99241424</dd>
                                    
                                    <dd>CADD: 0.992 (21.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000215029">TCP11X2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.102462896A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1432888360">rs1432888360</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1432888360&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453326.2">TCP11X2:ENST00000453326.2:c.776T&gt;C:p.(Val259Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000642911.1">TCP11X2:ENST00000642911.1:c.1061T&gt;C:p.(Val354Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618302.2">TCP11X2:ENST00000618302.2:c.171T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.959 (T)</dd>
                                    <dd>CADD: 0.994 (22.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000147996">CBWD5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.65681648G&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=202066212">rs202066212</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs202066212&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377384.5">CBWD5:ENST00000377384.5:c.316G&gt;T:p.(Gly106Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377395.8">CBWD5:ENST00000377395.8:c.316G&gt;T:p.(Gly106Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382405.7">CBWD5:ENST00000382405.7:c.316G&gt;T:p.(Gly106Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000429800.6">CBWD5:ENST00000429800.6:c.316G&gt;T:p.(Gly106Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430059.6">CBWD5:ENST00000430059.6:c.316G&gt;T:p.(Gly106Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377392.9">CBWD5:ENST00000377392.9:c.-1450G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.998 (27.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_NFE: 0.0014%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr9:g.65682821G&gt;T [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1210326103">rs1210326103</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1210326103&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.994</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377395.8">CBWD5:ENST00000377395.8:c.394G&gt;T:p.(Asp132Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382405.7">CBWD5:ENST00000382405.7:c.394G&gt;T:p.(Asp132Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000429800.6">CBWD5:ENST00000429800.6:c.394G&gt;T:p.(Asp132Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430059.6">CBWD5:ENST00000430059.6:c.394G&gt;T:p.(Asp132Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377392.9">CBWD5:ENST00000377392.9:c.-1372G&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.99411154</dd>
                        
                        <dd>CADD: 0.994 (22.300)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000184206">GOLGA6L4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.998</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.998</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr15:g.84240663C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1278379491">rs1278379491</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1278379491&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.998</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422563.6">GOLGA6L4:ENST00000422563.6:c.901C&gt;T:p.(Gln301*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000510439.7">GOLGA6L4:ENST00000510439.7:c.1309C&gt;T:p.(Gln437*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.996 (24.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_G_AFR: 0.0118%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000259511">UBE2Q2L</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr15:g.84182055G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1339078682">rs1339078682</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1339078682&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000558195.2">UBE2Q2L:ENST00000558195.2:c.308G&gt;A:p.(Arg103His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.983 (17.730)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_EAS: 0.0267%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000230626">AC011005.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.995</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.995</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.129126858T&gt;C [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.995</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000467614.2">AC011005.1:ENST00000467614.2:c.341T&gt;C:p.(Ile114Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9946297</dd>
                                    
                                    <dd>CADD: 0.995 (22.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154529">CNTNAP3B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.41991721G&gt;A [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3739623">rs3739623</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs3739623&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341990.8">CNTNAP3B:ENST00000341990.8:c.1222C&gt;T:p.(Leu408Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377561.6">CNTNAP3B:ENST00000377561.6:c.1222C&gt;T:p.(Leu408Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612828.4">CNTNAP3B:ENST00000612828.4:c.1222C&gt;T:p.(Leu408Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617422.4">CNTNAP3B:ENST00000617422.4:c.1222C&gt;T:p.(Leu408Phe)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99323916</dd>
                                    
                                    <dd>CADD: 0.993 (21.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211673">IGLV3-1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.22880780T&gt;C [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390319.2">IGLV3-1:ENST00000390319.2:c.23T&gt;C:p.(Leu8Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99323916</dd>
                                    
                                    <dd>CADD: 0.993 (21.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22881108T&gt;C [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=761874852">rs761874852</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs761874852&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.974</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390319.2">IGLV3-1:ENST00000390319.2:c.58T&gt;C:p.(Ser20Pro)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9751114</dd>
                        
                        <dd>CADD: 0.975 (16.040)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>ExAC NFE: 0.0061%</dd>
                        <dd>gnomAD_E_AFR: 0.0066%</dd>
                        <dd>gnomAD_E_AMR: 0.0030%</dd>
                        <dd>gnomAD_E_NFE: 0.0009%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22881160C&gt;G [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=776652018">rs776652018</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs776652018&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.974</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390319.2">IGLV3-1:ENST00000390319.2:c.110C&gt;G:p.(Ala37Gly)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9754529</dd>
                        
                        <dd>CADD: 0.975 (16.100)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>ExAC AFR: 0.0103%</dd>
                        <dd>ExAC NFE: 0.0015%</dd>
                        <dd>ExAC SAS: 0.0061%</dd>
                        <dd>gnomAD_E_AFR: 0.0066%</dd>
                        <dd>gnomAD_E_NFE: 0.0009%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178104">PDE4DIP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.991</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.991</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.149005097A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1747958">rs1747958</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1747958&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.991</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369354.7">PDE4DIP:ENST00000369354.7:c.4075A&gt;G:p.(Lys1359Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369356.8">PDE4DIP:ENST00000369356.8:c.4075A&gt;G:p.(Lys1359Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524974.5">PDE4DIP:ENST00000524974.5:c.4483A&gt;G:p.(Lys1495Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000585156.5">PDE4DIP:ENST00000585156.5:c.4483A&gt;G:p.(Lys1495Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618462.4">PDE4DIP:ENST00000618462.4:c.3943A&gt;G:p.(Lys1315Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9912904</dd>
                                    
                                    <dd>SIFT: 0.316 (T)</dd>
                                    <dd>CADD: 0.991 (20.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211900">IGHJ6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.990</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.990</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.105863254T&gt;C [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=369886334">rs369886334</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs369886334&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390560.2">IGHJ6:ENST00000390560.2:c.5A&gt;G:p.(Tyr2Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9912904</dd>
                                    
                                    <dd>CADD: 0.991 (20.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000268320">SCGB1C2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.985</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.985</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.137637G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1255969219">rs1255969219</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1255969219&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000595228.3">SCGB1C2:ENST00000595228.3:c.47G&gt;A:p.(Cys16Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99339306</dd>
                                    
                                    <dd>CADD: 0.993 (21.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.137603C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1239602891">rs1239602891</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1239602891&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.976</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000595228.3">SCGB1C2:ENST00000595228.3:c.13C&gt;T:p.(Arg5Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9761219</dd>
                                    
                                    <dd>CADD: 0.976 (16.220)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162825">NBPF20</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.983</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.983</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.145337786A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1342929556">rs1342929556</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1342929556&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.983</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369373.9">NBPF20:ENST00000369373.9:c.9569T&gt;C:p.(Val3190Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392971.6">NBPF20:ENST00000392971.6:c.8534T&gt;C:p.(Val2845Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000594479.7">NBPF20:ENST00000594479.7:c.2264T&gt;C:p.(Val755Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622803.4">NBPF20:ENST00000622803.4:c.1738+12505T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620821.1">NBPF20:ENST00000620821.1:c.*600-45297T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98313445</dd>
                                    
                                    <dd>CADD: 0.983 (17.730)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211666">IGLV2-14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.978</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.978</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.22759060T&gt;C [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.978</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390312.2">IGLV2-14:ENST00000390312.2:c.203T&gt;C:p.(Met68Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9780719</dd>
                                    
                                    <dd>CADD: 0.978 (16.590)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr22:g.22759013C&gt;T [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=373891287">rs373891287</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs373891287&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.000</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390312.2">IGLV2-14:ENST00000390312.2:c.156C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 4.6038628E-4</dd>
                        
                        <dd>CADD: 0.000 (0.002)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0599%</dd>
                        <dd>ExAC AFR: 0.0103%</dd>
                        <dd>ExAC AMR: 0.0173%</dd>
                        <dd>ExAC NFE: 0.0090%</dd>
                        <dd>ExAC SAS: 0.0061%</dd>
                        <dd>gnomAD_E_AFR: 0.0066%</dd>
                        <dd>gnomAD_E_AMR: 0.0060%</dd>
                        <dd>gnomAD_E_ASJ: 0.0306%</dd>
                        <dd>gnomAD_E_NFE: 0.0009%</dd>
                        <dd>gnomAD_E_OTH: 0.0183%</dd>
                        <dd>gnomAD_E_SAS: 0.0032%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104972">LILRB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.978</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.978</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54632040T&gt;G [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1061681">rs1061681</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1061681&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.978</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324602.11">LILRB1:ENST00000324602.11:c.464T&gt;G:p.(Ile155Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396315.5">LILRB1:ENST00000396315.5:c.464T&gt;G:p.(Ile155Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396317.5">LILRB1:ENST00000396317.5:c.464T&gt;G:p.(Ile155Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396327.7">LILRB1:ENST00000396327.7:c.464T&gt;G:p.(Ile155Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396331.5">LILRB1:ENST00000396331.5:c.464T&gt;G:p.(Ile155Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396332.8">LILRB1:ENST00000396332.8:c.464T&gt;G:p.(Ile155Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427581.6">LILRB1:ENST00000427581.6:c.572T&gt;G:p.(Ile191Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9780719</dd>
                                    
                                    <dd>CADD: 0.978 (16.590)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.54632001C&gt;T [1/1:0/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1061680">rs1061680</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1061680&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.971</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324602.11">LILRB1:ENST00000324602.11:c.425C&gt;T:p.(Thr142Ile)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396315.5">LILRB1:ENST00000396315.5:c.425C&gt;T:p.(Thr142Ile)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396317.5">LILRB1:ENST00000396317.5:c.425C&gt;T:p.(Thr142Ile)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396327.7">LILRB1:ENST00000396327.7:c.425C&gt;T:p.(Thr142Ile)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396331.5">LILRB1:ENST00000396331.5:c.425C&gt;T:p.(Thr142Ile)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396332.8">LILRB1:ENST00000396332.8:c.425C&gt;T:p.(Thr142Ile)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427581.6">LILRB1:ENST00000427581.6:c.533C&gt;T:p.(Thr178Ile)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9714241</dd>
                        
                        <dd>CADD: 0.971 (15.440)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000263956">NBPF11</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.972</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.965</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.148107750T&gt;C [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1488492448">rs1488492448</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1488492448&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.965</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.2039A&gt;G:p.(Tyr680Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.1934A&gt;G:p.(Tyr645Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.2039A&gt;G:p.(Tyr680Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.965086</dd>
                                    
                                    <dd>CADD: 0.965 (14.570)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.972</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.148118714C&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=71249509">rs71249509</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs71249509&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.977</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.997G&gt;C:p.(Glu333Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.892G&gt;C:p.(Glu298Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.997G&gt;C:p.(Glu333Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97740567</dd>
                                    
                                    <dd>CADD: 0.977 (16.460)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.148126846T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=57784339">rs57784339</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs57784339&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.967</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.158A&gt;G:p.(Asn53Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.158A&gt;G:p.(Asn53Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.156A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96673405</dd>
                                    
                                    <dd>CADD: 0.967 (14.780)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.148126839C&gt;G [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2064455">rs2064455</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2064455&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.892</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.165G&gt;C:p.(Gln55His)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.163G&gt;C:p.(Glu55Gln)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.165G&gt;C:p.(Gln55His)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8922543</dd>
                        
                        <dd>CADD: 0.892 (9.676)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.148120602C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1199265858">rs1199265858</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1199265858&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.857</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.887G&gt;A:p.(Cys296Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.782G&gt;A:p.(Cys261Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.887G&gt;A:p.(Cys296Tyr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.85724217</dd>
                        
                        <dd>CADD: 0.857 (8.454)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.148120603A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=74994984">rs74994984</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs74994984&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.704</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.886T&gt;C:p.(Cys296Arg)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.781T&gt;C:p.(Cys261Arg)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.886T&gt;C:p.(Cys296Arg)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.70351684</dd>
                        
                        <dd>CADD: 0.704 (5.280)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211721">TRBV6-5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.969</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.969</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.142451250A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1256127073">rs1256127073</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1256127073&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.969</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390368.2">TRBV6-5:ENST00000390368.2:c.146A&gt;G:p.(Glu49Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9693804</dd>
                                    
                                    <dd>CADD: 0.969 (15.140)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198521">ZNF43</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.963</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.963</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.21808523A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1360304639">rs1360304639</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1360304639&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.963</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354959.8">ZNF43:ENST00000354959.8:c.1514T&gt;C:p.(Ile505Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357491.10">ZNF43:ENST00000357491.10:c.1541T&gt;C:p.(Ile514Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000594012.5">ZNF43:ENST00000594012.5:c.1496T&gt;C:p.(Ile499Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000595461.5">ZNF43:ENST00000595461.5:c.1496T&gt;C:p.(Ile499Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000598381.5">ZNF43:ENST00000598381.5:c.1496T&gt;C:p.(Ile499Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96837723</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>CADD: 0.968 (15.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0409%</dd>
                                    <dd>gnomAD_G_FIN: 0.0358%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197978">GOLGA6L9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.954</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.954</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr15:g.82434605G&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1380876243">rs1380876243</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1380876243&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.954</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618348.1">GOLGA6L9:ENST00000618348.1:c.1002+3G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9541858</dd>
                                    
                                    <dd>CADD: 0.954 (13.390)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr15:g.82434051C&gt;G [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1305272775">rs1305272775</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1305272775&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.790</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618348.1">GOLGA6L9:ENST00000618348.1:c.451C&gt;G:p.(Gln151Glu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.79010606</dd>
                        
                        <dd>CADD: 0.790 (6.780)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000172969">FRG2C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.954</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.954</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.75665820C&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=13096122">rs13096122</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs13096122&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.954</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308062.7">FRG2C:ENST00000308062.7:c.628C&gt;A:p.(Leu210Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000464571.1">FRG2C:ENST00000464571.1:c.625C&gt;A:p.(Leu209Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000638439.1">FRG2C:ENST00000638439.1:n.137+4200G&gt;T:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9536553</dd>
                                    
                                    <dd>Polyphen2: 0.731 (P)</dd>
                                    <dd>SIFT: 0.217 (T)</dd>
                                    <dd>CADD: 0.954 (13.340)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000272674">PCDHB16</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.953</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.953</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr5:g.141184136C&gt;A [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=17844652">rs17844652</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs17844652&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.953</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000609684.2">PCDHB16:ENST00000609684.2:c.1577C&gt;A:p.(Ala526Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9534414</dd>
                                    
                                    <dd>CADD: 0.953 (13.320)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr5:g.141184133A&gt;G [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=17844651">rs17844651</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs17844651&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.901</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000609684.2">PCDHB16:ENST00000609684.2:c.1574A&gt;G:p.(Gln525Arg)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9011447</dd>
                        
                        <dd>CADD: 0.901 (10.050)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr5:g.141184137T&gt;G [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=17844653">rs17844653</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs17844653&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.012</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000609684.2">PCDHB16:ENST00000609684.2:c.1578T&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.011902034</dd>
                        
                        <dd>CADD: 0.012 (0.052)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211938">IGHV3-7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.945</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.010</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.945</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.106062336C&gt;T [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.945</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390598.2">IGHV3-7:ENST00000390598.2:c.166G&gt;A:p.(Val56Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9452984</dd>
                                    
                                    <dd>CADD: 0.945 (12.620)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.501</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.106062169G&gt;A [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=113047412">rs113047412</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs113047412&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.501</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390598.2">IGHV3-7:ENST00000390598.2:c.333C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.50146</dd>
                                    
                                    <dd>CADD: 0.501 (3.023)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr14:g.106062358A&gt;G [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=781843443">rs781843443</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs781843443&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.679</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390598.2">IGHV3-7:ENST00000390598.2:c.144T&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.680405</dd>
                        
                        <dd>CADD: 0.680 (4.954)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>ExAC AFR: 0.0103%</dd>
                        <dd>ExAC AMR: 0.0173%</dd>
                        <dd>ExAC NFE: 0.0030%</dd>
                        <dd>gnomAD_E_AFR: 0.0066%</dd>
                        <dd>gnomAD_E_AMR: 0.0119%</dd>
                        <dd>gnomAD_E_NFE: 0.0028%</dd>
                        <dd>gnomAD_E_SAS: 0.0032%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr14:g.106062439C&gt;T [0/1:0/0:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=782297935">rs782297935</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs782297935&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.059</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390598.2">IGHV3-7:ENST00000390598.2:c.63G&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.05941075</dd>
                        
                        <dd>CADD: 0.059 (0.266)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_E_NFE: 0.0018%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211935">IGHV1-3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.943</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.010</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.943</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.106005398G&gt;A [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1143496">rs1143496</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1143496&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.943</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390595.3">IGHV1-3:ENST00000390595.3:c.50C&gt;T:p.(Ala17Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.94337606</dd>
                                    
                                    <dd>CADD: 0.943 (12.470)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr14:g.106005385G&gt;A [1/1:0/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=36065697">rs36065697</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs36065697&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.730</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390595.3">IGHV1-3:ENST00000390595.3:c.63C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.7297909</dd>
                        
                        <dd>CADD: 0.730 (5.683)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000276950">GSTT4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.934</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.009</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.934</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.23998678T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1177906550">rs1177906550</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1177906550&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.983</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621179.5">GSTT4:ENST00000621179.5:c.590A&gt;G:p.(Gln197Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9830956</dd>
                                    
                                    <dd>CADD: 0.983 (17.720)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr22:g.24005240T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1299755982">rs1299755982</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1299755982&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.885</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621179.5">GSTT4:ENST00000621179.5:c.112+5A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.89316833</dd>
                                    
                                    <dd>CADD: 0.893 (9.713)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_EAS: 0.0617%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000276775">IGHV4-4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.924</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.008</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.924</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.106012008T&gt;C [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=76899215">rs76899215</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs76899215&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.924</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000455737.2">IGHV4-4:ENST00000455737.2:c.267A&gt;G:p.(Ile89Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.92414224</dd>
                                    
                                    <dd>CADD: 0.924 (11.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr14:g.106012218A&gt;G [1/1:0/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=78225485">rs78225485</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs78225485&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.817</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000455737.2">IGHV4-4:ENST00000455737.2:c.57T&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.81710577</dd>
                        
                        <dd>CADD: 0.817 (7.378)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr14:g.106011997T&gt;G [1/1:0/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=77628366">rs77628366</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs77628366&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.451</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000455737.2">IGHV4-4:ENST00000455737.2:c.278A&gt;C:p.(Lys93Thr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.4514705</dd>
                        
                        <dd>CADD: 0.451 (2.608)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr14:g.106012330G&gt;C [1/1:0/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1141386">rs1141386</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1141386&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.039</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000455737.2">IGHV4-4:ENST00000455737.2:c.27C&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.039272964</dd>
                        
                        <dd>CADD: 0.039 (0.174)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204577">LILRB3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.876</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.005</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.876</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54220202T&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=374515400">rs374515400</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs374515400&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.876</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245620.13">LILRB3:ENST00000245620.13:c.1262A&gt;C:p.(His421Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391750.5">LILRB3:ENST00000391750.5:c.1262A&gt;C:p.(His421Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346401.10">LILRB3:ENST00000346401.10:c.1259-286A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.88323843</dd>
                                    
                                    <dd>CADD: 0.883 (9.327)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0599%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.493</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                                <b>chr19:g.54217318C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=147121453">rs147121453</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs147121453&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.972</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245620.13">LILRB3:ENST00000245620.13:c.1752+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346401.10">LILRB3:ENST00000346401.10:c.1785+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391750.5">LILRB3:ENST00000391750.5:c.1749+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.95124716</dd>
                                    
                                    <dd>CADD: 0.951 (13.120)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0799%</dd>
                                    <dd>gnomAD_E_AFR: 0.1858%</dd>
                                    <dd>gnomAD_E_AMR: 0.0450%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0191%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_NFE: 0.0069%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54222365G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=138323850">rs138323850</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs138323850&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.014</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245620.13">LILRB3:ENST00000245620.13:c.268C&gt;G:p.(Gln90Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346401.10">LILRB3:ENST00000346401.10:c.268C&gt;G:p.(Gln90Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391750.5">LILRB3:ENST00000391750.5:c.268C&gt;G:p.(Gln90Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.013720512</dd>
                                    
                                    <dd>CADD: 0.014 (0.060)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.54222457T&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200199363">rs200199363</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200199363&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.000</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245620.13">LILRB3:ENST00000245620.13:c.176A&gt;G:p.(Gln59Arg)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346401.10">LILRB3:ENST00000346401.10:c.176A&gt;G:p.(Gln59Arg)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391750.5">LILRB3:ENST00000391750.5:c.176A&gt;G:p.(Gln59Arg)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 2.3025274E-4</dd>
                        
                        <dd>CADD: 0.000 (0.001)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187537">POTEG</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.852</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.004</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.852</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.19434064T&gt;C [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=57000026">rs57000026</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs57000026&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.852</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000547848.5">POTEG:ENST00000547848.5:c.226A&gt;G:p.(Ser76Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8521232</dd>
                                    
                                    <dd>CADD: 0.852 (8.301)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211596">IGKJ2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.004</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.850</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_INSERTION</span>
                                <b>chr2:g.88861550G&gt;GCCAAAAGTGTACACACAATGGTTC [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390241.3">IGKJ2:ENST00000390241.3:c.13_14insGAACCATTGTGTGTACACTTTTGG:p.(Leu4_Ala5insGlyThrIleValCysThrLeuLeu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000266714">MYO15B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.004</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.850</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">INTERNAL_FEATURE_ELONGATION</span>
                                <b>chr17:g.75600611G&gt;CTTTTTTTTT [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000610510.4">MYO15B:ENST00000610510.4:c.3297delGinsCTTTTTTTTT:p.(Glu1099Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000633867.1">MYO15B:ENST00000633867.1:c.299-20delGinsCTTTTTTTTT:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211941">IGHV3-11</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.843</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.004</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.843</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.106116715T&gt;C [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=75474934">rs75474934</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs75474934&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.843</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390601.3">IGHV3-11:ENST00000390601.3:c.273A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8428552</dd>
                                    
                                    <dd>CADD: 0.843 (8.037)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr14:g.106116768T&gt;C [1/1:0/1:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2003432">rs2003432</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2003432&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.531</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390601.3">IGHV3-11:ENST00000390601.3:c.220A&gt;G:p.(Ser74Gly)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.5309707</dd>
                        
                        <dd>CADD: 0.531 (3.288)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211934">IGHV1-2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.831</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.831</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr14:g.105986679A&gt;T [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=112806369">rs112806369</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs112806369&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.831</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390594.3">IGHV1-2:ENST00000390594.3:c.256T&gt;A:p.(Trp86Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.83149993</dd>
                                    
                                    <dd>CADD: 0.831 (7.734)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000244306">AL589743.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.831</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.831</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr14:g.19331482C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=140953425">rs140953425</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs140953425&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.831</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000648085.1">AL589743.1:ENST00000648085.1:n.711G&gt;A:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8305272</dd>
                                    
                                    <dd>CADD: 0.831 (7.709)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143199">ADCY10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.534</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADCY10%0DNME5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NME5</a> and phenotypic similarity 0.608 to mouse mutant of NME5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0001866, nasal inflammation</dd><dd>HP:0001047, Atopic dermatitis - MP:0001866, nasal inflammation</dd><dd>HP:0001581, Recurrent skin infections - MP:0001866, nasal inflammation</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0001866, nasal inflammation</dd><dd>HP:0002090, Pneumonia - MP:0001866, nasal inflammation</dd><dd>HP:0002205, Recurrent respiratory infections -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/143870">OMIM:143870</a> Hypercalciuria, absorptive, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2197">ORPHA:2197</a> Idiopathic hypercalciuria</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.534</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.167875175A&gt;G [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=142478979">rs142478979</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs142478979&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    <span class="label label-info">LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=706836%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.925</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367848.1">ADCY10:ENST00000367848.1:c.1142T&gt;C:p.(Met381Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367851.9">ADCY10:ENST00000367851.9:c.1418T&gt;C:p.(Met473Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545172.5">ADCY10:ENST00000545172.5:c.959T&gt;C:p.(Met320Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9898842</dd>
                                    
                                    <dd>Polyphen2: 0.342 (B)</dd>
                                    <dd>Mutation Taster: 0.988 (P)</dd>
                                    <dd>SIFT: 0.070 (T)</dd>
                                    <dd>CADD: 0.990 (19.950)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.1808%</dd>
                                    <dd>UK10K: 0.3967%</dd>
                                    <dd>ESP AA: 0.1362%</dd>
                                    <dd>ESP EA: 0.3140%</dd>
                                    <dd>ESP All: 0.2537%</dd>
                                    <dd>ExAC AFR: 0.0770%</dd>
                                    <dd>ExAC AMR: 0.0519%</dd>
                                    <dd>ExAC FIN: 0.0907%</dd>
                                    <dd>ExAC NFE: 0.3208%</dd>
                                    <dd>ExAC SAS: 0.0303%</dd>
                                    <dd>gnomAD_E_AFR: 0.0849%</dd>
                                    <dd>gnomAD_E_AMR: 0.0596%</dd>
                                    <dd>gnomAD_E_FIN: 0.0987%</dd>
                                    <dd>gnomAD_E_NFE: 0.3349%</dd>
                                    <dd>gnomAD_E_OTH: 0.1276%</dd>
                                    <dd>gnomAD_E_SAS: 0.0325%</dd>
                                    <dd>gnomAD_G_AFR: 0.1259%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_FIN: 0.1431%</dd>
                                    <dd>gnomAD_G_NFE: 0.2598%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.167818241T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=41270737">rs41270737</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs41270737&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=696239%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.143</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367848.1">ADCY10:ENST00000367848.1:c.4037A&gt;G:p.(Asn1346Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367851.9">ADCY10:ENST00000367851.9:c.4313A&gt;G:p.(Asn1438Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545172.5">ADCY10:ENST00000545172.5:c.3854A&gt;G:p.(Asn1285Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9661156</dd>
                                    
                                    <dd>Polyphen2: 0.011 (B)</dd>
                                    <dd>SIFT: 0.814 (T)</dd>
                                    <dd>CADD: 0.966 (14.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4792%</dd>
                                    <dd>TOPMed: 0.7406%</dd>
                                    <dd>UK10K: 1.0315%</dd>
                                    <dd>ESP AA: 0.3404%</dd>
                                    <dd>ESP EA: 1.3721%</dd>
                                    <dd>ESP All: 1.0226%</dd>
                                    <dd>ExAC AFR: 0.2316%</dd>
                                    <dd>ExAC AMR: 0.7949%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC FIN: 0.6501%</dd>
                                    <dd>ExAC NFE: 1.0626%</dd>
                                    <dd>ExAC OTH: 0.7743%</dd>
                                    <dd>ExAC SAS: 0.7268%</dd>
                                    <dd>gnomAD_E_AFR: 0.2157%</dd>
                                    <dd>gnomAD_E_AMR: 0.6581%</dd>
                                    <dd>gnomAD_E_ASJ: 0.9137%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_FIN: 0.6906%</dd>
                                    <dd>gnomAD_E_NFE: 1.1047%</dd>
                                    <dd>gnomAD_E_OTH: 1.1848%</dd>
                                    <dd>gnomAD_E_SAS: 0.6660%</dd>
                                    <dd>gnomAD_G_AFR: 0.2290%</dd>
                                    <dd>gnomAD_G_AMR: 0.7160%</dd>
                                    <dd>gnomAD_G_ASJ: 1.9868%</dd>
                                    <dd>gnomAD_G_FIN: 0.8014%</dd>
                                    <dd>gnomAD_G_NFE: 1.1454%</dd>
                                    <dd>gnomAD_G_OTH: 1.2220%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000275969">SPATA31A3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.816</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.816</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.66987715G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=62565523">rs62565523</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs62565523&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.973</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428649.4">SPATA31A3:ENST00000428649.4:c.2783C&gt;T:p.(Thr928Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97289807</dd>
                                    
                                    <dd>CADD: 0.973 (15.670)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.66989995G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=273319">rs273319</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs273319&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.659</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428649.4">SPATA31A3:ENST00000428649.4:c.503C&gt;T:p.(Thr168Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6586499</dd>
                                    
                                    <dd>CADD: 0.659 (4.668)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr9:g.66989082C&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=773528867">rs773528867</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs773528867&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.834</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428649.4">SPATA31A3:ENST00000428649.4:c.1416G&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8338119</dd>
                        
                        <dd>CADD: 0.834 (7.794)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr9:g.66986649T&gt;C [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=202070805">rs202070805</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs202070805&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.520</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428649.4">SPATA31A3:ENST00000428649.4:c.3849A&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.5204874</dd>
                        
                        <dd>CADD: 0.520 (3.192)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000124574">ABCC10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.808</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.808</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr6:g.43444857A&gt;G [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1186522803">rs1186522803</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1186522803&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.808</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000244533.7">ABCC10:ENST00000244533.7:c.2675A&gt;G:p.(Gln892Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372530.9">ABCC10:ENST00000372530.9:c.2759A&gt;G:p.(Gln920Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.80905867</dd>
                                    
                                    <dd>Polyphen2: 0.007 (B)</dd>
                                    <dd>SIFT: 0.564 (T)</dd>
                                    <dd>CADD: 0.809 (7.191)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000243501">AL365232.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr6:g.73216642C&gt;CAAAAAAAAA [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1198387515">rs1198387515</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1198387515&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423730.3">AL365232.1:ENST00000423730.3:c.*229-4_*229-3insTTTTTTTTT:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.24837708</dd>
                                    
                                    <dd>CADD: 0.248 (1.240)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000273907">CA15P2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr22:g.18637317C&gt;T [0/1:0/0:0/0]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611168.1">CA15P2:ENST00000611168.1:n.404+7G&gt;A:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6091591</dd>
                                    
                                    <dd>CADD: 0.609 (4.080)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000127914">AKAP9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.497</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=AKAP9%0DOFD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OFD1</a> and phenotypic similarity 0.701 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with OFD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0002090, Pneumonia</dd><dd>HP:0001047, Atopic dermatitis - HP:0002090, Pneumonia</dd><dd>HP:0001581, Recurrent skin infections - HP:0002090, Pneumonia</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0002090, Pneumonia</dd><dd>HP:0002090, Pneumonia - HP:0002090, Pneumonia</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0002205, Recurrent respiratory infections</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/611820">OMIM:611820</a> ?Long QT syndrome 11 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=101016">ORPHA:101016</a> Romano-Ward syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=130">ORPHA:130</a> Brugada syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.002</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.497</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr7:g.92083702T&gt;A [-/1:-/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=575797632">rs575797632</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs575797632&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.626</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359028.6">AKAP9:ENST00000359028.6:c.8719+7T&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356239.7">AKAP9:ENST00000356239.7:c.8646+47T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358100.6">AKAP9:ENST00000358100.6:c.8505+47T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.012584388</dd>
                                    
                                    <dd>CADD: 0.013 (0.055)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2995%</dd>
                                    <dd>TOPMed: 0.3186%</dd>
                                    <dd>ExAC AFR: 0.6319%</dd>
                                    <dd>ExAC AMR: 0.1504%</dd>
                                    <dd>ExAC EAS: 0.1058%</dd>
                                    <dd>ExAC FIN: 0.1039%</dd>
                                    <dd>ExAC NFE: 0.0687%</dd>
                                    <dd>ExAC OTH: 0.1736%</dd>
                                    <dd>ExAC SAS: 0.0839%</dd>
                                    <dd>gnomAD_E_AFR: 0.8958%</dd>
                                    <dd>gnomAD_E_AMR: 0.1979%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0672%</dd>
                                    <dd>gnomAD_E_EAS: 0.1374%</dd>
                                    <dd>gnomAD_E_FIN: 0.0359%</dd>
                                    <dd>gnomAD_E_NFE: 0.0692%</dd>
                                    <dd>gnomAD_E_OTH: 0.1365%</dd>
                                    <dd>gnomAD_E_SAS: 0.1491%</dd>
                                    <dd>gnomAD_G_AFR: 0.9572%</dd>
                                    <dd>gnomAD_G_AMR: 0.1366%</dd>
                                    <dd>gnomAD_G_EAS: 0.0627%</dd>
                                    <dd>gnomAD_G_FIN: 0.0691%</dd>
                                    <dd>gnomAD_G_NFE: 0.0423%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.91973801C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=35669569">rs35669569</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs35669569&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=189264%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.367</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356239.7">AKAP9:ENST00000356239.7:c.139C&gt;T:p.(His47Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358100.6">AKAP9:ENST00000358100.6:c.139C&gt;T:p.(His47Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359028.6">AKAP9:ENST00000359028.6:c.172C&gt;T:p.(His58Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394564.5">AKAP9:ENST00000394564.5:c.139C&gt;T:p.(His47Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000619023.4">AKAP9:ENST00000619023.4:c.163C&gt;T:p.(His55Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.914</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.086 (T)</dd>
                                    <dd>CADD: 0.876 (9.059)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2196%</dd>
                                    <dd>TOPMed: 0.6841%</dd>
                                    <dd>UK10K: 1.3356%</dd>
                                    <dd>ESP AA: 0.1362%</dd>
                                    <dd>ESP EA: 1.3953%</dd>
                                    <dd>ESP All: 0.9688%</dd>
                                    <dd>ExAC AFR: 0.1552%</dd>
                                    <dd>ExAC AMR: 0.1991%</dd>
                                    <dd>ExAC FIN: 1.4065%</dd>
                                    <dd>ExAC NFE: 1.2703%</dd>
                                    <dd>ExAC OTH: 0.2212%</dd>
                                    <dd>ExAC SAS: 0.0364%</dd>
                                    <dd>gnomAD_E_AFR: 0.1700%</dd>
                                    <dd>gnomAD_E_AMR: 0.3306%</dd>
                                    <dd>gnomAD_E_ASJ: 0.1220%</dd>
                                    <dd>gnomAD_E_FIN: 1.3813%</dd>
                                    <dd>gnomAD_E_NFE: 1.3324%</dd>
                                    <dd>gnomAD_E_OTH: 0.6567%</dd>
                                    <dd>gnomAD_E_SAS: 0.0390%</dd>
                                    <dd>gnomAD_G_AFR: 0.2519%</dd>
                                    <dd>gnomAD_G_AMR: 0.3589%</dd>
                                    <dd>gnomAD_G_FIN: 1.6905%</dd>
                                    <dd>gnomAD_G_NFE: 1.0800%</dd>
                                    <dd>gnomAD_G_OTH: 0.7143%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000011451">WIZ</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.778</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.002</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.778</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.15439162C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=188576887">rs188576887</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs188576887&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.893</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000643092.1">WIZ:ENST00000643092.1:c.1832G&gt;A:p.(Arg611His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000673675.1">WIZ:ENST00000673675.1:c.1832G&gt;A:p.(Arg611His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263381.12">WIZ:ENST00000263381.12:c.206-2033G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.94871384</dd>
                                    
                                    <dd>CADD: 0.949 (12.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1198%</dd>
                                    <dd>TOPMed: 0.1402%</dd>
                                    <dd>UK10K: 0.1058%</dd>
                                    <dd>ExAC NFE: 0.1014%</dd>
                                    <dd>gnomAD_E_AFR: 0.0481%</dd>
                                    <dd>gnomAD_E_AMR: 0.2404%</dd>
                                    <dd>gnomAD_E_FIN: 0.0953%</dd>
                                    <dd>gnomAD_E_NFE: 0.1579%</dd>
                                    <dd>gnomAD_E_OTH: 0.3378%</dd>
                                    <dd>gnomAD_G_AFR: 0.0688%</dd>
                                    <dd>gnomAD_G_AMR: 0.3580%</dd>
                                    <dd>gnomAD_G_FIN: 0.0574%</dd>
                                    <dd>gnomAD_G_NFE: 0.1135%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.15440167C&gt;T [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=567739882">rs567739882</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs567739882&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.663</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000643092.1">WIZ:ENST00000643092.1:c.827G&gt;A:p.(Arg276Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000673675.1">WIZ:ENST00000673675.1:c.827G&gt;A:p.(Arg276Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263381.12">WIZ:ENST00000263381.12:c.206-3038G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.996 (23.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1198%</dd>
                                    <dd>TOPMed: 0.0773%</dd>
                                    <dd>ExAC AFR: 0.3953%</dd>
                                    <dd>ExAC AMR: 1.2500%</dd>
                                    <dd>gnomAD_E_AFR: 0.3267%</dd>
                                    <dd>gnomAD_E_AMR: 0.0474%</dd>
                                    <dd>gnomAD_E_NFE: 0.0062%</dd>
                                    <dd>gnomAD_E_SAS: 0.0045%</dd>
                                    <dd>gnomAD_G_AFR: 0.2417%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.1022%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr19:g.15429971G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=189657005">rs189657005</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs189657005&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.830</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389282.8">WIZ:ENST00000389282.8:c.258C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545156.5">WIZ:ENST00000545156.5:c.258C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000599686.4">WIZ:ENST00000599686.4:c.267C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000599910.6">WIZ:ENST00000599910.6:c.267C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000643092.1">WIZ:ENST00000643092.1:c.2859C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000673675.1">WIZ:ENST00000673675.1:c.3030C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000674001.1">WIZ:ENST00000674001.1:c.561C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000674089.1">WIZ:ENST00000674089.1:c.561C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263381.12">WIZ:ENST00000263381.12:c.530-2438C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8609407</dd>
                        
                        <dd>CADD: 0.861 (8.568)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0799%</dd>
                        <dd>TOPMed: 0.0908%</dd>
                        <dd>UK10K: 0.0529%</dd>
                        <dd>ExAC NFE: 0.1123%</dd>
                        <dd>gnomAD_E_AFR: 0.0485%</dd>
                        <dd>gnomAD_E_AMR: 0.2275%</dd>
                        <dd>gnomAD_E_FIN: 0.0188%</dd>
                        <dd>gnomAD_E_NFE: 0.1473%</dd>
                        <dd>gnomAD_E_OTH: 0.2214%</dd>
                        <dd>gnomAD_G_AFR: 0.0230%</dd>
                        <dd>gnomAD_G_AMR: 0.2387%</dd>
                        <dd>gnomAD_G_NFE: 0.0668%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000168702">LRP1B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.135</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRP1B%0DDDR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DDR2</a> and phenotypic similarity 0.654 to <a href="http://www.omim.org/271665">Spondylometaepiphyseal dysplasia, short limb-hand type</a> associated with DDR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - HP:0006532, Recurrent pneumonia</dd><dd>HP:0001047, Atopic dermatitis - HP:0006532, Recurrent pneumonia</dd><dd>HP:0001581, Recurrent skin infections - HP:0006532, Recurrent pneumonia</dd><dd>HP:0004386, Gastrointestinal inflammation - HP:0006532, Recurrent pneumonia</dd><dd>HP:0002090, Pneumonia - HP:0006532, Recurrent pneumonia</dd><dd>HP:0002205, Recurrent respiratory infections - HP:0006532, Recurrent pneumonia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRP1B%0DDDR2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DDR2</a> and phenotypic similarity 0.303 to mouse mutant of DDR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000964, Eczema - MP:0002908, delayed wound healing</dd><dd>HP:0001047, Atopic dermatitis - MP:0002908, delayed wound healing</dd><dd>HP:0001581, Recurrent skin infections - MP:0002908, delayed wound healing</dd><dd>HP:0004386, Gastrointestinal inflammation - MP:0002908, delayed wound healing</dd><dd>HP:0002090, Pneumonia - MP:0000445, short snout</dd><dd>HP:0002205, Recurrent respiratory infections - MP:0000445, short snout</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.135</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.140334515T&gt;G [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=79879036">rs79879036</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs79879036&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    <span class="label label-info">LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=732776%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.135</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389484.8">LRP1B:ENST00000389484.8:c.12161A&gt;C:p.(Glu4054Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9969097</dd>
                                    
                                    <dd>Polyphen2: 0.484 (P)</dd>
                                    <dd>Mutation Taster: 0.983 (P)</dd>
                                    <dd>SIFT: 0.736 (T)</dd>
                                    <dd>CADD: 0.997 (25.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2196%</dd>
                                    <dd>TOPMed: 0.3823%</dd>
                                    <dd>UK10K: 0.4628%</dd>
                                    <dd>ESP AA: 0.1816%</dd>
                                    <dd>ESP EA: 0.5930%</dd>
                                    <dd>ESP All: 0.4536%</dd>
                                    <dd>ExAC AFR: 0.2028%</dd>
                                    <dd>ExAC AMR: 0.2195%</dd>
                                    <dd>ExAC EAS: 0.0118%</dd>
                                    <dd>ExAC FIN: 0.1062%</dd>
                                    <dd>ExAC NFE: 0.4809%</dd>
                                    <dd>ExAC OTH: 0.3472%</dd>
                                    <dd>ExAC SAS: 0.2423%</dd>
                                    <dd>gnomAD_E_AFR: 0.1598%</dd>
                                    <dd>gnomAD_E_AMR: 0.2610%</dd>
                                    <dd>gnomAD_E_ASJ: 1.4385%</dd>
                                    <dd>gnomAD_E_EAS: 0.0119%</dd>
                                    <dd>gnomAD_E_FIN: 0.0823%</dd>
                                    <dd>gnomAD_E_NFE: 0.4679%</dd>
                                    <dd>gnomAD_E_OTH: 0.4205%</dd>
                                    <dd>gnomAD_E_SAS: 0.2353%</dd>
                                    <dd>gnomAD_G_AFR: 0.1717%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_ASJ: 2.0000%</dd>
                                    <dd>gnomAD_G_FIN: 0.0859%</dd>
                                    <dd>gnomAD_G_NFE: 0.4201%</dd>
                                    <dd>gnomAD_G_OTH: 0.4082%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000268043">NBPF12</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.660</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.660</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.146988091A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200441040">rs200441040</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200441040&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.980</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439206.6">NBPF12:ENST00000439206.6:c.1403A&gt;G:p.(Glu468Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579935.8">NBPF12:ENST00000579935.8:c.2441A&gt;G:p.(Glu814Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611443.4">NBPF12:ENST00000611443.4:c.3254A&gt;G:p.(Glu1085Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613714.2">NBPF12:ENST00000613714.2:c.2216A&gt;G:p.(Glu739Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617614.4">NBPF12:ENST00000617614.4:c.2437A&gt;G:p.(Arg813Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617844.4">NBPF12:ENST00000617844.4:c.3254A&gt;G:p.(Glu1085Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617931.4">NBPF12:ENST00000617931.4:c.3254A&gt;G:p.(Glu1085Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617864.3">NBPF12:ENST00000617864.3:c.*564A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9803664</dd>
                                    
                                    <dd>CADD: 0.980 (17.070)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.146960192A&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=587660653">rs587660653</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs587660653&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.339</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439206.6">NBPF12:ENST00000439206.6:c.49A&gt;C:p.(Ile17Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579935.8">NBPF12:ENST00000579935.8:c.49A&gt;C:p.(Ile17Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611443.4">NBPF12:ENST00000611443.4:c.49A&gt;C:p.(Ile17Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617614.4">NBPF12:ENST00000617614.4:c.49A&gt;C:p.(Ile17Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617844.4">NBPF12:ENST00000617844.4:c.49A&gt;C:p.(Ile17Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617864.3">NBPF12:ENST00000617864.3:c.49A&gt;C:p.(Ile17Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617931.4">NBPF12:ENST00000617931.4:c.49A&gt;C:p.(Ile17Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613714.2">NBPF12:ENST00000613714.2:c.-36+186A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7229233</dd>
                                    
                                    <dd>CADD: 0.723 (5.574)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.1389%</dd>
                                    <dd>gnomAD_E_AMR: 0.6507%</dd>
                                    <dd>gnomAD_E_ASJ: 1.1924%</dd>
                                    <dd>gnomAD_E_FIN: 1.5850%</dd>
                                    <dd>gnomAD_E_NFE: 1.5932%</dd>
                                    <dd>gnomAD_E_OTH: 1.0796%</dd>
                                    <dd>gnomAD_E_SAS: 0.2300%</dd>
                                    <dd>gnomAD_G_AFR: 0.3405%</dd>
                                    <dd>gnomAD_G_AMR: 0.3807%</dd>
                                    <dd>gnomAD_G_ASJ: 0.4505%</dd>
                                    <dd>gnomAD_G_FIN: 0.9033%</dd>
                                    <dd>gnomAD_G_NFE: 1.3727%</dd>
                                    <dd>gnomAD_G_OTH: 0.6711%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211659">IGLV3-25</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.653</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.653</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.22687076A&gt;C [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=565679073">rs565679073</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs565679073&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.653</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390305.2">IGLV3-25:ENST00000390305.2:c.142A&gt;C:p.(Lys48Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.65683734</dd>
                                    
                                    <dd>CADD: 0.657 (4.645)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0399%</dd>
                                    <dd>ExAC AFR: 0.0204%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000275395">FCGBP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.612</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.612</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.39886439A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=746639255">rs746639255</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs746639255&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.737</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.7740T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.4335T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7463704</dd>
                                    
                                    <dd>CADD: 0.746 (5.958)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0267%</dd>
                                    <dd>gnomAD_E_AMR: 0.0662%</dd>
                                    <dd>gnomAD_E_EAS: 0.0099%</dd>
                                    <dd>gnomAD_E_FIN: 0.0087%</dd>
                                    <dd>gnomAD_E_NFE: 0.0876%</dd>
                                    <dd>gnomAD_E_OTH: 0.0313%</dd>
                                    <dd>gnomAD_E_SAS: 0.0455%</dd>
                                    <dd>gnomAD_G_NFE: 0.0785%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.39886277G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1231492703">rs1231492703</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1231492703&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.487</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.7902C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.4497C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.48737472</dd>
                                    
                                    <dd>CADD: 0.487 (2.902)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.39886422A&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=771631855">rs771631855</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs771631855&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.109</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.7757T&gt;A:p.(Leu2586Gln)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.4352T&gt;A:p.(Leu1451Gln)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.16131282</dd>
                        
                        <dd>CADD: 0.161 (0.764)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_E_AMR: 0.0046%</dd>
                        <dd>gnomAD_E_EAS: 1.2286%</dd>
                        <dd>gnomAD_E_OTH: 0.0604%</dd>
                        <dd>gnomAD_E_SAS: 0.0188%</dd>
                        <dd>gnomAD_G_EAS: 0.2000%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr19:g.39886283C&gt;G [0/1:0/1:0/0]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=770357345">rs770357345</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs770357345&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.082</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.7896G&gt;C:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.4491G&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.0820902</dd>
                        
                        <dd>CADD: 0.082 (0.372)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr19:g.39886355C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2916064">rs2916064</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2916064&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.053</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.7824G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.4419G&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.052890897</dd>
                        
                        <dd>CADD: 0.053 (0.236)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_E_AFR: 0.0323%</dd>
                        <dd>gnomAD_E_AMR: 0.0114%</dd>
                        <dd>gnomAD_E_ASJ: 0.0268%</dd>
                        <dd>gnomAD_E_NFE: 0.0028%</dd>
                        <dd>gnomAD_E_SAS: 0.0087%</dd>
                        <dd>gnomAD_G_AFR: 0.0396%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000132017">DCAF15</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.583</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.583</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr19:g.13956115G&gt;GC [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=565233977">rs565233977</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs565233977&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.760</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000254337.10">DCAF15:ENST00000254337.10:c.474-8_474-7insC:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8897192</dd>
                                    
                                    <dd>CADD: 0.890 (9.575)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1198%</dd>
                                    <dd>TOPMed: 0.1609%</dd>
                                    <dd>UK10K: 0.2248%</dd>
                                    <dd>ESP AA: 0.0939%</dd>
                                    <dd>ESP EA: 0.1213%</dd>
                                    <dd>ESP All: 0.1120%</dd>
                                    <dd>ExAC AFR: 0.0239%</dd>
                                    <dd>ExAC AMR: 0.1624%</dd>
                                    <dd>ExAC NFE: 0.2413%</dd>
                                    <dd>gnomAD_E_AFR: 0.0211%</dd>
                                    <dd>gnomAD_E_AMR: 0.1261%</dd>
                                    <dd>gnomAD_E_ASJ: 0.7298%</dd>
                                    <dd>gnomAD_E_FIN: 0.0048%</dd>
                                    <dd>gnomAD_E_NFE: 0.1868%</dd>
                                    <dd>gnomAD_E_OTH: 0.1528%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.0345%</dd>
                                    <dd>gnomAD_G_AMR: 0.2392%</dd>
                                    <dd>gnomAD_G_ASJ: 0.6623%</dd>
                                    <dd>gnomAD_G_NFE: 0.1808%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.13959660G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=111418954">rs111418954</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs111418954&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.406</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000254337.10">DCAF15:ENST00000254337.10:c.1298G&gt;A:p.(Arg433Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9918717</dd>
                                    
                                    <dd>Polyphen2: 0.000 (B)</dd>
                                    <dd>SIFT: 0.385 (T)</dd>
                                    <dd>CADD: 0.992 (20.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3395%</dd>
                                    <dd>TOPMed: 0.4794%</dd>
                                    <dd>ESP AA: 1.6795%</dd>
                                    <dd>ESP All: 0.5690%</dd>
                                    <dd>ExAC AFR: 1.4442%</dd>
                                    <dd>ExAC AMR: 0.0520%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 1.4477%</dd>
                                    <dd>gnomAD_E_AMR: 0.0447%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0548%</dd>
                                    <dd>gnomAD_G_AFR: 1.3902%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_EAS: 0.0618%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000270629">NBPF14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.529</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.529</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.148561435T&gt;C [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1354268501">rs1354268501</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1354268501&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.529</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614999.4">NBPF14:ENST00000614999.4:c.3912A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581201.5">NBPF14:ENST00000581201.5:c.2252-8605A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611826.4">NBPF14:ENST00000611826.4:c.1214-8605A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621066.4">NBPF14:ENST00000621066.4:c.2188+1562A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621070.4">NBPF14:ENST00000621070.4:c.2027-8605A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614329.4">NBPF14:ENST00000614329.4:c.*375-8605A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616120.4">NBPF14:ENST00000616120.4:c.*375-8605A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620240.1">NBPF14:ENST00000620240.1:c.*375-8605A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5285887</dd>
                                    
                                    <dd>CADD: 0.529 (3.266)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167613">LAIR1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.424</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.424</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54360069C&gt;G [1/1:0/1:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=190462445">rs190462445</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs190462445&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.424</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391742.7">LAIR1:ENST00000391742.7:c.368G&gt;C:p.(Ser123Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391743.7">LAIR1:ENST00000391743.7:c.314G&gt;C:p.(Ser105Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000434277.6">LAIR1:ENST00000434277.6:c.365G&gt;C:p.(Ser122Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000348231.8">LAIR1:ENST00000348231.8:c.364+847G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000474878.5">LAIR1:ENST00000474878.5:c.361+847G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4244275</dd>
                                    
                                    <dd>CADD: 0.424 (2.399)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000064042">LIMCH1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.401</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.401</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr4:g.41685780G&gt;A [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=61733763">rs61733763</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs61733763&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.656</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313860.11">LIMCH1:ENST00000313860.11:c.2886G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381753.8">LIMCH1:ENST00000381753.8:c.2385G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396595.7">LIMCH1:ENST00000396595.7:c.2421G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000503057.6">LIMCH1:ENST00000503057.6:c.4038G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000508501.5">LIMCH1:ENST00000508501.5:c.2883G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000509277.5">LIMCH1:ENST00000509277.5:c.2385G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000511496.5">LIMCH1:ENST00000511496.5:c.2406G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512632.5">LIMCH1:ENST00000512632.5:c.2655G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512820.5">LIMCH1:ENST00000512820.5:c.2922G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512946.5">LIMCH1:ENST00000512946.5:c.2886G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513024.5">LIMCH1:ENST00000513024.5:c.2445G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000514096.5">LIMCH1:ENST00000514096.5:c.2406G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8081331</dd>
                                    
                                    <dd>CADD: 0.808 (7.170)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4992%</dd>
                                    <dd>TOPMed: 0.7104%</dd>
                                    <dd>UK10K: 0.6083%</dd>
                                    <dd>ESP AA: 0.8398%</dd>
                                    <dd>ESP EA: 0.6628%</dd>
                                    <dd>ESP All: 0.7227%</dd>
                                    <dd>ExAC AFR: 0.8559%</dd>
                                    <dd>ExAC AMR: 0.4693%</dd>
                                    <dd>ExAC EAS: 0.0234%</dd>
                                    <dd>ExAC NFE: 0.6242%</dd>
                                    <dd>ExAC OTH: 0.3311%</dd>
                                    <dd>ExAC SAS: 0.7876%</dd>
                                    <dd>gnomAD_E_AFR: 0.8702%</dd>
                                    <dd>gnomAD_E_AMR: 0.5088%</dd>
                                    <dd>gnomAD_E_ASJ: 0.3661%</dd>
                                    <dd>gnomAD_E_EAS: 0.0233%</dd>
                                    <dd>gnomAD_E_NFE: 0.6633%</dd>
                                    <dd>gnomAD_E_OTH: 0.5865%</dd>
                                    <dd>gnomAD_E_SAS: 0.7906%</dd>
                                    <dd>gnomAD_G_AFR: 0.8137%</dd>
                                    <dd>gnomAD_G_AMR: 0.4785%</dd>
                                    <dd>gnomAD_G_ASJ: 0.3311%</dd>
                                    <dd>gnomAD_G_EAS: 0.0619%</dd>
                                    <dd>gnomAD_G_NFE: 0.5397%</dd>
                                    <dd>gnomAD_G_OTH: 0.7128%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.41646750G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=148133956">rs148133956</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs148133956&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.145</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313860.11">LIMCH1:ENST00000313860.11:c.1522G&gt;A:p.(Ala508Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381753.8">LIMCH1:ENST00000381753.8:c.1024G&gt;A:p.(Ala342Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396595.7">LIMCH1:ENST00000396595.7:c.1060G&gt;A:p.(Ala354Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000503057.6">LIMCH1:ENST00000503057.6:c.2677G&gt;A:p.(Ala893Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000508501.5">LIMCH1:ENST00000508501.5:c.1522G&gt;A:p.(Ala508Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000509277.5">LIMCH1:ENST00000509277.5:c.1024G&gt;A:p.(Ala342Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000511496.5">LIMCH1:ENST00000511496.5:c.1045G&gt;A:p.(Ala349Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512632.5">LIMCH1:ENST00000512632.5:c.1522G&gt;A:p.(Ala508Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512820.5">LIMCH1:ENST00000512820.5:c.1486G&gt;A:p.(Ala496Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512946.5">LIMCH1:ENST00000512946.5:c.1522G&gt;A:p.(Ala508Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513024.5">LIMCH1:ENST00000513024.5:c.1009G&gt;A:p.(Ala337Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000514096.5">LIMCH1:ENST00000514096.5:c.1045G&gt;A:p.(Ala349Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97435516</dd>
                                    
                                    <dd>Polyphen2: 0.269 (B)</dd>
                                    <dd>SIFT: 0.284 (T)</dd>
                                    <dd>CADD: 0.974 (15.910)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3594%</dd>
                                    <dd>TOPMed: 0.5702%</dd>
                                    <dd>UK10K: 0.6744%</dd>
                                    <dd>ESP AA: 0.2043%</dd>
                                    <dd>ESP EA: 0.6047%</dd>
                                    <dd>ESP All: 0.4690%</dd>
                                    <dd>ExAC AFR: 0.1538%</dd>
                                    <dd>ExAC AMR: 0.8464%</dd>
                                    <dd>ExAC FIN: 0.3175%</dd>
                                    <dd>ExAC NFE: 0.7133%</dd>
                                    <dd>ExAC OTH: 0.7709%</dd>
                                    <dd>ExAC SAS: 0.0908%</dd>
                                    <dd>gnomAD_E_AFR: 0.1438%</dd>
                                    <dd>gnomAD_E_AMR: 0.8070%</dd>
                                    <dd>gnomAD_E_ASJ: 1.3604%</dd>
                                    <dd>gnomAD_E_FIN: 0.4081%</dd>
                                    <dd>gnomAD_E_NFE: 0.7547%</dd>
                                    <dd>gnomAD_E_OTH: 0.8020%</dd>
                                    <dd>gnomAD_E_SAS: 0.1137%</dd>
                                    <dd>gnomAD_G_AFR: 0.1375%</dd>
                                    <dd>gnomAD_G_AMR: 0.8353%</dd>
                                    <dd>gnomAD_G_ASJ: 1.9868%</dd>
                                    <dd>gnomAD_G_FIN: 0.3150%</dd>
                                    <dd>gnomAD_G_NFE: 0.4596%</dd>
                                    <dd>gnomAD_G_OTH: 0.4073%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000276040">SPATA31A7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.254</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.254</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.61194086C&gt;T [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200832265">rs200832265</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200832265&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.254</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000619167.1">SPATA31A7:ENST00000619167.1:c.2000C&gt;T:p.(Pro667Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.25441015</dd>
                                    
                                    <dd>CADD: 0.254 (1.275)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000148513">ANKRD30A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.116</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.116</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr10:g.37125862C&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=546050115">rs546050115</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs546050115&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.200</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374660.6">ANKRD30A:ENST00000374660.6:c.75C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602533.6">ANKRD30A:ENST00000602533.6:c.75C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611781.4">ANKRD30A:ENST00000611781.4:c.75C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361713.1">ANKRD30A:ENST00000361713.1:c.-94C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.21584845</dd>
                                    
                                    <dd>CADD: 0.216 (1.056)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0998%</dd>
                                    <dd>TOPMed: 0.3122%</dd>
                                    <dd>UK10K: 0.3438%</dd>
                                    <dd>gnomAD_G_AFR: 0.1376%</dd>
                                    <dd>gnomAD_G_AMR: 0.3580%</dd>
                                    <dd>gnomAD_G_ASJ: 0.3311%</dd>
                                    <dd>gnomAD_G_FIN: 0.0573%</dd>
                                    <dd>gnomAD_G_NFE: 0.4341%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr10:g.37193080T&gt;C [0/1:0/1:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=189638660">rs189638660</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs189638660&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.032</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361713.1">ANKRD30A:ENST00000361713.1:c.2361T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374660.6">ANKRD30A:ENST00000374660.6:c.2886T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602533.6">ANKRD30A:ENST00000602533.6:c.2529T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611781.4">ANKRD30A:ENST00000611781.4:c.2529T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.039936423</dd>
                                    
                                    <dd>CADD: 0.040 (0.177)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1398%</dd>
                                    <dd>TOPMed: 0.2636%</dd>
                                    <dd>ESP AA: 0.0279%</dd>
                                    <dd>ESP EA: 0.5041%</dd>
                                    <dd>ESP All: 0.3584%</dd>
                                    <dd>ExAC AFR: 0.0715%</dd>
                                    <dd>ExAC AMR: 0.1987%</dd>
                                    <dd>ExAC FIN: 0.8923%</dd>
                                    <dd>ExAC NFE: 0.4378%</dd>
                                    <dd>ExAC OTH: 0.4454%</dd>
                                    <dd>ExAC SAS: 0.0182%</dd>
                                    <dd>gnomAD_E_AFR: 0.0524%</dd>
                                    <dd>gnomAD_E_AMR: 0.1489%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0305%</dd>
                                    <dd>gnomAD_E_FIN: 0.6191%</dd>
                                    <dd>gnomAD_E_NFE: 0.4328%</dd>
                                    <dd>gnomAD_E_OTH: 0.3473%</dd>
                                    <dd>gnomAD_E_SAS: 0.0163%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_AMR: 0.3589%</dd>
                                    <dd>gnomAD_G_FIN: 0.4590%</dd>
                                    <dd>gnomAD_G_NFE: 0.4731%</dd>
                                    <dd>gnomAD_G_OTH: 0.2037%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196275">GTF2IRD2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.064</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.064</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr7:g.74798012T&gt;C [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2529319">rs2529319</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2529319&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.064</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000451013.6">GTF2IRD2:ENST00000451013.6:c.1500A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000651129.1">GTF2IRD2:ENST00000651129.1:c.1986A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.063516796</dd>
                                    
                                    <dd>CADD: 0.064 (0.285)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000271425">NBPF10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.025</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.025</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.146121514G&gt;C [0/1:0/0:0/0]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1382137118">rs1382137118</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1382137118&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.025</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583866.9">NBPF10:ENST00000583866.9:c.2742C&gt;G:p.(Phe914Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617010.2">NBPF10:ENST00000617010.2:c.-5134C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.02545923</dd>
                                    
                                    <dd>CADD: 0.025 (0.112)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>

    

    <div id="about">
        <h3>About</h3>
        <p>
            The Exomizer is a Java program that functionally annotates variants from whole-exome
            sequencing data starting from a VCF file (version 4). The functional annotation code is
            based on <a href="https://github.com/charite/jannovar/">Jannovar</a> and uses
            <a href="http://genome.ucsc.edu/">UCSC</a> KnownGene transcript definitions and
            hg19 genomic coordinates
        </p>
        <p>
            Variants are prioritized according to user-defined criteria on variant frequency, pathogenicity,
            quality, inheritance pattern, and model organism phenotype data. Predicted pathogenicity data
            was extracted from the <a href="http://www.ncbi.nlm.nih.gov/pubmed/21520341">dbNSFP</a> resource.
        </p>
        <p>
            Developed by the Computational Biology and Bioinformatics group at the
            <a href="http://genetik.charite.de/">Institute for Medical Genetics and Human Genetics</a> of the
            <a href="http://www.charite.de">Charit&eacute; - Universit&auml;tsmedizin Berlin</a>, the Mouse
            Informatics Group at the <a href="http://www.sanger.ac.uk/">Sanger Institute</a> and the Smedley group at
            <a href="https://qmul.ac.uk/whri">Queen Mary University of London</a>.
        </p>


        <p>Problems, suggestions, or comments? Please <a href="mailto:peter.robinson@charite.de">let us know</a></p>
    </div>
</div>
</body>
</html>
