<!--
  ~ The Exomiser - A tool to annotate and prioritize genomic variants
  ~
  ~ Copyright (c) 2016-2019 Queen Mary University of London.
  ~ Copyright (c) 2012-2016 Charité Universitätsmedizin Berlin and Genome Research Ltd.
  ~
  ~ This program is free software: you can redistribute it and/or modify
  ~ it under the terms of the GNU Affero General Public License as
  ~ published by the Free Software Foundation, either version 3 of the
  ~ License, or (at your option) any later version.
  ~
  ~ This program is distributed in the hope that it will be useful,
  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  ~ GNU Affero General Public License for more details.
  ~
  ~ You should have received a copy of the GNU Affero General Public License
  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  -->

<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>
    <title>The Exomiser - A Tool to Annotate and Prioritise Whole-Exome Sequencing Data</title>
    <meta charset="UTF-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <style>
        /*!
 * Bootstrap v3.3.7 (http://getbootstrap.com)
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 *//*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */html{font-family:sans-serif;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block;vertical-align:baseline}audio:not([controls]){display:none;height:0}[hidden],template{display:none}a{background-color:transparent}a:active,a:hover{outline:0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:700}dfn{font-style:italic}h1{margin:.67em 0;font-size:2em}mark{color:#000;background:#ff0}small{font-size:80%}sub,sup{position:relative;font-size:75%;line-height:0;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}img{border:0}svg:not(:root){overflow:hidden}figure{margin:1em 40px}hr{height:0;-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}pre{overflow:auto}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}button,input,optgroup,select,textarea{margin:0;font:inherit;color:inherit}button{overflow:visible}button,select{text-transform:none}button,html input[type=button],input[type=reset],input[type=submit]{-webkit-appearance:button;cursor:pointer}button[disabled],html input[disabled]{cursor:default}button::-moz-focus-inner,input::-moz-focus-inner{padding:0;border:0}input{line-height:normal}input[type=checkbox],input[type=radio]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;padding:0}input[type=number]::-webkit-inner-spin-button,input[type=number]::-webkit-outer-spin-button{height:auto}input[type=search]{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;-webkit-appearance:textfield}input[type=search]::-webkit-search-cancel-button,input[type=search]::-webkit-search-decoration{-webkit-appearance:none}fieldset{padding:.35em .625em .75em;margin:0 2px;border:1px solid silver}legend{padding:0;border:0}textarea{overflow:auto}optgroup{font-weight:700}table{border-spacing:0;border-collapse:collapse}td,th{padding:0}/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */@media print{*,:after,:before{color:#000!important;text-shadow:none!important;background:0 0!important;-webkit-box-shadow:none!important;box-shadow:none!important}a,a:visited{text-decoration:underline}a[href]:after{content:" (" attr(href) ")"}abbr[title]:after{content:" (" attr(title) ")"}a[href^="javascript:"]:after,a[href^="#"]:after{content:""}blockquote,pre{border:1px solid #999;page-break-inside:avoid}thead{display:table-header-group}img,tr{page-break-inside:avoid}img{max-width:100%!important}h2,h3,p{orphans:3;widows:3}h2,h3{page-break-after:avoid}.navbar{display:none}.btn>.caret,.dropup>.btn>.caret{border-top-color:#000!important}.label{border:1px solid #000}.table{border-collapse:collapse!important}.table td,.table th{background-color:#fff!important}.table-bordered td,.table-bordered th{border:1px solid #ddd!important}}@font-face{font-family:'Glyphicons Halflings';src:url(../fonts/glyphicons-halflings-regular.eot);src:url(../fonts/glyphicons-halflings-regular.eot?#iefix) format('embedded-opentype'),url(../fonts/glyphicons-halflings-regular.woff2) format('woff2'),url(../fonts/glyphicons-halflings-regular.woff) format('woff'),url(../fonts/glyphicons-halflings-regular.ttf) format('truetype'),url(../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular) format('svg')}.glyphicon{position:relative;top:1px;display:inline-block;font-family:'Glyphicons Halflings';font-style:normal;font-weight:400;line-height:1;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.glyphicon-asterisk:before{content:"\002a"}.glyphicon-plus:before{content:"\002b"}.glyphicon-eur:before,.glyphicon-euro:before{content:"\20ac"}.glyphicon-minus:before{content:"\2212"}.glyphicon-cloud:before{content:"\2601"}.glyphicon-envelope:before{content:"\2709"}.glyphicon-pencil:before{content:"\270f"}.glyphicon-glass:before{content:"\e001"}.glyphicon-music:before{content:"\e002"}.glyphicon-search:before{content:"\e003"}.glyphicon-heart:before{content:"\e005"}.glyphicon-star:before{content:"\e006"}.glyphicon-star-empty:before{content:"\e007"}.glyphicon-user:before{content:"\e008"}.glyphicon-film:before{content:"\e009"}.glyphicon-th-large:before{content:"\e010"}.glyphicon-th:before{content:"\e011"}.glyphicon-th-list:before{content:"\e012"}.glyphicon-ok:before{content:"\e013"}.glyphicon-remove:before{content:"\e014"}.glyphicon-zoom-in:before{content:"\e015"}.glyphicon-zoom-out:before{content:"\e016"}.glyphicon-off:before{content:"\e017"}.glyphicon-signal:before{content:"\e018"}.glyphicon-cog:before{content:"\e019"}.glyphicon-trash:before{content:"\e020"}.glyphicon-home:before{content:"\e021"}.glyphicon-file:before{content:"\e022"}.glyphicon-time:before{content:"\e023"}.glyphicon-road:before{content:"\e024"}.glyphicon-download-alt:before{content:"\e025"}.glyphicon-download:before{content:"\e026"}.glyphicon-upload:before{content:"\e027"}.glyphicon-inbox:before{content:"\e028"}.glyphicon-play-circle:before{content:"\e029"}.glyphicon-repeat:before{content:"\e030"}.glyphicon-refresh:before{content:"\e031"}.glyphicon-list-alt:before{content:"\e032"}.glyphicon-lock:before{content:"\e033"}.glyphicon-flag:before{content:"\e034"}.glyphicon-headphones:before{content:"\e035"}.glyphicon-volume-off:before{content:"\e036"}.glyphicon-volume-down:before{content:"\e037"}.glyphicon-volume-up:before{content:"\e038"}.glyphicon-qrcode:before{content:"\e039"}.glyphicon-barcode:before{content:"\e040"}.glyphicon-tag:before{content:"\e041"}.glyphicon-tags:before{content:"\e042"}.glyphicon-book:before{content:"\e043"}.glyphicon-bookmark:before{content:"\e044"}.glyphicon-print:before{content:"\e045"}.glyphicon-camera:before{content:"\e046"}.glyphicon-font:before{content:"\e047"}.glyphicon-bold:before{content:"\e048"}.glyphicon-italic:before{content:"\e049"}.glyphicon-text-height:before{content:"\e050"}.glyphicon-text-width:before{content:"\e051"}.glyphicon-align-left:before{content:"\e052"}.glyphicon-align-center:before{content:"\e053"}.glyphicon-align-right:before{content:"\e054"}.glyphicon-align-justify:before{content:"\e055"}.glyphicon-list:before{content:"\e056"}.glyphicon-indent-left:before{content:"\e057"}.glyphicon-indent-right:before{content:"\e058"}.glyphicon-facetime-video:before{content:"\e059"}.glyphicon-picture:before{content:"\e060"}.glyphicon-map-marker:before{content:"\e062"}.glyphicon-adjust:before{content:"\e063"}.glyphicon-tint:before{content:"\e064"}.glyphicon-edit:before{content:"\e065"}.glyphicon-share:before{content:"\e066"}.glyphicon-check:before{content:"\e067"}.glyphicon-move:before{content:"\e068"}.glyphicon-step-backward:before{content:"\e069"}.glyphicon-fast-backward:before{content:"\e070"}.glyphicon-backward:before{content:"\e071"}.glyphicon-play:before{content:"\e072"}.glyphicon-pause:before{content:"\e073"}.glyphicon-stop:before{content:"\e074"}.glyphicon-forward:before{content:"\e075"}.glyphicon-fast-forward:before{content:"\e076"}.glyphicon-step-forward:before{content:"\e077"}.glyphicon-eject:before{content:"\e078"}.glyphicon-chevron-left:before{content:"\e079"}.glyphicon-chevron-right:before{content:"\e080"}.glyphicon-plus-sign:before{content:"\e081"}.glyphicon-minus-sign:before{content:"\e082"}.glyphicon-remove-sign:before{content:"\e083"}.glyphicon-ok-sign:before{content:"\e084"}.glyphicon-question-sign:before{content:"\e085"}.glyphicon-info-sign:before{content:"\e086"}.glyphicon-screenshot:before{content:"\e087"}.glyphicon-remove-circle:before{content:"\e088"}.glyphicon-ok-circle:before{content:"\e089"}.glyphicon-ban-circle:before{content:"\e090"}.glyphicon-arrow-left:before{content:"\e091"}.glyphicon-arrow-right:before{content:"\e092"}.glyphicon-arrow-up:before{content:"\e093"}.glyphicon-arrow-down:before{content:"\e094"}.glyphicon-share-alt:before{content:"\e095"}.glyphicon-resize-full:before{content:"\e096"}.glyphicon-resize-small:before{content:"\e097"}.glyphicon-exclamation-sign:before{content:"\e101"}.glyphicon-gift:before{content:"\e102"}.glyphicon-leaf:before{content:"\e103"}.glyphicon-fire:before{content:"\e104"}.glyphicon-eye-open:before{content:"\e105"}.glyphicon-eye-close:before{content:"\e106"}.glyphicon-warning-sign:before{content:"\e107"}.glyphicon-plane:before{content:"\e108"}.glyphicon-calendar:before{content:"\e109"}.glyphicon-random:before{content:"\e110"}.glyphicon-comment:before{content:"\e111"}.glyphicon-magnet:before{content:"\e112"}.glyphicon-chevron-up:before{content:"\e113"}.glyphicon-chevron-down:before{content:"\e114"}.glyphicon-retweet:before{content:"\e115"}.glyphicon-shopping-cart:before{content:"\e116"}.glyphicon-folder-close:before{content:"\e117"}.glyphicon-folder-open:before{content:"\e118"}.glyphicon-resize-vertical:before{content:"\e119"}.glyphicon-resize-horizontal:before{content:"\e120"}.glyphicon-hdd:before{content:"\e121"}.glyphicon-bullhorn:before{content:"\e122"}.glyphicon-bell:before{content:"\e123"}.glyphicon-certificate:before{content:"\e124"}.glyphicon-thumbs-up:before{content:"\e125"}.glyphicon-thumbs-down:before{content:"\e126"}.glyphicon-hand-right:before{content:"\e127"}.glyphicon-hand-left:before{content:"\e128"}.glyphicon-hand-up:before{content:"\e129"}.glyphicon-hand-down:before{content:"\e130"}.glyphicon-circle-arrow-right:before{content:"\e131"}.glyphicon-circle-arrow-left:before{content:"\e132"}.glyphicon-circle-arrow-up:before{content:"\e133"}.glyphicon-circle-arrow-down:before{content:"\e134"}.glyphicon-globe:before{content:"\e135"}.glyphicon-wrench:before{content:"\e136"}.glyphicon-tasks:before{content:"\e137"}.glyphicon-filter:before{content:"\e138"}.glyphicon-briefcase:before{content:"\e139"}.glyphicon-fullscreen:before{content:"\e140"}.glyphicon-dashboard:before{content:"\e141"}.glyphicon-paperclip:before{content:"\e142"}.glyphicon-heart-empty:before{content:"\e143"}.glyphicon-link:before{content:"\e144"}.glyphicon-phone:before{content:"\e145"}.glyphicon-pushpin:before{content:"\e146"}.glyphicon-usd:before{content:"\e148"}.glyphicon-gbp:before{content:"\e149"}.glyphicon-sort:before{content:"\e150"}.glyphicon-sort-by-alphabet:before{content:"\e151"}.glyphicon-sort-by-alphabet-alt:before{content:"\e152"}.glyphicon-sort-by-order:before{content:"\e153"}.glyphicon-sort-by-order-alt:before{content:"\e154"}.glyphicon-sort-by-attributes:before{content:"\e155"}.glyphicon-sort-by-attributes-alt:before{content:"\e156"}.glyphicon-unchecked:before{content:"\e157"}.glyphicon-expand:before{content:"\e158"}.glyphicon-collapse-down:before{content:"\e159"}.glyphicon-collapse-up:before{content:"\e160"}.glyphicon-log-in:before{content:"\e161"}.glyphicon-flash:before{content:"\e162"}.glyphicon-log-out:before{content:"\e163"}.glyphicon-new-window:before{content:"\e164"}.glyphicon-record:before{content:"\e165"}.glyphicon-save:before{content:"\e166"}.glyphicon-open:before{content:"\e167"}.glyphicon-saved:before{content:"\e168"}.glyphicon-import:before{content:"\e169"}.glyphicon-export:before{content:"\e170"}.glyphicon-send:before{content:"\e171"}.glyphicon-floppy-disk:before{content:"\e172"}.glyphicon-floppy-saved:before{content:"\e173"}.glyphicon-floppy-remove:before{content:"\e174"}.glyphicon-floppy-save:before{content:"\e175"}.glyphicon-floppy-open:before{content:"\e176"}.glyphicon-credit-card:before{content:"\e177"}.glyphicon-transfer:before{content:"\e178"}.glyphicon-cutlery:before{content:"\e179"}.glyphicon-header:before{content:"\e180"}.glyphicon-compressed:before{content:"\e181"}.glyphicon-earphone:before{content:"\e182"}.glyphicon-phone-alt:before{content:"\e183"}.glyphicon-tower:before{content:"\e184"}.glyphicon-stats:before{content:"\e185"}.glyphicon-sd-video:before{content:"\e186"}.glyphicon-hd-video:before{content:"\e187"}.glyphicon-subtitles:before{content:"\e188"}.glyphicon-sound-stereo:before{content:"\e189"}.glyphicon-sound-dolby:before{content:"\e190"}.glyphicon-sound-5-1:before{content:"\e191"}.glyphicon-sound-6-1:before{content:"\e192"}.glyphicon-sound-7-1:before{content:"\e193"}.glyphicon-copyright-mark:before{content:"\e194"}.glyphicon-registration-mark:before{content:"\e195"}.glyphicon-cloud-download:before{content:"\e197"}.glyphicon-cloud-upload:before{content:"\e198"}.glyphicon-tree-conifer:before{content:"\e199"}.glyphicon-tree-deciduous:before{content:"\e200"}.glyphicon-cd:before{content:"\e201"}.glyphicon-save-file:before{content:"\e202"}.glyphicon-open-file:before{content:"\e203"}.glyphicon-level-up:before{content:"\e204"}.glyphicon-copy:before{content:"\e205"}.glyphicon-paste:before{content:"\e206"}.glyphicon-alert:before{content:"\e209"}.glyphicon-equalizer:before{content:"\e210"}.glyphicon-king:before{content:"\e211"}.glyphicon-queen:before{content:"\e212"}.glyphicon-pawn:before{content:"\e213"}.glyphicon-bishop:before{content:"\e214"}.glyphicon-knight:before{content:"\e215"}.glyphicon-baby-formula:before{content:"\e216"}.glyphicon-tent:before{content:"\26fa"}.glyphicon-blackboard:before{content:"\e218"}.glyphicon-bed:before{content:"\e219"}.glyphicon-apple:before{content:"\f8ff"}.glyphicon-erase:before{content:"\e221"}.glyphicon-hourglass:before{content:"\231b"}.glyphicon-lamp:before{content:"\e223"}.glyphicon-duplicate:before{content:"\e224"}.glyphicon-piggy-bank:before{content:"\e225"}.glyphicon-scissors:before{content:"\e226"}.glyphicon-bitcoin:before{content:"\e227"}.glyphicon-btc:before{content:"\e227"}.glyphicon-xbt:before{content:"\e227"}.glyphicon-yen:before{content:"\00a5"}.glyphicon-jpy:before{content:"\00a5"}.glyphicon-ruble:before{content:"\20bd"}.glyphicon-rub:before{content:"\20bd"}.glyphicon-scale:before{content:"\e230"}.glyphicon-ice-lolly:before{content:"\e231"}.glyphicon-ice-lolly-tasted:before{content:"\e232"}.glyphicon-education:before{content:"\e233"}.glyphicon-option-horizontal:before{content:"\e234"}.glyphicon-option-vertical:before{content:"\e235"}.glyphicon-menu-hamburger:before{content:"\e236"}.glyphicon-modal-window:before{content:"\e237"}.glyphicon-oil:before{content:"\e238"}.glyphicon-grain:before{content:"\e239"}.glyphicon-sunglasses:before{content:"\e240"}.glyphicon-text-size:before{content:"\e241"}.glyphicon-text-color:before{content:"\e242"}.glyphicon-text-background:before{content:"\e243"}.glyphicon-object-align-top:before{content:"\e244"}.glyphicon-object-align-bottom:before{content:"\e245"}.glyphicon-object-align-horizontal:before{content:"\e246"}.glyphicon-object-align-left:before{content:"\e247"}.glyphicon-object-align-vertical:before{content:"\e248"}.glyphicon-object-align-right:before{content:"\e249"}.glyphicon-triangle-right:before{content:"\e250"}.glyphicon-triangle-left:before{content:"\e251"}.glyphicon-triangle-bottom:before{content:"\e252"}.glyphicon-triangle-top:before{content:"\e253"}.glyphicon-console:before{content:"\e254"}.glyphicon-superscript:before{content:"\e255"}.glyphicon-subscript:before{content:"\e256"}.glyphicon-menu-left:before{content:"\e257"}.glyphicon-menu-right:before{content:"\e258"}.glyphicon-menu-down:before{content:"\e259"}.glyphicon-menu-up:before{content:"\e260"}*{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}:after,:before{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}html{font-size:10px;-webkit-tap-highlight-color:rgba(0,0,0,0)}body{font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;line-height:1.42857143;color:#333;background-color:#fff}button,input,select,textarea{font-family:inherit;font-size:inherit;line-height:inherit}a{color:#337ab7;text-decoration:none}a:focus,a:hover{color:#23527c;text-decoration:underline}a:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}figure{margin:0}img{vertical-align:middle}.carousel-inner>.item>a>img,.carousel-inner>.item>img,.img-responsive,.thumbnail a>img,.thumbnail>img{display:block;max-width:100%;height:auto}.img-rounded{border-radius:6px}.img-thumbnail{display:inline-block;max-width:100%;height:auto;padding:4px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:all .2s ease-in-out;-o-transition:all .2s ease-in-out;transition:all .2s ease-in-out}.img-circle{border-radius:50%}hr{margin-top:20px;margin-bottom:20px;border:0;border-top:1px solid #eee}.sr-only{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0,0,0,0);border:0}.sr-only-focusable:active,.sr-only-focusable:focus{position:static;width:auto;height:auto;margin:0;overflow:visible;clip:auto}[role=button]{cursor:pointer}.h1,.h2,.h3,.h4,.h5,.h6,h1,h2,h3,h4,h5,h6{font-family:inherit;font-weight:500;line-height:1.1;color:inherit}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-weight:400;line-height:1;color:#777}.h1,.h2,.h3,h1,h2,h3{margin-top:20px;margin-bottom:10px}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small{font-size:65%}.h4,.h5,.h6,h4,h5,h6{margin-top:10px;margin-bottom:10px}.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-size:75%}.h1,h1{font-size:36px}.h2,h2{font-size:30px}.h3,h3{font-size:24px}.h4,h4{font-size:18px}.h5,h5{font-size:14px}.h6,h6{font-size:12px}p{margin:0 0 10px}.lead{margin-bottom:20px;font-size:16px;font-weight:300;line-height:1.4}@media (min-width:768px){.lead{font-size:21px}}.small,small{font-size:85%}.mark,mark{padding:.2em;background-color:#fcf8e3}.text-left{text-align:left}.text-right{text-align:right}.text-center{text-align:center}.text-justify{text-align:justify}.text-nowrap{white-space:nowrap}.text-lowercase{text-transform:lowercase}.text-uppercase{text-transform:uppercase}.text-capitalize{text-transform:capitalize}.text-muted{color:#777}.text-primary{color:#337ab7}a.text-primary:focus,a.text-primary:hover{color:#286090}.text-success{color:#3c763d}a.text-success:focus,a.text-success:hover{color:#2b542c}.text-info{color:#31708f}a.text-info:focus,a.text-info:hover{color:#245269}.text-warning{color:#8a6d3b}a.text-warning:focus,a.text-warning:hover{color:#66512c}.text-danger{color:#a94442}a.text-danger:focus,a.text-danger:hover{color:#843534}.bg-primary{color:#fff;background-color:#337ab7}a.bg-primary:focus,a.bg-primary:hover{background-color:#286090}.bg-success{background-color:#dff0d8}a.bg-success:focus,a.bg-success:hover{background-color:#c1e2b3}.bg-info{background-color:#d9edf7}a.bg-info:focus,a.bg-info:hover{background-color:#afd9ee}.bg-warning{background-color:#fcf8e3}a.bg-warning:focus,a.bg-warning:hover{background-color:#f7ecb5}.bg-danger{background-color:#f2dede}a.bg-danger:focus,a.bg-danger:hover{background-color:#e4b9b9}.page-header{padding-bottom:9px;margin:40px 0 20px;border-bottom:1px solid #eee}ol,ul{margin-top:0;margin-bottom:10px}ol ol,ol ul,ul ol,ul ul{margin-bottom:0}.list-unstyled{padding-left:0;list-style:none}.list-inline{padding-left:0;margin-left:-5px;list-style:none}.list-inline>li{display:inline-block;padding-right:5px;padding-left:5px}dl{margin-top:0;margin-bottom:20px}dd,dt{line-height:1.42857143}dt{font-weight:700}dd{margin-left:0}@media (min-width:768px){.dl-horizontal dt{float:left;width:160px;overflow:hidden;clear:left;text-align:right;text-overflow:ellipsis;white-space:nowrap}.dl-horizontal dd{margin-left:180px}}abbr[data-original-title],abbr[title]{cursor:help;border-bottom:1px dotted #777}.initialism{font-size:90%;text-transform:uppercase}blockquote{padding:10px 20px;margin:0 0 20px;font-size:17.5px;border-left:5px solid #eee}blockquote ol:last-child,blockquote p:last-child,blockquote ul:last-child{margin-bottom:0}blockquote .small,blockquote footer,blockquote small{display:block;font-size:80%;line-height:1.42857143;color:#777}blockquote .small:before,blockquote footer:before,blockquote small:before{content:'\2014 \00A0'}.blockquote-reverse,blockquote.pull-right{padding-right:15px;padding-left:0;text-align:right;border-right:5px solid #eee;border-left:0}.blockquote-reverse .small:before,.blockquote-reverse footer:before,.blockquote-reverse small:before,blockquote.pull-right .small:before,blockquote.pull-right footer:before,blockquote.pull-right small:before{content:''}.blockquote-reverse .small:after,.blockquote-reverse footer:after,.blockquote-reverse small:after,blockquote.pull-right .small:after,blockquote.pull-right footer:after,blockquote.pull-right small:after{content:'\00A0 \2014'}address{margin-bottom:20px;font-style:normal;line-height:1.42857143}code,kbd,pre,samp{font-family:Menlo,Monaco,Consolas,"Courier New",monospace}code{padding:2px 4px;font-size:90%;color:#c7254e;background-color:#f9f2f4;border-radius:4px}kbd{padding:2px 4px;font-size:90%;color:#fff;background-color:#333;border-radius:3px;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.25);box-shadow:inset 0 -1px 0 rgba(0,0,0,.25)}kbd kbd{padding:0;font-size:100%;font-weight:700;-webkit-box-shadow:none;box-shadow:none}pre{display:block;padding:9.5px;margin:0 0 10px;font-size:13px;line-height:1.42857143;color:#333;word-break:break-all;word-wrap:break-word;background-color:#f5f5f5;border:1px solid #ccc;border-radius:4px}pre code{padding:0;font-size:inherit;color:inherit;white-space:pre-wrap;background-color:transparent;border-radius:0}.pre-scrollable{max-height:340px;overflow-y:scroll}.container{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}@media (min-width:768px){.container{width:750px}}@media (min-width:992px){.container{width:970px}}@media (min-width:1200px){.container{width:1170px}}.container-fluid{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}.row{margin-right:-15px;margin-left:-15px}.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9,.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9,.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9,.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{position:relative;min-height:1px;padding-right:15px;padding-left:15px}.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{float:left}.col-xs-12{width:100%}.col-xs-11{width:91.66666667%}.col-xs-10{width:83.33333333%}.col-xs-9{width:75%}.col-xs-8{width:66.66666667%}.col-xs-7{width:58.33333333%}.col-xs-6{width:50%}.col-xs-5{width:41.66666667%}.col-xs-4{width:33.33333333%}.col-xs-3{width:25%}.col-xs-2{width:16.66666667%}.col-xs-1{width:8.33333333%}.col-xs-pull-12{right:100%}.col-xs-pull-11{right:91.66666667%}.col-xs-pull-10{right:83.33333333%}.col-xs-pull-9{right:75%}.col-xs-pull-8{right:66.66666667%}.col-xs-pull-7{right:58.33333333%}.col-xs-pull-6{right:50%}.col-xs-pull-5{right:41.66666667%}.col-xs-pull-4{right:33.33333333%}.col-xs-pull-3{right:25%}.col-xs-pull-2{right:16.66666667%}.col-xs-pull-1{right:8.33333333%}.col-xs-pull-0{right:auto}.col-xs-push-12{left:100%}.col-xs-push-11{left:91.66666667%}.col-xs-push-10{left:83.33333333%}.col-xs-push-9{left:75%}.col-xs-push-8{left:66.66666667%}.col-xs-push-7{left:58.33333333%}.col-xs-push-6{left:50%}.col-xs-push-5{left:41.66666667%}.col-xs-push-4{left:33.33333333%}.col-xs-push-3{left:25%}.col-xs-push-2{left:16.66666667%}.col-xs-push-1{left:8.33333333%}.col-xs-push-0{left:auto}.col-xs-offset-12{margin-left:100%}.col-xs-offset-11{margin-left:91.66666667%}.col-xs-offset-10{margin-left:83.33333333%}.col-xs-offset-9{margin-left:75%}.col-xs-offset-8{margin-left:66.66666667%}.col-xs-offset-7{margin-left:58.33333333%}.col-xs-offset-6{margin-left:50%}.col-xs-offset-5{margin-left:41.66666667%}.col-xs-offset-4{margin-left:33.33333333%}.col-xs-offset-3{margin-left:25%}.col-xs-offset-2{margin-left:16.66666667%}.col-xs-offset-1{margin-left:8.33333333%}.col-xs-offset-0{margin-left:0}@media (min-width:768px){.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9{float:left}.col-sm-12{width:100%}.col-sm-11{width:91.66666667%}.col-sm-10{width:83.33333333%}.col-sm-9{width:75%}.col-sm-8{width:66.66666667%}.col-sm-7{width:58.33333333%}.col-sm-6{width:50%}.col-sm-5{width:41.66666667%}.col-sm-4{width:33.33333333%}.col-sm-3{width:25%}.col-sm-2{width:16.66666667%}.col-sm-1{width:8.33333333%}.col-sm-pull-12{right:100%}.col-sm-pull-11{right:91.66666667%}.col-sm-pull-10{right:83.33333333%}.col-sm-pull-9{right:75%}.col-sm-pull-8{right:66.66666667%}.col-sm-pull-7{right:58.33333333%}.col-sm-pull-6{right:50%}.col-sm-pull-5{right:41.66666667%}.col-sm-pull-4{right:33.33333333%}.col-sm-pull-3{right:25%}.col-sm-pull-2{right:16.66666667%}.col-sm-pull-1{right:8.33333333%}.col-sm-pull-0{right:auto}.col-sm-push-12{left:100%}.col-sm-push-11{left:91.66666667%}.col-sm-push-10{left:83.33333333%}.col-sm-push-9{left:75%}.col-sm-push-8{left:66.66666667%}.col-sm-push-7{left:58.33333333%}.col-sm-push-6{left:50%}.col-sm-push-5{left:41.66666667%}.col-sm-push-4{left:33.33333333%}.col-sm-push-3{left:25%}.col-sm-push-2{left:16.66666667%}.col-sm-push-1{left:8.33333333%}.col-sm-push-0{left:auto}.col-sm-offset-12{margin-left:100%}.col-sm-offset-11{margin-left:91.66666667%}.col-sm-offset-10{margin-left:83.33333333%}.col-sm-offset-9{margin-left:75%}.col-sm-offset-8{margin-left:66.66666667%}.col-sm-offset-7{margin-left:58.33333333%}.col-sm-offset-6{margin-left:50%}.col-sm-offset-5{margin-left:41.66666667%}.col-sm-offset-4{margin-left:33.33333333%}.col-sm-offset-3{margin-left:25%}.col-sm-offset-2{margin-left:16.66666667%}.col-sm-offset-1{margin-left:8.33333333%}.col-sm-offset-0{margin-left:0}}@media (min-width:992px){.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9{float:left}.col-md-12{width:100%}.col-md-11{width:91.66666667%}.col-md-10{width:83.33333333%}.col-md-9{width:75%}.col-md-8{width:66.66666667%}.col-md-7{width:58.33333333%}.col-md-6{width:50%}.col-md-5{width:41.66666667%}.col-md-4{width:33.33333333%}.col-md-3{width:25%}.col-md-2{width:16.66666667%}.col-md-1{width:8.33333333%}.col-md-pull-12{right:100%}.col-md-pull-11{right:91.66666667%}.col-md-pull-10{right:83.33333333%}.col-md-pull-9{right:75%}.col-md-pull-8{right:66.66666667%}.col-md-pull-7{right:58.33333333%}.col-md-pull-6{right:50%}.col-md-pull-5{right:41.66666667%}.col-md-pull-4{right:33.33333333%}.col-md-pull-3{right:25%}.col-md-pull-2{right:16.66666667%}.col-md-pull-1{right:8.33333333%}.col-md-pull-0{right:auto}.col-md-push-12{left:100%}.col-md-push-11{left:91.66666667%}.col-md-push-10{left:83.33333333%}.col-md-push-9{left:75%}.col-md-push-8{left:66.66666667%}.col-md-push-7{left:58.33333333%}.col-md-push-6{left:50%}.col-md-push-5{left:41.66666667%}.col-md-push-4{left:33.33333333%}.col-md-push-3{left:25%}.col-md-push-2{left:16.66666667%}.col-md-push-1{left:8.33333333%}.col-md-push-0{left:auto}.col-md-offset-12{margin-left:100%}.col-md-offset-11{margin-left:91.66666667%}.col-md-offset-10{margin-left:83.33333333%}.col-md-offset-9{margin-left:75%}.col-md-offset-8{margin-left:66.66666667%}.col-md-offset-7{margin-left:58.33333333%}.col-md-offset-6{margin-left:50%}.col-md-offset-5{margin-left:41.66666667%}.col-md-offset-4{margin-left:33.33333333%}.col-md-offset-3{margin-left:25%}.col-md-offset-2{margin-left:16.66666667%}.col-md-offset-1{margin-left:8.33333333%}.col-md-offset-0{margin-left:0}}@media (min-width:1200px){.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9{float:left}.col-lg-12{width:100%}.col-lg-11{width:91.66666667%}.col-lg-10{width:83.33333333%}.col-lg-9{width:75%}.col-lg-8{width:66.66666667%}.col-lg-7{width:58.33333333%}.col-lg-6{width:50%}.col-lg-5{width:41.66666667%}.col-lg-4{width:33.33333333%}.col-lg-3{width:25%}.col-lg-2{width:16.66666667%}.col-lg-1{width:8.33333333%}.col-lg-pull-12{right:100%}.col-lg-pull-11{right:91.66666667%}.col-lg-pull-10{right:83.33333333%}.col-lg-pull-9{right:75%}.col-lg-pull-8{right:66.66666667%}.col-lg-pull-7{right:58.33333333%}.col-lg-pull-6{right:50%}.col-lg-pull-5{right:41.66666667%}.col-lg-pull-4{right:33.33333333%}.col-lg-pull-3{right:25%}.col-lg-pull-2{right:16.66666667%}.col-lg-pull-1{right:8.33333333%}.col-lg-pull-0{right:auto}.col-lg-push-12{left:100%}.col-lg-push-11{left:91.66666667%}.col-lg-push-10{left:83.33333333%}.col-lg-push-9{left:75%}.col-lg-push-8{left:66.66666667%}.col-lg-push-7{left:58.33333333%}.col-lg-push-6{left:50%}.col-lg-push-5{left:41.66666667%}.col-lg-push-4{left:33.33333333%}.col-lg-push-3{left:25%}.col-lg-push-2{left:16.66666667%}.col-lg-push-1{left:8.33333333%}.col-lg-push-0{left:auto}.col-lg-offset-12{margin-left:100%}.col-lg-offset-11{margin-left:91.66666667%}.col-lg-offset-10{margin-left:83.33333333%}.col-lg-offset-9{margin-left:75%}.col-lg-offset-8{margin-left:66.66666667%}.col-lg-offset-7{margin-left:58.33333333%}.col-lg-offset-6{margin-left:50%}.col-lg-offset-5{margin-left:41.66666667%}.col-lg-offset-4{margin-left:33.33333333%}.col-lg-offset-3{margin-left:25%}.col-lg-offset-2{margin-left:16.66666667%}.col-lg-offset-1{margin-left:8.33333333%}.col-lg-offset-0{margin-left:0}}table{background-color:transparent}caption{padding-top:8px;padding-bottom:8px;color:#777;text-align:left}th{text-align:left}.table{width:100%;max-width:100%;margin-bottom:20px}.table>tbody>tr>td,.table>tbody>tr>th,.table>tfoot>tr>td,.table>tfoot>tr>th,.table>thead>tr>td,.table>thead>tr>th{padding:8px;line-height:1.42857143;vertical-align:top;border-top:1px solid #ddd}.table>thead>tr>th{vertical-align:bottom;border-bottom:2px solid #ddd}.table>caption+thead>tr:first-child>td,.table>caption+thead>tr:first-child>th,.table>colgroup+thead>tr:first-child>td,.table>colgroup+thead>tr:first-child>th,.table>thead:first-child>tr:first-child>td,.table>thead:first-child>tr:first-child>th{border-top:0}.table>tbody+tbody{border-top:2px solid #ddd}.table .table{background-color:#fff}.table-condensed>tbody>tr>td,.table-condensed>tbody>tr>th,.table-condensed>tfoot>tr>td,.table-condensed>tfoot>tr>th,.table-condensed>thead>tr>td,.table-condensed>thead>tr>th{padding:5px}.table-bordered{border:1px solid #ddd}.table-bordered>tbody>tr>td,.table-bordered>tbody>tr>th,.table-bordered>tfoot>tr>td,.table-bordered>tfoot>tr>th,.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border:1px solid #ddd}.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border-bottom-width:2px}.table-striped>tbody>tr:nth-of-type(odd){background-color:#f9f9f9}.table-hover>tbody>tr:hover{background-color:#f5f5f5}table col[class*=col-]{position:static;display:table-column;float:none}table td[class*=col-],table th[class*=col-]{position:static;display:table-cell;float:none}.table>tbody>tr.active>td,.table>tbody>tr.active>th,.table>tbody>tr>td.active,.table>tbody>tr>th.active,.table>tfoot>tr.active>td,.table>tfoot>tr.active>th,.table>tfoot>tr>td.active,.table>tfoot>tr>th.active,.table>thead>tr.active>td,.table>thead>tr.active>th,.table>thead>tr>td.active,.table>thead>tr>th.active{background-color:#f5f5f5}.table-hover>tbody>tr.active:hover>td,.table-hover>tbody>tr.active:hover>th,.table-hover>tbody>tr:hover>.active,.table-hover>tbody>tr>td.active:hover,.table-hover>tbody>tr>th.active:hover{background-color:#e8e8e8}.table>tbody>tr.success>td,.table>tbody>tr.success>th,.table>tbody>tr>td.success,.table>tbody>tr>th.success,.table>tfoot>tr.success>td,.table>tfoot>tr.success>th,.table>tfoot>tr>td.success,.table>tfoot>tr>th.success,.table>thead>tr.success>td,.table>thead>tr.success>th,.table>thead>tr>td.success,.table>thead>tr>th.success{background-color:#dff0d8}.table-hover>tbody>tr.success:hover>td,.table-hover>tbody>tr.success:hover>th,.table-hover>tbody>tr:hover>.success,.table-hover>tbody>tr>td.success:hover,.table-hover>tbody>tr>th.success:hover{background-color:#d0e9c6}.table>tbody>tr.info>td,.table>tbody>tr.info>th,.table>tbody>tr>td.info,.table>tbody>tr>th.info,.table>tfoot>tr.info>td,.table>tfoot>tr.info>th,.table>tfoot>tr>td.info,.table>tfoot>tr>th.info,.table>thead>tr.info>td,.table>thead>tr.info>th,.table>thead>tr>td.info,.table>thead>tr>th.info{background-color:#d9edf7}.table-hover>tbody>tr.info:hover>td,.table-hover>tbody>tr.info:hover>th,.table-hover>tbody>tr:hover>.info,.table-hover>tbody>tr>td.info:hover,.table-hover>tbody>tr>th.info:hover{background-color:#c4e3f3}.table>tbody>tr.warning>td,.table>tbody>tr.warning>th,.table>tbody>tr>td.warning,.table>tbody>tr>th.warning,.table>tfoot>tr.warning>td,.table>tfoot>tr.warning>th,.table>tfoot>tr>td.warning,.table>tfoot>tr>th.warning,.table>thead>tr.warning>td,.table>thead>tr.warning>th,.table>thead>tr>td.warning,.table>thead>tr>th.warning{background-color:#fcf8e3}.table-hover>tbody>tr.warning:hover>td,.table-hover>tbody>tr.warning:hover>th,.table-hover>tbody>tr:hover>.warning,.table-hover>tbody>tr>td.warning:hover,.table-hover>tbody>tr>th.warning:hover{background-color:#faf2cc}.table>tbody>tr.danger>td,.table>tbody>tr.danger>th,.table>tbody>tr>td.danger,.table>tbody>tr>th.danger,.table>tfoot>tr.danger>td,.table>tfoot>tr.danger>th,.table>tfoot>tr>td.danger,.table>tfoot>tr>th.danger,.table>thead>tr.danger>td,.table>thead>tr.danger>th,.table>thead>tr>td.danger,.table>thead>tr>th.danger{background-color:#f2dede}.table-hover>tbody>tr.danger:hover>td,.table-hover>tbody>tr.danger:hover>th,.table-hover>tbody>tr:hover>.danger,.table-hover>tbody>tr>td.danger:hover,.table-hover>tbody>tr>th.danger:hover{background-color:#ebcccc}.table-responsive{min-height:.01%;overflow-x:auto}@media screen and (max-width:767px){.table-responsive{width:100%;margin-bottom:15px;overflow-y:hidden;-ms-overflow-style:-ms-autohiding-scrollbar;border:1px solid #ddd}.table-responsive>.table{margin-bottom:0}.table-responsive>.table>tbody>tr>td,.table-responsive>.table>tbody>tr>th,.table-responsive>.table>tfoot>tr>td,.table-responsive>.table>tfoot>tr>th,.table-responsive>.table>thead>tr>td,.table-responsive>.table>thead>tr>th{white-space:nowrap}.table-responsive>.table-bordered{border:0}.table-responsive>.table-bordered>tbody>tr>td:first-child,.table-responsive>.table-bordered>tbody>tr>th:first-child,.table-responsive>.table-bordered>tfoot>tr>td:first-child,.table-responsive>.table-bordered>tfoot>tr>th:first-child,.table-responsive>.table-bordered>thead>tr>td:first-child,.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.table-responsive>.table-bordered>tbody>tr>td:last-child,.table-responsive>.table-bordered>tbody>tr>th:last-child,.table-responsive>.table-bordered>tfoot>tr>td:last-child,.table-responsive>.table-bordered>tfoot>tr>th:last-child,.table-responsive>.table-bordered>thead>tr>td:last-child,.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.table-responsive>.table-bordered>tbody>tr:last-child>td,.table-responsive>.table-bordered>tbody>tr:last-child>th,.table-responsive>.table-bordered>tfoot>tr:last-child>td,.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}}fieldset{min-width:0;padding:0;margin:0;border:0}legend{display:block;width:100%;padding:0;margin-bottom:20px;font-size:21px;line-height:inherit;color:#333;border:0;border-bottom:1px solid #e5e5e5}label{display:inline-block;max-width:100%;margin-bottom:5px;font-weight:700}input[type=search]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}input[type=checkbox],input[type=radio]{margin:4px 0 0;margin-top:1px\9;line-height:normal}input[type=file]{display:block}input[type=range]{display:block;width:100%}select[multiple],select[size]{height:auto}input[type=file]:focus,input[type=checkbox]:focus,input[type=radio]:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}output{display:block;padding-top:7px;font-size:14px;line-height:1.42857143;color:#555}.form-control{display:block;width:100%;height:34px;padding:6px 12px;font-size:14px;line-height:1.42857143;color:#555;background-color:#fff;background-image:none;border:1px solid #ccc;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075);-webkit-transition:border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;-o-transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s;transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s}.form-control:focus{border-color:#66afe9;outline:0;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6);box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6)}.form-control::-moz-placeholder{color:#999;opacity:1}.form-control:-ms-input-placeholder{color:#999}.form-control::-webkit-input-placeholder{color:#999}.form-control::-ms-expand{background-color:transparent;border:0}.form-control[disabled],.form-control[readonly],fieldset[disabled] .form-control{background-color:#eee;opacity:1}.form-control[disabled],fieldset[disabled] .form-control{cursor:not-allowed}textarea.form-control{height:auto}input[type=search]{-webkit-appearance:none}@media screen and (-webkit-min-device-pixel-ratio:0){input[type=date].form-control,input[type=time].form-control,input[type=datetime-local].form-control,input[type=month].form-control{line-height:34px}.input-group-sm input[type=date],.input-group-sm input[type=time],.input-group-sm input[type=datetime-local],.input-group-sm input[type=month],input[type=date].input-sm,input[type=time].input-sm,input[type=datetime-local].input-sm,input[type=month].input-sm{line-height:30px}.input-group-lg input[type=date],.input-group-lg input[type=time],.input-group-lg input[type=datetime-local],.input-group-lg input[type=month],input[type=date].input-lg,input[type=time].input-lg,input[type=datetime-local].input-lg,input[type=month].input-lg{line-height:46px}}.form-group{margin-bottom:15px}.checkbox,.radio{position:relative;display:block;margin-top:10px;margin-bottom:10px}.checkbox label,.radio label{min-height:20px;padding-left:20px;margin-bottom:0;font-weight:400;cursor:pointer}.checkbox input[type=checkbox],.checkbox-inline input[type=checkbox],.radio input[type=radio],.radio-inline input[type=radio]{position:absolute;margin-top:4px\9;margin-left:-20px}.checkbox+.checkbox,.radio+.radio{margin-top:-5px}.checkbox-inline,.radio-inline{position:relative;display:inline-block;padding-left:20px;margin-bottom:0;font-weight:400;vertical-align:middle;cursor:pointer}.checkbox-inline+.checkbox-inline,.radio-inline+.radio-inline{margin-top:0;margin-left:10px}fieldset[disabled] input[type=checkbox],fieldset[disabled] input[type=radio],input[type=checkbox].disabled,input[type=checkbox][disabled],input[type=radio].disabled,input[type=radio][disabled]{cursor:not-allowed}.checkbox-inline.disabled,.radio-inline.disabled,fieldset[disabled] .checkbox-inline,fieldset[disabled] .radio-inline{cursor:not-allowed}.checkbox.disabled label,.radio.disabled label,fieldset[disabled] .checkbox label,fieldset[disabled] .radio label{cursor:not-allowed}.form-control-static{min-height:34px;padding-top:7px;padding-bottom:7px;margin-bottom:0}.form-control-static.input-lg,.form-control-static.input-sm{padding-right:0;padding-left:0}.input-sm{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-sm{height:30px;line-height:30px}select[multiple].input-sm,textarea.input-sm{height:auto}.form-group-sm .form-control{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.form-group-sm select.form-control{height:30px;line-height:30px}.form-group-sm select[multiple].form-control,.form-group-sm textarea.form-control{height:auto}.form-group-sm .form-control-static{height:30px;min-height:32px;padding:6px 10px;font-size:12px;line-height:1.5}.input-lg{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-lg{height:46px;line-height:46px}select[multiple].input-lg,textarea.input-lg{height:auto}.form-group-lg .form-control{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.form-group-lg select.form-control{height:46px;line-height:46px}.form-group-lg select[multiple].form-control,.form-group-lg textarea.form-control{height:auto}.form-group-lg .form-control-static{height:46px;min-height:38px;padding:11px 16px;font-size:18px;line-height:1.3333333}.has-feedback{position:relative}.has-feedback .form-control{padding-right:42.5px}.form-control-feedback{position:absolute;top:0;right:0;z-index:2;display:block;width:34px;height:34px;line-height:34px;text-align:center;pointer-events:none}.form-group-lg .form-control+.form-control-feedback,.input-group-lg+.form-control-feedback,.input-lg+.form-control-feedback{width:46px;height:46px;line-height:46px}.form-group-sm .form-control+.form-control-feedback,.input-group-sm+.form-control-feedback,.input-sm+.form-control-feedback{width:30px;height:30px;line-height:30px}.has-success .checkbox,.has-success .checkbox-inline,.has-success .control-label,.has-success .help-block,.has-success .radio,.has-success .radio-inline,.has-success.checkbox label,.has-success.checkbox-inline label,.has-success.radio label,.has-success.radio-inline label{color:#3c763d}.has-success .form-control{border-color:#3c763d;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-success .form-control:focus{border-color:#2b542c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168}.has-success .input-group-addon{color:#3c763d;background-color:#dff0d8;border-color:#3c763d}.has-success .form-control-feedback{color:#3c763d}.has-warning .checkbox,.has-warning .checkbox-inline,.has-warning .control-label,.has-warning .help-block,.has-warning .radio,.has-warning .radio-inline,.has-warning.checkbox label,.has-warning.checkbox-inline label,.has-warning.radio label,.has-warning.radio-inline label{color:#8a6d3b}.has-warning .form-control{border-color:#8a6d3b;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-warning .form-control:focus{border-color:#66512c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b}.has-warning .input-group-addon{color:#8a6d3b;background-color:#fcf8e3;border-color:#8a6d3b}.has-warning .form-control-feedback{color:#8a6d3b}.has-error .checkbox,.has-error .checkbox-inline,.has-error .control-label,.has-error .help-block,.has-error .radio,.has-error .radio-inline,.has-error.checkbox label,.has-error.checkbox-inline label,.has-error.radio label,.has-error.radio-inline label{color:#a94442}.has-error .form-control{border-color:#a94442;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-error .form-control:focus{border-color:#843534;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483}.has-error .input-group-addon{color:#a94442;background-color:#f2dede;border-color:#a94442}.has-error .form-control-feedback{color:#a94442}.has-feedback label~.form-control-feedback{top:25px}.has-feedback label.sr-only~.form-control-feedback{top:0}.help-block{display:block;margin-top:5px;margin-bottom:10px;color:#737373}@media (min-width:768px){.form-inline .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.form-inline .form-control{display:inline-block;width:auto;vertical-align:middle}.form-inline .form-control-static{display:inline-block}.form-inline .input-group{display:inline-table;vertical-align:middle}.form-inline .input-group .form-control,.form-inline .input-group .input-group-addon,.form-inline .input-group .input-group-btn{width:auto}.form-inline .input-group>.form-control{width:100%}.form-inline .control-label{margin-bottom:0;vertical-align:middle}.form-inline .checkbox,.form-inline .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.form-inline .checkbox label,.form-inline .radio label{padding-left:0}.form-inline .checkbox input[type=checkbox],.form-inline .radio input[type=radio]{position:relative;margin-left:0}.form-inline .has-feedback .form-control-feedback{top:0}}.form-horizontal .checkbox,.form-horizontal .checkbox-inline,.form-horizontal .radio,.form-horizontal .radio-inline{padding-top:7px;margin-top:0;margin-bottom:0}.form-horizontal .checkbox,.form-horizontal .radio{min-height:27px}.form-horizontal .form-group{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.form-horizontal .control-label{padding-top:7px;margin-bottom:0;text-align:right}}.form-horizontal .has-feedback .form-control-feedback{right:15px}@media (min-width:768px){.form-horizontal .form-group-lg .control-label{padding-top:11px;font-size:18px}}@media (min-width:768px){.form-horizontal .form-group-sm .control-label{padding-top:6px;font-size:12px}}.btn{display:inline-block;padding:6px 12px;margin-bottom:0;font-size:14px;font-weight:400;line-height:1.42857143;text-align:center;white-space:nowrap;vertical-align:middle;-ms-touch-action:manipulation;touch-action:manipulation;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;background-image:none;border:1px solid transparent;border-radius:4px}.btn.active.focus,.btn.active:focus,.btn.focus,.btn:active.focus,.btn:active:focus,.btn:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}.btn.focus,.btn:focus,.btn:hover{color:#333;text-decoration:none}.btn.active,.btn:active{background-image:none;outline:0;-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn.disabled,.btn[disabled],fieldset[disabled] .btn{cursor:not-allowed;filter:alpha(opacity=65);-webkit-box-shadow:none;box-shadow:none;opacity:.65}a.btn.disabled,fieldset[disabled] a.btn{pointer-events:none}.btn-default{color:#333;background-color:#fff;border-color:#ccc}.btn-default.focus,.btn-default:focus{color:#333;background-color:#e6e6e6;border-color:#8c8c8c}.btn-default:hover{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active.focus,.btn-default.active:focus,.btn-default.active:hover,.btn-default:active.focus,.btn-default:active:focus,.btn-default:active:hover,.open>.dropdown-toggle.btn-default.focus,.open>.dropdown-toggle.btn-default:focus,.open>.dropdown-toggle.btn-default:hover{color:#333;background-color:#d4d4d4;border-color:#8c8c8c}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{background-image:none}.btn-default.disabled.focus,.btn-default.disabled:focus,.btn-default.disabled:hover,.btn-default[disabled].focus,.btn-default[disabled]:focus,.btn-default[disabled]:hover,fieldset[disabled] .btn-default.focus,fieldset[disabled] .btn-default:focus,fieldset[disabled] .btn-default:hover{background-color:#fff;border-color:#ccc}.btn-default .badge{color:#fff;background-color:#333}.btn-primary{color:#fff;background-color:#337ab7;border-color:#2e6da4}.btn-primary.focus,.btn-primary:focus{color:#fff;background-color:#286090;border-color:#122b40}.btn-primary:hover{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active.focus,.btn-primary.active:focus,.btn-primary.active:hover,.btn-primary:active.focus,.btn-primary:active:focus,.btn-primary:active:hover,.open>.dropdown-toggle.btn-primary.focus,.open>.dropdown-toggle.btn-primary:focus,.open>.dropdown-toggle.btn-primary:hover{color:#fff;background-color:#204d74;border-color:#122b40}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{background-image:none}.btn-primary.disabled.focus,.btn-primary.disabled:focus,.btn-primary.disabled:hover,.btn-primary[disabled].focus,.btn-primary[disabled]:focus,.btn-primary[disabled]:hover,fieldset[disabled] .btn-primary.focus,fieldset[disabled] .btn-primary:focus,fieldset[disabled] .btn-primary:hover{background-color:#337ab7;border-color:#2e6da4}.btn-primary .badge{color:#337ab7;background-color:#fff}.btn-success{color:#fff;background-color:#5cb85c;border-color:#4cae4c}.btn-success.focus,.btn-success:focus{color:#fff;background-color:#449d44;border-color:#255625}.btn-success:hover{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active.focus,.btn-success.active:focus,.btn-success.active:hover,.btn-success:active.focus,.btn-success:active:focus,.btn-success:active:hover,.open>.dropdown-toggle.btn-success.focus,.open>.dropdown-toggle.btn-success:focus,.open>.dropdown-toggle.btn-success:hover{color:#fff;background-color:#398439;border-color:#255625}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{background-image:none}.btn-success.disabled.focus,.btn-success.disabled:focus,.btn-success.disabled:hover,.btn-success[disabled].focus,.btn-success[disabled]:focus,.btn-success[disabled]:hover,fieldset[disabled] .btn-success.focus,fieldset[disabled] .btn-success:focus,fieldset[disabled] .btn-success:hover{background-color:#5cb85c;border-color:#4cae4c}.btn-success .badge{color:#5cb85c;background-color:#fff}.btn-info{color:#fff;background-color:#5bc0de;border-color:#46b8da}.btn-info.focus,.btn-info:focus{color:#fff;background-color:#31b0d5;border-color:#1b6d85}.btn-info:hover{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active.focus,.btn-info.active:focus,.btn-info.active:hover,.btn-info:active.focus,.btn-info:active:focus,.btn-info:active:hover,.open>.dropdown-toggle.btn-info.focus,.open>.dropdown-toggle.btn-info:focus,.open>.dropdown-toggle.btn-info:hover{color:#fff;background-color:#269abc;border-color:#1b6d85}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{background-image:none}.btn-info.disabled.focus,.btn-info.disabled:focus,.btn-info.disabled:hover,.btn-info[disabled].focus,.btn-info[disabled]:focus,.btn-info[disabled]:hover,fieldset[disabled] .btn-info.focus,fieldset[disabled] .btn-info:focus,fieldset[disabled] .btn-info:hover{background-color:#5bc0de;border-color:#46b8da}.btn-info .badge{color:#5bc0de;background-color:#fff}.btn-warning{color:#fff;background-color:#f0ad4e;border-color:#eea236}.btn-warning.focus,.btn-warning:focus{color:#fff;background-color:#ec971f;border-color:#985f0d}.btn-warning:hover{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active.focus,.btn-warning.active:focus,.btn-warning.active:hover,.btn-warning:active.focus,.btn-warning:active:focus,.btn-warning:active:hover,.open>.dropdown-toggle.btn-warning.focus,.open>.dropdown-toggle.btn-warning:focus,.open>.dropdown-toggle.btn-warning:hover{color:#fff;background-color:#d58512;border-color:#985f0d}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{background-image:none}.btn-warning.disabled.focus,.btn-warning.disabled:focus,.btn-warning.disabled:hover,.btn-warning[disabled].focus,.btn-warning[disabled]:focus,.btn-warning[disabled]:hover,fieldset[disabled] .btn-warning.focus,fieldset[disabled] .btn-warning:focus,fieldset[disabled] .btn-warning:hover{background-color:#f0ad4e;border-color:#eea236}.btn-warning .badge{color:#f0ad4e;background-color:#fff}.btn-danger{color:#fff;background-color:#d9534f;border-color:#d43f3a}.btn-danger.focus,.btn-danger:focus{color:#fff;background-color:#c9302c;border-color:#761c19}.btn-danger:hover{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active.focus,.btn-danger.active:focus,.btn-danger.active:hover,.btn-danger:active.focus,.btn-danger:active:focus,.btn-danger:active:hover,.open>.dropdown-toggle.btn-danger.focus,.open>.dropdown-toggle.btn-danger:focus,.open>.dropdown-toggle.btn-danger:hover{color:#fff;background-color:#ac2925;border-color:#761c19}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{background-image:none}.btn-danger.disabled.focus,.btn-danger.disabled:focus,.btn-danger.disabled:hover,.btn-danger[disabled].focus,.btn-danger[disabled]:focus,.btn-danger[disabled]:hover,fieldset[disabled] .btn-danger.focus,fieldset[disabled] .btn-danger:focus,fieldset[disabled] .btn-danger:hover{background-color:#d9534f;border-color:#d43f3a}.btn-danger .badge{color:#d9534f;background-color:#fff}.btn-link{font-weight:400;color:#337ab7;border-radius:0}.btn-link,.btn-link.active,.btn-link:active,.btn-link[disabled],fieldset[disabled] .btn-link{background-color:transparent;-webkit-box-shadow:none;box-shadow:none}.btn-link,.btn-link:active,.btn-link:focus,.btn-link:hover{border-color:transparent}.btn-link:focus,.btn-link:hover{color:#23527c;text-decoration:underline;background-color:transparent}.btn-link[disabled]:focus,.btn-link[disabled]:hover,fieldset[disabled] .btn-link:focus,fieldset[disabled] .btn-link:hover{color:#777;text-decoration:none}.btn-group-lg>.btn,.btn-lg{padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.btn-group-sm>.btn,.btn-sm{padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.btn-group-xs>.btn,.btn-xs{padding:1px 5px;font-size:12px;line-height:1.5;border-radius:3px}.btn-block{display:block;width:100%}.btn-block+.btn-block{margin-top:5px}input[type=button].btn-block,input[type=reset].btn-block,input[type=submit].btn-block{width:100%}.fade{opacity:0;-webkit-transition:opacity .15s linear;-o-transition:opacity .15s linear;transition:opacity .15s linear}.fade.in{opacity:1}.collapse{display:none}.collapse.in{display:block}tr.collapse.in{display:table-row}tbody.collapse.in{display:table-row-group}.collapsing{position:relative;height:0;overflow:hidden;-webkit-transition-timing-function:ease;-o-transition-timing-function:ease;transition-timing-function:ease;-webkit-transition-duration:.35s;-o-transition-duration:.35s;transition-duration:.35s;-webkit-transition-property:height,visibility;-o-transition-property:height,visibility;transition-property:height,visibility}.caret{display:inline-block;width:0;height:0;margin-left:2px;vertical-align:middle;border-top:4px dashed;border-top:4px solid\9;border-right:4px solid transparent;border-left:4px solid transparent}.dropdown,.dropup{position:relative}.dropdown-toggle:focus{outline:0}.dropdown-menu{position:absolute;top:100%;left:0;z-index:1000;display:none;float:left;min-width:160px;padding:5px 0;margin:2px 0 0;font-size:14px;text-align:left;list-style:none;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.15);border-radius:4px;-webkit-box-shadow:0 6px 12px rgba(0,0,0,.175);box-shadow:0 6px 12px rgba(0,0,0,.175)}.dropdown-menu.pull-right{right:0;left:auto}.dropdown-menu .divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.dropdown-menu>li>a{display:block;padding:3px 20px;clear:both;font-weight:400;line-height:1.42857143;color:#333;white-space:nowrap}.dropdown-menu>li>a:focus,.dropdown-menu>li>a:hover{color:#262626;text-decoration:none;background-color:#f5f5f5}.dropdown-menu>.active>a,.dropdown-menu>.active>a:focus,.dropdown-menu>.active>a:hover{color:#fff;text-decoration:none;background-color:#337ab7;outline:0}.dropdown-menu>.disabled>a,.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{color:#777}.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{text-decoration:none;cursor:not-allowed;background-color:transparent;background-image:none;filter:progid:DXImageTransform.Microsoft.gradient(enabled=false)}.open>.dropdown-menu{display:block}.open>a{outline:0}.dropdown-menu-right{right:0;left:auto}.dropdown-menu-left{right:auto;left:0}.dropdown-header{display:block;padding:3px 20px;font-size:12px;line-height:1.42857143;color:#777;white-space:nowrap}.dropdown-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:990}.pull-right>.dropdown-menu{right:0;left:auto}.dropup .caret,.navbar-fixed-bottom .dropdown .caret{content:"";border-top:0;border-bottom:4px dashed;border-bottom:4px solid\9}.dropup .dropdown-menu,.navbar-fixed-bottom .dropdown .dropdown-menu{top:auto;bottom:100%;margin-bottom:2px}@media (min-width:768px){.navbar-right .dropdown-menu{right:0;left:auto}.navbar-right .dropdown-menu-left{right:auto;left:0}}.btn-group,.btn-group-vertical{position:relative;display:inline-block;vertical-align:middle}.btn-group-vertical>.btn,.btn-group>.btn{position:relative;float:left}.btn-group-vertical>.btn.active,.btn-group-vertical>.btn:active,.btn-group-vertical>.btn:focus,.btn-group-vertical>.btn:hover,.btn-group>.btn.active,.btn-group>.btn:active,.btn-group>.btn:focus,.btn-group>.btn:hover{z-index:2}.btn-group .btn+.btn,.btn-group .btn+.btn-group,.btn-group .btn-group+.btn,.btn-group .btn-group+.btn-group{margin-left:-1px}.btn-toolbar{margin-left:-5px}.btn-toolbar .btn,.btn-toolbar .btn-group,.btn-toolbar .input-group{float:left}.btn-toolbar>.btn,.btn-toolbar>.btn-group,.btn-toolbar>.input-group{margin-left:5px}.btn-group>.btn:not(:first-child):not(:last-child):not(.dropdown-toggle){border-radius:0}.btn-group>.btn:first-child{margin-left:0}.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn:last-child:not(:first-child),.btn-group>.dropdown-toggle:not(:first-child){border-top-left-radius:0;border-bottom-left-radius:0}.btn-group>.btn-group{float:left}.btn-group>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-bottom-left-radius:0}.btn-group .dropdown-toggle:active,.btn-group.open .dropdown-toggle{outline:0}.btn-group>.btn+.dropdown-toggle{padding-right:8px;padding-left:8px}.btn-group>.btn-lg+.dropdown-toggle{padding-right:12px;padding-left:12px}.btn-group.open .dropdown-toggle{-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn-group.open .dropdown-toggle.btn-link{-webkit-box-shadow:none;box-shadow:none}.btn .caret{margin-left:0}.btn-lg .caret{border-width:5px 5px 0;border-bottom-width:0}.dropup .btn-lg .caret{border-width:0 5px 5px}.btn-group-vertical>.btn,.btn-group-vertical>.btn-group,.btn-group-vertical>.btn-group>.btn{display:block;float:none;width:100%;max-width:100%}.btn-group-vertical>.btn-group>.btn{float:none}.btn-group-vertical>.btn+.btn,.btn-group-vertical>.btn+.btn-group,.btn-group-vertical>.btn-group+.btn,.btn-group-vertical>.btn-group+.btn-group{margin-top:-1px;margin-left:0}.btn-group-vertical>.btn:not(:first-child):not(:last-child){border-radius:0}.btn-group-vertical>.btn:first-child:not(:last-child){border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn:last-child:not(:first-child){border-top-left-radius:0;border-top-right-radius:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}.btn-group-vertical>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group-vertical>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group-vertical>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-top-right-radius:0}.btn-group-justified{display:table;width:100%;table-layout:fixed;border-collapse:separate}.btn-group-justified>.btn,.btn-group-justified>.btn-group{display:table-cell;float:none;width:1%}.btn-group-justified>.btn-group .btn{width:100%}.btn-group-justified>.btn-group .dropdown-menu{left:auto}[data-toggle=buttons]>.btn input[type=checkbox],[data-toggle=buttons]>.btn input[type=radio],[data-toggle=buttons]>.btn-group>.btn input[type=checkbox],[data-toggle=buttons]>.btn-group>.btn input[type=radio]{position:absolute;clip:rect(0,0,0,0);pointer-events:none}.input-group{position:relative;display:table;border-collapse:separate}.input-group[class*=col-]{float:none;padding-right:0;padding-left:0}.input-group .form-control{position:relative;z-index:2;float:left;width:100%;margin-bottom:0}.input-group .form-control:focus{z-index:3}.input-group-lg>.form-control,.input-group-lg>.input-group-addon,.input-group-lg>.input-group-btn>.btn{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-group-lg>.form-control,select.input-group-lg>.input-group-addon,select.input-group-lg>.input-group-btn>.btn{height:46px;line-height:46px}select[multiple].input-group-lg>.form-control,select[multiple].input-group-lg>.input-group-addon,select[multiple].input-group-lg>.input-group-btn>.btn,textarea.input-group-lg>.form-control,textarea.input-group-lg>.input-group-addon,textarea.input-group-lg>.input-group-btn>.btn{height:auto}.input-group-sm>.form-control,.input-group-sm>.input-group-addon,.input-group-sm>.input-group-btn>.btn{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-group-sm>.form-control,select.input-group-sm>.input-group-addon,select.input-group-sm>.input-group-btn>.btn{height:30px;line-height:30px}select[multiple].input-group-sm>.form-control,select[multiple].input-group-sm>.input-group-addon,select[multiple].input-group-sm>.input-group-btn>.btn,textarea.input-group-sm>.form-control,textarea.input-group-sm>.input-group-addon,textarea.input-group-sm>.input-group-btn>.btn{height:auto}.input-group .form-control,.input-group-addon,.input-group-btn{display:table-cell}.input-group .form-control:not(:first-child):not(:last-child),.input-group-addon:not(:first-child):not(:last-child),.input-group-btn:not(:first-child):not(:last-child){border-radius:0}.input-group-addon,.input-group-btn{width:1%;white-space:nowrap;vertical-align:middle}.input-group-addon{padding:6px 12px;font-size:14px;font-weight:400;line-height:1;color:#555;text-align:center;background-color:#eee;border:1px solid #ccc;border-radius:4px}.input-group-addon.input-sm{padding:5px 10px;font-size:12px;border-radius:3px}.input-group-addon.input-lg{padding:10px 16px;font-size:18px;border-radius:6px}.input-group-addon input[type=checkbox],.input-group-addon input[type=radio]{margin-top:0}.input-group .form-control:first-child,.input-group-addon:first-child,.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group>.btn,.input-group-btn:first-child>.dropdown-toggle,.input-group-btn:last-child>.btn-group:not(:last-child)>.btn,.input-group-btn:last-child>.btn:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.input-group-addon:first-child{border-right:0}.input-group .form-control:last-child,.input-group-addon:last-child,.input-group-btn:first-child>.btn-group:not(:first-child)>.btn,.input-group-btn:first-child>.btn:not(:first-child),.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group>.btn,.input-group-btn:last-child>.dropdown-toggle{border-top-left-radius:0;border-bottom-left-radius:0}.input-group-addon:last-child{border-left:0}.input-group-btn{position:relative;font-size:0;white-space:nowrap}.input-group-btn>.btn{position:relative}.input-group-btn>.btn+.btn{margin-left:-1px}.input-group-btn>.btn:active,.input-group-btn>.btn:focus,.input-group-btn>.btn:hover{z-index:2}.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group{margin-right:-1px}.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group{z-index:2;margin-left:-1px}.nav{padding-left:0;margin-bottom:0;list-style:none}.nav>li{position:relative;display:block}.nav>li>a{position:relative;display:block;padding:10px 15px}.nav>li>a:focus,.nav>li>a:hover{text-decoration:none;background-color:#eee}.nav>li.disabled>a{color:#777}.nav>li.disabled>a:focus,.nav>li.disabled>a:hover{color:#777;text-decoration:none;cursor:not-allowed;background-color:transparent}.nav .open>a,.nav .open>a:focus,.nav .open>a:hover{background-color:#eee;border-color:#337ab7}.nav .nav-divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.nav>li>a>img{max-width:none}.nav-tabs{border-bottom:1px solid #ddd}.nav-tabs>li{float:left;margin-bottom:-1px}.nav-tabs>li>a{margin-right:2px;line-height:1.42857143;border:1px solid transparent;border-radius:4px 4px 0 0}.nav-tabs>li>a:hover{border-color:#eee #eee #ddd}.nav-tabs>li.active>a,.nav-tabs>li.active>a:focus,.nav-tabs>li.active>a:hover{color:#555;cursor:default;background-color:#fff;border:1px solid #ddd;border-bottom-color:transparent}.nav-tabs.nav-justified{width:100%;border-bottom:0}.nav-tabs.nav-justified>li{float:none}.nav-tabs.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-tabs.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-tabs.nav-justified>li{display:table-cell;width:1%}.nav-tabs.nav-justified>li>a{margin-bottom:0}}.nav-tabs.nav-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs.nav-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border-bottom-color:#fff}}.nav-pills>li{float:left}.nav-pills>li>a{border-radius:4px}.nav-pills>li+li{margin-left:2px}.nav-pills>li.active>a,.nav-pills>li.active>a:focus,.nav-pills>li.active>a:hover{color:#fff;background-color:#337ab7}.nav-stacked>li{float:none}.nav-stacked>li+li{margin-top:2px;margin-left:0}.nav-justified{width:100%}.nav-justified>li{float:none}.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-justified>li{display:table-cell;width:1%}.nav-justified>li>a{margin-bottom:0}}.nav-tabs-justified{border-bottom:0}.nav-tabs-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border-bottom-color:#fff}}.tab-content>.tab-pane{display:none}.tab-content>.active{display:block}.nav-tabs .dropdown-menu{margin-top:-1px;border-top-left-radius:0;border-top-right-radius:0}.navbar{position:relative;min-height:50px;margin-bottom:20px;border:1px solid transparent}@media (min-width:768px){.navbar{border-radius:4px}}@media (min-width:768px){.navbar-header{float:left}}.navbar-collapse{padding-right:15px;padding-left:15px;overflow-x:visible;-webkit-overflow-scrolling:touch;border-top:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1)}.navbar-collapse.in{overflow-y:auto}@media (min-width:768px){.navbar-collapse{width:auto;border-top:0;-webkit-box-shadow:none;box-shadow:none}.navbar-collapse.collapse{display:block!important;height:auto!important;padding-bottom:0;overflow:visible!important}.navbar-collapse.in{overflow-y:visible}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse,.navbar-static-top .navbar-collapse{padding-right:0;padding-left:0}}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:340px}@media (max-device-width:480px) and (orientation:landscape){.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:200px}}.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:0;margin-left:0}}.navbar-static-top{z-index:1000;border-width:0 0 1px}@media (min-width:768px){.navbar-static-top{border-radius:0}}.navbar-fixed-bottom,.navbar-fixed-top{position:fixed;right:0;left:0;z-index:1030}@media (min-width:768px){.navbar-fixed-bottom,.navbar-fixed-top{border-radius:0}}.navbar-fixed-top{top:0;border-width:0 0 1px}.navbar-fixed-bottom{bottom:0;margin-bottom:0;border-width:1px 0 0}.navbar-brand{float:left;height:50px;padding:15px 15px;font-size:18px;line-height:20px}.navbar-brand:focus,.navbar-brand:hover{text-decoration:none}.navbar-brand>img{display:block}@media (min-width:768px){.navbar>.container .navbar-brand,.navbar>.container-fluid .navbar-brand{margin-left:-15px}}.navbar-toggle{position:relative;float:right;padding:9px 10px;margin-top:8px;margin-right:15px;margin-bottom:8px;background-color:transparent;background-image:none;border:1px solid transparent;border-radius:4px}.navbar-toggle:focus{outline:0}.navbar-toggle .icon-bar{display:block;width:22px;height:2px;border-radius:1px}.navbar-toggle .icon-bar+.icon-bar{margin-top:4px}@media (min-width:768px){.navbar-toggle{display:none}}.navbar-nav{margin:7.5px -15px}.navbar-nav>li>a{padding-top:10px;padding-bottom:10px;line-height:20px}@media (max-width:767px){.navbar-nav .open .dropdown-menu{position:static;float:none;width:auto;margin-top:0;background-color:transparent;border:0;-webkit-box-shadow:none;box-shadow:none}.navbar-nav .open .dropdown-menu .dropdown-header,.navbar-nav .open .dropdown-menu>li>a{padding:5px 15px 5px 25px}.navbar-nav .open .dropdown-menu>li>a{line-height:20px}.navbar-nav .open .dropdown-menu>li>a:focus,.navbar-nav .open .dropdown-menu>li>a:hover{background-image:none}}@media (min-width:768px){.navbar-nav{float:left;margin:0}.navbar-nav>li{float:left}.navbar-nav>li>a{padding-top:15px;padding-bottom:15px}}.navbar-form{padding:10px 15px;margin-top:8px;margin-right:-15px;margin-bottom:8px;margin-left:-15px;border-top:1px solid transparent;border-bottom:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1)}@media (min-width:768px){.navbar-form .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.navbar-form .form-control{display:inline-block;width:auto;vertical-align:middle}.navbar-form .form-control-static{display:inline-block}.navbar-form .input-group{display:inline-table;vertical-align:middle}.navbar-form .input-group .form-control,.navbar-form .input-group .input-group-addon,.navbar-form .input-group .input-group-btn{width:auto}.navbar-form .input-group>.form-control{width:100%}.navbar-form .control-label{margin-bottom:0;vertical-align:middle}.navbar-form .checkbox,.navbar-form .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.navbar-form .checkbox label,.navbar-form .radio label{padding-left:0}.navbar-form .checkbox input[type=checkbox],.navbar-form .radio input[type=radio]{position:relative;margin-left:0}.navbar-form .has-feedback .form-control-feedback{top:0}}@media (max-width:767px){.navbar-form .form-group{margin-bottom:5px}.navbar-form .form-group:last-child{margin-bottom:0}}@media (min-width:768px){.navbar-form{width:auto;padding-top:0;padding-bottom:0;margin-right:0;margin-left:0;border:0;-webkit-box-shadow:none;box-shadow:none}}.navbar-nav>li>.dropdown-menu{margin-top:0;border-top-left-radius:0;border-top-right-radius:0}.navbar-fixed-bottom .navbar-nav>li>.dropdown-menu{margin-bottom:0;border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.navbar-btn{margin-top:8px;margin-bottom:8px}.navbar-btn.btn-sm{margin-top:10px;margin-bottom:10px}.navbar-btn.btn-xs{margin-top:14px;margin-bottom:14px}.navbar-text{margin-top:15px;margin-bottom:15px}@media (min-width:768px){.navbar-text{float:left;margin-right:15px;margin-left:15px}}@media (min-width:768px){.navbar-left{float:left!important}.navbar-right{float:right!important;margin-right:-15px}.navbar-right~.navbar-right{margin-right:0}}.navbar-default{background-color:#f8f8f8;border-color:#e7e7e7}.navbar-default .navbar-brand{color:#777}.navbar-default .navbar-brand:focus,.navbar-default .navbar-brand:hover{color:#5e5e5e;background-color:transparent}.navbar-default .navbar-text{color:#777}.navbar-default .navbar-nav>li>a{color:#777}.navbar-default .navbar-nav>li>a:focus,.navbar-default .navbar-nav>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav>.active>a,.navbar-default .navbar-nav>.active>a:focus,.navbar-default .navbar-nav>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav>.disabled>a,.navbar-default .navbar-nav>.disabled>a:focus,.navbar-default .navbar-nav>.disabled>a:hover{color:#ccc;background-color:transparent}.navbar-default .navbar-toggle{border-color:#ddd}.navbar-default .navbar-toggle:focus,.navbar-default .navbar-toggle:hover{background-color:#ddd}.navbar-default .navbar-toggle .icon-bar{background-color:#888}.navbar-default .navbar-collapse,.navbar-default .navbar-form{border-color:#e7e7e7}.navbar-default .navbar-nav>.open>a,.navbar-default .navbar-nav>.open>a:focus,.navbar-default .navbar-nav>.open>a:hover{color:#555;background-color:#e7e7e7}@media (max-width:767px){.navbar-default .navbar-nav .open .dropdown-menu>li>a{color:#777}.navbar-default .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav .open .dropdown-menu>.active>a,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#ccc;background-color:transparent}}.navbar-default .navbar-link{color:#777}.navbar-default .navbar-link:hover{color:#333}.navbar-default .btn-link{color:#777}.navbar-default .btn-link:focus,.navbar-default .btn-link:hover{color:#333}.navbar-default .btn-link[disabled]:focus,.navbar-default .btn-link[disabled]:hover,fieldset[disabled] .navbar-default .btn-link:focus,fieldset[disabled] .navbar-default .btn-link:hover{color:#ccc}.navbar-inverse{background-color:#222;border-color:#080808}.navbar-inverse .navbar-brand{color:#9d9d9d}.navbar-inverse .navbar-brand:focus,.navbar-inverse .navbar-brand:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-text{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a:focus,.navbar-inverse .navbar-nav>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav>.active>a,.navbar-inverse .navbar-nav>.active>a:focus,.navbar-inverse .navbar-nav>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav>.disabled>a,.navbar-inverse .navbar-nav>.disabled>a:focus,.navbar-inverse .navbar-nav>.disabled>a:hover{color:#444;background-color:transparent}.navbar-inverse .navbar-toggle{border-color:#333}.navbar-inverse .navbar-toggle:focus,.navbar-inverse .navbar-toggle:hover{background-color:#333}.navbar-inverse .navbar-toggle .icon-bar{background-color:#fff}.navbar-inverse .navbar-collapse,.navbar-inverse .navbar-form{border-color:#101010}.navbar-inverse .navbar-nav>.open>a,.navbar-inverse .navbar-nav>.open>a:focus,.navbar-inverse .navbar-nav>.open>a:hover{color:#fff;background-color:#080808}@media (max-width:767px){.navbar-inverse .navbar-nav .open .dropdown-menu>.dropdown-header{border-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu .divider{background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#444;background-color:transparent}}.navbar-inverse .navbar-link{color:#9d9d9d}.navbar-inverse .navbar-link:hover{color:#fff}.navbar-inverse .btn-link{color:#9d9d9d}.navbar-inverse .btn-link:focus,.navbar-inverse .btn-link:hover{color:#fff}.navbar-inverse .btn-link[disabled]:focus,.navbar-inverse .btn-link[disabled]:hover,fieldset[disabled] .navbar-inverse .btn-link:focus,fieldset[disabled] .navbar-inverse .btn-link:hover{color:#444}.breadcrumb{padding:8px 15px;margin-bottom:20px;list-style:none;background-color:#f5f5f5;border-radius:4px}.breadcrumb>li{display:inline-block}.breadcrumb>li+li:before{padding:0 5px;color:#ccc;content:"/\00a0"}.breadcrumb>.active{color:#777}.pagination{display:inline-block;padding-left:0;margin:20px 0;border-radius:4px}.pagination>li{display:inline}.pagination>li>a,.pagination>li>span{position:relative;float:left;padding:6px 12px;margin-left:-1px;line-height:1.42857143;color:#337ab7;text-decoration:none;background-color:#fff;border:1px solid #ddd}.pagination>li:first-child>a,.pagination>li:first-child>span{margin-left:0;border-top-left-radius:4px;border-bottom-left-radius:4px}.pagination>li:last-child>a,.pagination>li:last-child>span{border-top-right-radius:4px;border-bottom-right-radius:4px}.pagination>li>a:focus,.pagination>li>a:hover,.pagination>li>span:focus,.pagination>li>span:hover{z-index:2;color:#23527c;background-color:#eee;border-color:#ddd}.pagination>.active>a,.pagination>.active>a:focus,.pagination>.active>a:hover,.pagination>.active>span,.pagination>.active>span:focus,.pagination>.active>span:hover{z-index:3;color:#fff;cursor:default;background-color:#337ab7;border-color:#337ab7}.pagination>.disabled>a,.pagination>.disabled>a:focus,.pagination>.disabled>a:hover,.pagination>.disabled>span,.pagination>.disabled>span:focus,.pagination>.disabled>span:hover{color:#777;cursor:not-allowed;background-color:#fff;border-color:#ddd}.pagination-lg>li>a,.pagination-lg>li>span{padding:10px 16px;font-size:18px;line-height:1.3333333}.pagination-lg>li:first-child>a,.pagination-lg>li:first-child>span{border-top-left-radius:6px;border-bottom-left-radius:6px}.pagination-lg>li:last-child>a,.pagination-lg>li:last-child>span{border-top-right-radius:6px;border-bottom-right-radius:6px}.pagination-sm>li>a,.pagination-sm>li>span{padding:5px 10px;font-size:12px;line-height:1.5}.pagination-sm>li:first-child>a,.pagination-sm>li:first-child>span{border-top-left-radius:3px;border-bottom-left-radius:3px}.pagination-sm>li:last-child>a,.pagination-sm>li:last-child>span{border-top-right-radius:3px;border-bottom-right-radius:3px}.pager{padding-left:0;margin:20px 0;text-align:center;list-style:none}.pager li{display:inline}.pager li>a,.pager li>span{display:inline-block;padding:5px 14px;background-color:#fff;border:1px solid #ddd;border-radius:15px}.pager li>a:focus,.pager li>a:hover{text-decoration:none;background-color:#eee}.pager .next>a,.pager .next>span{float:right}.pager .previous>a,.pager .previous>span{float:left}.pager .disabled>a,.pager .disabled>a:focus,.pager .disabled>a:hover,.pager .disabled>span{color:#777;cursor:not-allowed;background-color:#fff}.label{display:inline;padding:.2em .6em .3em;font-size:75%;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:baseline;border-radius:.25em}a.label:focus,a.label:hover{color:#fff;text-decoration:none;cursor:pointer}.label:empty{display:none}.btn .label{position:relative;top:-1px}.label-default{background-color:#777}.label-default[href]:focus,.label-default[href]:hover{background-color:#5e5e5e}.label-primary{background-color:#337ab7}.label-primary[href]:focus,.label-primary[href]:hover{background-color:#286090}.label-success{background-color:#5cb85c}.label-success[href]:focus,.label-success[href]:hover{background-color:#449d44}.label-info{background-color:#5bc0de}.label-info[href]:focus,.label-info[href]:hover{background-color:#31b0d5}.label-warning{background-color:#f0ad4e}.label-warning[href]:focus,.label-warning[href]:hover{background-color:#ec971f}.label-danger{background-color:#d9534f}.label-danger[href]:focus,.label-danger[href]:hover{background-color:#c9302c}.badge{display:inline-block;min-width:10px;padding:3px 7px;font-size:12px;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:middle;background-color:#777;border-radius:10px}.badge:empty{display:none}.btn .badge{position:relative;top:-1px}.btn-group-xs>.btn .badge,.btn-xs .badge{top:0;padding:1px 5px}a.badge:focus,a.badge:hover{color:#fff;text-decoration:none;cursor:pointer}.list-group-item.active>.badge,.nav-pills>.active>a>.badge{color:#337ab7;background-color:#fff}.list-group-item>.badge{float:right}.list-group-item>.badge+.badge{margin-right:5px}.nav-pills>li>a>.badge{margin-left:3px}.jumbotron{padding-top:30px;padding-bottom:30px;margin-bottom:30px;color:inherit;background-color:#eee}.jumbotron .h1,.jumbotron h1{color:inherit}.jumbotron p{margin-bottom:15px;font-size:21px;font-weight:200}.jumbotron>hr{border-top-color:#d5d5d5}.container .jumbotron,.container-fluid .jumbotron{padding-right:15px;padding-left:15px;border-radius:6px}.jumbotron .container{max-width:100%}@media screen and (min-width:768px){.jumbotron{padding-top:48px;padding-bottom:48px}.container .jumbotron,.container-fluid .jumbotron{padding-right:60px;padding-left:60px}.jumbotron .h1,.jumbotron h1{font-size:63px}}.thumbnail{display:block;padding:4px;margin-bottom:20px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:border .2s ease-in-out;-o-transition:border .2s ease-in-out;transition:border .2s ease-in-out}.thumbnail a>img,.thumbnail>img{margin-right:auto;margin-left:auto}a.thumbnail.active,a.thumbnail:focus,a.thumbnail:hover{border-color:#337ab7}.thumbnail .caption{padding:9px;color:#333}.alert{padding:15px;margin-bottom:20px;border:1px solid transparent;border-radius:4px}.alert h4{margin-top:0;color:inherit}.alert .alert-link{font-weight:700}.alert>p,.alert>ul{margin-bottom:0}.alert>p+p{margin-top:5px}.alert-dismissable,.alert-dismissible{padding-right:35px}.alert-dismissable .close,.alert-dismissible .close{position:relative;top:-2px;right:-21px;color:inherit}.alert-success{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.alert-success hr{border-top-color:#c9e2b3}.alert-success .alert-link{color:#2b542c}.alert-info{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.alert-info hr{border-top-color:#a6e1ec}.alert-info .alert-link{color:#245269}.alert-warning{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.alert-warning hr{border-top-color:#f7e1b5}.alert-warning .alert-link{color:#66512c}.alert-danger{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.alert-danger hr{border-top-color:#e4b9c0}.alert-danger .alert-link{color:#843534}@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@-o-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}.progress{height:20px;margin-bottom:20px;overflow:hidden;background-color:#f5f5f5;border-radius:4px;-webkit-box-shadow:inset 0 1px 2px rgba(0,0,0,.1);box-shadow:inset 0 1px 2px rgba(0,0,0,.1)}.progress-bar{float:left;width:0;height:100%;font-size:12px;line-height:20px;color:#fff;text-align:center;background-color:#337ab7;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);-webkit-transition:width .6s ease;-o-transition:width .6s ease;transition:width .6s ease}.progress-bar-striped,.progress-striped .progress-bar{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);-webkit-background-size:40px 40px;background-size:40px 40px}.progress-bar.active,.progress.active .progress-bar{-webkit-animation:progress-bar-stripes 2s linear infinite;-o-animation:progress-bar-stripes 2s linear infinite;animation:progress-bar-stripes 2s linear infinite}.progress-bar-success{background-color:#5cb85c}.progress-striped .progress-bar-success{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-info{background-color:#5bc0de}.progress-striped .progress-bar-info{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-warning{background-color:#f0ad4e}.progress-striped .progress-bar-warning{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-danger{background-color:#d9534f}.progress-striped .progress-bar-danger{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.media{margin-top:15px}.media:first-child{margin-top:0}.media,.media-body{overflow:hidden;zoom:1}.media-body{width:10000px}.media-object{display:block}.media-object.img-thumbnail{max-width:none}.media-right,.media>.pull-right{padding-left:10px}.media-left,.media>.pull-left{padding-right:10px}.media-body,.media-left,.media-right{display:table-cell;vertical-align:top}.media-middle{vertical-align:middle}.media-bottom{vertical-align:bottom}.media-heading{margin-top:0;margin-bottom:5px}.media-list{padding-left:0;list-style:none}.list-group{padding-left:0;margin-bottom:20px}.list-group-item{position:relative;display:block;padding:10px 15px;margin-bottom:-1px;background-color:#fff;border:1px solid #ddd}.list-group-item:first-child{border-top-left-radius:4px;border-top-right-radius:4px}.list-group-item:last-child{margin-bottom:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}a.list-group-item,button.list-group-item{color:#555}a.list-group-item .list-group-item-heading,button.list-group-item .list-group-item-heading{color:#333}a.list-group-item:focus,a.list-group-item:hover,button.list-group-item:focus,button.list-group-item:hover{color:#555;text-decoration:none;background-color:#f5f5f5}button.list-group-item{width:100%;text-align:left}.list-group-item.disabled,.list-group-item.disabled:focus,.list-group-item.disabled:hover{color:#777;cursor:not-allowed;background-color:#eee}.list-group-item.disabled .list-group-item-heading,.list-group-item.disabled:focus .list-group-item-heading,.list-group-item.disabled:hover .list-group-item-heading{color:inherit}.list-group-item.disabled .list-group-item-text,.list-group-item.disabled:focus .list-group-item-text,.list-group-item.disabled:hover .list-group-item-text{color:#777}.list-group-item.active,.list-group-item.active:focus,.list-group-item.active:hover{z-index:2;color:#fff;background-color:#337ab7;border-color:#337ab7}.list-group-item.active .list-group-item-heading,.list-group-item.active .list-group-item-heading>.small,.list-group-item.active .list-group-item-heading>small,.list-group-item.active:focus .list-group-item-heading,.list-group-item.active:focus .list-group-item-heading>.small,.list-group-item.active:focus .list-group-item-heading>small,.list-group-item.active:hover .list-group-item-heading,.list-group-item.active:hover .list-group-item-heading>.small,.list-group-item.active:hover .list-group-item-heading>small{color:inherit}.list-group-item.active .list-group-item-text,.list-group-item.active:focus .list-group-item-text,.list-group-item.active:hover .list-group-item-text{color:#c7ddef}.list-group-item-success{color:#3c763d;background-color:#dff0d8}a.list-group-item-success,button.list-group-item-success{color:#3c763d}a.list-group-item-success .list-group-item-heading,button.list-group-item-success .list-group-item-heading{color:inherit}a.list-group-item-success:focus,a.list-group-item-success:hover,button.list-group-item-success:focus,button.list-group-item-success:hover{color:#3c763d;background-color:#d0e9c6}a.list-group-item-success.active,a.list-group-item-success.active:focus,a.list-group-item-success.active:hover,button.list-group-item-success.active,button.list-group-item-success.active:focus,button.list-group-item-success.active:hover{color:#fff;background-color:#3c763d;border-color:#3c763d}.list-group-item-info{color:#31708f;background-color:#d9edf7}a.list-group-item-info,button.list-group-item-info{color:#31708f}a.list-group-item-info .list-group-item-heading,button.list-group-item-info .list-group-item-heading{color:inherit}a.list-group-item-info:focus,a.list-group-item-info:hover,button.list-group-item-info:focus,button.list-group-item-info:hover{color:#31708f;background-color:#c4e3f3}a.list-group-item-info.active,a.list-group-item-info.active:focus,a.list-group-item-info.active:hover,button.list-group-item-info.active,button.list-group-item-info.active:focus,button.list-group-item-info.active:hover{color:#fff;background-color:#31708f;border-color:#31708f}.list-group-item-warning{color:#8a6d3b;background-color:#fcf8e3}a.list-group-item-warning,button.list-group-item-warning{color:#8a6d3b}a.list-group-item-warning .list-group-item-heading,button.list-group-item-warning .list-group-item-heading{color:inherit}a.list-group-item-warning:focus,a.list-group-item-warning:hover,button.list-group-item-warning:focus,button.list-group-item-warning:hover{color:#8a6d3b;background-color:#faf2cc}a.list-group-item-warning.active,a.list-group-item-warning.active:focus,a.list-group-item-warning.active:hover,button.list-group-item-warning.active,button.list-group-item-warning.active:focus,button.list-group-item-warning.active:hover{color:#fff;background-color:#8a6d3b;border-color:#8a6d3b}.list-group-item-danger{color:#a94442;background-color:#f2dede}a.list-group-item-danger,button.list-group-item-danger{color:#a94442}a.list-group-item-danger .list-group-item-heading,button.list-group-item-danger .list-group-item-heading{color:inherit}a.list-group-item-danger:focus,a.list-group-item-danger:hover,button.list-group-item-danger:focus,button.list-group-item-danger:hover{color:#a94442;background-color:#ebcccc}a.list-group-item-danger.active,a.list-group-item-danger.active:focus,a.list-group-item-danger.active:hover,button.list-group-item-danger.active,button.list-group-item-danger.active:focus,button.list-group-item-danger.active:hover{color:#fff;background-color:#a94442;border-color:#a94442}.list-group-item-heading{margin-top:0;margin-bottom:5px}.list-group-item-text{margin-bottom:0;line-height:1.3}.panel{margin-bottom:20px;background-color:#fff;border:1px solid transparent;border-radius:4px;-webkit-box-shadow:0 1px 1px rgba(0,0,0,.05);box-shadow:0 1px 1px rgba(0,0,0,.05)}.panel-body{padding:15px}.panel-heading{padding:10px 15px;border-bottom:1px solid transparent;border-top-left-radius:3px;border-top-right-radius:3px}.panel-heading>.dropdown .dropdown-toggle{color:inherit}.panel-title{margin-top:0;margin-bottom:0;font-size:16px;color:inherit}.panel-title>.small,.panel-title>.small>a,.panel-title>a,.panel-title>small,.panel-title>small>a{color:inherit}.panel-footer{padding:10px 15px;background-color:#f5f5f5;border-top:1px solid #ddd;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.list-group,.panel>.panel-collapse>.list-group{margin-bottom:0}.panel>.list-group .list-group-item,.panel>.panel-collapse>.list-group .list-group-item{border-width:1px 0;border-radius:0}.panel>.list-group:first-child .list-group-item:first-child,.panel>.panel-collapse>.list-group:first-child .list-group-item:first-child{border-top:0;border-top-left-radius:3px;border-top-right-radius:3px}.panel>.list-group:last-child .list-group-item:last-child,.panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{border-bottom:0;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.panel-heading+.panel-collapse>.list-group .list-group-item:first-child{border-top-left-radius:0;border-top-right-radius:0}.panel-heading+.list-group .list-group-item:first-child{border-top-width:0}.list-group+.panel-footer{border-top-width:0}.panel>.panel-collapse>.table,.panel>.table,.panel>.table-responsive>.table{margin-bottom:0}.panel>.panel-collapse>.table caption,.panel>.table caption,.panel>.table-responsive>.table caption{padding-right:15px;padding-left:15px}.panel>.table-responsive:first-child>.table:first-child,.panel>.table:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child,.panel>.table:first-child>thead:first-child>tr:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table:first-child>thead:first-child>tr:first-child th:first-child{border-top-left-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table:first-child>thead:first-child>tr:first-child th:last-child{border-top-right-radius:3px}.panel>.table-responsive:last-child>.table:last-child,.panel>.table:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:first-child{border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:last-child{border-bottom-right-radius:3px}.panel>.panel-body+.table,.panel>.panel-body+.table-responsive,.panel>.table+.panel-body,.panel>.table-responsive+.panel-body{border-top:1px solid #ddd}.panel>.table>tbody:first-child>tr:first-child td,.panel>.table>tbody:first-child>tr:first-child th{border-top:0}.panel>.table-bordered,.panel>.table-responsive>.table-bordered{border:0}.panel>.table-bordered>tbody>tr>td:first-child,.panel>.table-bordered>tbody>tr>th:first-child,.panel>.table-bordered>tfoot>tr>td:first-child,.panel>.table-bordered>tfoot>tr>th:first-child,.panel>.table-bordered>thead>tr>td:first-child,.panel>.table-bordered>thead>tr>th:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:first-child,.panel>.table-responsive>.table-bordered>thead>tr>td:first-child,.panel>.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.panel>.table-bordered>tbody>tr>td:last-child,.panel>.table-bordered>tbody>tr>th:last-child,.panel>.table-bordered>tfoot>tr>td:last-child,.panel>.table-bordered>tfoot>tr>th:last-child,.panel>.table-bordered>thead>tr>td:last-child,.panel>.table-bordered>thead>tr>th:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:last-child,.panel>.table-responsive>.table-bordered>thead>tr>td:last-child,.panel>.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.panel>.table-bordered>tbody>tr:first-child>td,.panel>.table-bordered>tbody>tr:first-child>th,.panel>.table-bordered>thead>tr:first-child>td,.panel>.table-bordered>thead>tr:first-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>th,.panel>.table-responsive>.table-bordered>thead>tr:first-child>td,.panel>.table-responsive>.table-bordered>thead>tr:first-child>th{border-bottom:0}.panel>.table-bordered>tbody>tr:last-child>td,.panel>.table-bordered>tbody>tr:last-child>th,.panel>.table-bordered>tfoot>tr:last-child>td,.panel>.table-bordered>tfoot>tr:last-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>th,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>td,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}.panel>.table-responsive{margin-bottom:0;border:0}.panel-group{margin-bottom:20px}.panel-group .panel{margin-bottom:0;border-radius:4px}.panel-group .panel+.panel{margin-top:5px}.panel-group .panel-heading{border-bottom:0}.panel-group .panel-heading+.panel-collapse>.list-group,.panel-group .panel-heading+.panel-collapse>.panel-body{border-top:1px solid #ddd}.panel-group .panel-footer{border-top:0}.panel-group .panel-footer+.panel-collapse .panel-body{border-bottom:1px solid #ddd}.panel-default{border-color:#ddd}.panel-default>.panel-heading{color:#333;background-color:#f5f5f5;border-color:#ddd}.panel-default>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ddd}.panel-default>.panel-heading .badge{color:#f5f5f5;background-color:#333}.panel-default>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ddd}.panel-primary{border-color:#337ab7}.panel-primary>.panel-heading{color:#fff;background-color:#337ab7;border-color:#337ab7}.panel-primary>.panel-heading+.panel-collapse>.panel-body{border-top-color:#337ab7}.panel-primary>.panel-heading .badge{color:#337ab7;background-color:#fff}.panel-primary>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#337ab7}.panel-success{border-color:#d6e9c6}.panel-success>.panel-heading{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.panel-success>.panel-heading+.panel-collapse>.panel-body{border-top-color:#d6e9c6}.panel-success>.panel-heading .badge{color:#dff0d8;background-color:#3c763d}.panel-success>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#d6e9c6}.panel-info{border-color:#bce8f1}.panel-info>.panel-heading{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.panel-info>.panel-heading+.panel-collapse>.panel-body{border-top-color:#bce8f1}.panel-info>.panel-heading .badge{color:#d9edf7;background-color:#31708f}.panel-info>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#bce8f1}.panel-warning{border-color:#faebcc}.panel-warning>.panel-heading{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.panel-warning>.panel-heading+.panel-collapse>.panel-body{border-top-color:#faebcc}.panel-warning>.panel-heading .badge{color:#fcf8e3;background-color:#8a6d3b}.panel-warning>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#faebcc}.panel-danger{border-color:#ebccd1}.panel-danger>.panel-heading{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.panel-danger>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ebccd1}.panel-danger>.panel-heading .badge{color:#f2dede;background-color:#a94442}.panel-danger>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ebccd1}.embed-responsive{position:relative;display:block;height:0;padding:0;overflow:hidden}.embed-responsive .embed-responsive-item,.embed-responsive embed,.embed-responsive iframe,.embed-responsive object,.embed-responsive video{position:absolute;top:0;bottom:0;left:0;width:100%;height:100%;border:0}.embed-responsive-16by9{padding-bottom:56.25%}.embed-responsive-4by3{padding-bottom:75%}.well{min-height:20px;padding:19px;margin-bottom:20px;background-color:#f5f5f5;border:1px solid #e3e3e3;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.05);box-shadow:inset 0 1px 1px rgba(0,0,0,.05)}.well blockquote{border-color:#ddd;border-color:rgba(0,0,0,.15)}.well-lg{padding:24px;border-radius:6px}.well-sm{padding:9px;border-radius:3px}.close{float:right;font-size:21px;font-weight:700;line-height:1;color:#000;text-shadow:0 1px 0 #fff;filter:alpha(opacity=20);opacity:.2}.close:focus,.close:hover{color:#000;text-decoration:none;cursor:pointer;filter:alpha(opacity=50);opacity:.5}button.close{-webkit-appearance:none;padding:0;cursor:pointer;background:0 0;border:0}.modal-open{overflow:hidden}.modal{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1050;display:none;overflow:hidden;-webkit-overflow-scrolling:touch;outline:0}.modal.fade .modal-dialog{-webkit-transition:-webkit-transform .3s ease-out;-o-transition:-o-transform .3s ease-out;transition:transform .3s ease-out;-webkit-transform:translate(0,-25%);-ms-transform:translate(0,-25%);-o-transform:translate(0,-25%);transform:translate(0,-25%)}.modal.in .modal-dialog{-webkit-transform:translate(0,0);-ms-transform:translate(0,0);-o-transform:translate(0,0);transform:translate(0,0)}.modal-open .modal{overflow-x:hidden;overflow-y:auto}.modal-dialog{position:relative;width:auto;margin:10px}.modal-content{position:relative;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #999;border:1px solid rgba(0,0,0,.2);border-radius:6px;outline:0;-webkit-box-shadow:0 3px 9px rgba(0,0,0,.5);box-shadow:0 3px 9px rgba(0,0,0,.5)}.modal-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1040;background-color:#000}.modal-backdrop.fade{filter:alpha(opacity=0);opacity:0}.modal-backdrop.in{filter:alpha(opacity=50);opacity:.5}.modal-header{padding:15px;border-bottom:1px solid #e5e5e5}.modal-header .close{margin-top:-2px}.modal-title{margin:0;line-height:1.42857143}.modal-body{position:relative;padding:15px}.modal-footer{padding:15px;text-align:right;border-top:1px solid #e5e5e5}.modal-footer .btn+.btn{margin-bottom:0;margin-left:5px}.modal-footer .btn-group .btn+.btn{margin-left:-1px}.modal-footer .btn-block+.btn-block{margin-left:0}.modal-scrollbar-measure{position:absolute;top:-9999px;width:50px;height:50px;overflow:scroll}@media (min-width:768px){.modal-dialog{width:600px;margin:30px auto}.modal-content{-webkit-box-shadow:0 5px 15px rgba(0,0,0,.5);box-shadow:0 5px 15px rgba(0,0,0,.5)}.modal-sm{width:300px}}@media (min-width:992px){.modal-lg{width:900px}}.tooltip{position:absolute;z-index:1070;display:block;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:12px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;filter:alpha(opacity=0);opacity:0;line-break:auto}.tooltip.in{filter:alpha(opacity=90);opacity:.9}.tooltip.top{padding:5px 0;margin-top:-3px}.tooltip.right{padding:0 5px;margin-left:3px}.tooltip.bottom{padding:5px 0;margin-top:3px}.tooltip.left{padding:0 5px;margin-left:-3px}.tooltip-inner{max-width:200px;padding:3px 8px;color:#fff;text-align:center;background-color:#000;border-radius:4px}.tooltip-arrow{position:absolute;width:0;height:0;border-color:transparent;border-style:solid}.tooltip.top .tooltip-arrow{bottom:0;left:50%;margin-left:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-left .tooltip-arrow{right:5px;bottom:0;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-right .tooltip-arrow{bottom:0;left:5px;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.right .tooltip-arrow{top:50%;left:0;margin-top:-5px;border-width:5px 5px 5px 0;border-right-color:#000}.tooltip.left .tooltip-arrow{top:50%;right:0;margin-top:-5px;border-width:5px 0 5px 5px;border-left-color:#000}.tooltip.bottom .tooltip-arrow{top:0;left:50%;margin-left:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-left .tooltip-arrow{top:0;right:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-right .tooltip-arrow{top:0;left:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.popover{position:absolute;top:0;left:0;z-index:1060;display:none;max-width:276px;padding:1px;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.2);border-radius:6px;-webkit-box-shadow:0 5px 10px rgba(0,0,0,.2);box-shadow:0 5px 10px rgba(0,0,0,.2);line-break:auto}.popover.top{margin-top:-10px}.popover.right{margin-left:10px}.popover.bottom{margin-top:10px}.popover.left{margin-left:-10px}.popover-title{padding:8px 14px;margin:0;font-size:14px;background-color:#f7f7f7;border-bottom:1px solid #ebebeb;border-radius:5px 5px 0 0}.popover-content{padding:9px 14px}.popover>.arrow,.popover>.arrow:after{position:absolute;display:block;width:0;height:0;border-color:transparent;border-style:solid}.popover>.arrow{border-width:11px}.popover>.arrow:after{content:"";border-width:10px}.popover.top>.arrow{bottom:-11px;left:50%;margin-left:-11px;border-top-color:#999;border-top-color:rgba(0,0,0,.25);border-bottom-width:0}.popover.top>.arrow:after{bottom:1px;margin-left:-10px;content:" ";border-top-color:#fff;border-bottom-width:0}.popover.right>.arrow{top:50%;left:-11px;margin-top:-11px;border-right-color:#999;border-right-color:rgba(0,0,0,.25);border-left-width:0}.popover.right>.arrow:after{bottom:-10px;left:1px;content:" ";border-right-color:#fff;border-left-width:0}.popover.bottom>.arrow{top:-11px;left:50%;margin-left:-11px;border-top-width:0;border-bottom-color:#999;border-bottom-color:rgba(0,0,0,.25)}.popover.bottom>.arrow:after{top:1px;margin-left:-10px;content:" ";border-top-width:0;border-bottom-color:#fff}.popover.left>.arrow{top:50%;right:-11px;margin-top:-11px;border-right-width:0;border-left-color:#999;border-left-color:rgba(0,0,0,.25)}.popover.left>.arrow:after{right:1px;bottom:-10px;content:" ";border-right-width:0;border-left-color:#fff}.carousel{position:relative}.carousel-inner{position:relative;width:100%;overflow:hidden}.carousel-inner>.item{position:relative;display:none;-webkit-transition:.6s ease-in-out left;-o-transition:.6s ease-in-out left;transition:.6s ease-in-out left}.carousel-inner>.item>a>img,.carousel-inner>.item>img{line-height:1}@media all and (transform-3d),(-webkit-transform-3d){.carousel-inner>.item{-webkit-transition:-webkit-transform .6s ease-in-out;-o-transition:-o-transform .6s ease-in-out;transition:transform .6s ease-in-out;-webkit-backface-visibility:hidden;backface-visibility:hidden;-webkit-perspective:1000px;perspective:1000px}.carousel-inner>.item.active.right,.carousel-inner>.item.next{left:0;-webkit-transform:translate3d(100%,0,0);transform:translate3d(100%,0,0)}.carousel-inner>.item.active.left,.carousel-inner>.item.prev{left:0;-webkit-transform:translate3d(-100%,0,0);transform:translate3d(-100%,0,0)}.carousel-inner>.item.active,.carousel-inner>.item.next.left,.carousel-inner>.item.prev.right{left:0;-webkit-transform:translate3d(0,0,0);transform:translate3d(0,0,0)}}.carousel-inner>.active,.carousel-inner>.next,.carousel-inner>.prev{display:block}.carousel-inner>.active{left:0}.carousel-inner>.next,.carousel-inner>.prev{position:absolute;top:0;width:100%}.carousel-inner>.next{left:100%}.carousel-inner>.prev{left:-100%}.carousel-inner>.next.left,.carousel-inner>.prev.right{left:0}.carousel-inner>.active.left{left:-100%}.carousel-inner>.active.right{left:100%}.carousel-control{position:absolute;top:0;bottom:0;left:0;width:15%;font-size:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6);background-color:rgba(0,0,0,0);filter:alpha(opacity=50);opacity:.5}.carousel-control.left{background-image:-webkit-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.5)),to(rgba(0,0,0,.0001)));background-image:linear-gradient(to right,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#80000000', endColorstr='#00000000', GradientType=1);background-repeat:repeat-x}.carousel-control.right{right:0;left:auto;background-image:-webkit-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.0001)),to(rgba(0,0,0,.5)));background-image:linear-gradient(to right,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#00000000', endColorstr='#80000000', GradientType=1);background-repeat:repeat-x}.carousel-control:focus,.carousel-control:hover{color:#fff;text-decoration:none;filter:alpha(opacity=90);outline:0;opacity:.9}.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{position:absolute;top:50%;z-index:5;display:inline-block;margin-top:-10px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{left:50%;margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{right:50%;margin-right:-10px}.carousel-control .icon-next,.carousel-control .icon-prev{width:20px;height:20px;font-family:serif;line-height:1}.carousel-control .icon-prev:before{content:'\2039'}.carousel-control .icon-next:before{content:'\203a'}.carousel-indicators{position:absolute;bottom:10px;left:50%;z-index:15;width:60%;padding-left:0;margin-left:-30%;text-align:center;list-style:none}.carousel-indicators li{display:inline-block;width:10px;height:10px;margin:1px;text-indent:-999px;cursor:pointer;background-color:#000\9;background-color:rgba(0,0,0,0);border:1px solid #fff;border-radius:10px}.carousel-indicators .active{width:12px;height:12px;margin:0;background-color:#fff}.carousel-caption{position:absolute;right:15%;bottom:20px;left:15%;z-index:10;padding-top:20px;padding-bottom:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6)}.carousel-caption .btn{text-shadow:none}@media screen and (min-width:768px){.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{width:30px;height:30px;margin-top:-10px;font-size:30px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{margin-right:-10px}.carousel-caption{right:20%;left:20%;padding-bottom:30px}.carousel-indicators{bottom:20px}}.btn-group-vertical>.btn-group:after,.btn-group-vertical>.btn-group:before,.btn-toolbar:after,.btn-toolbar:before,.clearfix:after,.clearfix:before,.container-fluid:after,.container-fluid:before,.container:after,.container:before,.dl-horizontal dd:after,.dl-horizontal dd:before,.form-horizontal .form-group:after,.form-horizontal .form-group:before,.modal-footer:after,.modal-footer:before,.modal-header:after,.modal-header:before,.nav:after,.nav:before,.navbar-collapse:after,.navbar-collapse:before,.navbar-header:after,.navbar-header:before,.navbar:after,.navbar:before,.pager:after,.pager:before,.panel-body:after,.panel-body:before,.row:after,.row:before{display:table;content:" "}.btn-group-vertical>.btn-group:after,.btn-toolbar:after,.clearfix:after,.container-fluid:after,.container:after,.dl-horizontal dd:after,.form-horizontal .form-group:after,.modal-footer:after,.modal-header:after,.nav:after,.navbar-collapse:after,.navbar-header:after,.navbar:after,.pager:after,.panel-body:after,.row:after{clear:both}.center-block{display:block;margin-right:auto;margin-left:auto}.pull-right{float:right!important}.pull-left{float:left!important}.hide{display:none!important}.show{display:block!important}.invisible{visibility:hidden}.text-hide{font:0/0 a;color:transparent;text-shadow:none;background-color:transparent;border:0}.hidden{display:none!important}.affix{position:fixed}@-ms-viewport{width:device-width}.visible-lg,.visible-md,.visible-sm,.visible-xs{display:none!important}.visible-lg-block,.visible-lg-inline,.visible-lg-inline-block,.visible-md-block,.visible-md-inline,.visible-md-inline-block,.visible-sm-block,.visible-sm-inline,.visible-sm-inline-block,.visible-xs-block,.visible-xs-inline,.visible-xs-inline-block{display:none!important}@media (max-width:767px){.visible-xs{display:block!important}table.visible-xs{display:table!important}tr.visible-xs{display:table-row!important}td.visible-xs,th.visible-xs{display:table-cell!important}}@media (max-width:767px){.visible-xs-block{display:block!important}}@media (max-width:767px){.visible-xs-inline{display:inline!important}}@media (max-width:767px){.visible-xs-inline-block{display:inline-block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm{display:block!important}table.visible-sm{display:table!important}tr.visible-sm{display:table-row!important}td.visible-sm,th.visible-sm{display:table-cell!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-block{display:block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline{display:inline!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline-block{display:inline-block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md{display:block!important}table.visible-md{display:table!important}tr.visible-md{display:table-row!important}td.visible-md,th.visible-md{display:table-cell!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-block{display:block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline{display:inline!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline-block{display:inline-block!important}}@media (min-width:1200px){.visible-lg{display:block!important}table.visible-lg{display:table!important}tr.visible-lg{display:table-row!important}td.visible-lg,th.visible-lg{display:table-cell!important}}@media (min-width:1200px){.visible-lg-block{display:block!important}}@media (min-width:1200px){.visible-lg-inline{display:inline!important}}@media (min-width:1200px){.visible-lg-inline-block{display:inline-block!important}}@media (max-width:767px){.hidden-xs{display:none!important}}@media (min-width:768px) and (max-width:991px){.hidden-sm{display:none!important}}@media (min-width:992px) and (max-width:1199px){.hidden-md{display:none!important}}@media (min-width:1200px){.hidden-lg{display:none!important}}.visible-print{display:none!important}@media print{.visible-print{display:block!important}table.visible-print{display:table!important}tr.visible-print{display:table-row!important}td.visible-print,th.visible-print{display:table-cell!important}}.visible-print-block{display:none!important}@media print{.visible-print-block{display:block!important}}.visible-print-inline{display:none!important}@media print{.visible-print-inline{display:inline!important}}.visible-print-inline-block{display:none!important}@media print{.visible-print-inline-block{display:inline-block!important}}@media print{.hidden-print{display:none!important}}
        /*# sourceMappingURL=bootstrap.min.css.map */
    </style>
    <!--<link rel="stylesheet" type="text/css" media="all" href="../../resources/bootstrap-3.2.0/css/bootstrap.min.css" th:href="@{resources/bootstrap-3.2.0/css/bootstrap.min.css}"/>-->
</head>
<body>
<div id="content" class="container">
    <h2>
        Exomiser Analysis Results for
        proband
    </h2>
    <ul class="nav nav-pills" id="tabs">
        <li><a href="#settings" role="tab" data-toggle="tab"><b>Analysis Settings</b></a></li>
        <li><a href="#filtering" role="tab" data-toggle="tab"><b>Filtering Summary</b></a></li>
        <li><a href="#distribution" role="tab" data-toggle="tab"><b>Variant Type Distribution</b></a></li>
        <li><a href="#prioritisation" role="tab" data-toggle="tab"><b>Prioritised Genes</b></a></li>
        
        <li><a href="#about" role="tab" data-toggle="tab"><b>About</b></a></li>
    </ul>
    <div class="panel panel-default" id="settings">
        <div class="panel-heading">
            <h3>Analysis Settings</h3>
        </div>
        <div class="panel-body"><p>Settings used for this analysis:</p>
            <pre>---
vcf: &quot;/mnt/scratch/Public/TRAINING/GenomeAnalysisModule/CaseInformation/CaseFiles/Case9/Case9_Merged.vcf.gz&quot;
genomeAssembly: &quot;HG38&quot;
pedigree:
  individuals:
  - familyId: &quot;FAM&quot;
    id: &quot;proband&quot;
    motherId: &quot;mother&quot;
    fatherId: &quot;father&quot;
    sex: &quot;FEMALE&quot;
    status: &quot;AFFECTED&quot;
  - familyId: &quot;FAM&quot;
    id: &quot;mother&quot;
    motherId: &quot;&quot;
    fatherId: &quot;&quot;
    sex: &quot;FEMALE&quot;
    status: &quot;UNAFFECTED&quot;
  - familyId: &quot;FAM&quot;
    id: &quot;father&quot;
    motherId: &quot;&quot;
    fatherId: &quot;&quot;
    sex: &quot;MALE&quot;
    status: &quot;AFFECTED&quot;
proband: &quot;proband&quot;
hpoIds:
- &quot;HP:0001519&quot;
- &quot;HP:0011800&quot;
- &quot;HP:0008625&quot;
- &quot;HP:0000545&quot;
- &quot;HP:0000175&quot;
- &quot;HP:0000218&quot;
inheritanceModes:
  maxFreq: 2.0
  empty: false
  definedModes:
  - &quot;AUTOSOMAL_DOMINANT&quot;
  - &quot;AUTOSOMAL_RECESSIVE&quot;
  - &quot;X_RECESSIVE&quot;
  - &quot;X_DOMINANT&quot;
  - &quot;MITOCHONDRIAL&quot;
  definedSubModes:
  - &quot;AUTOSOMAL_DOMINANT&quot;
  - &quot;AUTOSOMAL_RECESSIVE_COMP_HET&quot;
  - &quot;AUTOSOMAL_RECESSIVE_HOM_ALT&quot;
  - &quot;X_RECESSIVE_COMP_HET&quot;
  - &quot;X_RECESSIVE_HOM_ALT&quot;
  - &quot;X_DOMINANT&quot;
  - &quot;MITOCHONDRIAL&quot;
analysisMode: &quot;PASS_ONLY&quot;
frequencySources:
- &quot;THOUSAND_GENOMES&quot;
- &quot;TOPMED&quot;
- &quot;UK10K&quot;
- &quot;ESP_AFRICAN_AMERICAN&quot;
- &quot;ESP_EUROPEAN_AMERICAN&quot;
- &quot;ESP_ALL&quot;
- &quot;EXAC_AFRICAN_INC_AFRICAN_AMERICAN&quot;
- &quot;EXAC_AMERICAN&quot;
- &quot;EXAC_EAST_ASIAN&quot;
- &quot;EXAC_FINNISH&quot;
- &quot;EXAC_NON_FINNISH_EUROPEAN&quot;
- &quot;EXAC_OTHER&quot;
- &quot;EXAC_SOUTH_ASIAN&quot;
- &quot;GNOMAD_E_AFR&quot;
- &quot;GNOMAD_E_AMR&quot;
- &quot;GNOMAD_E_ASJ&quot;
- &quot;GNOMAD_E_EAS&quot;
- &quot;GNOMAD_E_FIN&quot;
- &quot;GNOMAD_E_NFE&quot;
- &quot;GNOMAD_E_OTH&quot;
- &quot;GNOMAD_E_SAS&quot;
- &quot;GNOMAD_G_AFR&quot;
- &quot;GNOMAD_G_AMR&quot;
- &quot;GNOMAD_G_ASJ&quot;
- &quot;GNOMAD_G_EAS&quot;
- &quot;GNOMAD_G_FIN&quot;
- &quot;GNOMAD_G_NFE&quot;
- &quot;GNOMAD_G_OTH&quot;
- &quot;GNOMAD_G_SAS&quot;
pathogenicitySources:
- &quot;POLYPHEN&quot;
- &quot;MUTATION_TASTER&quot;
- &quot;SIFT&quot;
- &quot;CADD&quot;
analysisSteps:
- offTargetVariantTypes:
  - &quot;CODING_TRANSCRIPT_INTRON_VARIANT&quot;
  - &quot;FIVE_PRIME_UTR_EXON_VARIANT&quot;
  - &quot;THREE_PRIME_UTR_EXON_VARIANT&quot;
  - &quot;FIVE_PRIME_UTR_INTRON_VARIANT&quot;
  - &quot;THREE_PRIME_UTR_INTRON_VARIANT&quot;
  - &quot;NON_CODING_TRANSCRIPT_EXON_VARIANT&quot;
  - &quot;NON_CODING_TRANSCRIPT_INTRON_VARIANT&quot;
  - &quot;UPSTREAM_GENE_VARIANT&quot;
  - &quot;DOWNSTREAM_GENE_VARIANT&quot;
  - &quot;INTERGENIC_VARIANT&quot;
  - &quot;REGULATORY_REGION_VARIANT&quot;
- decoratedFilter:
    maxFreq: 2.0
- decoratedFilter:
    keepNonPathogenic: true
- compatibleModes:
  - &quot;AUTOSOMAL_DOMINANT&quot;
  - &quot;AUTOSOMAL_RECESSIVE&quot;
  - &quot;X_RECESSIVE&quot;
  - &quot;X_DOMINANT&quot;
  - &quot;MITOCHONDRIAL&quot;
- priorityType: &quot;OMIM_PRIORITY&quot;
- priorityType: &quot;HIPHIVE_PRIORITY&quot;
---
outputContributingVariantsOnly: false
outputPrefix: &quot;Case9_exomiser&quot;
outputFormats:
- &quot;HTML&quot;
- &quot;VCF&quot;
- &quot;TSV_GENE&quot;
- &quot;TSV_VARIANT&quot;
- &quot;JSON&quot;
numGenes: 0
</pre>
        </div>
    </div>

    <div class="panel panel-default" id="filtering">
        <div class="panel-heading">
            <h3>Filtering Summary</h3>
        </div>
        <div class="panel-body">
            <table class="table">
                <thead>
                <tr>
                    <th>Filter</th>
                    <th>Report</th>
                    <th>Passed filter</th>
                    <th>Failed filter</th>
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td><b>Variant effect</b></td>
                    <td>
                        <ul>Removed variants with effects of type: [CODING_TRANSCRIPT_INTRON_VARIANT, FIVE_PRIME_UTR_EXON_VARIANT, THREE_PRIME_UTR_EXON_VARIANT, FIVE_PRIME_UTR_INTRON_VARIANT, THREE_PRIME_UTR_INTRON_VARIANT, NON_CODING_TRANSCRIPT_EXON_VARIANT, NON_CODING_TRANSCRIPT_INTRON_VARIANT, UPSTREAM_GENE_VARIANT, DOWNSTREAM_GENE_VARIANT, INTERGENIC_VARIANT, REGULATORY_REGION_VARIANT]</ul>
                    </td>
                    <td>225</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Frequency</b></td>
                    <td>
                        <ul>Variants filtered for maximum allele frequency of 2.00%</ul>
                    </td>
                    <td>225</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Pathogenicity</b></td>
                    <td>
                        <ul>Retained all non-pathogenic variants of all types. Scoring was applied, but the filter passed all variants.</ul>
                    </td>
                    <td>225</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Inheritance</b></td>
                    <td>
                        <ul>Genes filtered for compatibility with AUTOSOMAL_DOMINANT, AUTOSOMAL_RECESSIVE, X_RECESSIVE, X_DOMINANT, MITOCHONDRIAL inheritance.</ul>
                    </td>
                    <td>161</td>
                    <td>0</td>
                </tr>
                </tbody>
            </table>
        </div>
    </div>
    <div class="panel panel-default" id="distribution">
        <div class="panel-heading">
            <h3>Variant Type Distribution</h3>
        </div>
        <div class="panel-body">
            <table class="table-striped table-condensed">
                <thead>
                <tr>
                    <th>Variant Type</th>
                    
                        <th>proband</th>
                    
                        <th>mother</th>
                    
                        <th>father</th>
                    
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td>FRAMESHIFT_ELONGATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_TRUNCATION</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_VARIANT</td>
                    
                        <td>2</td>
                    
                        <td>0</td>
                    
                        <td>2</td>
                    
                </tr>
                <tr>
                    <td>INTERNAL_FEATURE_ELONGATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FEATURE_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>MNV</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>STOP_GAINED</td>
                    
                        <td>3</td>
                    
                        <td>0</td>
                    
                        <td>3</td>
                    
                </tr>
                <tr>
                    <td>STOP_LOST</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>START_LOST</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_ACCEPTOR_VARIANT</td>
                    
                        <td>2</td>
                    
                        <td>0</td>
                    
                        <td>2</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_DONOR_VARIANT</td>
                    
                        <td>4</td>
                    
                        <td>0</td>
                    
                        <td>4</td>
                    
                </tr>
                <tr>
                    <td>MISSENSE_VARIANT</td>
                    
                        <td>129</td>
                    
                        <td>7</td>
                    
                        <td>129</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_INSERTION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_INSERTION</td>
                    
                        <td>2</td>
                    
                        <td>0</td>
                    
                        <td>2</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_DELETION</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_DELETION</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_REGION_VARIANT</td>
                    
                        <td>19</td>
                    
                        <td>2</td>
                    
                        <td>19</td>
                    
                </tr>
                <tr>
                    <td>STOP_RETAINED_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INITIATOR_CODON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SYNONYMOUS_VARIANT</td>
                    
                        <td>61</td>
                    
                        <td>1</td>
                    
                        <td>61</td>
                    
                </tr>
                <tr>
                    <td>CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>UPSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DOWNSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INTERGENIC_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>REGULATORY_REGION_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                </tbody>
            </table>
        </div>
    </div>

    

    <div class="panel panel-default" id="prioritisation">
        <div class="panel-heading">
            <h3>Prioritised Genes</h3>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000139219">COL2A1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.990</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.839</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.839 to <a href="http://www.omim.org/108300">Stickler syndrome, type I</a> associated with COL2A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001519, Disproportionate tall stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000175, Cleft palate</dd></dl><dl><dt>Phenotypic similarity 0.791 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=COL2A1">COL2A1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0012085, midface hypoplasia</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0005163, cyclopia</dd><dd>HP:0000175, Cleft palate - MP:0012085, midface hypoplasia</dd><dd>HP:0000218, High palate - MP:0012085, midface hypoplasia</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/108300">OMIM:108300</a> Stickler syndrome, type I - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/132450">OMIM:132450</a> Epiphyseal dysplasia, multiple, with myopia and deafness - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/150600">OMIM:150600</a> Legg-Calve-Perthes disease - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/151210">OMIM:151210</a> Platyspondylic skeletal dysplasia, Torrance type - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/156550">OMIM:156550</a> Kniest dysplasia - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/183900">OMIM:183900</a> SED congenita - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/184250">OMIM:184250</a> SMED Strudwick type - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/200610">OMIM:200610</a> Achondrogenesis, type II or hypochondrogenesis - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/271700">OMIM:271700</a> Spondyloperipheral dysplasia - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/604864">OMIM:604864</a> Osteoarthritis with mild chondrodysplasia - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/608805">OMIM:608805</a> Avascular necrosis of the femoral head - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/609162">OMIM:609162</a> Czech dysplasia - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/609508">OMIM:609508</a> Stickler sydrome, type I, nonsyndromic ocular - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/616583">OMIM:616583</a> Spondyloepiphyseal dysplasia, Stanescu type - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=166011">ORPHA:166011</a> Multiple epiphyseal dysplasia, Beighton type</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=166100">ORPHA:166100</a> Autosomal dominant otospondylomegaepiphyseal dysplasia</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1856">ORPHA:1856</a> Spondyloperipheral dysplasia-short ulna syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2380">ORPHA:2380</a> Legg-Calvé-Perthes disease</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=485">ORPHA:485</a> Kniest dysplasia</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=85166">ORPHA:85166</a> Platyspondylic dysplasia, Torrance type</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=86820">ORPHA:86820</a> Familial avascular necrosis of femoral head</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=90653">ORPHA:90653</a> Stickler syndrome type 1</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=90653">ORPHA:90653</a> Stickler syndrome type 1</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93296">ORPHA:93296</a> Achondrogenesis type 2</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93315">ORPHA:93315</a> Spondylometaphyseal dysplasia, 'corner fracture' type</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93316">ORPHA:93316</a> Spondylometaphyseal dysplasia, Schmidt type</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93346">ORPHA:93346</a> Spondyloepimetaphyseal dysplasia congenita, Strudwick type</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=94068">ORPHA:94068</a> Spondyloepiphyseal dysplasia congenita</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.990</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.839</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr12:g.47982940G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1555166555">rs1555166555</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1555166555&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-danger">PATHOGENIC</span>
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=445007%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                <span class="label label-danger">WHITELIST VARIANT</span>
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000337299.7">COL2A1:ENST00000337299.7:c.1894C&gt;T:p.(Arg632*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380518.8">COL2A1:ENST00000380518.8:c.2101C&gt;T:p.(Arg701*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 1.000 (44.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000133392">MYH11</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.926</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.643</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.995</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.643 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=91387">Familial thoracic aortic aneurysm and aortic dissection</a> associated with MYH11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0000098, Tall stature</dd><dd>HP:0011800, Midface retrusion - HP:0002705, High, narrow palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0002705, High, narrow palate</dd><dd>HP:0000218, High palate - HP:0002705, High, narrow palate</dd></dl><dl><dt>Phenotypic similarity 0.269 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MYH11">MYH11</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000462, abnormal digestive system morphology</dd><dd>HP:0000218, High palate - MP:0000462, abnormal digestive system morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MYH11%0DTPM3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TPM3</a> and phenotypic similarity 0.620 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=171439">Childhood-onset nemaline myopathy</a> associated with TPM3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001533, Slender build</dd><dd>HP:0011800, Midface retrusion - HP:0000275, Narrow face</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/132900">OMIM:132900</a> Aortic aneurysm, familial thoracic 4 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2241">ORPHA:2241</a> Megacystis-microcolon-intestinal hypoperistalsis syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=229">ORPHA:229</a> Familial aortic dissection</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=91387">ORPHA:91387</a> Familial thoracic aortic aneurysm and aortic dissection</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.926</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.643</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.995</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.15753412C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=566821376">rs566821376</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs566821376&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.995</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000300036.6">MYH11:ENST00000300036.6:c.1846G&gt;A:p.(Ala616Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396324.7">MYH11:ENST00000396324.7:c.1867G&gt;A:p.(Ala623Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000452625.7">MYH11:ENST00000452625.7:c.1867G&gt;A:p.(Ala623Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000576790.7">MYH11:ENST00000576790.7:c.1846G&gt;A:p.(Ala616Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9973085</dd>
                                    
                                    <dd>Polyphen2: 0.463 (P)</dd>
                                    <dd>Mutation Taster: 0.992 (P)</dd>
                                    <dd>SIFT: 0.007 (D)</dd>
                                    <dd>CADD: 0.997 (25.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196218">RYR1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.900</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.614</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.555 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98905">Congenital multicore myopathy with external ophthalmoplegia</a> associated with RYR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0000275, Narrow face</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Phenotypic similarity 0.614 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RYR1">RYR1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000440, domed cranium</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0009890, cleft secondary palate</dd><dd>HP:0000218, High palate - MP:0009890, cleft secondary palate</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RYR1%0DASPH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ASPH</a> and phenotypic similarity 0.281 to <a href="http://www.omim.org/601552">Traboulsi syndrome</a> associated with ASPH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0000689, Dental malocclusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000689, Dental malocclusion</dd><dd>HP:0000218, High palate - HP:0000689, Dental malocclusion</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RYR1%0DASPH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ASPH</a> and phenotypic similarity 0.621 to mouse mutant of ASPH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000445, short snout</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/117000">OMIM:117000</a> Central core disease - autosomal dominant/recessive</dd>
<dd><a href="http://www.omim.org/entry/145600">OMIM:145600</a> King-Denborough syndrome - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/145600">OMIM:145600</a> Malignant hyperthermia susceptibility 1 (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/255320">OMIM:255320</a> Minicore myopathy with external ophthalmoplegia - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=169186">ORPHA:169186</a> Autosomal recessive centronuclear myopathy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=169189">ORPHA:169189</a> Autosomal dominant centronuclear myopathy</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=178145">ORPHA:178145</a> Moderate multiminicore disease with hand involvement</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=423">ORPHA:423</a> Malignant hyperthermia of anesthesia</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=597">ORPHA:597</a> Central core disease</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98905">ORPHA:98905</a> Congenital multicore myopathy with external ophthalmoplegia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.900</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.614</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.38517416G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=553187437">rs553187437</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs553187437&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=847631%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355481.8">RYR1:ENST00000355481.8:c.9743G&gt;A:p.(Arg3248Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359596.8">RYR1:ENST00000359596.8:c.9743G&gt;A:p.(Arg3248Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9943766</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.588 (T)</dd>
                                    <dd>CADD: 0.994 (22.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000125107">CNOT1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.871</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.590</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.987</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.590 to <a href="http://www.omim.org/618500">Holoprosencephaly 12, with or without pancreatic agenesis</a> associated with CNOT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0005280, Depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/618500">OMIM:618500</a> Holoprosencephaly 12, with or without pancreatic agenesis - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.871</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.590</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.987</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.58543423A&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=556592424">rs556592424</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs556592424&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.987</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441024.6">CNOT1:ENST00000441024.6:c.4618T&gt;G:p.(Phe1540Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317147.10">CNOT1:ENST00000317147.10:c.4434+184T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000569240.5">CNOT1:ENST00000569240.5:c.4419+184T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.000 (B)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.535 (3.326)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0120%</dd>
                                    <dd>gnomAD_E_AFR: 0.0142%</dd>
                                    <dd>gnomAD_E_AMR: 0.0889%</dd>
                                    <dd>gnomAD_E_NFE: 0.0025%</dd>
                                    <dd>gnomAD_E_OTH: 0.0413%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000127616">SMARCA4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.868</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.585</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.990</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.585 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1465">Coffin-Siris syndrome</a> associated with SMARCA4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0005280, Depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000175, Cleft palate</dd></dl><dl><dt>Phenotypic similarity 0.403 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SMARCA4">SMARCA4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0002639, micrognathia</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0001293, anophthalmia</dd><dd>HP:0000175, Cleft palate - MP:0002639, micrognathia</dd><dd>HP:0000218, High palate - MP:0002639, micrognathia</dd></dl><dl><dt>Phenotypic similarity 0.221 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=SMARCA4">SMARCA4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - ZP:0001246, abnormal(ly) morphology inner ear</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SMARCA4%0DARID1B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ARID1B</a> and phenotypic similarity 0.640 to <a href="http://www.omim.org/135900">Coffin-Siris syndrome 1</a> associated with ARID1B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0005280, Depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/613325">OMIM:613325</a> Rhabdoid tumor predisposition syndrome 2 (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/614609">OMIM:614609</a> Coffin-Siris syndrome 4 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1465">ORPHA:1465</a> Coffin-Siris syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.868</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.585</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.990</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.11033439C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537004254">rs537004254</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537004254&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344626.9">SMARCA4:ENST00000344626.9:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413806.7">SMARCA4:ENST00000413806.7:c.3888C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000429416.8">SMARCA4:ENST00000429416.8:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000444061.8">SMARCA4:ENST00000444061.8:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000450717.7">SMARCA4:ENST00000450717.7:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541122.6">SMARCA4:ENST00000541122.6:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589677.5">SMARCA4:ENST00000589677.5:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000590574.6">SMARCA4:ENST00000590574.6:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000642628.1">SMARCA4:ENST00000642628.1:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000642726.1">SMARCA4:ENST00000642726.1:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000643296.1">SMARCA4:ENST00000643296.1:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000643549.1">SMARCA4:ENST00000643549.1:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000644737.1">SMARCA4:ENST00000644737.1:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000645460.1">SMARCA4:ENST00000645460.1:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000646484.1">SMARCA4:ENST00000646484.1:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000646510.1">SMARCA4:ENST00000646510.1:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000646693.1">SMARCA4:ENST00000646693.1:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000647230.1">SMARCA4:ENST00000647230.1:c.3696C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9922375</dd>
                                    
                                    <dd>CADD: 0.992 (21.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000070018">LRP6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.811</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.679</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.837</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.219 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=99798">Oligodontia</a> associated with LRP6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000218, High palate - HP:0000327, Hypoplasia of the maxilla</dd></dl><dl><dt>Phenotypic similarity 0.679 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LRP6">LRP6</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0000547, short limbs</dd><dd>HP:0011800, Midface retrusion - MP:0005170, cleft upper lip</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0001286, abnormal eye development</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dl><dt>Proximity score 0.529 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRP6%0DANTXR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANTXR1</a> and phenotypic similarity 0.670 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2067">GAPO syndrome</a> associated with ANTXR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0000174, Abnormal palate morphology</dd><dd>HP:0000218, High palate - HP:0000174, Abnormal palate morphology</dd></dl><dl><dt>Proximity score 0.529 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRP6%0DANTXR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANTXR1</a> and phenotypic similarity 0.463 to mouse mutant of ANTXR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0002835, abnormal cranial suture morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000120, malocclusion</dd><dd>HP:0000218, High palate - MP:0000120, malocclusion</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/610947">OMIM:610947</a> Coronary artery disease, autosomal dominant, 2 (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/616724">OMIM:616724</a> Tooth agenesis, selective, 7 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=99798">ORPHA:99798</a> Oligodontia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.811</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.679</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.837</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr12:g.12130897G&gt;GA [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=754240736">rs754240736</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs754240736&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.837</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261349.9">LRP6:ENST00000261349.9:c.3971-5_3971-4insT:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543091.1">LRP6:ENST00000543091.1:c.3836-5_3836-4insT:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8424567</dd>
                                    
                                    <dd>CADD: 0.842 (8.026)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ESP AA: 0.0469%</dd>
                                    <dd>ESP EA: 0.0485%</dd>
                                    <dd>ESP All: 0.0479%</dd>
                                    <dd>gnomAD_E_AMR: 0.0301%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_SAS: 0.0099%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000189186">DCAF8L2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.803</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.540</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.540 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DCAF8L2%0DPYROXD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PYROXD1</a> and phenotypic similarity 0.661 to <a href="http://www.omim.org/617258">Myopathy, myofibrillar, 8</a> associated with PYROXD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0000098, Tall stature</dd><dd>HP:0011800, Midface retrusion - HP:0000218, High palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Proximity score 0.540 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DCAF8L2%0DPYROXD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PYROXD1</a> and phenotypic similarity 0.253 to mouse mutant of PYROXD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000470, abnormal stomach morphology</dd><dd>HP:0000218, High palate - MP:0000470, abnormal stomach morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.803</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.540</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.27747991A&gt;G [0/1:0/0:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=774285351">rs774285351</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs774285351&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000451261.6">DCAF8L2:ENST00000451261.6:c.1096A&gt;G:p.(Lys366Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545306.2">DCAF8L2:ENST00000545306.2:c.1096A&gt;G:p.(Lys366Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.997487</dd>
                                    
                                    <dd>Mutation Taster: 0.997 (P)</dd>
                                    <dd>CADD: 0.994 (22.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0265%</dd>
                                    <dd>TOPMed: 0.0207%</dd>
                                    <dd>UK10K: 0.0152%</dd>
                                    <dd>ExAC AMR: 0.0110%</dd>
                                    <dd>ExAC NFE: 0.0086%</dd>
                                    <dd>gnomAD_E_AMR: 0.0602%</dd>
                                    <dd>gnomAD_E_NFE: 0.0063%</dd>
                                    <dd>gnomAD_E_OTH: 0.0247%</dd>
                                    <dd>gnomAD_G_NFE: 0.0095%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000132881">CPLANE2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.799</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.534</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.461 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CPLANE2">CPLANE2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0005170, cleft upper lip</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0005170, cleft upper lip</dd><dd>HP:0000218, High palate - MP:0005170, cleft upper lip</dd></dl><dl><dt>Proximity score 0.534 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CPLANE2%0DFUZ&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FUZ</a> and phenotypic similarity 0.308 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3027">Caudal regression sequence</a> associated with FUZ.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000202, Oral cleft</dd><dd>HP:0000218, High palate - HP:0002023, Anal atresia</dd></dl><dl><dt>Proximity score 0.534 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CPLANE2%0DFUZ&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FUZ</a> and phenotypic similarity 0.655 to mouse mutant of FUZ.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000457, maxilla hypoplasia</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0000031, abnormal cochlea morphology</dd><dd>HP:0000545, Myopia - MP:0001293, anophthalmia</dd><dd>HP:0000175, Cleft palate - MP:0009890, cleft secondary palate</dd><dd>HP:0000218, High palate - MP:0009890, cleft secondary palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.799</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.534</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.16232263G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=571056693">rs571056693</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs571056693&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375599.8">CPLANE2:ENST00000375599.8:c.562C&gt;T:p.(Arg188Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99563485</dd>
                                    
                                    <dd>Polyphen2: 0.394 (B)</dd>
                                    <dd>Mutation Taster: 0.965 (P)</dd>
                                    <dd>SIFT: 0.029 (D)</dd>
                                    <dd>CADD: 0.996 (23.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC NFE: 0.0093%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0059%</dd>
                                    <dd>gnomAD_E_NFE: 0.0056%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169548">ZNF280A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.789</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.527</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.527 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF280A%0DQRICH1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to QRICH1</a> and phenotypic similarity 0.321 to <a href="http://www.omim.org/617982">Ververi-Brady syndrome</a> associated with QRICH1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0000319, Smooth philtrum</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000319, Smooth philtrum</dd><dd>HP:0000218, High palate - HP:0000319, Smooth philtrum</dd></dl><dl><dt>Proximity score 0.527 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF280A%0DQRICH1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to QRICH1</a> and phenotypic similarity 0.604 to mouse mutant of QRICH1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000111, cleft palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.789</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.527</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.22514169T&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361666">rs361666</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361666&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.1462A&gt;C:p.(Ser488Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9939744</dd>
                                    
                                    <dd>CADD: 0.994 (22.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22514173C&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361762">rs361762</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361762&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.994</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.1458G&gt;C:p.(Leu486Phe)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9939744</dd>
                        
                        <dd>CADD: 0.994 (22.200)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22514799C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=362003">rs362003</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs362003&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.994</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.832G&gt;A:p.(Asp278Asn)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.99354345</dd>
                        
                        <dd>CADD: 0.994 (21.900)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr22:g.22514452A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=362173">rs362173</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs362173&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.976</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.1179T&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9757897</dd>
                        
                        <dd>CADD: 0.976 (16.160)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22515418A&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361959">rs361959</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361959&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.972</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.213T&gt;G:p.(Asn71Lys)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.97168607</dd>
                        
                        <dd>CADD: 0.972 (15.480)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22515221G&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361580">rs361580</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361580&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.938</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.410C&gt;A:p.(Ser137Tyr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9379131</dd>
                        
                        <dd>CADD: 0.938 (12.070)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr22:g.22514449A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361737">rs361737</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361737&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.919</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.1182T&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9187169</dd>
                        
                        <dd>CADD: 0.919 (10.900)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22514894C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=362132">rs362132</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs362132&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.905</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.737G&gt;A:p.(Ser246Asn)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.90493953</dd>
                        
                        <dd>CADD: 0.905 (10.220)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22514885G&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=362124">rs362124</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs362124&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.824</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.746C&gt;G:p.(Ala249Gly)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8244504</dd>
                        
                        <dd>CADD: 0.824 (7.556)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22515224C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=362011">rs362011</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs362011&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.533</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.407G&gt;A:p.(Ser136Asn)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.5326954</dd>
                        
                        <dd>CADD: 0.533 (3.304)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr22:g.22515214G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361611">rs361611</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361611&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.030</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302097.3">ZNF280A:ENST00000302097.3:c.417C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.030383468</dd>
                        
                        <dd>CADD: 0.030 (0.134)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154736">ADAMTS5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.773</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.520</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.520 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADAMTS5%0DB3GLCT&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to B3GLCT</a> and phenotypic similarity 0.627 to <a href="http://www.omim.org/261540">Peters-plus syndrome</a> associated with B3GLCT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0003561, Birth length less than 3rd percentile</dd><dd>HP:0011800, Midface retrusion - HP:0000276, Long face</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000175, Cleft palate</dd></dl><dl><dt>Proximity score 0.520 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADAMTS5%0DB3GLCT&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to B3GLCT</a> and phenotypic similarity 0.584 to mouse mutant of B3GLCT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0030066, short face</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0001300, ocular hypertelorism</dd><dd>HP:0000175, Cleft palate - MP:0030066, short face</dd><dd>HP:0000218, High palate - MP:0030066, short face</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.773</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.520</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr21:g.26966192C&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=373831167">rs373831167</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs373831167&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000284987.6">ADAMTS5:ENST00000284987.6:c.200G&gt;T:p.(Arg67Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9946297</dd>
                                    
                                    <dd>Polyphen2: 0.000 (B)</dd>
                                    <dd>SIFT: 0.080 (T)</dd>
                                    <dd>CADD: 0.995 (22.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>gnomAD_E_AMR: 0.0031%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000138768">USO1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.769</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.524</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.984</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.524 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=USO1%0DGOLGB1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GOLGB1</a> and phenotypic similarity 0.651 to mouse mutant of GOLGB1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0009886, failure of palatal shelf elevation</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.769</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.524</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.984</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.75805212G&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=544984728">rs544984728</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs544984728&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.984</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264904.8">USO1:ENST00000264904.8:c.2231G&gt;T:p.(Gly744Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000514213.6">USO1:ENST00000514213.6:c.2198G&gt;T:p.(Gly733Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9871175</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>CADD: 0.987 (18.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000151834">GABRA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.764</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.510</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.309 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=442835">Undetermined early-onset epileptic encephalopathy</a> associated with GABRA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000639, Nystagmus</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Phenotypic similarity 0.293 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GABRA2">GABRA2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0001314, corneal opacity</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.510 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GABRA2%0DGABRB3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GABRB3</a> and phenotypic similarity 0.718 to mouse mutant of GABRB3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000111, cleft palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0004740, sensorineural hearing loss</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/103780">OMIM:103780</a> Alcohol dependence, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=442835">ORPHA:442835</a> Undetermined early-onset epileptic encephalopathy</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.764</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.510</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                                <b>chr4:g.46475066C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=532022892">rs532022892</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs532022892&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000510861.5">GABRA2:ENST00000510861.5:c.-11+1G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9712922</dd>
                                    
                                    <dd>CADD: 0.971 (15.420)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000113810">SMC4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.759</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SMC4%0DSMC1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMC1A</a> and phenotypic similarity 0.673 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=199">Cornelia de Lange syndrome</a> associated with SMC1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001956, Truncal obesity</dd><dd>HP:0011800, Midface retrusion - HP:0005280, Depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.759</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                                <b>chr3:g.160425019G&gt;GGTGTGTGTGTGTGTGTGT [0/1:0/0:0/1]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344722.5">SMC4:ENST00000344722.5:c.2478_2478+1insGTGTGTGTGTGTGTGTGT:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357388.8">SMC4:ENST00000357388.8:c.2478_2478+1insGTGTGTGTGTGTGTGTGT:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000462787.5">SMC4:ENST00000462787.5:c.2478_2478+1insGTGTGTGTGTGTGTGTGT:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000469762.5">SMC4:ENST00000469762.5:c.2403_2403+1insGTGTGTGTGTGTGTGTGT:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483754.1">SMC4:ENST00000483754.1:c.952+13212_952+13213insACACACACACACACACAC:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000076555">ACACB</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.753</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACACB%0DACSL4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACSL4</a> and phenotypic similarity 0.645 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with ACSL4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001513, Obesity</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008587, Mild neurosensory hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000343, Long philtrum</dd><dd>HP:0000218, High palate - HP:0000343, Long philtrum</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACACB%0DACSL4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACSL4</a> and phenotypic similarity 0.299 to mouse mutant of ACSL4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0011967, increased or absent threshold for auditory brainstem response</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.753</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.109216914G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=539224114">rs539224114</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs539224114&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000338432.11">ACACB:ENST00000338432.11:c.3558G&gt;A:p.(Met1186Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377848.7">ACACB:ENST00000377848.7:c.3558G&gt;A:p.(Met1186Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377854.9">ACACB:ENST00000377854.9:c.-445G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.050 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.119 (T)</dd>
                                    <dd>CADD: 0.995 (23.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000101255">TRIB3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.751</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRIB3%0DMAP2K1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MAP2K1</a> and phenotypic similarity 0.657 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1340">Cardiofaciocutaneous syndrome</a> associated with MAP2K1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0000293, Full cheeks</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000639, Nystagmus</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0000176, Submucous cleft hard palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TRIB3%0DMAP2K1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MAP2K1</a> and phenotypic similarity 0.422 to mouse mutant of MAP2K1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0008526, decreased cranium width</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0030166, increased inner canthal distance</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.751</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_VARIANT</span>
                                <b>chr20:g.391492TG&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=771814056">rs771814056</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs771814056&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000217233.8">TRIB3:ENST00000217233.8:c.499del:p.(Ala167Profs*43)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422053.3">TRIB3:ENST00000422053.3:c.580del:p.(Ala194Profs*43)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9987411</dd>
                                    
                                    <dd>CADD: 0.999 (29.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000072310">SREBF1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.750</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SREBF1%0DSEC23A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SEC23A</a> and phenotypic similarity 0.622 to <a href="http://www.omim.org/607812">Craniolenticulosutural dysplasia</a> associated with SEC23A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000193, Bifid uvula</dd><dd>HP:0000218, High palate - HP:0000193, Bifid uvula</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SREBF1%0DSEC23A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SEC23A</a> and phenotypic similarity 0.178 to fish mutant of SEC23A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - ZP:0001249, abnormal(ly) aplastic semicircular canal</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.750</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.17819684G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=531373893">rs531373893</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs531373893&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261646.10">SREBF1:ENST00000261646.10:c.565C&gt;T:p.(Pro189Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355815.8">SREBF1:ENST00000355815.8:c.655C&gt;T:p.(Pro219Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395757.6">SREBF1:ENST00000395757.6:c.565C&gt;T:p.(Pro189Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.019 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.751 (T)</dd>
                                    <dd>CADD: 0.969 (15.090)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>gnomAD_E_AMR: 0.0090%</dd>
                                    <dd>gnomAD_E_NFE: 0.0028%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000275954">TBC1D3F</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.749</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBC1D3F%0DGDI1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GDI1</a> and phenotypic similarity 0.645 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with GDI1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001513, Obesity</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008587, Mild neurosensory hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000343, Long philtrum</dd><dd>HP:0000218, High palate - HP:0000343, Long philtrum</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.749</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.36435803G&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1235437204">rs1235437204</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1235437204&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620210.1">TBC1D3F:ENST00000620210.1:c.252C&gt;A:p.(Asp84Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.336 (1.778)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000005844">ITGAL</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.748</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ITGAL%0DITGB8&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ITGB8</a> and phenotypic similarity 0.627 to mouse mutant of ITGB8.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000111, cleft palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.748</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.30483943T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=142620368">rs142620368</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs142620368&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356798.11">ITGAL:ENST00000356798.11:c.839T&gt;C:p.(Ile280Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358164.9">ITGAL:ENST00000358164.9:c.590T&gt;C:p.(Ile197Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000433423.2">ITGAL:ENST00000433423.2:c.153+9656T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.199 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.108 (T)</dd>
                                    <dd>CADD: 0.994 (22.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000152578">GRIA4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.747</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.309 to <a href="http://www.omim.org/617864">Neurodevelopmental disorder with or without seizures and gait abnormalities</a> associated with GRIA4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000639, Nystagmus</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GRIA4%0DDLG3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DLG3</a> and phenotypic similarity 0.645 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with DLG3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001513, Obesity</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008587, Mild neurosensory hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000343, Long philtrum</dd><dd>HP:0000218, High palate - HP:0000343, Long philtrum</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/617864">OMIM:617864</a> Neurodevelopmental disorder with or without seizures and gait abnormalities - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.747</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.105924663G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=369753546">rs369753546</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs369753546&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000282499.10">GRIA4:ENST00000282499.10:c.1741G&gt;A:p.(Gly581Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393127.6">GRIA4:ENST00000393127.6:c.1741G&gt;A:p.(Gly581Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000525187.5">GRIA4:ENST00000525187.5:c.1741G&gt;A:p.(Gly581Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530497.1">GRIA4:ENST00000530497.1:c.1741G&gt;A:p.(Gly581Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.704 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.042 (D)</dd>
                                    <dd>CADD: 0.997 (25.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>ExAC SAS: 0.0242%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_E_SAS: 0.0195%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000157782">CABP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.747</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.999</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.397 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CABP1">CABP1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0005551, abnormal eye electrophysiology</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CABP1%0DRYR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RYR1</a> and phenotypic similarity 0.555 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98905">Congenital multicore myopathy with external ophthalmoplegia</a> associated with RYR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0000275, Narrow face</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CABP1%0DRYR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RYR1</a> and phenotypic similarity 0.614 to mouse mutant of RYR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000440, domed cranium</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0009890, cleft secondary palate</dd><dd>HP:0000218, High palate - MP:0009890, cleft secondary palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.747</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.999</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.120641293C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1048321578">rs1048321578</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1048321578&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.999</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000316803.7">CABP1:ENST00000316803.7:c.608C&gt;T:p.(Pro203Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999</dd>
                                    
                                    <dd>Polyphen2: 0.837 (P)</dd>
                                    <dd>Mutation Taster: 0.837</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.998 (27.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0016%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165115">KIF27</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.746</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.403 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=KIF27">KIF27</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0002236, abnormal internal nares morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0001850, increased susceptibility to otitis media</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0002236, abnormal internal nares morphology</dd><dd>HP:0000218, High palate - MP:0002236, abnormal internal nares morphology</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIF27%0DGLI3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GLI3</a> and phenotypic similarity 0.587 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=36">Acrocallosal syndrome</a> associated with GLI3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0000098, Tall stature</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIF27%0DGLI3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GLI3</a> and phenotypic similarity 0.677 to mouse mutant of GLI3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0002764, short tibia</dd><dd>HP:0011800, Midface retrusion - MP:0000455, abnormal maxilla morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0000023, abnormal ear position</dd><dd>HP:0000545, Myopia - MP:0001286, abnormal eye development</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.746</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr9:g.83908476G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537684585">rs537684585</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537684585&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000297814.7">KIF27:ENST00000297814.7:c.475C&gt;T:p.(Arg159*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000334204.6">KIF27:ENST00000334204.6:c.475C&gt;T:p.(Arg159*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413982.5">KIF27:ENST00000413982.5:c.475C&gt;T:p.(Arg159*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 1.000 (36.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>ExAC SAS: 0.0122%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0297%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143379">SETDB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.745</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.508</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.508 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SETDB1%0DPLOD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PLOD1</a> and phenotypic similarity 0.799 to <a href="http://www.omim.org/225400">Ehlers-Danlos syndrome, kyphoscoliotic type, 1</a> associated with PLOD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001519, Disproportionate tall stature</dd><dd>HP:0011800, Midface retrusion - HP:0005280, Depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0005280, Depressed nasal bridge</dd><dd>HP:0000218, High palate - HP:0005280, Depressed nasal bridge</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.745</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_VARIANT</span>
                                <b>chr1:g.150945147TG&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=755041728">rs755041728</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs755041728&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368962.6">SETDB1:ENST00000368962.6:c.1187del:p.(Gly396Glufs*6)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000271640.9">SETDB1:ENST00000271640.9:c.1140+40del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368969.8">SETDB1:ENST00000368969.8:c.1140+40del:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.013038993</dd>
                                    
                                    <dd>CADD: 0.013 (0.057)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0600%</dd>
                                    <dd>TOPMed: 0.0600%</dd>
                                    <dd>ExAC AFR: 0.0234%</dd>
                                    <dd>ExAC AMR: 0.0609%</dd>
                                    <dd>ExAC NFE: 0.0323%</dd>
                                    <dd>ExAC SAS: 0.0436%</dd>
                                    <dd>gnomAD_E_AFR: 0.0068%</dd>
                                    <dd>gnomAD_E_AMR: 0.0768%</dd>
                                    <dd>gnomAD_E_EAS: 0.0059%</dd>
                                    <dd>gnomAD_E_NFE: 0.0141%</dd>
                                    <dd>gnomAD_E_SAS: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000081803">CADPS2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.745</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CADPS2%0DACSL4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACSL4</a> and phenotypic similarity 0.645 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with ACSL4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001513, Obesity</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008587, Mild neurosensory hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000343, Long philtrum</dd><dd>HP:0000218, High palate - HP:0000343, Long philtrum</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CADPS2%0DACSL4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACSL4</a> and phenotypic similarity 0.299 to mouse mutant of ACSL4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0011967, increased or absent threshold for auditory brainstem response</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.745</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.122663470C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=571419420">rs571419420</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs571419420&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313070.11">CADPS2:ENST00000313070.11:c.235G&gt;A:p.(Val79Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000334010.11">CADPS2:ENST00000334010.11:c.553G&gt;A:p.(Val185Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000412584.6">CADPS2:ENST00000412584.6:c.553G&gt;A:p.(Val185Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449022.7">CADPS2:ENST00000449022.7:c.553G&gt;A:p.(Val185Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615869.4">CADPS2:ENST00000615869.4:c.553G&gt;A:p.(Val185Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.994 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.996 (24.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170035">UBE2E3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.745</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=UBE2E3%0DARX&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ARX</a> and phenotypic similarity 0.645 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with ARX.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001513, Obesity</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008587, Mild neurosensory hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000343, Long philtrum</dd><dd>HP:0000218, High palate - HP:0000343, Long philtrum</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=UBE2E3%0DARX&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ARX</a> and phenotypic similarity 0.220 to mouse mutant of ARX.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0005218, abnormal pancreatic delta cell morphology</dd><dd>HP:0000218, High palate - MP:0005218, abnormal pancreatic delta cell morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.745</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.180989882G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=533732420">rs533732420</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs533732420&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602475.5">UBE2E3:ENST00000602475.5:c.246G&gt;C:p.(Arg82Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602632.1">UBE2E3:ENST00000602632.1:c.303G&gt;C:p.(Arg101Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392415.6">UBE2E3:ENST00000392415.6:c.245+5789G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000410062.9">UBE2E3:ENST00000410062.9:c.245+5789G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602710.5">UBE2E3:ENST00000602710.5:c.245+5789G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602959.5">UBE2E3:ENST00000602959.5:c.245+5789G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.780 (6.577)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0193%</dd>
                                    <dd>gnomAD_E_NFE: 0.0020%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198890">PRMT6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.744</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PRMT6%0DHOXA2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HOXA2</a> and phenotypic similarity 0.569 to <a href="http://www.omim.org/612290">?Microtia, hearing impairment, and cleft palate (AR)</a> associated with HOXA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0000175, Cleft palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000410, Mixed hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000175, Cleft palate</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PRMT6%0DHOXA2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HOXA2</a> and phenotypic similarity 0.641 to mouse mutant of HOXA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0004423, abnormal temporal bone squamous part morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0002177, abnormal outer ear morphology</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.744</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.107057251A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=565706783">rs565706783</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs565706783&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370078.2">PRMT6:ENST00000370078.2:c.536A&gt;G:p.(Lys179Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.678 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.005 (D)</dd>
                                    <dd>CADD: 0.999 (30.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183549">ACSM5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.744</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACSM5%0DACSL4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACSL4</a> and phenotypic similarity 0.645 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with ACSL4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001513, Obesity</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008587, Mild neurosensory hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000343, Long philtrum</dd><dd>HP:0000218, High palate - HP:0000343, Long philtrum</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACSM5%0DACSL4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACSL4</a> and phenotypic similarity 0.299 to mouse mutant of ACSL4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0011967, increased or absent threshold for auditory brainstem response</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.744</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.20419282G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=572015052">rs572015052</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs572015052&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000331849.8">ACSM5:ENST00000331849.8:c.470G&gt;A:p.(Arg157Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000575584.5">ACSM5:ENST00000575584.5:c.470G&gt;A:p.(Arg157Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 1.000 (D)</dd>
                                    <dd>Mutation Taster: 0.997 (P)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.999 (28.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188611">ASAH2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.744</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ASAH2%0DAGPAT3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AGPAT3</a> and phenotypic similarity 0.622 to mouse mutant of AGPAT3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0002090, abnormal vision</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.744</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.50189586G&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1279780267">rs1279780267</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1279780267&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329428.10">ASAH2:ENST00000329428.10:c.1904C&gt;A:p.(Thr635Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395526.9">ASAH2:ENST00000395526.9:c.2066C&gt;A:p.(Thr689Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000443575.5">ASAH2:ENST00000443575.5:c.1592C&gt;A:p.(Thr531Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000447815.5">ASAH2:ENST00000447815.5:c.1961C&gt;A:p.(Thr654Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000656090.1">ASAH2:ENST00000656090.1:c.2066C&gt;A:p.(Thr689Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.996965</dd>
                                    
                                    <dd>Polyphen2: 0.787 (P)</dd>
                                    <dd>Mutation Taster: 0.997 (P)</dd>
                                    <dd>SIFT: 0.007 (D)</dd>
                                    <dd>CADD: 0.996 (24.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000141434">MEP1B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.743</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.367 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MEP1B">MEP1B</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0001260, increased body weight</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MEP1B%0DNECTIN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NECTIN1</a> and phenotypic similarity 0.601 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3253">Cleft lip/palate-ectodermal dysplasia syndrome</a> associated with NECTIN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0002744, Bilateral cleft lip and palate</dd><dd>HP:0000218, High palate - HP:0002744, Bilateral cleft lip and palate</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MEP1B%0DNECTIN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NECTIN1</a> and phenotypic similarity 0.442 to mouse mutant of NECTIN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000126, brittle teeth</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0001297, microphthalmia</dd><dd>HP:0000175, Cleft palate - MP:0000126, brittle teeth</dd><dd>HP:0000218, High palate - MP:0000126, brittle teeth</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.743</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr18:g.32204318C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=531662397">rs531662397</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs531662397&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000269202.11">MEP1B:ENST00000269202.11:c.505C&gt;T:p.(Arg169Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581447.1">MEP1B:ENST00000581447.1:c.505C&gt;T:p.(Arg169Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 1.000 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.999 (29.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ExAC AFR: 0.0182%</dd>
                                    <dd>ExAC NFE: 0.0234%</dd>
                                    <dd>gnomAD_E_AFR: 0.0076%</dd>
                                    <dd>gnomAD_E_NFE: 0.0117%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000119522">DENND1A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.743</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.154 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DENND1A">DENND1A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0004001, decreased hepatocyte proliferation</dd><dd>HP:0000218, High palate - MP:0004001, decreased hepatocyte proliferation</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DENND1A%0DSCLT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SCLT1</a> and phenotypic similarity 0.601 to mouse mutant of SCLT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000111, cleft palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.743</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.123381763G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=528747641">rs528747641</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs528747641&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373624.6">DENND1A:ENST00000373624.6:c.2699C&gt;T:p.(Pro900Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999369</dd>
                                    
                                    <dd>Polyphen2: 0.997 (D)</dd>
                                    <dd>Mutation Taster: 0.998 (P)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.999 (32.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0209%</dd>
                                    <dd>gnomAD_E_NFE: 0.0023%</dd>
                                    <dd>gnomAD_E_SAS: 0.0071%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000184083">FAM120C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.743</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FAM120C%0DTRPM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRPM1</a> and phenotypic similarity 0.496 to <a href="http://www.omim.org/613216">Night blindness, congenital stationary (complete), 1C, autosomal recessive</a> associated with TRPM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000639, Nystagmus</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FAM120C%0DTRPM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRPM1</a> and phenotypic similarity 0.644 to mouse mutant of TRPM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0002090, abnormal vision</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.743</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.54182952G&gt;A [0/1:0/0:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=781837490">rs781837490</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs781837490&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000328235.4">FAM120C:ENST00000328235.4:c.247C&gt;T:p.(Arg83Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375180.7">FAM120C:ENST00000375180.7:c.247C&gt;T:p.(Arg83Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000477084.1">FAM120C:ENST00000477084.1:c.247C&gt;T:p.(Arg83Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.002 (B)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.996 (24.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0265%</dd>
                                    <dd>TOPMed: 0.0265%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170122">FOXD4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.742</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FOXD4%0DTFAP2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TFAP2A</a> and phenotypic similarity 0.611 to <a href="http://www.omim.org/113620">Branchiooculofacial syndrome</a> associated with TFAP2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0005280, Depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000175, Cleft palate</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FOXD4%0DTFAP2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TFAP2A</a> and phenotypic similarity 0.647 to mouse mutant of TFAP2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000455, abnormal maxilla morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0013550, abnormal secondary palate morphology</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FOXD4%0DTFAP2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TFAP2A</a> and phenotypic similarity 0.320 to fish mutant of TFAP2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - ZP:0001979, abnormal(ly) split ethmoid cartilage</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - ZP:0001975, abnormal(ly) mislocalised palatoquadrate cartilage</dd><dd>HP:0000218, High palate - ZP:0001975, abnormal(ly) mislocalised palatoquadrate cartilage</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.742</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.116931G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=535147968">rs535147968</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs535147968&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382500.3">FOXD4:ENST00000382500.3:c.1189C&gt;T:p.(Arg397Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9946297</dd>
                                    
                                    <dd>Polyphen2: 0.639 (P)</dd>
                                    <dd>SIFT: 0.006 (D)</dd>
                                    <dd>CADD: 0.995 (22.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AMR: 0.0174%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000127362">TAS2R3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.740</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.995</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TAS2R3%0DKISS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KISS1</a> and phenotypic similarity 0.619 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=432">Normosmic congenital hypogonadotropic hypogonadism</a> associated with KISS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0000002, Abnormality of body height</dd><dd>HP:0011800, Midface retrusion - HP:0005280, Depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008527, Congenital sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000175, Cleft palate</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TAS2R3%0DKISS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KISS1</a> and phenotypic similarity 0.429 to mouse mutant of KISS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000618, small salivary gland</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000618, small salivary gland</dd><dd>HP:0000218, High palate - MP:0000618, small salivary gland</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.740</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.995</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.141764543G&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=551280379">rs551280379</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs551280379&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.995</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000247879.2">TAS2R3:ENST00000247879.2:c.385G&gt;T:p.(Val129Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000465582.5">TAS2R3:ENST00000465582.5:c.*30+14159G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.998 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.064 (T)</dd>
                                    <dd>CADD: 0.992 (20.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185304">RGPD2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.738</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.390 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RGPD2">RGPD2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0004021, abnormal rod electrophysiology</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RGPD2%0DDVL3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DVL3</a> and phenotypic similarity 0.662 to <a href="http://www.omim.org/616894">Robinow syndrome, autosomal dominant 3</a> associated with DVL3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000175, Cleft palate</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RGPD2%0DDVL3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DVL3</a> and phenotypic similarity 0.450 to mouse mutant of DVL3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000445, short snout</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0006198, enophthalmos</dd><dd>HP:0000175, Cleft palate - MP:0000445, short snout</dd><dd>HP:0000218, High palate - MP:0000445, short snout</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.738</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.87783164C&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=771070394">rs771070394</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs771070394&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398146.4">RGPD2:ENST00000398146.4:c.3860G&gt;T:p.(Ser1287Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000638730.1">RGPD2:ENST00000638730.1:c.3860G&gt;T:p.(Ser1287Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9981379</dd>
                                    
                                    <dd>Mutation Taster: 0.875</dd>
                                    <dd>SIFT: 0.006 (D)</dd>
                                    <dd>CADD: 0.998 (27.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0088%</dd>
                                    <dd>gnomAD_E_AMR: 0.0383%</dd>
                                    <dd>gnomAD_E_NFE: 0.0287%</dd>
                                    <dd>gnomAD_E_OTH: 0.0272%</dd>
                                    <dd>gnomAD_G_NFE: 0.0399%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196531">NACA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.737</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NACA%0DIRX5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRX5</a> and phenotypic similarity 0.630 to <a href="http://www.omim.org/611174">Hamamy syndrome</a> associated with IRX5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0000431, Wide nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia - HP:0011003, High myopia</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NACA%0DIRX5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRX5</a> and phenotypic similarity 0.297 to mouse mutant of IRX5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0005287, narrow eye opening</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.737</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.56717076G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=557754386">rs557754386</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs557754386&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454682.5">NACA:ENST00000454682.5:c.4454C&gt;T:p.(Ala1485Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356769.7">NACA:ENST00000356769.7:c.71-2389C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393891.8">NACA:ENST00000393891.8:c.71-2389C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000546392.5">NACA:ENST00000546392.5:c.71-2389C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000550920.6">NACA:ENST00000550920.6:c.131-2389C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000550952.6">NACA:ENST00000550952.6:c.1865-870C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000552540.5">NACA:ENST00000552540.5:c.71-2389C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000647707.1">NACA:ENST00000647707.1:c.513-2389C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000548563.5">NACA:ENST00000548563.5:c.-183-2373C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.995</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.005 (D)</dd>
                                    <dd>CADD: 0.184 (0.883)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0017%</dd>
                                    <dd>gnomAD_E_NFE: 0.0015%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000101019">UQCC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.734</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=UQCC1%0DIGF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IGF1</a> and phenotypic similarity 0.615 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=73272">Growth delay due to insulin-like growth factor type 1 deficiency</a> associated with IGF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001956, Truncal obesity</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008619, Bilateral sensorineural hearing impairment</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=UQCC1%0DIGF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IGF1</a> and phenotypic similarity 0.662 to mouse mutant of IGF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0004740, sensorineural hearing loss</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.734</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr20:g.35374234C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=529226801">rs529226801</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs529226801&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374380.6">UQCC1:ENST00000374380.6:c.152G&gt;A:p.(Arg51His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374384.6">UQCC1:ENST00000374384.6:c.356G&gt;A:p.(Arg119His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374385.10">UQCC1:ENST00000374385.10:c.356G&gt;A:p.(Arg119His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397554.5">UQCC1:ENST00000397554.5:c.356G&gt;A:p.(Arg119His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349714.9">UQCC1:ENST00000349714.9:c.326-7620G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359226.6">UQCC1:ENST00000359226.6:c.333+7684G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397556.7">UQCC1:ENST00000397556.7:c.188-7620G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.290 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.163 (T)</dd>
                                    <dd>CADD: 0.995 (22.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0080%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AMR: 0.0243%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_G_AFR: 0.0114%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000166582">CENPV</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.732</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.990</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CENPV%0DCISD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CISD2</a> and phenotypic similarity 0.446 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3463">Wolfram syndrome</a> associated with CISD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0002592, Gastric ulcer</dd><dd>HP:0000218, High palate - HP:0002592, Gastric ulcer</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CENPV%0DCISD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CISD2</a> and phenotypic similarity 0.623 to mouse mutant of CISD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0000021, prominent ears</dd><dd>HP:0000545, Myopia - MP:0002001, blindness</dd><dd>HP:0000175, Cleft palate - MP:0003866, abnormal defecation</dd><dd>HP:0000218, High palate - MP:0003866, abnormal defecation</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.732</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.990</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.16353172G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=760190524">rs760190524</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs760190524&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000299736.5">CENPV:ENST00000299736.5:c.265C&gt;T:p.(Pro89Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9920567</dd>
                                    
                                    <dd>Polyphen2: 0.104 (B)</dd>
                                    <dd>SIFT: 0.169 (T)</dd>
                                    <dd>CADD: 0.992 (21.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC NFE: 0.0067%</dd>
                                    <dd>gnomAD_E_AMR: 0.0133%</dd>
                                    <dd>gnomAD_E_NFE: 0.0030%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000160613">PCSK7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.732</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.265 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=PCSK7">PCSK7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - ZP:0007265, abnormal(ly) disrupted otolith development</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PCSK7%0DPCSK5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PCSK5</a> and phenotypic similarity 0.612 to mouse mutant of PCSK5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0003743, abnormal facial morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.732</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr11:g.117228294A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=534958050">rs534958050</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs534958050&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000320934.8">PCSK7:ENST00000320934.8:c.525T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540028.5">PCSK7:ENST00000540028.5:c.525T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9912904</dd>
                                    
                                    <dd>CADD: 0.991 (20.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000111271">ACAD10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.729</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACAD10%0DANTXR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANTXR1</a> and phenotypic similarity 0.670 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2067">GAPO syndrome</a> associated with ANTXR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0000174, Abnormal palate morphology</dd><dd>HP:0000218, High palate - HP:0000174, Abnormal palate morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACAD10%0DANTXR1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANTXR1</a> and phenotypic similarity 0.463 to mouse mutant of ANTXR1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0002835, abnormal cranial suture morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000120, malocclusion</dd><dd>HP:0000218, High palate - MP:0000120, malocclusion</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.729</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.111756411G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=142030532">rs142030532</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs142030532&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313698.8">ACAD10:ENST00000313698.8:c.3118G&gt;A:p.(Asp1040Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000455480.6">ACAD10:ENST00000455480.6:c.3211G&gt;A:p.(Asp1071Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000546840.3">ACAD10:ENST00000546840.3:c.104+666G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.979 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.002 (D)</dd>
                                    <dd>CADD: 0.997 (24.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0422%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP EA: 0.0582%</dd>
                                    <dd>ESP All: 0.0538%</dd>
                                    <dd>ExAC AFR: 0.0730%</dd>
                                    <dd>ExAC AMR: 0.0191%</dd>
                                    <dd>ExAC NFE: 0.0389%</dd>
                                    <dd>ExAC SAS: 0.0066%</dd>
                                    <dd>gnomAD_E_AFR: 0.0667%</dd>
                                    <dd>gnomAD_E_AMR: 0.0181%</dd>
                                    <dd>gnomAD_E_NFE: 0.0374%</dd>
                                    <dd>gnomAD_E_OTH: 0.0373%</dd>
                                    <dd>gnomAD_E_SAS: 0.0066%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_NFE: 0.0200%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000274808">TBC1D3B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.728</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.988</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBC1D3B%0DGDI1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GDI1</a> and phenotypic similarity 0.645 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with GDI1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001513, Obesity</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008587, Mild neurosensory hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000343, Long philtrum</dd><dd>HP:0000218, High palate - HP:0000343, Long philtrum</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.728</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.988</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.36166065C&gt;T [1/1:0/1:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=745769652">rs745769652</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs745769652&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.988</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611257.4">TBC1D3B:ENST00000611257.4:c.1580G&gt;A:p.(Cys527Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98825103</dd>
                                    
                                    <dd>CADD: 0.988 (19.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000149090">PAMR1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.724</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.986</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PAMR1%0DGABRD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GABRD</a> and phenotypic similarity 0.673 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1606">1p36 deletion syndrome</a> associated with GABRD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001513, Obesity</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia - HP:0008499, High hypermetropia</dd><dd>HP:0000175, Cleft palate - HP:0000343, Long philtrum</dd><dd>HP:0000218, High palate - HP:0000343, Long philtrum</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.724</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.986</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.35432738G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=554767403">rs554767403</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs554767403&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.986</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611014.4">PAMR1:ENST00000611014.4:c.869C&gt;T:p.(Ser290Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615849.4">PAMR1:ENST00000615849.4:c.1448C&gt;T:p.(Ser483Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000619888.4">PAMR1:ENST00000619888.4:c.1781C&gt;T:p.(Ser594Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621476.4">PAMR1:ENST00000621476.4:c.1661C&gt;T:p.(Ser554Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000622144.4">PAMR1:ENST00000622144.4:c.1832C&gt;T:p.(Ser611Phe)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9884389</dd>
                                    
                                    <dd>Polyphen2: 0.055 (B)</dd>
                                    <dd>CADD: 0.988 (19.370)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196862">RGPD4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.723</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.984</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.390 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RGPD4">RGPD4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0004021, abnormal rod electrophysiology</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RGPD4%0DFRAS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FRAS1</a> and phenotypic similarity 0.572 to <a href="http://www.omim.org/219000">Fraser syndrome 1</a> associated with FRAS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0005280, Depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000405, Conductive hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000175, Cleft palate</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=RGPD4%0DFRAS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FRAS1</a> and phenotypic similarity 0.649 to mouse mutant of FRAS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0009890, cleft secondary palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0005287, narrow eye opening</dd><dd>HP:0000175, Cleft palate - MP:0009890, cleft secondary palate</dd><dd>HP:0000218, High palate - MP:0009890, cleft secondary palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.723</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.984</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                                <b>chr2:g.107872584AAAC&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=770558604">rs770558604</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs770558604&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.984</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000408999.3">RGPD4:ENST00000408999.3:c.4589_4591del:p.(Thr1530del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98775387</dd>
                                    
                                    <dd>CADD: 0.988 (19.120)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AMR: 0.0242%</dd>
                                    <dd>ExAC NFE: 0.0040%</dd>
                                    <dd>gnomAD_E_AMR: 0.0151%</dd>
                                    <dd>gnomAD_E_NFE: 0.0019%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204174">NPY4R</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.721</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.985</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NPY4R%0DNXNL1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NXNL1</a> and phenotypic similarity 0.631 to mouse mutant of NXNL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0002090, abnormal vision</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.721</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.985</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.46462341C&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2229967">rs2229967</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2229967&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=701353%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.985</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374312.5">NPY4R:ENST00000374312.5:c.295G&gt;T:p.(Ala99Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612632.3">NPY4R:ENST00000612632.3:c.295G&gt;T:p.(Ala99Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.994</dd>
                                    
                                    <dd>SIFT: 0.006 (D)</dd>
                                    <dd>CADD: 0.982 (17.410)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0071%</dd>
                                    <dd>gnomAD_E_EAS: 0.0186%</dd>
                                    <dd>gnomAD_E_FIN: 0.0051%</dd>
                                    <dd>gnomAD_E_NFE: 0.0103%</dd>
                                    <dd>gnomAD_G_EAS: 0.0674%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000173706">HEG1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.721</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.984</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=HEG1%0DRAP1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAP1A</a> and phenotypic similarity 0.609 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2322">Kabuki syndrome</a> associated with RAP1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001513, Obesity</dd><dd>HP:0011800, Midface retrusion - HP:0000218, High palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.721</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.984</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.125055776G&gt;C [0/1:0/0:0/1]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.984</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311127.9">HEG1:ENST00000311127.9:c.115C&gt;G:p.(Arg39Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9842602</dd>
                                    
                                    <dd>Polyphen2: 0.044 (B)</dd>
                                    <dd>SIFT: 0.019 (D)</dd>
                                    <dd>CADD: 0.984 (18.030)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000059691">GATB</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.719</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.982</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GATB%0DEBP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EBP</a> and phenotypic similarity 0.672 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=401973">MEND syndrome</a> associated with EBP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.719</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.982</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.151760854G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=537155081">rs537155081</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs537155081&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.982</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263985.11">GATB:ENST00000263985.11:c.129C&gt;G:p.(Ser43Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000508611.1">GATB:ENST00000508611.1:c.129C&gt;G:p.(Ser43Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512306.5">GATB:ENST00000512306.5:c.129C&gt;G:p.(Ser43Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000515812.5">GATB:ENST00000515812.5:c.129C&gt;G:p.(Ser43Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98465383</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.045 (D)</dd>
                                    <dd>CADD: 0.985 (18.140)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000205213">LGR4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.719</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.514</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.967</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.491 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LGR4">LGR4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0001313, increased incidence of corneal inflammation</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LGR4%0DRSPO2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RSPO2</a> and phenotypic similarity 0.403 to <a href="http://www.omim.org/618021">Tetraamelia syndrome 2</a> associated with RSPO2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0100336, Bilateral cleft lip</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0100336, Bilateral cleft lip</dd><dd>HP:0000218, High palate - HP:0010296, Ankyloglossia</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LGR4%0DRSPO2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RSPO2</a> and phenotypic similarity 0.669 to mouse mutant of RSPO2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0002765, short fibula</dd><dd>HP:0011800, Midface retrusion - MP:0030302, abnormal maxillary zygomatic process morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0006020, decreased tympanic ring size</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dl><dt>Proximity score 0.514 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LGR4%0DRSPO2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RSPO2</a> and phenotypic similarity 0.198 to fish mutant of RSPO2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - ZP:0012199, abnormal(ly) hypoplastic pelvic fin lepidotrichium</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.719</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.514</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.967</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr11:g.27368638T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=561587043">rs561587043</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs561587043&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.967</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379214.9">LGR4:ENST00000379214.9:c.2085A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389858.4">LGR4:ENST00000389858.4:c.2013A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97007734</dd>
                                    
                                    <dd>CADD: 0.970 (15.240)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171566">PLRG1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.713</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.978</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.349 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=PLRG1">PLRG1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - ZP:0000026, abnormal(ly) decreased length trunk</dd><dd>HP:0011800, Midface retrusion - ZP:0000399, abnormal(ly) flat head</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLRG1%0DSNRPN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SNRPN</a> and phenotypic similarity 0.611 to <a href="http://www.omim.org/176270">Prader-Willi syndrome</a> associated with SNRPN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0012743, Abdominal obesity</dd><dd>HP:0011800, Midface retrusion - HP:0000219, Thin upper lip vermilion</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0000219, Thin upper lip vermilion</dd><dd>HP:0000218, High palate - HP:0000219, Thin upper lip vermilion</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLRG1%0DSNRPN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SNRPN</a> and phenotypic similarity 0.398 to mouse mutant of SNRPN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0001260, increased body weight</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.713</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.978</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr4:g.154544463T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=530993501">rs530993501</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs530993501&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.978</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302078.9">PLRG1:ENST00000302078.9:c.549A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000499023.7">PLRG1:ENST00000499023.7:c.576A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9803664</dd>
                                    
                                    <dd>CADD: 0.980 (17.070)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105976">MET</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.712</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.972</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.403 to <a href="http://www.omim.org/616705">?Deafness, autosomal recessive 97</a> associated with MET.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Phenotypic similarity 0.462 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MET">MET</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000762, abnormal tongue morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000762, abnormal tongue morphology</dd><dd>HP:0000218, High palate - MP:0000762, abnormal tongue morphology</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MET%0DPDGFC&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PDGFC</a> and phenotypic similarity 0.648 to mouse mutant of PDGFC.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0009891, abnormal palate bone morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0009890, cleft secondary palate</dd><dd>HP:0000218, High palate - MP:0009891, abnormal palate bone morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/114550">OMIM:114550</a> Hepatocellular carcinoma, childhood type, somatic - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/605074">OMIM:605074</a> Renal cell carcinoma, papillary, 1, familial and somatic - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/607278">OMIM:607278</a> Osteofibrous dysplasia, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/616705">OMIM:616705</a> ?Deafness, autosomal recessive 97 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=33402">ORPHA:33402</a> Pediatric hepatocellular carcinoma</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.712</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.972</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.116699164A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=555920594">rs555920594</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs555920594&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=561188%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.972</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000318493.11">MET:ENST00000318493.11:c.80A&gt;G:p.(Lys27Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397752.7">MET:ENST00000397752.7:c.80A&gt;G:p.(Lys27Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000436117.2">MET:ENST00000436117.2:c.80A&gt;G:p.(Lys27Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.974707</dd>
                                    
                                    <dd>Polyphen2: 0.002 (B)</dd>
                                    <dd>SIFT: 0.177 (T)</dd>
                                    <dd>CADD: 0.975 (15.970)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000125498">KIR2DL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.710</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.978</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIR2DL1%0DPTPN11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTPN11</a> and phenotypic similarity 0.814 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=500">Noonan syndrome with multiple lentigines</a> associated with PTPN11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0000431, Wide nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008625, Severe sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000431, Wide nasal bridge</dd><dd>HP:0000218, High palate - HP:0000431, Wide nasal bridge</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIR2DL1%0DPTPN11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTPN11</a> and phenotypic similarity 0.508 to mouse mutant of PTPN11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0030034, depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0000042, abnormal organ of Corti morphology</dd><dd>HP:0000545, Myopia - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000175, Cleft palate - MP:0030034, depressed nasal bridge</dd><dd>HP:0000218, High palate - MP:0030034, depressed nasal bridge</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIR2DL1%0DPTPN11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTPN11</a> and phenotypic similarity 0.195 to fish mutant of PTPN11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - ZP:0004159, abnormal(ly) position otolith</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.710</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.978</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54775329G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=62121640">rs62121640</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs62121640&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.978</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000291633.7">KIR2DL1:ENST00000291633.7:c.535G&gt;A:p.(Gly179Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336077.11">KIR2DL1:ENST00000336077.11:c.535G&gt;A:p.(Gly179Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9833275</dd>
                                    
                                    <dd>Polyphen2: 0.274 (B)</dd>
                                    <dd>SIFT: 0.399 (T)</dd>
                                    <dd>CADD: 0.983 (17.780)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0399%</dd>
                                    <dd>ESP EA: 0.0241%</dd>
                                    <dd>ESP All: 0.0159%</dd>
                                    <dd>gnomAD_E_AFR: 0.0074%</dd>
                                    <dd>gnomAD_E_AMR: 0.0064%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0237%</dd>
                                    <dd>gnomAD_E_NFE: 0.0010%</dd>
                                    <dd>gnomAD_E_SAS: 0.0279%</dd>
                                    <dd>gnomAD_G_AFR: 0.0141%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.54773534C&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=687485">rs687485</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs687485&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.915</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000291633.7">KIR2DL1:ENST00000291633.7:c.272C&gt;A:p.(Thr91Lys)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336077.11">KIR2DL1:ENST00000336077.11:c.272C&gt;A:p.(Thr91Lys)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.921</dd>
                        
                        <dd>Polyphen2: 0.841 (P)</dd>
                        <dd>SIFT: 0.079 (T)</dd>
                        <dd>CADD: 0.629 (4.308)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>ExAC AFR: 0.0392%</dd>
                        <dd>ExAC NFE: 0.0015%</dd>
                        <dd>gnomAD_E_AFR: 0.0466%</dd>
                        <dd>gnomAD_E_NFE: 0.0018%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.54775344G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=147072532">rs147072532</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs147072532&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.873</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000291633.7">KIR2DL1:ENST00000291633.7:c.550G&gt;A:p.(Asp184Asn)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336077.11">KIR2DL1:ENST00000336077.11:c.550G&gt;A:p.(Asp184Asn)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.878</dd>
                        
                        <dd>Polyphen2: 0.014 (B)</dd>
                        <dd>SIFT: 0.122 (T)</dd>
                        <dd>CADD: 0.804 (7.088)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>1000Genomes: 0.0399%</dd>
                        <dd>TOPMed: 0.0399%</dd>
                        <dd>ExAC SAS: 0.0073%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.54773593G&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=687885">rs687885</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs687885&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.708</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000291633.7">KIR2DL1:ENST00000291633.7:c.331G&gt;T:p.(Val111Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336077.11">KIR2DL1:ENST00000336077.11:c.331G&gt;T:p.(Val111Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.71529174</dd>
                        
                        <dd>CADD: 0.715 (5.456)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_E_AFR: 0.0532%</dd>
                        <dd>gnomAD_E_AMR: 0.0031%</dd>
                        <dd>gnomAD_E_NFE: 0.0075%</dd>
                        <dd>gnomAD_E_SAS: 0.0067%</dd>
                        <dd>gnomAD_G_AFR: 0.0749%</dd>
                        <dd>gnomAD_G_FIN: 0.0597%</dd>
                        <dd>gnomAD_G_NFE: 0.0219%</dd>
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr19:g.54775370C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=28465191">rs28465191</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs28465191&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.003</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000291633.7">KIR2DL1:ENST00000291633.7:c.576C&gt;T:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336077.11">KIR2DL1:ENST00000336077.11:c.576C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.0034478903</dd>
                        
                        <dd>CADD: 0.003 (0.015)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>ExAC EAS: 0.0118%</dd>
                        <dd>ExAC NFE: 0.0016%</dd>
                        <dd>ExAC SAS: 0.0206%</dd>
                        <dd>gnomAD_E_EAS: 0.0062%</dd>
                        <dd>gnomAD_E_NFE: 0.0011%</dd>
                        <dd>gnomAD_E_SAS: 0.0588%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000215182">MUC5AC</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.701</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.482</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.482 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MUC5AC">MUC5AC</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0005253, abnormal eye physiology</dd><dd>HP:0000175, Cleft palate - MP:0003449, abnormal intestinal goblet cell morphology</dd><dd>HP:0000218, High palate - MP:0003449, abnormal intestinal goblet cell morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.701</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.482</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.1190234G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1247756839">rs1247756839</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1247756839&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.12089G&gt;A:p.(Arg4030Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9938341</dd>
                                    
                                    <dd>CADD: 0.994 (22.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1184723G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=878932837">rs878932837</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs878932837&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.966</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.6578G&gt;A:p.(Arg2193Gln)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.96603745</dd>
                        
                        <dd>CADD: 0.966 (14.690)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1182814G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=879153734">rs879153734</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs879153734&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.966</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.4669G&gt;A:p.(Ala1557Thr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9659592</dd>
                        
                        <dd>CADD: 0.966 (14.680)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1183604G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=878949904">rs878949904</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs878949904&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.963</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.5459G&gt;A:p.(Arg1820Gln)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.96335626</dd>
                        
                        <dd>CADD: 0.963 (14.360)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1188446C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1269243287">rs1269243287</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1269243287&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.959</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.10301C&gt;T:p.(Pro3434Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.95860004</dd>
                        
                        <dd>CADD: 0.959 (13.830)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1187320A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1222080419">rs1222080419</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1222080419&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.942</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.9175A&gt;G:p.(Ser3059Gly)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9421904</dd>
                        
                        <dd>CADD: 0.942 (12.380)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1183901C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1284670614">rs1284670614</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1284670614&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.940</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.5756C&gt;T:p.(Pro1919Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9400209</dd>
                        
                        <dd>CADD: 0.940 (12.220)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1183358T&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=879118927">rs879118927</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs879118927&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.911</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.5213T&gt;C:p.(Leu1738Pro)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9110799</dd>
                        
                        <dd>CADD: 0.911 (10.510)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1188859A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1361954623">rs1361954623</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1361954623&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.878</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.10714A&gt;G:p.(Lys3572Glu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8777075</dd>
                        
                        <dd>CADD: 0.878 (9.126)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1182851C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=878901958">rs878901958</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs878901958&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.835</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.4706C&gt;T:p.(Pro1569Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8345754</dd>
                        
                        <dd>CADD: 0.835 (7.814)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1184516C&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1201547484">rs1201547484</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1201547484&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.832</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.6371C&gt;A:p.(Thr2124Lys)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8315388</dd>
                        
                        <dd>CADD: 0.832 (7.735)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1185584C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1403452853">rs1403452853</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1403452853&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.819</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.7439C&gt;T:p.(Ala2480Val)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.81911606</dd>
                        
                        <dd>CADD: 0.819 (7.426)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1190666C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1248958955">rs1248958955</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1248958955&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.798</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.12521C&gt;T:p.(Thr4174Ile)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.79816335</dd>
                        
                        <dd>CADD: 0.798 (6.950)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1189033T&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1202505797">rs1202505797</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1202505797&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.778</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.10888T&gt;C:p.(Ser3630Pro)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.77828246</dd>
                        
                        <dd>CADD: 0.778 (6.542)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1183472G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1313976795">rs1313976795</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1313976795&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.758</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.5327G&gt;A:p.(Ser1776Asn)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.75778556</dd>
                        
                        <dd>CADD: 0.758 (6.158)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1183769A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=74196458">rs74196458</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs74196458&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.701</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.5624A&gt;G:p.(Gln1875Arg)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.7007046</dd>
                        
                        <dd>CADD: 0.701 (5.239)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1188357C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1466012270">rs1466012270</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1466012270&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.638</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.10212C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.637757</dd>
                        
                        <dd>CADD: 0.638 (4.410)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1183837C&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1379396838">rs1379396838</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1379396838&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.616</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.5692C&gt;A:p.(Pro1898Thr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.6162927</dd>
                        
                        <dd>CADD: 0.616 (4.160)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1190429C&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1163100172">rs1163100172</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1163100172&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.592</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.12284C&gt;G:p.(Thr4095Ser)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.59186864</dd>
                        
                        <dd>CADD: 0.592 (3.892)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1188729A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1161621512">rs1161621512</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1161621512&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.590</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.10584A&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.59045655</dd>
                        
                        <dd>CADD: 0.590 (3.877)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1188020G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1554928444">rs1554928444</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1554928444&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.588</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.9875G&gt;A:p.(Arg3292His)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.5879025</dd>
                        
                        <dd>CADD: 0.588 (3.850)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1188993T&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1245095901">rs1245095901</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1245095901&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.575</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.10848T&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.57469547</dd>
                        
                        <dd>CADD: 0.575 (3.713)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1186818T&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1350264670">rs1350264670</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1350264670&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.553</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.8673T&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.55311066</dd>
                        
                        <dd>CADD: 0.553 (3.498)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1191424C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1158446987">rs1158446987</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1158446987&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.379</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.13279C&gt;T:p.(Pro4427Ser)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.3792739</dd>
                        
                        <dd>CADD: 0.379 (2.071)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1191146T&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1194119815">rs1194119815</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1194119815&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.349</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.13001T&gt;C:p.(Ile4334Thr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.3485216</dd>
                        
                        <dd>CADD: 0.349 (1.861)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1183369A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=878859236">rs878859236</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs878859236&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.227</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.5224A&gt;G:p.(Asn1742Asp)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.22731942</dd>
                        
                        <dd>CADD: 0.227 (1.120)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1183926G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1422036505">rs1422036505</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1422036505&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.101</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.5781G&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.100916564</dd>
                        
                        <dd>CADD: 0.101 (0.462)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1184798A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1245151647">rs1245151647</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1245151647&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.090</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.6653A&gt;G:p.(Lys2218Arg)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.09008676</dd>
                        
                        <dd>CADD: 0.090 (0.410)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1184488A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1365388725">rs1365388725</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1365388725&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.013</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.6343A&gt;G:p.(Asn2115Asp)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.013266206</dd>
                        
                        <dd>CADD: 0.013 (0.058)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr11:g.1190684C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1330735919">rs1330735919</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1330735919&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.009</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.12539C&gt;T:p.(Ala4180Val)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.009168029</dd>
                        
                        <dd>CADD: 0.009 (0.040)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1182855G&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=879181541">rs879181541</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs879181541&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.005</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.4710G&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.0048237443</dd>
                        
                        <dd>CADD: 0.005 (0.021)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1189815C&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1328764977">rs1328764977</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1328764977&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.004</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.11670C&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.003906727</dd>
                        
                        <dd>CADD: 0.004 (0.017)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1190214G&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1379570399">rs1379570399</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1379570399&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.000</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.12069G&gt;C:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 4.6038628E-4</dd>
                        
                        <dd>CADD: 0.000 (0.002)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1184544C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1292150048">rs1292150048</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1292150048&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.000</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.6399C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 2.3025274E-4</dd>
                        
                        <dd>CADD: 0.000 (0.001)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr11:g.1188726C&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1384018709">rs1384018709</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1384018709&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.000</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621226.2">MUC5AC:ENST00000621226.2:c.10581C&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 2.3025274E-4</dd>
                        
                        <dd>CADD: 0.000 (0.001)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000123983">ACSL3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.692</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.968</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACSL3%0DACSL4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACSL4</a> and phenotypic similarity 0.645 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=777">X-linked non-syndromic intellectual disability</a> associated with ACSL4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001513, Obesity</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008587, Mild neurosensory hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000343, Long philtrum</dd><dd>HP:0000218, High palate - HP:0000343, Long philtrum</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACSL3%0DACSL4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACSL4</a> and phenotypic similarity 0.299 to mouse mutant of ACSL4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0011967, increased or absent threshold for auditory brainstem response</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.692</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.968</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.222908823C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=542262128">rs542262128</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs542262128&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.968</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357430.8">ACSL3:ENST00000357430.8:c.51C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392066.7">ACSL3:ENST00000392066.7:c.51C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9702833</dd>
                                    
                                    <dd>CADD: 0.970 (15.270)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0135%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0107%</dd>
                                    <dd>gnomAD_E_AMR: 0.0092%</dd>
                                    <dd>gnomAD_E_NFE: 0.0091%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000120802">TMPO</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.689</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.968</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.403 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=154">Familial isolated dilated cardiomyopathy</a> associated with TMPO.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Phenotypic similarity 0.228 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TMPO">TMPO</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0004842, abnormal large intestine crypts of Lieberkuhn morphology</dd><dd>HP:0000218, High palate - MP:0004842, abnormal large intestine crypts of Lieberkuhn morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TMPO%0DSMCHD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMCHD1</a> and phenotypic similarity 0.672 to <a href="http://www.omim.org/603457">Bosma arhinia microphthalmia syndrome</a> associated with SMCHD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=154">ORPHA:154</a> Familial isolated dilated cardiomyopathy</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.689</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.968</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr12:g.98533631T&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=561730840">rs561730840</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs561730840&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.968</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000266732.8">TMPO:ENST00000266732.8:c.1374T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000261210.9">TMPO:ENST00000261210.9:c.565+1793T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343315.9">TMPO:ENST00000343315.9:c.565+1793T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393053.6">TMPO:ENST00000393053.6:c.565+1793T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556029.5">TMPO:ENST00000556029.5:c.565+1793T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9702833</dd>
                                    
                                    <dd>CADD: 0.970 (15.270)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000126467">TSKS</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.688</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.967</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TSKS%0DPKDCC&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKDCC</a> and phenotypic similarity 0.652 to mouse mutant of PKDCC.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0000547, short limbs</dd><dd>HP:0011800, Midface retrusion - MP:0004540, small maxilla</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.688</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.967</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr19:g.49740189G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=534346167">rs534346167</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs534346167&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.967</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000246801.8">TSKS:ENST00000246801.8:c.1498-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358830.3">TSKS:ENST00000358830.3:c.898-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9745317</dd>
                                    
                                    <dd>CADD: 0.975 (15.940)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>ExAC AFR: 0.0100%</dd>
                                    <dd>ExAC AMR: 0.0174%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AMR: 0.0241%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0554%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171763">SPATA5L1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.684</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.965</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPATA5L1%0DRAB3GAP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAB3GAP1</a> and phenotypic similarity 0.637 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1387">Cataract-intellectual disability-hypogonadism syndrome</a> associated with RAB3GAP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.684</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.965</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr15:g.45402633G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=574924028">rs574924028</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs574924028&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.965</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000305560.11">SPATA5L1:ENST00000305560.11:c.204G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000638607.1">SPATA5L1:ENST00000638607.1:c.204G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96987</dd>
                                    
                                    <dd>CADD: 0.970 (15.210)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0399%</dd>
                                    <dd>gnomAD_E_AMR: 0.0070%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000131471">AOC3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.682</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.964</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=AOC3%0DGAD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GAD1</a> and phenotypic similarity 0.606 to mouse mutant of GAD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0030248, narrow maxilla</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0009890, cleft secondary palate</dd><dd>HP:0000218, High palate - MP:0009890, cleft secondary palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.682</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.964</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.42854704C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=574123445">rs574123445</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs574123445&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.964</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308423.7">AOC3:ENST00000308423.7:c.1857C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000591562.1">AOC3:ENST00000591562.1:c.228C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613571.1">AOC3:ENST00000613571.1:c.1857C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617500.4">AOC3:ENST00000617500.4:c.228C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96673405</dd>
                                    
                                    <dd>CADD: 0.967 (14.780)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_NFE: 0.0013%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198793">MTOR</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.672</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.613</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.831</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.613 to <a href="http://www.omim.org/616638">Smith-Kingsmore syndrome</a> associated with MTOR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001520, Large for gestational age</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000343, Long philtrum</dd><dd>HP:0000218, High palate - HP:0000343, Long philtrum</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MTOR%0DIGBP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IGBP1</a> and phenotypic similarity 0.612 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=52055">Corpus callosum agenesis-intellectual disability-coloboma-micrognathia syndrome</a> associated with IGBP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0000426, Prominent nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/607341">OMIM:607341</a> Focal cortical dysplasia, type II, somatic - somatic</dd>
<dd><a href="http://www.omim.org/entry/616638">OMIM:616638</a> Smith-Kingsmore syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=457485">ORPHA:457485</a> Macrocephaly-intellectual disability-neurodevelopmental disorder-small thorax syndrome</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.672</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.613</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.831</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.11253872G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=528524512">rs528524512</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs528524512&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    <span class="label label-info">LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=685542%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.831</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361445.9">MTOR:ENST00000361445.9:c.807C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8353734</dd>
                                    
                                    <dd>CADD: 0.835 (7.835)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0387%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000180376">CCDC66</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.668</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.472</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.472 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CCDC66">CCDC66</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0004022, abnormal cone electrophysiology</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.668</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.472</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.56619843T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=199936575">rs199936575</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs199936575&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000326595.11">CCDC66:ENST00000326595.11:c.2600T&gt;C:p.(Met867Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394672.8">CCDC66:ENST00000394672.8:c.2702T&gt;C:p.(Met901Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9945046</dd>
                                    
                                    <dd>Polyphen2: 0.018 (B)</dd>
                                    <dd>SIFT: 0.008 (D)</dd>
                                    <dd>CADD: 0.995 (22.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0048%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AMR: 0.0387%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000174130">TLR6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.668</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.956</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TLR6%0DCHUK&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CHUK</a> and phenotypic similarity 0.304 to <a href="http://www.omim.org/613630">Cocoon syndrome</a> associated with CHUK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0010808, Protruding tongue</dd><dd>HP:0000218, High palate - HP:0010808, Protruding tongue</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TLR6%0DCHUK&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CHUK</a> and phenotypic similarity 0.628 to mouse mutant of CHUK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0000547, short limbs</dd><dd>HP:0011800, Midface retrusion - MP:0005358, abnormal incisor morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0009890, cleft secondary palate</dd><dd>HP:0000218, High palate - MP:0009890, cleft secondary palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.668</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.956</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.38827201C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=55963748">rs55963748</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs55963748&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.956</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381950.1">TLR6:ENST00000381950.1:c.2273G&gt;A:p.(Arg758Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000436693.6">TLR6:ENST00000436693.6:c.2273G&gt;A:p.(Arg758Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000610323.2">TLR6:ENST00000610323.2:c.1325G&gt;A:p.(Arg442Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96419036</dd>
                                    
                                    <dd>Polyphen2: 0.023 (B)</dd>
                                    <dd>SIFT: 0.040 (D)</dd>
                                    <dd>CADD: 0.964 (14.460)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>ExAC SAS: 0.0606%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0520%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188779">SKOR1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.660</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.948</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SKOR1%0DTBX2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TBX2</a> and phenotypic similarity 0.580 to <a href="http://www.omim.org/618223">Vertebral anomalies and variable endocrine and T-cell dysfunction</a> associated with TBX2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0005280, Depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000189, Narrow palate</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SKOR1%0DTBX2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TBX2</a> and phenotypic similarity 0.621 to mouse mutant of TBX2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0009885, abnormal palatal shelf elevation</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0009890, cleft secondary palate</dd><dd>HP:0000218, High palate - MP:0009890, cleft secondary palate</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SKOR1%0DTBX2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TBX2</a> and phenotypic similarity 0.214 to fish mutant of TBX2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - ZP:0000993, abnormal(ly) decreased size inner ear</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.660</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.948</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_INSERTION</span>
                                <b>chr15:g.67827107A&gt;AGCG [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=771405818">rs771405818</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs771405818&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.948</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380035.3">SKOR1:ENST00000380035.3:c.1292_1294dup:p.(Gly431dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000554054.5">SKOR1:ENST00000554054.5:c.1208_1210dup:p.(Gly403dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000554240.5">SKOR1:ENST00000554240.5:c.1175_1177dup:p.(Gly392dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341418.10">SKOR1:ENST00000341418.10:c.1585+15_1585+16insGCG:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.94751924</dd>
                                    
                                    <dd>CADD: 0.948 (12.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000137766">UNC13C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.648</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.947</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=UNC13C%0DERC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ERC2</a> and phenotypic similarity 0.628 to mouse mutant of ERC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0002090, abnormal vision</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.648</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.947</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr15:g.54012972A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=543353315">rs543353315</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs543353315&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.947</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000260323.15">UNC13C:ENST00000260323.15:c.69A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000647821.1">UNC13C:ENST00000647821.1:c.69A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.94964993</dd>
                                    
                                    <dd>CADD: 0.950 (12.980)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136040">PLXNC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.641</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.941</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLXNC1%0DERF&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ERF</a> and phenotypic similarity 0.624 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=207">Crouzon disease</a> associated with ERF.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000405, Conductive hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000189, Narrow palate</dd><dd>HP:0000218, High palate - HP:0000189, Narrow palate</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLXNC1%0DERF&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ERF</a> and phenotypic similarity 0.455 to mouse mutant of ERF.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000443, abnormal snout morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000443, abnormal snout morphology</dd><dd>HP:0000218, High palate - MP:0000443, abnormal snout morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.641</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.941</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr12:g.94248366G&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=568450392">rs568450392</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs568450392&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.941</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258526.9">PLXNC1:ENST00000258526.9:c.2732G&gt;T:p.(Gly911Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9435063</dd>
                                    
                                    <dd>Polyphen2: 0.077 (B)</dd>
                                    <dd>SIFT: 0.203 (T)</dd>
                                    <dd>CADD: 0.944 (12.480)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000129566">TEP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.629</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.937</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TEP1%0DPPP2R5D&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PPP2R5D</a> and phenotypic similarity 0.659 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=457279">Intellectual disability-macrocephaly-hypotonia-behavioral abnormalities syndrome</a> associated with PPP2R5D.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0001583, Rotary nystagmus</dd><dd>HP:0000545, Myopia - HP:0000483, Astigmatism</dd><dd>HP:0000175, Cleft palate - HP:0000176, Submucous cleft hard palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.629</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.937</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.20404728G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=542282942">rs542282942</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs542282942&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.937</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262715.10">TEP1:ENST00000262715.10:c.915C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556935.5">TEP1:ENST00000556935.5:c.870+723C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9457999</dd>
                                    
                                    <dd>CADD: 0.946 (12.660)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0080%</dd>
                                    <dd>UK10K: 0.0264%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>ExAC AMR: 0.0346%</dd>
                                    <dd>ExAC NFE: 0.0076%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0686%</dd>
                                    <dd>gnomAD_E_NFE: 0.0063%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136045">PWP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.611</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.926</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PWP1%0DPAK1IP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PAK1IP1</a> and phenotypic similarity 0.643 to mouse mutant of PAK1IP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0030095, abnormal midface morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0009890, cleft secondary palate</dd><dd>HP:0000218, High palate - MP:0009890, cleft secondary palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.611</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.926</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr12:g.107704646T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=532316838">rs532316838</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs532316838&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.926</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000412830.8">PWP1:ENST00000412830.8:c.976T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541166.1">PWP1:ENST00000541166.1:c.790T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9287147</dd>
                                    
                                    <dd>CADD: 0.929 (11.470)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000175311">ANKS4B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.553</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.904</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANKS4B%0DDNAJC5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DNAJC5</a> and phenotypic similarity 0.693 to mouse mutant of DNAJC5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0006325, impaired hearing</dd><dd>HP:0000545, Myopia - MP:0002090, abnormal vision</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.553</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.904</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr16:g.21233821G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=376343890">rs376343890</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs376343890&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.904</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311620.7">ANKS4B:ENST00000311620.7:c.84G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.90645945</dd>
                                    
                                    <dd>CADD: 0.906 (10.290)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0090%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000060237">WNK1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.549</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.902</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=WNK1%0DKLHL7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KLHL7</a> and phenotypic similarity 0.656 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=97297">Bohring-Opitz syndrome</a> associated with KLHL7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0000293, Full cheeks</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000175, Cleft palate</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/201300">OMIM:201300</a> Neuropathy, hereditary sensory and autonomic, type II - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/614492">OMIM:614492</a> Pseudohypoaldosteronism, type IIC - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=970">ORPHA:970</a> Hereditary sensory and autonomic neuropathy type 2</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.549</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.902</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr12:g.882055A&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=549781052">rs549781052</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs549781052&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=326914%5Balleleid%5D">(criteria_provided,_conflicting_interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.902</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000315939.11">WNK1:ENST00000315939.11:c.3354A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340908.8">WNK1:ENST00000340908.8:c.4110A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530271.6">WNK1:ENST00000530271.6:c.4593A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000535572.5">WNK1:ENST00000535572.5:c.2613A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537687.5">WNK1:ENST00000537687.5:c.4134A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9045007</dd>
                                    
                                    <dd>CADD: 0.905 (10.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0020%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000073614">KDM5A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.520</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.887</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KDM5A%0DEHMT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EHMT1</a> and phenotypic similarity 0.652 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=96147">Kleefstra syndrome due to 9q34 microdeletion</a> associated with EHMT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001513, Obesity</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0010808, Protruding tongue</dd><dd>HP:0000218, High palate - HP:0010808, Protruding tongue</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KDM5A%0DEHMT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EHMT1</a> and phenotypic similarity 0.428 to mouse mutant of EHMT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0003842, abnormal metopic suture morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000120, malocclusion</dd><dd>HP:0000218, High palate - MP:0000120, malocclusion</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.520</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.887</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr12:g.323210C&gt;CA [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=60377454">rs60377454</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs60377454&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.887</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399788.7">KDM5A:ENST00000399788.7:c.2151-5_2151-4insT:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8871764</dd>
                                    
                                    <dd>CADD: 0.887 (9.476)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000007174">DNAH9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.515</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.887</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.310 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with DNAH9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000405, Conductive hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAH9%0DPAFAH1B1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PAFAH1B1</a> and phenotypic similarity 0.661 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=217385">17p13.3 microduplication syndrome</a> associated with PAFAH1B1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0000098, Tall stature</dd><dd>HP:0011800, Midface retrusion - HP:0000218, High palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/618300">OMIM:618300</a> Ciliary dyskinesia, primary, 40 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">ORPHA:244</a> Primary ciliary dyskinesia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.515</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.887</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.11763608A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=539397342">rs539397342</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs539397342&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.887</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262442.9">DNAH9:ENST00000262442.9:c.7164A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454412.6">DNAH9:ENST00000454412.6:c.7164A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.88933766</dd>
                                    
                                    <dd>CADD: 0.889 (9.560)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000115590">IL1R2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.459</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.862</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.299 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=IL1R2">IL1R2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0011967, increased or absent threshold for auditory brainstem response</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=IL1R2%0DCASP3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CASP3</a> and phenotypic similarity 0.604 to mouse mutant of CASP3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000111, cleft palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.459</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.862</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.102009638G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=543195065">rs543195065</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs543195065&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.862</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000332549.8">IL1R2:ENST00000332549.8:c.144G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393414.6">IL1R2:ENST00000393414.6:c.144G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441002.1">IL1R2:ENST00000441002.1:c.144G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8646058</dd>
                                    
                                    <dd>CADD: 0.865 (8.684)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000132382">MYBBP1A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.453</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.860</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MYBBP1A%0DPOLR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLR1A</a> and phenotypic similarity 0.633 to <a href="http://www.omim.org/616462">Acrofacial dysostosis, Cincinnati type</a> associated with POLR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000175, Cleft palate</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=MYBBP1A%0DPOLR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLR1A</a> and phenotypic similarity 0.381 to fish mutant of POLR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - ZP:0002314, abnormal(ly) shape head</dd><dd>HP:0008625, Severe sensorineural hearing impairment - ZP:0000993, abnormal(ly) decreased size inner ear</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.453</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.860</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.4545709C&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=202167625">rs202167625</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs202167625&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.860</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000254718.8">MYBBP1A:ENST00000254718.8:c.1974G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381556.6">MYBBP1A:ENST00000381556.6:c.1974G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.86686265</dd>
                                    
                                    <dd>CADD: 0.867 (8.757)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0080%</dd>
                                    <dd>ExAC AMR: 0.0187%</dd>
                                    <dd>ExAC NFE: 0.0083%</dd>
                                    <dd>gnomAD_E_AFR: 0.0067%</dd>
                                    <dd>gnomAD_E_AMR: 0.0270%</dd>
                                    <dd>gnomAD_E_NFE: 0.0128%</dd>
                                    <dd>gnomAD_E_OTH: 0.0373%</dd>
                                    <dd>gnomAD_G_NFE: 0.0134%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000164291">ARSK</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.348</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.813</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARSK%0DIDUA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IDUA</a> and phenotypic similarity 0.544 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93473">Hurler syndrome</a> associated with IDUA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0000293, Full cheeks</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000158, Macroglossia</dd><dd>HP:0000218, High palate - HP:0000158, Macroglossia</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARSK%0DIDUA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IDUA</a> and phenotypic similarity 0.712 to mouse mutant of IDUA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0004469, abnormal zygomatic arch morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0004740, sensorineural hearing loss</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0004595, abnormal mandibular condyloid process morphology</dd><dd>HP:0000218, High palate - MP:0004595, abnormal mandibular condyloid process morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.348</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.813</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr5:g.95591471T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=536220801">rs536220801</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs536220801&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.813</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380009.9">ARSK:ENST00000380009.9:c.942T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8148601</dd>
                                    
                                    <dd>CADD: 0.815 (7.325)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143933">CALM2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.338</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.219 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3286">Catecholaminergic polymorphic ventricular tachycardia</a> associated with CALM2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0002321, Vertigo</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CALM2%0DCNGA3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CNGA3</a> and phenotypic similarity 0.440 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=49382">Achromatopsia</a> associated with CNGA3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0012043, Pendular nystagmus</dd><dd>HP:0000545, Myopia - HP:0000540, Hypermetropia</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CALM2%0DCNGA3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CNGA3</a> and phenotypic similarity 0.614 to mouse mutant of CNGA3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0011831, abnormal visual contrast sensitivity</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/616249">OMIM:616249</a> Long QT syndrome 15 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=101016">ORPHA:101016</a> Romano-Ward syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=130">ORPHA:130</a> Brugada syndrome</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3286">ORPHA:3286</a> Catecholaminergic polymorphic ventricular tachycardia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.338</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr2:g.47167693CAAAAAAAAAAA&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=58897204">rs58897204</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs58897204&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409563.5">CALM2:ENST00000409563.5:c.-40-14_-40-4del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000272298.12">CALM2:ENST00000272298.12:c.34+3030_34+3040del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000628793.2">CALM2:ENST00000628793.2:c.34+3030_34+3040del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000654798.1">CALM2:ENST00000654798.1:c.34+3030_34+3040del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422269.1">CALM2:ENST00000422269.1:c.102+3030_102+3040del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456319.6">CALM2:ENST00000456319.6:c.-75+3030_-75+3040del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000655450.1">CALM2:ENST00000655450.1:c.-75+3030_-75+3040del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000655728.1">CALM2:ENST00000655728.1:c.-75+3030_-75+3040del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000656538.1">CALM2:ENST00000656538.1:c.-75+3030_-75+3040del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000668667.1">CALM2:ENST00000668667.1:c.-75+3030_-75+3040del:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.24785775</dd>
                                    
                                    <dd>CADD: 0.248 (1.237)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000174796">THAP6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.330</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.804</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=THAP6%0DIRF6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRF6</a> and phenotypic similarity 0.551 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=199306">Cleft lip/palate</a> associated with IRF6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0000220, Velopharyngeal insufficiency</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000405, Conductive hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0100334, Unilateral cleft palate</dd><dd>HP:0000218, High palate - HP:0000220, Velopharyngeal insufficiency</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=THAP6%0DIRF6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRF6</a> and phenotypic similarity 0.688 to mouse mutant of IRF6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0000547, short limbs</dd><dd>HP:0011800, Midface retrusion - MP:0030188, round snout</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0005579, absent outer ear</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.330</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.804</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr4:g.75521783C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=568252076">rs568252076</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs568252076&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.804</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000311638.7">THAP6:ENST00000311638.7:c.336C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000507556.5">THAP6:ENST00000507556.5:c.336C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000507885.5">THAP6:ENST00000507885.5:c.213C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000508105.5">THAP6:ENST00000508105.5:c.213C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000514480.1">THAP6:ENST00000514480.1:c.336C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380837.7">THAP6:ENST00000380837.7:c.288+4804C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000502620.1">THAP6:ENST00000502620.1:c.165+4804C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000504190.5">THAP6:ENST00000504190.5:c.165+4804C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000507557.5">THAP6:ENST00000507557.5:c.165+4804C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8061794</dd>
                                    
                                    <dd>CADD: 0.806 (7.126)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000132746">ALDH3B2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.324</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.798</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALDH3B2%0DGAD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GAD1</a> and phenotypic similarity 0.606 to mouse mutant of GAD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0030248, narrow maxilla</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0009890, cleft secondary palate</dd><dd>HP:0000218, High palate - MP:0009890, cleft secondary palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.324</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.798</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr11:g.67666115C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=373409168">rs373409168</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs373409168&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.798</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349015.7">ALDH3B2:ENST00000349015.7:c.319+7G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530069.6">ALDH3B2:ENST00000530069.6:c.319+7G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000673966.1">ALDH3B2:ENST00000673966.1:c.319+7G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.2740612</dd>
                                    
                                    <dd>CADD: 0.274 (1.391)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC NFE: 0.0090%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0081%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000243772">KIR2DL3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.290</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.783</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIR2DL3%0DPTPN11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTPN11</a> and phenotypic similarity 0.814 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=500">Noonan syndrome with multiple lentigines</a> associated with PTPN11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0000431, Wide nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008625, Severe sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000431, Wide nasal bridge</dd><dd>HP:0000218, High palate - HP:0000431, Wide nasal bridge</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIR2DL3%0DPTPN11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTPN11</a> and phenotypic similarity 0.508 to mouse mutant of PTPN11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0030034, depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0000042, abnormal organ of Corti morphology</dd><dd>HP:0000545, Myopia - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000175, Cleft palate - MP:0030034, depressed nasal bridge</dd><dd>HP:0000218, High palate - MP:0030034, depressed nasal bridge</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=KIR2DL3%0DPTPN11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PTPN11</a> and phenotypic similarity 0.195 to fish mutant of PTPN11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - ZP:0004159, abnormal(ly) position otolith</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.290</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.783</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr19:g.54743789C&gt;CT [1/1:0/1:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=762937109">rs762937109</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs762937109&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.783</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000342376.4">KIR2DL3:ENST00000342376.4:c.371-6_371-5insT:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7509423</dd>
                                    
                                    <dd>CADD: 0.751 (6.037)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_G_AFR: 0.1476%</dd>
                                    <dd>gnomAD_G_NFE: 0.0082%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000138375">SMARCAL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.274</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.304</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.607 to <a href="http://www.omim.org/242900">Schimke immunoosseous dysplasia</a> associated with SMARCAL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0003521, Disproportionate short-trunk short stature</dd><dd>HP:0011800, Midface retrusion - HP:0005280, Depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0005280, Depressed nasal bridge</dd><dd>HP:0000218, High palate - HP:0005280, Depressed nasal bridge</dd></dl><dl><dt>Phenotypic similarity 0.299 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=SMARCAL1">SMARCAL1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - ZP:0000026, abnormal(ly) decreased length trunk</dd><dd>HP:0011800, Midface retrusion - ZP:0000062, abnormal(ly) decreased size palatoquadrate cartilage</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - ZP:0000062, abnormal(ly) decreased size palatoquadrate cartilage</dd><dd>HP:0000218, High palate - ZP:0000062, abnormal(ly) decreased size palatoquadrate cartilage</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SMARCAL1%0DTCF20&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TCF20</a> and phenotypic similarity 0.708 to <a href="http://www.omim.org/618430">Developmental delay with variable intellectual impairment and behavioral abnormalities</a> associated with TCF20.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0000098, Tall stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0010804, Tented upper lip vermilion</dd><dd>HP:0000218, High palate - HP:0010804, Tented upper lip vermilion</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/242900">OMIM:242900</a> Schimke immunoosseous dysplasia - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1830">ORPHA:1830</a> Schimke immuno-osseous dysplasia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.274</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.304</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.216482811T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=572658323">rs572658323</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs572658323&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357276.9">SMARCAL1:ENST00000357276.9:c.2699T&gt;C:p.(Leu900Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358207.9">SMARCAL1:ENST00000358207.9:c.2699T&gt;C:p.(Leu900Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.093 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.002 (D)</dd>
                                    <dd>CADD: 0.998 (26.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188306">LRRIQ4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.261</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.767</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRRIQ4%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.623 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=950">Acrodysostosis</a> associated with PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0005280, Depressed nasal bridge</dd><dd>HP:0000218, High palate - HP:0005280, Depressed nasal bridge</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRRIQ4%0DPRKAR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1A</a> and phenotypic similarity 0.258 to mouse mutant of PRKAR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000474, abnormal foregut morphology</dd><dd>HP:0000218, High palate - MP:0000474, abnormal foregut morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.261</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.767</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr3:g.169828833G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=535804958">rs535804958</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs535804958&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.767</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340806.6">LRRIQ4:ENST00000340806.6:c.1095G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7756118</dd>
                                    
                                    <dd>CADD: 0.776 (6.490)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0223%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ExAC AFR: 0.0102%</dd>
                                    <dd>ExAC AMR: 0.0432%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0775%</dd>
                                    <dd>gnomAD_E_NFE: 0.0161%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169894">MUC3A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.261</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.317</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.975</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.317 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MUC3A">MUC3A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0001284, absent vibrissae</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0001284, absent vibrissae</dd><dd>HP:0000218, High palate - MP:0001284, absent vibrissae</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.261</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.317</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.975</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.100965345C&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=73714276">rs73714276</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs73714276&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.975</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.9446C&gt;A:p.(Ala3149Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.9446C&gt;A:p.(Ala3149Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9745903</dd>
                                    
                                    <dd>CADD: 0.975 (15.950)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                    <b>chr7:g.100965350A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=75592954">rs75592954</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs75592954&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.964</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.9448+3A&gt;G:p.?</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.9448+3A&gt;G:p.?</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9644369</dd>
                        
                        <dd>CADD: 0.964 (14.490)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr7:g.100960786C&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2904852">rs2904852</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2904852&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.602</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.8901C&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000483366.5">MUC3A:ENST00000483366.5:c.8901C&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.6018928</dd>
                        
                        <dd>CADD: 0.602 (4.000)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100997">ABHD12</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.467</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.411 to <a href="http://www.omim.org/612674">Polyneuropathy, hearing loss, ataxia, retinitis pigmentosa, and cataract</a> associated with ABHD12.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Phenotypic similarity 0.467 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ABHD12">ABHD12</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0006325, impaired hearing</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/612674">OMIM:612674</a> Polyneuropathy, hearing loss, ataxia, retinitis pigmentosa, and cataract - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.252</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.467</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr20:g.25342103CA&gt;C [1/1:0/1:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=11476197">rs11476197</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs11476197&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000672566.1">ABHD12:ENST00000672566.1:c.-431-8del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339157.10">ABHD12:ENST00000339157.10:c.192-2753del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376542.8">ABHD12:ENST00000376542.8:c.192-2753del:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000673121.1">ABHD12:ENST00000673121.1:c.-280-2753del:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.054198503</dd>
                                    
                                    <dd>CADD: 0.054 (0.242)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136861">CDK5RAP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.180</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.403 to <a href="http://www.omim.org/604804">Microcephaly 3, primary, autosomal recessive</a> associated with CDK5RAP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Phenotypic similarity 0.303 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CDK5RAP2">CDK5RAP2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0009859, eye opacity</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CDK5RAP2%0DOFD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OFD1</a> and phenotypic similarity 0.610 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2754">Orofaciodigital syndrome type 6</a> associated with OFD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0000276, Long face</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000405, Conductive hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CDK5RAP2%0DOFD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OFD1</a> and phenotypic similarity 0.235 to mouse mutant of OFD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0004355, short radius</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CDK5RAP2%0DOFD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OFD1</a> and phenotypic similarity 0.207 to fish mutant of OFD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - ZP:0004021, abnormal(ly) present in greater numbers in organism otolith</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/604804">OMIM:604804</a> Microcephaly 3, primary, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2512">ORPHA:2512</a> Autosomal recessive primary microcephaly</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.180</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.120415090T&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=539439151">rs539439151</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs539439151&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349780.9">CDK5RAP2:ENST00000349780.9:c.4247A&gt;C:p.(Asn1416Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360190.8">CDK5RAP2:ENST00000360190.8:c.4247A&gt;C:p.(Asn1416Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360822.7">CDK5RAP2:ENST00000360822.7:c.3557A&gt;C:p.(Asn1186Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.999 (D)</dd>
                                    <dd>Mutation Taster: 0.967 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.998 (26.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000149925">ALDOA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.180</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.275 to <a href="http://www.omim.org/611881">Glycogen storage disease XII</a> associated with ALDOA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALDOA%0DTGFB3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TGFB3</a> and phenotypic similarity 0.692 to <a href="http://www.omim.org/615582">Loeys-Dietz syndrome 5</a> associated with TGFB3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0000098, Tall stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000193, Bifid uvula</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALDOA%0DTGFB3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TGFB3</a> and phenotypic similarity 0.633 to mouse mutant of TGFB3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0002239, abnormal nasal septum morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALDOA%0DTGFB3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TGFB3</a> and phenotypic similarity 0.288 to fish mutant of TGFB3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - ZP:0005170, abnormal(ly) ossified entopterygoid</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - ZP:0005170, abnormal(ly) ossified entopterygoid</dd><dd>HP:0000218, High palate - ZP:0005170, abnormal(ly) ossified entopterygoid</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/611881">OMIM:611881</a> Glycogen storage disease XII - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.180</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr16:g.30067000C&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=567803172">rs567803172</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs567803172&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000642816.2">ALDOA:ENST00000642816.2:c.103C&gt;G:p.(Gln35Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395248.6">ALDOA:ENST00000395248.6:c.*123C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395240.7">ALDOA:ENST00000395240.7:c.-21-234C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000412304.6">ALDOA:ENST00000412304.6:c.-21-234C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000563060.6">ALDOA:ENST00000563060.6:c.-21-234C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000569545.5">ALDOA:ENST00000569545.5:c.-21-234C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000569798.5">ALDOA:ENST00000569798.5:c.-21-234C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000643777.2">ALDOA:ENST00000643777.2:c.-21-234C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000627059.2">ALDOA:ENST00000627059.2:c.-21-234C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000338110.10">ALDOA:ENST00000338110.10:c.*489-234C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000566897.6">ALDOA:ENST00000566897.6:c.*489-234C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568435.6">ALDOA:ENST00000568435.6:c.*489-234C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.620 (T)</dd>
                                    <dd>CADD: 0.986 (18.420)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0132%</dd>
                                    <dd>gnomAD_E_NFE: 0.0015%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000115665">SLC5A7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.179</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.582</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.622</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.582 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98914">Presynaptic congenital myasthenic syndromes</a> associated with SLC5A7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0000276, Long face</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Phenotypic similarity 0.394 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SLC5A7">SLC5A7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0002092, abnormal eye morphology</dd><dd>HP:0000175, Cleft palate - MP:0000496, abnormal small intestine morphology</dd><dd>HP:0000218, High palate - MP:0000496, abnormal small intestine morphology</dd></dl><dl><dt>Proximity score 0.519 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC5A7%0DMN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MN1</a> and phenotypic similarity 0.650 to <a href="http://www.omim.org/618774">CEBALID syndrome</a> associated with MN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Proximity score 0.519 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC5A7%0DMN1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MN1</a> and phenotypic similarity 0.654 to mouse mutant of MN1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000109, abnormal parietal bone morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0009890, cleft secondary palate</dd><dd>HP:0000218, High palate - MP:0009890, cleft secondary palate</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/158580">OMIM:158580</a> Neuronopathy, distal hereditary motor, type VIIA - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/617143">OMIM:617143</a> Myasthenic syndrome, congenital, 20, presynaptic - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98914">ORPHA:98914</a> Presynaptic congenital myasthenic syndromes</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.179</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.582</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.622</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr2:g.107997929G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=574474443">rs574474443</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs574474443&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.622</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264047.3">SLC5A7:ENST00000264047.3:c.540G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409059.5">SLC5A7:ENST00000409059.5:c.540G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.62372965</dd>
                                    
                                    <dd>CADD: 0.624 (4.245)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000113262">GRM6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.178</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.258</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.480 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=215">Congenital stationary night blindness</a> associated with GRM6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000639, Nystagmus</dd><dd>HP:0000545, Myopia - HP:0011003, High myopia</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Phenotypic similarity 0.464 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GRM6">GRM6</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0005253, abnormal eye physiology</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.515 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GRM6%0DNYX&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NYX</a> and phenotypic similarity 0.507 to <a href="http://www.omim.org/310500">Night blindness, congenital stationary (complete), 1A, X-linked</a> associated with NYX.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.515 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=GRM6%0DNYX&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NYX</a> and phenotypic similarity 0.642 to mouse mutant of NYX.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0002090, abnormal vision</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/257270">OMIM:257270</a> Night blindness, congenital stationary (complete), 1B, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=215">ORPHA:215</a> Congenital stationary night blindness</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.178</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.258</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr5:g.178986674C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=149564055">rs149564055</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs149564055&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000231188.9">GRM6:ENST00000231188.9:c.1580G&gt;A:p.(Arg527Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000517717.3">GRM6:ENST00000517717.3:c.1580G&gt;A:p.(Arg527Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000650031.1">GRM6:ENST00000650031.1:c.1580G&gt;A:p.(Arg527Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999811</dd>
                                    
                                    <dd>Polyphen2: 0.913 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.066 (T)</dd>
                                    <dd>CADD: 0.998 (27.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0104%</dd>
                                    <dd>ExAC AMR: 0.0262%</dd>
                                    <dd>ExAC NFE: 0.0094%</dd>
                                    <dd>ExAC SAS: 0.0182%</dd>
                                    <dd>gnomAD_E_AMR: 0.0298%</dd>
                                    <dd>gnomAD_E_NFE: 0.0100%</dd>
                                    <dd>gnomAD_E_OTH: 0.0185%</dd>
                                    <dd>gnomAD_E_SAS: 0.0130%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0134%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000044446">PHKA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.177</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.275 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=264580">Glycogen storage disease due to liver phosphorylase kinase deficiency</a> associated with PHKA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=PHKA2%0DERCC3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ERCC3</a> and phenotypic similarity 0.646 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=33364">Trichothiodystrophy</a> associated with ERCC3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0002705, High, narrow palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008619, Bilateral sensorineural hearing impairment</dd><dd>HP:0000545, Myopia - HP:0000545, Myopia</dd><dd>HP:0000175, Cleft palate - HP:0002705, High, narrow palate</dd><dd>HP:0000218, High palate - HP:0002705, High, narrow palate</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/306000">OMIM:306000</a> Glycogen storage disease, type IXa2 - X-linked recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=264580">ORPHA:264580</a> Glycogen storage disease due to liver phosphorylase kinase deficiency</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.177</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.18906777C&gt;T [0/1:0/0:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=761309363">rs761309363</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs761309363&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379942.5">PHKA2:ENST00000379942.5:c.2635G&gt;A:p.(Glu879Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.435 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.059 (D)</dd>
                                    <dd>CADD: 0.994 (22.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0265%</dd>
                                    <dd>TOPMed: 0.0265%</dd>
                                    <dd>ExAC NFE: 0.0042%</dd>
                                    <dd>gnomAD_E_NFE: 0.0025%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000107611">CUBN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.173</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUBN%0DLRP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRP2</a> and phenotypic similarity 0.674 to <a href="http://www.omim.org/222448">Donnai-Barrow syndrome</a> associated with LRP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia - HP:0011003, High myopia</dd><dd>HP:0000175, Cleft palate - HP:0005280, Depressed nasal bridge</dd><dd>HP:0000218, High palate - HP:0005280, Depressed nasal bridge</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUBN%0DLRP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRP2</a> and phenotypic similarity 0.643 to mouse mutant of LRP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000111, cleft palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0002092, abnormal eye morphology</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUBN%0DLRP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRP2</a> and phenotypic similarity 0.401 to fish mutant of LRP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - ZP:0010561, abnormal(ly) process quality visual perception</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/261100">OMIM:261100</a> Imerslund-Grasbeck syndrome 1 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.173</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.16937678G&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=147617753">rs147617753</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs147617753&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=314885%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377833.10">CUBN:ENST00000377833.10:c.5840C&gt;A:p.(Ser1947Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.998</dd>
                                    
                                    <dd>Polyphen2: 0.995 (D)</dd>
                                    <dd>SIFT: 0.002 (D)</dd>
                                    <dd>CADD: 0.998 (26.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0295%</dd>
                                    <dd>UK10K: 0.0397%</dd>
                                    <dd>ESP EA: 0.0465%</dd>
                                    <dd>ESP All: 0.0308%</dd>
                                    <dd>ExAC AMR: 0.0347%</dd>
                                    <dd>ExAC FIN: 0.0151%</dd>
                                    <dd>ExAC NFE: 0.0465%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0358%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0440%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000119535">CSF3R</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.171</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.992</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSF3R%0DCBL&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CBL</a> and phenotypic similarity 0.676 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=648">Noonan syndrome</a> associated with CBL.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSF3R%0DCBL&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CBL</a> and phenotypic similarity 0.367 to mouse mutant of CBL.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0001260, increased body weight</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617014">OMIM:617014</a> Neutropenia, severe congenital, 7, autosomal recessive - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.171</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.992</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.36466434C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=528303671">rs528303671</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs528303671&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.992</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361632.8">CSF3R:ENST00000361632.8:c.2434G&gt;A:p.(Val812Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373103.5">CSF3R:ENST00000373103.5:c.2515G&gt;A:p.(Val839Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373106.5">CSF3R:ENST00000373106.5:c.2434G&gt;A:p.(Val812Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000331941.6">CSF3R:ENST00000331941.6:c.2247+187G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373104.5">CSF3R:ENST00000373104.5:c.2247+187G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 0.003 (B)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.608 (T)</dd>
                                    <dd>CADD: 0.979 (16.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0143%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_AMR: 0.0598%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0369%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000119977">TCTN3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.169</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.305</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.928</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.610 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2754">Orofaciodigital syndrome type 6</a> associated with TCTN3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0000276, Long face</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000405, Conductive hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Phenotypic similarity 0.298 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TCTN3">TCTN3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0001303, abnormal lens morphology</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/258860">OMIM:258860</a> Orofaciodigital syndrome IV - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/614815">OMIM:614815</a> Joubert syndrome 18 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2753">ORPHA:2753</a> Orofaciodigital syndrome type 4</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2754">ORPHA:2754</a> Orofaciodigital syndrome type 6</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.169</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.305</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.928</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr10:g.95684595A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=549194555">rs549194555</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs549194555&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.928</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265993.13">TCTN3:ENST00000265993.13:c.1053T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371209.5">TCTN3:ENST00000371209.5:c.999T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371217.10">TCTN3:ENST00000371217.10:c.999T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614499.4">TCTN3:ENST00000614499.4:c.999T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430368.6">TCTN3:ENST00000430368.6:c.652-966T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9308169</dd>
                                    
                                    <dd>CADD: 0.931 (11.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000109906">ZBTB16</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.169</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.990</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.334 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=520">Acute promyelocytic leukemia</a> associated with ZBTB16.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0002321, Vertigo</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000212, Gingival overgrowth</dd><dd>HP:0000218, High palate - HP:0000212, Gingival overgrowth</dd></dl><dl><dt>Phenotypic similarity 0.311 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ZBTB16">ZBTB16</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0000547, short limbs</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZBTB16%0DHDAC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HDAC4</a> and phenotypic similarity 0.635 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1001">2q37 microdeletion syndrome</a> associated with HDAC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001513, Obesity</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000405, Conductive hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0005280, Depressed nasal bridge</dd><dd>HP:0000218, High palate - HP:0005280, Depressed nasal bridge</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZBTB16%0DHDAC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HDAC4</a> and phenotypic similarity 0.403 to mouse mutant of HDAC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000440, domed cranium</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZBTB16%0DHDAC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HDAC4</a> and phenotypic similarity 0.247 to fish mutant of HDAC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - ZP:0008435, abnormal(ly) shortened ethmoid cartilage</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/612447">OMIM:612447</a> Skeletal defects, genital hypoplasia, and mental retardation - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=520">ORPHA:520</a> Acute promyelocytic leukemia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.169</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.990</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr11:g.114250366C&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=532283203">rs532283203</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs532283203&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.990</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335953.9">ZBTB16:ENST00000335953.9:c.1833C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392996.2">ZBTB16:ENST00000392996.2:c.1833C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99241424</dd>
                                    
                                    <dd>CADD: 0.992 (21.200)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000153113">CAST</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.161</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.983</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CAST%0DACTA1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTA1</a> and phenotypic similarity 0.620 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=171439">Childhood-onset nemaline myopathy</a> associated with ACTA1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001533, Slender build</dd><dd>HP:0011800, Midface retrusion - HP:0000275, Narrow face</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=CAST%0DACTA1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ACTA1</a> and phenotypic similarity 0.489 to mouse mutant of ACTA1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0030140, facial muscle atrophy</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0001344, blepharoptosis</dd><dd>HP:0000175, Cleft palate - MP:0030140, facial muscle atrophy</dd><dd>HP:0000218, High palate - MP:0030140, facial muscle atrophy</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/616295">OMIM:616295</a> Peeling skin with leukonychia, acral punctate keratoses, cheilitis, and knuckle pads - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.161</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.983</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr5:g.96695891C&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=532536058">rs532536058</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs532536058&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.983</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395812.6">CAST:ENST00000395812.6:c.194C&gt;G:p.(Thr65Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000508608.5">CAST:ENST00000508608.5:c.149C&gt;G:p.(Thr50Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000508830.5">CAST:ENST00000508830.5:c.194C&gt;G:p.(Thr65Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000510756.5">CAST:ENST00000510756.5:c.194C&gt;G:p.(Thr65Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000338252.7">CAST:ENST00000338252.7:c.-56C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341926.7">CAST:ENST00000341926.7:c.-56C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.986</dd>
                                    
                                    <dd>Polyphen2: 0.731 (P)</dd>
                                    <dd>SIFT: 0.014 (D)</dd>
                                    <dd>CADD: 0.932 (11.660)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_AMR: 0.0090%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130402">ACTN4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.155</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.694</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.478 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ACTN4">ACTN4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000445, short snout</dd><dd>HP:0008625, Severe sensorineural hearing impairment - MP:0004666, absent stapedial artery</dd><dd>HP:0000545, Myopia - MP:0004269, abnormal optic cup morphology</dd><dd>HP:0000175, Cleft palate - MP:0000445, short snout</dd><dd>HP:0000218, High palate - MP:0000445, short snout</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACTN4%0DTGFB3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TGFB3</a> and phenotypic similarity 0.692 to <a href="http://www.omim.org/615582">Loeys-Dietz syndrome 5</a> associated with TGFB3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0000098, Tall stature</dd><dd>HP:0011800, Midface retrusion - HP:0011800, Midface retrusion</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000193, Bifid uvula</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACTN4%0DTGFB3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TGFB3</a> and phenotypic similarity 0.633 to mouse mutant of TGFB3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0002239, abnormal nasal septum morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACTN4%0DTGFB3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TGFB3</a> and phenotypic similarity 0.288 to fish mutant of TGFB3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - ZP:0005170, abnormal(ly) ossified entopterygoid</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - ZP:0005170, abnormal(ly) ossified entopterygoid</dd><dd>HP:0000218, High palate - ZP:0005170, abnormal(ly) ossified entopterygoid</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/603278">OMIM:603278</a> Glomerulosclerosis, focal segmental, 1 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.155</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.694</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.38729097C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=372396071">rs372396071</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs372396071&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.694</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000252699.7">ACTN4:ENST00000252699.7:c.2520C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390009.7">ACTN4:ENST00000390009.7:c.1863C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424234.6">ACTN4:ENST00000424234.6:c.1350C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.697657</dd>
                                    
                                    <dd>CADD: 0.698 (5.195)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0175%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>ExAC NFE: 0.0243%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0206%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000133863">TEX15</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.137</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.964</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=TEX15%0DRAX&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAX</a> and phenotypic similarity 0.662 to mouse mutant of RAX.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0000111, cleft palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0001286, abnormal eye development</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617960">OMIM:617960</a> Spermatogenic failure 25 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=399805">ORPHA:399805</a> Male infertility with azoospermia or oligozoospermia due to single gene mutation</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.137</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.964</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr8:g.30837922T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=139095018">rs139095018</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs139095018&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.964</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000256246.5">TEX15:ENST00000256246.5:c.7213A&gt;G:p.(Lys2405Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000638951.1">TEX15:ENST00000638951.1:c.8374A&gt;G:p.(Lys2792Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000643185.1">TEX15:ENST00000643185.1:c.8362A&gt;G:p.(Lys2788Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96973085</dd>
                                    
                                    <dd>Polyphen2: 0.391 (B)</dd>
                                    <dd>SIFT: 0.170 (T)</dd>
                                    <dd>CADD: 0.970 (15.190)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0446%</dd>
                                    <dd>UK10K: 0.0264%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0116%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000174640">SLCO2A1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.134</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.256</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.954</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.195 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2796">Pachydermoperiostosis</a> associated with SLCO2A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0004398, Peptic ulcer</dd><dd>HP:0000218, High palate - HP:0004398, Peptic ulcer</dd></dl><dl><dt>Proximity score 0.512 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLCO2A1%0DHPGD&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HPGD</a> and phenotypic similarity 0.762 to <a href="http://www.omim.org/259100">Hypertrophic osteoarthropathy, primary, autosomal recessive 1</a> associated with HPGD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0001519, Disproportionate tall stature</dd><dd>HP:0011800, Midface retrusion - HP:0000218, High palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000218, High palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614441">OMIM:614441</a> Hypertrophic osteoarthropathy, primary, autosomal recessive 2 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2796">ORPHA:2796</a> Pachydermoperiostosis</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.134</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.256</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.954</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.133945240G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=147644142">rs147644142</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs147644142&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.954</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000310926.11">SLCO2A1:ENST00000310926.11:c.1316C&gt;T:p.(Pro439Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000493729.5">SLCO2A1:ENST00000493729.5:c.1088C&gt;T:p.(Pro363Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96451867</dd>
                                    
                                    <dd>Polyphen2: 0.012 (B)</dd>
                                    <dd>SIFT: 0.391 (T)</dd>
                                    <dd>CADD: 0.965 (14.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0159%</dd>
                                    <dd>UK10K: 0.0264%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AMR: 0.0779%</dd>
                                    <dd>ExAC EAS: 0.0232%</dd>
                                    <dd>ExAC NFE: 0.0167%</dd>
                                    <dd>gnomAD_E_AMR: 0.0400%</dd>
                                    <dd>gnomAD_E_EAS: 0.0177%</dd>
                                    <dd>gnomAD_E_NFE: 0.0118%</dd>
                                    <dd>gnomAD_E_OTH: 0.0186%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000124299">PEPD</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.126</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.209</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.418 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=742">Prolidase deficiency</a> associated with PEPD.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - HP:0005280, Depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0005280, Depressed nasal bridge</dd><dd>HP:0000218, High palate - HP:0005280, Depressed nasal bridge</dd></dl><dl><dt>Phenotypic similarity 0.356 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PEPD">PEPD</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0002764, short tibia</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0008259, abnormal optic disk morphology</dd><dd>HP:0000175, Cleft palate - MP:0002691, small stomach</dd><dd>HP:0000218, High palate - MP:0002691, small stomach</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/170100">OMIM:170100</a> Prolidase deficiency - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=742">ORPHA:742</a> Prolidase deficiency</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.126</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.209</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.33387892C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=121917724">rs121917724</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs121917724&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    
                                    
                                    
                                    <span class="label label-danger">PATHOGENIC</span>
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=15252%5Balleleid%5D">(criteria_provided,_multiple_submitters,_no_conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                <span class="label label-danger">WHITELIST VARIANT</span>
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000244137.12">PEPD:ENST00000244137.12:c.1342G&gt;A:p.(Gly448Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397032.8">PEPD:ENST00000397032.8:c.1219G&gt;A:p.(Gly407Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000436370.7">PEPD:ENST00000436370.7:c.1150G&gt;A:p.(Gly384Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Polyphen2: 1.000 (D)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.997 (24.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC NFE: 0.0270%</dd>
                                    <dd>gnomAD_E_NFE: 0.0031%</dd>
                                    <dd>gnomAD_E_SAS: 0.0042%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000215041">NEURL4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.126</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.668</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=NEURL4%0DCCP110&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCP110</a> and phenotypic similarity 0.663 to mouse mutant of CCP110.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion - MP:0009882, absent palatal shelf</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0005262, coloboma</dd><dd>HP:0000175, Cleft palate - MP:0000111, cleft palate</dd><dd>HP:0000218, High palate - MP:0000111, cleft palate</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.126</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.668</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr17:g.7323939G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200538912">rs200538912</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200538912&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.668</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000315614.11">NEURL4:ENST00000315614.11:c.2136C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399464.7">NEURL4:ENST00000399464.7:c.2136C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000570460.5">NEURL4:ENST00000570460.5:c.2070C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6713755</dd>
                                    
                                    <dd>CADD: 0.671 (4.833)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0048%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP EA: 0.0240%</dd>
                                    <dd>ESP All: 0.0162%</dd>
                                    <dd>ExAC AMR: 0.0260%</dd>
                                    <dd>ExAC EAS: 0.0117%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0357%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0102%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0116%</dd>
                                    <dd>gnomAD_E_OTH: 0.0366%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187741">FANCA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.097</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.296</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.869</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.593 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with FANCA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0004322, Short stature</dd><dd>HP:0011800, Midface retrusion - HP:0000218, High palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000365, Hearing impairment</dd><dd>HP:0000545, Myopia - HP:0000483, Astigmatism</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000218, High palate</dd></dl><dl><dt>Phenotypic similarity 0.288 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FANCA">FANCA</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0001297, microphthalmia</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FANCA%0DCHUK&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CHUK</a> and phenotypic similarity 0.304 to <a href="http://www.omim.org/613630">Cocoon syndrome</a> associated with CHUK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0010808, Protruding tongue</dd><dd>HP:0000218, High palate - HP:0010808, Protruding tongue</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=FANCA%0DCHUK&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CHUK</a> and phenotypic similarity 0.628 to mouse mutant of CHUK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0000547, short limbs</dd><dd>HP:0011800, Midface retrusion - MP:0005358, abnormal incisor morphology</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0009890, cleft secondary palate</dd><dd>HP:0000218, High palate - MP:0009890, cleft secondary palate</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/227650">OMIM:227650</a> Fanconi anemia, complementation group A - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">ORPHA:84</a> Fanconi anemia</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.097</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.296</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.869</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr16:g.89808371AAAAAC&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=755643283">rs755643283</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs755643283&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    
                                    
                                    <span class="label label-info">LIKELY_BENIGN</span>
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=690164%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.869</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389301.8">FANCA:ENST00000389301.8:c.523-9_523-5del:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389302.7">FANCA:ENST00000389302.7:c.523-9_523-5del:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000534992.5">FANCA:ENST00000534992.5:c.523-9_523-5del:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543736.5">FANCA:ENST00000543736.5:c.427-9_427-5del:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000563673.5">FANCA:ENST00000563673.5:c.523-9_523-5del:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568369.5">FANCA:ENST00000568369.5:c.523-9_523-5del:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.87046117</dd>
                                    
                                    <dd>CADD: 0.870 (8.876)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182566">CLEC4G</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.076</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.157</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.157 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CLEC4G">CLEC4G</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0001860, liver inflammation</dd><dd>HP:0000218, High palate - MP:0001860, liver inflammation</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.076</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.157</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.7731666G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=530697988">rs530697988</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs530697988&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000328853.9">CLEC4G:ENST00000328853.9:c.161C&gt;T:p.(Ser54Phe)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999</dd>
                                    
                                    <dd>Polyphen2: 0.997 (D)</dd>
                                    <dd>SIFT: 0.001 (D)</dd>
                                    <dd>CADD: 0.996 (24.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC AFR: 0.0200%</dd>
                                    <dd>ExAC AMR: 0.0088%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000275395">FCGBP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.035</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.309</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.737</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.309 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FCGBP">FCGBP</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia - MP:0002092, abnormal eye morphology</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.035</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.309</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.737</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.39886439A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=746639255">rs746639255</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs746639255&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.737</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.7740T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.4335T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7463704</dd>
                                    
                                    <dd>CADD: 0.746 (5.958)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0267%</dd>
                                    <dd>gnomAD_E_AMR: 0.0662%</dd>
                                    <dd>gnomAD_E_EAS: 0.0099%</dd>
                                    <dd>gnomAD_E_FIN: 0.0087%</dd>
                                    <dd>gnomAD_E_NFE: 0.0876%</dd>
                                    <dd>gnomAD_E_OTH: 0.0313%</dd>
                                    <dd>gnomAD_E_SAS: 0.0455%</dd>
                                    <dd>gnomAD_G_NFE: 0.0785%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr19:g.39886355C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2916064">rs2916064</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2916064&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.053</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616721.6">FCGBP:ENST00000616721.6:c.7824G&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620799.1">FCGBP:ENST00000620799.1:c.4419G&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.052890897</dd>
                        
                        <dd>CADD: 0.053 (0.236)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        
                        <dd>gnomAD_E_AFR: 0.0323%</dd>
                        <dd>gnomAD_E_AMR: 0.0114%</dd>
                        <dd>gnomAD_E_ASJ: 0.0268%</dd>
                        <dd>gnomAD_E_NFE: 0.0028%</dd>
                        <dd>gnomAD_E_SAS: 0.0087%</dd>
                        <dd>gnomAD_G_AFR: 0.0396%</dd>
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196873">CBWD3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_ACCEPTOR_VARIANT</span>
                                <b>chr9:g.68256920G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2225470">rs2225470</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2225470&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360171.10">CBWD3:ENST00000360171.10:c.431-1G&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377342.9">CBWD3:ENST00000377342.9:c.431-1G&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616550.4">CBWD3:ENST00000616550.4:c.431-1G&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618217.4">CBWD3:ENST00000618217.4:c.431-1G&gt;C:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99645185</dd>
                                    
                                    <dd>CADD: 0.996 (24.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000147996">CBWD5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                                <b>chr9:g.65682859T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1282520379">rs1282520379</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1282520379&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377392.9">CBWD5:ENST00000377392.9:c.-1336+2T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377395.8">CBWD5:ENST00000377395.8:c.430+2T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382405.7">CBWD5:ENST00000382405.7:c.430+2T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000429800.6">CBWD5:ENST00000429800.6:c.430+2T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000430059.6">CBWD5:ENST00000430059.6:c.430+2T&gt;C:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99601895</dd>
                                    
                                    <dd>CADD: 0.996 (24.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000259156">CHEK2P2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                                <b>chr15:g.20282975G&gt;GTGAA [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=374719316">rs374719316</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs374719316&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555186.5">CHEK2P2:ENST00000555186.5:n.231+1_231+2insTGAA:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9729604</dd>
                                    
                                    <dd>CADD: 0.973 (15.680)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000221957">KIR2DS4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_TRUNCATION</span>
                                <b>chr19:g.54847788AAC&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=138992022">rs138992022</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs138992022&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339924.12">KIR2DS4:ENST00000339924.12:c.838_839del:p.(Gln280Valfs*18)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9938341</dd>
                                    
                                    <dd>CADD: 0.994 (22.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.54835079G&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=112522228">rs112522228</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs112522228&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.990</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339924.12">KIR2DS4:ENST00000339924.12:c.35G&gt;C:p.(Gly12Ala)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.9898375</dd>
                        
                        <dd>CADD: 0.990 (19.930)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188763">FZD9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>1.000</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.73434381T&gt;G [0/1:0/0:0/1]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344575.5">FZD9:ENST00000344575.5:c.374T&gt;G:p.(Ile125Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999978</dd>
                                    
                                    <dd>Polyphen2: 0.574 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.002 (D)</dd>
                                    <dd>CADD: 0.998 (28.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163749">CCDC158</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_ACCEPTOR_VARIANT</span>
                                <b>chr4:g.76334169T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=574121898">rs574121898</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs574121898&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000388914.7">CCDC158:ENST00000388914.7:c.2665-2A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.996 (24.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135913">USP37</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.218474816C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=529125220">rs529125220</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs529125220&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258399.8">USP37:ENST00000258399.8:c.2113G&gt;A:p.(Glu705Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000415516.5">USP37:ENST00000415516.5:c.1831G&gt;A:p.(Glu611Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000418019.5">USP37:ENST00000418019.5:c.2113G&gt;A:p.(Glu705Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454775.5">USP37:ENST00000454775.5:c.2113G&gt;A:p.(Glu705Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999993</dd>
                                    
                                    <dd>Polyphen2: 0.448 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.019 (D)</dd>
                                    <dd>CADD: 0.996 (23.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000110013">SIAE</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.997</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.997</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr11:g.124649635C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=533417722">rs533417722</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs533417722&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.997</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263593.8">SIAE:ENST00000263593.8:c.706G&gt;A:p.(Gly236Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545756.5">SIAE:ENST00000545756.5:c.601G&gt;A:p.(Gly201Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618733.4">SIAE:ENST00000618733.4:c.601G&gt;A:p.(Gly201Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999975</dd>
                                    
                                    <dd>Polyphen2: 0.507 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.008 (D)</dd>
                                    <dd>CADD: 0.995 (23.100)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000180257">ZNF816</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.996</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.52956082C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=535358313">rs535358313</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs535358313&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000270457.8">ZNF816:ENST00000270457.8:c.8G&gt;A:p.(Arg3His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357666.8">ZNF816:ENST00000357666.8:c.8G&gt;A:p.(Arg3His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391786.6">ZNF816:ENST00000391786.6:c.8G&gt;A:p.(Arg3His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000438970.6">ZNF816:ENST00000438970.6:c.8G&gt;A:p.(Arg3His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000444460.6">ZNF816:ENST00000444460.6:c.8G&gt;A:p.(Arg3His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000535506.5">ZNF816:ENST00000535506.5:c.8G&gt;A:p.(Arg3His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391777.3">ZNF816:ENST00000391777.3:c.8G&gt;A:p.(Arg3His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>SIFT: 0.000 (D)</dd>
                                    <dd>CADD: 0.176 (0.841)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187545">PRAMEF10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.995</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.016</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.995</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                                <b>chr1:g.12894694C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2260233">rs2260233</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2260233&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.995</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000235347.4">PRAMEF10:ENST00000235347.4:c.758G&gt;A:p.(Trp253*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 1.0</dd>
                                    
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>CADD: 0.996 (23.900)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0348%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204175">GPRIN2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.995</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.995</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.46549695C&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4926046">rs4926046</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs4926046&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.995</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374314.5">GPRIN2:ENST00000374314.5:c.1042G&gt;T:p.(Val348Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374317.1">GPRIN2:ENST00000374317.1:c.1042G&gt;T:p.(Val348Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99563485</dd>
                                    
                                    <dd>SIFT: 0.036 (D)</dd>
                                    <dd>CADD: 0.996 (23.600)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0073%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                    <b>chr10:g.46554580G&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=873876">rs873876</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs873876&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.800</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374317.1">GPRIN2:ENST00000374317.1:c.-7+5C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.46972638</dd>
                        
                        <dd>CADD: 0.470 (2.755)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000270339">PPIAL4H</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.146344411T&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1427493799">rs1427493799</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1427493799&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000584068.3">PPIAL4H:ENST00000584068.3:c.305A&gt;T:p.(Asn102Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000643391.1">PPIAL4H:ENST00000643391.1:n.376-16200A&gt;T:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9942456</dd>
                                    
                                    <dd>CADD: 0.994 (22.400)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000118200">CAMSAP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.994</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.994</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">INFRAME_DELETION</span>
                                <b>chr1:g.200852612AAAGGAAACT&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=747209889">rs747209889</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs747209889&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.994</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000236925.8">CAMSAP2:ENST00000236925.8:c.3571_3579del:p.(Lys1191_Thr1193del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358823.6">CAMSAP2:ENST00000358823.6:c.3538_3546del:p.(Lys1180_Thr1182del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413307.6">CAMSAP2:ENST00000413307.6:c.3490_3498del:p.(Lys1164_Thr1166del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9946297</dd>
                                    
                                    <dd>CADD: 0.995 (22.700)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000268320">SCGB1C2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.993</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.993</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.137637G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1255969219">rs1255969219</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1255969219&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.993</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000595228.3">SCGB1C2:ENST00000595228.3:c.47G&gt;A:p.(Cys16Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99339306</dd>
                                    
                                    <dd>CADD: 0.993 (21.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000074935">TUBE1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.991</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.991</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr6:g.112087425A&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=199513778">rs199513778</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs199513778&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.991</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368662.10">TUBE1:ENST00000368662.10:c.10T&gt;G:p.(Ser4Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.999991</dd>
                                    
                                    <dd>Polyphen2: 0.872 (P)</dd>
                                    <dd>Mutation Taster: 1.000 (P)</dd>
                                    <dd>SIFT: 0.277 (T)</dd>
                                    <dd>CADD: 0.997 (25.500)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0247%</dd>
                                    <dd>UK10K: 0.0529%</dd>
                                    <dd>ExAC NFE: 0.0137%</dd>
                                    <dd>gnomAD_E_NFE: 0.0441%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                    <dd>gnomAD_G_NFE: 0.0668%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196275">GTF2IRD2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.989</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.989</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.74798013G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2529320">rs2529320</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2529320&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.989</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000451013.6">GTF2IRD2:ENST00000451013.6:c.1499C&gt;T:p.(Ser500Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000651129.1">GTF2IRD2:ENST00000651129.1:c.1985C&gt;T:p.(Ser662Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98923534</dd>
                                    
                                    <dd>CADD: 0.989 (19.680)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000161082">CELF5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.988</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.015</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.988</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.3286015G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=376828058">rs376828058</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs376828058&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.988</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000292672.7">CELF5:ENST00000292672.7:c.1176G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541430.6">CELF5:ENST00000541430.6:c.1101G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9925869</dd>
                                    
                                    <dd>CADD: 0.993 (21.300)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ESP EA: 0.0125%</dd>
                                    <dd>ESP All: 0.0084%</dd>
                                    <dd>ExAC AMR: 0.0340%</dd>
                                    <dd>ExAC NFE: 0.0021%</dd>
                                    <dd>gnomAD_E_AMR: 0.0160%</dd>
                                    <dd>gnomAD_E_NFE: 0.0022%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163001">CFAP36</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.986</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.986</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr2:g.55519825G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=551462553">rs551462553</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs551462553&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.986</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339012.7">CFAP36:ENST00000339012.7:c.24G&gt;C:p.(Glu8Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349456.8">CFAP36:ENST00000349456.8:c.24G&gt;C:p.(Glu8Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000403007.4">CFAP36:ENST00000403007.4:c.24G&gt;C:p.(Glu8Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000406691.7">CFAP36:ENST00000406691.7:c.24G&gt;C:p.(Glu8Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000407816.7">CFAP36:ENST00000407816.7:c.24G&gt;C:p.(Glu8Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.99168235</dd>
                                    
                                    <dd>Polyphen2: 0.193 (B)</dd>
                                    <dd>Mutation Taster: 0.947 (P)</dd>
                                    <dd>SIFT: 0.485 (T)</dd>
                                    <dd>CADD: 0.992 (20.800)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC NFE: 0.0090%</dd>
                                    <dd>ExAC SAS: 0.0363%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_ASJ: 0.0406%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0260%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197978">GOLGA6L9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.985</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.008</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.921</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr15:g.82434369G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=17846317">rs17846317</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs17846317&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.921</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618348.1">GOLGA6L9:ENST00000618348.1:c.769G&gt;A:p.(Glu257Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.92056715</dd>
                                    
                                    <dd>CADD: 0.921 (11.000)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.985</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr15:g.82431921C&gt;T [1/1:0/1:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1228564009">rs1228564009</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1228564009&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.985</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618348.1">GOLGA6L9:ENST00000618348.1:c.176C&gt;T:p.(Thr59Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9850721</dd>
                                    
                                    <dd>CADD: 0.985 (18.260)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000270629">NBPF14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.984</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.984</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.148593602G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=71231122">rs71231122</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs71231122&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.984</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581201.5">NBPF14:ENST00000581201.5:c.274C&gt;T:p.(Leu92Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611826.4">NBPF14:ENST00000611826.4:c.274C&gt;T:p.(Leu92Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614999.4">NBPF14:ENST00000614999.4:c.274C&gt;T:p.(Leu92Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621066.4">NBPF14:ENST00000621066.4:c.67C&gt;T:p.(Leu23Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621070.4">NBPF14:ENST00000621070.4:c.274C&gt;T:p.(Leu92Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616120.4">NBPF14:ENST00000616120.4:c.175+1941C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620240.1">NBPF14:ENST00000620240.1:c.174-836C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614329.4">NBPF14:ENST00000614329.4:c.-7+2122C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9838935</dd>
                                    
                                    <dd>CADD: 0.984 (17.930)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.148559961T&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1397909621">rs1397909621</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1397909621&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.946</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614999.4">NBPF14:ENST00000614999.4:c.4054A&gt;G:p.(Ser1352Gly)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581201.5">NBPF14:ENST00000581201.5:c.2252-7131A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611826.4">NBPF14:ENST00000611826.4:c.1214-7131A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621066.4">NBPF14:ENST00000621066.4:c.2188+3036A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621070.4">NBPF14:ENST00000621070.4:c.2027-7131A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614329.4">NBPF14:ENST00000614329.4:c.*375-7131A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616120.4">NBPF14:ENST00000616120.4:c.*375-7131A&gt;G:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620240.1">NBPF14:ENST00000620240.1:c.*375-7131A&gt;G:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.946173</dd>
                        
                        <dd>CADD: 0.946 (12.690)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr1:g.148559962G&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1339643523">rs1339643523</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1339643523&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.220</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614999.4">NBPF14:ENST00000614999.4:c.4053C&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581201.5">NBPF14:ENST00000581201.5:c.2252-7132C&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611826.4">NBPF14:ENST00000611826.4:c.1214-7132C&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621066.4">NBPF14:ENST00000621066.4:c.2188+3035C&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621070.4">NBPF14:ENST00000621070.4:c.2027-7132C&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614329.4">NBPF14:ENST00000614329.4:c.*375-7132C&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616120.4">NBPF14:ENST00000616120.4:c.*375-7132C&gt;A:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620240.1">NBPF14:ENST00000620240.1:c.*375-7132C&gt;A:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.22034943</dd>
                        
                        <dd>CADD: 0.220 (1.081)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000206559">ZCWPW2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.983</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.983</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.28413268G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=374756161">rs374756161</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs374756161&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.983</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000383768.7">ZCWPW2:ENST00000383768.7:c.200G&gt;C:p.(Arg67Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000635992.1">ZCWPW2:ENST00000635992.1:c.200G&gt;C:p.(Arg67Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.98862374</dd>
                                    
                                    <dd>Polyphen2: 0.272 (B)</dd>
                                    <dd>SIFT: 0.184 (T)</dd>
                                    <dd>CADD: 0.989 (19.440)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000276950">GSTT4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.983</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.983</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.23998678T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1177906550">rs1177906550</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1177906550&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.983</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621179.5">GSTT4:ENST00000621179.5:c.590A&gt;G:p.(Gln197Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9830956</dd>
                                    
                                    <dd>CADD: 0.983 (17.720)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000260220">CCDC187</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.981</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.981</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.136292176G&gt;A [0/1:0/0:0/1]</b>
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.981</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000569961.5">CCDC187:ENST00000569961.5:c.658C&gt;T:p.(Leu220Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000624277.3">CCDC187:ENST00000624277.3:c.952C&gt;T:p.(Leu318Phe)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000638797.1">CCDC187:ENST00000638797.1:c.952C&gt;T:p.(Leu318Phe)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9814647</dd>
                                    
                                    <dd>SIFT: 0.174 (T)</dd>
                                    <dd>CADD: 0.981 (17.320)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000268043">NBPF12</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.980</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.014</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.980</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.146988091A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200441040">rs200441040</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200441040&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.980</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439206.6">NBPF12:ENST00000439206.6:c.1403A&gt;G:p.(Glu468Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579935.8">NBPF12:ENST00000579935.8:c.2441A&gt;G:p.(Glu814Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611443.4">NBPF12:ENST00000611443.4:c.3254A&gt;G:p.(Glu1085Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613714.2">NBPF12:ENST00000613714.2:c.2216A&gt;G:p.(Glu739Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617614.4">NBPF12:ENST00000617614.4:c.2437A&gt;G:p.(Arg813Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617844.4">NBPF12:ENST00000617844.4:c.3254A&gt;G:p.(Glu1085Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617931.4">NBPF12:ENST00000617931.4:c.3254A&gt;G:p.(Glu1085Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617864.3">NBPF12:ENST00000617864.3:c.*564A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9803664</dd>
                                    
                                    <dd>CADD: 0.980 (17.070)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000004866">ST7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.979</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.979</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr7:g.117129833T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=547128860">rs547128860</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs547128860&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.979</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265437.9">ST7:ENST00000265437.9:c.435T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000323984.7">ST7:ENST00000323984.7:c.435T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393443.5">ST7:ENST00000393443.5:c.285T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393444.7">ST7:ENST00000393444.7:c.306T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393446.6">ST7:ENST00000393446.6:c.435T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393447.8">ST7:ENST00000393447.8:c.306T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393449.5">ST7:ENST00000393449.5:c.435T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393451.7">ST7:ENST00000393451.7:c.435T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422922.5">ST7:ENST00000422922.5:c.297T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000432298.5">ST7:ENST00000432298.5:c.297T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000465133.5">ST7:ENST00000465133.5:c.306T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9818866</dd>
                                    
                                    <dd>CADD: 0.982 (17.420)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000215548">FRG1JP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.977</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.977</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr9:g.63842823CCTT&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=144201632">rs144201632</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs144201632&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.977</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376334.3">FRG1JP:ENST00000376334.3:n.410_412del:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9765037</dd>
                                    
                                    <dd>CADD: 0.977 (16.290)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000275969">SPATA31A3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.013</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.973</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.013</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.973</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr9:g.66987715G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=62565523">rs62565523</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs62565523&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.973</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428649.4">SPATA31A3:ENST00000428649.4:c.2783C&gt;T:p.(Thr928Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97289807</dd>
                                    
                                    <dd>CADD: 0.973 (15.670)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr9:g.66989082C&gt;A [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=773528867">rs773528867</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs773528867&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.834</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428649.4">SPATA31A3:ENST00000428649.4:c.1416G&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8338119</dd>
                        
                        <dd>CADD: 0.834 (7.794)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000138161">CUZD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.968</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.968</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.122837549G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=558554477">rs558554477</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs558554477&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.968</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392790.6">CUZD1:ENST00000392790.6:c.454C&gt;G:p.(Pro152Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368904.6">CUZD1:ENST00000368904.6:c.454C&gt;G:p.(Pro152Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.97021484</dd>
                                    
                                    <dd>Polyphen2: 0.053 (B)</dd>
                                    <dd>SIFT: 0.181 (T)</dd>
                                    <dd>CADD: 0.970 (15.260)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0010%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000247315">ZCCHC3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.968</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.968</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_INSERTION</span>
                                <b>chr20:g.298046C&gt;CCGG [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=5839847">rs5839847</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs5839847&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.968</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000500893.4">ZCCHC3:ENST00000500893.4:c.471_473dup:p.(Ala158dup)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.967566</dd>
                                    
                                    <dd>CADD: 0.968 (14.890)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000263956">NBPF11</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.967</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.967</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.148126846T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=57784339">rs57784339</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs57784339&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.967</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.158A&gt;G:p.(Asn53Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.158A&gt;G:p.(Asn53Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.156A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.96673405</dd>
                                    
                                    <dd>CADD: 0.967 (14.780)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.148120602C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1199265858">rs1199265858</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1199265858&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.857</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.887G&gt;A:p.(Cys296Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.782G&gt;A:p.(Cys261Tyr)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.887G&gt;A:p.(Cys296Tyr)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.85724217</dd>
                        
                        <dd>CADD: 0.857 (8.454)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr1:g.148120603A&gt;G [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=74994984">rs74994984</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs74994984&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.704</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614015.4">NBPF11:ENST00000614015.4:c.886T&gt;C:p.(Cys296Arg)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614785.4">NBPF11:ENST00000614785.4:c.781T&gt;C:p.(Cys261Arg)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615281.4">NBPF11:ENST00000615281.4:c.886T&gt;C:p.(Cys296Arg)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.70351684</dd>
                        
                        <dd>CADD: 0.704 (5.280)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000173821">RNF213</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.965</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2573">ORPHA:2573</a> Moyamoya disease</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.012</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.965</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.80363278A&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=561810522">rs561810522</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs561810522&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.965</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000508628.6">RNF213:ENST00000508628.6:c.11679A&gt;C:p.(Glu3893Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000582970.5">RNF213:ENST00000582970.5:c.11532A&gt;C:p.(Glu3844Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9679373</dd>
                                    
                                    <dd>SIFT: 0.207 (T)</dd>
                                    <dd>CADD: 0.968 (14.940)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000265018">AGAP12P</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.961</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.961</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr10:g.48025898T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=370794753">rs370794753</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs370794753&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.961</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000585227.6">AGAP12P:ENST00000585227.6:n.396A&gt;G:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000603888.2">AGAP12P:ENST00000603888.2:n.437A&gt;G:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9609159</dd>
                                    
                                    <dd>CADD: 0.961 (14.080)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183248">PRR36</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.957</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.011</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.957</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.7869534C&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=540887892">rs540887892</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs540887892&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.957</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000618550.5">PRR36:ENST00000618550.5:c.3540G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9594492</dd>
                                    
                                    <dd>CADD: 0.959 (13.920)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>gnomAD_E_SAS: 0.0047%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000274070">CASTOR2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.944</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.010</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.944</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr7:g.75017680C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1209469316">rs1209469316</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1209469316&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.944</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616305.2">CASTOR2:ENST00000616305.2:c.267C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.94363624</dd>
                                    
                                    <dd>CADD: 0.944 (12.490)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000120159">CAAP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.941</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.010</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.941</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr9:g.26887352T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=569193262">rs569193262</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs569193262&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.941</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333916.8">CAAP1:ENST00000333916.8:c.465A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000625311.1">CAAP1:ENST00000625311.1:c.30A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000520187.1">CAAP1:ENST00000520187.1:c.303+5061A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.94402426</dd>
                                    
                                    <dd>CADD: 0.944 (12.520)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0023%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000142661">MYOM3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.937</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.009</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.937</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.24075462G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=531893682">rs531893682</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs531893682&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.937</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374434.4">MYOM3:ENST00000374434.4:c.2715C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9405708</dd>
                                    
                                    <dd>CADD: 0.941 (12.260)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0264%</dd>
                                    <dd>gnomAD_E_NFE: 0.0020%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211647">IGLV5-48</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.937</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.009</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.937</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.22353209G&gt;C [1/1:0/1:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201334749">rs201334749</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201334749&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.937</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390293.1">IGLV5-48:ENST00000390293.1:c.145G&gt;C:p.(Val49Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9370494</dd>
                                    
                                    <dd>CADD: 0.937 (12.010)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr22:g.22353228A&gt;T [1/1:0/1:1/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=117042445">rs117042445</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs117042445&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.000</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390293.1">IGLV5-48:ENST00000390293.1:c.164A&gt;T:p.(Tyr55Phe)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 2.3025274E-4</dd>
                        
                        <dd>CADD: 0.000 (0.001)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000172969">FRG2C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.921</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.008</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.921</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr3:g.75665948T&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2118760">rs2118760</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2118760&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.921</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308062.7">FRG2C:ENST00000308062.7:c.756T&gt;A:p.(Asp252Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000464571.1">FRG2C:ENST00000464571.1:c.753T&gt;A:p.(Asp251Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000638439.1">FRG2C:ENST00000638439.1:n.137+4072A&gt;T:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9214764</dd>
                                    
                                    <dd>Polyphen2: 0.001 (B)</dd>
                                    <dd>SIFT: 0.624 (T)</dd>
                                    <dd>CADD: 0.921 (11.050)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                    <b>chr3:g.75665865T&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4507282">rs4507282</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs4507282&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.818</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308062.7">FRG2C:ENST00000308062.7:c.673T&gt;C:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000464571.1">FRG2C:ENST00000464571.1:c.670T&gt;C:p.(=)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000638439.1">FRG2C:ENST00000638439.1:n.137+4155A&gt;G:</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8184902</dd>
                        
                        <dd>CADD: 0.818 (7.411)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000211641">IGLV11-55</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.916</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.008</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.916</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr22:g.22201974T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=9306341">rs9306341</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs9306341&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.916</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000390286.3">IGLV11-55:ENST00000390286.3:c.182T&gt;C:p.(Leu61Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000640991.1">IGLV11-55:ENST00000640991.1:c.203-13147T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.916054</dd>
                                    
                                    <dd>CADD: 0.916 (10.760)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000158747">NBL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.912</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.007</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.912</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr1:g.19643329G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=552896981">rs552896981</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs552896981&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.912</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000289749.6">NBL1:ENST00000289749.6:c.18G&gt;A:p.(Met6Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602662.1">NBL1:ENST00000602662.1:c.-19-11683G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602384.5">NBL1:ENST00000602384.5:c.*433+2600G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9144933</dd>
                                    
                                    <dd>Polyphen2: 0.000 (B)</dd>
                                    <dd>SIFT: 0.094 (T)</dd>
                                    <dd>CADD: 0.914 (10.680)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000233369">GTF2IP4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.901</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.007</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.901</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr7:g.73179276ACT&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1428697221">rs1428697221</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1428697221&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.901</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453092.3">GTF2IP4:ENST00000453092.3:n.361+7_361+8del:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544802.5">GTF2IP4:ENST00000544802.5:n.767+7_767+8del:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9009168</dd>
                                    
                                    <dd>CADD: 0.901 (10.040)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000282939">TRBV7-2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.006</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.891</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.006</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.891</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr7:g.142353283G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=361383">rs361383</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs361383&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.891</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000634605.1">TRBV7-2:ENST00000634605.1:c.272G&gt;A:p.(Gly91Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8908309</dd>
                                    
                                    <dd>CADD: 0.891 (9.619)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000233524">RANP8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.006</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.889</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.006</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.889</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr13:g.20747462C&gt;CTT [1/1:0/-:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=60730051">rs60730051</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs60730051&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.889</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000444720.1">RANP8:ENST00000444720.1:n.293-8_293-7insAA:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382754.4">RANP8:ENST00000382754.4:c.145-9658_145-9657insAA:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382758.6">RANP8:ENST00000382758.6:c.145-9658_145-9657insAA:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8892612</dd>
                                    
                                    <dd>CADD: 0.889 (9.557)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000250312">ZNF718</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.006</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.886</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.006</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.886</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr4:g.131416G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1280682274">rs1280682274</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1280682274&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.886</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000510175.6">ZNF718:ENST00000510175.6:c.137G&gt;C:p.(Ser46Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000609714.1">ZNF718:ENST00000609714.1:c.41G&gt;C:p.(Ser14Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8861849</dd>
                                    
                                    <dd>CADD: 0.886 (9.438)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr4:g.130852C&gt;T [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1482910683">rs1482910683</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1482910683&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.854</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000510175.6">ZNF718:ENST00000510175.6:c.68C&gt;T:p.(Thr23Ile)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000609714.1">ZNF718:ENST00000609714.1:c.-29C&gt;T:p.(=)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.8544206</dd>
                        
                        <dd>CADD: 0.854 (8.369)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187391">MAGI2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.871</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617609">OMIM:617609</a> Nephrotic syndrome, type 15 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.005</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.871</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr7:g.78168010G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=144611762">rs144611762</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs144611762&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.871</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354212.9">MAGI2:ENST00000354212.9:c.2502C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000419488.5">MAGI2:ENST00000419488.5:c.2460C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000519748.5">MAGI2:ENST00000519748.5:c.1281C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000522391.3">MAGI2:ENST00000522391.3:c.2502C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000535697.5">MAGI2:ENST00000535697.5:c.2046C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000626691.2">MAGI2:ENST00000626691.2:c.2013C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000628980.2">MAGI2:ENST00000628980.2:c.2088C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000629359.2">MAGI2:ENST00000629359.2:c.1971C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000637282.1">MAGI2:ENST00000637282.1:c.1281C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000637441.1">MAGI2:ENST00000637441.1:c.2460C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8809581</dd>
                                    
                                    <dd>CADD: 0.881 (9.243)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0088%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0288%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0238%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_G_AFR: 0.0459%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185028">LRRC14B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.842</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.004</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.842</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr5:g.191754C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=544622005">rs544622005</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs544622005&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.842</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000328278.4">LRRC14B:ENST00000328278.4:c.216C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.843937</dd>
                                    
                                    <dd>CADD: 0.844 (8.067)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000270775">AP000436.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.824</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.824</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr11:g.65025779CT&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=36027416">rs36027416</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs36027416&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.824</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000605239.1">AP000436.1:ENST00000605239.1:n.410del:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000301886.3">AP000436.1:ENST00000301886.3:c.340-1538del:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.82372123</dd>
                                    
                                    <dd>CADD: 0.824 (7.538)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197181">PIWIL2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.806</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.806</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr8:g.22315022A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201716037">rs201716037</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201716037&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.806</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356766.11">PIWIL2:ENST00000356766.11:c.2092-7A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454009.6">PIWIL2:ENST00000454009.6:c.2092-7A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000521356.5">PIWIL2:ENST00000521356.5:c.2092-7A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000611073.1">PIWIL2:ENST00000611073.1:c.2092-7A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.808398</dd>
                                    
                                    <dd>CADD: 0.808 (7.176)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000259425">AC011767.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr15:g.22774667G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=2347177">rs2347177</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs2347177&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000559762.1">AC011767.1:ENST00000559762.1:n.228C&gt;T:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617410.4">AC011767.1:ENST00000617410.4:n.87C&gt;T:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000437912.6">AC011767.1:ENST00000437912.6:c.-48+354G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.05027032</dd>
                                    
                                    <dd>CADD: 0.050 (0.224)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000253362">AC012400.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.800</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr8:g.84163656C&gt;CT [0/1:0/0:-/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3077436">rs3077436</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs3077436&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000518452.1">AC012400.1:ENST00000518452.1:n.177-4_177-3insA:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4028272</dd>
                                    
                                    <dd>CADD: 0.403 (2.239)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197054">ZNF763</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.798</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.003</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.798</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                                <b>chr19:g.11977364A&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=562370017">rs562370017</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs562370017&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.798</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000343949.9">ZNF763:ENST00000343949.9:c.140-7A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358987.8">ZNF763:ENST00000358987.8:c.131-7A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000590798.1">ZNF763:ENST00000590798.1:c.191-7A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000592625.5">ZNF763:ENST00000592625.5:c.130+200A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000591944.1">ZNF763:ENST00000591944.1:c.337+200A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.12923694</dd>
                                    
                                    <dd>CADD: 0.129 (0.601)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000277758">FO681492.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.788</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.002</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.788</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr10:g.47762988G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1226594908">rs1226594908</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1226594908&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.788</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000623662.3">FO681492.1:ENST00000623662.3:c.17C&gt;T:p.(Ser6Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.78772444</dd>
                                    
                                    <dd>CADD: 0.788 (6.731)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198573">SPANXC</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.766</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.002</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.766</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.141241609C&gt;G [1/1:0/1:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=3208371">rs3208371</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs3208371&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.766</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358993.3">SPANXC:ENST00000358993.3:c.202G&gt;C:p.(Val68Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.76611626</dd>
                                    
                                    <dd>CADD: 0.766 (6.310)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000223547">ZNF844</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.702</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.702</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.12075835T&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=554342006">rs554342006</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs554342006&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.702</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439326.8">ZNF844:ENST00000439326.8:c.715T&gt;A:p.(Ser239Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000652448.1">ZNF844:ENST00000652448.1:c.619T&gt;A:p.(Ser207Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000651606.1">ZNF844:ENST00000651606.1:c.509T&gt;A:p.(Phe170Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441304.2">ZNF844:ENST00000441304.2:c.*438T&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.704</dd>
                                    
                                    <dd>Polyphen2: 0.020 (B)</dd>
                                    <dd>SIFT: 0.296 (T)</dd>
                                    <dd>CADD: 0.000 (0.001)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC AMR: 0.0109%</dd>
                                    <dd>ExAC NFE: 0.0017%</dd>
                                    <dd>gnomAD_E_AMR: 0.0061%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104972">LILRB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.661</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.661</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr19:g.54631288C&gt;A [1/1:0/1:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1985501">rs1985501</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1985501&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.661</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324602.11">LILRB1:ENST00000324602.11:c.52C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396315.5">LILRB1:ENST00000396315.5:c.52C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396317.5">LILRB1:ENST00000396317.5:c.52C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396327.7">LILRB1:ENST00000396327.7:c.52C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396331.5">LILRB1:ENST00000396331.5:c.52C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396332.8">LILRB1:ENST00000396332.8:c.52C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427581.6">LILRB1:ENST00000427581.6:c.160C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6609217</dd>
                                    
                                    <dd>CADD: 0.661 (4.697)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.54631632C&gt;T [1/1:0/1:1/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1061679">rs1061679</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1061679&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.010</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324602.11">LILRB1:ENST00000324602.11:c.203C&gt;T:p.(Pro68Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396315.5">LILRB1:ENST00000396315.5:c.203C&gt;T:p.(Pro68Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396317.5">LILRB1:ENST00000396317.5:c.203C&gt;T:p.(Pro68Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396327.7">LILRB1:ENST00000396327.7:c.203C&gt;T:p.(Pro68Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396331.5">LILRB1:ENST00000396331.5:c.203C&gt;T:p.(Pro68Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396332.8">LILRB1:ENST00000396332.8:c.203C&gt;T:p.(Pro68Leu)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427581.6">LILRB1:ENST00000427581.6:c.311C&gt;T:p.(Pro104Leu)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 0.009624243</dd>
                        
                        <dd>CADD: 0.010 (0.042)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182330">PRAMEF8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.625</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.001</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.625</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.13281806C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1891607">rs1891607</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1891607&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.625</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357367.6">PRAMEF8:ENST00000357367.6:c.990G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000614831.1">PRAMEF8:ENST00000614831.1:c.990G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000621519.4">PRAMEF8:ENST00000621519.4:c.990G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6254585</dd>
                                    
                                    <dd>CADD: 0.625 (4.265)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000127129">EDN2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.027</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=EDN2%0DTAC3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TAC3</a> and phenotypic similarity 0.619 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=432">Normosmic congenital hypogonadotropic hypogonadism</a> associated with TAC3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0000002, Abnormality of body height</dd><dd>HP:0011800, Midface retrusion - HP:0005280, Depressed nasal bridge</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0008527, Congenital sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0000175, Cleft palate</dd><dd>HP:0000218, High palate - HP:0000175, Cleft palate</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=EDN2%0DTAC3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TAC3</a> and phenotypic similarity 0.428 to mouse mutant of TAC3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0001260, increased body weight</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0003550, short perineum</dd><dd>HP:0000218, High palate - MP:0003550, short perineum</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.027</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.41484166C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=41269469">rs41269469</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs41269469&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.027</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372587.5">EDN2:ENST00000372587.5:c.102G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.02702874</dd>
                                    
                                    <dd>CADD: 0.027 (0.119)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0406%</dd>
                                    <dd>UK10K: 0.0793%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0581%</dd>
                                    <dd>ESP All: 0.0461%</dd>
                                    <dd>ExAC AFR: 0.0101%</dd>
                                    <dd>ExAC AMR: 0.0177%</dd>
                                    <dd>ExAC FIN: 0.0342%</dd>
                                    <dd>ExAC NFE: 0.0495%</dd>
                                    <dd>gnomAD_E_AFR: 0.0394%</dd>
                                    <dd>gnomAD_E_AMR: 0.0149%</dd>
                                    <dd>gnomAD_E_FIN: 0.0226%</dd>
                                    <dd>gnomAD_E_NFE: 0.0702%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_NFE: 0.0534%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000018625">ATP1A2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.016</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.411 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=569">Familial or sporadic hemiplegic migraine</a> associated with ATP1A2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment - HP:0000407, Sensorineural hearing impairment</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Phenotypic similarity 0.367 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ATP1A2">ATP1A2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - MP:0001260, increased body weight</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate -</dd><dd>HP:0000218, High palate -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ATP1A2%0DPRKG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKG1</a> and phenotypic similarity 0.643 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=91387">Familial thoracic aortic aneurysm and aortic dissection</a> associated with PRKG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature - HP:0000098, Tall stature</dd><dd>HP:0011800, Midface retrusion - HP:0002705, High, narrow palate</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - HP:0002705, High, narrow palate</dd><dd>HP:0000218, High palate - HP:0002705, High, narrow palate</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://string-db.org/newstring_cgi/show_network_section.pl?identifiers=ATP1A2%0DPRKG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKG1</a> and phenotypic similarity 0.292 to mouse mutant of PRKG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001519, Disproportionate tall stature -</dd><dd>HP:0011800, Midface retrusion -</dd><dd>HP:0008625, Severe sensorineural hearing impairment -</dd><dd>HP:0000545, Myopia -</dd><dd>HP:0000175, Cleft palate - MP:0000477, abnormal intestine morphology</dd><dd>HP:0000218, High palate - MP:0000477, abnormal intestine morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/104290">OMIM:104290</a> Alternating hemiplegia of childhood 1 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/602481">OMIM:602481</a> Migraine, familial hemiplegic, 2 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=569">ORPHA:569</a> Familial or sporadic hemiplegic migraine</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.016</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr1:g.160123263C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=201540165">rs201540165</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs201540165&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.016</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361216.8">ATP1A2:ENST00000361216.8:c.228C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392233.7">ATP1A2:ENST00000392233.7:c.228C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.016441941</dd>
                                    
                                    <dd>CADD: 0.016 (0.072)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000276070">CCL4L2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.471</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.471</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr17:g.36212342C&gt;G [1/1:0/1:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=4796193">rs4796193</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs4796193&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.471</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615418.4">CCL4L2:ENST00000615418.4:c.200C&gt;G:p.(Pro67Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620098.4">CCL4L2:ENST00000620098.4:c.221C&gt;G:p.(Pro74Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613173.4">CCL4L2:ENST00000613173.4:c.126C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617405.5">CCL4L2:ENST00000617405.5:c.231C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000610565.4">CCL4L2:ENST00000610565.4:c.192-22C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620055.4">CCL4L2:ENST00000620055.4:c.77-155C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620250.1">CCL4L2:ENST00000620250.1:c.192-140C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620576.4">CCL4L2:ENST00000620576.4:c.192-155C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000617416.4">CCL4L2:ENST00000617416.4:c.*448C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620732.4">CCL4L2:ENST00000620732.4:c.*46C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.470946</dd>
                                    
                                    <dd>CADD: 0.471 (2.765)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000126890">CTAG2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.401</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            
            
            
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.401</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr23:g.154653499C&gt;T [0/1:0/0:1/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=17855367">rs17855367</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs17855367&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            <div class="col-sm-12">
                                <b>ClinVar: </b>
                                
                                    <span class="label label-success">BENIGN</span>
                                    
                                    
                                    
                                    
                                    
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=706180%5Balleleid%5D">(criteria_provided,_single_submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.401</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000247306.4">CTAG2:ENST00000247306.4:c.17G&gt;A:p.(Arg6Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369585.3">CTAG2:ENST00000369585.3:c.17G&gt;A:p.(Arg6Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4011749</dd>
                                    
                                    <dd>CADD: 0.401 (2.227)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197253">TPSB2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.341</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.341</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr16:g.1228996G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1356110016">rs1356110016</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1356110016&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.341</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000606293.5">TPSB2:ENST00000606293.5:c.567C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000612142.1">TPSB2:ENST00000612142.1:c.588C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.34173614</dd>
                                    
                                    <dd>CADD: 0.342 (1.816)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0095%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000228300">FAM174C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.224</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.224</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.1278863T&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=564786703">rs564786703</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs564786703&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.224</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000469144.5">FAM174C:ENST00000469144.5:c.53T&gt;C:p.(Leu18Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409293.6">FAM174C:ENST00000409293.6:c.*86T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.2255382</dd>
                                    
                                    <dd>Polyphen2: 0.002 (B)</dd>
                                    <dd>CADD: 0.226 (1.110)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AFR: 0.0099%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000261150">EPPK1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.186</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.186</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr8:g.143866876C&gt;T [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=553724064">rs553724064</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs553724064&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.186</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568225.2">EPPK1:ENST00000568225.2:c.6303G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000615648.2">EPPK1:ENST00000615648.2:c.6378G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.18660778</dd>
                                    
                                    <dd>CADD: 0.187 (0.897)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000280411">IGHV1-69D</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.115</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.115</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr14:g.106762382C&gt;G [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1478403490">rs1478403490</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs1478403490&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.115</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000624687.1">IGHV1-69D:ENST00000624687.1:c.63G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.115088165</dd>
                                    
                                    <dd>CADD: 0.115 (0.531)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000177943">MAMDC4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.035</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.035</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                                <b>chr9:g.136856983G&gt;A [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=541764542">rs541764542</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs541764542&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.035</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317446.7">MAMDC4:ENST00000317446.7:c.1914G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445819.5">MAMDC4:ENST00000445819.5:c.2151G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.034616232</dd>
                                    
                                    <dd>CADD: 0.035 (0.153)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0200%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204577">LILRB3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.014</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <!-- TODO: implement comparable on GeneScore th:each="geneScore: ${#lists.sort(gene.getGeneScores(), geneScoreComparator)}">-->
            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                    <div class="panel-heading">
                        <div class="row">
                            <div class="col-sm-3">
                                <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Variant Score: <b>0.014</b>
                                </h4>
                            </div>
                        </div>
                    </div>
                    <div class="panel-body">
                        <div class="row">
                            <div class="col-sm-12">
                                <b>Variants contributing to score:</b>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                                <b>chr19:g.54222365G&gt;C [0/1:0/0:0/1]</b>
                                
                                    <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=138323850">rs138323850</a>
                                    <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs138323850&amp;assm=GCF_000001405.25">
                                        (variation viewer)
                                    </a>
                                    <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                                
                            </div>
                            
                            <div class="col-sm-6">
                                <b>Variant score: 0.014</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245620.13">LILRB3:ENST00000245620.13:c.268C&gt;G:p.(Gln90Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346401.10">LILRB3:ENST00000346401.10:c.268C&gt;G:p.(Gln90Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391750.5">LILRB3:ENST00000391750.5:c.268C&gt;G:p.(Gln90Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.013720512</dd>
                                    
                                    <dd>CADD: 0.014 (0.060)</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>
            
            
            
             <!-- end gene scores -->
            <div class="row">
                <div class="col-sm-12">
                    <b>Other passed variants:</b>
                </div>
            </div>
            
            <div class="row">
                <div class="col-sm-12">
            <span class="label label-danger">MISSENSE_VARIANT</span>
                    <b>chr19:g.54222457T&gt;C [0/1:0/0:0/1]</b>
                    
                        <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=200199363">rs200199363</a>
                        <a href="http://www.ncbi.nlm.nih.gov/variation/view/?q=rs200199363&amp;assm=GCF_000001405.25">
                            (variation viewer)
                        </a>
                        <!--see https://www.ncbi.nlm.nih.gov/variation/view/help/ GRCh38.p2 (RefSeq: GCF_000001405.28)-->
                    
                </div>
                
                <div class="col-sm-6">
                    <b>Variant score: 0.000</b>
                    
                    <dl class="list-unstyled">
                        <dt>Transcripts:</dt>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245620.13">LILRB3:ENST00000245620.13:c.176A&gt;G:p.(Gln59Arg)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346401.10">LILRB3:ENST00000346401.10:c.176A&gt;G:p.(Gln59Arg)</a>
                            
                        </dd>
                        <dd>
                            
                            <a href="http://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391750.5">LILRB3:ENST00000391750.5:c.176A&gt;G:p.(Gln59Arg)</a>
                            
                        </dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Pathogenicity Score: 1.00</b>-->
                    <dl class="list-unstyled">
                        <dt>Pathogenicity Data:</dt>
                        <dd>Best Score: 2.3025274E-4</dd>
                        
                        <dd>CADD: 0.000 (0.001)</dd>
                    </dl>
                </div>
                <div class="col-sm-3">
                    <!--<b>Frequency Score: 0.95</b>-->
                    <dl>
                        <dt>Frequency Data:</dt>
                        <dd>No frequency data</dd>
                        
                    </dl>
                </div>
            </div>
        </div>
    </div>

    

    <div id="about">
        <h3>About</h3>
        <p>
            The Exomizer is a Java program that functionally annotates variants from whole-exome
            sequencing data starting from a VCF file (version 4). The functional annotation code is
            based on <a href="https://github.com/charite/jannovar/">Jannovar</a> and uses
            <a href="http://genome.ucsc.edu/">UCSC</a> KnownGene transcript definitions and
            hg19 genomic coordinates
        </p>
        <p>
            Variants are prioritized according to user-defined criteria on variant frequency, pathogenicity,
            quality, inheritance pattern, and model organism phenotype data. Predicted pathogenicity data
            was extracted from the <a href="http://www.ncbi.nlm.nih.gov/pubmed/21520341">dbNSFP</a> resource.
        </p>
        <p>
            Developed by the Computational Biology and Bioinformatics group at the
            <a href="http://genetik.charite.de/">Institute for Medical Genetics and Human Genetics</a> of the
            <a href="http://www.charite.de">Charit&eacute; - Universit&auml;tsmedizin Berlin</a>, the Mouse
            Informatics Group at the <a href="http://www.sanger.ac.uk/">Sanger Institute</a> and the Smedley group at
            <a href="https://qmul.ac.uk/whri">Queen Mary University of London</a>.
        </p>


        <p>Problems, suggestions, or comments? Please <a href="mailto:peter.robinson@charite.de">let us know</a></p>
    </div>
</div>
</body>
</html>
